# Can I generate a comprehensive restriction enzyme map of the genome in PomBase?
<!-- pombase_categories: Tools and Resources -->

No, but this can be done within Artemis.\
\
Install Artemis (available from
<http://www.sanger.ac.uk/resources/software/artemis/>).\
\
You can then read in the EMBL formatÂ chromosome contig files of sequence
and annotation (available from the [Genome
Datasets](/downloads/genome-datasets#sequences)page). To generate a
restriction map:

1.  Create a new entry using the "Create" menu item "New Entry"
2.  Toggle off the main annotation by un-checking the [
    chromsome]{data-scayt_word="chromsome" data-scaytid="3"}
    [contig]{data-scayt_word="contig" data-scaytid="4"}file (this will
    make your new file "no name" the active entry).
3.  Save your new file with your preferred name.
4.  Use the Create menu option "Mark From Pattern" to create features
    for any restriction patterns of interest and save them into your
    file.
5.  You can add "color" labels to distinguish the different restriction
    sites. See the [Artemis
    FAQ](/faqs/there-equivalent-artemis-java-applet-pombase)and the
    [Artemis
    manual](ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf)(pdf;
    Sanger site) for additional information.


