# Can I view my protein of interest in a multiple alignment with related proteins in other species?
<!-- pombase_categories: Orthology,Tools and Resources -->

You can use Compara via the genome browser to see multiple alignments:

1.  On any gene page, go to the Orthologs section (scroll or use the
    Quick Links box).
2.  Follow the relevant link to Compara - for fungal alignments, choose
    "View orthologs in other fungal species with Compara", or for all
    eukaryotic species choose "View orthologs across taxonomic space
    using pan-species Compara".
3.  You should see a "collapsed" gene tree highlighting your fission
    yeast gene of interest. From here you can click on any node to see a
    menu of options:
    1.  Expand or collapes specific sub-nodes of the tree, or expand the
        tree fully
    2.  View the alignment in FASTA format
    3.  Launch the jalview multiple alignment viewer to see the full
        alignment and colour by residue conservation, hydrophobicity,
        etc.

To configure the protein entries visible in the alignment, select the
most "inclusive" node you require. You can reduce the number of entries
by collapsing individual sub-trees (step 4) before you generate your
alignment.

Information about how the Compara trees are generated, homology types,
and species are available here:
<http://fungi.ensembl.org/info/genome/compara/homology_method.html>

