# How can I retrieve a gene sequence, including upstream and downstream sequences?
<!-- pombase_categories: Gene Page,Genome Browser,Sequence Retrieval -->

You can retrieve sequences from a gene page or in the Genome Browser.

**On the gene page:**Scroll down or click the quick link to the Sequence
section of the page, where there is a set of pre-set one-click options
and a Custom option. For protein-coding genes, there are pre-set options
to retrieve the coding sequence (CDS), CDS + UTRs, CDS + UTRs + introns,
or a translation of the CDS; for non-coding RNA genes only the relevant
options are offered. Under Display Options, you can choose whether to
retrieve plain text or add color highlighting of different regions.

To include flanking sequences, use the Custom Sequence option. Clicking
the View button takes you to a page where you can specify whether to
include UTRs and introns, and how much upstream and downstream sequence
to include. Click the Download button to see the sequence. You can save
by copying and pasting from the browser.

**To use the Genome Browser:**Click the "View in Genome Browser" link
under the map graphic on a gene page, or go directly to the Genome
Browser via the Tools menu, and search for a gene name or systematic ID.
Click Export Data (a button on the left hand side). Select the number of
bases up- and downstream, which strand and the features you would like.
Click "next". Select your download option. Your browser will save or
display the data, depending on which format you select.

To retrieve flanking regions for more than one gene at a time, at
present you must use the Ensembl Genomes Biomart query, as described in
[this
FAQ](/faq/can-i-download-sequences-many-genes-once-including-flanking-regions).

