# How can I see nucleotide-level similarity between *S. pombe* and other *Schizosaccharomyces* species?
<!-- pombase_categories: Orthology,Finding data,Tools and resources -->

PomBase offers two ways to view nucleotide-level similarity between *S.
pombe* and *S. japonicus*, *S. octosporus*, or *S. cryophilus*. Both use
the genome browser.

1.  To view nucleotide similarity data tracks in the browser, follow the
    usual steps as described in the [data track FAQ](/faq/how-can-i-show-or-hide-tracks-in-the-genome-browser). Select the
    data type "Comparative Genomics".
2.  Display syntenic regions as follows:
    1.  Go to your region of interest in the browser (e.g. follow the
        link from a gene page or [use sequence coordinates](/faq/how-can-i-display-a-sequence-region-using-sequence-coordinates-in-the-genome-browser)).
        Make sure the "Location" tab is selected in the horizontal set
        of tabs along the top.
    2.  In the left-hand menu, find the "Comparative Genomics" heading,
        and click on "Region Comparison".
    3.  To select a species for comparison, go to the bottom of the
        left-hand menu, and click the "Select species or regions" link
        (it may appear to be subtly blinking; we apologise for this
        anomaly).
    4.  In the popup, click the "+" beside any species in the
        "Unselected species or regions" list to move it to the "Selected
        species or regions" list. Note: "lastz" is the nucleotide
        alignment algorithm used. Close the popup - click the tick/check
        mark in the upper right corner, or click outside the popup.
    5.  Synteny views will now be visible in the bottom-most graphical
        display (scroll down if necessary). For any region in the *S.
        pombe* genome, pink tracks show the region in the second genome
        with the best nucleotide alignment. Green bands connect the
        best-aligned regions to highlight synteny.


