# Is there an equivalent to the Artemis java applet in PomBase?
<!-- pombase_categories: Locating Genomic Regions,Querying/Searching,Tools and Resources -->

No; at present only the Ensembl genome browser is available via the
PomBase web site. (As of May 2013, we are investigating the possibility
of adding an Artemis applet to PomBase, and will update this FAQ
accordingly when it becomes available.)\
\
If you want to browse the *S. pombe*genome in the Artemis environment,
it is fairly easy to download and run locally:

-   Artemis is available from
    <http://www.sanger.ac.uk/resources/software/artemis/>
-   Read in the chromosome contig files available from the [Genome
    Datasets](/downloads/genome-datasets#sequences)page. (You can read
    in one or more files in a session.)

Once you have loaded the file(s), you can do many different things,
e.g.:

-   Find features by name or ID
-   Find all features of a given type (e.g. see the " [can I find
    transposons](/faqs/how-can-i-find-transposons-s-pombe-genome)" FAQ)
-   Find matches to a specific nucleotide sequence (e.g. see the "
    [restriction enzyme
    map](/faqs/can-i-generate-comprehensive-restriction-enzyme-map-genome-pombase)"
    FAQ)
-   View the nucleotide or amino acid sequence of a region or feature
-   Export selected sequences

See the [Artemis
FAQ](/faqs/there-equivalent-artemis-java-applet-pombase)and the [Artemis
manual](ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf)(pdf;
Sanger site) for additional information.

