# Is there an equivalent to the Artemis java applet in PomBase?
<!-- pombase_categories: Finding data,Tools and resources -->

No; at present only the genome browser is available via the PomBase
web site. (In the future we may investigate the possibility of adding
an Artemis applet to PomBase, and will update this FAQ accordingly
when it becomes available.)

If you want to browse the *S. pombe* genome in the Artemis environment,
it is fairly easy to download and run locally:

-   Artemis is available from
    http://www.sanger.ac.uk/science/tools/artemis
-   Read in the chromosome contig files available from the [Genome sequences](/downloads/genome-datasets) page. (You can read
    in one or more files in a session.)

Once you have loaded the file(s), you can do many different things,
e.g.:

-   Find features by name or ID
-   Find all features of a given type (e.g. see the "[can I find transposons](/faq/how-can-i-find-transposons-s.-pombe-genome)" FAQ)
-   Find matches to a specific nucleotide sequence (e.g. see the 
    "[restriction enzyme map](/faq/can-i-generate-comprehensive-restriction-enzyme-map-genome-pombase)" FAQ)
-   View the nucleotide or amino acid sequence of a region or feature
-   Export selected sequences

See the [Artemis manual](ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf) (pdf;
Sanger site) for additional information.

