<div class="recent-news">
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/dropdown_menu.png"/>
<h4 id="new-fission-yeast-go-slims"><strong>New fission yeast GO slims</strong></h4>
<p><em>2020-05-20</em></p>
<p>To complement the overview provided by the <a href="/browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO biological process slim</a>, we have created GO slims for the <a href="/browse-curation/fission-yeast-mf-go-slim-terms">molecular function</a> and <a routerLink="/browse-curation/fission-yeast-cc-go-slim-terms">cellular component</a> branches of GO. Each slim page provides links to ontology term pages, annotated genes, and to download slim terms and IDs from the PomBase ftp site.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/dropdown_menu.png"/>
<h4 id="new-ontology-slimming-options-for-advanced-search-results"><strong>New ontology slimming options for advanced search results</strong></h4>
<p><em>2020-05-20</em></p>
<p>The PomBase <a routerLink="/query">advanced search</a> results panel now allows you to retrieve annotations to any of the fission yeast <a href="documentation/pombase-go-slim-documentation">GO slims</a> or the <a href="https://www.pombase.org/browse-curation/disease-slim">Mondo disease slim</a> for genes in the results list. For example, you can query for all genes involved in a process and slim the resulting list by molecular function or disease association.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/human disease.png"/>
<h4 id="pombase-adopts-mondo-for-disease-gene-curation"><strong>PomBase adopts MONDO for disease gene curation</strong></h4>
<p><em>2020-05-18</em></p>
<p>PomBase has switched from the <a href="https://disease-ontology.org/">Disease Ontology</a> (DO) to the Monarch Initiative’s <a href="https://mondo.monarchinitiative.org/">Mondo Disease Ontology</a> (Mondo) for disease gene curation. Mondo covers the same set of disease descriptions as DO, but has a richer hierarchical structure that classifies more specific descriptions into broad categories (e.g. anemia, cancer, kidney disease) suitable for a disease “slim” term set.</p>
<p>PomBase curators are collaborating with Mondo to improve its disease classification, especially in areas that will support inferences that improve fission yeast disease annotation coverage in the new <a href="https://www.pombase.org/browse-curation/disease-slim">PomBase Mondo slim</a>. The new disease slim is a work in progress, so if there is a particular disease grouping that you would find useful, please <a href="mailto:helpdesk@pombase.org">let us know</a>.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/human disease.png"/>
<h4 id="improved-gene-disease-curation"><strong>Improved gene-disease curation</strong></h4>
<p><em>2020-05-18</em></p>
<p>PomBase disease gene curation associates disease descriptors with fission yeast orthologs of human disease-causing genes. We have now increased coverage by adding new gene–disease term connections, with 3954 individual annotations to 1195 genes (up from 2588 and 905 respectively in January 2019). Disease associations now cover 24.5% of all fission yeast protein-coding genes, and over one third of those with human orthologs.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/canto.png"/>
<h4 id="published-community-curation-in-pombase"><strong>Published: Community curation in PomBase</strong></h4>
<p><em>2020-05-11</em></p>
<p>The PomBase team has published an overview of our experience with community curation for fission yeast. In the article, out this week in <a href="https://academic.oup.com/database/article/doi/10.1093/database/baaa028/5827230">Database</a>, we reflect on the factors that have made our community’s remarkable, standard-setting achievements possible, and on the benefits we and PomBase users derive from this effort. We highlight the collaboration between authors and professional curators that arises via community curation, and how annotation quality improves as a result.</p>
<p>Watch for invitations to curate your new papers, or see our <a href="https://www.pombase.org/community/fission-yeast-community-curation-project">community curation page</a> for more information.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/>
<h4 id="pombetalks-may-13th"><strong>PombeTalks May 13th</strong></h4>
<p><em>2020-05-07</em></p>
<p>The next online PombeTalks will take place on Wednesday 13 May 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Sarah Lambert</strong>, Institut Curie, Paris, France: <em>Resolution of replication stress in space and time for maintaining genome stability</em></p></li>
<li><p><strong>Cornelia Kilchert</strong>, Justus-Liebig-University, Giessen, Germany: <em>RNA-binding proteins in fission yeast - a global perspective</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also mark your calendars for the next two sessions on <strong>May 27</strong> and <strong>June 10</strong>, and if you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/nucleosome.png"/>
<h4 id="chromatin-silencing-ontology-annotation-overhaul"><strong>Chromatin silencing ontology &amp; annotation overhaul</strong></h4>
<p><em>2020-05-07</em></p>
<p>PomBase curators have collaborated with the GO Consortium to improve the representation of chromatin silencing and the underlying heterochromatin organization processes in the GO biological process ontology and annotations.</p>
<p>Notably, “chromatin silencing” terms have been removed from GO on the grounds that they conflated various heterochromatin assembly, formation, and maintenance pathways with processes that affect chromatin-mediated repression more indirectly (e.g. tethering to the nuclear envelope). Chromatin silencing is a phenotype resulting from the cumulative effects of these processes, and the Fission Yeast Phenotype Ontology (FYPO) accordingly retains a full suite of “chromatin silencing” terms.</p>
<p>Annotations using the GO chromatin silencing terms were reviewed, and either removed or reannotated based on what could be inferred from the phenotypes, resulting in a substantially revised set of <a href="https://www.pombase.org/term/GO:0031507">heterochromatin assembly annotations</a>. Further work is required, so please <a href="mailto:helpdesk@pombase.org">send us</a> any corrections.</p>

</div>

<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

