<div class="recent-news">
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="genetic-interaction-annotation-model-updated"><strong>Genetic Interaction annotation model updated</strong></h4>
<p><em>2022-11-29</em></p>
<p>We have recently implemented an improved way to <strong>annotate and display genetic interactions so they are linked to phenotype annotations and alleles</strong>.</p>
<p>Previously, our Genetic Interaction annotations only mentioned the interacting genes and the type of genetic interaction. For example, if a strain with genotype <code>asp1-H397A</code> has the phenotype <code>decreased acid phosphatase activity affecting activity of pho1</code>, but this phenotype is suppressed when <code>rhn1</code> is deleted in that strain, we would annotate that the genes <code>asp1</code> and <code>rhn1</code> are part of a <code>Phenotypic Suppression</code> interaction.</p>
<figure>
<img src="assets/newsfeed/gi-update-1.png" style="width:100.0%" alt="GI old format" />
<figcaption aria-hidden="true">GI old format</figcaption>
</figure>
<p>We continue to display interactions in this format by default (showing only genes and interaction type), but if you expand the annotation, you can view the associated genotypes and phenotypes.</p>
<figure>
<img src="assets/newsfeed/gi-update-2.png" style="width:100.0%" alt="GI new format" />
<figcaption aria-hidden="true">GI new format</figcaption>
</figure>
<p>In the revised Canto interface, you can only annotate a genetic interaction from the double mutant phenotype annotation (by clicking on <code>add..</code>, as shown below).</p>
<figure>
<img src="assets/newsfeed/gi-update-3.png" style="width:100.0%" alt="GI annotation in Canto" />
<figcaption aria-hidden="true">GI annotation in Canto</figcaption>
</figure>
<p>Of course, genetic interactions predating this update are not linked to phenotypes or genotypes, but we are hoping to auto-annotate several of those. We will also prioritise for update any interactions where community curators have provided these details in an annotation comment. Finally, <strong>a big shoutout to Ana Sanchez and Angad Garg from the Shuman lab</strong>, for testing the new interface in numerous recently curated publications. The examples provided here are from <a routerLink="/reference/PMID:31276588">Sanchez et al. 2019</a>. Go read it and see the annotations in PomBase.</p>
<p>Best, The PomBase team</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/protein-structure.png"/>
<h4 id="using-alphafold-models-to-discover-distant-human-and-budding-yeast-homologues"><strong>Using AlphaFold models to discover distant human and budding yeast homologues</strong></h4>
<p><em>2022-10-14</em></p>
<p>We collaborated with the Pfam group at the EBI to evaluate predictions generated from AlphaFold reciprocal best structure hits to identify potential distant orthologs. The Reciprocal Best Structure Hits (RBSH) approach provided 11 novel human homologues, including <strong>Pho86 -&gt; NAT8</strong> (ER acetyltransferase), <strong>Mug174 -&gt; COIL</strong> (Coilin), <strong>Ach1 -&gt; OXCT1</strong> (succinyl-CoA:3-ketoacid coenzyme A transferase), <strong>SPAC1952.08c -&gt; CREG1</strong>, <strong>imt1 -&gt; A4GALT</strong> (Lactosylceramide 4-alpha-galactosyltransferase), <strong>Rtc5 -&gt; MEAK7</strong> (MTOR associated protein). A further 41 novel orthologs were predicted between <em>S. pombe</em> and <em>S. cerevisiae</em> which had fallen under the radar for all other methods used at PomBase. Most of the novel connections provided additional functional information, or supported existing knowledge for poorly characterised proteins. See supporting data tables S4 and S5 for the
complete list of predictions included in PomBase. <a href="https://academic.oup.com/bioinformaticsadvances/advance-article/doi/10.1093/bioadv/vbac072/6749558">Article</a>.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-to-fission-yeast-check-out-our-quick-start-guide-for-new-users"><strong>New to fission yeast? check out our quick start guide for new users</strong></h4>
<p><em>2022-10-13</em></p>
<p>We have added a new page providing useful information for new fission yeast researchers:<br />
<a routerLink="/documentation/getting-started">Getting started with <em>S.pombe</em> and PomBase</a></p>
<p>Includes details of how to join the community mailing list and the Slack channel, links to useful tools and resources, information about fission yeast as a model organism, an overview of PomBase and more.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/human_disease.png"/>
<h4 id="new-disease-gene-associations"><strong>77 new disease gene associations</strong></h4>
<p><em>2022-09-16</em></p>
<p>We have added 77 new disease-gene associations for 71 fission yeast human gene orthologs. These were identified using “PombeMine” to identify the disease genes curated by OMIM not annotated with an existing MONDO mapping. The number of human disease gene associations is currently 1471. Disease genes can be browsed via the <a routerLink="/browse-curation/disease-slim">disease slim set</a> or from the <a routerLink="/term/MONDO:0000001">MONDO root node term</a>.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/intermine.png"/>
<h4 id="pombemine-an-intermine-instance-for-s.-pombe"><strong>PombeMine: an InterMine instance for <em>S. pombe</em></strong></h4>
<p><em>2022-09-16</em></p>
<p>As part of an Elixir funded collaboration with the <a href="http://intermine.org/">InterMine team</a> we have created <a href="https://www.pombase.org/im/pombemine">PombeMine</a>. Gene lists can be sent directly from PomBase query results pages directly to Intermine (under the “export” tab), providing direct (2 click) access to GO and phenotype enrichment tools.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="protein-sequence-changes"><strong>Protein sequence changes</strong></h4>
<p><em>2022-09-06</em></p>
<p>78 protein have been shortened (at the N-term). This set includes 34 published proteins (Apl4, Brc1, Cdc48, Cdt1, Cho1, Cmb1, Cut2, Cut6, Cwf26, Dbr1, Dri1, Elo2, Eri1, Lsd2, Lys2, Med13, Naa38, Nup107, Nup82, Orc2, Pof10, Ppt1, Rec24, Rga2, Rmt3, Rns1, RRpn7, Sap145, Skb1, Snf5, Snt2, Spn3, Tpp2, Trm1, and Tup12). Allele description changes and modification position changes are pending.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-phenotype-slim"><strong>New phenotype slim</strong></h4>
<p><em>2022-04-13</em></p>
<p>We have added a <a routerLink="/browse-curation/fypo-slim">phenotype slim overview</a> to complement those provided for <a routerLink="/browse-curation/disease-slim">disease association</a>, <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">biological process</a>, <a routerLink="/browse-curation/fission-yeast-mf-go-slim-terms">molecular function</a> and <a routerLink="/browse-curation/fission-yeast-cc-go-slim-terms">cellular component</a> annotation. The purpose of the phenotype slim is to provide subsets of commonly used ‘broad’ phenotypic classes or annotation subsets that can provide a useful starting point for accessing phenotype lists. The phenotype slim page provides links to ontology term pages, annotated genes, and to download files containing the slim terms and IDs.</p>
<p>The phenotype slim has also been added to the PomBase <a routerLink="/query">advanced search</a> results panel “Slim with” menu. For example, you can query for all genes involved in a GO process, another phenotype, or any other list, and “slim” the results using the “Phenotype slim” option to view categories of phenotypes assigned to the list.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="making-biological-knowledge-useful-for-humans-and-machines"><strong>Making biological knowledge useful for humans and machines</strong></h4>
<p><em>2022-04-04</em></p>
<p>A <a href="https://academic.oup.com/genetics/issue/220/4">GENETICS special issue</a> featuring model organism database updates is published today. This issue features the recent PomBase and <a href="https://www.japonicusdb.org/">JaponicusDB</a> publications and is accompanied by an editorial <a href="https://academic.oup.com/genetics/article-abstract/220/4/iyac001/6563297">“Making biological knowledge useful for humans and machines”</a> co-authored by Val Wood, Paul Sternberg and Howard Lipshitz.</p>
<ul>
<li><a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab222/6481557?login=false">Fission stories: using PomBase to understand Schizosaccharomyces pombe biology</a></li>
<li><a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab223/6481558?login=false">JaponicusDB: rapid deployment of a model organism database for an emerging model species</a></li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="the-fission-yeast-community-have-now-curated-over-1000-publications"><strong>The fission yeast community have now curated over 1000 publications</strong></h4>
<p><em>2022-03-26</em></p>
<p>We would like to extend a huge “thank you” to the fission yeast community for curation contributions. The community have now curated 1008 publications providing 19,156 independent annotations, representing 25% of the curation from small-scale publications. In addition, another 80,000 annotations have been provided via the submission of HTP datasets.</p>
<p>Please contact us <a href="mailto:helpdesk@pombase.org?subject=New%20manuscript%20for%20curation">via the helpdesk</a> if you would like to provide curation for your manuscript but don’t know how.</p>
<p>Links:</p>
<ul>
<li><a href="https://www.pombase.org/reference_list/community">All community curated publications</a></li>
<li><a href="https://www.pombase.org/archive/spotlight">Spotlights</a></li>
<li><a href="https://curation.pombase.org/pombe">Canto curation tool</a></li>
<li><a href="https://www.pombase.org/submit-data">HTP data and browser track submission</a></li>
</ul>

</div>

<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

