<div class="recent-news">
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/elixir_uk_thumb.png"/>
<h4 id="pombase-is-now-an-elixir-node-service"><strong>PomBase is now an ELIXIR Node Service</strong></h4>
<p><em>2020-01-17</em></p>
<p>PomBase has been awarded Node Service status by the <a href="https://elixiruknode.org/">UK node</a> of <a href="https://elixir-europe.org/">ELIXIR</a>. ELIXIR-UK Node Services support the bioinformatics and broader biological research communities by providing training and resources that help researchers to find and share data, exchange expertise, and agree on best practices at national, European and international levels. The review panel describes PomBase as a “mature, leading model organism database which is popular, unique, well used, and has a strong user community.”</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/stable_pAde6_vector.png"/>
<h4 id="new-vectors-for-simple-reliable-s.-pombe-molecular-biology"><strong>New vectors for simple, reliable <em>S. pombe</em> molecular biology</strong></h4>
<p><em>2020-01-07</em></p>
<p>To enable fission yeast researchers to manipulate <em>S. pombe</em> molecular biology reproducibly and easily, Aleks Vještica and Magdalena Marek in Sophie Martin’s lab have designed and constructed a series of simple, fully characterized plasmids.</p>
<p>The Stable Integration Vector (SIV) series provides a highly modular toolbox to introduce heterologous sequences more stably was possible with than previously available vectors. The toolkit includes antibiotic resistance markers, promoters, fluorescent tags, and terminators, as well as large set of ready-to-use fluorescent probes to mark organelles and visualize cellular processes.</p>
<p>The work is published in the <a href="https://jcs.biologists.org/content/early/2019/11/29/jcs.240754.long">Journal of Cell Science</a>, and a PomBase <a href="https://www.pombase.org/reference/PMID:31801797">publication page</a> is available.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/>
<h4 id="unique-permanent-urls-for-search-results"><strong>Unique permanent URLs for search results</strong></h4>
<p><em>2019-11-28</em></p>
<p>All result pages from the <a routerLink="/query">Advanced search</a> now have a unique permanent URL that can be bookmarked and shared with your colleagues.</p>
<p>The <a routerLink="/documentation/quick-little-tool">QuiLT</a> and <a routerLink="/browse-curation/fission-yeast-go-slim-terms">GO slim</a> pages also now have permanent URLs.</p>
<p>Examples:</p>
<ul>
<li><a routerLink="/results/from/id/7474e390-ddc0-4f70-8c2e-d6a69d471a50">All disease associated genes</a></li>
<li><a routerLink="/vis/from/id/7474e390-ddc0-4f70-8c2e-d6a69d471a50">QuiLT example for all disease associated genes</a></li>
<li><a routerLink="/slim:bp_goslim_pombe/from/id/7474e390-ddc0-4f70-8c2e-d6a69d471a50">GO slim for disease associated genes</a></li>
</ul>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/new_tracks.png"/>
<h4 id="browser-tracks-now-loadable-from-publication-pages"><strong>Browser tracks now loadable from publication pages</strong></h4>
<p><em>2019-10-16</em></p>
<p>Data tracks from datasets hosted in the PomBase genome browser can now be browsed and loaded from their respective publication pages. For an example, see <a href="https://www.pombase.org/reference/PMID:29914874">Atkinson et al. (2018)</a>. Data tracks are now also downloadable from the publication pages.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-pombase-funding-from-the-wellcome-trust"><strong>New PomBase funding from the Wellcome Trust</strong></h4>
<p><em>2019-10-07</em></p>
<p>We are pleased to announce that the recent PomBase application for continued Wellcome Trust funding was successful. Although the grant was not fully funded, we are confident that we can cover the shortfall by small grants for stand-alone projects and collaborations. We would like to thank the pombe community for their support with the application, and the Wellcome Trust for their continued funding. We look forward to supporting your research until 2025 (and beyond).</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/>
<h4 id="pombase-now-uses-interpro-version-76.0"><strong>PomBase now uses InterPro Version 76.0</strong></h4>
<p><em>2019-10-06</em></p>
<p>PomBase now uses InterPro Version 76.0, which integrates 277 new methods from the CATH-Gene3D (1), PANTHER (178) and CDD (98) databases. InterPro cites 59846 publications in PubMed. See the InterPro <a href="https://www.ebi.ac.uk/interpro/release_notes/">release notes</a> for further information.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/>
<h4 id="replication-origin-data-loaded-into-jbrowse"><strong>Replication origin data loaded into JBrowse</strong></h4>
<p><em>2019-08-30</em></p>
<p>We have loaded data from: <a href="https://www.pombase.org/reference/PMID:14566325">Segurado et al. (2003) “A+T-rich islands”</a>, <a href="https://www.pombase.org/reference/PMID:17304213">Hayashi et al. (2007) “Pre-replicative complex localization; early and late firing origins”</a>, and <a href="https://www.pombase.org/reference/PMID:18093330">Mickle et al. (2007) “Replication origins with functional classification”</a>.</p>
<p>To view the tracks, either follow the hyperlinks above to the respective PomBase publication pages, and click on the “view” link after “Datasets from this publication are available in the PomBase JBrowse genome browser”, or go directly to <a href="http://www.pombase.org/jbrowse/">the browser</a> and click on the “select tracks” button to find the tracks manually.</p>
<p>For anyone new to JBrowse we have a <a href="https://www.pombase.org/documentation/JBrowse_quick_start">quick start guide</a>.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/slack_icon-32px.png"/>
<h4 id="join-the-conversations-on-slack"><strong>Join the conversations on Slack</strong></h4>
<p><em>2019-07-19</em></p>
<p>The vibrant fission yeast community now has a Slack channel. Slack provides a forum for the research community. Follow conversations you care about, message colleagues privately, or in groups, ask questions, post responses. All archived and searchable.</p>
<ul>
<li>See the <a href="/assets/external_media/PombeSlack-flyer.pdf">PombeSlack flyer</a> for more information, including how to join.</li>
</ul>
<!-- included with original news item; commented out after expiry:
 - <a href="https://bit.ly/2XYkcEi">Join PombeSlack</a> -->

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/>
<h4 id="fitness-landscape-of-the-fission-yeast-genome-dataset-loaded-into-jbrowse"><strong>“Fitness Landscape of the Fission Yeast Genome” dataset loaded into JBrowse</strong></h4>
<p><em>2019-07-11</em></p>
<p>We have loaded the <a href="https://www.pombase.org/reference/PMID:31077324">Grech et al. (2019) “Fitness Landscape of the Fission Yeast Genome”</a> dataset into JBrowse. In this study, transposon mutagenesis libraries were created to map transposon insertion sites in the S. pombe genome. From this data, functional elements of the genome were inferred. The tracks from this study can be loaded by a single click from the linked publication page above</p>
<p>Thanks Dan Jeffares for sending us the data.</p>
<p>For anyone new to JBrowse we have a <a href="https://www.pombase.org/documentation/JBrowse_quick_start">quick start guide</a>.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/rna_metabolism_32px.png"/>
<h4 id="trna-metabolism-go-annotation-update"><strong>tRNA metabolism GO annotation update</strong></h4>
<p><em>2019-05-19</em></p>
<p>The process of tRNA metabolism, and the associated molecular functions have recently been reviewed.</p>
<ul>
<li><a href="https://www.pombase.org/term/GO:0006399">“tRNA metabolic process” annotation</a></li>
<li><a href="https://www.pombase.org/term_genes/GO:0006399">genes for “tRNA metabolic process”</a></li>
</ul>
<p>Please let us know if the annotation can be further improved.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/>
<h4 id="customisable-fasta-download"><strong>Customisable FASTA download</strong></h4>
<p><em>2019-04-18</em></p>
<p>You can now download nucleotide or peptide sequences for genes in Advanced search results in FASTA format, and customise what is included in the FASTA headers (e.g. gene names, product descriptions, sequence coordinates, or various IDs can be included).</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/tree_for_news.png"/>
<h4 id="new-homology-modelling-and-ortholog-links-on-gene-pages"><strong>New homology modelling and ortholog links on gene pages</strong></h4>
<p><em>2019-04-18</em></p>
<p>We have added new external links to PomBase gene pages for structure and ortholog predictions:</p>
<ul>
<li><p>Protein-specific links to <a href="https://swissmodel.expasy.org/">SWISS-MODEL</a>, a fully automated protein structure homology-modelling server, accessible via the ExPASy web server, lead to a SWISS-MODEL Repository page for each sequence and present results. If no structure or model is available, you can either trigger adding an entry to the repository with a single click or easily interactively search for templates and build models in your own SWISS-MODEL workspace.</p></li>
<li><p>Ensembl Fungi Compara and Ensembl Pan-taxonomic Compara links lead to orthology predictions from the Ensembl Compara pipeline for fungi and all species, respectively.</p></li>
<li><p><a href="http://www.pantherdb.org/">PANTHER</a> links retrieve gene information, classification, and predicted orthologs.</p></li>
</ul>

</div>

<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

