<div class="recent-news">
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/human_disease.png"/>
<h4 id="human-disease-association-changes---monarch"><strong>Human disease association changes - Monarch</strong></h4>
<p><em>2024-06-21</em></p>
<p>We have changed our major source of human disease associations from <a href="https://www.malacards.org/">MalaCards</a> to <a href="https://monarchinitiative.org/">Monarch</a>.</p>
<p>This change has:</p>
<ol type="1">
<li>Increased the number of fission yeast genes with disease associations from 1463 to 1551, made the associations more accurate and up to date</li>
<li>Allowed us to automate the inclusion of new disease gene imports from Monarch based on our ortholog mappings</li>
<li>Enabled us to add the qualifiers “causal” or “correlated with” (for multi-genic disorders) to the disease association (you can see this on the gene pages under “show details”)</li>
</ol>
<p>Disease associations in PomBase:</p>
<ul>
<li><a routerLink="/term/MONDO:0700096">Browse the new disease list</a></li>
<li><a routerLink="/browse-curation/disease-slim">Overview of disease associations (disease slim)</a></li>
</ul>
<p>If you have knowledge of a specific disease class or notice any issues with the ontology you can <a href="https://github.com/monarch-initiative/mondo/issues/">report this to Monarch</a></p>
<p>The Monarch initiative also host our fission yeast phenotype associations in their <a href="https://monarchinitiative.org/">knowledge graph</a>.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/>
<h4 id="pombase-now-includes-data-from-interpro-v100.0"><strong>PomBase now includes data from InterPro v100.0</strong></h4>
<p><em>2024-06-14</em></p>
<p>PomBase now uses <a href="https://www.ebi.ac.uk/interpro/">InterPro</a> <a href="https://www.ebi.ac.uk/interpro/release_notes/">release 100.0</a>.</p>
<p>Features include:</p>
<ul>
<li>The addition of 570 InterPro entries.</li>
<li>An update to Pfam [37.0].</li>
<li>Integration of 582 new methods from the NCBIfam (26), Pfam (545), PANTHER (3), CDD (8) databases.</li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-database-update-publication-from-pombase"><strong>New database update publication from PomBase</strong></h4>
<p><em>2024-02-22</em></p>
<p>The fission yeast 2024 database update “PomBase: a Global Core Biodata Resource—growth, collaboration, and sustainability” has been <a href="https://doi.org/10.1093/genetics/iyae007">published in <em>Genetics</em></a>. This update describes PomBase’s continuing growth and improvement over the past two years and showcases the depth of existing connections both within the biocuration ecosystem and between PomBase and its user community.</p>
<p>We discuss ways to track knowledge coverage including the tracking of unknown proteins, and the use of slimsets. We describe how we provide value by data and software sharing and how we have reused software provided by others in our new views. We provide author-led curation updates, describe contributions to global standards, future plans to support machine learning with high quality curated data and more.</p>
<p><a href="https://doi.org/10.1093/genetics/iyae007">Rutherford <em>et al.</em>, <em>Genetics</em>, iyae007</a></p>
<p><a href="assets/newsfeed/genetics_2024_paper_fig_5.png"><img src="assets/newsfeed/genetics_2024_paper_fig_5.png" title="Data dissemination from PomBase to other resources" class="screenshot" loading="lazy" style="width:100.0%" alt="“Fig 5 Data dissemination from PomBase to other resources”" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="viewing-protein-modifications"><strong>Viewing protein modifications</strong></h4>
<p><em>2024-02-08</em></p>
<p>Modified amino acids in the protein feature viewer are now coloured by modification type. See the <a routerLink="/gene_protein_features/SPAC1834.04">histone H3.1 gene page</a> for an example.</p>
<p><a href="assets/newsfeed/gene-hht1-protein-feature-viewer-page.png"><img src="assets/newsfeed/gene-hht1-protein-feature-viewer-page.png" class="screenshot" loading="lazy" style="width:100.0%" alt="Protein feature view for hht1" /></a></p>
<p><a href="assets/newsfeed/protein-feature-modification-key.png"><img src="assets/newsfeed/protein-feature-modification-key.png" class="screenshot" loading="lazy" style="width:100.0%" alt="Protein feature view modification colour key" /></a></p>
<p>Visit the <a routerLink="/documentation/gene-page-protein-features-widget">protein feature view documentation page</a> for more information.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="access-target-genes-of-individual-gene-products"><strong>Access target genes of individual gene products</strong></h4>
<p><em>2024-02-07</em></p>
<p>It is now possible to send lists of annotated molecular function substrates to the advanced search from gene pages:</p>
<p><a href="assets/newsfeed/cdc2-substrates-to-advanced-search.png"><img src="assets/newsfeed/cdc2-substrates-to-advanced-search.png" class="screenshot" loading="lazy" style="width:100.0%" alt="cdc2 - send target genes to the Advanced Search" /></a></p>
<p>Lists of annotated molecular function substrates of individual genes (i.e protein kinase or transcription factor target genes) can now be sent directly to the Advanced Search. You can send complete lists, or pre-filtered by cell cycle phase.</p>

</div>

<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

