<div class="recent-news">
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="revised-canonical-5-utrs"><strong>Revised canonical 5’ UTRs</strong></h4>
<p><em>2023-01-24</em></p>
<p>We have revised the curated 5’UTRs using Transcription Start Sites (TSS) data (in vegetative growth/ minimal media) from the Cap Analysis of Gene Expression (CAGE) data provided by <a routerLink="/reference/PMID:30566651">Thodberg et al</a>.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/gbc-news-icon.png"/>
<h4 id="global-core-biodata-resource-status-for-pombase"><strong>Global Core Biodata Resource status for PomBase</strong></h4>
<p><em>2022-12-15</em></p>
<p>We are very pleased to announce that PomBase has been selected as one of the first <a href="https://globalbiodata.org/scientific-activities/global-core-biodata-resources">Global Core Biodata Resource (GCBR)</a> — a collection of 37 resources whose long term funding and sustainability is critical to life science and biomedical research worldwide. This accreditation recognizes PomBase as a primary knowledge base (adding value to data through expert curation) and as a crucial component of the research ecosystem. The candidate biodata resources were assessed against a series of rigorous criteria that included their scientific focus, the size and reach of their user communities, their quality of service, their governance, and their impact on global research.</p>
<p>Thank you to the entire community, especially the community curators who contribute regularly to our content, and our <a routerLink="/about/pombase-sab">Scientific Advisory Board</a> for their help and support.</p>
<p>For more information about the <a href="https://globalbiodata.org/">Global Biodata Coalition</a> and PomBase’s new status, see <a href="https://globalbiodata.org/global-biodata-coalition-announces-the-first-set-of-global-core-biodata-resources/">the full press release</a>.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="querying-by-rna-length-in-the-advanced-search-tool"><strong>Querying by RNA length in the Advanced Search tool</strong></h4>
<p><em>2022-12-07</em></p>
<p>You can now query the RNA length of genes (spliced or unspliced) under the “Transcripts and exons” query grouping in the <a href="https://www.pombase.org/query">Advanced search</a>.</p>
<p>You can also add RNA sequence length as a field in tables downloaded from the query builder.</p>
<figure>
<img src="assets/newsfeed/advanced-search-rna-length-query.png" style="width:100.0%" alt="RNA length queries are available under the “Transcripts and exons” tab" />
<figcaption aria-hidden="true">RNA length queries are available under the “Transcripts and exons” tab</figcaption>
</figure>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="genetic-interaction-annotation-model-updated"><strong>Genetic Interaction annotation model updated</strong></h4>
<p><em>2022-11-29</em></p>
<p>We have recently implemented an improved way to <strong>annotate and display genetic interactions so they are linked to phenotype annotations and alleles</strong>.</p>
<p>Previously, our Genetic Interaction annotations only mentioned the interacting genes and the type of genetic interaction. For example, if a strain with genotype <code>asp1-H397A</code> has the phenotype <code>decreased acid phosphatase activity affecting activity of pho1</code>, but this phenotype is suppressed when <code>rhn1</code> is deleted in that strain, we would annotate that the genes <code>asp1</code> and <code>rhn1</code> are part of a <code>Phenotypic Suppression</code> interaction.</p>
<figure>
<img src="assets/newsfeed/gi-update-1.png" style="width:100.0%" alt="GI old format" />
<figcaption aria-hidden="true">GI old format</figcaption>
</figure>
<p>We continue to display interactions in this format by default (showing only genes and interaction type), but if you expand the annotation, you can view the associated genotypes and phenotypes.</p>
<figure>
<img src="assets/newsfeed/gi-update-2.png" style="width:100.0%" alt="GI new format" />
<figcaption aria-hidden="true">GI new format</figcaption>
</figure>
<p>In the revised Canto interface, you can only annotate a genetic interaction from the double mutant phenotype annotation (by clicking on <code>add..</code>, as shown below).</p>
<figure>
<img src="assets/newsfeed/gi-update-3.png" style="width:100.0%" alt="GI annotation in Canto" />
<figcaption aria-hidden="true">GI annotation in Canto</figcaption>
</figure>
<p>Of course, genetic interactions predating this update are not linked to phenotypes or genotypes, but we are hoping to auto-annotate several of those. We will also prioritise for update any interactions where community curators have provided these details in an annotation comment. Finally, <strong>a big shoutout to Ana Sanchez and Angad Garg from the Shuman lab</strong>, for testing the new interface in numerous recently curated publications. The examples provided here are from <a routerLink="/reference/PMID:31276588">Sanchez et al. 2019</a>. Go read it and see the annotations in PomBase.</p>
<p>Best, The PomBase team</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/protein-structure.png"/>
<h4 id="using-alphafold-models-to-discover-distant-human-and-budding-yeast-homologues"><strong>Using AlphaFold models to discover distant human and budding yeast homologues</strong></h4>
<p><em>2022-10-14</em></p>
<p>We collaborated with the Pfam group at the EBI to evaluate predictions generated from AlphaFold reciprocal best structure hits to identify potential distant orthologs. The Reciprocal Best Structure Hits (RBSH) approach provided 11 novel human homologues, including <strong>Pho86 -&gt; NAT8</strong> (ER acetyltransferase), <strong>Mug174 -&gt; COIL</strong> (Coilin), <strong>Ach1 -&gt; OXCT1</strong> (succinyl-CoA:3-ketoacid coenzyme A transferase), <strong>SPAC1952.08c -&gt; CREG1</strong>, <strong>imt1 -&gt; A4GALT</strong> (Lactosylceramide 4-alpha-galactosyltransferase), <strong>Rtc5 -&gt; MEAK7</strong> (MTOR associated protein). A further 41 novel orthologs were predicted between <em>S. pombe</em> and <em>S. cerevisiae</em> which had fallen under the radar for all other methods used at PomBase. Most of the novel connections provided additional functional information, or supported existing knowledge for poorly characterised proteins. See supporting data tables S4 and S5 for the
complete list of predictions included in PomBase. <a href="https://academic.oup.com/bioinformaticsadvances/advance-article/doi/10.1093/bioadv/vbac072/6749558">Article</a>.</p>

</div>

<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

