<div class="recent-news">
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/>
<h4 id="customisable-fasta-download"><strong>Customisable FASTA download</strong></h4>
<p><em>2019-04-18</em></p>
<p>You can now download nucleotide or peptide sequences for genes in Advanced search results in FASTA format, and customise what is included in the FASTA headers (e.g. gene names, product descriptions, sequence coordinates, or various IDs can be included).</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/tree_for_news.png"/>
<h4 id="new-homology-modelling-and-ortholog-links-on-gene-pages"><strong>New homology modelling and ortholog links on gene pages</strong></h4>
<p><em>2019-04-18</em></p>
<p>We have added new external links to PomBase gene pages for structure and ortholog predictions:</p>
<ul>
<li><p>Protein-specific links to <a href="https://swissmodel.expasy.org/">SWISS-MODEL</a>, a fully automated protein structure homology-modelling server, accessible via the ExPASy web server, lead to a SWISS-MODEL Repository page for each sequence and present results. If no structure or model is available, you can either trigger adding an entry to the repository with a single click or easily interactively search for templates and build models in your own SWISS-MODEL workspace.</p></li>
<li><p>Ensembl Fungi Compara and Ensembl Pan-taxonomic Compara links lead to orthology predictions from the Ensembl Compara pipeline for fungi and all species, respectively.</p></li>
<li><p><a href="http://www.pantherdb.org/">PANTHER</a> links retrieve gene information, classification, and predicted orthologs.</p></li>
</ul>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/>
<h4 id="pombase-interpro-update"><strong>PomBase InterPro Update</strong></h4>
<p><em>2019-04-17</em></p>
<p>PomBase now uses InterPro Version 73.0, which integrates 1,531 new methods from the CATH-Gene3D (122), CDD (330), PANTHER (1075), Pfam (2), PROSITE profiles (1) and TIGRFAMs (1) databases, and covers 81.2% of UniProt Knowledgebase release 2019_02.</p>
<p>See <a href="https://www.ebi.ac.uk/about/news/service-news/InterPro-73.0">the news item at InterPro</a> for additional information, including release notes.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/gix_icon.png"/>
<h4 id="s.-pombe-included-in-gene-info-browser-extension"><strong><em>S. pombe</em> included in Gene Info browser extension</strong></h4>
<p><em>2019-04-16</em></p>
<p><em>S. pombe</em> gene information is now included in the <a href="https://gene-info.org/">Gene Info</a> extension (GIX) for the Chrome and Firefox web browsers. GIX allows you to retrieve information about a gene product directly on any webpage simply by double clicking an official gene name, synonym or supported accession. Searching or double-clicking on text terms retrieves gene function annotation, GO terms, external database links, and interaction data drawn from BioGRID and IntAct. Retrieved gene names are automatically hyperlinked for rapid recursive searches.</p>
<p>GeneInfo is fully open source, available online at <a href="https://github.com/knightjdr/gene-info/">GitHub</a>. Tutorial videos, a step-by-step guide, and download links for Firefox Add-ons and the Chrome web storeare available <a href="https://gene-info.org/">online</a>. GeneInfo was developed by James Knight in the <a href="https://gingraslab.lunenfeld.ca/">Gingras Lab</a> at the <a href="http://www.lunenfeld.ca/">Lunenfeld-Tanenbaum Research Institute</a> in Toronto, Canada.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/motif_search_cropped.png"/>
<h4 id="integrated-motif-search"><strong>Integrated motif search</strong></h4>
<p><em>2019-03-19</em></p>
<p>The PomBase <a routerLink="/motif_search">motif search</a> has been fully integrated into the website, and allows users to find protein motifs and send them directly to the PomBase <a routerLink="/query">advanced search</a>.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/icygmb_tiny.png"/>
<h4 id="icygmb-2019---registration-open"><strong>ICYGMB 2019 - registration open</strong></h4>
<p><em>2019-03-13</em></p>
<p>Registration is now open for the <a href="https://yeast2019.org">29th International Conference on Yeast Genetics and Molecular Biology (ICYGMB)</a>, which returns to Gothenburg, Sweden, August 18-22, 2019.</p>
<p>Yeast2019 is the meeting of the international yeast research community where the latest, and even unpublished results are exchanged, and new projects, alliances, and collaborations are founded. Featuring 55 confirmed speakers including keynote lectures by Susan Gasser, Roger Kornberg and Frederick Roth, this conference will contain important news and information for all yeast researchers. A do-not-miss-event.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/slim_tiny.png"/>
<h4 id="go-slim-for-any-s.-pombe-gene-list"><strong>GO slim for any <em>S. pombe</em> gene list</strong></h4>
<p><em>2019-03-05</em></p>
<p>PomBase's advanced search now allows you to retrieve <a href="documentation/pombase-go-slim-documentation">GO slim</a> annotations for any set of search results. To find GO slim annotations for your own list of <em>S. pombe</em> genes, use the advanced search &quot;Gene IDs&quot; option, and then use the &quot;Slim&quot; button on the search results page.</p>
<p>See the <a href="browse-curation/fission-yeast-go-slim-terms">fission yeast GO slim page</a> and the <a href="documentation/advanced-search">advanced search documentation</a> for more information.</p>

</div>

<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

