<div class="recent-news">
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-phenotype-slimming-option-for-advanced-search-results"><strong>New phenotype slimming option for advanced search results</strong></h4>
<p><em>2022-04-14</em></p>
<p>The <a routerLink="/browse-curation/fypo-slim">phenotype slim</a> has also been added to the PomBase <a routerLink="/query">advanced search</a> results panel “Slim with” menu. For example, you can query for all genes involved in a GO process, another phenotype, or any other list, and “slim” the results using the “Phenotype slim” option to view categories of phenotypes assigned to the list.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-phenotype-slim"><strong>New phenotype slim</strong></h4>
<p><em>2022-04-13</em></p>
<p>We have added a <a routerLink="/browse-curation/fypo-slim">phenotype slim overview</a> to complement those provided for disease biological process, molecular function and cellular component annotation. The purpose of the phenotype slim is to provide subsets of commonly used ‘broad’ phenotypic classes or annotation subsets that can provide a useful starting point for accessing phenotype lists. The phenotype slim page provides links to ontology term pages, annotated genes, and to download files containing the slim terms and IDs.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="making-biological-knowledge-useful-for-humans-and-machines"><strong>Making biological knowledge useful for humans and machines</strong></h4>
<p><em>2022-04-04</em></p>
<p>A <a href="https://academic.oup.com/genetics/issue/220/4">GENETICS special issue</a> featuring model organism database updates is published today. This issue features the recent PomBase and <a href="https://www.japonicusdb.org/">JaponicusDB</a> publications and is accompanied by an editorial <a href="https://academic.oup.com/genetics/article-abstract/220/4/iyac001/6563297">“Making biological knowledge useful for humans and machines”</a> co-authored by Val Wood, Paul Sternberg and Howard Lipshitz.</p>
<ul>
<li><a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab222/6481557?login=false">Fission stories: using PomBase to understand Schizosaccharomyces pombe biology</a></li>
<li><a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab223/6481558?login=false">JaponicusDB: rapid deployment of a model organism database for an emerging model species</a></li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="the-fission-yeast-community-have-now-curated-over-1000-publications"><strong>The fission yeast community have now curated over 1000 publications</strong></h4>
<p><em>2022-03-26</em></p>
<p>We would like to extend a huge “thank you” to the fission yeast community for curation contributions. The community have now curated 1008 publications providing 19,156 independent annotations, representing 25% of the curation from small-scale publications. In addition, another 80,000 annotations have been provided via the submission of HTP datasets.</p>
<p>Please contact us <a href="mailto:helpdesk@pombase.org?subject=New%20manuscript%20for%20curation">via the helpdesk</a> if you would like to provide curation for your manuscript but don’t know how.</p>
<p>Links:</p>
<ul>
<li><a href="https://www.pombase.org/reference_list/community">All community curated publications</a></li>
<li><a href="https://www.pombase.org/archive/spotlight">Spotlights</a></li>
<li><a href="https://curation.pombase.org/pombe">Canto curation tool</a></li>
<li><a href="https://www.pombase.org/submit-data">HTP data and browser track submission</a></li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-human-orthologs-including-3-mrp-complex-subunits"><strong>New human orthologs including 3 MRP complex subunits</strong></h4>
<p><em>2022-03-23</em></p>
<p>We continue to identify distant human orthologs. Four new 1:1 human ortholog connections have been added to PomBase this week:</p>
<ul>
<li>RNase MRP subunit Rmp1 = human NEPRO (family submitted to Pfam)</li>
<li>RNase P and RNase MRP subunit Pop23 = human RPP38 (members of the same Pfam clan)</li>
<li>RNase P and RNase MRP subunit Pop8 = human RPP14 (family submitted to Pfam)</li>
<li>glutamyl-tRNA amidotransferase complex subunit 3 (Gtf1) = human GATC</li>
</ul>
<p>Human NEPRO is a poorly characterised protein linked to the disease Anauxetic dysplasia 3, and GATC is the causal gene for “combined oxidative phosphorylation deficiency 42”</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/>
<h4 id="pombase-now-uses-interpro-version-88.0"><strong>PomBase now uses InterPro Version 88.0</strong></h4>
<p><em>2022-03-12</em></p>
<p>PomBase now uses InterPro Version 88.0.</p>
<p>Features include:</p>
<ul>
<li>The addition of 39 InterPro entries (40,071 total entries)</li>
<li>Integration of 45 new methods from the PRINTS (1), SMART (1), Pfam (2), SUPERFAMILY (7), CATH-Gene3D (14), PANTHER (13), CDD (7) databases</li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="tor-and-nutritional-phosphoproteome-dataset-loaded"><strong>TOR and nutritional phosphoproteome dataset loaded</strong></h4>
<p><em>2022-03-11</em></p>
<p>We have loaded the TOR and nutritional phosphoproteome dataset described in <a href="https://www.pombase.org/reference/PMID:33823663">Halova <em>et al</em>.</a> (9424 annotations). Many thanks to Janni Petersen for preparing the files.</p>
<p><a href="https://www.pombase.org/reference/PMID:33823663">Publication page for PMID:33823663</a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="an-additional-5775-novel-curated-lncrnas-from-atkinson-et-al."><strong>An additional 5775 novel curated lncRNAs from Atkinson <em>et al</em>.</strong></h4>
<p><em>2022-03-10</em></p>
<p>We have added an additional 5775 novel curated lncRNAs from <a href="https://www.pombase.org/reference/PMID:29914874">Atkinson <em>et al</em>.</a> to PomBase. We will refine the descriptions of these gene products to align with Sequence Ontology (SO) terms describing RNA features in the coming months.</p>
<p>Thanks to María Rodríguez-López for preparing the files.</p>
<p><a href="https://www.pombase.org/reference/PMID:29914874">Publication page for PMID:29914874</a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="pombase-japonicusdb-publications-in-the-genetics-mod-reports-special-issue"><strong>PomBase &amp; JaponicusDB publications in the GENETICS MOD reports special issue</strong></h4>
<p><em>2022-02-02</em></p>
<p>Papers describing PomBase and JaponicusDB are now published (early online). These articles are part of a special issue of GENETICS devoted to model organism database (MOD) reports. The MOD papers will highlight the journal’s new section on Computational Resources, Software &amp; Databases.</p>
<ul>
<li><a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab222/6481557">Fission stories: using PomBase to understand <em>Schizosaccharomyces pombe</em> biology</a></li>
<li><a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab223/6481558">JaponicusDB: rapid deployment of a model organism database for an emerging model species</a></li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="improved-options-for-filtering-annotations-by-cell-cycle-phase"><strong>Improved options for filtering annotations by cell cycle phase</strong></h4>
<p><em>2021-12-19</em></p>
<p>The Gene Ontology annotation filter for “during” specific cell cycle phases is now included on the “Summary” view in addition to the “Details” view. Available phases have been extended to cover all phases used, and to provide<br />
useful grouping terms. This filter is “ontology aware” (i.e. a search on interphase will also display G1/S/G2 phase annotation). The phase filter is most useful on pages that display increasing volume of phase-specific curation (such as cdc2). The revised phase filter options are also available in the gene expression section.</p>
<p>The phase filters are located at the top right of GO and gene expression annotation sections:</p>
<p><img src="assets/newsfeed/during-filter.png" style="padding-left:5em" alt="During filter" /><br />
</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/>
<h4 id="pombase-now-uses-interpro-version-87.0"><strong>PomBase now uses InterPro Version 87.0</strong></h4>
<p><em>2021-11-23</em></p>
<p>PomBase now uses InterPro Version 87.0, which integrates:</p>
<ul>
<li>1,155 new InterPro entries</li>
<li>Update to Pfam 34.0</li>
<li>1,256 new methods</li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/>
<h4 id="coils-disorder-and-more-new-protein-feature-queries"><strong>Coils, disorder, and more: new protein feature queries</strong></h4>
<p><em>2021-10-18</em></p>
<p>The PomBase <a routerLink="/query">advanced search</a> now allows you to find proteins that have coiled-coil regions, disordered regions, and low-complexity regions. The query-building interface also now organises query options more sensibly, and the documentation has been updated.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/>
<h4 id="pombase-japonicusdb-preprints"><strong>PomBase &amp; JaponicusDB preprints</strong></h4>
<p><em>2021-09-27</em></p>
<p>Papers describing PomBase and JaponicusDB will appear in an issue of <a href="https://academic.oup.com/genetics/">GENETICS</a> devoted to model organism database (MOD) reports. The MOD papers will highlight the journal’s new section on Computational Resources, Software &amp; Databases.</p>
<p>Follow the links to the <a href="https://www.biorxiv.org/content/10.1101/2021.09.07.459264v1.full">PomBase preprint</a> and <a href="https://www.biorxiv.org/content/10.1101/2021.09.23.461587v1">JaponicusDB preprint</a>, and watch for the full-fledged publications to appear in March 2022.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/japonicusdb-32x32.png"/>
<h4 id="japonicusdb-a-new-fission-yeast-database"><strong>JaponicusDB: a new fission yeast database</strong></h4>
<p><em>2021-09-01</em></p>
<p>We are delighted to announce the official release of PomBase’s new sister: <a href="https://www.japonicusdb.org">JaponicusDB</a> is a new, curated model organism database for the fission yeast <em>Schizosaccharomyces japonicus</em>. JaponicusDB highlights include revised gene structures, distant ortholog detection, improved GO annotation, community literature curation, and reciprocal gene page links with PomBase, providing a familiar environment for all fission yeast researchers.</p>
<p>The <em>S. japonicus</em> community will maintain JaponicusDB from now on. Join the <a href="https://mailman.kcl.ac.uk/mailman/listinfo/japonicus-list">new mailing list</a>, and follow <span class="citation" data-cites="japonicusdb">@japonicusdb</span> on Twitter.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/Ofd2_structure_32px.png"/>
<h4 id="alphafold-links"><strong>AlphaFold links</strong></h4>
<p><em>2021-07-28</em></p>
<p>PomBase gene pages now have links to the AlphaFold Protein Structure Database, the collection of structures predicted by AI developed by DeepMind, hosted at EBI. Look in the “External references” section of your favorite gene page, or check out <a href="https://alphafold.ebi.ac.uk/entry/Q01663">this example (pap1)</a>, or read more at the <a href="https://alphafold.ebi.ac.uk/">EBI AlphaFold home</a>.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/kegg3.png"/>
<h4 id="kegg-pathway-links"><strong>KEGG pathway links</strong></h4>
<p><em>2021-07-06</em></p>
<p>PomBase gene pages now have links to pathway entries in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database, as well as links to gene lists for each linked pathway (example: <a href="https://www.pombase.org/term/KEGG_PW:spo01210">2-Oxocarboxylic acid metabolism</a>). The KEGG pathway links are the first entry in a new gene page section, “Molecular pathway”, dedicated to connecting genes in PomBase to depictions of biochemical and signaling pathways.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/>
<h4 id="new-diploid-genotype-phenotype-display"><strong>New diploid genotype &amp; phenotype display</strong></h4>
<p><em>2021-04-29</em></p>
<p>PomBase now includes pages for curated diploid genotypes, and displays phenotypes annotated to them on gene and publication pages. For more details see the documentation for <a href="documentation/gene-page-phenotypes">phenotype annotations</a> and <a href="documentation/genotype-page">genotype pages</a>.</p>

</div>

<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

