<div class="recent-news">
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/>
<h4 id="new-genome-browser-datasets"><strong>New genome browser datasets</strong></h4>
<p><em>2020-09-17</em></p>
<p>Two new datasets in are now available in PomBase JBrowse (links go to PomBase publication pages, which in turn link to the browser with the tracks enabled):</p>
<ul>
<li><a routerLink="/reference/PMID:23395004">Meiotic DSBs</a> from<br />
Fowler KR, Gutiérrez-Velasco S, Martín-Castellanos C, Smith GR. 2013.<br />
Protein determinants of meiotic DNA break hot spots.<br />
PMID:25747261 DOI: 10.1016/j.molcel.2013.01.008</li>
</ul>
<p>and</p>
<ul>
<li><a routerLink="/reference/PMID:32152323">Time-lapse single-cell transcripts for dormancy breaking</a> from<br />
Tsuyuzaki H, Hosokawa M, Arikawa K, Yoda T, Okada N, Takeyama H, Sato. 2020.<br />
Time-lapse single-cell transcriptomics reveals modulation of histone H3 for dormancy breaking in fission yeast.<br />
PMID:32152323 DOI: 10.1016/j.molcel.2013.01.008</li>
</ul>
<p>More datasets are always welcome, so check out our <a routerLink="/documentation/data-submission-form-for-HTP-sequence-linked-data">instructions for submission</a>.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/transposon.png"/>
<h4 id="hermes-transposon-insertions-in-pombase"><strong>Hermes transposon insertions in PomBase</strong></h4>
<p><em>2020-09-11</em></p>
<p><a routerLink="/news/2020-09-11-hermes-transposon-data-long">Read more</a> about the new data in PomBase from Henry Levin’s lab: <a href="/reference/PMID:32101745">Lee et al. 2020</a> applied Hermes and Transposon Integration Sequencing to identify genes important for the formation of heterochromatin. Insertion sites from eight independent cultures can be visualized from PomBase as custom tracks on Jbrowse, and additional data are visible on the <a href="/reference/PMID:32101745">publication page</a> and gene pages. The lab has also provided a <a routerLink="/news/2020-09-11-hermes-transposon-data-long">summary</a> of the work.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/>
<h4 id="new-genome-browser-datasets"><strong>New genome browser datasets</strong></h4>
<p><em>2020-09-08</em></p>
<p>Three new datasets in are now available in PomBase JBrowse (links go to PomBase publication pages, which in turn link to the browser with the tracks enabled):</p>
<ul>
<li><p><a href="https://www.pombase.org/reference/PMID:25747261">Transcription start sites</a> from<br />
Li H, Hou J, Bai L, Hu C, Tong P, Kang Y, Zhao X, Shao Z. 2015.<br />
Genome-wide analysis of core promoter structures in Schizosaccharomyces pombe with DeepCAGE.<br />
PMID:25747261 DOI:10.1080/15476286.2015.1022704</p></li>
<li><p><a href="https://www.pombase.org/reference/PMID:26883383">Transcript data</a> from<br />
Eser P, Wachutka L, Maier KC, Demel C, Boroni M, Iyer S, Cramer P, Gagneur J. 2016<br />
Determinants of RNA metabolism in the Schizosaccharomyces pombe genome.<br />
PMID:26883383 DOI:10.15252/msb.20156526</p></li>
<li><p><a href="https://www.pombase.org/reference/PMID:32101745">Transposon insertion sites</a> from<br />
Lee SY, Hung S, Esnault C, Pathak R, Johnson KR, Bankole O, Yamashita A, Zhang H Levin HL.<br />
Dense Transposon Integration Reveals Essential Cleavage and Polyadenylation Factors Promote Heterochromatin Formation.<br />
PMID:32101745 DOI:10.1016/j.celrep.2020.01.094</p></li>
</ul>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/>
<h4 id="submit-your-genome-browser-data"><strong>Submit your genome browser data</strong></h4>
<p><em>2020-09-08</em></p>
<p>We have updated our HTP data submission procedure to make it easier for you to contribute your datasets for PomBase JBrowse:</p>
<p>We now provide spreadsheet templates in Excel and Open Document formats that gather the metadata we need to load and display your data. You can download a template from the <a href="https://www.pombase.org/documentation/data-submission-form-for-HTP-sequence-linked-data">documentation page on HTP data submission</a>. Send completed spreadsheets to the <a href="mailto:helpdesk@pombase.org">PomBase helpdesk</a>.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/go-term-matrix.png"/>
<h4 id="published-go-term-matrix-for-annotation-qc"><strong>Published: GO Term Matrix for annotation QC</strong></h4>
<p><em>2020-09-02</em></p>
<p>In collaboration with the GO Consortium, the PomBase team has published a report on the <strong>Term Matrix</strong> approach to GO annotation quality control. The article, out this week in <a href="https://royalsocietypublishing.org/doi/10.1098/rsob.200149">Open Biology</a>, describes biological processes that do, or don’t, share annotated gene products, and how we use co-annotation patterns to build rules to detect, correct, and prevent errors.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/pombe2022-32px.png"/>
<h4 id="th-pombe-meeting-postponed-to-2022"><strong>11th Pombe meeting postponed to 2022</strong></h4>
<p><em>2020-08-19</em></p>
<p>Due to the ongoing Covid-19 pandemic, the <a href="http://www.fission-yeast2022.org/">11th International Fission Yeast Meeting</a>, due to take place in Hiroshima, Japan, has been postponed.</p>
<p>The new dates will be <strong>12th (Sun -17th (Fri) June, 2022</strong>.</p>
<p>Please see the <a href="http://www.fission-yeast2022.org/">conference website</a> and pombelist for further announcements.</p>

</div>

<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

