### 2006 GeneDB *S. pombe* survey, question 16 responses

<p><strong>If you could add one feature to the database immediately, or improve one feature immediately, what would it be?</strong></p>
<ol>
<li>Location of the genes based on their chromosomal positions.</li>
<li>sequence variation in S. pombe related strains</li>
<li>Links to S. cerevisiae and other eukaryotic databases for orthologues etc.</li>
<li>Adding protein/gene interaction data. Either biochemical or from synthetic interaction studies.</li>
<li>-</li>
<li>Improve the link to microarrays results to obtain an easier view of the microarrays results, including the mutant strains</li>
<li>Sequences of upstream 1000bp + gene sequences + downstream 1000bp</li>
<li>Epigenome information!!</li>
<li>Improve the gene search tool by including full-text genepage search. Add transcriptional regulatory motifs of genes.</li>
<li>Promoter Motif Finder</li>
<li>Add phenotypes.</li>
<li>direct link to orthologs</li>
<li>mainly I want to know the best homolgues in S. cerevisiae, C.elegans and mammals.</li>
<li>mutant and deletion phenotypes</li>
<li>as indicaeted above</li>
<li>The web page needs to be more attractive.</li>
<li>More information on the gene functions curated from the literature.</li>
<li>the sited literature for a specific gene.</li>
<li>transcription start and stop</li>
<li>Link to find orthologs of my pombe gene of interest in other species directly from the &quot;Gene Page.&quot;</li>
<li>Add phylogenetic trees</li>
<li>I want to download the datasets for ORFs and chromosomal elements in a format that I can directly import into a relational database such as MS Access. Any tab-delimited format is fine, but the ORF start and stop on the chromosome coordinates AND the gene aliases (or blank spaces) need to be linkable and to import consistantly. The current format is unusable because, for example, the names in some lines are not tab-delimited spaces in all of the other lines where no name exists. The unique identifier for all elements needs to be on all tables. We need such information to do our own whole-genome searches and mutageneses.</li>
<li>looks easier</li>
<li>3&#39; UTR data</li>
<li>Quick access to a specific gene in the choromosome map. Quick access to the cosmid sequence that contains a specific gene.</li>
<li>The Gene Description would be more informative. Especially when you use the LIST Download feature.</li>
<li>genome browser - needs to be more user friendly</li>
<li>Making it easy to access to available resources such as mutant strains, plasmids for gene expression; probably by adding links to cooperative researchers or institutions (for example, NBRP in Japan http://yeast.lab.nig.ac.jp/nig/english/index.html).</li>
<li>useful</li>
<li>Finding orthologues. Integrating orthiologue&not;&yen;s information Comparing two or more genes for expression profiles or other features</li>
<li>I like SGD very much</li>
<li>Make it more easy to identify a query sequence by Blast. You often get the name of a cosmid, instead of teh name of the gene</li>
<li>Microarray datasets</li>
<li>improved more user-friendly motif searching</li>
<li>A tool tp search protein-protein interaction database</li>
<li>addition of a function: mapping of two genes, quick calculation of intergenic distance between linked genes.</li>
<li>the information from the sequencing for plasmid vs cosmid is confusing. e.g. you find your gene and blast search a region of it within the pombe sequence, for example to check that a primer only binds to your gene and not elsewhere. the ID for the hits (which should be 100% matching with your gene) are the plasmid numbers which don&#39;t correspnd to the cosmid number which makes things a bit confusing and requires you to browse through all the balst hits (e.g. the systematic gene name - SPCC###.##). Maybe when you see results of the blast the systematic gene names/cosmid could be immediately visible rather than the plasmid name?</li>
<li>better links to other orthologs ie. human</li>
<li>Don&#39;t know</li>
<li>interacting factors if known (genetic and physical)</li>
<li>FASTA format of sequence data not punctuated by space</li>
<li>email addresses of experts for the gene</li>
<li>Phenotypes of the mutant</li>
<li>Improve Gene Ontology Annotaions. It is often not trivial to track down why a specific term is associated with a gene. Thi smay just be my lack of knowledge about how to correctly use the resource.</li>
<li>gene boundary: where on the chromosome a gene start and where it ends (promoter and transcription termination signal).</li>
<li>phenotype information</li>
<li>Two things actually: 1. To have chromosomal position along with contig position. 2. Consolidated information about interacting partners and/or functional partners.</li>
<li>Ability to search for partial systematic ORF names</li>
<li>protein- protein interaction</li>
<li>more clear identification of physical distances between pairs of genes</li>
<li>Add transcriptional regulatory regions of genes</li>
<li>qq</li>
<li>Overview of UTRs</li>
<li>Direct links to major publications for genes.</li>
<li>microarray data</li>
<li>show orthologs with % similarity and sequence alignments</li>
<li>transcript start/stop</li>
<li>add a clustal multiple alignment tool, to compare all orthologues of a given fission yeast gene.</li>
<li>Full text literature annotation with references to all mutant alleles of a gene and phenotypes of the mutants (often a paper contains information on mutant alleles that is not reflected in the abstract) note: There are many features I consider a very close second in priority.</li>
<li>Easy access to genome/proteome interaction studies</li>
<li>otholog sequence alignment, such as clustal and shown as box-shadowing type.</li>
<li>description of mutant phenotype and available alleles, description of localisation</li>
<li>It would be very helpful to have a section on published mutants for each gene along with a reference. Often I would like to know what kind of mutations of a particular gene have been constructed and what their phenotypes are. If there was a brief compilation of that data in one place it would make GeneDB much better.</li>
<li>Link the gene names that appear in Blast searches to the genedb page for that gene so that the info on the gene is jus a click away.</li>
<li>List Download</li>
<li>I find the graphic interface of amiGO to be very difficult to use. I really have never found it to be useful. It also has mistakes in process names. For example it refers to the transposons as Ty1 not the true name Tf1.</li>
<li>ability to download gene sequence data from the gene page, plus the ability to add a specified distance of the sequence either side of the gene. For example, retrieving sequence data for gene Y plus 0.5 Kb of sequence either side.</li>
<li>something that alows me to look at at gene in the genome, then click on said gene and expending it into its DNA sequence along with adjacent sequence</li>
<li>It would be helpful to be able to get sequence of gene plus bases up and downstream, as with SGD, instead of having to look through the cosmid library</li>
<li>protein interactions</li>
<li>x</li>
<li>Direct access link to PombePD under some form of licensing or one off fee pricing model (although nocost is obviously the ideal).</li>
<li>blast features, display of results in the form of map like in cerevisiae geome project</li>
<li>Mutant phenotypes and protein interaction data</li>
<li>to become like pombePD</li>
<li>Direct link to NCBI/Entrez</li>
<li>Protein regulation</li>
<li>(1)possibility of easily carrying out certain predictions manually, like prediction of subcellular localisation, introns, homologues. This would enable the user to test how good the predictions are and if there are alternative possibilities. (2)facility for getting a list of genes with similar expression profiles ; also more DNA microarray data (e.g. DNA damage) (3) better (i.e. more complex) facility for searching the whole set of genes: e.g. enable the user to specify a number of &#39;ideal&#39; criteria of a gene/protein (combination of pI, size, expression profile, protein domains, kinase motifs, other sequence motifs, predicted subcellular localisation), which would then return a set of genes/proteins that best fulfill these criteria, without necessarilly fulfilling all of them. Such a feature would be very useful in the identification of very distant homologues that are not easily identified in Blast searches.</li>
<li>More informations on phenotypes</li>
<li>Community annotation/discussion</li>
<li>easier access to batch downloads of gene/protein information and more information on what it exactly is (how it was constructed)</li>
<li>I find it very good. For my purposes all the available features are already a lot. I understand however the need for supporting curation of annotation and constant update. This of cours requires more than one curator and appropriate funding.</li>
<li>Phenotype data</li>
<li>Phenotypes of deletion mutants</li>
<li>Search information by gene list</li>
<li>SDS PAGE mobility</li>
<li>No comments</li>
<li>Complete list of phenotypes</li>
<li>no</li>
<li>metabolic pathway maps tuned to yeast</li>
<li>Position of the introns into the sequence add as colored letters</li>
<li>-</li>
<li>I would like to wish to get also the information of the S.cerevisiae gene name of the respective gene</li>
<li>To easy up data down-loads: where to find what</li>
<li>interactions (Y 2 H) IP info for each gene genetic interactions (null, slow growth)</li>
<li>An end-to-end searchable gene map of each chromosome (or contig), so that I can easily see what genes are closely linked to my gene-of-interest.</li>
<li>Make it easier</li>
<li>--</li>
<li>integrated resource for accessing all microarray data</li>
<li>Completion of the genome sequence, with linking of all contigs and addition of telomeres.</li>
<li>somewhat more details on the phenotype</li>
<li>Add mutant phenotype description on each Gene Page</li>
<li>Annotate phenotypes associated with a given gene (deletion, mutants, overexpression, doublemutants,..)</li>
<li>clear literature summary</li>
<li>Phenotypes and lit</li>
<li>IDs of and links to physical and genetic interaction partners.</li>
<li>Results of prerun BLAST against other organisms with links built in to other publicly available databases.</li>
<li>don&#39;t know</li>
<li>Provide the ORF+ 1KB upstream and 1 Kb downstream of genes</li>
<li>I would be useful to have the identity/similarity % with homologs of each gene compared to other organisms</li>
<li>phenotypes, and genetic and physical interactions as is found in SGD and YPD would make the database hugely more powerful.</li>
<li>when we could access PombePD the cross referencing between orthologues in Worms, humans and cerevisiae was so useful. YOGY has been great but it would be good if it was linked into Wormbase, SGD more so you could move between Gene pages in different organisms more quickly.</li>
<li>Web services for better integration with BioPerl and automated scripts in general.</li>
<li>Description of homologues in higher organisms.</li>
<li>Information on transciption start and stop sites and promoter elements.</li>
<li>No Thanks</li>
<li>more microarray datasets</li>
<li>mutant gene phenotype and genetic interactions</li>
<li>Better hyperlinks to look at orthologues in other organisms</li>
<li>Having easy access to information about regulatory motifs and promoters for each gene.</li>
<li>phosphorylation site predictor</li>
<li>Integrating GBrowse with expression data.</li>
<li>Probably better links to as much as possible of published literature relating to the gene, so one can read the details of a gene of interest when the gene name is not mentioned explicitly in the abstract of the paper. however, this could be huge for some genes such as cdc2.</li>
<li>I would change the genome browser to be based on chromosomal coordinates rather than contig coordinates. Right now I&#39;m using the &quot;chromosome&quot; assembly of the genome. Since your implementation of gbrowse does not allow genome browsing based on chromosomal coordinates, I&#39;ve been using this one that does: http://fungal.genome.duke.edu/cgi-bin/gbrowse/spom/ . I understand that the chromosomal coordinates may not be as precise, but if they&#39;re good enough to create chromosome assembly files, why not just allow people to access the genome that way as well? I think that&#39;s what most people have come to expect from genome browsers elsewhere.</li>
<li>Add a Candida albicans GeneDB and a Cryptococcus neoformans GeneDB.</li>
<li>curated (or non-curated) interaction grids genetic, physical, text mining based. with links to data</li>
<li>identification of genes with human orthologues especially causing disease</li>
<li>Non-coding sequences, particularly replication origins.</li>
<li>Identification of transcription units.</li>
<li>Be able to get alignments of my favorite gene with the closest ortologs directly</li>
<li>To show the homolog names in other model organisms</li>
<li>Direct X-ref to SGD</li>
<li>Graphic Interface</li>
<li>More GO curation</li>
<li>UTR data for genes</li>
</ol>