# How can I use Artemis to browse the genome?
<!-- pombase_categories: Finding data,Tools and resources -->

If you want to browse the *S. japonicus* genome in the Artemis environment,
it is fairly easy to download and run locally:

-    *Important:* For *S. japonicus* we recommend using [Artemis Version 16](ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/v16/),
     not the most recent version available from the [Artemis home
     page](http://www.sanger.ac.uk/science/tools/artemis). Newer
     versions of Artemis may be incompatible with the default Java
     installed on some computers, especially Macs.

-   Read in the chromosome contig files available from the [Genome sequences](/downloads/genome-datasets) page. (You can read
    in one or more files in a session.)

Once you have loaded the file(s), you can do many different things,
e.g.:

-   Find features by name or ID
-   Find all features of a given type (e.g. see the "[can I find transposons](/faq/how-can-i-find-transposons-genome)" FAQ)
-   Find matches to a specific nucleotide sequence (e.g. see the 
    "[restriction enzyme map](/faq/can-i-generate-comprehensive-restriction-enzyme-map-genome-pombase)" FAQ)
-   View the nucleotide or amino acid sequence of a region or feature
-   Export selected sequences

See the [Artemis manual](ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf) (pdf;
Sanger site) for additional information.

