# How are non-coding RNA genes identified?
<!-- pombase_categories: Genome statistics and lists -->

Most non-coding RNAs in ${database_name} are based on transcriptome data, either
from Jürg Bähler's lab (Solexa/deep sequencing;
[PMID:18488015](http://www.ncbi.nlm.nih.gov/pubmed?term=18488015)) or
Nick Rhind's lab (RNA sequencing;
[PMID:21511999](http://www.ncbi.nlm.nih.gov/pubmed?term=21511999)). For
any ncRNA, the source should be linked as a publication in the
"Literature" section at the bottom of the ${database_name} gene page. To get an
idea of the transcription in a region, you can look at the Bähler Lab
Transcriptome Viewer, which is linked from most gene pages, e.g.
[SPNCRNA.200](http://www.bahlerlab.info/cgi-bin/pombetv/pombetv?condition=13&genename=SPNCRNA.200&action=display).
Unfortunately some genes, such as SPNCRNA.1115, post-date the viewer and
therefore do not have entries, but you can look at transcription in this
region by finding a neighboring gene.

