# Is there an equivalent to the Artemis java applet in PomBase?
<!-- pombase_categories: Finding data,Tools and resources -->

Unfortunately, the Java Applet technology has been
[depricated](http://openjdk.java.net/jeps/289) and is not
supported by the major browsers.

We now use JBrowse and links to the Ensembl genome browser in place of
the Artemis applet.

If you want to browse the *S. pombe* genome in Artemis on your laptop
or desktop, it is free to download and run locally:

-   Download and installation instructions are available from the
    [Sanger Institute Artemis pages](https://www.sanger.ac.uk/tool/artemis/)
-   Read in the chromosome contig files available from the [Genome sequences](/downloads/genome-datasets) page. (You can read
    in one or more files in a session.)

Once you have loaded the file(s), you can do many different things,
e.g.:

-   Find features by name or ID
-   Find all features of a given type (e.g. see the "[can I find transposons](/faq/how-can-i-find-transposons-genome)" FAQ)
-   Find matches to a specific nucleotide sequence (e.g. see the 
    "[restriction enzyme map](/faq/can-i-generate-comprehensive-restriction-enzyme-map-genome-pombase)" FAQ)
-   View the nucleotide or amino acid sequence of a region or feature
-   Export selected sequences
