<div class="recent-news">
<h4 id="pombase-nar-database-issue-2018-10-15"><strong>PomBase NAR Database Issue <em>2018-10-15</em> </strong></h4>
<p>PomBase has published a new paper in Nucleic Acids Research's well-known annual Database Issue, now available as an <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky961/5128936">open-access Advance Article</a>.</p>
<p>Lock A, Rutherford K, Harris MA, Hayles J, Oliver SG, BÃ¤hler J; Wood V.<br />
PomBase 2018: user-driven reimplementation of the fission yeast database provides rapid and intuitive access to diverse, interconnected information.<br />
<em>Nucleic Acids Res.</em> 2018 (Database issue): gky961 Epub 2018 Oct 13.<br />
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30321395">PMID:30321395</a>; <a href="https://doi.org/10.1093/nar/gky961">DOI:10.1093/nar/gky961</a></p>
<p>We have updated the <a routerLink="/about/citing-pombase">Citing PomBase</a> to recommend citing this new paper. Thank you all for guiding the development of the new, improved PomBase, and for your continued usage, curation contributions, and suggestions!</p>

<h4 id="fungal-pathogen-genomics-course-2019---registration-open-2018-10-10"><strong>Fungal Pathogen Genomics Course 2019 - registration open <em>2018-10-10</em> </strong></h4>
<p>Registration for the <a href="https://coursesandconferences.wellcomegenomecampus.org/our-events/fungal-pathogen-genomics-2019/">2019 Fungal Pathogen Genomics Course</a> is now open.</p>
<p>The course is hosted by Wellcome Genome Advanced Courses and Scientific Conferences, and will take place May 7-12, 2019, at the Wellcome Genome Campus, Hinxton, UK.</p>
<p>Course content provides hands-on training on how to:</p>
<ul>
<li>Take advantage of unique tools offered by FungiDB, EnsemblFungi, PomBase, SGD/CGD, and MycoCosm/JGI;</li>
<li>Develop testable hypotheses;</li>
<li>Investigate transcriptomics, proteomics and genomics datasets across multiple databases and different user interfaces.</li>
</ul>
<p>Please see the <a href="https://coursesandconferences.wellcomegenomecampus.org/our-events/fungal-pathogen-genomics-2019/">course website</a> for more information, including how to apply, costs (limited bursaries are available), programme, and logistics.</p>

<h4 id="transcript-tracks-from-atkinson-et-al-2018-loaded-2018-10-06"><strong>Transcript tracks from Atkinson et al (2018) loaded <em>2018-10-06</em> </strong></h4>
<p>We are very pleased to announce that we have loaded the transcript tracks from <a href="https://www.ncbi.nlm.nih.gov/pubmed/29914874">Atkinson et al (2018)</a> into the PomBase JBrowse genome browser. For a brief introduction to getting started with PomBase JBrowse, please see our <a href="https://www.pombase.org/documentation/JBrowse_quick_start">documentation page</a>. If you have published data that you would like to see hosted, please get in touch.</p>

<h4 id="pombelist-changes-2018-08-31"><strong>pombelist changes <em>2018-08-31</em> </strong></h4>
<p>The pombe community mailing list, &quot;pombelist&quot;, is now hosted by the University of Cambridge. The new address for posting messages is <a href="mailto:pombelist@pombase.org">pombelist@pombase.org</a>. The link to <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">subscribe</a> has also changed.</p>

<h4 id="new-genome-browser-tracks-2018-05-28"><strong>New genome browser tracks <em>2018-05-28</em> </strong></h4>
<p>We are very pleased to announce that we have loaded a number of new datasets into the PomBase <a href="https://www.pombase.org/jbrowse/">JBrowse genome browser</a>.</p>
<p>These include:</p>
<ul>
<li><a href="https://www.biorxiv.org/content/early/2018/03/13/281642">Thodberg <em>et al.</em> (2018)</a> - CAGE-defined <a href="https://www.pombase.org/jbrowse/?loc=II%3A27586..65254&amp;tracks=DNA%20sequence%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20start%20sites%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)&amp;highlight=">transcription start sites</a> across 5 different conditions</li>
<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23163955">Yadav <em>et al.</em> (2012)</a> - <a href="https://www.pombase.org/jbrowse/?loc=I%3A2364901..2436900&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CPosition%20specific%20G(x)%20scores%20(free%20energy%20needed%20to%20separate%20a%20base%20pair)%20-%20Yadav%20et%20al.%20(2012)&amp;highlight=">G(x) scores</a> specifying the amount of free energy needed to melt base pairs in the DNA duplex at different genomic locations</li>
<li>A PomBase-generated <a href="https://www.pombase.org/jbrowse/?loc=III%3A725762..736349&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20factor%20DNA%20binding%20motifs&amp;highlight=">dataset of promoter elements</a> across the genome (computational matching of the <a routerLink="/browse-curation/dna-binding-sites">consensus promoter sequences</a> to the reference DNA sequence)</li>
<li><a href="https://www.pombase.org/jbrowse/?loc=I%3A2178501..2207300&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CIntron%20Branch%20Points%20-%20Bitton%20et%20al.%20(2014)&amp;highlight=">Intron branch points</a> from <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">Bitton <em>et al.</em> (2014)</a> which was also available in the Ensembl browser.</li>
</ul>
<p>For anyone wanting a quick introduction to our genome browser, Antonia Lock has written <a routerLink="/documentation/JBrowse_quick_start">&quot;Getting started with PomBase JBrowse&quot;</a>, a basic guide that covers loading tracks, navigating the browser, what metadata we provide, and more.</p>

<h4 id="new-book-chapter-about-pombase-2018-05-22"><strong>New book chapter about PomBase <em>2018-05-22</em> </strong></h4>
<p>PomBase has a <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4">new book chapter</a> in <a href="https://link.springer.com/book/10.1007/978-1-4939-7737-6"><em>Eukaryotic Genomic Databases</em> (Methods and Protocols)</a>.</p>
<p>This chapter provides insight into the curation philosophy and data organization at PomBase, and provides a guide to using PomBase tailored for infrequent visitors and anyone considering extending their research to include <em>S. pombe</em>.</p>
<p>The chapter is <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4">free to download</a> courtesy of the <a href="https://wellcome.ac.uk/">Wellcome Trust</a>.</p>


<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

