<div class="recent-news">
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/mobile_thumb.png"/>
<h4 id="pombase-in-your-pocket"><strong>PomBase in your pocket</strong></h4>
<p><em>2018-11-14</em></p>
<p>Our usage statistics informed us that over 20% of devices accessing PomBase are smartphones or tablets. We therefore spent some time optimizing the display for small screens. We hope that you will continue to enjoy PomBase on the go!</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/unknowns.png"/>
<h4 id="hidden-in-plain-sight-what-remains-to-be-discovered-in-the-eukaryotic-proteome"><strong>Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</strong></h4>
<p><em>2018-10-26</em></p>
<p>In a new publication, PomBase curators consider the challenges and opportunities that conserved, but persistently unstudied, proteins pose for diverse areas of basic and applied biology. To draw attention to these proteins, we develop metrics to define unknown lists, provide unknown inventories for human and yeast, classify <em>S. pombe</em> unknowns by numerous orthogonal attributes, and speculate about reasons for their neglect.</p>
<p>A <a href="https://www.biorxiv.org/content/early/2018/11/16/469569">pre-publication manuscript</a> is available at bioRxiv.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/>
<h4 id="new-pombase-genomic-region-graphics"><strong>New PomBase genomic region graphics</strong></h4>
<p><em>2018-10-23</em></p>
<p>PomBase gene pages now use interactive graphics from PomBase JBrowse to depict the genomic region around the gene. Drag to scroll left and right, double-click to zoom in, shift-double-click to zoom out, and click a feature to see details in a popup. The &quot;Full-screen view&quot; link in the corner opens the fully functional JBrowse in a new tab or window. Reloading a gene page restores the display to the default location and zoom level.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/NAR_paper.png"/>
<h4 id="pombase-nar-database-issue"><strong>PomBase NAR Database Issue</strong></h4>
<p><em>2018-10-15</em></p>
<p>Our NAR database update &quot;PomBase 2018: user-driven reimplementation of the fission yeast database provides rapid and intuitive access to diverse, interconnected information&quot; is <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky961/5128936">now available</a>. We have updated the <a routerLink="/about/citing-pombase">Citing PomBase</a> to recommend citing this new paper. Thank you all for guiding the development of the new, improved PomBase, and for your continued usage, curation contributions, and suggestions!</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/WT_course.png"/>
<h4 id="fungal-pathogen-genomics-course-2019---registration-open"><strong>Fungal Pathogen Genomics Course 2019 - registration open</strong></h4>
<p><em>2018-10-10</em></p>
<p>Registration for the <a href="https://coursesandconferences.wellcomegenomecampus.org/our-events/fungal-pathogen-genomics-2019/">2019 Fungal Pathogen Genomics Course</a> is now open. The course is hosted by Wellcome Genome Advanced Courses and Scientific Conferences, and will take place May 7-12, 2019, at the Wellcome Genome Campus, Hinxton, UK. Course content provides hands-on training on how to: - Take advantage of unique tools offered by FungiDB, EnsemblFungi, PomBase, SGD/CGD, and MycoCosm/JGI; - Develop testable hypotheses; - Investigate transcriptomics, proteomics and genomics datasets across multiple databases and different user interfaces. Please see the <a href="https://coursesandconferences.wellcomegenomecampus.org/our-events/fungal-pathogen-genomics-2019/">course website</a> for more information, including how to apply, costs (limited bursaries are available), programme, and logistics.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/Atkinson.png"/>
<h4 id="transcript-tracks-from-atkinson-et-al-2018-loaded"><strong>Transcript tracks from Atkinson et al (2018) loaded</strong></h4>
<p><em>2018-10-06</em></p>
<p>We are very pleased to announce that we have loaded the transcript tracks from <a href="https://www.ncbi.nlm.nih.gov/pubmed/29914874">Atkinson et al (2018)</a> into the PomBase JBrowse genome browser. For a brief introduction to getting started with PomBase JBrowse, please see our <a href="https://www.pombase.org/documentation/JBrowse_quick_start">documentation page</a>. If you have published data that you would like to see hosted, please get in touch.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/>
<h4 id="pombelist-changes"><strong>pombelist changes</strong></h4>
<p><em>2018-08-31</em></p>
<p>The pombe community mailing list, &quot;pombelist&quot;, is now hosted by the University of Cambridge. The new address for posting messages is <a href="mailto:pombelist@pombase.org">pombelist@pombase.org</a>. The link to <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">subscribe</a> has also changed.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/new_tracks.png"/>
<h4 id="new-genome-browser-tracks"><strong>New genome browser tracks</strong></h4>
<p><em>2018-05-28</em></p>
<p>We are very pleased to announce that we have loaded a number of new datasets into the PomBase [JBrowse genome browser (https://www.pombase.org/jbrowse/). These include:</p>
<ul>
<li><a href="https://www.biorxiv.org/content/early/2018/03/13/281642">Thodberg <em>et al.</em> (2018)</a> - CAGE-defined <a href="https://www.pombase.org/jbrowse/?loc=II%3A27586..65254&amp;tracks=DNA%20sequence%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20start%20sites%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)&amp;highlight=">transcription start sites</a> across 5 different conditions</li>
<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23163955">Yadav <em>et al.</em> (2012)</a> - <a href="https://www.pombase.org/jbrowse/?loc=I%3A2364901..2436900&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CPosition%20specific%20G(x)%20scores%20(free%20energy%20needed%20to%20separate%20a%20base%20pair)%20-%20Yadav%20et%20al.%20(2012)&amp;highlight=">G(x) scores</a> specifying the amount of free energy needed to melt base pairs in the DNA duplex at different genomic locations</li>
<li>A PomBase-generated <a href="https://www.pombase.org/jbrowse/?loc=III%3A725762..736349&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20factor%20DNA%20binding%20motifs&amp;highlight=">dataset of promoter elements</a> across the genome (computational matching of the <a routerLink="/browse-curation/dna-binding-sites">consensus promoter sequences</a> to the reference DNA sequence)</li>
<li><a href="https://www.pombase.org/jbrowse/?loc=I%3A2178501..2207300&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CIntron%20Branch%20Points%20-%20Bitton%20et%20al.%20(2014)&amp;highlight=">Intron branch points</a> from <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">Bitton <em>et al.</em> (2014)</a> which was also available in the Ensembl browser.</li>
</ul>
<p>For anyone wanting a quick introduction to our genome browser, Antonia Lock has written <a routerLink="/documentation/JBrowse_quick_start">&quot;Getting started with PomBase JBrowse&quot;</a>, a basic guide that covers loading tracks, navigating the browser, what metadata we provide, and more.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/book_chapter.png"/>
<h4 id="new-book-chapter-about-pombase"><strong>New book chapter about PomBase</strong></h4>
<p><em>2018-05-22</em></p>
<p>PomBase has a <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4">new book chapter</a> in <a href="https://link.springer.com/book/10.1007/978-1-4939-7737-6"><em>Eukaryotic Genomic Databases</em> (Methods and Protocols)</a>. This chapter provides insight into the curation philosophy and data organization at PomBase, and provides a guide to using PomBase tailored for infrequent visitors and anyone considering extending their research to include <em>S. pombe</em>. The chapter is <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4">free to download</a> courtesy of the <a href="https://wellcome.ac.uk/">Wellcome Trust</a>.</p>

</div>

<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

