<div class="recent-news">
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/human disease.png"/>
<h4 id="improved-disease-association-dataset-released"><strong>Improved disease association dataset released</strong></h4>
<p><em>2019-01-09</em></p>
<p>We are pleased to announce the release of our improved human disease mappings dataset. This dataset connects human disease causing genes to their <em>S. pombe</em> orthologs.</p>
<p>Diseases are now mapped to the Disease Ontology (DO) and the dataset has been extended by data from Malacards. All disease associations can be accessed from the <a routerLink="/term/DOID:4">top level disease page</a>. A disease slim has been created to facilitate browsing of disease categories. Currently, 907 S. pombe genes are associated with disease (up from 610 in the original dataset). This number is due to increase as mappings are still in progress.</p>
<p>Many thanks to DO and Malacards for help in improving this annotation set. Icon courtesy of <a href="https://pixabay.com/en/users/mcmurryjulie-2375405/">Julie McMurry</a>.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/mito_news.png"/>
<h4 id="mitochondrial-go-annotation-update"><strong>Mitochondrial GO annotation update</strong></h4>
<p><em>2018-12-17</em></p>
<p>Responding to increasing interest in mitochondrial biology, especially relating to ageing, neurogenerative diseases, and processes at the ER-mitochondrion interface, we have reviewed <em>S. pombe</em> mitochondrial GO annotations. Although there is still relatively little fission yeast-derived experimental data in this area, we have refined many inferred annotations for mitochondrial complexes and sub-components as well as some for processes.</p>
<p>You can see all 753 <em>S. pombe</em> mitochondrial annotations on the ontology term page for <a href="https://www.pombase.org/term/GO:0005739">mitochondrion (GO:0005739)</a>.</p>
<p>Icon courtesy of Reactome.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/>
<h4 id="new-nucleosome-occupancy-maps-loaded"><strong>New nucleosome occupancy maps loaded</strong></h4>
<p><em>2018-12-03</em></p>
<p>We have loaded the nucleosome occupancy maps as described in González et al. (2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27662899">PMID: 27662899</a>. This dataset was generated using the paired-end sequencing protocol of Illumina and thus those maps are of higher resolution than those made with single-end (SE) sequencing hosted in the browser since before.</p>
<p>Here is a link that <a href="https://www.pombase.org/jbrowse/?loc=II%3A2751621..2767510&tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2Cnucleosome%20positioning%20during%20vegetative%20growth%3B%20repeat%201%20-%20Gonz%C3%A1lez%20et%20al.%20%282016%29%2Cnucleosome%20positioning%20during%20vegetative%20growth%3B%20repeat%202%20-%20Gonz%C3%A1lez%20et%20al.%20%282016%29%2CDNA%20sequence&highlight=">loads the tracks in PomBase JBrowse</a>. And here is a link to our <a href="https://www.pombase.org/documentation/JBrowse_quick_start">JBrowse quickstart guide</a>.</p>
<p>Many thanks to Paco Antequera for sending us the bigwig files! If you would like us to load any datasets then please get in touch.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/QLT.png"/>
<h4 id="see-your-genes-in-a-quilt"><strong>See your genes in a QuiLT</strong></h4>
<p><em>2018-11-21</em></p>
<p>PomBase now offers a new way to display gene lists graphically based on multiple orthogonal annotation types — the Quick Little Tool (QuiLT) for visualisation.</p>
<p>Inspired by our recent analysis of conserved unstudied proteins (see figures 4 and S1 in the <a href="https://www.biorxiv.org/content/early/2018/11/16/469569">manuscript at bioRxiv</a>), QuiLT allows you to create a similar figure for any gene list you create or import using the advanced search. To use QuiLT, follow the link to your search results, then click the &quot;Visualise&quot; button. QuiLT visualisation is also available from the PomBase pages that list genes annotated to an ontology term, and on the Priority unstudied genes page.</p>
<p>To see the Unknowns dataset in QuiLT, visit the <a routerLink="/query/results/from/predefined/priority-unstudied-genes:conserved_unknown">unknowns results page</a> and click &quot;Visualise&quot;.</p>
<p>The QuiLT display is interactive, and you can:</p>
<ul>
<li>Highlight subsets of the list, and filter the display</li>
<li>Toggle annotation types on and off</li>
<li>Reorder the list to focus on features of most interest</li>
<li>Download the image</li>
</ul>
<p>See the <a routerLink="/documentation/quick-little-tool">QuiLT documentation</a> for more information, and <a href="mailto:helpdesk@pombase.org">contact the curators</a> if you have comments, questions or suggestions.</p>
<p>Many thanks to our star (and only) programmer, Kim Rutherford, for developing QuiLT.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/antiporter.png"/>
<h4 id="fission-yeast-transmembrane-transport-overhaul"><strong>Fission yeast transmembrane transport overhaul</strong></h4>
<p><em>2018-11-20</em></p>
<p>The Gene Ontology &quot;<a href="https://www.pombase.org/term_genes/GO:0055085">transmembrane transport</a>&quot; branch has recently been substantially revised. In line with these revisions, PomBase has standardised gene product descriptions for transporters, and overhauled GO annotations to be as complete and comprehensive as possible based on current knowledge.</p>
<p>Icon courtesy of <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29077811">Reactome</a>.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/unknowns.png"/>
<h4 id="hidden-in-plain-sight-what-remains-to-be-discovered-in-the-eukaryotic-proteome"><strong>Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</strong></h4>
<p><em>2018-11-17</em></p>
<p>In a new publication, PomBase curators consider the challenges and opportunities that conserved, but persistently unstudied, proteins pose for diverse areas of basic and applied biology. To draw attention to these proteins, we develop metrics to define unknown lists, provide unknown inventories for human and yeast, classify <em>S. pombe</em> unknowns by numerous orthogonal attributes, and speculate about reasons for their neglect.</p>
<p>A <a href="https://www.biorxiv.org/content/early/2018/11/16/469569">pre-publication manuscript</a> is available at bioRxiv.</p>

</div>
<div class="news-item">
<img class="newsfeed-thumbnail" src="/assets/newsfeed/mobile_thumb.png"/>
<h4 id="pombase-in-your-pocket"><strong>PomBase in your pocket</strong></h4>
<p><em>2018-11-14</em></p>
<p>Our usage statistics informed us that over 20% of devices accessing PomBase are smartphones or tablets. We therefore spent some time optimizing the display for small screens. We hope that you will continue to enjoy PomBase on the go!</p>

</div>

<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

