<div class="docs">
<div *ngIf="section == 'about'">
  <div class="docs-menu">
<h3 id="about"><a routerLink="/about">About</a></h3>
<ul>
<li><a routerLink="/about/citing-pombase">Citing PomBase</a></li>
<li><a routerLink="/help">Help</a></li>
<li><a routerLink="/about/resource-metrics">Resource Metrics</a></li>
</ul>

  </div>
  <div class="docs-content">
  <div *ngIf="pageName == 'citing-pombase'">
<h1 id="citing-pombase">Citing PomBase</h1>
<p>To cite <em>S. pombe</em> genome data and annotation, please use:</p>
<p>Wood V, Harris MA, McDowall MD, Rutherford K, Vaughan BW, Staines DM, Aslett M, Lock A, Bähler J, Kersey PJ, Oliver SG.<br />
PomBase: a comprehensive online resource for fission yeast.<br />
<em>Nucleic Acids Res.</em> 2012 Jan;40(Database issue):D695-9. Epub 2011 Oct 28.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">PMID:22039153</a> <a href="http://dx.doi.org/DOI:10.1093/nar/gkr853">DOI:10.1093/nar/gkr853</a></p>
<p>You may also wish to cite the most recent PomBase NAR Database Issue paper:</p>
<p>McDowall MD, Harris MA, Lock A, Rutherford K, Staines DM, Bähler J, Kersey PJ, Oliver SG, Wood V.<br />
PomBase 2015: updates to the fission yeast database.<br />
<em>Nucleic Acids Res.</em> 2015 Jan;43(Database issue):D665-61. Epub 2014 Oct 31.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25361970">PMID:25361970</a> <a href="http://dx.doi.org/10.1093/nar/gku1040">DOI:10.1093/nar/gku1040</a></p>
<h3 id="genome-sequence">Genome Sequence</h3>
<p>To cite the <em>S. pombe</em> genome sequence, use:</p>
<p>Wood V, Gwilliam R, Rajandream MA, Lyne M, Lyne R, Stewart A, Sgouros J, Peat N, Hayles J, Baker S, <em>et al.</em><br />
The genome sequence of <em>Schizosaccharomyces pombe</em>.<br />
<em>Nature</em>. 2002 Feb 21;415(6874):871-80. Erratum in: Nature 2003 Jan 2;421(6918):94. Cerrutti L [corrected to Cerutti L].</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/11859360">PMID:11859360</a></p>
<p><a href="http://www.nature.com/nature/journal/v415/n6874/full/nature724.html">Full text at Nature</a></p>
<h3 id="go-annotations">GO Annotations</h3>
<p>To cite the <em>S. pombe</em> Gene Ontology annotation data, use the 2012 NAR Database Issue paper noted above.</p>
<p>(Note that GO annotation for <em>S. pombe</em> has also been previously published, but we now recommend the 2012 NAR paper as it is more up-to-date. Aslett M, Wood V. Gene Ontology annotation status of the fission yeast genome: preliminary coverage approaches 100%. <em>Yeast</em> 2006 Oct 15;23(13):913-9. Erratum in <em>Yeast</em> 2007 May;24(5):457. PMID:17072883)</p>
<h3 id="phenotype-ontology-and-annotations">Phenotype Ontology and Annotations</h3>
<p>To cite the Fission Yeast Phenotype Ontology (FYPO) and/or <em>S. pombe</em> FYPO annotation data, use:</p>
<p>Harris MA, Lock A, **Bähler J, Oliver SG, Wood V.<br />
FYPO: The Fission Yeast Phenotype Ontology.<br />
<em>Bioinformatics</em>. 2013 Jul 1;29(13):1671-8. doi: <a href="http://dx.doi.org/10.1093/bioinformatics/btt266">10.1093/bioinformatics/btt266</a>. Epub 2013 May 8.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/23658422">PMID:23658422</a></p>
<p>Full text at Bioinformatics: <a href="http://bioinformatics.oxfordjournals.org/content/29/13/1671.long">HTML</a> or <a href="http://bioinformatics.oxfordjournals.org/content/29/13/1671.full.pdf+html">PDF</a></p>
<h3 id="canto">Canto</h3>
<p>To cite the online curaton tool Canto, use:</p>
<p>Rutherford KM, Harris MA, Lock A, Oliver SG, Wood V.<br />
Canto: An online tool for community literature curation.<br />
<em>Bioinformatics</em>. 2014 Jun 15;30(12):1791-2. Epub 2014 Feb 25.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/24574118">PMID:24574118</a> <a href="http://dx.doi.org/10.1093/bioinformatics/btu103">DOI: 10.1093/bioinformatics/btu103</a></p>
<h3 id="manually-curated-orthologs">Manually Curated Orthologs</h3>
<p>To cite the manually curated <em>S. cerevisiae</em> or human orthologs (displayed on gene pages or downloaded), use the NAR database paper as noted above. Note that the manually curated orthologs use several data sources including Compara, but are distinct from the Compara results themselves.</p>
<p>Wood V, Harris MA, McDowall MD, Rutherford K, Vaughan BW, Staines DM, Aslett M, Lock A, Bähler J, Kersey PJ, Oliver SG.<br />
PomBase: a comprehensive online resource for fission yeast.<br />
<em>Nucleic Acids Res.</em> 2012 Jan;40(Database issue):D695-9. Epub 2011 Oct 28.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">PMID:22039153</a></p>
<p><a href="http://nar.oxfordjournals.org/content/40/D1/D695.long">Full text at NAR</a></p>
<h3 id="compara">Compara</h3>
<p>Note that the manually curated <em>S. cerevisiae</em> and human orthologs (displayed on gene pages or downloadable) use several data sources in addition to Compara, and are therefore distinct from the Compara data. To cite the <em>S. pombe</em> Compara data, use:</p>
<p>Vilella AJ, Severin J, Ureta-Vidal A, Heng L, Durbin R, Birney E.<br />
EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates.<br />
<em>Genome Res</em> 2009 Feb;19(2):327-35. Epub 2008 Nov 24.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/19029536">PMID:19029536</a></p>
<p><a href="http://genome.cshlp.org/content/19/2/327.long">Full text at Genome Research</a></p>

  </div>
  <div *ngIf="pageName == 'index'">
<h1 id="about-pombase">About PomBase</h1>
<p>PomBase is a model organism database that provides organization of and access to scientific data for the fission yeast <em>Schizosaccharomyces pombe</em>. PomBase supports genomic sequence and features, genome-wide datasets and manual literature curation.</p>
<p>PomBase also provides a community hub for researchers, providing genome statistics, a community curation interface, news, events, documentation and mailing lists.</p>
<hr />
<h2 id="contact-pombase-staff">Contact PomBase staff</h2>
<p>Email: <a routerLink="/mailto:helpdesk@pombase.org">helpdesk@pombase.org</a></p>
<p>Post:</p>
<p>PomBase<br />
Department of Biochemistry<br />
University of Cambridge<br />
80 Tennis Court Road<br />
Cambridge<br />
CB2 1GA<br />
United Kingdom</p>
<h2 id="submit-data">Submit Data</h2>
<ul>
<li>To register a gene name, please see the <a href="submit-data/gene-naming-guidelines">naming guidelines</a> and complete the <a routerLink="/submit-data/registration-form">submission form</a></li>
<li>To submit information on individual genes, use the Contact Curators link provided on the relevant <a href="http://www.pombase.org/">PomBase</a> gene page</li>
<li>To submit sequence-linked data, use the <a href="submit-data/data-submission-form">HTP data submission form</a></li>
<li>To submit large sets of phenotype data, use the <a href="submit-data/phenotype-data-submission-form">phenotype data submission form</a></li>
</ul>
<h2 id="data-versions">Data versions</h2>
<p>Versions of internal and external data loaded into current and past Pombase releases.</p>
<p><a routerLink="/about/version-history">Version history</a></p>
<h2 id="receive-project-updates">Receive project updates</h2>
<p>Subscribe to <a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">Pombelist</a> to receive annotation and data updates</p>
<hr />
<h2 id="pombase-staff">PomBase staff</h2>
<h3 id="university-of-cambridge"><a href="http://www.cam.ac.uk/">University of Cambridge</a></h3>
<ul>
<li>Val Wood (Curator &amp; co-PI)</li>
<li>Midori Harris (Curator)</li>
<li>Kim Rutherford (Programmer)</li>
<li>Steve Oliver (PI)</li>
</ul>
<h3 id="university-college-london"><a href="http://www.ucl.ac.uk/">University College London</a></h3>
<ul>
<li>Antonia Lock (Curator)</li>
<li>Jürg Bähler (co-PI)</li>
</ul>
<hr />
<p>Funding: PomBase is funded by the Wellcome Trust [104967/Z/14/Z].</p>

  </div>
  <div *ngIf="pageName == 'pombase-staff'">
<h2 id="contact-us">Contact us</h2>
<p>Email: <a routerLink="/mailto:helpdesk@pombase.org">helpdesk@pombase.org</a></p>
<p>Post:</p>
<p>PomBase<br />
Department of Biochemistry<br />
University of Cambridge<br />
80 Tennis Court Road<br />
Cambridge<br />
CB2 1GA<br />
United Kingdom</p>

  </div>
  <div *ngIf="pageName == 'resource-metrics'">
<h1 id="resource-metrics">Resource Metrics</h1>
<h2 id="curation">Curation</h2>
<p>The <a href="https://curation.pombase.org/pombe/stats/annotation">Curation Statistics</a> page (currently part of PomBase Canto) provides a set of metrics reflecting the current state of literature curation in PomBase.</p>
<h2 id="web-usage">Web usage</h2>
<p>Statistics forthcoming</p>

  </div>
  <div *ngIf="pageName == 'version-history'">
<h1 id="pombase-data-version-history">PomBase data version history</h1>
<p>This table records versions of software, annotations, etc. used in the numbered releases of Pombase made between September 2012 and January 2017.</p>
<h4 id="past-pombase-releases">Past PomBase Releases</h4>
<table border="1">
 <tbody>
 <tr>
  <td>Release Date</td>
  <td>INSDC</td>
  <td>Gene Build</td>
  <td>Ensembl Software Release</td>
  <td>Annotation Version</td>
  <td>GO GAF</td>
  <td>InterPro</td>
  <td>Manually Curated Ortholog table</td>
 </tr>
 <tr>
  <td><span class="no-break">2017-01-30</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>62</td>
  <td><span class="no-break">2017-03-17</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-10-21</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>61</td>
  <td><span class="no-break">2016-10-21</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-06-01</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>60</td>
  <td><span class="no-break">2016-06-01</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-05-10</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>59</td>
  <td><span class="no-break">2016-05-11</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-04-12</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>58</td>
  <td><span class="no-break">2016-04-12</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-02-11</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!83, EG30</span></td>
  <td>57</td>
  <td><span class="no-break">2016-02-11</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-12-02</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!82, EG29</span></td>
  <td>56</td>
  <td><span class="no-break">2015-12-02</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-09-29</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!81, EG28</span></td>
  <td>55</td>
  <td><span class="no-break">2015-09-29</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-09-07</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!81, EG28</span></td>
  <td>54</td>
  <td><span class="no-break">2015-09-01</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-06-17</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!79, EG26</span></td>
  <td>53</td>
  <td><span class="no-break">2015-06-17</span></td>
  <td>InterProScan5 (5.8-49.0 20-November-2014)</td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-05-27</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-05-28</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!79, EG26</span></td>
  <td>52</td>
  <td><span class="no-break">2015-05-29</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-05-27</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-04-21</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!79, EG26</span></td>
  <td>51</td>
  <td><span class="no-break">2015-04-21</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-03-10</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-03-25</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!78, EG25</span></td>
  <td>50</td>
  <td><span class="no-break">2015-03-25</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-03-10</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-02-19</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!78, EG25</span></td>
  <td>49</td>
  <td><span class="no-break">2015-02-17</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-01-27</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!78, EG25</span></td>
  <td>48</td>
  <td><span class="no-break">2015-01-20</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2014-11-13</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!76, EG23</span></td>
  <td>47</td>
  <td><span class="no-break">2014-11-13</span></td>
  <td><span>InterProScan5 (5.7-48.0 August-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2014-09-16</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!76, EG23</span></td>
  <td>46</td>
  <td><span class="no-break">2014-09-15</span></td>
  <td><span>InterProScan5 (5.7-48.0 August-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2014-08-18</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!76, EG23</span></td>
  <td>45</td>
  <td><span class="no-break">2014-08-18</span></td>
  <td><span>InterProScan5 (5.7-48.0 August-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-07-18</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!75, EG22</span></td>
  <td>44</td>
  <td><span class="no-break">2014-07-18</span></td>
  <td><span>InterProScan5 (5.4-47.0 June-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-07-08</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!75, EG22</span></td>
  <td>43</td>
  <td><span class="no-break">2014-07-08</span></td>
  <td><span>InterProScan5 (5.4-47.0 June-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-05-15</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!75, EG22</span></td>
  <td>42</td>
  <td><span class="no-break">2014-05-13</span></td>
  <td><span>InterProScan5 (5.3-46.0 January-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-03-20</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!74, EG21</span></td>
  <td>41</td>
  <td><span class="no-break">2014-03-20</span></td>
  <td><span>InterProScan5 (5.3-46.0 January-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-02-20</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!74, EG21</span></td>
  <td>40</td>
  <td><span class="no-break">2014-02-20</span></td>
  <td><span>InterProScan5 (5.3-46.0 January-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-12-10</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!73, EG20</span></td>
  <td>39</td>
  <td><span class="no-break">2013-12-04</span></td>
  <td><span>InterProScan5 (<span class="no-break">2013-04-24</span>)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-10-22</span></td>
  <td>ASM294v2</td>
  <td>2.1&nbsp;</td>
  <td><span>e!73, EG20</span></td>
  <td>38</td>
  <td><span class="no-break">2013-11-11</span></td>
  <td><span>InterProScan5 (<span class="no-break">2013-04-24</span>)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-09-16</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!71, EG18</span></td>
  <td>37</td>
  <td><span class="no-break">2013-09-18</span></td>
  <td><span>InterProScan5 (<span class="no-break">2013-04-24</span>)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-06-21</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td>e!71, EG18</td>
  <td>36</td>
  <td><span class="no-break">2013-07-02</span></td>
  <td>InterProScan5 (<span class="no-break">2013-04-24</span>)</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-05-21</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!71, EG18</span>&nbsp;</td>
  <td>35</td>
  <td><span class="no-break">2013-06-06</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-05-07</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!71, EG18</span>&nbsp;</td>
  <td>34</td>
  <td><span class="no-break">2013-04-18</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-04-02</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td>e!70, EG17</td>
  <td>33</td>
  <td><span class="no-break">2013-03-07</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-02-26</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!70, EG17</span>&nbsp;</td>
  <td>31</td>
  <td><span class="no-break">2013-01-16</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2012-11-19</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!69, EG16</span>&nbsp;</td>
  <td>30</td>
  <td><span class="no-break">2012-11-27</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span>2012-10</span></td>
  <td><span>ASM294v2</span></td>
  <td>2.1</td>
  <td><span>e!69, EG16</span>&nbsp;</td>
  <td>29&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><span>2012-09</span></td>
  <td><span>ASM294v2</span></td>
  <td>2.1&nbsp;</td>
  <td><span>e!68, EG15</span>&nbsp;</td>
  <td>28&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 </tbody>
</table>
<h4 id="brief-explanations">Brief explanations</h4>
<table>
<thead>
<tr class="header">
<th>Item</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>INSDC</td>
<td>Accession of the current <em>S. pombe</em> sequence in the International Nucleotide Sequence Data Collaboration (ENA, GenBank, DDBJ) databases</td>
</tr>
<tr class="even">
<td>Gene Build</td>
<td>Version of gene feature annotation</td>
</tr>
<tr class="odd">
<td>Ensembl Software Release</td>
<td>Versions of ensembl (e!) and Ensembl Genomes (EG) data and software used to build PomBase</td>
</tr>
<tr class="even">
<td>Annotation Version</td>
<td>Version of curated PomBase annotations (sequence features, ontology annotations, etc.)</td>
</tr>
<tr class="odd">
<td>GO GAF</td>
<td>Date on which GO annotation data were retrieved from the GO repository</td>
</tr>
<tr class="even">
<td>InterPro</td>
<td>Date and version of InterProScan used to find matches to InterPro entries</td>
</tr>
<tr class="odd">
<td>Manually Curated Ortholog table</td>
<td>Date and version of the manually curated sets of pombe/cerevisiae (and, where included, pombe/human) orthologs</td>
</tr>
</tbody>
</table>

  </div>
  </div>
</div>
<div *ngIf="section == 'browse-curation'">
  <div class="docs-menu">
<h3 id="browse-curation"><a routerLink="/browse-curation">Browse curation</a></h3>
<ul>
<li><a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</a></li>
<li><a routerLink="/browse-curation/fission-yeast-go-slim-terms">GO slim terms</a></li>
<li><a routerLink="/browse-curation/drugs-known-pombe-targets">Drugs with known targets</a></li>
</ul>

  </div>
  <div class="docs-content">
  <div *ngIf="pageName == 'drugs-known-pombe-targets'">
<h1 id="drugs-with-known-s.-pombe-targets">Drugs with known S. pombe targets</h1>
<p>This table lists drugs that affect <em>S. pombe</em>, and summarizes what is known about their targets or mechanisms of action.</p>
<table>
<thead>
<tr class="header">
<th>Drug</th>
<th>Targeted gene product, process, etc.</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1,10-phenanthroline</td>
<td>inhibits RNA polymerases II and III</td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27518095">PMID:27518095</a></td>
</tr>
<tr class="even">
<td>CT2108A, CT2108B</td>
<td>fatty acid synthase</td>
<td> </td>
</tr>
<tr class="odd">
<td>alpha-amanitin</td>
<td>RNA polymerase II inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>alverine citrate</td>
<td>proteasome/lipid synthesis?</td>
<td> </td>
</tr>
<tr class="odd">
<td>amiloride</td>
<td>sodium transporter inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>amphotericin B</td>
<td>sterol biosynthesis inhibitor; binds ergosterol; forms membrane pores</td>
<td> </td>
</tr>
<tr class="odd">
<td>anisomycin</td>
<td>translation inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>antifungal azoles</td>
<td>sterol biosynthesis inhibitors</td>
<td> </td>
</tr>
<tr class="odd">
<td>arborcandin C</td>
<td>1,3-beta-glucan synthase</td>
<td> </td>
</tr>
<tr class="even">
<td>benomyl</td>
<td>destabilizes microtubules</td>
<td> </td>
</tr>
<tr class="odd">
<td>bleomycins</td>
<td>fungal cell wall septation and cytokinesis inhibitors</td>
<td> </td>
</tr>
<tr class="even">
<td>brefeldin A</td>
<td>inhibits intracellular transport</td>
<td> </td>
</tr>
<tr class="odd">
<td>calcofluor white</td>
<td>cell wall synthesis</td>
<td> </td>
</tr>
<tr class="even">
<td>camptothecin</td>
<td>topoisomerase 1 (Top1) inhibitor</td>
<td> </td>
</tr>
<tr class="odd">
<td>canavanine</td>
<td>naturally occuring (alfafa) L-arginine antimetabolite</td>
<td> </td>
</tr>
<tr class="even">
<td>chloramphenicol</td>
<td>protein synthesis inhibitor</td>
<td> </td>
</tr>
<tr class="odd">
<td>compactin</td>
<td>ergosterol biosynthesis</td>
<td> </td>
</tr>
<tr class="even">
<td>Cutin-1</td>
<td>fatty acid synthase beta subunit (Fas1); inhibits nuclear envelope expansion during mitosis</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26869222">PMID:26869222</a></td>
</tr>
<tr class="odd">
<td>cycloheximide</td>
<td>translation inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>cyclosporin A</td>
<td>binds cyclophilin</td>
<td> </td>
</tr>
<tr class="odd">
<td>dihydromotuporamine C</td>
<td>proteasome/lipid synthesis?</td>
<td> </td>
</tr>
<tr class="even">
<td>echinocandin</td>
<td>cell wall synthesis</td>
<td> </td>
</tr>
<tr class="odd">
<td>fenpropimorph</td>
<td>proteasome/lipid synthesis?</td>
<td> </td>
</tr>
<tr class="even">
<td>hydroxyurea</td>
<td>inhibits ribonucleotide reductase; slows DNA synthesis; causes replication fork stalling</td>
<td> </td>
</tr>
<tr class="odd">
<td>hygromycin B</td>
<td>translation inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>isopropyl N-3-chlorophenyl carbamate</td>
<td>spindle poison</td>
<td> </td>
</tr>
<tr class="odd">
<td>itraconazole</td>
<td>spindle poison</td>
<td> </td>
</tr>
<tr class="even">
<td>latrunculin</td>
<td>inhibits actin polymerization</td>
<td> </td>
</tr>
<tr class="odd">
<td>leptomycin B</td>
<td>nuclear export receptor Crm1</td>
<td> </td>
</tr>
<tr class="even">
<td>lovastatin</td>
<td>competitive HMG CoA-reductase inhibitor</td>
<td> </td>
</tr>
<tr class="odd">
<td>methyl-2-benzimidazolecarbamate</td>
<td>reversible MT inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>monensin</td>
<td>intracellular transport</td>
<td> </td>
</tr>
<tr class="odd">
<td>natamycin</td>
<td>blocks fungal growth by binding specifically to ergosterol without permeabilizing the membrane</td>
<td> </td>
</tr>
<tr class="even">
<td>nystatin</td>
<td>ergosterol biosynthesis; antifungal; ergosta-5,7,22,24,(28)-tetraenol binding</td>
<td> </td>
</tr>
<tr class="odd">
<td>oxazolidinones</td>
<td>protein synthesis inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>phleomycin</td>
<td>causes DNA strand breaks</td>
<td> </td>
</tr>
<tr class="odd">
<td>pravastatin</td>
<td>HMG-CoA reductase inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>ricin</td>
<td>disrupts translation by inactivating the ribosome</td>
<td> </td>
</tr>
<tr class="odd">
<td>sanglifehrin A</td>
<td>binds cyclophilin</td>
<td> </td>
</tr>
<tr class="even">
<td>spliceostatin A</td>
<td>binds SF3b components and inhibits splicing</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17961508">PMID:17961508</a></td>
</tr>
<tr class="odd">
<td>staurosporine</td>
<td>protein kinase inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>tagetitoxin/tagetin</td>
<td>RNA polymerase III inhibitor</td>
<td> </td>
</tr>
<tr class="odd">
<td>thialysine</td>
<td>naturally occuring lysine analogue</td>
<td> </td>
</tr>
<tr class="even">
<td>trichostatin A (TSA)</td>
<td>histone deacetylase inhibitor</td>
<td> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19723888">PMID:19723888</a></td>
</tr>
<tr class="odd">
<td>tunicamycin</td>
<td>inhibits N-glycosylation</td>
<td> </td>
</tr>
<tr class="even">
<td>zaragozic acid</td>
<td>inhibits N-glycosylation</td>
<td> </td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'fission-yeast-go-slim-terms'">
<h1 id="fission-yeast-go-slim-terms">Fission Yeast GO slim terms</h1>
<p>The table below shows terms in the current fission yeast biological process GO slim, and the number of annotations to each term. GO IDs link to the QuickGO browser, where you can explore the ontology and annotations further. Icons beside each GO term link to <a href="http://www.esyn.org/">esyN</a>, which provides a graphical view of interactions for the genes from the PomBase High Confidence Physical Interaction Network (HCPIN) dataset. Only the subset of genes linked into the interaction network will be displayed in the esyN network view. The annotation totals link to pages with information about the term and a list of annotated genes.</p>
<p>Further information is available from:</p>
<ul>
<li><a routerLink="/documentation/pombase-go-slim-documentation">PomBase GO slim documentation</a></li>
<li><a routerLink="/documentation/high-confidence-physical-interaction-network">PomBase HCPIN documentation</a></li>
<li><a href="http://www.esyn.org/tutorial.html">esyN documentation</a></li>
<li><a routerLink="/browse-curation/fission-yeast-go-slimming-tips">PomBase tips for using GO slim terms and annotations</a></li>
<li><a href="http://geneontology.org/page/go-slim-and-subset-guide">GO Slim Guide at the Gene Ontology website</a></li>
</ul>
<p>Note that both proteins and RNAs can be annotated to GO terms, and the tables on this page include annotated RNAs. For some GO terms, notably 'cytoplasmic translation', RNAs make up a significant proportion of the total annotated genes.</p>
<p><app-go-slim-table></app-go-slim-table></p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-go-slimming-tips'">
<h1 id="pombase-go-slim-usage-tips">PomBase GO Slim Usage Tips</h1>
<p>=== General GO Slim hints ===</p>
<p>These points will help you understand GO slims, and highlight some features of the fission yeast slim terms and annotations.</p>
<ol style="list-style-type: decimal">
<li>Note that the counts in the &quot;Genes&quot; column of the GO slim table are not additive, because many gene products are annotated to multiple terms.</li>
<li>It not possible to create a slim with no overlaps between terms. Although the <em>S. pombe</em> slim has been defined to include biologically informative terms, and minimise overlaps between terms, large overlaps cannot be entirely avoided. For example, most of the gene products annotated to <em>signal transduction</em> are also annotated other terms in the slim.</li>
<li>As a consequence of points 1 and 2, GO slim annotation summaries should not be presented using pie charts. Although a pie chart could show the fraction of total annotations for any slim term, it can too easily be mistaken for the fraction of total annotated gene products, which is not the same.</li>
<li>It is difficult to define a slim which includes all annotated gene products without including terms with very small numbers of annotations (for example, the cell aging branch has very few annotations), or very high level terms which are not particularly biologically informative (e.g. cellular process). Because we have opted not to include such terms, some gene products are annotated to process terms but do not appear in the slim annotation set.</li>
<li>Bear in mind that both proteins and RNAs can be annotated to GO terms. If you are working only with proteins you will need to make adjustments for this. For example, many tRNAs and rRNAs are annotated to cytoplasmic translation, and are therefore included in the slim set.</li>
<li>There is a difference between &quot;unknown&quot; and &quot;unannotated&quot;. All fission yeast and budding yeast gene products have been assessed and are classed as &quot;unknown&quot; for biological process if no biological process information is found (experimental or inferred). If you are making comparisons with other organisms, remember that it is possible that not all gene products have been assessed and that the &quot;unknown&quot; set is underestimated.</li>
<li>The default S. pombe slim includes all evidence codes for fission yeast. The evidence code <strong>IEA</strong> (inferred from electronic annotation) is often considered to be less accurate than other evidence codes, but it is very useful for increasing the coverage of some of the high level GO terms. Accurate annotation counts for some terms currently depends on including this evidence code (for example, there are 26 gene products annotated to transmembrane transport with IEA evidence, which are not yet covered by a manual annotation). For fission yeast, the IEA annotations improve slim coverage, but only represent a small number of annotations (654 biological process annotations as of September 2011), and have a low rate of false positives. We therefore recommend that you include them.</li>
<li>If you are making comparisons with budding yeast (or other organisms), you should consider excluding the evidence code <strong>RCA</strong>. This evidence code is used for functional predictions, and has a very high rate of false positives (for example, including RCA for budding yeast hugely will greatly and artificially inflate the number of annotations to translation).</li>
</ol>
<p>=== Creating a user-defined slim ===</p>
<ol style="list-style-type: decimal">
<li>You can create your own slim, or retrieve slim annotations for a gene set, using online slimming tools such as the <a href="http://go.princeton.edu/cgi-bin/GOTermMapper">GOTermMapper</a> at Princeton.</li>
<li>When creating a slim for the entire genome, you should try to ensure that it covers as many annotated genes in your set as possible (see #3 in list above). You should be aware of how many genes are annotated but not in your slim, and how many are &quot;unknown&quot; (i.e., annotated only to the root node; see #5 in list above).</li>
<li>For display purposes you usually want to keep the number of terms as small as possible to convey your results. However, you should ensure that the terms you include are specific enough to capture biologically relevant information. Many terms (e.g. metabolic process (2915 annotations), cellular process (4083 annotations)) are too general for the purpose of most slim-based analyses.</li>
<li>On a related note, if you are using your slim for data analysis (e.g. to summarize an enrichment), you should ensure that the terms are specific enough to demonstrate their relevance to the biological topic of interest. For example, lumping all genes involved in transport my mask overrepresentation of transmembrane transport vs. underrepresentation of vesicle-mediated transport in your results set, so you need to ensure that the slim has categories to represent your results effectively.</li>
<li>Most current implementations of software to create &quot;GO slims&quot; include the <em>regulates</em> relationship by default, so that (for example) genes involved in <em>regulation of cytokinesis</em> will be included with the set of genes annotated to <em>cytokinesis</em>. See the <a href="http://www.geneontology.org/GO.ontology.relations.shtml">GO Ontology Relations documentation</a> for further information about relationships in GO. The annotation totals presented for the <a routerLink="/browse-curation/fission-yeast-go-slim-terms">PomBase default S. pombe slim</a>, by contrast, are calculated both explicitly including or excluding the genes which are involved in a process via regulation only. We expect this distinction to be available in future versions of slimming software.</li>
</ol>

  </div>
  <div *ngIf="pageName == 'fission-yeast-phenotype-ontology'">
<h1 id="fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</h1>
<p>PomBase curators develop and use the Fission Yeast Phenotype Ontology (FYPO) to annotate phenotypes of mutant alleles and of over- or under-expressing wild type genes.</p>
<p>The contents and structure of FYPO are described on the <a href="http://curation.pombase.org/pombase-trac/wiki/FissionYeastPhenotypeOntology">FYPO page of the PomBase wiki</a> and in the publication:</p>
<p>Harris MA, Lock A, Bähler J, Oliver SG, Wood V. FYPO: The Fission Yeast Phenotype Ontology. Bioinformatics. 2013 July 1;29(13): 1671–1678. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23658422">PubMed abstract (PMID:23658422)</a> or <a href="http://bioinformatics.oxfordjournals.org/content/29/13/1671.long">Full text at Bioinformatics</a></p>
<h2 id="browsing-fypo-terms">Browsing FYPO terms</h2>
<p>At present, FYPO is included collections such as the National Center for Biomedical Ontology's <a href="http://bioportal.bioontology.org/">BioPortal</a> and EBI's <a href="http://www.ebi.ac.uk/ontology-lookup/">Ontology Lookup Service (OLS)</a>, both of which allow searching and browsing.</p>
<ul>
<li>BioPortal <a href="http://bioportal.bioontology.org/ontologies/FYPO">FYPO summary page</a></li>
<li>BioPortal <a href="http://bioportal.bioontology.org/ontologies/FYPO/?p=classes">FYPO terms</a> (the term display supports tree-viewer browsing, text searches, and a graphical display)</li>
<li><a href="http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=FYPO">OLS FYPO browser</a></li>
</ul>
<p>We hope to deploy a browser at PomBase in the not-too-distant future that will include annotations as well as terms.</p>
<h2 id="finding-fypo-terms-and-annotations-in-pombase">Finding FYPO terms and annotations in PomBase</h2>
<p>FYPO terms are displayed on PomBase gene pages, along with supporting evidence and allele and expression details, as described in the PomBase gene page documentation.</p>
<p>The PomBase advanced search can use either term names or IDs to search FYPO, and returns a list of genes that have alleles annotated to the specified term or any of its descendants. For example, see the <a href="http://www.pombase.org/faqs/can-i-get-list-essential-pombe-genes">FAQ on finding essential genes</a>.</p>
<h2 id="curating-phenotypes-with-fypo">Curating phenotypes with FYPO</h2>
<p>The PomBase online curation tool, <a href="http://curation.pombase.org/pombe">Canto</a>, uses FYPO terms for annotation of S. pombe phenotypes. <a href="http://curation.pombase.org/pombe/docs/fypo_annotation">Phenotype curation documentation</a> is available via Canto.</p>
<h2 id="submitting-bulk-data">Submitting Bulk Data</h2>
<p>If you have a large set of phenotype data to submit, you may want to do a bulk submission. See the documentation on the <a href="http://www.pombase.org/submit-data/phenotype-data-bulk-upload-format">recommended file format</a>, and use the Phenotype Data Submission Form.</p>

  </div>
  <div *ngIf="pageName == 'index'">


  </div>
  </div>
</div>
<div *ngIf="section == 'community'">
  <div class="docs-menu">
<h3 id="community"><a routerLink="/community">Community</a></h3>
<ul>
<li><a routerLink="/community/fission-yeast-community-curation-project">Community curation project</a></li>
<li><a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">Mailing list</a></li>
<li><a routerLink="/community/internet-resources">External resources</a></li>
</ul>

  </div>
  <div class="docs-content">
  <div *ngIf="pageName == 'fission-yeast-community-curation-pilot-project'">
<h2 id="participating-groups-annotated-papers">Participating groups &amp; Annotated papers</h2>
<p>This table summarizes the results of the pilot project that preceded the <a routerLink="/community/fission-yeast-community-curation-project">Fission Yeast Community Curation Project</a>. The first phase of the project began in January 2009, and the second in November 2009. We thank all of the researchers who participated for making the pilot project a success.</p>
<table border="0" cellpadding="2" width="100%">
 <thead>
  <tr>
  <th><span>PMID</span></th>
  <th>Laboratory</th>
  <th>Submitter if different</th>
  <th>GO terms</th>
  <th>Interactions</th>
  <th>Phenotypes</th>
  <th>Other <span>curation</span></th>
  <th>Total</th>
  <th>Date</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td>PHASE 1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19101542">19101542</a></td>
  <td>Peter Folk</td>
  <td>Martin <span>Převorovský</span></td>
  <td>19</td>
  <td>0</td>
  <td>15</td>
  <td>3</td>
  <td>37</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19098712">19098712</a></td>
  <td>Juan Mata</td>
  <td>&nbsp;</td>
  <td>3</td>
  <td>7</td>
  <td>0</td>
  <td>1</td>
  <td>11</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19076239">19076239</a></td>
  <td>Paul Russell</td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>0</td>
  <td>2</td>
  <td>2</td>
  <td>9</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18675827">18675827</a></td>
  <td>Per <span>Sunnerhagen</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>1</td>
  <td>3</td>
  <td>0</td>
  <td>6</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17353264">17353264</a></td>
  <td>Chris <span>Norbury</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>2</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19057642">19057642</a></td>
  <td>Luis <span>Rokeach</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19056896">19056896</a></td>
  <td>Fred Winston</td>
  <td>Dom <span>Helmlinger</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19041767">19041767</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>John Burg</td>
  <td>4</td>
  <td>2</td>
  <td>2</td>
  <td>1</td>
  <td>9</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18556659">18556659</a></td>
  <td>Kevin <span>Hardwick</span></td>
  <td>&nbsp;</td>
  <td>16</td>
  <td>16</td>
  <td>2</td>
  <td>0</td>
  <td>34</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18430926">18430926</a></td>
  <td>Charlie Hoffman</td>
  <td>&nbsp;</td>
  <td>4</td>
  <td>0</td>
  <td>2</td>
  <td>0</td>
  <td>6</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17304215">17304215</a></td>
  <td><span>Hiroshi</span> <span>Iwasaki</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>5</td>
  <td>9</td>
  <td>19</td>
  <td>35</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18426916">18426916</a></td>
  <td><span>Hiro</span> <span>Yamano</span></td>
  <td>Michelle Tricky</td>
  <td>2</td>
  <td>1</td>
  <td>2</td>
  <td>1</td>
  <td>6</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19037101">19037101</a></td>
  <td>Nick <span>Rhind</span></td>
  <td>Nicholas Willis</td>
  <td>7</td>
  <td>6</td>
  <td>7</td>
  <td>0</td>
  <td>20</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19001497">19001497</a></td>
  <td>Ken <span>Sawin</span></td>
  <td>Lynda <span>Groocock</span></td>
  <td>9</td>
  <td>6</td>
  <td>2</td>
  <td>1</td>
  <td>18</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18368917">18368917</a></td>
  <td>Steve Aves</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18799626">18799626</a></td>
  <td>Takashi <span>Toda</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>3</td>
  <td>3</td>
  <td>0</td>
  <td>8</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19109429">19109429</a></td>
  <td>Stephen <span>Kearsey</span></td>
  <td>&nbsp;</td>
  <td>9</td>
  <td>2</td>
  <td>2</td>
  <td>0</td>
  <td>13</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17688408">17688408</a></td>
  <td>Susan <span>Forsburg</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17409062">17409062</a></td>
  <td>Iain Hagan</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19139281">19139281</a></td>
  <td>Tony Carr</td>
  <td>Edgar <span>Hartsuiker</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-15</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18065650">18065650</a></td>
  <td>Per <span>Sunnerhagen</span></td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>2</td>
  <td>4</td>
  <td>0</td>
  <td>11</td>
  <td>-</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18406331">18406331</a></td>
  <td><span>Kaz</span> <span>Shiozaki</span></td>
  <td>&nbsp;</td>
  <td>4</td>
  <td>4</td>
  <td>1</td>
  <td>1</td>
  <td>10</td>
  <td><span class="no-break">2009-01-15</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19139265">19139265</a></td>
  <td>Kathy Gould</td>
  <td>Rachel Roberts</td>
  <td>8</td>
  <td>11</td>
  <td>1</td>
  <td>1</td>
  <td>21</td>
  <td><span class="no-break">2009-01-17</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19197239">19197239</a></td>
  <td>Benoit <span>Arcangioli</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>4</td>
  <td>0</td>
  <td>6</td>
  <td><span class="no-break">2009-02-10</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19328067">19328067</a></td>
  <td>Rob Fisher</td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>4</td>
  <td>2</td>
  <td>25</td>
  <td>36</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19330768">19330768</a></td>
  <td><span>Katya</span> <span>Grishchuk</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19217404">19217404</a></td>
  <td>Robin <span>Allshire</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19217403">19217403</a></td>
  <td>Paul Russell</td>
  <td>Jessica Williams</td>
  <td>5</td>
  <td>7</td>
  <td>3</td>
  <td>0</td>
  <td>15</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19362535">19362535</a></td>
  <td>Janet Partridge</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19250904">19250904</a></td>
  <td><span>Songtao</span> <span>Jia</span></td>
  <td><span>Bahrat</span> Reddy</td>
  <td>4</td>
  <td>2</td>
  <td>2</td>
  <td>2</td>
  <td>10</td>
  <td><span class="no-break">2009-03-01</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19373772">19373772</a></td>
  <td>Jeremy <span>Hyams</span></td>
  <td>Isabelle <span>Jourdain</span></td>
  <td>5</td>
  <td>1</td>
  <td>6</td>
  <td>0</td>
  <td>12</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19299465">19299465</a></td>
  <td>Sara Mole</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19299465">19299465</a></td>
  <td>Julie Cooper</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19286980">19286980</a></td>
  <td><span>Stevan</span> Marcus</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19033386">19033386</a></td>
  <td>Iain Hagan</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19351719">19351719</a></td>
  <td>Luis <span>Rokeach</span></td>
  <td><span>Pascale</span> Beauregard</td>
  <td>2</td>
  <td>1</td>
  <td>1</td>
  <td>0</td>
  <td>4</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19160458">19160458</a></td>
  <td>Ken <span>Sawin</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19211838">19211838</a></td>
  <td>Nick <span>Rhind</span></td>
  <td>Mary Porter-Goff</td>
  <td>2</td>
  <td>2</td>
  <td>8</td>
  <td>0</td>
  <td>12</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19273851">19273851</a></td>
  <td>Ramsay <span>Macfarlane</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>0</td>
  <td>1</td>
  <td>0</td>
  <td>2</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19243310">19243310</a></td>
  <td>Takashi <span>Toda</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>4</td>
  <td>1</td>
  <td>0</td>
  <td>7</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19158663">19158663</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>1</td>
  <td>1</td>
  <td>0</td>
  <td>4</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19237545">19237545</a></td>
  <td><span>Henar</span> <span>Valdivieso</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>1</td>
  <td>3</td>
  <td>0</td>
  <td>5</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18562692">18562692</a></td>
  <td>Jonathan Miller</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16143612">16143612</a></td>
  <td>Charlie Hoffman</td>
  <td>&nbsp;</td>
  <td>0</td>
  <td>8</td>
  <td>0</td>
  <td>0</td>
  <td>8</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15448137">15448137</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>15</td>
  <td>0</td>
  <td>0</td>
  <td>5</td>
  <td>20</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18375981">18375981</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>1</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9391101">9391101</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>2</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7518718">7518718</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>2</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7958849">7958849</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>7</td>
  <td>6</td>
  <td>0</td>
  <td>0</td>
  <td>13</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19542312">19542312</a></td>
  <td>John Armstrong</td>
  <td>James <span>Dodgeson</span></td>
  <td>18</td>
  <td>0</td>
  <td>16</td>
  <td>0</td>
  <td>34</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>PHASE 2</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19620282">19620282</a></td>
  <td>Simon Whitehall</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19643199">19643199</a></td>
  <td><span>Jeramy</span> <span>Hyams</span></td>
  <td>Isabelle <span>Jourdain</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19436749">19436749</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19627505">19627505</a></td>
  <td><span>Henar</span> <span>Valdivieso</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>4</td>
  <td>5</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19664060">19664060</a></td>
  <td>Stuart <span>Macneill</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19686603">19686603</a></td>
  <td>Stuart <span>Macneill</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19666000">19666000</a></td>
  <td><span>Murakami</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19646873">19646873</a></td>
  <td><span>Fulvia</span> Verde</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19714215">19714215</a></td>
  <td><span>Jurg</span> <span>Bahler</span></td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>4</td>
  <td>5</td>
  <td>2</td>
  <td>16</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18923417">18923417</a></td>
  <td>Nick <span>Boddy</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18667531">18667531</a></td>
  <td>Nick <span>Boddy</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19686339">19686339</a></td>
  <td>Luis <span>Rokeach</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17114925">17114925</a></td>
  <td><span>Danesh</span> <span>Moazed</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19707600">19707600</a></td>
  <td>Felicity Watts</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19394293">19394293</a></td>
  <td><span>Danesh</span> <span>Moazed</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18574244">18574244</a></td>
  <td>Peter <span>Baumann</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19520858">19520858</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>&nbsp;</td>
  <td>9</td>
  <td>1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>10</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18503029">18503029</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>1</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19427212">19427212</a></td>
  <td>Anna <span>Paoletti</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18223116">18223116</a></td>
  <td>Simon <span>Labbe</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19915076">19915076</a></td>
  <td>Simon <span>Labbe</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19571115">19571115</a></td>
  <td>Shelly <span>Sazer</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18272791">18272791</a></td>
  <td>Rosa <span>Aligue</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19838064">19838064</a></td>
  <td>Jan <span>Paluh</span></td>
  <td>&nbsp;</td>
  <td>3</td>
  <td>29</td>
  <td>9</td>
  <td>4</td>
  <td>55</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19696784">19696784</a></td>
  <td>Takashi <span>Toda</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>3</td>
  <td>2</td>
  <td>&nbsp;</td>
  <td>7</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17409354">17409354</a></td>
  <td>Elena Hidalgo</td>
  <td>&nbsp;</td>
  <td>14</td>
  <td>7</td>
  <td>3</td>
  <td>2</td>
  <td>26</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19416828">19416828</a></td>
  <td>Jacob <span>Dalgaard</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19473263">19473263</a></td>
  <td>Mathias <span>Sipiczki</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19836238">19836238</a></td>
  <td>Caroline Wilkinson</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18252721">18252721</a></td>
  <td>Caroline Wilkinson</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17986863">17986863</a></td>
  <td>Erik <span>Boye</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>3</td>
  <td>0</td>
  <td>5</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=20531409">20531409</a></td>
  <td>Juan Mata</td>
  <td>&nbsp;</td>
  <td>3</td>
  <td>90</td>
  <td>0</td>
  <td>0</td>
  <td>93</td>
  <td>N/A</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19720063">19720063</a></td>
  <td><span>Nimisha</span> Sharma</td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>3</td>
  <td>3</td>
  <td>2</td>
  <td>10</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19625445">19625445</a></td>
  <td>Jose <span>Cansado</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19486165">19486165</a></td>
  <td><span>Takayoshi</span> <span>Kuno</span></td>
  <td><span>Yue</span> Fang</td>
  <td>2</td>
  <td>10</td>
  <td>4</td>
  <td>0</td>
  <td>16</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19713940">19713940</a></td>
  <td><span>Kentaro</span> <span>Nakano</span></td>
  <td><span>Masakatsu</span> <span>Takaine</span></td>
  <td>4</td>
  <td>2</td>
  <td>2</td>
  <td>0</td>
  <td>8</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19409973">19409973</a></td>
  <td>Kurt <span>Runge</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 </tbody>
</table>

  </div>
  <div *ngIf="pageName == 'fission-yeast-community-curation-project'">
<h1 id="the-fission-yeast-community-curation-project">The Fission Yeast Community Curation Project</h1>
<p>Comprehensive manual curation of the S. pombe literature is a primary goal of the PomBase project, and among the most important activities of the curation staff. Increasing numbers of new publications, and a large backlog of older papers, however, mean that complete literature curation will require input from the research community as well as the efforts of dedicated professional curators.</p>
<p>The PomBase Community Curation system enables researchers to contribute annotations directly to PomBase based on their publications using Canto, a web-based tool that allows both curators and researchers to create annotations. Canto supports GO, phenotype, interaction, and modification annotations, and can be configured for use with other ontologies as the need arises. Annotations made in the community curation system will be prioritized for inclusion in PomBase and will therefore also be more rapidly disseminated to other databases (e.g. GenBank/ENA/DDBJ, UniProtKB, BioGRID and GO), making data from annotated papers widely visible.</p>
<p><strong>How to contribute</strong>: For newly published papers, PomBase curators email authors inviting them to participate, with specific instructions and links. Lab members are also welcome to evaluate existing annotations and create new annotations from past publications — use the PubMed ID search on the <a href="https://curation.pombase.org/pombe">main PomBase Canto page</a>. Anyone can also try the <a href="https://curation.pombase.org/demo">demo version of Canto</a>.</p>
<p><strong>Progress and attribution</strong>: <a href="https://curation.pombase.org/pombe/stats/annotation">Curation statistics</a> track S. pombe papers curated in Canto by PomBase and community curators, and the annotations obtained. Canto will soon allow contributors to register <a href="http://orcid.org/">ORCID</a> identifiers in the system, which will support more formalised attribution of community contributions to PomBase, and thereby enable researchers to cite community curation in institutional reports, funding applications, etc. To supplement the existing permanent publication-centred links in Canto, we are currently investigating possible ways to link to external resources such as ORCID or OpenRIF via ORCID IDs.</p>
<p>For a bit of historical perspective, you can browse the results of the <a routerLink="/community/fission-yeast-community-curation-pilot-project">Community Curation Pilot Project</a> that took place in 2009.</p>

  </div>
  <div *ngIf="pageName == 'index'">
<h1 id="fission-yeast-community">Fission Yeast Community</h1>
<ul>
<li><a routerLink="/community/fission-yeast-community-curation-project">Community curation project</a>, including the old <a routerLink="/community/fission-yeast-community-curation-pilot-project">pilot project</a></li>
<li><a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">Mailing list</a></li>
<li><a routerLink="/community/internet-resources">External resources</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'internet-resources'">
<h2 id="s.-pombe-community-websites"><em>S. pombe</em> Community Websites</h2>
<ul>
<li><a href="http://www-bcf.usc.edu/~forsburg/pombeweb.html">PombeWeb</a> maintained by Susan Forsburg
<ul>
<li><a href="http://www-bcf.usc.edu/~forsburg/pombeweb.html#people">Fission yeast researchers</a> at PombeWeb</li>
</ul></li>
<li><a href="http://www-bcf.usc.edu/~forsburg/main.html">PombeNet</a> maintained by Susan Forsburg
<ul>
<li><a href="http://www-bcf.usc.edu/~forsburg/main.html">About pombe</a> at PombeNet</li>
<li><a href="http://www-bcf.usc.edu/~forsburg/plasmids.html">Working with pombe</a> at PombeNet</li>
</ul></li>
<li><a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/erss.cgi?rss_guid=1ToPSyjDNLLW2R_UIOHFTxnRdlJouyMeQX4YfwMFXtO8d87M4r">PubMed RSS Feed</a> Feed for current <em>S. pombe</em> related publications.</li>
</ul>
<h2 id="ontologies">Ontologies</h2>
<ul>
<li><a href="http://www.geneontology.org/">The Gene Ontology Consortium</a> controlled vocabulary</li>
<li><a href="http://amigo.geneontology.org/">AmiGO</a> Search and browse GO annotation</li>
<li>The <a href="http://www.ebi.ac.uk/quickgo/">QuickGO</a> browser at EBI</li>
<li><a href="http://www.yeastgenome.org/help/function-help/gene-ontology-go">SGD GO documentation</a> to help SGD users become familiar with GO</li>
<li><a href="http://vortex.cs.wayne.edu/projects.htm#Onto-Express">Onto-Express</a> find significant GO terms in lists of genes</li>
<li><a href="http://go.princeton.edu/cgi-bin/GOTermFinder">GOTermFinder</a> find significant GO terms in lists of genes</li>
<li><a href="http://bahlerweb.cs.ucl.ac.uk/cgi-bin/GLA/GLA_input">AnGeLi: Analysis of Gene Lists</a> (UCL) uses GO, phenotypes, interactions, etc.</li>
<li><a href="http://www.sequenceontology.org/">Sequence Ontology</a></li>
</ul>
<h2 id="interactions-pathways-and-networks">Interactions, Pathways and Networks</h2>
<ul>
<li><a href="http://www.thebiogrid.org/">BioGRID</a> Genetic and physical interactions</li>
<li><a href="http://www.ebi.ac.uk/intact/pages/interactions/interactions.xhtml?query=species:4896">Interactions (IntAct)</a> Known interactions for <em>S. pombe</em> from the IntAct database</li>
<li><a href="http://reactome.org/cgi-bin/eventbrowser?DB=gk_current&FOCUS_SPECIES=Schizosaccharomyces%20pombe&ID=68324&">Reactome pathway data</a> <em>S. pombe</em> entry point.</li>
<li><a href="http://www.esyn.org/index.php">esyN</a> Web-based tool for building, sharing, and viewing network data</li>
<li><a href="http://string-db.org/">STRING</a> Protein–protein interaction network database</li>
</ul>
<h2 id="sequence-analysis-tools">Sequence Analysis Tools</h2>
<h3 id="general-tools">General tools</h3>
<ul>
<li><a href="http://gbrowse.org/wiki/index.php/Schizosaccharomyces_pombe">GBrowse Wiki</a> for <em>S. pombe</em></li>
<li><a href="http://www.sanger.ac.uk/science/tools/artemis">Artemis</a> Sequence viewer and analysis tool</li>
<li><a href="http://www.cbs.dtu.dk/services/GenomeAtlas-2.0/show-kingdom.php?kingdom=Eukaryotes&KLquickSearch=pombe">Genome Atlas</a> DNA structural atlas</li>
<li><a href="http://www.ncbi.nlm.nih.gov/mapview/map_search.cgi?chr=spombe.inf">Map viewer</a> at the NCBI</li>
<li><a href="http://megasun.bch.umontreal.ca/People/lang/species/spo/spombe.html">Mitochondrial Genome Information</a></li>
</ul>
<h3 id="sequence-features-protein-families-orthologues">Sequence Features, Protein Families, Orthologues</h3>
<ul>
<li>Todd Lowe's <em>S. pombe</em> genomic <a href="http://gtrnadb.ucsc.edu/GtRNAdb2/genomes/eukaryota/Schi_pomb_972h/">tRNA analysis</a></li>
<li><a href="http://wardlab.cbs.umn.edu/pombe/"><em>S. pombe</em> membrane protein library</a></li>
<li><a href="http://www.rsat.eu/">RSAT</a> Regulatory sequence analysis tools (includes <a href="http://rsat-tagc.univ-mrs.fr/rsat/">Fungi</a> server)</li>
<li><a href="http://pfam.xfam.org/">Pfam</a> protein families (search or browse)</li>
<li><a href="http://supfam.org/SUPERFAMILY/cgi-bin/gen_list.cgi?genome=iz">SCOP</a> Superfamily assignments</li>
<li><a href="https://rostlab.org/owiki/index.php/PredictNLS">PredictNLS</a> Nuclear localization signal prediction</li>
<li><a href="http://www.bahlerlab.info/YOGY/">YOGY</a> A resource for retrieving orthologous proteins</li>
<li><a href="http://epd.vital-it.ch/S_pombe/S_pombe_database.php">EPD</a> Eukaryotic Promoter Database, <em>S. pombe</em> section</li>
</ul>
<h2 id="gene-expression">Gene Expression</h2>
<ul>
<li><a href="http://www.bahlerlab.info/cgi-bin/SPGE/geexview">Gene Expression Viewer</a> at UCL</li>
<li><a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/buildDetails?atlas_build_id=355"><em>S. pombe</em> PeptideAtlas</a> compendium of peptides identified in tandem mass spectrometry proteomics experiments</li>
</ul>
<h2 id="experimental-tools">Experimental tools</h2>
<ul>
<li><a href="http://www.massgeneral.org/cancer/research/motamediresources.aspx">PRIMED</a> Primers for deleting and tagging yeast genes (<em>S. pombe</em> and <em>S. cerevisiae</em>; downloadable spreadsheets)</li>
</ul>
<h3 id="strains-and-constructs">Strains and Constructs</h3>
<ul>
<li><a href="http://yeast.lab.nig.ac.jp/nig/index_en.html">Japanese National Bio Resource Project</a></li>
<li><a href="http://www.riken.jp/SPD/index.html">Postgenome database service</a> Chemical Genetics Laboratory, Riken</li>
<li><a href="http://dna.brc.riken.jp/en/yoshidayeast_en.html">Orfeome clones</a> Riken Bioresource Centre</li>
<li><a href="http://www.addgene.org">Addgene</a> Collects and distributes plasmids</li>
</ul>
<h3 id="experimental-protocols">Experimental protocols</h3>
<ul>
<li>The <a href="http://www-bcf.usc.edu/~forsburg/plasmids.html">Forsburg lab</a> technical information e.g. <a href="http://www-bcf.usc.edu/~forsburg/Protocols.html">Protocols</a></li>
<li><a href="http://www.dieter-wolf-lab.org/protocols/">Wolf Lab Protocols</a></li>
</ul>
<h2 id="s.-cerevisiae-resources"><em>S. cerevisiae</em> Resources</h2>
<ul>
<li><a href="http://www.yeastgenome.org/">SGD</a> <em>Saccharomyces</em> Genome Database</li>
<li><a href="http://www.sanger.ac.uk/research/projects/genomeinformatics/sgrp.html"><em>Saccharomyces</em> genome resequencing</a> at Sanger</li>
</ul>

  </div>
  </div>
</div>
<div *ngIf="section == 'datasets'">
  <div class="docs-menu">
<h3 id="datasets"><a routerLink="/datasets">Datasets</a></h3>
<ul>
<li><a routerLink="/downloads/genome-datasets">Genome Datasets</a></li>
<li><a routerLink="/downloads/cds-coordinates">CDS Coordinates</a></li>
<li><a routerLink="/downloads/protein-datasets">Protein Datasets</a></li>
<li><a routerLink="/downloads/go-associations">GO Associations</a></li>
<li><a routerLink="/downloads/intron-data">Intron Data</a></li>
<li><a routerLink="/downloads/data-mappings">Data Mappings</a></li>
<li><a routerLink="/downloads/utr">UTRs</a></li>
</ul>

  </div>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h2 id="datasets">Datasets</h2>
<p>Links below go to web pages, except where &quot;ftp&quot; indicates that they go directly to ftp site directories. Note that the ftp subdirectories are organised slightly differently from the links on this page. If you have trouble finding anything, please ask the <a routerLink="/mailto:helpdesk@pombase.org">helpdesk</a>.</p>
<h5 id="ftp-site">FTP site</h5>
<p><a href="ftp://ftp.pombase.org/pombe/"><em>S. pombe</em> ftp site root directory</a></p>
<h5 id="genome-sequence-and-features"><a routerLink="/downloads/genome-datasets">Genome sequence and features</a></h5>
<p>Files with genome sequence, feature coordinates, or both (including enriched EMBL for Artemis), plus others (UTRs, CDS and exon coordinates, etc.)</p>
<h5 id="annotation-datasets">Annotation datasets</h5>
<table>
<thead>
<tr class="header">
<th>Annotation type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a routerLink="/downloads/protein-datasets">Protein datasets</a></td>
<td>Protein sequence FASTA database, peptide features, properties, etc.</td>
</tr>
<tr class="even">
<td><a routerLink="/downloads/go-annotations">GO annotations</a></td>
<td>Gene Ontology annotation files</td>
</tr>
<tr class="odd">
<td><a href="ftp://ftp.pombase.org/pombe/annotations/GO_complexes/">Macromolecular complexes</a></td>
<td>Subunits of protein and ribonucleoprotein complexes (GO cellular component terms and annotated genes; ftp)</td>
</tr>
<tr class="even">
<td><a routerLink="/downloads/phenotype-annotations">Phenotype annotations</a></td>
<td>FYPO phenotype annotation files - complete annotation set (PHAF) or viability summary</td>
</tr>
<tr class="odd">
<td><a href="ftp://ftp.pombase.org/pombe/exports/">HCPIN datasets</a></td>
<td>Physical interaction and GO substrate data that make up the High Confidence Physical Interaction Network datasets (ftp)</td>
</tr>
<tr class="even">
<td><a href="ftp://ftp.pombase.org/pombe/annotations/modifications/">Modifications</a></td>
<td>Protein modification data file (RNA modifications to be added in future; ftp)</td>
</tr>
<tr class="odd">
<td><a routerLink="/downloads/manually-curated-orthologs">Orthologs</a></td>
<td>Manually curated ortholog sets for human and <em>S. cerevisiae</em>, with documentation links</td>
</tr>
</tbody>
</table>
<h5 id="gene-names-and-ids"><a routerLink="/downloads/data-mappings">Gene names and IDs</a></h5>
<p>Mappings between PomBase systematic IDs, gene names, product descriptions, and UniProt accession numbers</p>
<h5 id="genomic-regions">Genomic regions</h5>
<table>
<thead>
<tr class="header">
<th>Region</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a routerLink="/downloads/intron-data">Introns</a></td>
<td>Intron sequence and data</td>
</tr>
<tr class="even">
<td><a routerLink="/status/telomeres">Telomeres</a></td>
<td>Telomeric sequence</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/centromeres">Centromeres</a></td>
<td>Centromeric sequence and maps</td>
</tr>
<tr class="even">
<td><a routerLink="/status/mating-type-region">Mating Type Region</a></td>
<td>Links to genome browser for 972 h- (chromosome 2 coordinates) and h+ (mating region contig)</td>
</tr>
<tr class="odd">
<td>Cosmids</td>
<td><a href="ftp://ftp.pombase.org/pombe/Archived_directories/Cosmid_sequences/">Sequences</a> and <a href="ftp://ftp.pombase.org/pombe/Archived_directories/Cosmid_assembly_data/">Chromosome tiling paths from genome sequence assembly</a></td>
</tr>
</tbody>
</table>

  </div>
  </div>
</div>
<div *ngIf="section == 'documentation'">
  <div class="docs-menu">
<h3 id="documentation"><a routerLink="/documentation">Documentation</a></h3>
<ul>
<li>Gene pages
<ul>
<li><a routerLink="/documentation/gene-page-basic-information">Basic information</a></li>
<li><a routerLink="/documentation/gene-page-gene-ontology">Gene Ontology</a></li>
<li><a routerLink="/documentation/gene-page-phenotypes">Phenotypes</a></li>
<li><a routerLink="/documentation/gene-page-target">Target of</a></li>
<li><a routerLink="/documentation/gene-page-protein-features">Protein features</a></li>
<li><a routerLink="/documentation/gene-page-modifications">Modifications</a></li>
<li><a routerLink="/documentation/gene-page-gene-expression">Gene expression</a></li>
<li><a routerLink="/documentation/genetic-and-physical-interactions">Genetic and physical interactions</a></li>
<li><a routerLink="/documentation/orthologs">Ortholog curation</a></li>
<li><a routerLink="/documentation/taxonomic-conservation">Taxonomic conservation</a></li>
<li><a routerLink="/documentation/gene-page-sequence">Sequence</a></li>
<li><a routerLink="/documentation/controlled_curation">Controlled curation</a></li>
<li><a routerLink="/documentation/gene-page-external-references">External references</a></li>
<li><a routerLink="/documentation/gene-page-literature">Literature</a></li>
</ul></li>
<li><a routerLink="/documentation/pombase-go-slim-documentation">GO Slim</a></li>
<li><a routerLink="/documentation/ontology-term-page">Ontology term pages</a></li>
<li><a routerLink="/documentation/publication-page">Publication pages</a></li>
<li><a routerLink="/documentation/genotype-page">Genotype pages</a></li>
<li><a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</a></li>
<li><a routerLink="/documentation/simple-search-documentation">Simple search</a></li>
<li><a routerLink="/documentation/advanced-search">Advanced search</a></li>
<li>Data submission file formats
<ul>
<li><a routerLink="/documentation/phenotype-data-bulk-upload-format">Phenotype data</a></li>
<li><a routerLink="/documentation/modification-data-bulk-upload-format">Modification data</a></li>
<li><a routerLink="/documentation/qualitative-gene-expression-data-bulk-upload-format">Qualitative gene expression data</a></li>
<li><a routerLink="/documentation/quantitative-gene-expression-data-bulk-upload-format">Quantitative gene expression data</a></li>
</ul></li>
</ul>

  </div>
  <div class="docs-content">
  <div *ngIf="pageName == 'annotation-extension-relation-display'">
<h1 id="annotation-extension-relation-display">Annotation Extension Relation Display</h1>
<p>Annotation extensions are used to provide additional specificity for annotations to GO, FYPO, and protein modification annotations. Each extension consists of a relation and another &quot;entity&quot;, which may be a gene in PomBase or another database, or another ontology term. (The <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">GO Consortium wiki page on annotation extensions</a> contains useful information on relations.) Because some relations have unwieldy names, the PomBase gene page display substitutes more readable text.</p>
<p>This table shows the underlying relation name and corresponding display text for the affected relations, and the reciprocal relation displayed in &quot;Target of&quot; annotations (derived from extension data):</p>
<table>
<thead>
<tr class="header">
<th>Relation name</th>
<th>Display name</th>
<th>Reciprocal</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>activated_by</td>
<td>activated by</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>coincident_with</td>
<td>at</td>
<td>N/A</td>
</tr>
<tr class="odd">
<td>during</td>
<td>during</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>exists_during</td>
<td>during</td>
<td>N/A</td>
</tr>
<tr class="odd">
<td>happens_during</td>
<td>during</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>has_regulation_target</td>
<td>regulates</td>
<td>regulated by</td>
</tr>
<tr class="odd">
<td>in_absence_of</td>
<td>in absence of</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>in_presence_of</td>
<td>in presence of</td>
<td>N/A</td>
</tr>
<tr class="odd">
<td>inhibited_by</td>
<td>inhibited by</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>not_exists_during</td>
<td>except during</td>
<td>N/A</td>
</tr>
<tr class="odd">
<td>not_happens_during</td>
<td>except during</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>occurs_at</td>
<td>at</td>
<td>N/A</td>
</tr>
<tr class="odd">
<td>occurs_in</td>
<td>in</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>required_for</td>
<td>required for</td>
<td>N/A</td>
</tr>
</tbody>
</table>
<p>For some relations, the best display string depends on term and its ancestry, as indicated in these tables:</p>
<p>The display for FYPO extensions indicating genes used in assays depends on whether the phenotype is normal or abnormal:</p>
<table>
<thead>
<tr class="header">
<th>Relation name</th>
<th>If descendant of</th>
<th>Display name</th>
<th>Reciprocal</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>assayed_using</td>
<td>FYPO:0001985 abnormal phenotype</td>
<td>affecting</td>
<td>affected by mutation in</td>
</tr>
<tr class="even">
<td>assayed_enzyme</td>
<td>FYPO:0001985 abnormal phenotype</td>
<td>affecting activity of</td>
<td>activity affected by mutation in</td>
</tr>
<tr class="odd">
<td>assayed_substrate</td>
<td>FYPO:0001985 abnormal phenotype</td>
<td>affecting substrate</td>
<td>affected by mutation in</td>
</tr>
<tr class="even">
<td>assayed_using</td>
<td>FYPO:0000257 normal phenotype</td>
<td>affecting</td>
<td>N/A (normal means no effect to report in &quot;Target of&quot; section)</td>
</tr>
<tr class="odd">
<td>assayed_enzyme</td>
<td>FYPO:0000257 normal phenotype</td>
<td>affecting activity of</td>
<td>N/A (normal means no effect to report in &quot;Target of&quot; section)</td>
</tr>
<tr class="even">
<td>assayed_substrate</td>
<td>FYPO:0000257 normal phenotype</td>
<td>affecting substrate</td>
<td>N/A (normal means no effect to report in &quot;Target of&quot; section)</td>
</tr>
</tbody>
</table>
<p>GO annotation extensions display for part_of:</p>
<table>
<thead>
<tr class="header">
<th>If descendant of</th>
<th>Display name</th>
<th>Reciprocal</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>GO:0003674 molecular_function</td>
<td>involved in</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>GO:0008150 biological process</td>
<td>part of</td>
<td>N/A</td>
</tr>
</tbody>
</table>
<p>The display for the &quot;input&quot; relations (has_input or has_direct_input) follows this pattern:</p>
<table>
<thead>
<tr class="header">
<th>If descendant of</th>
<th>Display name</th>
<th>Reciprocal</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>GO:0003824 catalytic activity</td>
<td>has substrate</td>
<td>substrate of</td>
</tr>
<tr class="even">
<td>GO:0006810 transport</td>
<td>transports</td>
<td>transported by</td>
</tr>
<tr class="odd">
<td>GO:0005215 transporter activity</td>
<td>transports</td>
<td>transported by</td>
</tr>
<tr class="even">
<td>GO:0030234 enzyme regulator activity</td>
<td>regulates</td>
<td>regulated by</td>
</tr>
<tr class="odd">
<td>GO:0008047 enzyme activator activity</td>
<td>positively regulates</td>
<td>positively regulated by</td>
</tr>
<tr class="even">
<td>GO:0004857 enzyme inhibitor activity</td>
<td>negatively regulates</td>
<td>negatively regulated by</td>
</tr>
<tr class="odd">
<td>GO:0051179 localization</td>
<td>localizes</td>
<td>localized by</td>
</tr>
<tr class="even">
<td>GO:0005488 binding</td>
<td>binds</td>
<td>binds</td>
</tr>
<tr class="odd">
<td>(anything not listed above)</td>
<td>has input</td>
<td>input for</td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'controlled_curation'">
<h1 id="pombase-controlled-curation">PomBase Controlled Curation</h1>
<p>Gene and publication pages include manually curated annotations of several types that cannot be represented using available ontologies.</p>
<p>For each annotation, the summary view shows a text description, which corresponds to an entry in the internal <a routerLink="/PBO">http://www.pombase.org/faqs/what-pbo-option-advanced-search</a> term set. The detailed view adds the PBO ID, evidence (if available), reference, and count. The count links to the <a href="/documentation/ontology-term-page">ontology term page</a> for the description.</p>
<p>Links in the left-hand navigation menu include:</p>
<ul>
<li><p><strong>Complementation</strong> - Indicates that an <em>S. pombe</em> gene complements, or is complemented by, a gene from another species</p></li>
<li><p><strong>Disease association</strong> - Indicates that the gene's human ortholog has been implicated in disease</p></li>
<li><p><strong>Miscellaneous</strong></p></li>
<li><p><strong>Warnings</strong> - Alerts to changes or anomalies that can affect interpretation of experimental results, such as changes to the sequence or feature annotation</p></li>
</ul>
<p>In the Miscellaneous category, annotations fall under one of these sub-headings:</p>
<ul>
<li><p><strong>Catalytic activity attributes:</strong> Features of a gene product's catalytic activity, such as K<sub>M</sub>, k<sub>cat</sub>, substrate specificity</p></li>
<li><p><strong>Experimental tools:</strong> Indicates that the gene or its product is used as an experimental tool (e.g. reporter gene construct, selectable marker, antibody)</p></li>
<li><p><strong>Genome organization:</strong> Attributes of the genomic region around the gene, e.g. duplications, repeats, alternative transcripts</p></li>
<li><p><strong>Miscellaneous functional group:</strong> Function-related descriptions that are not covered by GO molecular function or biological process annotations</p></li>
<li><p><strong>Comment:</strong> Experimental observations and other notes that do not fit into ontologies or any of the above controlled curation categories; truly miscellaneous but interesting information</p></li>
</ul>

  </div>
  <div *ngIf="pageName == 'gene-page-basic-information'">
<h1 id="gene-page-basic-information">Gene Page: Basic Information</h1>
<p>At the top of each gene page is a set of basic information about the gene:</p>
<div class="figure">
<img src="assets/basic_info_gene_page.png" title="Gene page basic information" alt="top of gene page" width="800" />
<p class="caption">top of gene page</p>
</div>
<ol style="list-style-type: decimal">
<li>See the <a routerLink="/submit-data/gene-name-registry">Gene Name Registry</a> for more information about <em>S. pombe</em> gene names. Synonyms include all names other than the standard name that have been published for a gene, and thus are not guaranteed to be unique.</li>
<li>Free text description of the gene product</li>
<li>Chromosome coordinates for the gene. For protein-coding genes, the first set of numbers indicate the gene start, end, and length including the default UTRs chosen by PomBase curators (see <a href="http://www.pombase.org/faqs/how-do-you-determine-genes-full-length-transcript-utr-coordinates-transcription-start-and-end-s">this FAQ</a> for more information). The &quot;CDS&quot; numbers give the start, end, and length excluding UTRs but including introns (other than introns located entirely within UTRs).</li>
<li>What the three-letter acronym stands for; where available, name descriptions are provided for synonyms as well as primary names.</li>
<li>Category describing how well-studied a gene is; see the <a href="/status/gene-characterisation">Gene Characterisation</a> page for details</li>
<li>Indicates whether a gene encodes protein, specifies a non-coding RNA, or is a pseudogene</li>
<li>The size of the mature gene product. For protein-coding genes, the length (number of amino acid residues) and molecular weight are shown.</li>
<li>Static graphic from the PomBase Genome Browser showing the region around the gene</li>
<li>Link to the PomBase Genome Browser, centred on the location of the gene</li>
<li>Links to gene pages for nearby genes</li>
</ol>

  </div>
  <div *ngIf="pageName == 'gene-page-external-references'">
<h1 id="gene-page-external-references">Gene Page: External References</h1>
<p>The External References section of a gene page provides links to information on a gene or its product in each of several external databases. For each database, a category, the name and a brief description are shown, and the identifier used by the database is displayed and linked.</p>
<p>The resources linked in this section cover a wide range of databases and data tyeps. Because all are external, PomBase staff cannot provide assistance with any of the resources other than problems with the actual links.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-gene-expression'">
<p>Gene Page: Gene Expression</p>
<p>The Gene Expression section of a gene page displays curated qualitative and quantitative information about the level and timing of the gene's expression. In each subsection, a simple summary appears by default (see explanations below the detail view illustration):</p>
<p><br />
<img src="assets/gene_page_expression_summary.png" title="Gene Page: Gene Expression Summary" alt="gene page expression section, summary view" width="800" /></p>
<p>Click &quot;Show details&quot; to reveal additional information:</p>
<p><br />
<img src="assets/gene_page_expression_details.png" title="Gene Page: Gene Expression Details" alt="gene page expression section, detail view" width="800" /></p>
<ol style="list-style-type: decimal">
<li>Qualitative descriptions are drawn from a small internal controlled vocabulary, with supporting extensions. Each description links to a list of all genes annotated to the same term.
<ul>
<li>a) What was measured, i.e. RNA or protein.</li>
<li>b) A qualitative description of the expression level (one of: present, absent, increased, decreased, unchanged, constant, fluctuates).</li>
<li>c) An optional extension indicating that expression is observed during a particular process or phase (using GO terms) or in the presence of a particular chemical (using ChEBI terms).</li>
<li>d) A brief descriptor for the type of evidence that supports the annotation. The evidence categories come from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO).</li>
<li>e) The paper from which the annotation comes.</li>
</ul></li>
<li>Quantitative data include the number of RNA or protein molecules measured per cell, and may optionally include a copy number range. Quantitative expression annotations also indicate whether the experiment measures expression in a population or a single cell, and usually include a brief description of experimental conditions, drawn from a small ontology maintained by PomBase curators.
<ul>
<li>f) Molecules per cell (RNA or protein, as indicated in the sub-header). For population-scale experiment types this is an average, whereas for single-cell scale experiments it is an absolute number.</li>
<li>g) The life cycle stage or cell cycle phase in which expression was measured.</li>
<li>h) Information about experimental conditions, such as temperature, type of medium used, etc. Descriptions come from a small ontology maintained by PomBase curators.</li>
<li>i) Whether the experiment was done at the level of a single cell or a population of cells.</li>
<li>j) A brief descriptor for the type of evidence that supports the annotation. The evidence categories come from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO). If a more specific description is not available, the general term &quot;experimental evidence&quot; is shown.</li>
<li>k) The paper from which the annotation comes.</li>
</ul></li>
</ol>
<p><strong>Important:</strong> Note that the condition descriptions are rather broad, and therefore do not necessarily capture all details of every aspect that may affect gene expression. We are also unable to capture strain details at present. We therefore recommend consulting the papers cited for the data before comparing or combining data from different publications.</p>
<p>Several of the <a href="/documentation/gene-page-external-references">External References</a> also link to gene expression data.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-gene-ontology'">
<h1 id="gene-page-gene-ontology">Gene Page: Gene Ontology</h1>
<p>PomBase uses the <a href="http://www.geneontology.org/">Gene Ontology</a> (GO) to describe the biological context of genes.</p>
<p>GO consists of three distinct <a href="http://www.geneontology.org/page/ontology-documentation">ontologies</a> (or sets of vocabularies) that describe a gene's:</p>
<ol style="list-style-type: decimal">
<li>Molecular Function (MF) — single-step activities such as <em>kinase activity</em> or <em>transporter activity</em></li>
<li>Biological Process (BP) — a series of functions with a common goal such as <em>MAPK cascade</em> or <em>cytokinesis</em></li>
<li>Cellular component (CC) — the location where the gene product can be found, and what complexes it is part of, for instance <em>nucleus</em> or <em>fatty acid synthase complex</em></li>
</ol>
<p>A gene product may be <a href="http://www.geneontology.org/page/annotation">annotated</a> to several GO terms from each of the three ontologies; <a routerLink="/gene/SPCC1682.02c">mcm3</a>, for instance, is annotated to ATP-dependent DNA helicase activity, ATP binding, and DNA replication origin binding (MFs), it acts in mitotic DNA replication initiation and negatively regulates the MCM helicase activity (BPs), and is found locations including the replication fork and the pre-replicative complex (CC).</p>
<p>Each table includes ontology term details and supporting data. The GO annotation display on PomBase gene pages includes ontology term details and supporting data. The summary view shows just the essentials: The list of terms is filtered, using the ontology structure, so that it shows only the most specific terms used to annotate a genotype, and each unique combination of gene, GO term, and extension(s) is shown once:</p>
<div class="figure">
<img src="assets/go_gene_page_summary.png" alt="summary gene page GO annotations" width="600" />
<p class="caption">summary gene page GO annotations</p>
</div>
<ol style="list-style-type: decimal">
<li>The GO term name, which links to a page with additional information, including the term definition, any synonyms, <a routerLink="/#go_str">relationships to other GO terms</a>, and annotations to the term or its descendants. (See the PomBase <a href="/documentation/ontology-term-page">ontology term page documentation</a> and the GO documentation on <a href="http://geneontology.org/page/ontology-structure">Ontology Structure</a> and <a href="http://geneontology.org/page/ontology-relations">Ontology Relations</a> for more information.)</li>
<li><a routerLink="/browse-curation/fission-yeast-go-slim-terms">GO Slim</a> terms applicable to the gene.</li>
<li>GO annotations may have <a routerLink="/#ann_ext">extensions</a> to capture any of several types of additional detail. <em>S. pombe</em> genes link to PomBase gene pages, and ontology term names link to ontology term pages.</li>
</ol>
<p>The detailed view shows annotations to all GO terms, and includes more details for each annotation. It shows separate entries for repeat determinations of a given gene/term/extension combination (if supported by more than one line of evidence and/or reported in more than one paper), and annotations to terms hidden in the summary view:</p>
<div class="figure">
<img src="assets/go_gene_page_full.png" alt="full gene page GO annotations" width="800" />
<p class="caption">full gene page GO annotations</p>
</div>
<ol style="list-style-type: decimal">
<li>The unique <a routerLink="/#id">ID and name</a> for a GO term, linked to an ontology term page as described above.</li>
<li><p>An abbreviation (code) for the type of <a routerLink="/#evid">evidence</a> that supports the annotation. The evidence categories come from the set of <a href="http://geneontology.org/page/guide-go-evidence-codes">evidence codes</a> defined by the GO Consortium.</p></li>
<li>An additional ontology term or identifier that provides supporting details for annotations using certain evidence codes (see below and <a href="http://geneontology.org/page/go-annotation-file-format-20">GO documentation</a>).</li>
<li>An optional qualifier that modifies the connection between the gene product and the GO term. Entries come from <a href="http://geneontology.org/page/go-annotation-conventions#qual">GO's set of allowed qualifiers</a> or internal PomBase usage.</li>
<li>The paper from which the annotation comes.</li>
<li>The number of genes annotated to the term, linked to an ontology term page as described above.</li>
<li><a routerLink="/browse-curation/fission-yeast-go-slim-terms">GO Slim</a> terms applicable to the gene.</li>
<li><p>GO annotations may have <a routerLink="/#ann_ext">extensions</a> to capture any of several types of additional detail. <em>S. pombe</em> genes link to PomBase gene pages.</p></li>
</ol>
<h4 id="id">Term ID and name</h4>
<p>Annotations are made to the definition of a term, not the term name itself, so we recommend that users always read the term definition. The definition of a term can be found on the ontology term page linked to the term name and ID.</p>
<h4 id="go_str">GO Structure</h4>
<p>GO is structured in a hierarchal order with less specific terms being parents of more specific child terms. A child term may have multiple parents and multiple children; the BP <em>mitotic sister chromatid segregation</em>, for instance, has child terms representing each part of the segregation process (sister chromatid cohesion, separation, etc.) as well as parent terms connecting it to both the mitotic cell cycle and chromosome segregation. Crucially, whenever an annotation is made to a term, the gene product is automatically annotated to all the parent terms. The ancestry of a term can be viewed in browsers such as AmiGO or QuickGO, accessible via links on the ontology term page. For more information, see the <a href="http://geneontology.org/page/ontology-structure">GO Ontology Structure</a> documentation.</p>
<p>Multiple relationships exist to describe the links within the ontologies. A child term can have an <em>is_a</em> relation to the parent term, where the child is a more specific type of the parent, or a <em>part_of</em> relationship where the child makes up a part of the parent. For instance, the mitochondrion <em>is_a</em> intracellular organelle and is part_of a cell. Additionally, GO also include regulatory relationships. For more information on relationships in GO, see the <a href="http://geneontology.org/page/ontology-relations">GO Ontology Relations</a> documentation.</p>
<h4 id="evid">Evidence codes</h4>
<p>In PomBase, every annotation is supported by a reference that states where the annotation comes from, and an evidence code that describes the type of data that supports the annotation. An annotation may be inferred from experimental ‘wet lab’ data, backed by a literature reference and citing experimental evidence such as <a href="http://geneontology.org/page/ida-inferred-direct-assay">IDA</a> (Inferred from Direct Assay) or <a href="http://geneontology.org/page/imp-inferred-mutant-phenotype">IMP</a> (Inferred from Mutant Phenotype). Further information on evidence codes is available in the <a href="http://geneontology.org/page/guide-go-evidence-codes">GO Evidence Codes</a> documentation.</p>
<p>Another source of annotations come from computational methods. Please note that all computational annotations are based on predictions. In cases where a sequence model has been used to annotate genes, but the genes annotated based on the model have not been manually checked, the <a href="http://geneontology.org/page/automatically-assigned-evidence-codes">IEA</a> (Inferred from Electronic Annotation) evidence code is assigned. If the annotations have been manually checked other evidence codes may be used, for instance <a href="http://geneontology.org/page/iso-inferred-sequence-orthology/">ISO</a> (Inferred from Sequence Orthology) or <a href="http://geneontology.org/page/ism-inferred-sequence-model/">ISM</a> (Inferred from Sequence Model). PomBase uses ISO to cross-reference to the roles of known <em>S. cerevisiae</em> genes, and uses ISM when domains present in a gene product can give clues to its biological role.</p>
<p>For some types of evidence, such as sequence comparisons or interaction data, it is important to note what gene or gene product was used in the comparison or detected in the interaction. In these cases the <a href="http://geneontology.org/page/go-annotation-file-gaf-format-20/">With/From column</a> provides more information regarding the source of the information.</p>
<h4 id="ann_ext">Annotation extensions</h4>
<p>Where available, annotation extensions are displayed underneath the GO term name. The extensions provide additional specificity to the annotation by linking the term to another ontology term or a gene product via a relationship. Examples include specifying substrates of molecular functions or specifying the cellular localization during a process (for instance, <a routerLink="/gene/SPBC106.10">pka1</a> has protein serine/threonine kinase activity and has the substrates <a routerLink="/gene/SPBC119.04">mei3</a> and <a routerLink="/gene/SPAC6F12.02">rst2</a>. It is a cellular component of the nucleus during nitrogen starvation, but found in the vacuole during glucose starvation).</p>
<p>The GO Consortium provides further information on annotation extensions in its <a href="http://geneontology.org/page/annotation-extension">annotation documentation</a>, including the <a href="http://geneontology.org/page/go-annotation-file-gaf-format-20/">file format guide</a>, on a <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">wiki page</a>, and in a <a href="http://www.biomedcentral.com/1471-2105/15/155/abstract">publication</a>. PomBase converts many extension names to more human-friendly text, as described <a routerLink="/documentation/annotation-extension-relation-display">here</a>.</p>
<p>If an extension mentions another <em>S. pombe</em> gene, the extension data will also be displayed as an annotation in the &quot;<a href="/documentation/gene-page-target">Target of</a>&quot; section of the page for that gene.</p>
<h4 id="finding-other-genes-annotated-to-a-specific-go-term">Finding other genes annotated to a specific GO term</h4>
<p>From a gene page, all <em>S. pombe</em> genes annotated to a term (or its children) can be found by clicking on a term name or ID to reach the ontology term page.</p>
<p>Additionally, the <a routerLink="/query">advanced search</a> can be used to search for all genes annotated to a particular GO term (see the <a href="/documentation/advanced-search-documentation">advanced search documentation</a> for more information). To find annotations to specific GO terms in organisms other than <em>S. pombe</em> we recommend using <a href="http://amigo.geneontology.org">AmiGO</a> or <a href="http://www.ebi.ac.uk/QuickGO">QuickGO</a>.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-literature'">
<h1 id="gene-page-literature">Gene Page: Literature</h1>
<p>The Literature section of a gene page lists the papers that have curated data for the gene or its product.</p>
<p>At the top of the section, there is a link to search PubMed for any papers on the gene. This may retrieve papers not on the gene page Literature list, if there are papers not yet curated (or spurious matches to the search criteria).</p>
<p>For each paper, the full citation and the number of genes curated from the paper are shown, and a &quot;Details&quot; link goes to the <a href="/documentation/publication-page.md">publication page</a>.</p>
<p>By default, the list is sorted alphabetically by first author. Use the &quot;Sort by&quot; links to change the order.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-modifications'">
<h1 id="gene-page-modifications">Gene Page: Modifications</h1>
<p>The Modifications section lists protein modifications that have been manually curated, using terms from the <a href="http://www.psidev.info/MOD">PSI-MOD</a> ontology, for protein-coding genes. PomBase will add RNA modifications to this section in the future, when relevant data are curated.</p>
<h4 id="ontology-annotations-for-protein-features">Ontology Annotations for Protein Features</h4>
<p>The summary view shows only the names of the most specific terms used to annotate the gene:</p>
<div class="figure">
<img src="assets/gene_page_modification_summary.png" title="Protein modifications" alt="summary gene page modification ontology annotation" width="250" />
<p class="caption">summary gene page modification ontology annotation</p>
</div>
<p>The detailed view shows more information for each annotation, and may display additional terms:</p>
<div class="figure">
<img src="assets/gene_page_modification_full.png" title="Protein modifications" alt="full gene page modification ontology annotation" width="800" />
<p class="caption">full gene page modification ontology annotation</p>
</div>
<ol style="list-style-type: decimal">
<li>Name and ID of the ontology term</li>
<li>Modification annotations may have <a routerLink="/#ann_ext">extensions</a> to capture any of several types of additional detail. <em>S. pombe</em> genes link to PomBase gene pages, and GO term names link to PomBase ontology term summary pages.</li>
<li>The summary view is filtered, using the ontology structure, so that it shows only the most specific terms used to annotate a gene. Modification annotations curated for a gene using less specific terms (&quot;superclasses&quot; or &quot;ancestors&quot; in the ontology) appear only in the detailed view.</li>
<li>An abbreviation (code) or brief description for the type of evidence that supports the annotation. Modification annotations use some of the <a href="http://www.geneontology.org/page/guide-go-evidence-codes">codes</a> defined by the GO Consortium, plus a small subset of the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO).</li>
<li>The residue(s) modified (where available)</li>
<li>The published source of the annotation (where available)</li>
<li>The total number of genes annotated to this term or any of its descendants, linked to a list of those genes. The gene list page includes basic information about the ontology term, and a link to send the gene list to the advanced search.</li>
</ol>
<h4 id="ann_ext">Annotation extensions</h4>
<p>Where available, annotation extensions are displayed underneath the ontology term name. The extensions provide additional specificity to the annotation, often by linking the term to another ontology term or a gene product via a relationship. Examples include specifying the gene product that adds or removes a modification, specifying modified residues, or specifying that the modification is observed during a phase or process. For example, Lys4 (shown above) is phosphorylated on a serine residue during M phase of the mitotic cell cycle. Likewise, <a routerLink="/gene/SPBC11B10.09">Cdc2</a> is phosphorylated during G2, and phosphorylated on Tyrosine 15 by Wee1; it is phosphorylated during G2 but not M phase of the mitotic cell cycle; other extensions are also available for Cdc2.</p>
<p>The annotation extension field can also be used to indicate modification site occupancy, for experiments that measure the proportion of copies of the protein (or RNA) that have the modification.</p>
<p>Note: unlike GO annotations, for modification annotations extensions are not considered for display filtering. Annotations to less specific terms are hidden in the summary view whether they have extensions or not.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-phenotypes'">
<h1 id="gene-page-phenotypes">Gene Page: Phenotypes</h1>
<p>PomBase defines a phenotype as an observable characteristic, or set of characteristics, of an organism that results from the interaction of its genotype with a given environment. In PomBase, phenotypes are annotated using terms from the Fission Yeast Phenotype Ontology (FYPO). FYPO uses several existing ontologies from the <a href="http://obofoundry.org/">Open Biological and Biomedical Ontologies</a> (OBO) collection to construct formal definitions. Basic documentation for FYPO is available at the <a href="http://obofoundry.org/ontology/fypo.html">OBO Foundry</a>, and further information is available on the <a href="http://curation.pombase.org/pombase-trac/wiki/FissionYeastPhenotypeOntology">PomBase wiki</a>.</p>
<p>In the phenotype annotation display on PomBase gene pages, the first item shown is a brief summary indicating whether cells with a null (deletion) allele of the gene are viable or inviable, or either depending on experimental conditions. Next, single-allele phenotypes are shown in two tables. The first table lists phenotypes observed at the population level, such as viability in culture, and the second shows cell-level phenotypes. Note that the viable/inviable population terms describe whether a gene is essential or not, and see the wiki <a href="http://curation.pombase.org/pombase-trac/wiki/FYPOContentStructure">FYPO Content and Structure</a> documentation for more information on cell and population phenotypes. Finally, two more tables list population-level and cell-level multi-allele phenotypes, i.e. phenotypes associated with double mutants, triple mutants, etc, and the relevant genotype details.</p>
<p>Each table includes ontology term details and supporting data. The summary view shows just the essentials: The list of terms is filtered, using the ontology structure, so that it shows only the most specific terms used to annotate a genotype, and each unique combination of gene, FYPO term, allele(s) and extension(s) is shown once. For single-allele phenotypes, the display includes:</p>
<div class="figure">
<img src="assets/single_fypo_gene_page_summary.png" title="Single-allele FYPO summary on gene page" alt="gene page phenotype annotations" width="800" />
<p class="caption">gene page phenotype annotations</p>
</div>
<ol style="list-style-type: decimal">
<li>The FYPO term name, which links to a page with additional information, including the term definition, any synonyms, relationships to other FYPO terms, and annotations to the term or its descendants. (See the <a href="/documentation/ontology-term-page">ontology term page documentation</a> for more.)</li>
<li>Allele details, including a name (if one is used in the literature) and description (where known). Mouse over the allele name to show the allele type, which indicates whether partial deletions or altered residues refer to amino acids or nucleotides, and expression level. If you can provide a description for any allele shown as &quot;unknown&quot;, please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a>.</li>
<li><p>Phenotype annotations may have extensions to capture expressivity (the extent to which a phenotype is expressed; for PomBase annotations expressivity extensions are often used to capture information about phenotype severity) or penetrance (proportion of a population that shows the phenotype), or to document which gene or protein used in an assay for level, localisation, etc. <em>S. pombe</em> genes link to PomBase gene pages. Expressivity and penetrance use the relations <em>has_expressivity</em> and <em>has_penetrance</em> respectively, and can have values such as &quot;high&quot;, &quot;medium&quot;, or &quot;low&quot;. Penetrance can also use numerical values. A gene or gene product used in an assay is stored using the appropriate PomBase systematic ID and the relation <em>assayed_using</em>; the relation is converted to <em>affecting</em> in the gene page display. If a mutation affects an activity that modifies another gene product, extensions may capture the affected enzyme, the affected substrate, or both. The relation <em>assayed_enzyme</em> is displayed as <em>affecting activity of</em> on gene pages, and <em>assayed_substrate</em> is shown as <em>affecting substrate</em>.</p></li>
<li><p>The phenotype display can be filtered to show subsets of the total set of annotations. The term filter, available in the summary or detailed view, lists several broad phenotypic categories derived from high-level FYPO terms:</p></li>
</ol>
<div class="figure">
<img src="assets/fypo_term_filter_pulldown.png" title="FYPO annotation term filter" alt="FYPO term filter" width="160" />
<p class="caption">FYPO term filter</p>
</div>
<p>Choose one to restrict the annotation display to terms in the selected branch of FYPO. When term filtering is active, a message appears to indicate that not all annotations are shown:</p>
<div class="figure">
<img src="assets/fypo_showing_n_annotations.png" title="filtered annotation display message" alt="filtering message" width="600" />
<p class="caption">filtering message</p>
</div>
<p>Change the selection back to &quot;No filter&quot; to see annotations to all terms.</p>
<p>The detailed single-allele phenotype view shows annotations to all FYPO terms, and includes more details for each annotation. It shows separate entries for repeat determinations of a given gene/term/allele/extension combination (if supported by more than one line of evidence and/or reported in more than one paper), and annotations to terms hidden in the summary view:</p>
<div class="figure">
<img src="assets/single_fypo_gene_page_full.png" title="Single-allele FYPO detailed view on gene page" alt="gene page phenotype annotations" width="800" />
<p class="caption">gene page phenotype annotations</p>
</div>
<ol style="list-style-type: decimal">
<li>The unique ID and name for a term in the phenotype ontology. The ID links to a page with additional information, including the term definition, any synonyms, relationships to other FYPO terms, and annotations to the term or its descendants. (See the <a href="/documentation/ontology-term-page">ontology term page documentation</a> for more.)</li>
<li>Allele details, including a name (if one is used in the literature) and description (where known). The column is headed &quot;Genotypes&quot; for consistency between the single- and multi-allele displays. Each genotype name links to a page with full details (type, description, and expression) for its allele(s), links to gene pages, and a list of all phenotype annotations for the genotype. If you can provide a description for any allele shown as &quot;unknown&quot;, please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a>.</li>
<li>Mouse over the allele name to show the allele type, which indicates whether partial deletions or altered residues refer to amino acids or nucleotides, and expression level.</li>
<li>The phenotype display can be filtered to show subsets of the total set of annotations. The term filter, available in the summary or detailed view, lists several broad phenotypic categories derived from high-level FYPO terms. Choose one to restrict the annotation display to terms in the selected branch of FYPO. Change the selection back to &quot;No filter&quot; to see annotations to all terms. In the detailed view, evidence can also be filtered by choosing an evidence description from a pulldown. Change the selection back to &quot;No filter&quot; to see annotations using any evidence type. Term and evidence filters can be combined (note that some term/evidence combinations have no matching annotations). When either term or evidence filtering is active, a message appears to indicate that not all annotations are shown.</li>
<li>A brief descriptor for the type of evidence that supports the annotation. The evidence categories come from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO).</li>
<li>Information about experimental conditions, such as temperature, type of medium used, etc. Descriptions come from a small ontology maintained by PomBase curators.</li>
<li>The paper from which the annotation comes.</li>
<li>Phenotype annotations may have extensions to capture expressivity (the extent to which a phenotype is expressed; for PomBase annotations expressivity extensions are often used to capture information about phenotype severity) or penetrance (proportion of a population that shows the phenotype), or to document which gene or protein used in an assay for level, localisation, etc. <em>S. pombe</em> genes link to PomBase gene pages. Expressivity and penetrance use the relations <em>has_expressivity</em> and <em>has_penetrance</em> respectively, and can have values such as &quot;high&quot;, &quot;medium&quot;, or &quot;low&quot; (numerical values may be added in the future if and when the need arises). A gene or gene product used in an assay is stored using the appropriate PomBase systematic ID and the relation <em>assayed_using</em>; the relation is converted to <em>affecting</em> in the gene page display. If a mutation affects an activity that modifies another gene product, extensions may capture the affected enzyme, the affected substrate, or both. The relation <em>assayed_enzyme</em> is displayed as <em>affecting activity of</em> on gene pages, and <em>assayed_substrate</em> is shown as <em>affecting substrate</em>.</li>
</ol>
<p>Similar displays are used for multi-allele phenotypes. Unless otherwise noted below, all items are as described above for single-allele phenotype displays.</p>
<p>The multi-allele summary shows FYPO terms and genotype descriptions:</p>
<div class="figure">
<img src="assets/multi_fypo_gene_page_summary.png" alt="multi-allele FYPO summary view on gene page" width="800" />
<p class="caption">multi-allele FYPO summary view on gene page</p>
</div>
<ol style="list-style-type: decimal">
<li>The FYPO term name, which links to the page for the term.</li>
<li>Genotype details, including a name (if one is used in the literature) and descriptions for the relevant alleles (where known). Mouse over any genotype name to show more allele details, or click to go to the genotype detail page. Multiple genotypes annotated to the same term are separated by commas.</li>
<li>Display filter for FYPO terms.</li>
</ol>
<p>The detailed view includes:</p>
<div class="figure">
<img src="assets/multi_fypo_gene_page_full.png" alt="multi-allele FYPO full view on gene page" width="800" />
<p class="caption">multi-allele FYPO full view on gene page</p>
</div>
<ol style="list-style-type: decimal">
<li>The unique ID and name for a term in the phenotype ontology, with links to the FYPO term page.</li>
<li>Genotype details, including a name (if one is used in the literature) and descriptions for the relevant alleles (where known). Mouse over any genotype name to show more allele details, or click to go to the genotype detail page. Multiple genotypes annotated to the same term are listed one per line.</li>
<li>Display filter for FYPO terms and evidence (see above).</li>
<li>Evidence description (see above).</li>
<li>Experimental conditions (see above).</li>
<li>The paper from which the annotation comes.</li>
</ol>
<p>Annotation extensions are also included in the multi-allele Summary and Full views (although none are included for the annotations in the above illustrations), using the same display as for single-allele phenotypes.</p>
<p>Also see the <a href="/documentation/advanced-search-documentation">Advanced Search documentation</a> for information on finding genes annotated to phenotype terms.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-protein-features'">
<h1 id="gene-page-protein-features">Gene Page: Protein Features</h1>
<p>Gene pages for protein-coding genes have a section describing protein features. Also see the <a routerLink="/documentation/gene-page-modifications">Modifications</a> documentation.</p>
<div class="figure">
<img src="assets/gene_page_protein_features.png" title="Protein features" alt="gene page protein features" width="800" />
<p class="caption">gene page protein features</p>
</div>
<ol style="list-style-type: decimal">
<li>The graphical view is a static graphic from the PomBase Genome Browser, and shows the positions of domains and other features. Below the graphic is a link to the Genome Browser's protein view.</li>
<li>The table of protein families and domains is described in more detail below.</li>
<li>This section provides manual curation that ensures that large protein families (e.g. the WD family) include all known members. These manual annotations are provided because many families in protein family databases are rebuilt during the release cycle to include new sequences, and as a result gene products are sometimes lost or gained from protein families, and false negatives are common. This section is also used to capture published subfamily members which may be collected into a single family in protein family databases.</li>
<li>Any motifs or features annotated using Sequence Ontology (SO) terms.</li>
<li>Table of the protein's physical properties</li>
</ol>
<h4 id="protein-families-and-domains">Protein Families and Domains</h4>
<div class="figure">
<img src="assets/gene_page_protein_domains.png" title="Protein families and domains" alt="gene page protein domains" width="800" />
<p class="caption">gene page protein domains</p>
</div>
<ol style="list-style-type: lower-alpha">
<li>Feature ID of this family or domain in the originating database. Where feasible, the ID links to the source database. Databases include Pfam, SMART, Prosite, Gene3D, SUPERFAMILY, TMHMM and Panther.</li>
<li>Source database name</li>
<li>Name of the matching feature in the contributing database</li>
<li>ID of the <a href="http://www.ebi.ac.uk/interpro/">InterPro</a> entry that includes the feature. InterPro classifies proteins into families using predictive models provided by several different databases that make up the InterPro consortium. InterPro is useful for assessing the species distribution of a particular family or domains.</li>
<li>The description of the domain in InterPro, from the contributing database</li>
<li>Start and end coordinates, showing the location of the feature within the protein</li>
<li>Link to a list of other fission yeast proteins which are members of the family or have the domain</li>
<li>Link to the InterPro entry for the feature<br />
</li>
<li>Link to the Pfam entry for the protein, which shows domain organization</li>
<li>Transmembrane domain coordinates, where applicable</li>
</ol>
<p>Note that some of the listed features (e.g. transmembrane domains from TMHMM) are predictions. Consult the contributing databases for further information, or <a routerLink="/mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you notice any problems with annotated or predicted features.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-sequence'">
<h1 id="gene-page-sequence">Gene Page: Sequence</h1>
<p>The Sequence section of a gene page provides a widget to download DNA sequences for any gene, and amino acid sequences for protein-coding genes, as well as links to send sequences to BLAST. For protein-coding genes, the DNA sequence of the coding exon(s) is displayed by default.</p>
<div class="figure">
<img src="assets/gene_page_sequence.png" alt="DNA sequence section" width="800" />
<p class="caption">DNA sequence section</p>
</div>
<ol style="list-style-type: decimal">
<li><p>Toggle to show the amino acid sequence (not available for non-coding RNA genes).</p></li>
<li><p>Controls to add UTRs, introns, or flanking sequences to the displayed sequence. For upstream or downstream sequence, use the up/down control to increment by one base, or type or paste a number in the box.</p></li>
<li><p>The sequence header indicates what is included.</p></li>
<li><p>Link to save the displayed sequence to a file.</p></li>
<li><p>Links to send the displayed sequence to BLAST at <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">NCBI</a> or <a href="http://fungi.ensembl.org/Multi/Tools/Blast?db=core">Ensembl</a>. DNA sequences link to BLASTN by default.</p></li>
</ol>
<p>When the &quot;Show translation&quot; option is selected, irrelevant controls are hidden and BLASTP links are shown:</p>
<div class="figure">
<img src="assets/gene_page_sequence_translated.png" alt="sequence section showing translation" width="800" />
<p class="caption">sequence section showing translation</p>
</div>

  </div>
  <div *ngIf="pageName == 'gene-page-target'">
<h1 id="gene-page-target-of">Gene Page: Target of</h1>
<p>The &quot;Target of&quot; section describes genes that affect the gene of interest. Genes listed under &quot;Target of&quot; may include upstream regulators or enzymes that modify the product of the gene of interest. For example, the &quot;Target of&quot; annotations for <a routerLink="/gene/SPBC11B10.09">cdc2</a> indicate that it is a substrate of, and regulated by, the kinase Wee1 and the phosphatase Cdc25 (among others).</p>
<p>Target of data are derived by taking the reciprocal of <a href="http://www.pombase.org/faqs/what-annotation-extension">ontology annotation extensions</a> (using <a href="documentation/gene-page-gene-ontology#ann_ext">GO annotation extensions</a> and <a routerLink="/documentation/gene-page-phenotypes">phenotype</a> annotation extensions). The table includes:</p>
<ul>
<li>Ontology: GO or FYPO, i.e. the ontology in which the annotation was made</li>
<li>Relationship: indicates how the genes are connected. The relationship is the reciprocal of the relation used in the originating annotation extension, as noted in the <a href="/documentation/annotation-extension-relation-display">Annotation extension relation display documentation</a>.</li>
<li>Gene: the gene whose product affects the gene of interest.</li>
<li>Product: the targeting gene's product description.</li>
<li>Reference (hidden in &quot;summary&quot; view): the source of the annotation.</li>
</ul>

  </div>
  <div *ngIf="pageName == 'genetic-and-physical-interactions'">
<h1 id="genetic-and-physical-interactions">Genetic and Physical Interactions</h1>
<p>The Interactions section of a gene page displays genetic and physical interactions for a gene (or its product), in <a href="http://thebiogrid.org/">BioGRID</a> format. All interactions are curated manually by PomBase or BioGRID curators.</p>
<p>Each table has five columns:</p>
<p>The first column (with no header) describes the interaction. The text reflects the type of evidence listed in the Evidence column and, where applicable, the directionality of the interaction.</p>
<p><strong>Interacting gene:</strong> The gene that interacts with the gene of interest, linked to its gene page</p>
<p><strong>Interacting product:</strong> The description of the interacting gene's product, from its gene page</p>
<p><strong>Evidence:</strong> The type of genetic interaction observed, or the type of experiment performed to detect a physical interaction. The evidence categories come from BioGRID, and are described on their <a href="http://wiki.thebiogrid.org/doku.php/experimental_systems">Experimental Evidence Codes</a> documentation page.</p>
<p><strong>Reference:</strong> The paper cited to support the interaction</p>

  </div>
  <div *ngIf="pageName == 'genotype-page'">
<h1 id="genotype-pages">Genotype pages</h1>
<p>Genotype pages are analogous to gene pages, but describe the alleles that make up a genotype, and show phenotypes associated with the genotype.</p>
<div class="figure">
<img src="assets/genotype_page.png" alt="genotype page" width="800" />
<p class="caption">genotype page</p>
</div>
<ol style="list-style-type: decimal">
<li>The genotype name. A short and/or memorable name can be assigned, but otherwise the default name is constructed by concatenating the names of the alleles that make up the genotype.</li>
<li><p>Each genotype in composed of one or more alleles. The table lists details for each allele of interest in the genotype. Any curated background details will be shown above the table of alleles.</p>
<ol style="list-style-type: lower-alpha">
<li>Gene name, linked to its gene page</li>
<li>Allele name and description. For point mutations and partial deletions, the description specifies the positions, and indicates altered residues for point mutations.</li>
<li>The allele type specifies what kind of change the allele description represents, e.g. amino acid mutation, deletion, other.</li>
<li>The expression level indicates the amount of gene product present in the assayed cells, or &quot;Not assayed&quot; if the level was not directly measured.</li>
</ol></li>
<li><p>Phenotypes annotated to the genotype are displayed as described in the <a routerLink="/documentation/gene-page-phenotypes">gene page documentation</a>.</p></li>
</ol>

  </div>
  <div *ngIf="pageName == 'high-confidence-physical-interaction-network'">
<h1 id="high-confidence-physical-interaction-network-hcpin">High Confidence Physical Interaction Network (HCPIN)</h1>
<p>The PomBase High Confidence Physical Interaction Network (HCPIN) is a collection of physical interaction data based on three subsets Gene Ontology (GO) annotations:</p>
<ol style="list-style-type: decimal">
<li>Manually curated protein binding annotations. PomBase curators annotate genes to the GO &quot;protein binding&quot; molecular function term (GO:0005515) when proteins are shown experimentally to interact in a pairwise, and presumably direct, manner, often by more than one method</li>
<li>Targets or substrates of biochemical activity (e.g. protein kinases or phosphatases, protein methyltransferases, ubiquitin-protein ligases). When one protein is shown experimentally to modify another, a GO molecular function annotation represents the activity, and the target(s) can be captured using <a href="/documentation/gene-page-gene-ontology#ann_ext">annotation extensions</a>. For example, <a routerLink="/gene/SPBC11B10.09">Cdc2</a> phosphorylates <a routerLink="/gene/SPBC146.03c">Cut3</a>, so it is annotated to 'protein serine/threonine kinase activity' with the extension 'has substrate cut3'.</li>
<li>GO protein complex annotations. The HCPIN set includes both annotations supported directly by fission yeast experiments and those inferred from an experimentally characterised ortholog.</li>
</ol>
<p>The data can be downloaded in tab-delimited format from the <a href="/downloads/datasets">Download Datasets</a> page.</p>

  </div>
  <div *ngIf="pageName == 'index'">
<h1 id="pombase-web-site-help-documentation">PomBase Web Site Help &amp; Documentation</h1>
<h3 id="DocsGenePage">Gene Pages</h3>
<ul>
<li><a routerLink="/documentation/gene-page-basic-information">Basic Information</a></li>
<li><a routerLink="/documentation/gene-page-gene-ontology">Gene Ontology</a></li>
<li><a routerLink="/documentation/gene-page-phenotypes">Phenotypes</a></li>
<li><a routerLink="/documentation/gene-page-target">Target of</a></li>
<li><a routerLink="/documentation/gene-page-transcript">Transcript</a></li>
<li><a routerLink="/documentation/gene-page-protein-features">Protein features</a></li>
<li><a routerLink="/documentation/gene-page-modifications">Modifications</a></li>
<li><a routerLink="/documentation/gene-page-sequence">Sequence</a></li>
<li><a routerLink="/documentation/gene-page-gene-expression">Gene expression</a></li>
<li><a routerLink="/documentation/taxonomic-conservation">Taxonomic conservation</a></li>
<li><a routerLink="/documentation/orthologs">Orthologs</a></li>
<li><a routerLink="/documentation/genetic-and-physical-interactions">Interactions</a></li>
<li><a href="documentation/gene-page-external-references">External references</a></li>
<li><a routerLink="/documentation/gene-page-literature">Literature</a></li>
</ul>
<h3 id="DocsSearch">Searching</h3>
<ul>
<li><a routerLink="/documentation/simple-search-documentation">Simple Search</a></li>
<li><a routerLink="/documentation/advanced-search-documentation">Advanced Search</a></li>
</ul>
<h3 id="other-pombase-pages">Other PomBase Pages</h3>
<ul>
<li>GO Slim <a routerLink="/documentation/pombase-go-slim-documentation">Documentation</a> and <a routerLink="/browse-curation/fission-yeast-go-slimming-tips">Usage tips</a></li>
<li><a routerLink="/documentation/ontology-term-page">Ontology term pages</a></li>
<li><a routerLink="/documentation/publication-page">Publication pages</a></li>
<li><a routerLink="/documentation/genotype-page">Genotype pages</a></li>
<li><a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology (FYPO) page</a></li>
<li><a routerLink="/documentation/high-confidence-physical-interaction-network">High Confidence Physical Interaction Network (HCPIN)</a></li>
<li><a routerLink="/about/version-history">Data versions</a></li>
</ul>
<h3 id="how-to-cite-pombase">How to Cite PomBase</h3>
<ul>
<li>See the <a routerLink="/about/citing-pombase">Citing PomBase</a> page</li>
</ul>
<h3 id="file-formats-for-data-submission">File Formats for Data Submission</h3>
<p>PomBase accepts batch submissions of certain types of data that appear on PomBase gene pages. For these data types, we use dedicated PomBase-specific formats:</p>
<ul>
<li><a routerLink="/documentation/phenotype-data-bulk-upload-format">Phenotypes</a></li>
<li><a routerLink="/documentation/modification-bulk-upload-file-format">Modifications</a></li>
<li><a routerLink="/documentation/qualitative-gene-expression-bulk-upload-file-format">Qualitative gene expression</a></li>
<li><a routerLink="/documentation/quantitative-gene-expression-bulk-upload-file-format">Quantitative gene expression</a></li>
</ul>
<p>For data that can be connected with sequence features or coordinates, and displayed as tracks in the genome browser, see the <a href="faqs/what-file-formats-can-i-use-submit-high-throughput-data">data format FAQ</a> and further details linked there.</p>
<h3 id="linking-to-and-from-pombase">Linking to and from PomBase</h3>
<ul>
<li>Linking to PomBase: To link to any PomBase gene page, use the systematic ID for the gene in a URL with the syntax http://www.pombase.org/gene/[systematic ID]. For example, <a routerLink="/gene/SPBC11B10.09">http://www.pombase.org/gene/SPBC11B10.09</a> links to the gene page for <a routerLink="/spombe/result/SPBC11B10.09">cdc2</a>.</li>
<li>Linking from PomBase to external resources: We can provide links from PomBase gene pages to gene- or gene product-specific <em>S. pombe</em> data for any resource that uses URLs with PomBase systematic IDs . Please contact the PomBase Curators for more information.</li>
</ul>
<h3 id="comparative-genomics">Comparative Genomics</h3>
<p>We plan to add specific documentation about comparative genomics using <em>S. pombe</em> here. In the meantime, you can look at the documentation for <a href="http://genomebrowser.pombase.org/info/docs/compara/index.html">Ensembl Compara</a>.</p>
<h3 id="more-information-...">More information ...</h3>
<p>For help with anything in PomBase not covered here, you can <app-contact-email class="contact-email" [linkName]="'contact the
curators.'"  [subject]="'Comment or problem report
...'"></app-contact-email></p>

  </div>
  <div *ngIf="pageName == 'ontology-term-page'">
<h1 id="ontology-term-pages">Ontology term pages</h1>
<p>For each ontology term loaded into PomBase, a page summarizes essential details about the term, and shows any annotations to it or its descendants via <em>is_a</em>, <em>part_of</em>, and the <em>regulates</em> relations (the GO documentation on <a href="http://geneontology.org/page/ontology-structure">Ontology Structure</a> and <a href="http://geneontology.org/page/ontology-relations">Ontology Relations</a> has some useful information on this topic). For FYPO terms only, the has_part relation is also used.</p>
<p>The main illustrations use a GO term page. Note that not all ontology terms have all of the depicted features.</p>
<div class="figure">
<img src="assets/ontology_term_page_summary.png" alt="ontology term page" width="800" />
<p class="caption">ontology term page</p>
</div>
<ol style="list-style-type: decimal">
<li>Ontology term details. The term's name, ID, and definition are shown.</li>
<li>The term's immediate parent(s), linked to the corresponding ontology term page(s).</li>
<li>Links to external browsers are available for some ontologies. GO, FYPO and PSI-MOD terms have links to <a href="http://bioportal.bioontology.org/">BioPortal</a>, and GO term pages also link to <a href="http://amigo.geneontology.org/amigo">AmiGO</a> and <a href="http://www.ebi.ac.uk/QuickGO/">QuickGO</a>. SO IDs link to <a href="http://sequenceontology.org/browser/obob.cgi">MISO</a>.</li>
<li>Link to a page that lists genes annotated to the term or its descendants. From the linked page, the gene list can be sent to the advanced search for use in queries.</li>
<li>In the annotation summary view, the term of interest is listed first, followed by any descendant term that has direct annotations. For each term, genes (or genotypes) are listed in a compact display, with separate lines only for unique combinations of gene, term, and extension. As in the <a href="/documentation/index#DocsGenePage">gene page summary display</a>, most details are hidden.</li>
</ol>
<p>As on gene pages, a <a href="documentation/index#DocsGenePage">more detailed annotation display</a> is available:</p>
<div class="figure">
<img src="assets/ontology_term_page_detail.png" alt="ontology term page" width="800" />
<p class="caption">ontology term page</p>
</div>
<ol style="list-style-type: decimal">
<li>The detailed view shows one line per annotation, with separate entries for repeat determinations of a given gene/term/extension combination (if supported by more than one line of evidence and/or reported in more than one paper). The annotated gene (or genotype) is listed in the first column.</li>
<li>The rest of the detailed annotation display (term name, extensions, evidence, reference, etc.) is as described in the gene page documentation for each annotation type.</li>
<li>The table is organized into sections, one per descendant term, and each section includes genes annotated directly to the indicated term. An icon near each descendant term ID shows the relationship type that links the terms. For example, a meiotic spindle astral microtubule (GO:1990574) is a type of astral microtubule, and a mitotic spindle astral microtubule is part of an astral microtubule.</li>
</ol>
<p>FYPO term pages have a few distinctive features:</p>
<div class="figure">
<img src="assets/ontology_term_page_fypo.png" alt="ontology term page" width="800" />
<p class="caption">ontology term page</p>
</div>
<ol style="list-style-type: decimal">
<li>Links near the top of the page go to lists of genes and genotypes.</li>
<li>The annotation display is split into single-allele and multi-allele sections, as on gene pages.</li>
<li>Annotated genotypes (rather than genes) are listed for each term.</li>
<li>In the detailed view, evidence filtering is available as described in the <a href="documentation/gene-page-phenotypes">gene page documentation</a>.</li>
</ol>
<p>Ontology term pages are available for <a href="http://geneontology.org/">GO</a>, <a routerLink="/browse-curation/fission-yeast-phenotype-ontology">FYPO</a>, <a href="http://sequenceontology.org/">SO</a>, <a href="http://www.psidev.info/MOD">PSI-MOD</a>, <a routerLink="/faqs/what-pbo-option-advanced-search">PBO</a>, and PECO, the internal ontology we use for phenotype experimental conditions.</p>

  </div>
  <div *ngIf="pageName == 'orthologs'">
<h1 id="ortholog-curation">Ortholog curation</h1>
<p>The &quot;Orthologs&quot; section of a gene page lists any orthologous genes in budding yeast (<em>Saccharomyces cerevisiae</em>) or human (<em>Homo sapiens</em>) that have been assessed and recorded manually by curators. Budding yeast entries link to the <a href="http://www.yeastgenome.org/"><em>Saccharomyces</em> Genome Database</a> (SGD), and human genes link to <a href="http://www.genenames.org/">HGNC</a>; the text descriptions come from these databases.</p>
<p>Ortholog prediction: The source of the ortholog prediction may vary. In some cases the consensus ortholog from the major ortholog predictors (<a href="http://www.ensembl.org/info/docs/compara/index.html">Compara</a>, <a href="http://inparanoid.sbc.su.se/">Inparanoid</a>, <a href="http://www.orthomcl.org/">OrthoMCL</a>) is used. Many distant orthologs have also been identified by <span class="st">PSI-BLAST</span> matches that are uniformly one-to one across species; these alignments have been submitted to the <a href="http://pfam.xfam.org/">Pfam</a> protein family database. Other ortholog predictions come from experimental data demonstrating functional correspondence or involving membership of corresponding complexes. These predictions are also aligned and submitted to Pfam before inclusion. PomBase's approach ensures that the breadth of coverage is greater than any individual prediction method, and includes many ortholog calls which are not detected by any automated method. Gradually, we will add and display supporting references for all orthology calls.</p>
<p>For species other than budding yeast and human, we provide links to Compara to display orthologs from fungi or from all species. for further information, see the <a href="http://www.pombase.org/faqs/how-can-i-find-s-pombe-orthologs-species-other-human-and-s-cerevisiae">FAQ on orthologs in additional species</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-go-slim-documentation'">
<h1 id="go-slims---gene-ontology-subsets">GO Slims - Gene Ontology subsets</h1>
<p>&quot;GO slims&quot; are subsets of the Gene Ontology (GO) that provide a broad overview of annotation distribution. Slims can offer a useful overview of a genome or the results of a large-scale experiment. For more information on GO slims, please see PomBase <a href="/browse-curation/fission-yeast-go-slimming-tips">GO Slimming Tips</a> and the <a href="http://www.geneontology.org/GO.slims.shtml">GO Slim Guide</a> at the Gene Ontology website.</p>
<p>The <a routerLink="/browse-curation/fission-yeast-go-slim-terms">GO slim term set</a> constructed for fission yeast provides almost complete coverage of gene products annotated to biological process terms. This slim set is a good starting point for users who wish to identify terms of biological interest to create a &quot;custom slim&quot;, or to become familiar with the genome contents.</p>
<p>In the GO slim <a routerLink="/browse-curation/fission-yeast-go-slim-terms">table</a>, GO IDs link to the QuickGO browser, where you can explore the ontology and annotations further. The annotation totals include annotations to the slim term and to descendants following the <em>is_a</em>, <em>part_of, regulates, positively_regulates,</em>and <em>negatively_regulates</em> relationships, and link to gene lists.</p>

  </div>
  <div *ngIf="pageName == 'publication-page'">
<h1 id="publication-pages">Publication pages</h1>
<p>For each publication loaded into PomBase, a page summarizes essential details about paper, and shows any annotations that cite it.</p>
<div class="figure">
<img src="assets/publication_page_top.png" alt="publication page top details" width="800" />
<p class="caption">publication page top details</p>
</div>
<ol style="list-style-type: decimal">
<li>Publication details, including links to <a href="https://www.ncbi.nlm.nih.gov/pubmed/">PubMed</a> and <a href="http://europepmc.org/">EuropePMC</a></li>
<li>Any community curators who have contributed annotations via Canto are acknowledged.</li>
<li>Summary views are identical to those on <a href="/documentation/index#DocsGenePage">gene pages</a>.</li>
<li>In detailed views annotations are displayed as on <a href="/documentation/index#DocsGenePage">gene pages</a>, except that a column for the annotated gene is included and the Reference column is omitted.</li>
<li>Term and evidence filtering are available for phenotype annotations as described in the <a href="documentation/gene-page-phenotypes">gene page documentation</a>.</li>
</ol>
<p>Like gene pages, publication pages include annotations of all types, with summary and detailed views available for most types:</p>
<div class="figure">
<img src="assets/publication_page_overview.png" alt="publication page" width="800" />
<p class="caption">publication page</p>
</div>

  </div>
  <div *ngIf="pageName == 'simple-search-documentation'">
<h1 id="simple-search">Simple search</h1>
<p>The simple search is available via the search box in the header on every PomBase page.</p>
<p>It searches gene names (standard names and synonyms), systematic IDs, and gene product descriptions for fission yeast genes. Human and budding yeast orthologs can also be searched by standard gene name and ORF name, respectively.</p>
<p>Start typing to see a pulldown of autocomplete matches, then choose one, or if there is a single match simply hit return, to go to the matching gene page.</p>

  </div>
  <div *ngIf="pageName == 'taxonomic-conservation'">
<h1 id="taxonomic-conservation">Taxonomic Conservation</h1>
<p>The Taxonomic conservation section provides manually assigned classifiers of taxon distribution (at the domain/kingdom level) of the product of a protein-coding gene. The classifiers come from a small controlled vocabulary maintained by PomBase curators. All applicable terms are assigned to a protein.</p>
<p>For each annotation, the summary view shows a text description, which corresponds to an entry in the internal <a routerLink="/PBO">http://www.pombase.org/faqs/what-pbo-option-advanced-search</a> term set. The detailed view adds the PBO ID and a count that links to the <a routerLink="/documentation/ontology-term-page">ontology term page</a> for the description.</p>
<h3 id="taxonomic-conservation-terms">Taxonomic conservation terms</h3>
<ul>
<li>These terms identify the taxonomic groups in which gene product ortholog(s) have been identified:
<ul>
<li>conserved in archaea</li>
<li>conserved in bacteria</li>
<li>conserved in eukaryotes</li>
<li>conserved in fungi</li>
<li>conserved in metazoa</li>
<li>conserved in vertebrates</li>
</ul></li>
<li>These two terms represent taxon restrictions, and indicate that a gene product has not been detected outside these taxa:
<ul>
<li>conserved in fungi only</li>
<li>conserved in eukaryotes only</li>
</ul></li>
<li>Other terms
<ul>
<li><em>Schizosaccharomyces</em> specific - indicates that a gene product is only identified in the genus <em>Schizosaccharomyces</em> at present, but is present in more than one species in this genus</li>
<li><em>Schizosaccharomyces pombe</em> specific - indicates that a gene product is identified only in <em>S. pombe</em>, not in any other species even in the <em>Schizosaccharomyces</em> genus</li>
<li>no apparent <em>S. cerevisiae</em> ortholog - indicates a gene product which is conserved at least outside the <em>Schizosaccharomyces</em> but is absent from <em>S. cerevisiae</em></li>
<li>faster evolving duplicate - indicates that the copy of a gene product which has duplicated after the divergence of <em>Schizosaccharomyces</em> from other yeasts is evolving faster than its duplicate partner (judged because the fission yeast duplicate is more closely related to the orthologs in other species, but is the best hit of this gene product)</li>
<li>predominantly single copy (one to one) - the gene product is conserved largely one-to one across species with minor exceptions (for example, frequently duplicated in the vertebrate lineage)</li>
<li>orthologs cannot be distinguished - the gene product appears to be conserved, but it is not possible to identify orthologs unambiguously</li>
</ul></li>
</ul>
<h3 id="additional-notes">Additional notes:</h3>
<ul>
<li>Any term which has &quot;conserved in fungi&quot; but does <em>not</em> have &quot;no apparent <em>S. cerevisiae</em> ortholog&quot; will have a manually annotated <em>S. cerevisiae</em> ortholog.</li>
<li>Any gene product which is annotated as &quot;conserved in vertebrates&quot; will have a manually annotated human ortholog.</li>
</ul>

  </div>
  </div>
</div>
<div *ngIf="section == 'documents'">
  <div class="docs-menu">
<h3 id="documents"><a routerLink="/documents">Documents</a></h3>
<ul>
<li><a routerLink="/documents/pombe2015">pombe2015</a></li>
<li><a routerLink="/documents/pombe2013">pombe2013</a></li>
<li><a routerLink="/documents/pombe2011">pombe2011</a></li>
</ul>

  </div>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h2 id="conferences-and-workshops">Conferences and Workshops</h2>
<h3 id="international-fission-yeast-meeting-series">International Fission Yeast Meeting series</h3>
<!--
<a routerLink="/documents/pombe2017">pombe2017</a>
Banff, Alberta, Canada
14-19 May 2017
-->
<p><a routerLink="/documents/pombe2015">pombe2015</a> Kobe, Japan 21-25 June 2015</p>
<p><a routerLink="/documents/pombe2013">pombe2013</a> London, UK 24-29 June 2013</p>
<p><a routerLink="/documents/pombe2011">pombe2011</a> Boston, MA, USA 25-30 June 2011</p>
<h3 id="other-conferences-and-workshops">Other Conferences and Workshops</h3>
<ul>
<li><a href="ftp://ftp.pombase.org/pombe/Archived_directories/Presentations_workshops_and_meeting_abstracts/EMBO_workshop/">2009 EMBO Workshop</a>, Manchester, UK (FTP directory)</li>
<li><a href="ftp://ftp.pombase.org/pombe/Archived_directories/Presentations_workshops_and_meeting_abstracts/Computing_Workshop/">Computing workshops, 2006 &amp; 2007</a>, various dates and locations (FTP directory)</li>
<li><a href="ftp://ftp.pombase.org/pombe/Archived_directories/Presentations_workshops_and_meeting_abstracts/Computing_Workshop/Fission_yeast_computing_workshop_man_Mar2006.ppt">2006 Fission yeast genome analysis workshop</a>, Hinxton, UK - course manual (file at FTP site)</li>
<li><a href="ftp://ftp.pombase.org/pombe/Archived_directories/Presentations_workshops_and_meeting_abstracts/Pombe_meeting_abstract_books/Wellcome_Euro_Yeast_27_2_06.pdf">2006 European fission yeast meeting</a>, Hinxton, UK - abstracts (file at FTP site)</li>
</ul>
<h2 id="other-documents">Other documents</h2>
<!--
-   [2017 PomBase infographic]() (PDF)
-->
<ul>
<li><a routerLink="/sites/pombase.org/files/documents/2013_pombase_survey_summary.pdf">2013 PomBase user survey results</a> (PDF)</li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombe2011'">
<h1 id="pombe2011---6th-international-fission-yeast-meeting">pombe2011 - 6th International Fission Yeast Meeting</h1>
<p>Boston, MA, USA, 25-30 June 2011</p>
<h2 id="pombase-posters">PomBase Posters</h2>
<ul>
<li>P50: Phenotypes in PomBase and Beyond</li>
<li>P96: <a routerLink="/sites/pombase.org/files/documents/pombe2011_pombase.pdf">PomBase.org</a></li>
<li>P159: PomBase Communty Curation: A fast track to capture expert knowledge</li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombe2013'">
<h1 id="pombe2013---seventh-interational-fission-yeast-meeting">pombe2013 - Seventh Interational Fission Yeast Meeting</h1>
<p>London, UK, 24-29 June 2013</p>
<h2 id="abstracts">Abstracts</h2>
<ul>
<li><a href="/sites/pombase.org/files/documents/pombe2013_abstractbook-final2.pdf">Abstract book</a> (PDF)</li>
</ul>
<h2 id="pombase-presentations">PomBase presentations</h2>
<h3 id="talks">Talks</h3>
<ul>
<li>Tuesday Lunchtime: Paper Curation - Midori Harris and Antonia Lock</li>
<li>Thursday Lunchtime: PomBase Gene Pages and the Genome Browser - Val Wood and Mark McDowall</li>
</ul>
<h3 id="posters">Posters</h3>
<ul>
<li>219: A configurable tool for community literature curation</li>
<li>220: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2013/PomBase.org.pdf">PomBase.org</a></li>
<li>221: Phenotype curation in PomBase</li>
<li>222: Using Gene Ontology biological process co-annotation for ontology and annotation quality control</li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombe2015'">
<h1 id="pombe2015---eighth-international-fission-yeast-meeting">pombe2015 - Eighth International Fission Yeast Meeting</h1>
<p>Kobe, Japan, 21-26 June 2015</p>
<h2 id="pombase-presentations">PomBase presentations</h2>
<h3 id="talks">Talks</h3>
<ul>
<li>PomBase Workshop
<ul>
<li><a href="/sites/pombase.org/files/documents/workshop_pombase_tour_vw2015.pdf">How to get more from PomBase</a> - a brief tour of PomBase gene pages, curated data types, querying, tools, and external links</li>
<li><a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2015/pombe2015_UsingGenomeBrowser.pdf">Using the PomBase Ensembl Browser</a> - a demo of the PomBase genome browser features and functions (PDF; large file)</li>
<li><a href="/sites/pombase.org/files/documents/pombe2015_curation_workshop_mah.pdf">Community curation using Canto</a> - how to create, edit and submit annotations based on published experiments</li>
</ul></li>
<li>81: Hidden in plain sight: The eukaryotically conserved unstudied proteins and a framework for their classification and characterisation</li>
<li>82: <a href="http://dev.pombase.org/sites/pombase.org/files/documents/multi_allele_pombe2015.pdf">Curating multi-allele phenotypes in PomBase</a> (PDF; poster version)</li>
</ul>
<h3 id="posters">Posters</h3>
<ul>
<li>33: Putting it all together with fission yeast: Data connections within and beyond PomBase</li>
<li>73: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2015/PomBaseGenomeBrowser.pdf">Using the PomBase Genome Browser</a> (PDF)</li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombe2017'">
<h1 id="pombe2017---9th-international-fission-yeast-meeting">pombe2017 - 9th International Fission Yeast Meeting</h1>
<p>Banff, Alberta, Canada, 14-19 May 2017</p>
<h2 id="pombase-posters">PomBase Posters</h2>
<!--
-   Infographic
-   community curation
-   Using fission yeast QC methods to correct and improve GO annotation across eukaryotic species

Talk
The new, improved PomBase
-->

  </div>
  </div>
</div>
<div *ngIf="section == 'downloads'">
  <div class="docs-menu">
<h3 id="downloads"><a routerLink="/downloads">Downloads</a></h3>
<ul>
<li><a routerLink="/datasets">Datasets</a></li>
<li><a routerLink="/documents">Documents</a></li>
<li><a routerLink="/downloads/chado-database-dumps">Chado database dumps</a></li>
</ul>

  </div>
  <div class="docs-content">
  <div *ngIf="pageName == 'cds-coordinates'">
<h3 id="cds-coordinates">CDS coordinates</h3>
<p><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/CDS_Coordinates/">Directory</a></p>
<ul>
<li>CDS coordinates (start and stop only; one line per gene)
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/CDS_Coordinates/chromosome1.cds.coords">Chromosome 1</a></li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/CDS_Coordinates/chromosome2.cds.coords">Chromosome 2</a></li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/CDS_Coordinates/chromosome3.cds.coords">Chromosome 3</a></li>
</ul></li>
<li>CDS exon coordinates (one line per exon)
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Exon_Coordinates/chromosome1.exon.coords">Chromosome 1</a></li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Exon_Coordinates/chromosome2.exon.coords">Chromosome 2</a></li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Exon_Coordinates/chromosome3.exon.coords">Chromosome 3</a></li>
</ul></li>
</ul>

  </div>
  <div *ngIf="pageName == 'chado-database-dumps'">
<h2 id="chado-database-dumps">Chado database dumps</h2>
<p>Monthly data dump from the PomBase <a href="http://gmod.org/wiki/Chado">Chado</a> database are available for <a href="http://curation.pombase.org/dumps/releases/">download</a>.</p>
<p>For data dumps made between July 2012 and January 2017, the file names include release numbers that correspond to the &quot;Annotation Version&quot; numbers in the <a routerLink="/about/version-history">Data version history</a> table.</p>

  </div>
  <div *ngIf="pageName == 'data-mappings'">
<p><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/">Mapping file directory</a></p>
<p><strong>Gene Product to UniProt Accession Number</strong></p>
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/PomBase2UniProt.tsv">PomBase2UniProt.tsv</a> tab-delimited file of systematic identifier mapped to UniProt Accession Number</li>
</ul>
<h3 id="all-gene-names">All Gene Names</h3>
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/allNames.tsv">allNames.tsv</a> tab-delimited file of systematic name, primary name (empty if not assigned), followed by all synonyms</li>
</ul>
<h3 id="systematic-id-to-gene-product">Systematic ID to Gene Product</h3>
<p>Files include systematic name, primary name (empty if not assigned), synonyms (empty if not assigned), followed by gene product description</p>
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/sysID2product.tsv">sysID2product.tsv</a> - protein-coding genes</li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/sysID2product.rna.tsv">sysID2product.rna.tsv</a> - non-coding RNA genes</li>
</ul>
<p>Note: A tab-delimited file of systematic identifiers mapped to EC numbers was previously maintained, but has not been updated since March 2012. The most recent version of the <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/OLD/gp2EC.txt">gp2EC.txt</a> file is available in the FTP archive, but because it is out of date it may contain errors or omissions.</p>

  </div>
  <div *ngIf="pageName == 'genome-datasets'">
<p>Note that most of the datasets available here are compressed (.gz), and can be uncompressed by utilities available in all common operating systems.</p>
<h2 id="genome-feature-annotations"><span id="features"></span>Genome Feature Annotations</h2>
<p>Each link below goes to a directory within the PomBase FTP site, in which there is one file for the whole genome, plus individual files for each chromosome. Choose the file(s) you need and click to download.</p>
<h3 id="feature-coordinates-only">Feature coordinates only</h3>
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Chromosome_Dumps/gff3/">GFF3 format - feature coordinates</a></li>
</ul>
<h3 id="feature-coordinates-and-sequence-data">Feature coordinates and sequence data</h3>
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Chromosome_Dumps/embl/">EMBL format</a></li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Chromosome_Dumps/genbank/">GenBank format</a></li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Chromosome_contigs/">Chromosome contigs</a> in enriched EMBL-based format for loading into <a href="http://www.sanger.ac.uk/resources/software/artemis/">Artemis</a></li>
</ul>
<h2 id="sequence-datasets-fasta-format"><span id="sequences"></span>Sequence Datasets (FASTA format)</h2>
<p>These files contain only DNA sequence data and headers.</p>
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Chromosome_Dumps/fasta/">Chromosomes</a> - link to FTP site</li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Chromosome_Dumps/fasta/">Coding sequences (CDS)</a> - link to FTP site</li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Chromosome_Dumps/fasta/">cDNA</a> - link to FTP site</li>
<li><a routerLink="/downloads/utr">UTRs</a></li>
</ul>
<p>You can also browse the files available from the <a href="ftp://ftp.ensemblgenomes.org/pub/current/fungi/">Ensembl Genomes FTP site</a>.</p>

  </div>
  <div *ngIf="pageName == 'go-associations'">
<p><a href="ftp://ftp.geneontology.org/pub/go/gene-associations/gene_association.pombase.gz"><em>S. pombe</em> GO annotations</a> (link downloads gzipped file from the GO repository) include annotations made by manual literature curation, annotations inferred from keyword mappings based on curated descriptions, and annotations shared by the UniProt <a href="http://www.ebi.ac.uk/GOA">GOA</a> team. The file is tab-delimited text, so it can be opened in a spreadsheet application or parsed with a script; the <a href="http://geneontology.org/page/go-annotation-file-gaf-format-20">format</a> is described on the GO website.</p>
<p>Also see the list of protein complexes in the FTP directory (<a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Complexes/" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Complexes/</a>), which uses GO macromolecular complex terms and IDs.</p>
<p>Previous versions of the <em>S. pombe</em> GO annotation file can be retrieved from the <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Gene_ontology/OLD/">PomBase FTP archive</a> or via the GO CVSweb interface (<a href="http://cvsweb.geneontology.org/cgi-bin/cvsweb.cgi/go/gene-associations/gene_association.pombase.gz">PomBase</a> files and the very old <a href="http://cvsweb.geneontology.org/cgi-bin/cvsweb.cgi/go/gene-associations/gene_association.GeneDB_Spombe.gz">GeneDB</a> files are at separate locations on the GO site).</p>
<p>To cite the fission yeast GO data, please see <a routerLink="/about/citing-pombase">Citing PomBase</a>.</p>

  </div>
  <div *ngIf="pageName == 'index'">
<h2 id="downloads" class="title block-title">Downloads</h2>
<ul>
<li><a routerLink="/datasets">Datasets</a>: Genome sequence and features, annotations, etc.</li>
<li><a routerLink="/documents">Documents</a>: Conference materials, manuals, reports, etc.</li>
<li><a routerLink="/downloads/chado-database-dumps">Chado database dumps</a>: Dumps of curated data stored in the PomBase Chado database.</li>
</ul>

  </div>
  <div *ngIf="pageName == 'intron-data'">
<p>Downloadable intron datasets are available in FASTA format from the <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/FASTA/">PomBase FTP site</a>.</p>
<p>Note that you can also use the Advanced Search to find all genes containing introns, as described in the <a href="http://www.pombase.org/faqs/how-can-i-retrieve-intron-coordinates-or-sequences">FAQ on introns</a>.</p>
<hr />
<p>We also provide access to archived <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Archived_directories/Intron_Data/OLD/">Intron Data</a>. Please note that this set of intron data reflect the dataset at the time of publication of the <em>S. pombe</em> genome sequence, and does not include any new introns, or changes to introns, since then. The intron data archive includes:</p>
<ul>
<li>List of all <em>S. pombe</em> intron-containing genes</li>
<li>List of all <em>S. pombe</em> genes with exon numbers</li>
<li><em>S. pombe</em> intron sequences in FASTA format (donor sequence and final 35 bases of acceptor)</li>
<li>All introns with branch site aligned</li>
<li>Donor frequency</li>
<li>Branch frequency</li>
<li>A README file is available with additional descriptions of file contents</li>
</ul>
<p>Also see the <a routerLink="/faqs/how-can-i-retrieve-intron-coordinates-or-sequences">FAQ on how to retrieve intron coordinates and sequences</a>.</p>

  </div>
  <div *ngIf="pageName == 'manually-curated-orthologs'">
<h2 id="manually-curated-ortholog-sets">Manually curated ortholog sets</h2>
<ul>
<li>Human: <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/orthologs/human-orthologs.txt.gz">human-orthologs.txt.gz</a> (gzipped file; curation information in <a routerLink="/faqs/how-can-i-find-s-pombe-orthologs-human-gene">FAQ</a> and <a routerLink="/documentation/orthologs">documentation</a>)</li>
<li><em>S. cerevisiae</em> <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/orthologs/cerevisiae-orthologs.txt">cerevisiae-orthologs.txt</a> (curation information in <a routerLink="/faqs/how-can-i-search-s-cerevisiae-orthologshomologs-s-pombe-gene">FAQ</a> and <a routerLink="/documentation/orthologs">documentation</a>)</li>
</ul>

  </div>
  <div *ngIf="pageName == 'phenotype-annotations'">
<h2 id="full-phenotype-annotations">Full phenotype annotations</h2>
<p><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Phenotype_annotations/phenotype_annotations.pombase.phaf.gz">Phenotype annotations</a> (link downloads gzipped file from PomBase ftp) for alleles of <em>S. pombe</em> genes are manually curated from the literature using Fission Yeast Phenotype Ontology (FYPO) terms. Note: this file contains annotations for single allele phenotypes (single mutants) only.</p>
<p>The file is in a version of the PomBase phenotype data bulk annotation format (PHAF), detailed below. This format in nearly identical to the one that can be used to submit phenotype annotations to PomBase in bulk, as described on the <a routerLink="/submit-data/phenotype-data-bulk-upload-format">Phenotype Data Bulk Upload Format</a> page, with the addition of the Database column.</p>
<p>Propagating phenotype annotations: Note that the file contains only direct annotations to FYPO terms. It does not include annotations that can be inferred by propagating between terms within the ontology. To make full use of the FYPO annotation data, we strongly recommend also using the ontology structure and inferred annotations. Please contact the <a routerLink="/mailto:helpdesk@pombase.org">PomBase helpdesk</a> if you need assistance.</p>
<h2 id="viability-summary">Viability summary</h2>
<p>A <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Phenotype_annotations/FYPOviability.tsv">set of &quot;viability summary&quot; data</a>, as shown at the top of the FYPO table on each gene page, is available as a downloadable file. The file has two columns: the gene systematic ID and one of three values: &quot;viable&quot;, &quot;inviable&quot; or &quot;condition-dependent&quot;.</p>
<p>To cite the fission yeast phenotype data (complete or viability summary), please see <a routerLink="/about/citing-pombase">Citing PomBase</a>.</p>
<h2 id="phaf-download-format">PHAF download format</h2>
<p>A column marked &quot;mandatory&quot; will always have an entry; non-mandatory columns may be empty.</p>
<table>
<thead>
<tr class="header">
<th align="left">Column</th>
<th align="left">Contents</th>
<th align="left">Example</th>
<th align="left">Mandatory?</th>
<th align="left">Multiple entries allowed?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">1</td>
<td align="left">Database</td>
<td align="left">PomBase</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">2</td>
<td align="left">Gene systematic ID</td>
<td align="left">SPBC11B10.09</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">3</td>
<td align="left">FYPO ID</td>
<td align="left">FYPO:0000001</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">4</td>
<td align="left">Allele description</td>
<td align="left">G146D Yes</td>
<td align="left">No</td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="left">5</td>
<td align="left">Expression</td>
<td align="left">endogenous Yes</td>
<td align="left">No</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">6</td>
<td align="left">Parental strain</td>
<td align="left">975 h+</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">7</td>
<td align="left">Strain name (background)</td>
<td align="left">SP286</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">8</td>
<td align="left">Genotype description</td>
<td align="left">h+ ura4-D18 leu1-32 ade6-M210</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">9</td>
<td align="left">Gene name</td>
<td align="left">cdc2</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">10</td>
<td align="left">Allele name</td>
<td align="left">cdc2-1w</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">11</td>
<td align="left">Allele synonym</td>
<td align="left">cdc2-1W</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">12</td>
<td align="left">Allele type</td>
<td align="left">amino acid mutation</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">13</td>
<td align="left">Evidence</td>
<td align="left">ECO:0000000</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">14</td>
<td align="left">Condition</td>
<td align="left">at high temperature</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">15</td>
<td align="left">Penetrance</td>
<td align="left">85%</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">16</td>
<td align="left">Expressivity</td>
<td align="left">medium</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">17</td>
<td align="left">Extension</td>
<td align="left">assayed_using(PomBase:SPBC582.03)</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">18</td>
<td align="left">Reference</td>
<td align="left">PMID:23697806</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">19</td>
<td align="left">taxon</td>
<td align="left">taxon:4896</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">20</td>
<td align="left">Date</td>
<td align="left">20120101</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
</tbody>
</table>
<h2 id="notes">Notes:</h2>
<ol style="list-style-type: decimal">
<li>The database that produced the file -- included to facilitate data sharing with other databases. Obviously always PomBase for this file.</li>
<li>Gene systematic ID should be self-explanatory.</li>
<li>To use the FYPO IDs, see the <a href="/browse-curation/fission-yeast-phenotype-ontology">FYPO summary page</a> and the <a href="/faqs/how-can-i-browse-phenotype-ontology-fypo">FAQ on browsing FYPO</a>. If you can't find a term you need, email the <a routerLink="/mailto:helpdesk@pombase.org">Helpdesk</a> for assistance; we can add new FYPO terms as needed.</li>
<li>The allele description specifically describes the change; see the table at the bottom of the the <a href="/submit-data/phenotype-data-bulk-upload-format">Phenotype Data Bulk Upload Format</a> page.</li>
<li>The Expression column contains one of these values: 'overexpression', 'knockdown', 'endogenous', 'null', 'not specified'. Deletions should always have 'null' expression.</li>
<li>The Parental strain column is for the parental strain designation, such as 972 h-, 975 h+, etc. This column must be filled in, but &quot;unknown&quot; is allowed.</li>
<li>The Strain name (background) column is used for a lab's in-house name/ID/designation for the background strain (i.e. the derivative of the parental strain that has selectable marker alleles etc.). The description in the Genotype column should match this background strain.</li>
<li>The Genotype description column is for alleles in the background, such as selectable markers; these details are optional. To avoid redundancy, it does not repeat the allele of interest (from column 4 or 10) in the genotype column.</li>
<li>Gene names are optional for upload, but are included where available in the download.</li>
<li>Allele names are optional for upload, but are included where available in the download. Column 10 lists the preferred allele name, and any alternative names are in column 11.</li>
<li>See note 10 above. Multiple entries are separated with pipes (|).</li>
<li>Allowed allele types, example descriptions, etc. are shown in the table at the bottom of the the <a href="/submit-data/phenotype-data-bulk-upload-format">Phenotype Data Bulk Upload Format</a> page.</li>
<li>For the Evidence column, we use a small selection from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO). Please contact the <a routerLink="/mailto:helpdesk@pombase.org">Helpdesk</a> if you need assistance using these IDs</li>
<li>Similarly, Conditions use a small ontology maintained in-house by PomBase curators, and we can help you interpret the IDs. Multiple entries are shown in cases where more than one condition detail applies at the same time (e.g. high temperature, minimal medium), and are separated with commas (,).</li>
<li>Penetrance describes the proportion of a population that shows a cell-level phenotype. Penetrance data are represented as percents or entries from the in-house FYPO_EXT ontology (FYPO_EXT:0000001 = high; FYPO_EXT:0000002 = medium; FYPO_EXT:0000003 = low; FYPO_EXT:0000004 = full).</li>
<li>Expressivity captures information about phenotype severity, using the FYPO_EXT ontology described in note 15.</li>
<li>The Extension column is used to record when a mutation in one gene affects another gene or its product. For example, if a mutation in gene A decreases its ability to phosphorylate protein B, the A allele is annotated to the phenotype &quot;decreased protein kinase activity&quot; with the ID for gene B in an extension. In the downloadable file, an annotation can have multiple comma-separated extensions  if they combine to form a &quot;compound&quot; extension (e.g. two or more genes assayed together).</li>
<li>The Reference column has the publication's PubMed ID (PMID).</li>
<li>The taxon will usually be 4896 (the NCBI taxon ID for <em>Schizosaccharomyces pombe</em>), although an NCBI taxon ID for a specific <em>S. pombe</em> strain would be allowed.</li>
<li>The date is the date on which the annotations are created.</li>
</ol>

  </div>
  <div *ngIf="pageName == 'protein-datasets'">
<p><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/FASTA/">Protein dataset in FASTA format</a> (parent directory; use the pep.fa.gz file)<br />
<a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Protein_data">Other protein data</a> (parent directory; contains README with release notes)</p>
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Protein_data/PeptideStats.tsv">PeptideStats</a> - Predicted molecular weight (kDa), predicted pI, charge, length (residues), codon adaptation index (CAI)</li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/sysID2product.tsv">Product list</a> Systematic ID and product</li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Protein_data/aa_composition.tsv">Amino acid composition</a></li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Protein_data/Protein_Features.tsv">Protein features</a>, such as domains and family assignments</li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Protein_data/pombase.modifications.gz">Protein modification</a> annotations</li>
</ul>

  </div>
  <div *ngIf="pageName == 'utr'">
<h2 id="available-utr-datasets">Available UTR datasets</h2>
<ul>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/UTR/5UTR.fa.gz">5' UTR</a></li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/UTR/3UTR.fa.gz">3' UTR</a></li>
<li><a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/UTR/">Parent directory link</a></li>
</ul>
<p>These datasets contain the default UTR features, determined by PomBase curators as described in <a href="/faqs/how-do-you-determine-genes-full-length-transcript-utr-coordinates-transcription-start-and-end-s">this FAQ</a>.</p>
<p>The headers are set out like so:</p>
<pre style="margin-left: 40px;"><code>&gt;SPCC338.13|1351474|1354474|-1|cog4|III|protein_coding|Golgi transport complex subunit Cog4 (predicted)
 ^          ^       ^       ^  ^    ^   ^              ^
 |          |       |       |  |    |   |              \_ Description
 |          |       |       |  |    |   |
 |          |       |       |  |    |   \_ Feature type
 |          |       |       |  |    |
 |          |       |       |  |    \_ Chromosome
 |          |       |       |  |
 |          |       |       |  \_ Gene Name  
 |          |       |       |
 |          |       |       \_ Strand
 |          |       |
 |          |       \_ Gene End
 |          |     
 |          \_ Gene Start
 |
 \_ Systematic_ID</code></pre>
<p>The UTR files are updated with each PomBase data update. Old versions are <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Archived_directories/UTRs/">archived</a>.</p>

  </div>
  </div>
</div>
<div *ngIf="section == 'faq'">
  <div class="docs-menu">
<h3 id="faq"><a routerLink="/faq">FAQ</a></h3>
<ul>
<li><a routerLink="/faq/data-submission-and-formats">Data submission and formats</a></li>
<li><a routerLink="/faq/datasets">Datasets</a></li>
<li><a routerLink="/faq/finding-data">Finding data</a></li>
<li><a routerLink="/faq/gene-page">Gene page</a></li>
<li><a routerLink="/faq/genome-browser">Genome browser</a></li>
<li><a routerLink="/faq/genome-statistics-and-lists">Genome statistics and lists</a></li>
<li><a routerLink="/faq/orthology">Orthology</a></li>
<li><a routerLink="/faq/querying-searching">Querying/Searching</a></li>
<li><a routerLink="/faq/sequence-retrieval">Sequence Retrieval</a></li>
<li><a routerLink="/faq/tools-and-resources">Tools and resources</a></li>
<li><a routerLink="/faq/using-ontologies">Using ontologies</a></li>
</ul>

  </div>
  <div class="docs-content">
  <div *ngIf="pageName == 'data-submission-and-formats'">
<h2 id="data-submission-and-formats">Data submission and formats</h2>

  </div>
  <div *ngIf="pageName == 'datasets'">
<h2 id="datasets">Datasets</h2>

  </div>
  <div *ngIf="pageName == 'finding-data'">
<h2 id="finding-data">Finding data</h2>

  </div>
  <div *ngIf="pageName == 'gene-page'">
<h2 id="gene-page">Gene page</h2>

  </div>
  <div *ngIf="pageName == 'genome-browser'">
<h2 id="genome-browser">Genome browser</h2>

  </div>
  <div *ngIf="pageName == 'genome-statistics-and-lists'">
<h2 id="genome-statistics-and-lists">Genome statistics and lists</h2>

  </div>
  <div *ngIf="pageName == 'index'">
<h1 id="frequently-asked-questions">Frequently asked questions</h1>
<p>Welcome to the PomBase FAQ. Choose a link in the left-hand menu to see the questions and answers in that category.</p>
<p>If you can't find the information you need here or in the PomBase documentation, please contact the helpdesk.</p>

  </div>
  <div *ngIf="pageName == 'orthology'">
<h2 id="orthology">Orthology</h2>

  </div>
  <div *ngIf="pageName == 'querying-searching'">
<h2 id="queryingsearching">Querying/Searching</h2>

  </div>
  <div *ngIf="pageName == 'sequence-retrieval'">
<h2 id="sequence-retrieval">Sequence Retrieval</h2>

  </div>
  <div *ngIf="pageName == 'tools-and-resources'">
<h2 id="tools-and-resources">Tools and resources</h2>

  </div>
  <div *ngIf="pageName == 'using-ontologies'">
<h2 id="using-ontologies">Using ontologies</h2>

  </div>
<div *ngIf="pageName == 'tools-and-resources' || pageName == 'can-i-access-pombase-via-api'">
<h3 id="can-i-access-pombase-via-api">Can I access PomBase via an API?</h3>
<!-- pombase_categories: Tools and resources -->
<p>Yes, the Ensembl API can be used with PomBase, as documented here:</p>
<ol style="list-style-type: decimal">
<li><a href="http://genomebrowser.pombase.org/info/docs/api/api_installation.html">Ensembl Perl API installation instructions</a></li>
<li><a href="http://www.ensembl.org/info/docs/Doxygen/core-api/index.html">Ensembl core database API documentation</a></li>
<li><a href="http://genomebrowser.pombase.org/info/docs/api/core/core_tutorial.html">Tutorial for using the API with the core database</a> - Includes examples about connecting to the database, retrieving chromosomes, genes, transcript and translations along with the corresponding xrefs.</li>
</ol>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'can-i-access-pombase-via-sql'">
<h3 id="can-i-access-pombase-via-sql">Can I access PomBase via SQL?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>PomBase does not have a publicly accessible SQL query server, but <a routerLink="/downloads/chado-database-dumps">Chado database dumps</a> (PostgresQL) are available to download and query locally.</p>
<p>The <a href="http://ensemblgenomes.org/info/access/mysql">Ensembl Genomes (EG) MySQL database</a> server does provide access to query <em>S. pombe</em> data. Note, however, that EG is updated much less frequently than PomBase, so EG data will rarely be as up-to-date as the PomBase web site.</p>
<p>MySQL dumps of EG data, including <em>Schizosaccharomyces</em> species, are available from <a href="ftp://ftp.ensemblgenomes.org/pub/current/fungi/mysql/">EG's FTP site</a>.</p>

</div>
<div *ngIf="pageName == 'tools-and-resources' || pageName == 'can-i-convert-file-gff-gtf'">
<h3 id="can-i-convert-file-gff-gtf">Can I convert a file from GFF to GTF?</h3>
<!-- pombase_categories: Tools and resources -->
<p>PomBase does not offer a GFF-to-GTF converter. There is a perl script on SEQanswers, which uses the module Bio::Tools::GFF from the BioPerl library, available from http://seqanswers.com/forums/showpost.php?p=22529&amp;postcount=4</p>

</div>
<div *ngIf="pageName == 'tools-and-resources' || pageName == 'can-i-convert-file-gtf-gff'">
<h3 id="can-i-convert-file-gtf-gff">Can I convert a file from GTF to GFF3?</h3>
<!-- pombase_categories: Tools and resources -->
<p>PomBase does not offer a converter. The Sequence Ontology site has a conversion script that can be used via a web form (at http://www.sequenceontology.org/cgi-bin/converter.cgi) or checked out from their CVS repository.</p>

</div>
<div *ngIf="pageName == 'tools-and-resources' || pageName == 'can-i-convert-ids-other-databases-or-pombase-ids'">
<h3 id="can-i-convert-ids-other-databases-or-pombase-ids">Can I convert IDs from other databases to or from PomBase IDs?</h3>
<!-- pombase_categories: Tools and resources -->
<p>PomBase does not have its own tool for ID conversion. We suggest you try the EBI's PICR web service ( <a href="http://www.ebi.ac.uk/Tools/picr/" class="uri">http://www.ebi.ac.uk/Tools/picr/</a>), which can convert between UniProtKB, RefSeq, Ensembl Genomes (including <em>S. pombe</em> systematic IDs), and many other common database IDs.</p>
<p>For UniProt IDs, we provide a static mapping file of PomBase systematic IDs and UniProtKB accessions, available on the <a routerLink="/downloads/data-mappings">Data Mapping</a> page and by FTP from <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/PomBase2UniProt.tsv" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/PomBase2UniProt.tsv</a>.</p>
<p>For other <em>Schizosaccharomyces</em> species (<em>S. japonicus</em>, <em>S. octosporus</em>, <em>S. cryophilus</em>), the PICR service converts between UniProtKB accessions and the identifiers used in Rhind <em>et al.</em> Comparative functional genomics of the fission yeasts (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>), but with one caveat: the IDs used by PICR, UniProtKB, and QuickGO contain underscores, whereas those in Rhind <em>et al.</em> do not (e.g. SJAG_00455 versus SJAG00455). The IDs with underscores are correct.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'can-i-do-enrichment-analysis-using-phenotypes'">
<h3 id="can-i-do-enrichment-analysis-using-phenotypes">Can I do an enrichment analysis using phenotypes?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>A small number of enrichment tools use phenotype data. See the <a routerLink="/faq/how-can-i-find-significant-shared-fypo-annotations-for-genes-in-a-list">FAQ on FYPO term enrichment</a>.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'can-i-do-go-term-enrichment-other-schizosaccharomyces-species-e.g.-s.-japonicus'">
<h3 id="can-i-do-go-term-enrichment-other-schizosaccharomyces-species-e.g.-s.-japonicus">Can I do GO term enrichment for other <em>Schizosaccharomyces</em> species (e.g. <em>S. japonicus</em>)?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>There are two possible approaches:</p>
<ol style="list-style-type: decimal">
<li>Retrieve a set of GO annotations in GAF format for <em>S. japonicus</em>, <em>S. octosporus</em> or <em>S. cryophilus</em>, as described in the <a routerLink="/faq/how-can-i-find-go-annotations-for-other-schizosaccharomyces-species-e-g-s-japonicus">relevant FAQ</a>. Use the GO annotation dataset and your gene list for enrichment.</li>
</ol>
<p>OR</p>
<ol start="2" style="list-style-type: decimal">
<li>In your gene list of interest, substitute the <em>Schizosaccharomyces</em> species gene IDs with the IDs of orthologous <em>S. pombe</em> genes. For ortholog IDs, see the <a routerLink="/faq/how-can-i-find-orthologs-between-s-pombe-and-other-schizosaccharomyces-species">FAQ on <em>Schizosaccharomyces</em> orthologs</a>, and use the indicated table from Rhind <em>et al.</em> Comparative functional genomics of the fission yeasts (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>).</li>
</ol>
<p>In either case, you can then proceed as described in the <a routerLink="/faq/how-can-i-find-significant-shared-go-annotations-for-genes-in-a-list">FAQ on <em>S. pombe</em> GO enrichment</a>. For the first option, use the Princeton GO Term Finder or another enrichment tool that allows you to use your own GAF, and include the GO Slim analysis using GO Term Mapper as recommended in the <a routerLink="/faq/how-can-i-find-significant-shared-go-annotations-genes-list">GO enrichment FAQ</a>.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'can-i-download-all-s.-pombe-phenotype-data'">
<h3 id="can-i-download-all-s.-pombe-phenotype-data">Can I download all <em>S. pombe</em> phenotype data?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>Yes, the <a routerLink="/downloads/phenotype-annotations">Phenotype annotations</a> page offers two options, a complete phenotype annotation file and a &quot;viability summary&quot; for deletion mutants. At present, the full file contains all manually curated single mutant phenotypes, and is in the same format as PomBase uses for bulk phenotype data submissions (see the <a routerLink="/faq/what-file-formats-can-i-use-to-submit-high-throughput-data">file formats FAQ</a>). Further information on the viability summary is available in the <a routerLink="/faq/can-i-get-a-list-of-essential-pombe-genes">essential genes FAQ</a>.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'can-i-download-interaction-data-used-by-esyn'">
<h3 id="can-i-download-interaction-data-used-by-esyn">Can I download the interaction data used by esyN?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists -->
<p>The <em>S. pombe</em> networks in <a href="http://www.esyn.org/">esyN</a> use the PomBase <a routerLink="/documentation/high-confidence-physical-interaction-network">High Confidence Physical Interaction Network</a> (HCPIN) data. The data can be downloaded in tab-delimited format from the <a routerLink="/downloads/datasets">Download Datasets</a> page, which links to an FTP directory with two files. One contains physical interaction data, and the other contains GO substrate data. Each file lists two gene systematic IDs and a reference identifier (usually a PubMed ID).</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'can-i-download-sequence-whole-chromosome'">
<h3 id="can-i-download-sequence-whole-chromosome">Can I download the sequence for a whole chromosome?</h3>
<!-- pombase_categories: Finding data -->
<p>Yes, sequence and feature annotations data files are available for each chromosome on the <a routerLink="/downloads/genome-datasets">Genome datasets</a> page (FTP download).</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'can-i-download-sequences-many-genes-once-including-flanking-regions'">
<h3 id="can-i-download-sequences-many-genes-once-including-flanking-regions">Can I download sequences for many genes at once, including flanking regions?</h3>
<!-- pombase_categories: Finding data -->
<p>If you only need coding sequence, introns, and/or curated UTRs, use the <a routerLink="/query">advanced search</a> to find the set of genes you want (see the <a routerLink="/documentation/advanced-search">documentation</a> as needed).</p>
<p>Click on a count in the query history to see the results, with a button for &quot;Download&quot; options including coordinates and sequence. In the popup, click on the Sequence tab, select Nucleotide, then tick boxes to add any of: 5' UTR, introns, 3' UTR.</p>
<p>At present, there isn't a good way to retrieve user-specified amounts of flanking sequence for multiple genes in bulk directly from PomBase. We hope to add a more convenient option in the near future, but in the meantime, we recommend using the Ensembl Genomes Biomart query:</p>
<ol style="list-style-type: decimal">
<li>Go to <a href="http://fungi.ensembl.org/biomart/martview/" class="uri">http://fungi.ensembl.org/biomart/martview/</a></li>
<li>Select the database &quot;Ensembl Fungi Genes&quot; from the &quot;CHOOSE DATABASE&quot; drop-down menu.</li>
<li>Select &quot;Schizosaccharomyces pombe genes&quot; from the &quot;CHOOSE DATASET&quot; drop-down menu. Additional options will appear in the left-hand sidebar.</li>
<li>In the left-hand menu, click on the header &quot;Filters&quot;.</li>
<li>Expand the section &quot;GENE&quot; by clicking the + sign</li>
<li>In the drop-down menu in the section &quot;Limit to genes (external references)&quot; select &quot;PomBase Gene ID(s)&quot;. (This will automatically tick the &quot;Limit to genes&quot; box.) In the box underneath, type or paste a list of <em>S. pombe</em> gene names or systematic IDs.</li>
<li>In the left-hand menu, click on the header &quot;Attributes&quot;.</li>
<li>Click the &quot;Sequences&quot; button, and expand the &quot;SEQUENCES&quot; section.</li>
<li>Click a button to select which sequences you want. In the cartoon, red or black highlighting indicates what each option retrieves. Key: |---, 5' flanking region; leftmost box, 5' UTR; inner boxes, coding exons; rightmost box, 3' UTR; ---|, 3' flanking region; ^, introns.</li>
<li>To include flanking regions, tick one or both of the &quot;Upstream flank&quot; and &quot;Downstream flank&quot; boxes, and enter the length you want. (Note: the &quot;flank&quot; options in the button selections retrieve ONLY flanking squence, and will only retrieve 3' or 5' in any given query, not both.)</li>
<li>When you have specified what you want, find the &quot;Results&quot; button in the header and click it. You will be able to view or download the results, or have them emailed to you.</li>
</ol>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'finding-data' || pageName == 'can-i-find-all-unconserved-orphan-genes-fission-yeast'">
<h3 id="can-i-find-all-unconserved-orphan-genes-fission-yeast">Can I find all of the unconserved (orphan) genes in fission yeast?</h3>
<!-- pombase_categories: Genome statistics and lists,Finding data -->
<p>Orphan genes are generally defined as genes without homologs in other organisms. In PomBase, protein-coding genes conserved in the <em>Schizosaccharomyces</em> genus are distinguished from genes conserved only in <em>S. pombe</em>.</p>
<p>To retrieve either set of genes, use the &quot;Taxonomic conservation&quot; query in the <a routerLink="/query">Advanced Search</a>. Choose &quot;Schizosaccharomyces specific&quot; for genes found in more than one <em>Schizosaccharomyces</em> species, or &quot;Schizosaccharomyces pombe specific&quot; for genes found only in <em>S. pombe.</em> See the <a routerLink="/documentation/advanced-search">Advanced Search documentation</a> for help with performing searches.</p>
<p>Historical note: Prior to August 2014, PomBase and its predecessor GeneDB referred to single-copy genes conserved within, but not outside, the <em>Schizosaccharomyces</em> genus as &quot;sequence orphans&quot;. See the <a routerLink="/status/gene-characterisation-statistics-history">Gene characterisation statistics history</a> page for more details (note that the gene characterisation classifications reflect whether a gene has been studied experimentally as well as the extent of its conservation).</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'can-i-find-promoters-pombase'">
<h3 id="can-i-find-promoters-pombase">Can I find promoters in PomBase?</h3>
<!-- pombase_categories: Finding data -->
<p>The PomBase genome browser includes a data track of core promoter locations from Li <em>et al.</em> (2015) Genome-wide analysis of core promoter structures in <em>Schizosaccharomyces pombe</em> with DeepCAGE. RNA Biol. 12:525-37 (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25747261">PMID:25747261</a>). The promoter track is listed under the Transcription Starts Site menu, and is best viewed using the &quot;Labels&quot; track style, which labels promoter features using the associated gene IDs (see the <a routerLink="/faq/how-can-i-show-or-hide-tracks-in-the-genome-browser">browser configuration FAQ</a> for more information).</p>
<p>We will add new tracks for any more promoter data sets that are submitted to us.</p>
<p>There are very few manually curated promoters in PomBase, which are displayed on a genome browser track, &quot;PomBase Annotated Promoter (SO:0000167)&quot;. This track is under the Sequence and assembly menu, and is activated in the default configuration. To search the manually curated promoters, we suggest that you use Artemis. Follow the instructions in <a routerLink="/faq/is-there-an-equivalent-to-the-artemis-java-applet-in-pombase">this FAQ</a>, and search for features with &quot;Key&quot; = &quot;promoter&quot;.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'can-i-find-replication-origins-pombase'">
<h3 id="can-i-find-replication-origins-pombase">Can I find replication origins in PomBase?</h3>
<!-- pombase_categories: Finding data -->
<p>Replication origin coordinates are not yet included in PomBase. We hope to obtain comprehensive origin data in the genome browser in the future, but we rely on user submissions to set priorities for adding data tracks to the browser.</p>
<p>Until replication origins are available in PomBase, we suggest that you use the data collated by Conrad Nieduszynski in <em>S. pombe</em> OriDB:</p>
<ul>
<li>Home page: http://pombe.oridb.org/index.php</li>
<li>Search: http://pombe.oridb.org/search.php?chr=all&amp;confirmed=true&amp;likely=true&amp;dubious=true&amp;name=</li>
</ul>

</div>
<div *ngIf="pageName == 'tools-and-resources' || pageName == 'can-i-generate-comprehensive-restriction-enzyme-map-genome-pombase'">
<h3 id="can-i-generate-comprehensive-restriction-enzyme-map-genome-pombase">Can I generate a comprehensive restriction enzyme map of the genome in PomBase?</h3>
<!-- pombase_categories: Tools and resources -->
<p>No, but this can be done within Artemis.</p>
<p>Install Artemis (available from http://www.sanger.ac.uk/science/tools/artemis).</p>
<p>You can then read in the EMBL format chromosome contig files of sequence and annotation (available from the <a routerLink="/downloads/genome-datasets">Genome datasets</a> page). To generate a restriction map:</p>
<ol style="list-style-type: decimal">
<li>Create a new entry using the &quot;Create&quot; menu item &quot;New Entry&quot;</li>
<li>Toggle off the main annotation by un-checking the chromsome contig file (this will make your new file &quot;no name&quot; the active entry).</li>
<li>Save your new file with your preferred name.</li>
<li>Use the Create menu option &quot;Mark From Pattern&quot; to create features for any restriction patterns of interest and save them into your file.</li>
<li>You can add &quot;color&quot; labels to distinguish the different restriction sites. See the <a routerLink="/faq/is-there-an-equivalent-to-the-artemis-java-applet-in-pombase">Artemis FAQ</a> and the <a href="ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf">Artemis manual</a> (pdf; Sanger site) for additional information.</li>
</ol>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'can-i-get-cdna-sequences-s.-pombe'">
<h3 id="can-i-get-cdna-sequences-s.-pombe">Can I get cDNA sequences for <em>S. pombe</em>?</h3>
<!-- pombase_categories: Finding data -->
<p>A file of cDNA sequences in FASTA format is available on the <a routerLink="/downloads/genome-datasets">Genome datasets</a> page.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'can-i-get-file-specific-set-genome-features'">
<h3 id="can-i-get-file-specific-set-genome-features">Can I get a file with a specific set of genome features?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>The genomic GFF3 file available on the <a routerLink="/downloads/genome-datasets">Genome datasets</a> page includes all gene features, but not LTRs or other repeats. The files in EMBL and GenBank formats contain all annotated sequence features.</p>
<p>Another option for extracting all annotated features (or if you need to specify which feature types to include) is to use the Ensembl API. See the FAQ &quot;<a routerLink="/faq/can-i-access-pombase-via-an-api">Can I access PomBase via an API?</a>&quot; for more information on using the API.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'using-ontologies' || pageName == 'can-i-get-list-essential-pombe-genes'">
<h3 id="can-i-get-list-essential-pombe-genes">Can I get a list of essential pombe genes?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists,Using ontologies -->
<p>If an essential gene is deleted, the cell cannot survive under normal laboratory conditions. A search for deletion alleles annotated to the Fission Yeast Phenotype Ontology term &quot;inviable vegetative cell population&quot; (FYPO:0002061) would therefore identify essential fission yeast genes. Similarly, deletion alleles annotated to &quot;viable vegetative cell population&quot; (FYPO:0002060) represent non-essential genes.</p>
<p><strong>Downloadable summary</strong></p>
<p>A <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Phenotype_annotations/FYPOviability.tsv">set of &quot;viability summary&quot; data</a>, as shown at the top of the FYPO table on each gene page, is available as a downloadable file. The file has two columns: the gene systematic ID and one of three values: &quot;viable&quot;, &quot;inviable&quot; or &quot;condition-dependent&quot;.</p>
<p><strong>Querying</strong></p>
<ul>
<li>To find genes annotated to &quot;inviable vegetative cell population&quot;, select the &quot;phenotype&quot; query and type or paste the ID, FYPO:0002061. Select the &quot;Null&quot; option for &quot;Expresion level&quot; and submit the query. The results include all genes that showed inviable phenotypes in the HTP deletion project as well some manually annotated genes. Do the same for viable (FYPO:0002060).</li>
<li><p>For some deletion mutants, viability depends on experimental conditions, which cannot yet be queried in PomBase. These genes are annotated to both viable (FYPO:0002060) and inviable (FYPO:0002061) at once. To find them, do both searches above, then click the boxes beside both in the query history table and click the &quot;Intersect&quot; button (equivalent to the &quot;AND&quot; operator).</p></li>
<li><p>See the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more information on performing the searches described here.</p></li>
</ul>
<p><strong>A brief note about FYPO terms</strong></p>
<p>At present, there are very few null mutants annotated as inviable in life cycle stages other than vegetative growth, and &quot;inviable vegetative cell population&quot; best fits the most common usage of &quot;essential gene&quot;. If you do want to include other stages (such as &quot;inviable spore&quot;), you can use the very generic term &quot;inviable cell population&quot; (FYPO:0002059) or &quot;viable cell population&quot; (FYPO:0002058) in your query. All of the caveats about alleles and conditions still apply.</p>
<!--
**Query links**

-   <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2219%22,%22query_1%22:%22FYPO:0002061%22,%22query_2%22:%22Null%22%7D%7D,%22filter_count%22:%221%22%7D%5D">Null alleles annotated to "inviable vegetative cell population"     (FYPO:0002061)</a> 
-   <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2219%22,%22query_1%22:%22FYPO:0002060%22,%22query_2%22:%22Null%22%7D%7D,%22filter_count%22:%221%22%7D%5D">Null alleles annotated to "viable vegetative cell population"     (FYPO:0002060)</a> 
-   <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2219%22,%22query_1%22:%22FYPO:0002061%22,%22query_2%22:%22Null%22%7D,%22filter_2%22:%7B%22operator%22:%22AND%22,%22filter%22:%2219%22,%22query_1%22:%22FYPO:0002060%22,%22query_2%22:%22Null%22%7D%7D,%22filter_count%22:%222%22%7D%5D">Null alleles annotated to both FYPO:0002061 and     FYPO:0002060</a> 
-->

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'can-i-get-list-systematic-ids-primary-names-synonyms-and-gene-products-s.-pombe'">
<h3 id="can-i-get-list-systematic-ids-primary-names-synonyms-and-gene-products-s.-pombe">Can I get a list of systematic IDs, primary names, synonyms and gene products in <em>S. pombe</em>?</h3>
<!-- pombase_categories: Genome statistics and Lists -->
<p>Yes: From the <a routerLink="/downloads/data-mappings">Data Mapping</a> page, retrieve this file: ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/sysID2product.tsv</p>

</div>
<div *ngIf="pageName == 'gene-page' || pageName == 'can-i-hide-quick-links-box'">
<h3 id="can-i-hide-quick-links-box">Can I hide the Quick Links box?</h3>
<!-- pombase_categories: Gene Page -->
<p>Yes, the Quick Links menu can be expanded and collapsed by clicking the &quot;Quick Links&quot; text in its header.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'can-i-obtain-dump-all-curated-data-pombase'">
<h3 id="can-i-obtain-dump-all-curated-data-pombase">Can I obtain a dump of all curated data in PomBase?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>Data that appear on gene pages -- sequence feature annotations, ontology annotations, etc. -- are stored in a database that uses the <a href="http://gmod.org/wiki/Chado">Chado</a> schema. Monthly snapshots of the PomBase Chado database are available via the <a routerLink="/downloads">Downloads</a> page.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'can-i-provide-list-genes-search-on'">
<h3 id="can-i-provide-list-genes-search-on">Can I provide a list of genes to search on?</h3>
<!-- pombase_categories: Finding data -->
<p>Yes, you can type or paste systematic IDs into the &quot;Gene filter&quot; query in the <a routerLink="/query">advanced search</a>. IDs can be separated by spaces, commas, or newlines. Click &quot;Lookup&quot; to retrieve the results. At present you can use the list in the query history to combine with other queries, or download a fixed set of details (systematic ID, gene name (if any), feature type, coordinates, and strand). We plan to offer more flexible options in the near future, and to allow you to upload a file with your gene list.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'can-i-retrieve-annotations-metabolic-databases-such-as-ec-metacyc-reactome-or-rhea-s.-pombe-genes'">
<h3 id="can-i-retrieve-annotations-metabolic-databases-such-as-ec-metacyc-reactome-or-rhea-s.-pombe-genes">Can I retrieve annotations to metabolic databases such as EC, MetaCyc, Reactome, or Rhea for <em>S. pombe</em> genes?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>The best way to find metabolism-related annotations for <em>S. pombe</em> genes is to use the GO annotation data available from PomBase in combination with mappings between GO terms and entries in the various metabolism-oriented databases.</p>
<p>For example, many GO molecular function (MF) terms representing enzymatic activities are mapped to the corresponding <a href="http://www.chem.qmul.ac.uk/iubmb/enzyme/">Enzyme Commission</a> (EC) number for the reaction, and some are also mapped to entries from <a href="http://www.genome.jp/kegg/">KEGG</a> or from the <a href="http://www.rhea-db.org/home">Rhea</a> database of annotated chemical reactions. GO MF and biological process (BP) terms may be annotated to reactions or pathways, respectively, in <a href="http://metacyc.org/">MetaCyc</a> or <a href="http://www.reactome.org/">Reactome</a>.</p>
<p>A complete list, with descriptions and links, is available on the <a href="http://geneontology.org/page/download-mappings">GO Consortium's Download Mappings</a> page.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'can-i-retrieve-functional-annotations-genes-list'">
<h3 id="can-i-retrieve-functional-annotations-genes-list">Can I retrieve functional annotations for genes in a list?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>You can find the GO annotations for your genes corresponding to functional roles and localizations. Our recommended approach depends on how many specific topics you are interested in:</p>
<ul>
<li><p>For a small number of specific GO terms (e.g. localization to the nucleus or cytoplasm, or a role in signaling or DNA metabolism), you can use the <a routerLink="/query">advanced search</a>. Paste your gene list into the &quot;Gene IDs&quot; query, and then combine it with a GO query for each term of interest (see the <a routerLink="/documentation/advanced-search">search documentation</a> for more information).</p></li>
<li><p>If you are interested in many GO terms, or if you do not know in advance which terms may be relevant, we recommend that you use a &quot;GO term enrichment&quot; tool. Such tools are typically used to find terms overrepresented for a gene list, but can be used to retrieve all GO annotations if the p-value threshold is set artificially high.</p></li>
</ul>
<p>Both the Advanced Search and term enrichment tools take advantage of the hierarchical structure of GO, such that annotations to specific terms are propagated to &quot;ancestor&quot; terms via <em>is_a</em> and <em>part_of</em> relations. See the <a routerLink="/documentation/gene-page-gene-ontology">PomBase GO documentation</a>, and the GO Consortium documentation linked there, for more information. (These approaches also make it easier to maintain and update your data than storing individual GO annotations locally.)</p>
<p>Also see the <a routerLink="/faq/how-can-i-find-significant-shared-go-annotations-for-genes-in-a-list">FAQ on GO term enrichment</a> and the <a routerLink="/browse-curation/fission-yeast-go-slim-terms">PomBase GO Slim</a> page.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'can-i-retrieve-sequence-any-cosmids-used-build-s.-pombe-genome-sequence'">
<h3 id="can-i-retrieve-sequence-any-cosmids-used-build-s.-pombe-genome-sequence">Can I retrieve the sequence for any of the cosmids used to build the <em>S. pombe</em> genome sequence?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>Although old cosmid sequences used in the reference assembly are not available in PomBase directly, they are all stored in the International Nucleotide Sequence Database Collaboration database (ENA, GenBank, DDBJ) archives. For ease of searching, PomBase curators recommend finding the accession, e.g. AL137130, for a cosmid, and using GenBank to retrieve the sequence:</p>
<p>Go to http://www.ncbi.nlm.nih.gov/nucleotide/ (or choose &quot;Nucleotide&quot; in the search pull-down menu on any NCBI search page). Enter the accession. The resulting page will inform you that the sequence has been replaced by one of the whole-chromosome entries, but offers links to both the current chromosome entry and the obsolete contig entry.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'can-i-search-gene-list-and-retrieve-results-same-order-as-input-list'">
<h3 id="can-i-search-gene-list-and-retrieve-results-same-order-as-input-list">Can I search for a gene list and retrieve results in the same order as in the input list?</h3>
<!-- pombase_categories: Finding data -->
<p>At present this is not possible; all advanced search results are ordered alphabetically by gene name, or by systematic ID for unnamed genes, by default. Click the &quot;Product&quot; column header to sort alphabetically by product description. We plan to add an option to preserve input order soon.</p>
<p>In the meantime, we suggest that you download the results using the &quot;Details&quot; link, and import them into a spreadsheet. You can then combine the results with other data in the spreadsheet, and use the spreadsheet software to sort on any column.</p>

</div>
<div *ngIf="pageName == 'orthology' || pageName == 'finding-data' || pageName == 'can-i-search-genes-based-on-conservation-different-taxa'">
<h3 id="can-i-search-genes-based-on-conservation-different-taxa">Can I search for genes based on conservation in different taxa?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p>Yes, use the &quot;Taxonomic conservation&quot; query in the <a routerLink="/query">advanced search</a>.</p>
<!--
Example query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2226%22,%22query%22:%22PBO:0011070%22%7D%7D,%22filter_count%22:%221%22%7D%5D">Genes conserved in vertebrates</a> 
-->

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'can-i-search-pombase-protein-features'">
<h3 id="can-i-search-pombase-protein-features">Can I search PomBase for protein features?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>A selection of protein sequence motifs and features have been manually curated using terms from the <a href="http://sequenceontology.org/">Sequence Ontology</a> (SO). For example, <a routerLink="/gene/SPAC15A10.03c">Rad54</a> has a KEN box (a motif recognized by the anaphase-promoting complex; SO:0001807), and <a routerLink="/gene/SPAC31A2.11c">Cuf1</a> and <a routerLink="/gene/SPAC1D4.10">Trz1</a> have nuclear localization signals (NLS; SO:0001528). These annotations are included in the Protein Features section of the gene page.</p>
<p>To search for these features, use the &quot;Protein feature&quot; query in the <a routerLink="/query">advanced search</a> (enter a SO ID or description; see the <a routerLink="/documentation/advanced-search">documentation</a> for help with searching).</p>
<p>Also see the FAQs on <a routerLink="/faq/how-can-i-find-proteins-that-have-transmembrane-domains">transmembrane domains</a> and <a routerLink="/faq/how-can-i-find-all-s-pombe-proteins-in-a-particular-protein-family">protein families</a>, and the section of the search documentation on protein feature and protein domain queries.</p>
<!--
Example query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2229%22,%22query%22:%22SO:0001528%22%7D%7D,%22filter_count%22:%221%22%7D%5D">nuclear localization signal (SO:0001528)</a> 
-->

</div>
<div *ngIf="pageName == 'tools-and-resources' || pageName == 'can-i-use-blast-search-short-sequences'">
<h3 id="can-i-use-blast-search-short-sequences">Can I use BLAST to search for short sequences?</h3>
<!-- pombase_categories: Tools and resources -->
<p>Yes, you can search for short nucleotide sequences, such We recommend NCBI BLAST, where you can search for short nucleotide sequences, such as primers or other oligomers (PomBase does not maintain its own BLAST server).</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'can-i-use-wild-card-beginning-search-string'">
<h3 id="can-i-use-wild-card-beginning-search-string">Can I use a wild card at the beginning of a search string?</h3>
<!-- pombase_categories: Finding data -->
<p>The simple search finds search strings anywhere in IDs, gene names, and product descriptions. It does not require any wildcard character, and shows autocomplete suggestions as you type.</p>

</div>
<div *ngIf="pageName == 'genome-browser' || pageName == 'can-i-view-my-data-privately-genome-browser'">
<h3 id="can-i-view-my-data-privately-genome-browser">Can I view my data privately in the genome browser?</h3>
<!-- pombase_categories: Genome browser -->
<p>Yes, if you have locally stored data that you want to see in the context of the genome browser, you can add it as a custom track. The <a href="http://genomebrowser.pombase.org/info/website/upload/index.html">Ensembl documentation on adding custom tracks</a> describes supported file formats, options for uploading or linking your data, and instructions on using the web interface to configure your tracks.</p>

</div>
<div *ngIf="pageName == 'orthology' || pageName == 'tools-and-resources' || pageName == 'can-i-view-my-protein-interest-multiple-alignment-related-proteins-other-species'">
<h3 id="can-i-view-my-protein-interest-multiple-alignment-related-proteins-other-species">Can I view my protein of interest in a multiple alignment with related proteins in other species?</h3>
<!-- pombase_categories: Orthology,Tools and resources -->
<p>You can use Compara via the genome browser to see multiple alignments:</p>
<ol style="list-style-type: decimal">
<li>On any gene page, go to the Orthologs section (scroll or use the Quick Links box).</li>
<li>Follow the relevant link to Compara - for fungal alignments, choose &quot;View orthologs in other fungal species with Compara&quot;, or for all eukaryotic species choose &quot;View orthologs across taxonomic space using pan-species Compara&quot;.</li>
<li>You should see a &quot;collapsed&quot; gene tree highlighting your fission yeast gene of interest. From here you can click on any node to see a menu of options:
<ol style="list-style-type: decimal">
<li>Expand or collapes specific sub-nodes of the tree, or expand the tree fully</li>
<li>View the alignment in FASTA format</li>
<li>Launch the jalview multiple alignment viewer to see the full alignment and colour by residue conservation, hydrophobicity, etc.</li>
</ol></li>
</ol>
<p>To configure the protein entries visible in the alignment, select the most &quot;inclusive&quot; node you require. You can reduce the number of entries by collapsing individual sub-trees (step 4) before you generate your alignment.</p>
<p>Information about how the Compara trees are generated, homology types, and species are available here: <a href="http://fungi.ensembl.org/info/genome/compara/homology_method.html" class="uri">http://fungi.ensembl.org/info/genome/compara/homology_method.html</a></p>

</div>
<div *ngIf="pageName == 'genome-browser' || pageName == 'tools-and-resources' || pageName == 'can-i-view-nucleotide-sequence-genome-browser'">
<h3 id="can-i-view-nucleotide-sequence-genome-browser">Can I view nucleotide sequence in the genome browser?</h3>
<!-- pombase_categories: Genome browser,Tools and resources -->
<p>Yes. First, make sure sequence display is &quot;on&quot;:</p>
<ol style="list-style-type: decimal">
<li>From the Location tab click on the &quot;Configure this Page&quot;</li>
<li>In the left hand menu of the pop-up, click on &quot;Sequence and assembly&quot;</li>
<li>If the box next to &quot;Sequence&quot; is blank, click it</li>
</ol>
<p>(also see the <a routerLink="/faq/how-can-i-show-or-hide-tracks-in-the-genome-browser">FAQ on configuring tracks</a>).</p>
<p>The Location tab shows two graphics. Between the two images there is a slider that controls the zoom. Click the '+' or drag the vertical bar to the left to zoom in. The lower graphic will first display colored blocks representing color-coded nucleotides, and then, at maximum zoom, legible sequence. (Note that the vertical bar may appear to be all the way to the left of the slider before you actually zoom in enough to read the sequence; keep clicking '+' if necessary.)</p>
<p>At any zoom level, use the arrows flanking the zoom slider to scroll along the sequence.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'genome-browser' || pageName == 'can-i-view-variation-data-pombase'">
<h3 id="can-i-view-variation-data-pombase">Can I view variation data in PomBase?</h3>
<!-- pombase_categories: Finding data,Genome browser -->
<p>The genome browser includes variation data from natural <em>S. pombe</em> isolates, published in:</p>
<p>Jeffares DC <em>et al.</em> 2015. The genomic and phenotypic diversity of Schizosaccharomyces pombe. <em>Nat Genet.</em> <strong>47(3):</strong>235-241. doi: <a href="http://dx.doi.org/10.1038/ng.3215">10.1038/ng.3215</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25665008">PMID:25665008</a></p>
<p>To view the variation data, enable one or more of the tracks under &quot;Variation&quot;. <a routerLink="/faq/how-can-i-show-or-hide-tracks-in-the-genome-browser">Help</a> is available for enabling tracks.</p>
<p>Insertions/deletions (indels) and SNPs can be enabled as separate tracks, or &quot;Sequence variants (all sources)&quot; displays both types on a single track. Clicking on any variation feature in the track brings up a small pop-up box with a summary and link to further details about the variation.</p>
<p>A Variscan track showing variation diversity along the sequence is also available.</p>
<p>Ensembl provides a <a href="http://test.genomebrowser.pombase.org/info/website/tutorials/Ensembl_variation_quick_reference_card.pdf">quick reference card</a> with more information on the browser interface for variations.</p>

</div>
<div *ngIf="pageName == 'tools-and-resources' || pageName == 'can-i-visualise-networks-pombase'">
<h3 id="can-i-visualise-networks-pombase">Can I visualise networks in PomBase?</h3>
<!-- pombase_categories: Tools and resources -->
<p>This is an area of active development at PomBase. At present, PomBase links GO slim terms to the web-based network tool <a href="http://www.esyn.org/">esyN</a> <sup>1</sup>: on the <a routerLink="/browse-curation/fission-yeast-go-slim-terms">GO Slim page</a> and on ontology term pages for GO Slim biological process terms, each GO Slim term links to the <a routerLink="/documentation/high-confidence-physical-interaction-network">HCPIN physical interaction network</a> in esyN. For example, the GO Slim page and the ontology term page for &quot;regulation of mitotic cell cycle phase transition&quot; (<a routerLink="/term/GO:1901990">GO:1901990</a>) link to http://www.esyn.org//builder.php?type=Graph&amp;term=GO:1901990&amp;interactionType=physical&amp;source=pombase&amp;includeInteractors=false .</p>
<p>We plan to provide gene-centred network displays on gene pages in the near future.</p>
<!--
On gene pages, we have links to gene-specific interaction networks in
esyN in the table headers of the Interactions sections:

-   The Genetic Interactions section links to all interactions centred
    on the gene and curated in BioGRID
-   The Physical interactions section has links to two datasets:
    -   All physical interactions curated in BioGRID for the gene
    -   Interactions for the gene in the PomBase <a routerLink="/documentation/high-confidence-physical-interaction-network">High Confidence         Physical Interaction Network         (HCPIN)</a> 

For example, the Genetic Interactions header for cdc2 links to
<http://www.esyn.org/builder.php?type=Graph&query=SPBC11B10.09&organism=4896&interactionType=genetic&source=biogrid>

-->
<p>Using the esyN network display:</p>
<ul>
<li>A brief simulation is used to position the nodes initially. This layout can often be improved by continuing the simulation: click &quot;Layouts&quot; in the left-hand panel, then click &quot;Force-Directed (improve)&quot;. Repeat until you like the network arrangement.</li>
<li>You can extend any network in esyN:</li>
</ul>
<ol style="list-style-type: decimal">
<li>Click on a node in the display.</li>
<li>Click &quot;Extend Network&quot; in the right-hand bar.</li>
<li>In the Advanced Tools box under the network display, first click &quot;Get Interactions&quot;.</li>
<li>In the table that appears, click &quot;Add&quot; buttons to add individual interactions, or click &quot;Add all&quot;.</li>
</ol>
<p>You can also use EsyN to:</p>
<p>Visualize interactions for a user-defined list of genes. To do this, visit http://www.esyn.org/builder.php?type=Graph, click on the &quot;Network from list&quot; option in the left-hand panel, and follow the instructions in the pop-up.</p>
<p>Build your own network -- either an Interactome graph or a Petri net -- from scratch (see the tutorial at http://www.esyn.org/tutorial.html). In both cases you can use the Advanced Tools to retrieve the interactions for a number of model organisms from several databases (see http://www.esyn.org/builder.php?type=Graph#interactions).</p>
<p>Save and share your networks. By logging in via the &quot;My esyN&quot; link (at the top of every esyN page), any user can save, share privately with collaborators, or publish any network.</p>
<p>Browse, view and modify, previously published, models (both graphs and Petri nets) at http://www.esyn.org/browse.php. We describe these networks are &quot;public&quot; in the sense of the open source movement, so that they are not only free to be copied, modified and (when possible) re-published, but we also actively encourage any collaborative effort to build and improve these biological networks.</p>
<p><sup>1</sup>esyN reference: Bean DM, Heimbach J, Ficorella L, Micklem G, Oliver SG, Favrin G. 2014. esyN: network building, sharing and publishing. <em>PLoS One</em>. 2014 Sep 2; <strong>9(9)</strong>:e106035. doi: <a href="http://dx.doi.org/10.1371/journal.pone.0106035">10.1371/journal.pone.0106035</a>. eCollection 2014. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25181461">PMID:25181461</a></p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'do-you-have-polyadenylation-data-fission-yeast'">
<h3 id="do-you-have-polyadenylation-data-fission-yeast">Do you have polyadenylation data for fission yeast?</h3>
<!-- pombase_categories: Finding data -->
<p>PomBase offers two ways to find polyadenylation sites and usage:</p>
<p>Fission yeast polyadenylation data are available in the genome browser. To display the data:</p>
<ol style="list-style-type: decimal">
<li>From any gene page, click the &quot;View in Genome Browser&quot; link.</li>
<li>Go to &quot;Configure this page&quot; option in the left-hand menu.</li>
<li>Select &quot;Polyadenylation sites&quot; in the left-hand menu of the popup window. Select the tracks you want to show, then click the &quot;tick&quot; in the corner.</li>
</ol>
<p><a routerLink="/faq/how-can-i-show-or-hide-tracks-in-the-genome-browser">Further help with configuring browser tracks</a> is available.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'how-are-non-coding-rna-genes-identified'">
<h3 id="how-are-non-coding-rna-genes-identified">How are non-coding RNA genes identified?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>Most non-coding RNAs in PomBase are based on transcriptome data, either from Jürg Bähler's lab (Solexa/deep sequencing; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>) or Nick Rhind's lab (RNA sequencing; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>). For any ncRNA, the source should be linked as a publication in the &quot;Literature&quot; section at the bottom of the PomBase gene page. To get an idea of the transcription in a region, you can look at the Bähler Lab Transcriptome Viewer, which is linked from most gene pages, e.g. <a href="http://www.bahlerlab.info/cgi-bin/pombetv/pombetv?condition=13&genename=SPNCRNA.200&action=display">SPNCRNA.200</a>. Unfortunately some genes, such as SPNCRNA.1115, post-date the viewer and therefore do not have entries, but you can look at transcription in this region by accessing a neighboring gene.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'how-are-pombase-systematic-ids-determined'">
<h3 id="how-are-pombase-systematic-ids-determined">How are PomBase systematic IDs determined?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>Systematic IDs follow patterns based on the feature type, and in some cases the chromosome, as shown in the table below.</p>
<p>Open reading frame (ORF) IDs also indicate which cosmid or plasmid they were found on in genome sequencing. In most cases, ORF IDs that end with a digit indicate that the ORF is on the forward (Watson) strand, and an ORF with an ID that ends with 'c' is on the reverse (Crick) stand. There are a few exceptions, however, because some cosmids were moved and their orientation reversed late in the sequence assembly procedure</p>
<p>IDs with '.1' appended are transcript IDs; the dot-and-digit IDs follow Ensembl's standard. At present, PomBase has only one transcript annotated for any given feature, but in the future when alternative transcripts are annotated the digit will be incremented (.2, .3, etc.).</p>
<p><strong>Systematic ID patterns</strong></p>
<table>
<thead>
<tr class="header">
<th>ID pattern</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>SPAC*</td>
<td>features, usually ORFs, on chromosome 1, sequenced on cosmids</td>
</tr>
<tr class="even">
<td>SPBC*</td>
<td>features, usually ORFs, on chromosome 2, sequenced on cosmids</td>
</tr>
<tr class="odd">
<td>SPCC*</td>
<td>features, usually ORFs, on chromosome 3, sequenced on cosmids</td>
</tr>
<tr class="even">
<td>SPAP*</td>
<td>features, usually ORFs, on chromosome 1, sequenced on plasmids</td>
</tr>
<tr class="odd">
<td>SPBP*</td>
<td>features, usually ORFs, on chromosome 2, sequenced on plasmids</td>
</tr>
<tr class="even">
<td>SBCP*</td>
<td>features, usually ORFs, on chromosome 3, sequenced on plasmids</td>
</tr>
<tr class="odd">
<td>SPATRNA*</td>
<td>tRNA genes on chromosome 1</td>
</tr>
<tr class="even">
<td>SPBTRNA*</td>
<td>tRNA genes on chromosome 2</td>
</tr>
<tr class="odd">
<td>SPCTRNA*</td>
<td>tRNA genes on chromosome 3</td>
</tr>
<tr class="even">
<td>SPLTRA*</td>
<td>LTRs on chromosome 1</td>
</tr>
<tr class="odd">
<td>SPLTRB*</td>
<td>LTRs on chromosome 2</td>
</tr>
<tr class="even">
<td>SPLTRC*</td>
<td>LTRs on chromosome 3</td>
</tr>
<tr class="odd">
<td>SPNCRNA*</td>
<td>non-coding RNA genes (no chromosome info in ID)</td>
</tr>
<tr class="even">
<td>SPRPTA.*</td>
<td>repeats (other than LTRs or centromeric repeats) on chromosome 1</td>
</tr>
<tr class="odd">
<td>SPRPTB.*</td>
<td>repeats (other than LTRs or centromeric repeats) on chromosome 2</td>
</tr>
<tr class="even">
<td>SPRPTC.*</td>
<td>repeats (other than LTRs or centromeric repeats) on chromosome 3</td>
</tr>
<tr class="odd">
<td>SPRPTCENA*</td>
<td>centromeric repeats on chromosome 1</td>
</tr>
<tr class="even">
<td>SPRPTCENB*</td>
<td>centromeric repeats on chromosome 2</td>
</tr>
<tr class="odd">
<td>SPRPTCENC*</td>
<td>centromeric repeats on chromosome 3</td>
</tr>
<tr class="even">
<td>SPRRNA*</td>
<td>rRNA genes (no chromosome info in ID)</td>
</tr>
<tr class="odd">
<td>SPSNORNA*</td>
<td>snoRNA genes (no chromosome info in ID)</td>
</tr>
<tr class="even">
<td>SPSNRNA*</td>
<td>snRNA genes (no chromosome info in ID)</td>
</tr>
<tr class="odd">
<td>SPTF*</td>
<td>transposons (no chromosome info in ID)</td>
</tr>
<tr class="even">
<td>SPMTR*</td>
<td>features on the separately sequenced mating type region contig</td>
</tr>
<tr class="odd">
<td>SPMIT*</td>
<td>features on the mitochondrial chromosome</td>
</tr>
<tr class="even">
<td>SPMITTRNA*</td>
<td>subset of SPMIT*; tRNA genes on mitochondrial chromosome</td>
</tr>
<tr class="odd">
<td>SPNUMT*</td>
<td>NUMTs (nuclear mitochondrial pseudogenes) (no chromosome info in ID)</td>
</tr>
</tbody>
</table>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'how-are-sequence-feature-types-defined-pombase'">
<h3 id="how-are-sequence-feature-types-defined-pombase">How are sequence feature types defined in PomBase?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists -->
<p>S. pombe genome features were originally annotated using <a href="http://www.sanger.ac.uk/science/tools/artemis">Artemis</a>. As noted in the manual (ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf - see p. 9), Artemis draws from a list of feature keys that is documented at EBI: ftp://ftp.ebi.ac.uk/pub/databases/embl/doc/FT_current.html#7.2<br />
In the genome sequence data files, features are defined using <a href="http://sequenceontology.org/">Sequence Ontology</a> terms. Gene pages use a selection of human-friendly text descriptions for feature types.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'how-can-i-browse-phenotype-ontology-fypo'">
<h3 id="how-can-i-browse-phenotype-ontology-fypo">How can I browse the phenotype ontology (FYPO)?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>In the future, we plan to make Fission Yeast Phenotype Ontology (FYPO) terms and annotations available in a browser analogous to AmiGO or QuickGO. Until such a browser becomes available, FYPO is accessible in these external resources:</p>
<p>NCBO BioPortal - search on the <a href="http://bioportal.bioontology.org/">BioPortal home page</a>, go to the <a href="http://bioportal.bioontology.org/ontologies/FYPO">FYPO summary page</a>, or go to the <a href="http://bioportal.bioontology.org/ontologies/FYPO/?p=classes&conceptid=root">FYPO terms page</a>. For assistance, see the &quot;User Interface&quot; part of the <a href="http://bioportal.bioontology.org/help">BioPortal Help</a>.</p>
<p>EBI's Ontology Lookup Service (OLS) - search on the <a href="http://www.ebi.ac.uk/ontology-lookup/">OLS home page</a> or go to the <a href="http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=FYPO">FYPO page</a>. Help is provided on each page.</p>

</div>
<div *ngIf="pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-display-sequence-region-using-sequence-coordinates-genome-browser'">
<h3 id="how-can-i-display-sequence-region-using-sequence-coordinates-genome-browser">How can I display a sequence region using sequence coordinates in the genome browser?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>Go to the <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Info/Index">genome browser</a>, and enter coordinates in the 'Search for:' box. The format is 'I:100000..200000' or 'I:100000-200000' (i.e. use Roman numerals to specify the chromosome, and don't include the word &quot;chromosome&quot;; use either '..' or '-' between the start and end coordinates.)</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-all-genes-have-given-mutant-phenotype'">
<h3 id="how-can-i-find-all-genes-have-given-mutant-phenotype">How can I find all of the genes that have a given mutant phenotype?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>You can search for genes annotated to a Fission Yeast Phenotype Ontology term in the <a routerLink="/query">advanced search</a>. In the &quot;Phenotype&quot; query, if you know the ID (for example, &quot;inviable cell&quot; is FYPO:0000049, and &quot;elongated cell&quot; is FYPO:0000017) you can type or paste the ID into the box. Otherwise, start typing a term name or description; the autocomplete feature will suggest phenotypes. Choose one to retrieve annotated genes.</p>
<p>Note that the FYPO search retrieves annotations by following the <em>is_a</em>, <em>part_of</em>, <em>output_of</em>, <em>has_output</em>, and <em>has_part</em> relationships in the ontology. For example, FYPO includes the relation &quot;inviable swollen elongated cell with enlarged nucleus&quot; (FYPO:0002083) <em>has_part</em> &quot;swollen cell&quot; (FYPO:0000025). Genes annotated to FYPO:0002083 will therefore be retrieved in a search for FYPO:0000025. See the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more information.</p>
<!--
Example query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2219%22,%22query_1%22:%22FYPO:0000017%22,%22query_2%22:%22all_alleles%22%7D%7D,%22filter_count%22:%221%22%7D%5D">Genes annotated to "elongated cell" (FYPO:0000017), all alleles</a> 

-->
<p>Also see the FAQ on <a routerLink="/faq/can-i-get-a-list-of-essential-pombe-genes">finding essential genes</a>.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'orthology' || pageName == 'how-can-i-find-all-s.-pombe-genes-are-conserved-human'">
<h3 id="how-can-i-find-all-s.-pombe-genes-are-conserved-human">How can I find all <em>S. pombe</em> genes that are conserved in human?</h3>
<!-- pombase_categories: Genome statistics and lists,Orthology -->
<p>Almost all genes that are conserved between fission yeast and human are also conserved in other vertebrates (there are two exceptions, both of which are genes encoding amino acid biosynthesis proteins that have become pseudogenes in human). To retreive these genes, go to the <a routerLink="/query">advanced search</a>, click the &quot;Taxonomic conservation&quot; query option, and choose the description &quot;Conserved in vertebrates&quot;.</p>
<p>Also see the FAQs on <a routerLink="/faq/how-can-i-find-s-pombe-genes-associated-with-human-disease">finding disease gene orthologs</a>, <a routerLink="/faq/how-can-i-find-s-pombe-ortholog-s-of-a-human-gene">finding the ortholog of a specific gene</a>, and on <a routerLink="/faq/how-can-i-obtain-the-list-of-human-and-s-pombe-orthologs">downloading the full set of curated orthologs</a>.</p>
<!--
Query link: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2226%22,%22query%22:%22PBO:0011070%22%7D%7D,%22filter_count%22:%221%22%7D%5D">Genes conserved in vertebrates</a> 
-->

</div>
<div *ngIf="pageName == 'orthology' || pageName == 'finding-data' || pageName == 'how-can-i-find-all-s.-pombe-proteins-particular-protein-family'">
<h3 id="how-can-i-find-all-s.-pombe-proteins-particular-protein-family">How can I find all <em>S. pombe</em> proteins in a particular protein family?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p>There are various ways you can find protein family members.</p>
<ol style="list-style-type: decimal">
<li>If you know the Pfam, PRINTs, PROSITE, or InterPro accession for the family or domain you want, you can use the <a routerLink="/query">advanced search</a>). Click &quot;Protein domain&quot; and then enter the accession.</li>
<li>If you don't have an accession, but do know any member of the family, go directly to its gene page. In the &quot;Protein Features&quot; section of the gene page there is a table of protein domains and families, which includes a link to a list of all family members in S. pombe.</li>
<li>If you know neither accessions nor family members, you can search for keywords in the <a href="http://www.ebi.ac.uk/interpro/">InterPro</a> database, which combines signatures from a number of member databases, including Pfam. Record the accession number(s) of the family, and use them in the PomBase advanced search as described in item 1 above. (If necessary, you can use the query history to combine the results of several queries.)</li>
</ol>
<!--
Example query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%224%22,%22query_1%22:%22pfam_pf%22,%22query_2%22:%22PF00004%22%7D%7D,%22filter_count%22:%221%22%7D%5D">Proteins matching "ATPase, AAA-type, core" (Pfam:PF00004)</a> 
-->

</div>
<div *ngIf="pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-find-all-sequence-features-region-using-chromosome-coordinates'">
<h3 id="how-can-i-find-all-sequence-features-region-using-chromosome-coordinates">How can I find all sequence features in a region using chromosome coordinates?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>You can do this in the genome browser (from a gene page or the Tools menu). First, <a routerLink="/faq/how-can-i-display-a-sequence-region-using-sequence-coordinates-in-the-genome-browser">enter the coordinates</a>, then click the Export Data button on the left-hand side. In the Output pulldown (topmost in the popup window) choose one of the formats under the &quot;Feature File&quot; header. Then follow the remaining steps to <a routerLink="/faq/how-can-i-retrieve-the-sequence-of-a-region-using-sequence-coordinates">retrieve the sequence features</a> -- add flanking sequences, select options for your selected output format, etc.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'how-can-i-find-genes-region-using-chromosome-coordinates'">
<h3 id="how-can-i-find-genes-region-using-chromosome-coordinates">How can I find genes in a region using chromosome coordinates?</h3>
<!-- pombase_categories: Finding data -->
<!-- 
* denotes things to confirm in actual interface when this works:
In the <a routerLink="/query">advanced search</a>, click *"more queries" to see
the "Genes between coordinates" query option. Choose a chromosome from
the *pulldown, and enter start and end coordinates in the *text boxes.
-->
<p>The <a routerLink="/query">advanced search</a> query to retrieve genes in a specified genomic region is temporarily unavailable.</p>
<p>In the meantime, we suggest that you use the Genome Browser to find all features (not only genes), <a routerLink="/faq/how-can-i-retrieve-the-sequence-of-a-region-using-sequence-coordinates">as described here</a>.</p>
<!--
Example query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2210%22,%22query%22:%22II%22%7D,%22filter_2%22:%7B%22operator%22:%22AND%22,%22filter%22:%2215%22,%22query_1%22:%221000000%22,%22query_2%22:%222000000%22%7D%7D,%22filter_count%22:%222%22%7D%5D">Genes between coordinates 1000000-2000000 on chromosome 2</a> 
-->

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-genes-specific-activity'">
<h3 id="how-can-i-find-genes-specific-activity">How can I find genes with a specific activity?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>PomBase uses Gene Ontology (GO) molecular function terms to capture the activities -- including enzymatic activities, binding, transporters, etc. -- of gene products. You can therefore use the GO term query in the <a routerLink="/query">advanced search</a> to retrieve genes whose products have a given activity.</p>
<p>In the &quot;GO&quot; query, if you know the ID (for example, &quot;histone acetyltransferase activity&quot; is GO:0004402, and &quot;calcium ion transmembrane transporter activity&quot; is GO:0015085), type or paste it into the box. Otherwise, and start typing a name or description; the autocomplete feature will suggest terms. Choose one to retrieve annotated genes. You can try using more specific or less specific terms to retrieve the results that best fit your expectations and needs. See the <a routerLink="/documentation/advaced-search">advanced search documentation</a> and the <a routerLink="/documentation/gene-page-gene-ontology">Gene Page GO documentation</a> for more information, including how [ontology searches retrieve annotations] to general terms.</p>
<!--
Example query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%221%22,%22query%22:%22GO:0004721%22%7D%7D,%22filter_count%22:%221%22%7D%5D">phosphoprotein phosphatase activity (GO:0004721)</a> 
-->

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-go-annotations-other-schizosaccharomyces-species-e.g.-s.-japonicus'">
<h3 id="how-can-i-find-go-annotations-other-schizosaccharomyces-species-e.g.-s.-japonicus">How can I find GO annotations for other <em>Schizosaccharomyces</em> species (e.g. <em>S. japonicus</em>)?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>For the sequenced strains of <em>S. japonicus</em>, <em>S. octosporus</em> and <em>S. cryophilus</em>, the Ensembl group has generated GO annotation data sets for protein-coding genes by transferring experiment-based annotations from <em>S. pombe</em> orthologs. You can use the <a href="http://www.ebi.ac.uk/QuickGO/">QuickGO</a> browser to retrieve the data for each species -- follow the &quot;Search and Filter GO annotation sets&quot; link, then click &quot;Filter&quot; to set a taxon filter for the taxon ID:</p>
<ul>
<li><em>S. japonicus</em> (strain yFS275): 402676</li>
<li><em>S. octosporus</em> (strain yFS286): 483514</li>
<li><em>S. cryophilus</em> (strain OY26): 653667</li>
</ul>
<p>Because these automated annotations are inferred only from experimentally-derived <em>S. pombe</em> annotations, coverage will not be complete.</p>
<p>Note that the GAF downloaded from QuickGO uses UniProtKB accessions in the gene product ID column (column 2). To use the GAF in any further analysis, such as term enrichment, you will have to convert the accessions to systematic IDs. See the <a routerLink="/faq/can-i-convert-ids-from-other-databases-to-or-from-pombase-ids">FAQ on ID mapping</a> for hints.</p>
<p>One feasible approach to improve annotation coverage is to download the <em>S. pombe</em> GO annotations (see the <a routerLink="/downloads/go-associations">GO Associations download</a></p>
<p>page), and then substitute the <em>S. pombe</em> IDs with the IDs of orthologous genes from the other <em>Schizosaccharomyces</em> species of interest. For ortholog IDs, see the <a routerLink="/faq/how-can-i-find-orthologs-between-s-pombe-and-other-schizosaccharomyces-species">FAQ on <em>Schizosaccharomyces</em> orthologs</a>, and use the indicated table from Rhind <em>et al.</em> Comparative functional genomics of the fission yeasts (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>).</p>
<p>Note that some genes are present in <em>S. japonicus</em>, <em>S. octosporus</em> or <em>S. cryophilus</em> but absent from <em>S. pombe</em>. For some of these gene products, GO annotations can be transferred from other species. If you wish to include annotations for these genes in your analysis you will need to use this option, and extend your GAF with the relevant annotation lines (contact the <a routerLink="/mailto:helpdesk@pombase.org">Helpdesk</a> if you need assistance). Combining all approaches gives the best coverage possible at present. You can use a &quot;GO Slim&quot; tool such as Princeton's GO Term Mapper to see if there are any gaps in coverage, as described in the <a routerLink="/faq/how-can-i-find-significant-shared-go-annotations-for-genes-in-a-list">FAQ on enrichment in <em>S. pombe</em></a>. Also see the FAQs on GO term enrichment in <a routerLink="/faq/can-i-do-go-term-enrichment-for-other-schizosaccharomyces-species-e-g-s-japonicus">other <em>Schizosaccharomyces</em> species</a>.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'how-can-i-find-intron-branch-sites'">
<h3 id="how-can-i-find-intron-branch-sites">How can I find intron branch sites?</h3>
<!-- pombase_categories: Finding data -->
<p>A dataset of intron branch sites is available as a track in the genome browser. The data were published in:</p>
<p>Bitton DA, Rallis C, Jeffares DC, Smith GC, Chen YY, Codlin S, Marguerat S, Bähler J. LaSSO, a strategy for genome-wide mapping of intronic lariats and branch points using RNA-seq. <em>Genome Res.</em> 2014 Jul; <strong>24(7)</strong>:1169-79. (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">PMID:24709818</a>; <a href="http://dx.doi.org/10.1101/gr.166819.113">DOI:10.1101/gr.166819.113</a>)</p>
<p>To view this data track, follow the instructions in the <a routerLink="/faq/how-can-i-show-or-hide-tracks-in-the-genome-browser">track configuration FAQ</a>, and select the Intron Branch Point track.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-modifications-my-protein-interest'">
<h3 id="how-can-i-find-modifications-my-protein-interest">How can I find modifications for my protein of interest?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>Protein modifications (where curated) are included in the Modifications section on gene pages. (We plan to include RNA modifications later.) The <a routerLink="/documentation/gene-page-modifications">gene page modifications documentation</a> describes the display.</p>
<p>To retrieve all genes whose products have a given modification, use the &quot;Protein modifications&quot; query in the [Advanced Search]. If you know the ID (for example, &quot;phosphorylated residue&quot; is MOD:00696), type or paste it into the box. Otherwise, start typing a name or description; the autocomplete feature will suggest terms. Choose one to retrieve annotated genes. See the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more information, including how ontology searches retrieve annotations to general terms.</p>
<!--
Example query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2220%22,%22query%22:%22MOD:00696%22%7D%7D,%22filter_count%22:%221%22%7D%5D">phosphorylated residue (MOD:00696)</a> 
-->

</div>
<div *ngIf="pageName == 'orthology' || pageName == 'finding-data' || pageName == 'how-can-i-find-orthologs-between-s.-pombe-and-other-schizosaccharomyces-species'">
<h3 id="how-can-i-find-orthologs-between-s.-pombe-and-other-schizosaccharomyces-species">How can I find orthologs between <em>S. pombe</em> and other Schizosaccharomyces species?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p>On a gene-by-gene basis, you can use the Ensembl genome browser link to reach Fungal Compara as described in the <a routerLink="/faq/how-can-i-find-s-pombe-orthologs-for-species-other-than-human-and-s-cerevisiae">FAQ on orthologs in other species</a>.</p>
<p>For a full set of orthologous genes in <em>S. pombe</em>, <em>S. cryophilus</em>, <em>S.japonicus</em> and <em>S. octosporus</em>, see Table S12, columns AD-AG, in Rhind <em>et al.</em> Comparative functional genomics of the fission yeasts (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>).</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-protein-localization-data'">
<h3 id="how-can-i-find-protein-localization-data">How can I find protein localization data?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>Gene Ontology (GO) cellular component annotations capture the localizations of gene products to subcellular structures such as organelles or complexes. GO Cellular Component annotations are displayed on PomBase gene pages as described in the <a routerLink="/documentation/gene-page-gene-ontology">PomBase GO documentation</a>. The GO Consortium provides <a href="http://geneontology.org/page/cellular-component-ontology-guidelines">documentation</a> that describes what the Cellular Component ontology includes. To search for proteins (or functional RNAs) with a particular localization, use the GO query in the <a routerLink="/query">advanced search</a> to find genes annotated to the relevant GO Cellular Component term(s).</p>
<p>Pombase GO Cellular Component annotations include data from the whole-genome localization study (Matsuyama et al. 2006) as well as manually curated data from papers on small-scale experiments, and inferences from ortholog annotations. Macromolecular complex annotations are also available in a file (see <a routerLink="/faq/is-there-a-list-of-protein-complexes-in-s-pombe-and-their-subunits">FAQ</a>).</p>
<!--
Example query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%221%22,%22query%22:%22GO:0005634%22%7D%7D,%22filter_count%22:%221%22%7D%5D">nucleus (GO:0005634)</a>
-->

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'how-can-i-find-proteins-have-transmembrane-domains'">
<h3 id="how-can-i-find-proteins-have-transmembrane-domains">How can I find proteins that have transmembrane domains?</h3>
<!-- pombase_categories: Finding data -->
<p>In the <a routerLink="/query">advanced search</a>, use the &quot;Number of TM domains&quot; query. Enter the minimum and maximum number of domains (use the same number as minimum and maximum to retrieve proteins with, e.g. exactly 7 transmembrane domains).</p>
<!--
Example query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%226%22,%22query_1%22:%227%22,%22query_2%22:%227%22%7D%7D,%22filter_count%22:%221%22%7D%5D">Genes whose products have 7 transmembrane domains</a> 
-->

</div>
<div *ngIf="pageName == 'querying-searching' || pageName == 'sequence-retrieval' || pageName == 'how-can-i-find-rrna-genes'">
<h3 id="how-can-i-find-rrna-genes">How can I find rRNA genes?</h3>
<!-- pombase_categories: Querying/Searching,Sequence Retrieval -->
<p>In the <a routerLink="/query">advanced search</a>, click &quot;Product type&quot;, then select &quot;rRNA&quot; from the pulldown.</p>
<p>Also see the FAQ on <a routerLink="/faq/are-there-any-rdna-repeat-sequences-in-pombase">rDNA sequences</a>.</p>
<!--
Query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%229%22,%22query%22:%22rRNA%22%7D%7D,%22filter_count%22:%221%22%7D%5D">rRNA genes</a> 
-->

</div>
<div *ngIf="pageName == 'orthology' || pageName == 'finding-data' || pageName == 'how-can-i-find-s.-pombe-genes-associated-human-disease'">
<h3 id="how-can-i-find-s.-pombe-genes-associated-human-disease">How can I find <em>S. pombe</em> genes associated with human disease?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p>S. pombe genes whose human orthologs have been implicated in disease are annotated with terms from the internal PBO vocabulary. To retrieve all of these genes, you can use the most general &quot;disease associated&quot; term in a query. In the <a routerLink="/query">advanced search</a>, select &quot;Disease&quot;. You can either start typing 'disease associated', and choose 'disease_associated' from the autocomplete options, or enter the ID 'PBO:5000000'.</p>
<p>Also see the FAQs on <a routerLink="/faq/how-can-i-find-all-s-pombe-genes-that-are-conserved-in-human">finding genes conserved in human</a>, <a routerLink="/faq/how-can-i-find-s-pombe-ortholog-s-of-a-human-gene">finding the ortholog of a specific gene</a>, and on <a routerLink="/faq/how-can-i-obtain-the-list-of-human-and-s-pombe-orthologs">downloading the full set of curated orthologs</a>.</p>
<!--
Example queries:

-   <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2223%22,%22query%22:%22PBO:5000000%22%7D%7D,%22filter_count%22:%221%22%7D%5D">disease\_associated     (PBO:5000000)</a> 
-   <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2223%22,%22query%22:%22PBO:0000239%22%7D%7D,%22filter_count%22:%221%22%7D%5D">cancer     (PBO:0000239)</a> 

-->

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-s.-pombe-genes-complement-or-are-complemented-by-gene-another-species'">
<h3 id="how-can-i-find-s.-pombe-genes-complement-or-are-complemented-by-gene-another-species">How can I find <em>S. pombe</em> genes that complement or are complemented by a gene from another species?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>If there is complementation data available for an <em>S. pombe</em> gene, it will be displayed in the Complementation section of the gene page. For example, <a routerLink="/spombe/result/SPAC57A10.12c"><em>ura3</em></a> can be complemented by <em>S. cerevisiae</em> <a href="http://www.yeastgenome.org/locus/S000001699/overview"><em>URA1</em></a>, and itself complements human <a href="http://www.genenames.org/cgi-bin/gene_symbol_report?hgnc_id=HGNC:2867">DHODH</a>.</p>
<!-- To search for complementation annotations, use one of the "PBO"
filters in the <a routerLink="/query">advanced search</a> (see the
<a routerLink="/documentation/advanced-search">documentation</a> for help with
searching). The complementation descriptions are stored as entries in
the PBO internal ontology, so a search for PBO term names that match
"complements" or "complemented by" will retrieve genes with
complementation data curated. The most general term, "complementation"
(PBO:2000000) retrieves all genes that have any complementation
annotation.

Example queries:

-   <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2223%22,%22query%22:%22PBO:2000000%22%7D%7D,%22filter_count%22:%221%22%7D%5D">Genes with any complementation annotation     (PBO:2000000)</a> 
-   <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2223%22,%22query%22:%22PBO:0011087%22%7D%7D,%22filter_count%22:%221%22%7D%5D">functionally complements S. cerevisiae URA1     (PBO:0011087)</a> 
-->
<p>g</p>

</div>
<div *ngIf="pageName == 'orthology' || pageName == 'finding-data' || pageName == 'how-can-i-find-s.-pombe-ortholog-s-human-gene'">
<h3 id="how-can-i-find-s.-pombe-ortholog-s-human-gene">How can I find <em>S. pombe</em> ortholog(s) of a human gene?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p>In PomBase, human orthologs are curated for <em>S. pombe</em> genes as described in the <a routerLink="/documentation/orthologs">Orthologs documentation</a>.</p>
<p>To find <em>S. pombe</em> orthologs for a human gene, you can search for the standard human gene name in the simple search box in the page header. For example, searching for human <em>ABTB1</em> will retrieve the <em>S. pombe</em> gene <em>btb3</em>. To find standard human gene names, you can search <a href="http://www.genenames.org/">HGNC</a>. Note that in a few cases, a human gene name will coincidentally match a name or synonym of a non-orthologous <em>-S. pombe</em> gene as well as the actual curated ortholog(s), so please check the gene pages carefully, especially if your search retrieves more than one result.</p>
<p>Also see the FAQs on on <a routerLink="/faq/how-can-i-find-all-s-pombe-genes-that-are-conserved-in-human">finding genes conserved in human</a>, <a routerLink="/faq/how-can-i-find-s-pombe-genes-associated-with-human-disease">finding disease gene orthologs</a>, and on <a routerLink="/faq/how-can-i-obtain-the-list-of-human-and-s-pombe-orthologs">downloading the full set of curated orthologs</a>.</p>

</div>
<div *ngIf="pageName == 'orthology' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'how-can-i-find-s.-pombe-orthologs-species-other-than-human-and-s.-cerevisiae'">
<h3 id="how-can-i-find-s.-pombe-orthologs-species-other-than-human-and-s.-cerevisiae">How can I find <em>S. pombe</em> orthologs for species other than human and <em>S. cerevisiae</em>?</h3>
<!-- pombase_categories: Orthology,Finding data,Tools and resources -->
<p>For orthologs that are not manually curated by PomBase, we suggest two approaches:</p>
<p><strong>DIOPT</strong></p>
<p>From any gene page, follow the link to DIOPT under External References.</p>
<p>The <a href="http://www.flyrnai.org/diopt">Drosophila RNAi Screening Center Integrative Ortholog Prediction Tool</a> (DIOPT) is a web-based tool that integrates several orthology prediction tools to identify orthologous proteins for nine species (<em>Caenorhabditis elegans, Danio rerio, Drosophila melanogaster, Homo sapiens, Mus musculus, Rattus norvegicus, Saccharomyces cerevisiae, Schizosaccharomyces pombe, and Xenopus tropicalis</em>).</p>
<p><strong>Compara</strong></p>
<p>You can search for orthologs/paralogs in Fungi, or in a pan-taxonomic comparison (eukaryotes), using Compara in the Ensembl browser.</p>
<ol style="list-style-type: decimal">
<li>On any gene page, follow the &quot;View in Ensembl genome browser&quot; link.</li>
<li>Click the &quot;Gene:&quot; tab to show Compara links in the left-hand margin - for fungal alignments, choose &quot;Fungal Compara&quot;, or for all eukaryotic species choose &quot;Pan-taxonomic Compara&quot;.</li>
<li>You should see a &quot;collapsed&quot; gene tree highlighting your fission yeast gene of interest. From here you can click on any node to see a menu of options:
<ol style="list-style-type: decimal">
<li>Expand or collapes specific sub-nodes of the tree, or expand the tree fully</li>
<li>View the alignment in FASTA format</li>
<li>Launch the jalview multiple alignment viewer to see the full alignment and colour by residue conservation, hydrophobicity, etc.</li>
</ol></li>
<li>To configure the protein entries visible in the alignment, select the most &quot;inclusive&quot; node you require. You can reduce the number of entries by collapsing individual sub-trees before you generate your alignment.</li>
</ol>
<p>Information about how the Compara trees are generated, homology types, and species is available from the <a href="http://ensemblgenomes.org/info/data/comparative_genomics">Ensembl comparative genomics documentation</a>.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-significant-shared-fypo-annotations-genes-list'">
<h3 id="how-can-i-find-significant-shared-fypo-annotations-genes-list">How can I find significant shared FYPO annotations for genes in a list?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>FYPO enrichment analysis is analogous to GO term enrichment, using phenotypes rather than GO annotations, i.e. analysing a gene list by finding FYPO terms that are significantly over- or under-represented among the annotations for the genes.</p>
<p>At present, PomBase does not have its own FYPO enrichment tool, and very few ontology enrichment tools can use phenotype data. One that does is <a href="http://bahlerweb.cs.ucl.ac.uk/cgi-bin/GLA/GLA_input">AnGeLi</a>, produced by Jürg Bähler's lab.</p>

</div>
<div *ngIf="pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-significant-shared-go-annotations-genes-list'">
<h3 id="how-can-i-find-significant-shared-go-annotations-genes-list">How can I find significant shared GO annotations for genes in a list?</h3>
<!-- pombase_categories: Tools and resources,Using ontologies -->
<p>GO term enrichment identifies GO terms that are significantly overrepresented (or underrepresented) among a set of genes.</p>
<p>At present PomBase does not have its own GO enrichment tool. We recommend using the <a href="http://go.princeton.edu/cgi-bin/GOTermFinder">Generic GO Term Finder at Princeton</a>, because it offers a simple interface and up-to-date ontology and annotation data, including the current PomBase GO annotation dataset (you can upload your own backgound set, GO annotation file, or both). You can also use the GO Term Finder to retrieve all annotations for your gene list by setting the p-value to 1.</p>
<p>Before you perform an enrichment analysis, we recommend that you use the accompanying &quot;slimming&quot; tool, <a href="http://go.princeton.edu/cgi-bin/GOTermMapper">GO Term Mapper</a>, which is useful for a broad overview of the annotation set (for more information, gsee the <a routerLink="/browse-curation/fission-yeast-go-slim-terms">Fission Yeast GO slim terms</a> page and <a routerLink="/faq/how-can-i-use-go-slims-with-s-pombe">FAQ</a>). GO Term Mapper is especially useful if you use your own GAF for the enrichment, because it will show:</p>
<ul>
<li>IDs in your gene list that are missing from the annotation set (the annotations in GO Term Mapper's database or your uploaded GAF)</li>
<li>Genes in your list that have annotations but do not map to the slim</li>
<li>Genes in your list that have no annotations</li>
</ul>
<p>A few other enrichment tools are described on the GO Consortium's <a href="http://geneontology.org/page/go-enrichment-analysis">GO Enrichment Analysis</a> page. If you choose one of these, we recommend that you only use an enrichment tool that allows you to upload a background set representing the genes used in your experiment. Even if you regard the whole genome as the relevant background, it is important to specify the background gene set explicitly to obtain meaningful results, especially if you have data only for protein-coding genes. For example, annotated tRNA genes can obscure enrichment of protein-coding genes annotated to translation.</p>
<p>For any GO analysis, we strongly recommend that you describe your approach fully in methods, and include the release details (number and/or date) for PomBase and the GO terms and annotations you use.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'sequence-retrieval' || pageName == 'how-can-i-find-snorna-genes'">
<h3 id="how-can-i-find-snorna-genes">How can I find snoRNA genes?</h3>
<!-- pombase_categories: Finding data,Sequence Retrieval -->
<p>In the <a routerLink="/query">advanced search</a>, click &quot;Product type&quot;, then select &quot;snoRNA&quot; from the pulldown.</p>
<p>Note that there are likely a number of snoRNAs that have not yet been identified and annotated in <em>S. pombe</em>; we hope to investigate further in the future.</p>
<!--
Query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%229%22,%22query%22:%22snoRNA%22%7D%7D,%22filter_count%22:%221%22%7D%5D">snoRNA genes</a> 
-->

</div>
<div *ngIf="pageName == 'datasets' || pageName == 'querying-searching' || pageName == 'how-can-i-find-transcription-factor-binding-sites-pombase'">
<h3 id="how-can-i-find-transcription-factor-binding-sites-pombase">How can I find transcription factor binding sites in PomBase?</h3>
<!-- pombase_categories: Datasets,Querying/Searching -->
<p>There is a data track available for transcription factor binding sites in the genome browser. Follow the <a routerLink="/faq/how-can-i-show-or-hide-tracks-in-the-genome-browser">instructions for showing tracks</a>, and choose &quot;Transcription Factor Binding Sites&quot; in the left-hand menu of the pop-up.</p>
<p>Also see the <a routerLink="/faq/how-can-i-find-transcription-factors-and-their-targets-in-pombase">FAQ on finding transcription factors</a>.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'how-can-i-find-transcription-factors-and-their-targets-pombase'">
<h3 id="how-can-i-find-transcription-factors-and-their-targets-pombase">How can I find transcription factors and their targets in PomBase?</h3>
<!-- pombase_categories: Finding data -->
<p>All sequence-specific DNA-binding transcription factors should be annotated to at least two GO Molecular Function terms, either directly or by transitivity (i.e. annotated to a more specific &quot;descendant&quot; term linked to one of these terms):</p>
<ul>
<li>GO:0000976 transcription regulatory region sequence-specific DNA binding (view in <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000976">QuickGO</a> or <a href="http://amigo.geneontology.org/amigo/term/GO:0000976">AmiGO</a>)</li>
<li>GO:0003700 sequence-specific DNA binding transcription factor activity (view in <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003700">QuickGO</a> or <a href="http://amigo.geneontology.org/amigo/term/GO:0003700">AmiGO</a>)</li>
</ul>
<p><a routerLink="/faq/what-is-an-annotation-extension">Annotation extensions</a> are used to capture two types of &quot;target&quot; data (where available):</p>
<ul>
<li>Annotations to GO:0000976 (or a descendant) may have extensions that capture DNA binding specificity using <a href="http://sequenceontology.org">Sequence Ontology</a> (SO) terms. A list of DNA binding sites identified in <em>S. pombe</em> is available on the <a routerLink="/browse-curation/dna-binding-sites">DNA Binding Sites</a> page.</li>
<li>Annotations to GO:0003700 (or a descendant) may have extensions identifying target genes.</li>
</ul>
<p>Because it is not yet possible to query annotation extensions in the PomBase advanced search, to identify target genes you must either inspect transcription factor gene pages manually, or search the GO annotation dataset. For the latter:</p>
<ol style="list-style-type: decimal">
<li>Download the GO annotation file (GAF) from the <a routerLink="/downloads/go-annotations">GO Annotations</a> page. The file is tab-delimited text, so it can be opened in a spreadsheet application or parsed with a script; the <a href="http://geneontology.org/page/go-annotation-file-format-20">format</a> is described on the GO website.</li>
<li>Look up the GO IDs for the specific terms to which genes are directly annotated -- the &quot;Child Terms&quot; feature in QuickGO is good for this (for &quot;transcription factor activity&quot;, the most commonly used term is GO:0000978, RNA polymerase II core promoter proximal region sequence-specific DNA binding).</li>
<li>Find annotations to the GO IDs of interest (GO ID is column 5; gene ID column 2), and then look at the annotation extensions (column 16).</li>
<li>Contact the <a routerLink="/mailto:helpdesk@pombase.org">Helpdesk</a> if you have any problems or questions.</li>
</ol>
<p>Finally, note that not all <em>S. pombe</em> transcription factors have been extensively characterised with respect to target genes, and for those that have, target curation in PomBase may be incomplete. You may therefore wish to query for transcription factors that have been have been experimentally characterised, and therefore might have targets which are not yet curated. To do so, use the Advanced Search to find which of the genes annotated to the transcription factor-related GO terms above have the annotation status &quot;published&quot; (e.g. GO ID &quot;GO:0000978&quot; AND Annotation Status &quot;published&quot;; see the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more tips on setting up the query).</p>
<!--
Query links:

<a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22operator%22:%22OR%22,%22param%22:%7B%22set_1%22:%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%221%22,%22query%22:%22GO:0000976%22%7D%7D,%22filter_count%22:%221%22%7D,%22set_2%22:%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%221%22,%22query%22:%22GO:0003700%22%7D%7D,%22filter_count%22:%221%22%7D%7D%7D%5D">Genes annotated to 'transcription regulatory region sequence-specific DNA binding' (GO:0000976) or 'sequence-specific DNA binding transcription factor activity' (GO:0003700)</a> 

<a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%221%22,%22query%22:%22GO:0000978%22%7D,%22filter_2%22:%7B%22operator%22:%22AND%22,%22filter%22:%2211%22,%22query%22:%22PBO:0000001%22%7D%7D,%22filter_count%22:%222%22%7D%5D">Genes annotated to 'RNA polymerase II core promoter proximal region sequence-specific DNA binding' (GO:0000978) with annotation status 'published'</a> 
-->

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'how-can-i-find-transposons-s.-pombe-genome'">
<h3 id="how-can-i-find-transposons-s.-pombe-genome">How can I find transposons in the <em>S. pombe</em> genome?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>In the <a routerLink="/query">advanced search</a>, click &quot;Characterisation status&quot;, then select &quot;transposon&quot; from the pulldown.</p>
<p>At present, there are 11 full-length transposons annotated, and two frameshifted copies.</p>
<!--
Query link:
<a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2211%22,%22query%22:%22PBO:0000007%22%7D%7D,%22filter_count%22:%221%22%7D%5D">Transposons</a> \
-->
<p>Lone LTRs are also annotated as sequence features. They cannot yet be retrieved by the simple or advanced searches, but they can be displayed on a <a routerLink="/faq/how-can-i-show-or-hide-tracks-in-the-genome-browser">track in the Ensembl browser</a> (under &quot;Repeats&quot;).</p>
<p>Finally, if you wish to install Artemis (available from http://www.sanger.ac.uk/science/tools/artemis), you can use it to view LTRs in more detail. Read in the EMBL format files of sequence and annotation (available from the <a routerLink="/downloads/genome-datasets">Genome Feature Annotations</a> page). To see LTRs,</p>
<ol style="list-style-type: decimal">
<li>In the Select menu, choose &quot;By Key&quot;.</li>
<li>In the pulldown that pops up, choose &quot;LTR&quot;.</li>
</ol>
<p>See the <a routerLink="/faq/is-there-an-equivalent-to-the-artemis-java-applet-in-pombase">Artemis FAQ</a> and the <a href="ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf">Artemis manual</a> (pdf; Sanger site) for additional information.</p>

</div>
<div *ngIf="pageName == 'tools-and-resources' || pageName == 'how-can-i-get-cosmids-my-favorite-gene'">
<h3 id="how-can-i-get-cosmids-my-favorite-gene">How can I get cosmids for my favorite gene?</h3>
<!-- pombase_categories: Tools and resources -->
<p>Please enquire with the Sanger Institute <a routerLink="/mailto:archives@sanger.ac.uk">archives@sanger.ac.uk</a> about clones included in the final genome sequence.</p>

</div>
<div *ngIf="pageName == 'querying-searching' || pageName == 'using-ontologies' || pageName == 'how-can-i-get-go-annotation-search-results-match-amigo'">
<h3 id="how-can-i-get-go-annotation-search-results-match-amigo">How can I get GO annotation search results to match AmiGO?</h3>
<!-- pombase_categories: Querying/Searching,Using ontologies -->
<p>In <a href="http://amigo.geneontology.org/">AmiGO</a>, the &quot;term association&quot; search retrieves gene products annotated to a GO term and to any of its child terms, following all relationships in the ontology, including <em>regulates</em>.<br />
<br />
The PomBase GO search excludes <em>regulates</em> relationships by default, so annotation totals will differ from those in AmiGO for any terms that have child terms connected by <em>regulates</em> links. For example, a search for &quot;cytokinesis&quot; in AmiGO will include genes annotated to &quot;regulation of cytokinesis&quot;, whereas a search in PomBase will not. To include regulation, and have results that match AmiGO, you must search for both terms -- see <a routerLink="/faq/how-can-i-search-genes-involved-both-go-process-and-regulation-process">instructions here</a>.<br />
<br />
For more information on <em>regulates</em>, see the <a href="http://www.geneontology.org/GO.ontology.relations.shtml">GO Ontology Relations documentation</a>.</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'how-can-i-get-my-data-into-pombase'">
<h3 id="how-can-i-get-my-data-into-pombase">How can I get my data into PomBase?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>The best route depends on the amount and type of data you have.</p>
<ul>
<li>If you have a published paper with &quot;small-scale&quot; data (a few genes, several different data types, etc.), you can curate it in <a href="http://curation.pombase.org/pombe">Canto</a>, PomBase's online <a routerLink="/community/fission-yeast-community-curation-project">community curation</a> tool. The <a href="http://curation.pombase.org/pombe/docs/index/">Canto documentation</a> describes the supported data types (Gene Ontology, phenotypes, interactions, modifications) and how to use the curation interface.</li>
<li>If you have data from large-scale experiments that is associated with sequence coordinates, it can be displayed as a track in the Genome Browser. Use the <a routerLink="/submit-data/data-submission-form">HTP data submission form</a> to send details.</li>
<li>If you have a large set of phenotype data (e.g. from a screen of the deletion collection), especially if there are many annotations to one or a few <a routerLink="/browse-curation/fission-yeast-phenotype-ontology">FYPO</a> term(s), you can send them to the helpdesk in the <a routerLink="/documentation/phenotype-data-bulk-upload-format">phenotype annotation spreadsheet format</a>.</li>
<li>For large sets of genetic or physical interaction data, we recommend that you prepare a spreadsheet using the <a href="http://wiki.thebiogrid.org/doku.php/contribute">template provided by BioGRID</a> (see &quot;Step 1. Send Us Your Interaction Data&quot;). You are welcome to send the spreadsheet either to BioGRID or to PomBase (via the helpdesk); PomBase and BioGRID exchange data regularly so all interactions will appear in both databases.</li>
</ul>
<p>You can do both Canto curation and large-scale data submission for a single publication, if it reports both large- and small-scale experimental results.</p>
<p>For any data types not listed above, or if you have any questions, please contact the <a routerLink="/mailto:helpdesk@pombase.org">PomBase helpdesk</a>.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-identify-all-genes-affect-process'">
<h3 id="how-can-i-identify-all-genes-affect-process">How can I identify all of the genes that affect a process?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>The best way to find genes that have any effect on a process, we recommend searching for both GO and FYPO terms relevant to the process.</p>
<p>As described in the <a routerLink="/faq/why-are-some-genes-with-an-abnormal-phenotype-annotated-to-the-corresponding-go-process-while-others-are-not">FAQ on GO and FYPO annotations</a>, PomBase curators annotate all genes with phenotypes that affect a process, whereas GO annotations are restricted to genes whose products act directly in a process or its regulation. By querying for genes annotated to either a GO term or a FYPO term, you can find genes with relevant phenotypes (including &quot;downstream effects&quot;) as well as genes involved in a process (with or without mutant phenotypes affecting the process).</p>
<p>Use the union (OR operator) button in the PomBase <a routerLink="/query">advanced search</a>, available in the query history, as described in the <a routerLink="/documentation/advanced-search">advanced search documentation</a>. For example, to find genes that affect cellular respiration, search for &quot;FYPO:0000078 (abnormal cellular respiration)&quot;; search for &quot;GO:0045333 (cellular respiration)&quot;; and then combine them in a third query. For any process, you can try using more specific or less specific terms to retrieve the results that best fit your expectations and needs.</p>
<!--
Example query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22operator%22:%22OR%22,%22param%22:%7B%22set_1%22:%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%221%22,%22query%22:%22GO:0045333%22%7D%7D,%22filter_count%22:%221%22%7D,%22set_2%22:%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2219%22,%22query_1%22:%22FYPO:0000078%22,%22query_2%22:%22all_alleles%22%7D%7D,%22filter_count%22:%221%22%7D%7D%7D%5D">genes annotated to 'abnormal cellular respiration' (FYPO:0000078) or 'cellular respiration' (GO:0045333)</a> 
-->

</div>
<div *ngIf="pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-locate-centromeres'">
<h3 id="how-can-i-locate-centromeres">How can I locate centromeres?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>Centromeres can be retrieved in the PomBase Ensembl browser; the coordinates are:</p>
<p>Chromosome I: <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=I%3A3753687-3789421;site=ensemblunit">3753687-3789421</a> Chromosome II: <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=II%3A1602264-1644747;site=ensemblunit">1602264-1644747</a> Chromosome III: <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=III%3A1070904-1137003;site=ensemblunit">1070904-1137003</a></p>
<p>Sequence features within the centromeres, such as repeats, are annotated with Sequence Ontology terms. For more details, see the <a routerLink="/status/centromeres">Centromeres</a> page.</p>

</div>
<div *ngIf="pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-locate-mating-type-region'">
<h3 id="how-can-i-locate-mating-type-region">How can I locate the mating type region?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>Browse for Chromosome <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=II%3A2129208-2137121;site=ensemblunit">II:2129208-2137121</a>, and see the <a routerLink="/status/mating-type-region">Mating type region</a> page.</p>
<p>The mating type region will soon be annotated as a feature, and refer to a Sequence Ontology term.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'how-can-i-locate-telomeres-and-subtelomeric-regions'">
<h3 id="how-can-i-locate-telomeres-and-subtelomeric-regions">How can I locate telomeres and subtelomeric regions?</h3>
<!-- pombase_categories: Finding data -->
<p>The current <em>S. pombe</em> genome assembly does not include the complete telomeric regions or the telomeric short repeats. These omissions are beyond the control of PomBase curators. Subtelomeric repeats are also not explicitly defined at present, although we hope to provide this information in the future. Additional information about <em>S. pombe</em> telomeres is available at on the <a routerLink="/status/telomeres">Telomeres</a> page.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'orthology' || pageName == 'how-can-i-obtain-list-human-and-s.-pombe-orthologs'">
<h3 id="how-can-i-obtain-list-human-and-s.-pombe-orthologs">How can I obtain the list of human and <em>S. pombe</em> orthologs?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists,Orthology -->
<p>The current version of the manually curated list of orthologs and orthologous groups identified between fission yeast and human is available for download from the <a routerLink="/downloads/manually-curated-orthologs">Orthologs</a> page (linked from the <a routerLink="/downloads/datasets">Datasets</a> page). Also see the FAQs on <a routerLink="/faq/how-can-i-find-all-s-pombe-genes-that-are-conserved-in-human">finding genes conserved in human</a>, <a routerLink="/faq/how-can-i-find-s-pombe-genes-associated-with-human-disease">finding disease gene orthologs</a>, and <a routerLink="/faq/how-can-i-find-s-pombe-orthologs-of-a-human-gene">finding the ortholog of a specific gene</a>.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'finding-data' || pageName == 'how-can-i-retrieve-all-s.-pombe-genes'">
<h3 id="how-can-i-retrieve-all-s.-pombe-genes">How can I retrieve all <em>S. pombe</em> genes?</h3>
<!-- pombase_categories: Genome statistics and lists,Finding data -->
<p>The <a routerLink="/downloads/names-and-identifiers">Names and identifiers page</a> provides lists of all genes, including non-coding RNA genes. The allNames.tsv file contains the systematic IDs and names for both protein-coding and non-coding RNA genes, whereas two separate files are available that also include product descriptions.</p>
<p>The <a routerLink="/query">advanced search</a> includes a query that retrieves all protein-coding genes at once, as decribed in <a routerLink="/faq/how-can-i-retrieve-all-s-pombe-protein-coding-genes">this FAQ</a>.</p>
<!--
Query link: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2235%22%7D%7D,%22filter_count%22:%221%22%7D%5D">All genes</a> 
-->

</div>
<div *ngIf="pageName == 'datasets' || pageName == 'genome-statistics-and-lists' || pageName == 'querying-searching' || pageName == 'how-can-i-retrieve-all-s.-pombe-protein-coding-genes'">
<h3 id="how-can-i-retrieve-all-s.-pombe-protein-coding-genes">How can I retrieve all <em>S. pombe</em> protein-coding genes?</h3>
<!-- pombase_categories: Datasets,Genome statistics and lists,Querying/Searching -->
<p>The <a routerLink="/query">advanced search</a> includes a query that retrieves all protein-coding genes at once. Click &quot;Canned queries&quot;, then click &quot;All protein coding genes (ex. dubious and transposon)&quot;.</p>
<p>Further explanation: All protein coding genes have the type &quot;protein coding&quot;, but this type also includes a few transposon genes and several genes that are dubious (i.e. predicted by automated methods considered unlikely to actually encode protein), which you will presumably want to exclude from the set. The canned query does this.</p>
<!--
Query link: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B">Protein-coding genes (excluding 'dubious' and 'transposon' status)</a> 
-->

</div>
<div *ngIf="pageName == 'gene-page' || pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-retrieve-gene-sequence-including-upstream-and-downstream-sequences'">
<h3 id="how-can-i-retrieve-gene-sequence-including-upstream-and-downstream-sequences">How can I retrieve a gene sequence, including upstream and downstream sequences?</h3>
<!-- pombase_categories: Gene page,Genome browser,Finding data -->
<p>You can retrieve sequences from a gene page or in the Genome Browser.</p>
<p>On the gene page: Scroll down or click the left-hand menu link to the Sequence section of the page, where a DNA sequence is displayed by default. For protein-coding genes, the CDS is shown, and you can click boxes to add the sequences of any annotated introns or 5' or 3' UTRs. To include flanking sequences, enter a number of nucleotides in the &quot;upstream&quot; and/or &quot;downstream&quot; boxes. Only applicable options appear for non-coding RNA genes. The &quot;Download sequence&quot; button saves a FASTA file with your selection.</p>
<p><strong>To use the Genome Browser</strong>: Click the &quot;View in Genome Browser&quot; link under the map graphic on a gene page. Click Export Data (a button on the left hand side). Select the number of bases up- and downstream, which strand and the features you would like. Click &quot;next&quot;. Select your download option. Your browser will save or display the data, depending on which format you select.</p>
<p>To retrieve flanking regions for more than one gene at a time, at present you must use the Ensembl Genomes Biomart query, as described in <a routerLink="/faq/can-i-download-sequences-for-many-genes-at-once-including-flanking-regions">this FAQ</a>.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'how-can-i-retrieve-intron-coordinates-or-sequences'">
<h3 id="how-can-i-retrieve-intron-coordinates-or-sequences">How can I retrieve intron coordinates or sequences?</h3>
<!-- pombase_categories: Finding data -->
<p>Downloadable intron datasets are available in FASTA format from the <a routerLink="/downloads/intron-data">Intron Data</a> page.</p>
<p>You can also find genes with introns using the PomBase <a routerLink="/query">advanced search</a>. To find all genes with introns, search for genes with a specified number of exons, and use the range 2 (i.e. at least one intron) to 20 (more than the maximum known, 16 introns). You can also restrict the search to protein-coding genes. Note that the PomBase count includes introns in UTRs.</p>
<p>Instructions for searching PomBase</p>
<ol style="list-style-type: decimal">
<li>Go to the <a routerLink="/query">advanced search</a>.</li>
<li>Select the &quot;Coding exons&quot; query.</li>
<li>Enter values: min 2, max 20. Click &quot;Search&quot;.</li>
<li>Optional: to restrict to protein-coding genes, select the &quot;Product type&quot; query, then choose &quot;protein&quot;. Use the &quot;Intersect&quot; (AND operator&quot;) button to combine the queries.</li>
</ol>
<p>Click on a count in the query history to see the results, with a button for &quot;Download&quot; options including coordinates and sequence.</p>
<p>Also see the <a routerLink="/faq/how-can-i-find-all-sequence-features-in-a-region-using-chromosome-coordinates">FAQ on finding sequence features in a region</a>.</p>
<!--
Query link: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%228%22,%22query_1%22:%222%22,%22query_2%22:%2220%22%7D,%22filter_2%22:%7B%22operator%22:%22AND%22,%22filter%22:%229%22,%22query%22:%22protein_coding%22%7D%7D,%22filter_count%22:%222%22%7D%5D">protein-coding genes with 2-20 exons</a> 
-->

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'how-can-i-retrieve-sequence-coordinates-all-features-particular-type'">
<h3 id="how-can-i-retrieve-sequence-coordinates-all-features-particular-type">How can I retrieve sequence coordinates for all features of a particular type?</h3>
<!-- pombase_categories: Finding data -->
<p>Available options:</p>
<ol style="list-style-type: decimal">
<li><p>Download one of the files available via the <a routerLink="/downloads/genome-datasets">Genome Datasets</a> page. The GFF3 files contain coordinates, and you can parse the files for the feature type you need. For example, to find all non-coding RNAs, search for &quot;ncRNA_gene&quot;; for coding sequences, use &quot;CDS&quot;, etc. There are also separate files available for CDS and UTR data.</p></li>
<li><p>If you only need genes, you can use the <a routerLink="/query">advanced search</a> to find all genes of a given type. (Note that non-gene features such as repeats cannot be retrieved by this method.) Select the &quot;Product type&quot; query, then choose a type from the pulldown menu. The &quot;Download&quot; options include coordinates and sequences in FASTA format. If you need more than one feature type, query for each type and then use Query Management to combine the individual queries with the &quot;Union&quot; (OR operator) button. See the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more information. Click on a count in the query history to see the results, with a button for &quot;Download&quot; options including coordinates and sequence.</p></li>
<li><p>The bioinformatically inclined can also use the Ensembl Genomes REST API to retrieve transcript feature coordinates, as described in the <a routerLink="/faq/is-the-s-pombe-transcriptome-available-in-fasta-format">FAQ on pombe transcriptome sequences</a>. Select the desired feature type(s) from the output file of stable IDs (bear in mind that Ensembl idiosyncratically uses &quot;biotype&quot; to mean feature type).</p></li>
</ol>
<!--
Example advanced search query: <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%229%22,%22query%22:%22snoRNA%22%7D%7D,%22filter_count%22:%221%22%7D%5D">snoRNA genes</a> 
-->

</div>
<div *ngIf="pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-retrieve-sequence-region-using-sequence-coordinates'">
<h3 id="how-can-i-retrieve-sequence-region-using-sequence-coordinates">How can I retrieve the sequence of a region using sequence coordinates?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>Go to the <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Info/Index">Genome Browser</a> (or click the &quot;View in Ensembl genome browser ...&quot; link on any gene page), and find the region of interest by searching for a gene name, a systematic ID, or a set of coordinates. Then click the Export Data button on the left-hand side.</p>
<p>Select the number of bases up- and downstream (even if you have searched using coordinates, you can add flanking sequence to what you download), which strand and the features you would like. Click &quot;next&quot;. Select your download option. Your browser will save or display the data, depending on which format you select.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'how-can-i-retrieve-sequences-non-protein-coding-genes'">
<h3 id="how-can-i-retrieve-sequences-non-protein-coding-genes">How can I retrieve sequences for non-protein-coding genes?</h3>
<!-- pombase_categories: Finding data -->
<p>A file of all non-coding RNA gene sequences is available on the <a routerLink="/downloads/genome-datasets">Genome Feature Annotations page</a>.</p>
<p>If you need sequences for all genes of a single type (tRNAs, rRNAs, other ncRNAs, etc.) we recommend using the <a routerLink="/query">advanced search</a> and results download as described in the <a routerLink="/faq/how-can-i-retrieve-sequence-coordinates-for-all-features-of-a-particular-type">FAQ on retrieving sequence coordinates for all features of a particular type</a>.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'how-can-i-retrieve-utr-sequences'">
<h3 id="how-can-i-retrieve-utr-sequences">How can I retrieve UTR sequences?</h3>
<!-- pombase_categories: Finding data -->
<p>Transcript start and end coordinates from all sources will be available as individual data tracks in the genome browser in the near future, which will allow you to view, evaluate and download them. We also provide downloadable <a routerLink="/downloads/utr">UTR data sets</a> that are updated periodically.</p>
<p>Also see the <a routerLink="/faq/how-do-you-determine-a-gene-s-full-length-transcript-utr-coordinates-transcription-start-and-end-sites">precedence criteria</a> used to choose default UTR features to display on gene pages.</p>
<p>To retrieve UTRs for a specified list of genes, see the FAQ on <a routerLink="/faq/can-i-download-sequences-for-many-genes-at-once-including-flanking-regions">downloading sequences for multiple genes</a> (choose 5' UTR and/or 3' UTR in the popup).</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-search-genes-involved-both-go-process-and-regulation-process'">
<h3 id="how-can-i-search-genes-involved-both-go-process-and-regulation-process">How can I search for genes involved in both a GO process and regulation of the process?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>The <a routerLink="/query">advanced search</a> GO query retrieves gene products annotated to a GO term and to any of its child terms, following the <em>is_a</em>, <em>part_of</em>, and <em>regulates</em> relationships in the ontology (also see the <a routerLink="/documentation/gene-page-gene-ontology">PomBase GO documentation</a>. GO search results for Biological Process terms therefore include genes involved in the process and its regulation.</p>
<p>Genes annotated directly to the process can be distinguished from those annotated to regulation on the <a routerLink="/documentation/ontology-term-page">ontology term page</a> for the term.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-search-or-browse-go-annotations'">
<h3 id="how-can-i-search-or-browse-go-annotations">How can I search or browse GO annotations?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>You can search for GO terms by name or ID in the PomBase <a href="/query">advanced search</a>, and retrieve a list of all genes annotated to the term and its descendants via the relations <em>is_a</em>, <em>part_of</em>, <em>regulates</em>, <em>positively_regulates,</em> and <em>negatively_regulates</em>. For example, a search for &quot;cytokinesis&quot; will include genes annotated to &quot;regulation of cytokinesis&quot;. (See the GO documentation on <a href="http://geneontology.org/page/ontology-structure">Ontology Structure</a> and <a href="http://geneontology.org/page/ontology-relations">Ontology Relations</a> for more information.)</p>
<p><em>S. pombe</em> GO annotations are also available in browsers that use the GO repository, notably <a href="http://amigo.geneontology.org/">AmiGO</a> and <a href="http://www.ebi.ac.uk/QuickGO/">QuickGO</a>. Both browsers have extensive documentation available:</p>
<ul>
<li><a href="http://wiki.geneontology.org/index.php/AmiGO_2_Manual:_Overview">AmiGO Manual</a></li>
<li><a href="http://www.ebi.ac.uk/QuickGO/help.html">QuickGO Help page</a></li>
</ul>
<p>Hint: to find <em>S. pombe</em> annotations, use Organism: Schizosaccharomyces pombe, Taxon: 4896 (<em>Schizosaccharomyces pombe</em>) or Source: PomBase. You can download the results in GAF format.</p>
<p>In PomBase, GO term names and IDs on gene pages link to ontology term pages for GO terms, which in turn offer links to AmiGO, QuickGO and BioPortal.</p>

</div>
<div *ngIf="pageName == 'orthology' || pageName == 'finding-data' || pageName == 'how-can-i-search-s.-cerevisiae-ortholog-s-homolog-s-s.-pombe-gene'">
<h3 id="how-can-i-search-s.-cerevisiae-ortholog-s-homolog-s-s.-pombe-gene">How can I search for the <em>S. cerevisiae</em> ortholog(s)/homolog(s) of an <em>S. pombe</em> gene?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p>In PomBase, <em>S. cerevisiae</em> orthologs are curated for <em>S. pombe</em> genes as described in the <a routerLink="/documentation/orthologs">Orthologs documentation</a>.</p>
<p>To find <em>S. pombe</em> orthologs for a budding yeast gene, you can search for the systematic name (ORF name) of the <em>S. cerevisiae</em> gene in the simple search box in the page header. For example, <em>S. cerevisiae</em> <em>LRP1</em> has the systematic name YHR081W, and a search on this in PomBase will retrieve the <em>S. pombe</em> gene <em>cti1</em>. Note that only systematic names can be searched for <em>S. cerevisiae</em>, to avoid confusion in cases where unrelated genes coincidentally have the same name in <em>S. pombe</em> and <em>S. cerevisiae</em>. To find systematic names of <em>S. cerevisiae</em> genes, you can search <a href="https://www.yeastgenome.org/locus">SGD</a>.</p>
<p>Also see the <a routerLink="/faq/how-do-i-receive-updates-to-the-list-of-s-cerevisiae-and-s-pombe-orthologs">FAQ on downloading the full set of orthologs</a>.</p>

</div>
<div *ngIf="pageName == 'orthology' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'how-can-i-see-nucleotide-level-similarity-between-s.-pombe-and-other-schizosaccharomyces-species'">
<h3 id="how-can-i-see-nucleotide-level-similarity-between-s.-pombe-and-other-schizosaccharomyces-species">How can I see nucleotide-level similarity between <em>S. pombe</em> and other <em>Schizosaccharomyces</em> species?</h3>
<!-- pombase_categories: Orthology,Finding data,Tools and resources -->
<p>PomBase offers two ways to view nucleotide-level similarity between <em>S. pombe</em> and <em>S. japonicus</em>, <em>S. octosporus</em>, or <em>S. cryophilus</em>. Both use the genome browser.</p>
<ol style="list-style-type: decimal">
<li>To view nucleotide similarity data tracks in the browser, follow the usual steps as described in the <a routerLink="/faq/how-can-i-show-or-hide-tracks-in-the-genome-browser">data track FAQ</a>. Select the data type &quot;Comparative Genomics&quot;.</li>
<li>Display syntenic regions as follows:
<ol style="list-style-type: decimal">
<li>Go to your region of interest in the browser (e.g. follow the link from a gene page or <a routerLink="/faq/how-can-i-display-a-sequence-region-using-sequence-coordinates-in-the-genome-browser">use sequence coordinates</a>). Make sure the &quot;Location&quot; tab is selected in the horizontal set of tabs along the top.</li>
<li>In the left-hand menu, find the &quot;Comparative Genomics&quot; heading, and click on &quot;Region Comparison&quot;.</li>
<li>To select a species for comparison, go to the bottom of the left-hand menu, and click the &quot;Select species or regions&quot; link (it may appear to be subtly blinking; we apologise for this anomaly).</li>
<li>In the popup, click the &quot;+&quot; beside any species in the &quot;Unselected species or regions&quot; list to move it to the &quot;Selected species or regions&quot; list. Note: &quot;lastz&quot; is the nucleotide alignment algorithm used. Close the popup - click the tick/check mark in the upper right corner, or click outside the popup.</li>
<li>Synteny views will now be visible in the bottom-most graphical display (scroll down if necessary). For any region in the <em>S. pombe</em> genome, pink tracks show the region in the second genome with the best nucleotide alignment. Green bands connect the best-aligned regions to highlight synteny.</li>
</ol></li>
</ol>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'genome-browser' || pageName == 'how-can-i-show-or-hide-tracks-genome-browser'">
<h3 id="how-can-i-show-or-hide-tracks-genome-browser">How can I show or hide tracks in the genome browser?</h3>
<!-- pombase_categories: Finding data,Genome Browser -->
<p>In the Ensembl genome browser, click the &quot;Configure this page&quot; button in the left-hand bar. A pop-up box will appear. Note that this box has several tabs along its top, and the exact selection of tabs and configuration options depends on whether you are configuring the &quot;Location&quot;, &quot;Gene&quot;, or &quot;Transcript&quot; tab of the main browser.</p>
<p>To turn a track on or off, click the small box to the left of its description. Note that some tracks simply toggle on and off, whereas for others a small popup appears, in which you can select from a set of options controlling exactly how the track appears. The left-hand bar of the configuration popup organizes available tracks into subsets, and offers a few additional options (including &quot;Reset configuration&quot;, which restores the default display).</p>
<p>For example, to show or hide repeat regions, make sure you have the &quot;Location&quot; tab selected. The tabs for this configuration allow you to configure the &quot;Region&quot; (lower) and &quot;Overview&quot; (lower) images separately. In the &quot;Configure Region Image&quot; tab, click &quot;Repeat regions&quot; in the popup's left-hand bar. You can then check one box to show all repeats, or select specific types of repeat to display.</p>
<p>When you are finished choosing tracks, click the tick/check mark in the upper right corner of the configuration popup.</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'how-can-i-submit-high-throughput-data-pombase'">
<h3 id="how-can-i-submit-high-throughput-data-pombase">How can I submit high-throughput data to PomBase?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>PomBase welcomes large sets of published data. The recommended submission route depends on the data type:</p>
<ul>
<li>For any data that can be associated with genome sequence coordinates (e.g. gene expression, ChIP-seq protein localisation, variation, etc.), please use the <a routerLink="/submit-data/data-submission-form">data submission form for HTP sequence-linked data</a>.</li>
<li>Several types of data associated with genes can be displayed on gene pages. PomBase has developed bulk upload formats for phenotype, modification, and gene expression data -- see the <a routerLink="/faq/what-file-formats-can-i-use-to-submit-high-throughput-data">FAQ on file formats</a> for links to the file format descriptions and data submission forms.</li>
</ul>
<p>If you have any other type of large-scale data -- or if you have problems or questions regarding the available submission forms -- please contact the <a routerLink="/mailto:helpdesk@pombase.org">helpdesk</a>.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'how-can-i-use-go-slims-s.-pombe'">
<h3 id="how-can-i-use-go-slims-s.-pombe">How can I use GO slims with S. pombe?</h3>
<!-- pombase_categories: Genome statistics and lists,Tools and resources,Using ontologies -->
<p>The <a routerLink="/browse-curation/fission-yeast-go-slim-terms">Fission Yeast GO slim terms</a> page provides a generic GO biological process slim for <em>S. pombe</em>, and shows total genes annotated to each term directly or to any of its descendants.</p>
<p>If you want GO slim annotations for your own list of <em>S. pombe</em> genes, we recommend the <a href="http://go.princeton.edu/cgi-bin/GOTermMapper">GO Term Mapper</a> at Princeton. Upload your list of genes, and select &quot;PomBase (S. pombe GOslim) (Process only)&quot; from the &quot;Choose GO slim&quot; pulldown. GO Term Mapper's interface and documentation should make the rest straightforward, but let PomBase staff know if you have any problems.</p>
<p>For further information on using the generic <em>S. pombe</em> slim, or on creating your own GO slim, please see the <a routerLink="/browse-curation/fission-yeast-go-slimming-tips">Fission Yeast GO slimming tips</a> page.</p>

</div>
<div *ngIf="pageName == 'tools-and-resources' || pageName == 'how-do-i-cite-data-pombase-how-do-i-cite-genome-sequence-comparison-data-etc.'">
<h3 id="how-do-i-cite-data-pombase-how-do-i-cite-genome-sequence-comparison-data-etc.">How do I cite data from PomBase? How do I cite the genome sequence, comparison data, etc.?</h3>
<!-- pombase_categories: Tools and resources -->
<p>See the <a routerLink="/about/citing-pombase">Citing PomBase</a> page, which lists papers to cite for PomBase, the <em>S. pombe</em> genome sequence, FYPO, annotations and Compara. Additional key papers may be added as needed.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'orthology' || pageName == 'how-do-i-receive-updates-list-s.-cerevisiae-and-s.-pombe-orthologs'">
<h3 id="how-do-i-receive-updates-list-s.-cerevisiae-and-s.-pombe-orthologs">How do I receive updates to the list of <em>S. cerevisiae</em> and <em>S. pombe</em> orthologs?</h3>
<!-- pombase_categories: Genome statistics and lists,Orthology -->
<p>The current version of the manually curated list of orthologs and orthologous groups identified between fission and budding yeast is available for download from the <a routerLink="/downloads/manually-curated-orthologs">Orthologs</a> page (linked from the <a routerLink="/downloads/datasets">Datasets</a> page).</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'finding-data' || pageName == 'how-do-you-determine-gene-s-full-length-transcript-utr-coordinates-transcription-start-and-end-sites'">
<h3 id="how-do-you-determine-gene-s-full-length-transcript-utr-coordinates-transcription-start-and-end-sites">How do you determine a gene's full-length transcript? / UTR coordinates? / transcription start and end sites?</h3>
<!-- pombase_categories: Genome statistics and lists,Finding data -->
<p>PomBase curators have chosen default UTR features using three data sources and a set of precedence criteria:</p>
<ol style="list-style-type: decimal">
<li>Highest priority is given to data from low-throughput &quot;conventional&quot; experiments performed on individual mRNAs and reported in publications or submitted to EMBL. Where low-throughput data are not available, one of three high-throughput datasets is used.</li>
<li>The Broad data published in 2011 by Rhind <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>) is given precedence because it is the most recent, is higher resolution and detected splicing within the UTRs. <strong>Note</strong>: This study used a &quot;greedy&quot; algorithm to determine the longest possible transcript from transcriptome reads, which may result in the prediction of longer UTRs than are actually present. Use these data with caution, and refer to the transcript profiling data in the genome browser for genes of interest.</li>
<li>For genes not covered by (1) or (2), start/end data from Lantermann <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=20118936">PMID:20118936</a>) based on transcriptome data from Dutrow <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a>) are used where available.</li>
<li>For genes not covered by (1), (2) or (3), we use data from Wilhelm <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed/18488015">PMID:18488015</a>).</li>
</ol>
<p>More information is available in the mailing list archive for two HTP datasets (Broad: http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000856.html ; Lanterman/Dutrow: http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000814.html).</p>
<p>Transcript start and end coordinates from all sources will be available as individual data tracks in the Ensembl genome browser in the near future, which will allow you to view and evaluate them. PomBase will also curate splice and transcript variants as data become available.</p>
<!-- remove this when ticket #350 done -->
<p>Note: exon and CDS coordinates are temporarily unavailable on PomBase gene pages.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'how-many-conserved-unstudied-proteins-are-there-how-list-generated'">
<h3 id="how-many-conserved-unstudied-proteins-are-there-how-list-generated">How many conserved unstudied proteins are there? How is the list generated?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>As genes are annotated, each is assigned a status, as described on the <a routerLink="/status/gene-characterisation">Gene characterisation status</a> page. Taxonomic conservation of a gene is assigned manually on a case-by-case basis, taking into account multiple criteria. Additional information is available from PomBase curators upon request.</p>
<p>Genes listed on the <a routerLink="/status/priority-unstudied-genes">Priority unstudied genes</a> page are those that have &quot;conserved unknown&quot; characterisation status and the &quot;conserved in vertebrates&quot; taxonomic distribution.</p>
<p>You can also use the <a routerLink="/query">advanced search</a> to find conserved unstudied genes as described in the FAQs on <a routerLink="/faq/what-does-the-characterisation-status-mean-for-a-gene">characterisation status</a> and <a routerLink="/faq/can-i-search-for-genes-based-on-conservation-in-different-taxa">taxonomic conservation</a>. Start by searching for Characterisation status &quot;conserved unknown&quot;, and refine the search by adding a Taxonomic conservation query if you wish.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'i-found-discrepancy-between-genbank-embl-ena-or-ddbj-entry-and-sequence-pombase.-what-should-i-do'">
<h3 id="i-found-discrepancy-between-genbank-embl-ena-or-ddbj-entry-and-sequence-pombase.-what-should-i-do">I found a discrepancy between a GenBank (EMBL/ENA or DDBJ) entry and a sequence in Pombase. What should I do?</h3>
<!-- pombase_categories: Finding data -->
<p>We recommend using only the genome sequence, either from PomBase <a routerLink="/downloads/genome-datasets">downloadable files</a> or from the <a routerLink="/faq/how-can-i-retrieve-a-gene-sequence-including-upstream-and-downstream-sequences">sequence retrieval tools</a> on the gene pages and in the genome browser. Although there are some sequence updates still pending, the genome sequence is more accurate than individual gene sequences that predate the genome.</p>
<p>Many older <em>S. pombe</em> sequence submissions to the DNA databases (International Nucleotide Sequence Database Collaboration databases, i.e. ENA, GenBank, DDBJ) contain one or more errors (sometimes with an error rate as high as 20%), and we do not have the resources to maintain past sequences or flag every error in PomBase.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 's.-pombe-transcriptome-available-fasta-format'">
<h3 id="s.-pombe-transcriptome-available-fasta-format">Is the <em>S. pombe</em> transcriptome available in FASTA format?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>There is no single transcriptome sequence file available from PomBase at present. Several transcriptomic data sets are available as <a routerLink="/faq/how-can-i-show-or-hide-tracks-in-the-genome-browser">tracks</a> in the PomBase <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Info/Index">genome browser</a>. The GFF3 genome feature files available from the <a routerLink="/downloads/genome-datasets">Genome Datasets</a> page include the coordinates of the <a routerLink="/faq/how-do-you-determine-a-gene-s-full-length-transcript-utr-coordinates-transcription-start-and-end-sites">annotated full-length transcript features</a>.</p>
<p>The bioinformatically inclined can also use the Ensembl Genomes REST API to retrieve transcript feature coordinates. The <a routerLink="/faq/there-any-programmatic-access-pombase-data">FAQ on programmatic access to PomBase</a> provides an introduction to using the API, some pombe-specific examples, and links to additional documentation.</p>
<ul>
<li>For transcript coordinates, first retrieve all stable IDs using this URL:
<ul>
<li>http://rest.ensemblgenomes.org/lookup/genome/schizosaccharomyces_pombe?content-type=application/json</li>
</ul></li>
<li>For each ID in the list, retrieve details via URLs in this format:
<ul>
<li>http://rest.ensemblgenomes.org/lookup/id/SPBC11B10.09?content-type=application/json;expand=1;</li>
</ul></li>
<li>Use your preferred scripting method to iterate through the ID list and parse the desired coordinates from the data obtained via the second URL.</li>
</ul>
<p>The Broad Institute has <a href="http://www.broadinstitute.org/ftp/pub/annotation/fungi/schizosaccharomyces/">archived genomic data files</a> for the <em>Schizosaccharomyces</em> species, including transcript files.</p>

</div>
<div *ngIf="pageName == 'tools-and-resources' || pageName == 'there-any-programmatic-access-pombase-data'">
<h3 id="there-any-programmatic-access-pombase-data">Is there any programmatic access to PomBase data?</h3>
<!-- pombase_categories: Tools and resources -->
<p>The Ensembl Genomes REST API Endpoints page provides a REST-ful interface allows language-independent programmatic access to all genomes accessible through Ensembl Genomes, including the same Schizosaccharomyces pombe genome sequence data available in PomBase. (Note that, while the sequence has not changed recently, annotation data are likely to be out of date relative to the PomBase web site Ensembl Genomes is updated much less frequently than PomBase.) The REST interface provides data in a variety of formats including GFF3, FASTA and JSON. Data types accessible via this interface include:</p>
<ul>
<li>genomic features, including genes and CDSs</li>
<li>genomic and protein sequences</li>
<li>cross-references including ontologies</li>
<li>gene trees and orthologues</li>
</ul>
<p>In addition, the interface also provides access to the Variant Effect Predictor tool and a tool for mapping genomic coordinates between different versions of genome assemblies.</p>
<p>The <a href="https://github.com/Ensembl/ensembl-rest/wiki">user guide</a> provides comprehensive descriptions of interface functionality, plus examples using a variety of languages and interfaces. The following URLs are examples specific to <em>S. pombe</em>:</p>
<ul>
<li>retrieve the cDNA sequence of the specified gene in FASTA: http://rest.ensemblgenomes.org/sequence/id/SPAC2F7.03c?content-type=text/x-fasta;type=cdna</li>
<li>retrieve list of homologues from the pan-taxonomic comparative database for the specified gene in JSON format: http://rest.ensemblgenomes.org/homology/id/SPAC2F7.03c?content-type=application/json&amp;format=condensed&amp;compara=pan_homology</li>
</ul>

</div>
<div *ngIf="pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'there-any-rdna-repeat-sequences-pombase'">
<h3 id="there-any-rdna-repeat-sequences-pombase">Are there any rDNA repeat sequences in PomBase?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>The reference genome sequence excludes most of the ribosomal DNA (rDNA) repeats, which are present in two tandem arrays on chromosome III. These arrays are estimated to be 1225 kb and 240 kb in size for the sequenced strain (972 h-). The reference sequence includes two partial and one complete representative rDNA repeats:</p>
<ul>
<li>At the left-hand end a partial repeat contains <a routerLink="/gene/SPRRNA.42">SPRRNA.42</a> (partial 18S gene)</li>
<li>A complete repeat contains <a routerLink="/gene/SPRRNA.47">SPRRNA.47</a> (28S), <a routerLink="/gene/SPRRNA.50">SPRRNA.50</a> (5.8S), and <a routerLink="/gene/SPRRNA.43">SPRRNA.43</a> (18S)</li>
<li>A partial repeat at the right-hand end contains <a routerLink="/gene/SPRRNA.45">SPRRNA.45</a></li>
</ul>
<p>The complete repeat sequence coordinates are <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=III%3A5542-13722">Chromosome 3:5542-13722</a> (note that the reverse strand is transcribed). The link goes to the PomBase Genome Browser, where you can view and <a routerLink="/faq/how-can-i-retrieve-sequence-region-using-sequence-coordinates">download</a> the sequence. Because the reverse strand is transcribed, you may want to choose &quot;-1&quot; in the location settings.</p>
<p>Also see the FAQ on <a routerLink="/faq/how-can-i-find-rrna-genes">finding rRNA genes</a>.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'there-equivalent-artemis-java-applet-pombase'">
<h3 id="there-equivalent-artemis-java-applet-pombase">Is there an equivalent to the Artemis java applet in PomBase?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>No; at present only the genome browser is available via the PomBase web site. (In the future we may investigate the possibility of adding an Artemis applet to PomBase, and will update this FAQ accordingly when it becomes available.)</p>
<p>If you want to browse the <em>S. pombe</em> genome in the Artemis environment, it is fairly easy to download and run locally:</p>
<ul>
<li>Artemis is available from http://www.sanger.ac.uk/science/tools/artemis</li>
<li>Read in the chromosome contig files available from the <a routerLink="/downloads/genome-datasets">Genome feature annotations</a> page. (You can read in one or more files in a session.)</li>
</ul>
<p>Once you have loaded the file(s), you can do many different things, e.g.:</p>
<ul>
<li>Find features by name or ID</li>
<li>Find all features of a given type (e.g. see the &quot;<a routerLink="/faq/how-can-i-find-transposons-in-the-s-pombe-genome">can I find transposons</a>&quot; FAQ)</li>
<li>Find matches to a specific nucleotide sequence (e.g. see the &quot;<a routerLink="/faq/can-i-generate-a-comprehensive-restriction-enzyme-map-of-the-genome-in-pombase">restriction enzyme map</a>&quot; FAQ)</li>
<li>View the nucleotide or amino acid sequence of a region or feature</li>
<li>Export selected sequences</li>
</ul>
<p>See the <a href="ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf">Artemis manual</a> (pdf; Sanger site) for additional information.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'finding-data' || pageName == 'there-list-drug-targets-s.-pombe'">
<h3 id="there-list-drug-targets-s.-pombe">Is there a list of drug targets in <em>S. pombe</em>?</h3>
<!-- pombase_categories: Genome statistics and lists,Finding data -->
<p>The &quot;<a routerLink="/browse-curation/drugs-known-pombe-targets">Drugs with knowns S. pombe targets</a>&quot; page lists drugs that have been shown to affect <em>S. pombe</em>, with brief summaries of their targets.</p>
<p>If you notice any errors or omissions on this page, or can provide any supporting references, please email the helpdesk.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'using-ontologies' || pageName == 'there-list-protein-complexes-s.-pombe-and-their-subunits'">
<h3 id="there-list-protein-complexes-s.-pombe-and-their-subunits">Is there a list of protein complexes in <em>S. pombe</em>, and their subunits?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists,Using ontologies -->
<p>Yes, there is a file that lists GO macromolecular complex assignments for fission yeast gene products in the FTP directory:</p>
<p>ftp://ftp.pombase.org/pombe/annotations/GO_complexes/</p>
<p>Note that the complex inventory includes the RNA subunits of ribonucleoprotein complexes. There is some redundancy in the list, because some gene products are annotated to both complexes and subcomplexes. For example, subunits of the DASH complex (GO:0042729) are annotated to 'condensed chromosome outer kinetochore' (GO:0000940) as well as GO:0042729. Additional notes are available in a README file: ftp://ftp.pombase.org/pombe/annotations/GO_complexes/README</p>
<p>Also see the <a routerLink="/faq/how-can-i-find-protein-localization-data">FAQ on localization</a>.</p>

</div>
<div *ngIf="pageName == 'using-ontologies' || pageName == 'what-annotation-extension'">
<h3 id="what-annotation-extension">What is an annotation extension?</h3>
<!-- pombase_categories: Using ontologies -->
<p>Annotation extensions can be used with annotations to terms from various ontologies, such as GO, FYPO, modifications, etc. Extensions provide additional specificity to the annotation by linking the term to another ontology term or a gene product via a relationship.</p>
<p>Extensions are most commonly used with GO annotations, where they can be used to capture details such as substrates of molecular functions or cell cycle phases during which a localization is observed. More information is available in the <a routerLink="/documentation/gene-page-gene-ontology">gene page GO annotation documentation</a>. The GO Consortium provides further information on annotation extensions in its <a href="http://geneontology.org/page/annotation-extension">annotation documentation</a>, including the <a href="http://www.geneontology.org/page/go-annotation-file-format-20">file format guide</a>, on a <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">wiki page</a>, and in a <a href="http://www.biomedcentral.com/1471-2105/15/155/abstract">publication</a>. PomBase converts many extension names to more human-friendly text, as described <a routerLink="/documentation/annotation-extension-relation-display">here</a>.</p>
<p>Phenotype annotations using FYPO may have extensions that capture expressivity (severity) or penetrance, or identify a gene or gene product used in an assay, as described in the <a routerLink="/documentation/gene-page-phenotypes">gene page phenotype documentation</a>.</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'what-bam-format'">
<h3 id="what-bam-format">What is BAM format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>BAM is a binary file format used for nucleotide sequence alignment data.</p>
<p>The <a href="http://samtools.github.io/hts-specs/SAMv1.pdf">file format specification</a> (PDF) is available from the <a href="http://samtools.sourceforge.net/">SAMtools</a> web site.</p>
<p>The <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format5.1">UCSC Genome Bioinformatics FAQ</a> and the <a href="http://www.broadinstitute.org/software/igv/BAM">Broad Institute file format guide</a> provide additional information.</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'what-bed-format'">
<h3 id="what-bed-format">What is BED format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>BED is a tab-delimited text format that defines a feature track for a genome browser.</p>
<p>BED format is described in the <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format1">UCSC Genome Bioinformatics FAQ</a>, and the <a href="http://www.broadinstitute.org/software/igv/BED">Broad Institute file format guide</a> provides additional information.</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'what-bedgraph-format'">
<h3 id="what-bedgraph-format">What is bedGraph format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>BedGraph is a file format that allows display of continuous-valued data in a genome browser track.</p>
<p>BedGraph format is described at the <a href="http://genome.ucsc.edu/goldenPath/help/bedgraph.html">UCSC Genome Bioinformatics web site</a>, and the <a href="http://www.broadinstitute.org/software/igv/bedgraph">Broad Institute file format guide</a> provides additional information.</p>
<p>If you have strand-specific data that can be represented in bedGraph format, we recommend submitting two bedGraph files, one per strand.</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'what-bigbed-format'">
<h3 id="what-bigbed-format">What is bigBed format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>BigBed is a binary file format that is created by conversion from <a routerLink="/faq/what-is-bed-format">BED</a>, and thus stores similar types of data for display in a genome browser track.</p>
<p>BigBed format is described at the <a href="http://genome.ucsc.edu/goldenPath/help/bigBed.html">UCSC Genome Bioinformatics web site</a>, and the <a href="http://www.broadinstitute.org/software/igv/bigbed">Broad Institute file format guide</a> provides additional information.</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'what-bigwig-format'">
<h3 id="what-bigwig-format">What is bigWig format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>BigWig is a file format for display of dense, continuous data in a genome browser track, created by conversion from Wiggle ( <a routerLink="/faq/what-is-wig-format">WIG</a>) format.</p>
<p>BigWig format is described at the <a href="http://genome.ucsc.edu/goldenPath/help/bigWig.html">UCSC Genome Bioinformatics web site</a>, and the <a href="http://www.broadinstitute.org/software/igv/bigwig">Broad Institute file format guide</a> provides additional information.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'what-does-characterisation-status-mean-gene'">
<h3 id="what-does-characterisation-status-mean-gene">What does the &quot;characterisation status&quot; mean for a gene?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>Each gene is assigned exactly one characterisation status that reflects how much is known about the gene, whether it is conserved, etc. Specific status descriptions:</p>
<ul>
<li><strong>Published</strong>: Completely or partially characterised in a small scale experiment, with some published information about the biological role (corresponding to any of the <a routerLink="/browse-curation/fission-yeast-go-slim-terms">fission yeast GO slim</a> biological process terms)</li>
<li><strong>Biological role inferred</strong>: A biological role (as above, a <a routerLink="/browse-curation/fission-yeast-go-slim-terms">fission yeast GO slim</a> term) is inferred from homology to an experimentally characterised gene product</li>
<li><strong>Conserved unknown</strong>: Conserved outside the <em>Schizosaccharomyces</em>, but nothing known about the biological role in any organism</li>
<li><strong><em>Schizosaccharomyces</em> specific protein, uncharacterised</strong>: Unpublished and found only in fission yeast (<em>S. pombe</em>, <em>S. octosporus</em>, <em>S. japonicus</em>, <em>S. cryophilus</em>); nothing known about biological role. May be single copy or a member of a multi-member family.</li>
<li><strong><em>S. pombe</em> specific protein, uncharacterised</strong>: Unpublished and found only in <em>S. pombe</em> (not detected in other <em>Schizosaccharomyces</em> species); nothing known about biological role</li>
<li><strong>Transposon</strong>: Known or predicted transposable element</li>
<li><strong>Dubious</strong>: Unlikely to be protein coding</li>
</ul>
<p>A current summary of <a routerLink="/status/gene-characterisation">gene characterisation status for the <em>S. pombe</em> genome</a> is available, as well as a table of <a routerLink="/status/gene-characterisation-statistics-history">historical characterisation status</a> counts.</p>
<p>You can also retrieve current lists of genes with each characterisation status using the <a routerLink="/query">advanced search</a>. Select the Characterisation status query, then choose a status from the pulldown menu, and submit.</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'what-file-formats-can-i-use-submit-high-throughput-data'">
<h3 id="what-file-formats-can-i-use-submit-high-throughput-data">What file formats can I use to submit high-throughput data?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>At present, PomBase can host any types of data that can be connected with sequence features or coordinates, and can display the data as tracks in the genome browser. We <a routerLink="/submit-data/data-submission-form">accept data</a> in any of several formats, listed on the genome browser documentation page on <a href="http://genomebrowser.pombase.org/info/website/upload/index.html#formats">Adding Custom Tracks to Ensembl</a>. To choose a file format for your data, consult the table below and the linked FAQs. Please <a routerLink="/mailto:helpdesk@pombase.org">consult the helpdesk</a> if you need further assistance.</p>
<table>
<thead>
<tr class="header">
<th>File format</th>
<th>Recommended for</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a routerLink="/faq/what-is-bam-format">BAM</a></td>
<td>sequence alignments, especially from high-throughput experiments</td>
</tr>
<tr class="even">
<td><a routerLink="/faq/what-is-bed-format">BED</a></td>
<td>sequence features with coordinates</td>
</tr>
<tr class="odd">
<td><a routerLink="/faq/what-is-bedgraph-format">bedGraph</a></td>
<td>values attached to genome locations/regions</td>
</tr>
<tr class="even">
<td><a routerLink="/faq/what-is-bigbed-format">bigBed</a></td>
<td>sequence features with coordinates</td>
</tr>
<tr class="odd">
<td><a routerLink="/faq/what-is-bigwig-format">bigWig</a></td>
<td>values attached to genome locations/regions</td>
</tr>
<tr class="even">
<td><a routerLink="/faq/what-is-gff3">GFF3</a></td>
<td>sequence features with coordinates</td>
</tr>
<tr class="odd">
<td><a routerLink="/faq/what-is-psl-format">PSL</a></td>
<td>sequence alignments</td>
</tr>
<tr class="even">
<td><a routerLink="/faq/what-is-vcf">VCF</a></td>
<td>structural variations, such as SNPs, insertions, deletions, or copy number variants</td>
</tr>
<tr class="odd">
<td><a routerLink="/faq/what-is-wig-format">WIG</a></td>
<td>values attached to genome locations/regions</td>
</tr>
</tbody>
</table>
<p>We can also accept batch submissions of certain types of data that appear on PomBase gene pages. For these data types, we use dedicated PomBase-specific formats as shown in the table:</p>
<table>
<thead>
<tr class="header">
<th>Data type</th>
<th>File format description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Phenotypes</td>
<td><a routerLink="/submit-data/phenotype-data-bulk-upload-format">phenotype file format</a></td>
</tr>
<tr class="even">
<td>Modifications</td>
<td><a routerLink="/submit-data/modification-bulk-upload-file-format">modification file format</a></td>
</tr>
<tr class="odd">
<td>Qualitative gene expression</td>
<td><a routerLink="/submit-data/qualitative-gene-expression-bulk-upload-file-format">qualitative gene expression file format</a></td>
</tr>
<tr class="even">
<td>Quantitative gene expression</td>
<td><a routerLink="/submit-data/quantitative-gene-expression-bulk-upload-file-format">quantitative gene expression file format</a></td>
</tr>
</tbody>
</table>
<p>We may be able to accept data in other text formats. Please enquire via the PomBase <a routerLink="/mailto:helpdesk@pombase.org">helpdesk</a> if you have any questions about your data format.</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'what-gff'">
<h3 id="what-gff">What is GFF3?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>Generic Feature Format Version 3 (GFF3) is a tab-delimited text file format used to represent genomic sequence features.</p>
<p>PomBase produces a GFF3 <a href="ftp://ftp.ensemblgenomes.org/pub/fungi/current/gff3/schizosaccharomyces_pombe/">file of <em>S. pombe</em> sequence features</a>, and accepts high-throughput data submissions in this format.</p>
<p>The <a href="http://www.sequenceontology.org/resources/gff3.html">file format specification</a> is available from the <a href="http://www.sequenceontology.org/">Sequence Ontology</a> web site, which also provides a link to validation software.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'what-go-term-enrichment-how-can-i-do-it-my-genes'">
<h3 id="what-go-term-enrichment-how-can-i-do-it-my-genes">What is GO term enrichment? How can I do it for my genes?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>&quot;GO term enrichment&quot; refers to analysing a gene list by finding GO terms that are significantly over- or under-represented among the annotations for the genes. Finding GO terms that are shared by genes in your list can help you find out what they have in common biologically.</p>
<p>PomBase does not have its own GO enrichment tool, but we recommend one, and provide a bit more information, in the <a routerLink="/faq/how-can-i-find-significant-shared-go-annotations-for-genes-in-a-list">FAQ on GO term enrichment</a>.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'what-pbo-option-advanced-search'">
<h3 id="what-pbo-option-advanced-search">What is the &quot;PBO&quot; option in the Advanced Search?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>PBO is an internal set of terms used for various PomBase annotations that do not fit into any of the other ontologies in use. PBO IDs and term names can be queried in the advanced search, and are most useful if you have noted a term or ID from a gene page. Examples include complementation annotations (e.g. cdc2 'functionally complemented by H. sapiens CDK1' PBO:0012584), disease association, and &quot;miscellaneous&quot; annotations (e.g. pom1 'forms a polar gradient' PBO:0000437).</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'what-psl-format'">
<h3 id="what-psl-format">What is PSL format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>PSL is a tab-delimited text format that represents sequence alignments.</p>
<p>PSL format is described in the <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format2">UCSC Genome Bioinformatics FAQ</a>, and the <a href="http://www.broadinstitute.org/software/igv/node/171">Broad Institute file format guide</a> provides additional information.</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'what-vcf'">
<h3 id="what-vcf">What is VCF?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>Variant Call Format (VCF) is a text file format used to describe structural variations, such as SNPs, insertions, deletions, or copy number variants.</p>
<p>The <a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">file format specification</a> is available from the <a href="http://www.1000genomes.org/">1000 Genomes</a> web site.</p>
<p>The <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format10.1">UCSC Genome Bioinformatics FAQ</a> and the <a href="http://www.broadinstitute.org/software/igv/VCF">Broad Institute file format guide</a> provide additional information.</p>

</div>
<div *ngIf="pageName == 'data-submission-and-formats' || pageName == 'what-wig-format'">
<h3 id="what-wig-format">What is WIG format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>Wiggle (WIG) is a file format for display of continuous-value data in a genome browser track.</p>
<p>BigWig format is described at the <a href="http://genome.ucsc.edu/goldenPath/help/wiggle.html">UCSC Genome Bioinformatics web site</a>, and the <a href="http://www.broadinstitute.org/software/igv/WIG">Broad Institute file format guide</a> provides additional information.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'when-was-genomic-sequence-last-updated'">
<h3 id="when-was-genomic-sequence-last-updated">When was the genomic sequence last updated?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>The reference sequence was last updated in January 2007; only feature coordinates and annotation have changed since then. See <a routerLink="/status/sequence-updates">Sequence updates</a> and <a routerLink="/status/sequence-updates-pending">Pending sequence updates</a> for more information.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'where-can-i-download-genome-sequence'">
<h3 id="where-can-i-download-genome-sequence">Where can I download the genome sequence?</h3>
<!-- pombase_categories: Finding data -->
<p>Genome sequence files can be downloaded from the <a routerLink="/downloads/genome-datasets">Genome feature annotations</a> page in several different formats.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'where-can-i-find-basic-statistics-on-s.-pombe-genome-such-as-genome-size-total-number-genes-mean-intergenic-distance-etc.'">
<h3 id="where-can-i-find-basic-statistics-on-s.-pombe-genome-such-as-genome-size-total-number-genes-mean-intergenic-distance-etc.">Where can I find basic statistics on the <em>S. pombe</em> genome, such as genome size, total number of genes, mean intergenic distance, etc.?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>A <a routerLink="/status/statistics">page of statistics</a> is available, but note that it was last updated in January 2017.</p>

</div>
<div *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'where-can-i-find-information-about-pombase-data-versions'">
<h3 id="where-can-i-find-information-about-pombase-data-versions">Where can I find information about PomBase data versions?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>PomBase annotations are updated each time the daily upload to the <a href="http://preview.pombase.org/">preview site</a> succeeds, with snapshots archived on a monthly basis. To refer to PomBase in your publications, we recommend citing the date on which you view or download data. If you download an archived snapshot, cite its date.</p>
<p>Prior to the September 2017 PomBase upgrade, periodic data releases were flagged with version numbers and release dates. Each legacy version number has two parts, of which the first is the Ensembl Genomes (EG) version and the second is the version of curated PomBase annotations (sequence features, ontology annotations, etc.). For example, PomBase version 20_39 used EG version 20 and PomBase annotation data version 39. The <a routerLink="/about/version-history">Data version history</a> page shows additional information about the versions of various data and software portions of the legacy PomBase releases.</p>

</div>
<div *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'why-are-go-annotations-different-between-pombase-and-uniprot-goa'">
<h3 id="why-are-go-annotations-different-between-pombase-and-uniprot-goa">Why are GO annotations different between PomBase and UniProt/GOA?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>The GO annotations available from PomBase (gene pages, advanced search, etc.) and the GO Consortium site (AmiGO; GO downloads) differ from those available from the UniProt GOA site (including QuickGO) for three main reasons:</p>
<ol style="list-style-type: decimal">
<li>RNA - PomBase provides GO annotations for functional RNAs (e.g. rRNA, tRNA, snRNA), but at present the UniProt GOA dataset only includes annotations for protein-coding genes.</li>
<li>Time lag - <em>S. pombe</em> GO data are updated at the same time on the PomBase and GO Consortium sites, but the UniProt GOA site may be up to a few weeks behind.</li>
<li>Filtering - PomBase does not include automated annotations that are redundant with manual annotations (contact the <a routerLink="/mailto:helpdesk@pombase.org">helpdesk</a> for further details). The GO Consortium site uses the same filtered annotation dataset as PomBase, whereas the UniProt GOA site includes the automated annotations.</li>
</ol>

</div>
<div *ngIf="pageName == 'using-ontologies' || pageName == 'why-are-some-genes-abnormal-phenotype-annotated-corresponding-go-process-while-others-are-not'">
<h3 id="why-are-some-genes-abnormal-phenotype-annotated-corresponding-go-process-while-others-are-not">Why are some genes with an abnormal phenotype annotated to the corresponding GO process while others are not?</h3>
<!-- pombase_categories: Using ontologies -->
<p>PomBase curators use GO Biological Process annotations to indicate that a gene product is directly involved in a process or its regulation. FYPO annotations indicate when a mutation in a gene causes a change in a process, but do not say whether the effect is direct or indirect.</p>
<p>Many mutant phenotypes reflect downstream effects of compromising an upstream process. In these cases, we annotate the phenotypes using FYPO terms, but do not annotate to the GO corresponding biological process term. We use &quot;regulation of biological process&quot; GO terms in cases where there is evidence for a gene playing a regulatory role in wild-type cells, but not where defects in an upstream process affect a downstream process (even though the latter is sometimes described as &quot;regulating&quot; or &quot;modulating&quot; the downstream process).</p>
<p>For example, a defect in cellular respiration may arise from mutations in genes directly involved in respiration, but also as a downstream effect of mutations in genes involved in mitochondrial translation, respiratory chain complex assembly, or ubiquinone biosynthesis. Similarly, DNA replication defects often also lead to defects in chromosome segregation; for the genes involved we annotate both replication and segregation phenotypes, but only replication in GO biological process.</p>
<p>The cell cycle offers an even more dramatic example of why we restrict usage of GO annotations. Over 750 genes can be mutated to give an elongated vegetative cell phenotype, which is traditionally interpreted as indicating that cell cycle progression is blocked in interphase. Most of these genes, however, are involved in transcription, translation, transport or splicing, and cell cycle delays seen in mutants are due to activation of cell cycle checkpoints by the abnormal processes. To annotate all 750 genes to &quot;regulation of mitotic cell cycle&quot; would obscure the genes that actually are part of the cell cycle regulatory network, greatly reducing the usefulness and precision of GO annotations.</p>
<p>Also see the <a routerLink="/faq/how-can-i-identify-all-of-the-genes-that-affect-a-process">FAQ on finding genes that affect a process</a>.</p>

</div>
<div *ngIf="pageName == 'gene-page' || pageName == 'using-ontologies' || pageName == 'why-are-some-genes-annotated-both-viable-and-inviable-phenotypes'">
<h3 id="why-are-some-genes-annotated-both-viable-and-inviable-phenotypes">Why are some genes annotated to both viable and inviable phenotypes?</h3>
<!-- pombase_categories: Gene page,Using ontologies -->
<p>One gene can be correctly annotated to both a &quot;viable&quot; term and an &quot;inviable&quot; term from FYPO, under certain circumstances:</p>
<ul>
<li>Different alleles may have different phenotypes; e.g., a deletion may be inviable, but a point mutation may be fully viable or conditionally lethal.</li>
<li>One allele may cause death under some, but not all, conditions.</li>
<li>An allele may cause only some cells in a population to die (this would be annotated using an &quot;inviable cell&quot; term, with an extension to indicate incomplete penetrance (&quot;low&quot; or &quot;medium&quot;), plus an annotation to a &quot;viable cell population&quot; term).</li>
<li>Cells that can divide for a few generations but then die are annotated as inviable, but can acquire suppressor mutations at a high enough frequency for populations to appear viable.</li>
</ul>
<p>At present, alleles cannot be queried directly in the PomBase advanced search, but the FYPO phenotype filters do allow you to retrieve annotations for all alleles, or to restrict to null expression (deletions etc.) or overexpression of the wild-type allele. Comparing results with and without the allele restrictions may help resolve apparent discrepancies.</p>
<p>Note that it not yet possible to search for specific conditions, or for penetrance, but we plan to add these features to the Advanced Search.</p>
<p>If, however, the allele and condition details are identical, annotation to both viable and inviable terms is probably an error (either one of the terms is wrong, or there are missing or incorrect details for the alleles and/or conditions). Please let us know via the <a routerLink="/mailto:helpdesk@pombase.org">helpdesk</a> if you notice any potential errors.</p>

</div>
<div *ngIf="pageName == 'gene-page' || pageName == 'genome-statistics-and-lists' || pageName == 'orthology' || pageName == 'why-do-gene-pages-only-show-curated-orthologs-human-and-s.-cerevisiae'">
<h3 id="why-do-gene-pages-only-show-curated-orthologs-human-and-s.-cerevisiae">Why do gene pages only show curated orthologs for human and <em>S. cerevisiae</em>?</h3>
<!-- pombase_categories: Gene page,Genome statistics and lists,Orthology -->
<p>In PomBase, human and <em>S. cerevisiae</em> orthologs are manually curated for <em>S. pombe</em> genes as described in the <a routerLink="/documentation/orthologs">Orthologs documentation</a>. Because manual ortholog curation is extremely time-consuming, it is not done for any species other than human and <em>S. cerevisiae</em>. For automated ortholog prediction of orthologs in other species please see the <a routerLink="/faq/how-can-i-find-s-pombe-orthologs-for-species-other-than-human-and-s-cerevisiae">relevant FAQ</a>.</p>
<p>In the future we will add a tree view of consensus orthologs in key species to the gene pages.</p>

</div>
  </div>
</div>
<div *ngIf="section == 'help'">
  <div class="docs-menu">
<h3 id="help"><a routerLink="/help">Help</a></h3>
<ul>
<li><a href="http://www.pombase.org/faq">Frequently Asked Questions (FAQ)</a></li>
<li><a routerLink="/documentation">PomBase Documentation</a></li>
<li><a routerLink="/about/contacts">Contact Curators</a></li>
</ul>

  </div>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h1 id="help">Help</h1>
<h3 id="old-frequently-asked-questions-faq"><a href="http://www.pombase.org/faq">Old Frequently Asked Questions (FAQ)</a></h3>
<h3 id="pombase-documentation"><a routerLink="/documentation">PomBase Documentation</a></h3>
<h3 id="contact-curators"><a routerLink="/about/contacts">Contact Curators</a></h3>

  </div>
  </div>
</div>
<div *ngIf="section == 'news'">
  <div class="docs-menu">
<h3 id="news"><a routerLink="/news">News</a></h3>
<ul>
<li><a simplePageScroll href="#global-transcriptional-responses-fission-yeast-environmental-stress">Global transcriptional responses of fission yeast to environmental stress</a></li>
<li><a simplePageScroll href="#schizosaccharomyces-pombe-essential-genes-pilot-study">Schizosaccharomyces pombe Essential Genes: A pilot Study</a></li>
<li><a simplePageScroll href="#the-new-fission-yeast-book-now-published">The 'new' fission yeast book is now published</a></li>
<li><a simplePageScroll href="#recent-genome-wide-surveys">Recent Genome wide surveys</a></li>
<li><a simplePageScroll href="#methods-volume-issue">Methods Volume 33 Issue 3</a></li>
<li><a simplePageScroll href="#the-third-international-fission-yeast-meeting">The Third International Fission Yeast Meeting</a></li>
<li><a simplePageScroll href="#general-repository-interaction-datasets">General Repository for Interaction Datasets</a></li>
<li><a simplePageScroll href="#second-east-coast-regional-pombe-meeting">Second East Coast Regional pombe Meeting</a></li>
<li><a simplePageScroll href="#comparative-genomics-eukaryotic-microorganisms">Comparative Genomics of Eukaryotic Microorganisms</a></li>
<li><a simplePageScroll href="#european-fission-yeast-meeting">European Fission Yeast Meeting</a></li>
<li><a simplePageScroll href="#fission-yeast-database-survey">Fission yeast database survey</a></li>
<li><a simplePageScroll href="#the-first-fission-yeast-whole-proteome-localization-study-now-published">The first fission yeast whole proteome localization study is now published</a></li>
<li><a simplePageScroll href="#yeast-special-issue-european-fission-yeast-meeting">Yeast Special Issue from the 2006 European Fission Yeast Meeting</a></li>
<li><a simplePageScroll href="#genedb-representation-fission-yeast-data-moved-contigs-chromosomes">GeneDB representation of the fission yeast data moved from contigs to chromosomes</a></li>
<li><a simplePageScroll href="#th-international-fission-yeast-meeting">4th International Fission Yeast Meeting</a></li>
<li><a simplePageScroll href="#wellcome-trust-advanced-course-genome-wide-approaches-fission-yeast">Wellcome Trust Advanced Course 'Genome-wide approaches with fission yeast</a></li>
<li><a simplePageScroll href="#baumann-and-zakian-labs-identify-telomerase-rna">Baumann and Zakian labs identify telomerase RNA</a></li>
<li><a simplePageScroll href="#the-h-mating-type-region-has-been-provided">The h- mating type region has been provided</a></li>
<li><a simplePageScroll href="#global-sequence-and-chip-study-examines-eukaryotic-transcription">Global sequence and chip study examines eukaryotic transcription</a></li>
<li><a simplePageScroll href="#s.-pombe-genedb-now-includes-deep-links-biological-general-repository-interaction-datasets-biogrid">S. pombe GeneDB now includes &quot;deep links&quot; to the Biological General Repository for Interaction Datasets (BioGRID)</a></li>
<li><a simplePageScroll href="#genedb-now-using-version-pfam-protein-family-database.">GeneDB is now using Version 23 of the Pfam protein family database.</a></li>
<li><a simplePageScroll href="#fission-yeast-ensembl-fungi">Fission yeast in Ensembl Fungi</a></li>
<li><a simplePageScroll href="#genedb-s.-pombe-now-uses-latest-update-pfam-release-.">GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0</a></li>
<li><a simplePageScroll href="#fission-yeast-one-key-organisms-reference-genomes-project">Fission yeast is one of the 12 key organisms of the reference genomes project</a></li>
<li><a simplePageScroll href="#funding-pombase">Funding for PomBase</a></li>
<li><a simplePageScroll href="#analysis-fission-yeast-deletion-publication">Analysis of Fission Yeast Deletion Publication</a></li>
<li><a simplePageScroll href="#annotated-transcription-start-and-termination-sites-fission-yeast">Annotated transcription start and termination sites for fission yeast</a></li>
<li><a simplePageScroll href="#genome-reappraisal-reveals-new-genes-and-revised-gene-structures">Genome reappraisal reveals new genes and revised gene structures</a></li>
<li><a simplePageScroll href="#schizosaccharomyces-comparative-genome-paper-published">Schizosaccharomyces Comparative Genome Paper Published</a></li>
<li><a simplePageScroll href="#genedb-s.-pombe-now-uses-latest-release-pfam-protein-family-database-.-.">GeneDB (S. pombe) now uses the latest release of the Pfam protein family database(25.0).</a></li>
<li><a simplePageScroll href="#pombase-nar-paper-published-online">PomBase NAR paper published online</a></li>
<li><a simplePageScroll href="#pombase-preview-launch">PomBase preview launch</a></li>
<li><a simplePageScroll href="#pombase-web-site-fully-live">PomBase web site fully live</a></li>
<li><a simplePageScroll href="#new-data-and-new-features-on-pombase-web-site">New data and new features on PomBase web site</a></li>
<li><a simplePageScroll href="#speed-improvements-and-new-data-on-pombase-web-site">Speed improvements and new data on PomBase web site</a></li>
<li><a simplePageScroll href="#data-update-on-pombase-web-site">Data update on PomBase web site</a></li>
<li><a simplePageScroll href="#pombe-registration-abstracts-by-mon-th-april">Pombe 2013: registration &amp; abstracts by Mon 8th April</a></li>
<li><a simplePageScroll href="#carl-singer-foundation-established">Carl Singer Foundation Established</a></li>
<li><a simplePageScroll href="#quantitative-gene-expression-data-available-pombase">Quantitative gene expression data available in PomBase</a></li>
<li><a simplePageScroll href="#genedb-s.-pombe-decommissioned">GeneDB S. pombe decommissioned</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-launches-community-curation">PomBase launches community curation</a></li>
<li><a simplePageScroll href="#pombase-website-update">PomBase website update</a></li>
<li><a simplePageScroll href="#pombelist-has-moved">pombelist has moved</a></li>
<li><a simplePageScroll href="#connecting-pombase">Connecting With PomBase</a></li>
<li><a simplePageScroll href="#send-htp-data-pombase">Send HTP data to PomBase</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-user-survey-open">PomBase User Survey open</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#new-target-gene-page-section">New &quot;Target Of&quot; gene page section</a></li>
<li><a simplePageScroll href="#pombase-survey-results-available">PomBase survey results available</a></li>
<li><a simplePageScroll href="#meeting-mini-reviews-published">2013 meeting mini-reviews published</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#human-ortholog-data-correction-coming-next-month">Human ortholog data correction coming next month</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#gene-ontology-publication-on-annotation-extensions">Gene Ontology publication on annotation extensions</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#new-compact-go-annotation-display">New compact GO annotation display</a></li>
<li><a simplePageScroll href="#esyn-network-visualizations-pombase">esyN network visualizations in PomBase</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombe-registration-now-open">Pombe 2015 registration now open</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombe-abstract-deadline-approaching">Pombe 2015 abstract deadline approaching</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombe-poster-abstract-deadline-extended">Pombe 2015 poster abstract deadline extended</a></li>
<li><a simplePageScroll href="#pombe-travel-fellowships">Pombe 2015 travel fellowships</a></li>
<li><a simplePageScroll href="#canto-downtime-new-version">Canto downtime &amp; new version</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update-multi-allele-phenotypes">PomBase data update with multi-allele phenotypes</a></li>
<li><a simplePageScroll href="#pombase-data-update-viability-summary-correction-alert">PomBase data update; viability summary correction alert</a></li>
<li><a simplePageScroll href="#new-genetics-primer-fission-yeast">New genetics primer for fission yeast</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#new-advanced-search-features">New Advanced Search features</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#our-model-organism-database-commentary">Our model organism database commentary</a></li>
<li><a simplePageScroll href="#show-your-support-database-funding">Show your support for database funding</a></li>
<li><a simplePageScroll href="#pombase-data-update">PomBase data update</a></li>
<li><a simplePageScroll href="#th-international-fission-yeast-meeting-registration-open">9th International Fission Yeast Meeting registration open</a></li>
<li><a simplePageScroll href="#th-international-fission-yeast-meeting-early-registration-closes-soon">9th International Fission Yeast Meeting - early registration closes soon</a></li>
<li><a simplePageScroll href="#new-pombase">New PomBase</a></li>
</ul>

  </div>
  <div class="docs-content">
  <div *ngIf="pageName == 'global-transcriptional-responses-fission-yeast-environmental-stress'">
<h3 id="global-transcriptional-responses-of-fission-yeast-to-environmental-stress">Global transcriptional responses of fission yeast to environmental stress</h3>
<p>Chen D, Toone WM, Mata J, Lyne R, Burns G, Kivinen K, Brazma A, Jones N, Bähler J. Mol Biol Cell. 2003 Jan;14(1):214-29. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12529438">PMID:12529438</a></p>

  </div>
  <div *ngIf="pageName == 'schizosaccharomyces-pombe-essential-genes-pilot-study'">
<h3 id="schizosaccharomyces-pombe-essential-genes-a-pilot-study">Schizosaccharomyces pombe Essential Genes: A pilot Study</h3>
<p>Decottignies A, Sanchez-Perez I, Nurse P Genome Res. 2003 Mar;13(3):399-406. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12618370">PMID:12618370</a></p>

  </div>
  <div *ngIf="pageName == 'new-fission-yeast-book-now-published'">
<h3 id="the-new-fission-yeast-book-is-now-published">The 'new' fission yeast book is now published</h3>
<p>Egel, R., Copenhagen, Denmark (Ed.) The Molecular Biology of Schizosaccharomyces pombe Genetics, Genomics and Beyond ISBN:3-540-00693-1</p>

  </div>
  <div *ngIf="pageName == 'recent-genome-wide-surveys'">
<h3 id="recent-genome-wide-surveys">Recent Genome wide surveys</h3>
<p>Correlations Between Gene Expression and Gene Conservation in Fission Yeast. Mata J, Bahler J. Genome Res. 2003 Nov 12 <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14613978">PMID:14613978</a><br />
<br />
FELINES: a utility for extracting and examining EST-defined introns and exons. Drabenstot SD et al Nucleic Acids Res. 2003 Nov 15;31(22):e141. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14602934">PMID:14602934</a><br />
<br />
Genome-wide distribution of DNA replication origins at A+T-rich islands in Schizosaccharomyces pombe. Segurado M, De Luis A, Antequera F. EMBO Rep. 2003 Nov;4(11):1048-53. Epub 2003 Oct 17. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14566325">PMID:14566325</a><br />
<br />
Retrotransposons and their recognition of pol II promoters: a comprehensive survey... Bowen NJ et al Genome Res. 2003 Sep;13(9):1984-97. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12952871">PMID:12952871</a></p>

  </div>
  <div *ngIf="pageName == 'methods-volume-33-issue-3'">
<h3 id="methods-volume-33-issue-3">Methods Volume 33 Issue 3</h3>
<p>This issue of Methods includes 11 papers for fission yeast protocols including DNA damage checkpoint assays, cell wall analysis, TAP, nuclear envelope integrity assays, GFP imaging, TS mutant creation and plasmid use and construction. See the <a href="http://www.sciencedirect.com/science/journal/10462023/33/3">Methods site</a> or the <a routerLink="/fission-yeast-papers-methods-volume-33-issue-3">Table of Contents</a> (at PomBase)<a href="/fission-yeast-papers-methods-volume-33-issue-3"></a> for details of the papers including PMIDs.</p>

  </div>
  <div *ngIf="pageName == 'third-international-fission-yeast-meeting'">
<h3 id="the-third-international-fission-yeast-meeting">The Third International Fission Yeast Meeting</h3>
<p>The meeting was held at UC San Diego on August 24-29, 2004.</p>

  </div>
  <div *ngIf="pageName == 'general-repository-interaction-datasets'">
<h3 id="general-repository-for-interaction-datasets">General Repository for Interaction Datasets</h3>
<p>A project to record published genetic and physical interactions is underway with Mike Tyers and the GRID group at Toronto.</p>

  </div>
  <div *ngIf="pageName == 'second-east-coast-regional-pombe-meeting'">
<h3 id="second-east-coast-regional-pombe-meeting">Second East Coast Regional pombe Meeting</h3>
<p>Second East Coast Regional pombe Meeting<br />
<br />
This meeting took place from November 11-13, 2005 in Miami Beach, Florida.</p>

  </div>
  <div *ngIf="pageName == 'comparative-genomics-eukaryotic-microorganisms'">
<h3 id="comparative-genomics-of-eukaryotic-microorganisms">Comparative Genomics of Eukaryotic Microorganisms</h3>
<p>Comparative Genomics of Eukaryotic Microorganisms:<br />
Eukaryotic Genome Evolution, Approaches with Yeasts and Fungi</p>
<p><br />
This conference took place from 12th-17th November 2005 in Sant Feliu de Guixols, Spain. Full details can be found <a href="http://www.esf.org/index.php?id=940">here</a>.</p>

  </div>
  <div *ngIf="pageName == 'european-fission-yeast-meeting'">
<h3 id="european-fission-yeast-meeting">European Fission Yeast Meeting</h3>
<p>The European Fission Yeast Meeting (16th-18th March 2006) and The Fission Yeast Bioinformatics workshop (15th - 16th Mar 2006) both took place at the Wellcome Trust Genome Campus in Hinxton (Cambridge, UK).</p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-database-survey'">
<h3 id="fission-yeast-database-survey">Fission yeast database survey</h3>
<p>The fission yeast database survey is now closed. You can view the survey results <a routerLink="/community/2006-fission-yeast-database-survey-results">here</a>.</p>

  </div>
  <div *ngIf="pageName == 'first-fission-yeast-whole-proteome-localization-study-now-published'">
<h3 id="the-first-fission-yeast-whole-proteome-localization-study-is-now-published">The first fission yeast whole proteome localization study is now published</h3>
<p>The first fission yeast whole proteome localization study is now published: <a href="http://www.nature.com/nbt/journal/v24/n7/abs/nbt1222.html">Matsuyama A. et al (2006): ORFeome cloning and global analysis of protein localization in the fission yeast Schizosaccharomyces pombe. Nat Biotech 24, 841-7.</a><br />
 </p>

  </div>
  <div *ngIf="pageName == 'yeast-special-issue-2006-european-fission-yeast-meeting'">
<h3 id="yeast-special-issue-from-the-2006-european-fission-yeast-meeting">Yeast Special Issue from the 2006 European Fission Yeast Meeting</h3>
<p>The October issue of the journal Yeast is a <a href="http://onlinelibrary.wiley.com/doi/10.1002/yea.v23:13/issuetoc">fission yeast special issue</a> containing 13 articles and reviews commissioned as a result of the European Fission Yeast Meeting, which are FREE to download.</p>

  </div>
  <div *ngIf="pageName == 'genedb-representation-fission-yeast-data-moved-contigs-chromosomes'">
<h3 id="genedb-representation-of-the-fission-yeast-data-moved-from-contigs-to-chromosomes">GeneDB representation of the fission yeast data moved from contigs to chromosomes</h3>
<p>GeneDB representation of the fission yeast data moved from contigs to chromosomes. See the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2007/000128.html">pombelist archive</a> for details.</p>

  </div>
  <div *ngIf="pageName == '4th-international-fission-yeast-meeting'">
<h3 id="th-international-fission-yeast-meeting">4th International Fission Yeast Meeting</h3>
<p>4th International Fission Yeast Meeting held in Copenhagen.</p>

  </div>
  <div *ngIf="pageName == 'wellcome-trust-advanced-course-genome-wide-approaches-fission-yeast'">
<h3 id="wellcome-trust-advanced-course-genome-wide-approaches-with-fission-yeast">Wellcome Trust Advanced Course 'Genome-wide approaches with fission yeast</h3>
<p>Wellcome Trust Advanced Course 'Genome-wide approaches with fission yeast' held in Hinxton.</p>

  </div>
  <div *ngIf="pageName == 'baumann-and-zakian-labs-identify-telomerase-rna'">
<h3 id="baumann-and-zakian-labs-identify-telomerase-rna">Baumann and Zakian labs identify telomerase RNA</h3>
<p>Baumann and Zakian labs identify elusive <a href="/spombe/result/SPNCRNA.214">telomerase RNA</a> (PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18157152">18157152</a> and PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18157149">18157149</a>)</p>

  </div>
  <div *ngIf="pageName == 'h-mating-type-region-has-been-provided'">
<h3 id="the-h--mating-type-region-has-been-provided">The h- mating type region has been provided</h3>
<p>The <a routerLink="/status/mating-type-region">h- mating type region</a> has been provided by Xavier Marsellach and Lorena Aguilar.</p>

  </div>
  <div *ngIf="pageName == 'global-sequence-and-chip-study-examines-eukaryotic-transcription'">
<h3 id="global-sequence-and-chip-study-examines-eukaryotic-transcription">Global sequence and chip study examines eukaryotic transcription</h3>
<p>Dynamic repertoire of the fission yeast transcriptome reveals: 94% of the genome is transcribed; extensive variation in different stages and conditions; global and condition-specific coupling between splicing efficiency and transcription; confirms the majority of introns; refines ~75 gene structures; identifies 453 new transcripts 26 of which were predicted to code for proteins.</p>

  </div>
  <div *ngIf="pageName == 's-pombe-genedb-now-includes-deep-links-biological-general-repository-interaction-datasets-biogr'">
<h3 id="s.-pombe-genedb-now-includes-deep-links-to-the-biological-general-repository-for-interaction-datasets-biogrid">S. pombe GeneDB now includes &quot;deep links&quot; to the Biological General Repository for Interaction Datasets (BioGRID)</h3>
<p><em>S. pombe</em> GeneDB now includes &quot;deep links&quot; to the Biological General Repository for Interaction Datasets (BioGRID) interaction datasets from the 'Database Cross References' section of the individual Gene Pages.</p>

  </div>
  <div *ngIf="pageName == 'genedb-now-using-version-23-pfam-protein-family-database'">
<h3 id="genedb-is-now-using-version-23-of-the-pfam-protein-family-database.">GeneDB is now using Version 23 of the Pfam protein family database.</h3>
<p>GeneDB is now using Version 23 of the Pfam protein family database. A total of 4154 (83%) S. pombe proteins now have at least one Pfam domain or family assignment (compared to 76% for S. cerevisiae), the highest percentage coverage for any eukaryote.</p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-ensembl-fungi'">
<h3 id="fission-yeast-in-ensembl-fungi">Fission yeast in Ensembl Fungi</h3>
<p>The fission yeast genome and annotation dataset is now available as part of <a href="http://fungi.ensembl.org/index.html">Ensembl Fungi</a>.</p>

  </div>
  <div *ngIf="pageName == 'genedb-s-pombe-now-uses-latest-update-pfam-release-240'">
<h3 id="genedb-s.-pombe-now-uses-the-latest-update-to-pfam-release-24.0">GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0</h3>
<p>GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0 and 88.5% of fission yeast proteins now contain a match to at least one Pfam domain (increased from 83% in version 23).</p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-one-12-key-organisms-reference-genomes-project'">
<h3 id="fission-yeast-is-one-of-the-12-key-organisms-of-the-reference-genomes-project">Fission yeast is one of the 12 key organisms of the reference genomes project</h3>
<p>Fission yeast is one of the 12 key organisms of the <a href="http://geneontology.org/GO.refgenome.shtml?all">reference genomes project</a>. The goal of this project is to completely annotate twelve reference genomes so that those annotations may be used to effectively seed the automatic annotation efforts of other genome.</p>

  </div>
  <div *ngIf="pageName == 'funding-pombase'">
<h3 id="funding-for-pombase">Funding for PomBase</h3>
<p>Funding was awarded by the Wellcome Trust for a fission yeast Model Organism Database, <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2010-March/002731.html">PomBase</a>.</p>

  </div>
  <div *ngIf="pageName == 'analysis-fission-yeast-deletion-publication'">
<h3 id="analysis-of-fission-yeast-deletion-publication">Analysis of Fission Yeast Deletion Publication</h3>
<p>The analysis of the fission yeast deletion collection is now published online in <a href="http://www.nature.com/nbt/journal/v28/n6/abs/nbt.1628.html">Nature Biotechnology</a>.</p>

  </div>
  <div *ngIf="pageName == 'annotated-transcription-start-and-termination-sites-fission-yeast'">
<h3 id="annotated-transcription-start-and-termination-sites-for-fission-yeast">Annotated transcription start and termination sites for fission yeast</h3>
<p>Further details are available on the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000814.html">pombe mailing list</a>.</p>

  </div>
  <div *ngIf="pageName == 'genome-reappraisal-reveals-new-genes-and-revised-gene-structures'">
<h3 id="genome-reappraisal-reveals-new-genes-and-revised-gene-structures">Genome reappraisal reveals new genes and revised gene structures</h3>
<p>Further information on the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000815.html">pombe mailing list</a>.</p>

  </div>
  <div *ngIf="pageName == 'schizosaccharomyces-comparative-genome-paper-published'">
<h3 id="schizosaccharomyces-comparative-genome-paper-published">Schizosaccharomyces Comparative Genome Paper Published</h3>
<p>A paper describing the major findings of the <em>Schizosaccharomyces</em> Comparative Genome Project was <a href="http://www.sciencemag.org/content/early/2011/04/20/science.1203357">published today</a> in Science Express and reported changes are included in GeneDB.</p>
<p>Further details are described in the pombe mailing list posts:</p>
<ul>
<li><a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000855.html">Schizosaccharomyces Comparative Genome Paper Published</a></li>
<li><a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000856.html">Import of the fission yeast revisions from the Broad Institute comparative genome paper into GeneDB</a>.</li>
</ul>

  </div>
  <div *ngIf="pageName == 'genedb-s-pombe-now-uses-latest-release-pfam-protein-family-database250'">
<h3 id="genedb-s.-pombe-now-uses-the-latest-release-of-the-pfam-protein-family-database25.0.">GeneDB (S. pombe) now uses the latest release of the Pfam protein family database(25.0).</h3>

  </div>
  <div *ngIf="pageName == 'pombase-nar-paper-published-online'">
<h3 id="pombase-nar-paper-published-online">PomBase NAR paper published online</h3>
<p>A paper describing <a href="http://www.pombase.org">PomBase</a> has been published online will be included in the 2012 Database Issue of <em>Nucleic Acids Research</em>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">Abstract</a> and <a href="http://nar.oxfordjournals.org/content/early/2011/10/28/nar.gkr853">open access full text</a> are available.</p>

  </div>
  <div *ngIf="pageName == 'pombase-preview-launch'">
<h3 id="pombase-preview-launch">PomBase preview launch</h3>
<p>A preview of <a href="http://www.pombase.org">PomBase</a>, the new model organism database for the fission yeast <em>Schizosaccharomyces pombe</em>, has been <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000970.html">announced to the <em>S. pombe</em> community</a> for testing and feedback. For more on PomBase, see the NAR Database Issue paper (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">PubMed abstract</a>) or <a href="/about/contacts">contact the PomBase staff</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-web-site-fully-live'">
<h3 id="pombase-web-site-fully-live">PomBase web site fully live</h3>
<p>We are pleased to announce that the PomBase web site, <a href="http://www.pombase.org">www.pombase.org</a>, is now fully live; the preview phase has ended. The site has been updated with an assortment of new features, datatypes, and bug fixes.</p>
<p>More recent data, reflecting additions and changes through March 20, 2012, are now available on gene pages and in search results.</p>
<p>The updated site features a Gene List Search that provides behavior equivalent to GeneDB's List Download. You can now type or paste lists into the Gene Systematic IDs and Gene Names filters, and use the Query History to combine a gene list search with other search options. For convenience, there is a direct link to a search page pre-configured to accept a list of systematic IDs available in the Find menu, on the Find page, and here: <a routerLink="/spombe/query/builder?filter=12">http://www.pombase.org/spombe/query/builder?filter=12</a></p>
<p>The Advanced Search also now offers:</p>
<ul>
<li>options to search GO, FYPO, and other ontologies by term name or ID;</li>
<li>autocomplete for ontology term name search;</li>
<li>ability to search for genes in a region, such as centromeres or telomeres;</li>
<li>improved organization of filter selections.</li>
</ul>
<p>We have also fixed a Sequence Download error reported by some users, so that the &quot;CDS&quot;, CDS + UTRs&quot;, and &quot;CDS + UTRs + Introns&quot; options now retrieve the correct sequences.<br />
<br />
In addition, numerous minor improvements have been made. Please send any questions or comments on the PomBase web site to us at &lt;<a href="mailto:helpdesk@pombase.org">helpdesk@pombase.org</a>&gt;.</p>

  </div>
  <div *ngIf="pageName == 'new-data-and-new-features-pombase-web-site'">
<h3 id="new-data-and-new-features-on-pombase-web-site">New data and new features on PomBase web site</h3>
<p>We are pleased to announce that we have updated both data and web site features for PomBase.</p>
<p>Most importantly, we have added new data types, and upgraded the gene pages to display them.</p>
<p>We have also added more annotations of existing data types, bringing the web site content up to September 11, 2012. The new annotations include the first contributions to come in via the new community curation system, and we thank the researchers who are participating in the initial phase of community curation.</p>
<p><strong>New annotation types:</strong></p>
<ul>
<li>Phenotype annotations now use the Fission Yeast Phenotype Ontology (FYPO), and include allele details, expression levels, and experimental conditions. With FYPO, more detailed phenotypes can be described, and links between terms for related phenotypes support improved phenotype searches.</li>
<li>Many GO annotations now include &quot;annotation extensions&quot; that provide additional specificity. For example, extensions may capture the substrate of a catalytic activity, the cell cycle phase during which a function or process occurs, or any of several other types of supporting information for the annotation. Annotation extensions are described in more detail below.</li>
</ul>
<p>You can see these new data types on many gene pages, such as <a routerLink="/spombe/result/SPBC11B10.09">cdc2</a> or <a routerLink="/spombe/result/SPBC106.10">pka1</a>.</p>
<p><strong>New web site features:</strong></p>
<ul>
<li>Annotation display - Gene page GO and phenotype displays have been revamped to show new annotation types described above.</li>
<li>Ontology term pages - Each ontology term ID now links to pages with information about the term and lists of genes annotated to it.</li>
<li>Ontology graph links - GO and phenotype annotation sections now include links to graphical ontology displays in the genome browser.</li>
<li>Sequence highlighting - Sequence download now offers an option to show colour highlighting of regions such as UTRs, introns and exons.</li>
<li>Versions - Each gene page now shows the current data version in the format PomBase:x.y, where <em>x</em> is the Ensembl Genomes (EG) version, and <em>y</em> is the Chado version. The sequence, and sequence feature locations, remain stable within any EG version, whereas annotations change with each Chado update.</li>
<li>Protein family information is now included in the Protein Features gene page section.</li>
<li>The Protein Feature section includes a link to the Pfam entry for a protein.</li>
<li>Transcript source data (e.g. for UTR coordinates) is now displayed in the Transcript Features section.</li>
<li>A Documentation page contains links to relevant portions of the Ensembl Genomes documentation. (More documentation will be added over the coming months.)</li>
</ul>
<p><strong>What are annotation extensions?</strong></p>
<p>Annotation extensions are a form of supporting data that can be added GO annotations (or other ontology annotations) to capture additional details not provided by the ontology term itself.</p>
<p>The information in GO annotation extensions encompasses several effector-target relationships, such as</p>
<ul>
<li>localisation dependencies</li>
<li>substrates of functions, e.g. targets of a protein kinase -- see the has_substrate extensions on Cdc2's &quot;protein serine/threonine kinase&quot; (GO:0004674) annotations</li>
<li>activators and inhibitors</li>
<li>regulation targets of signalling pathways or transcription factors</li>
</ul>
<p>Additional extensions describe spatial and temporal aspects of processes. For example, several S. pombe annotations now include extensions that indicate in which phase of the cell cycle a gene product is found in a cellular component or involved in a process -- see the pka1 annotations to &quot;nucleus&quot; (GO:0005634) and &quot;cytoplasm&quot; (GO:0005737).</p>
<p>You may also find the <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">GO wiki page</a> on annotation extensions informative, although it is primarily aimed at curators.</p>
<p>Annotation extensions can also be used with phenotype annotations. The most common usage of phenotype annotation extensions is to capture which gene, protein, etc. was used in an assay. For example, the sam5 (G441E) mutation of <em>pka1</em> causes nuclear accumulation of Ste11. This is represented by annotation to the ontology term &quot;nuclear protein accumulation&quot; (FYPO:0000255), with the extension &quot;assayed_using(PomBase:SPBC32C12.02)&quot;. Extensions can also indicate expressivity or penetrance for a phenotype.<br />
<br />
<br />
</p>

  </div>
  <div *ngIf="pageName == 'speed-improvements-and-new-data-pombase-web-site'">
<h3 id="speed-improvements-and-new-data-on-pombase-web-site">Speed improvements and new data on PomBase web site</h3>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through December 17, 2012, and reflects complete curation of an additional 70 papers.<br />
<br />
We have also made some improvements &quot;under the hood&quot; that should make gene page loading much faster. Please let us know if you have any problems with gene pages loading slowly or incompletely, whether or not you have reported issues in the past.<br />
<br />
We are aware that there is an intermittent problem with the &quot;Reference&quot; column display in the data tables -- sometimes a PubMed ID appears instead of an author name and year. This problem will be fixed as soon as possible. Please alert us if you notice anything else odd or wrong.</p>

  </div>
  <div *ngIf="pageName == 'data-update-pombase-web-site'">
<h3 id="data-update-on-pombase-web-site">Data update on PomBase web site</h3>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through March 6, 2013.<br />
<br />
We now expect to be able to update PomBase data every month, and will soon have an automated pipeline in place. We thank all of you for your patience during the long months when updates were infrequent.<br />
<br />
You should also see a few small improvements in the site:</p>
<ul>
<li>Ontology term pages now display the text definition for each term.</li>
<li>FASTA sequence retrieval should be quicker, and less likely to time out, for large gene lists.</li>
<li>There has been some tidying of the display of &quot;extension&quot; data for GO and phenotype annotations.</li>
</ul>
<p>Last month we noted an intermittent problem with the &quot;Reference&quot; column display in the data tables. The occurrence of this problem should now be greatly reduced, so please let us know if you see it recurring.<br />
<br />
As usual, please don't hesitate to alert us of any other problems with data or site performance, or if you have any questions.<br />
<br />
</p>

  </div>
  <div *ngIf="pageName == 'pombe-2013-registration-abstracts-mon-8th-april'">
<h3 id="pombe-2013-registration-abstracts-by-mon-8th-april">Pombe 2013: registration &amp; abstracts by Mon 8th April</h3>
<pre><code>Dear Pombe Fans,
      Please remember the imminent deadline (Monday 8th April) to register and submit abstracts for Pombe 2013: http://events.embo.org/13-pombe
      Abstracts are also required from all who have already been invited to talk.
      And do book your accommodation if you haven&#39;t yet done so.
      More details are in previous email forwarded below.
      Cheers,
      -Jürg &amp; Jacky
      From: On Behalf Of Bahler, Jurg
      Sent: 18 March 2013 17:49
      To: pombelist at sanger.ac.uk
      Subject: [Pombelist] Pombe 2013: Accommodation, registration &amp; abstracts
      Dear Pombe Afficionados,
      Only three weeks left to register and submit abstracts for Pombe 2013, by Monday 8th April: http://events.embo.org/13-pombe
      Speakers for 10 plenary talks and all workshop talks will be selected from abstracts, and there will be attractive poster prizes.
      Payment is only requested after registration, by 10th May.
      Important: if you require accommodation, please do book this real soon now. Especially the most cost-effective student accommodation (comfortable, with private bathrooms) may not be available much longer, as it will be put on general sale shortly. Both hotels and student accommodation will sell out in June, so you have to arrange it now. Information on accommodation is available here: http://events.embo.org/13-pombe/application.html
      We will provide a number of free registrations for which you can apply during online registration (a few of which are reserved for student members of The Genetics Society: you become eligible if you join them now). The meeting is also supported by the Biochemical Society, so if you are, or become, a member you can apply to them for student bursaries or, if you have been a member for at least 1 year, also for travel grants.
      We highly appreciate all the generous contributions from our sponsors so far:
      Platinum: EMBO
      Gold: Biochemical Society, Genetics Society, Formedium, Sunrise Science Products, Singer Instruments, F1000Research, PomBase/Wellcome Trust
      Silver: MDPI - Open Access Publishing, Hybrigenics, Infors, Life Technologies, Bioneer
      Bronze: Nature Communications, m2p labs, Imsol, Open Biology
      We look very much forward to welcoming you in London this June!
      All the best,
      -Jürg &amp; Jacky</code></pre>

  </div>
  <div *ngIf="pageName == 'carl-singer-foundation-established'">
<h3 id="carl-singer-foundation-established">Carl Singer Foundation Established</h3>
<p>Carl Singer, who was an integral part of the yeast research community for many years, passed away on February 8, 2013. Throughout his career, Carl supported yeast research both with his engineering expertise and with his good cheer. In tribute to Carl, the Singer family has now set up <a href="http://www.thecarlsingerfoundation.org/">The Carl Singer Foundation</a>, a charitable foundation dedicated to supporting scientific education in the field of yeast genetics. Questions about the foundation may be directed to Harry Singer at harry [at] thecarlsingerfoundation.org.<br />
<br />
Carl’s family would be happy to receive memories of Carl’s life at regards [at] singerinstruments.com.</p>
<p>H/T <a href="http://www.yeastgenome.org/carl-singer-foundation-established">SGD</a></p>

  </div>
  <div *ngIf="pageName == 'quantitative-gene-expression-data-available-pombase'">
<h3 id="quantitative-gene-expression-data-available-in-pombase">Quantitative gene expression data available in PomBase</h3>
<p>We have extended the Gene Expression section of each gene page to support the display of quantitative expression data, and are now showing data from two publications:</p>
<ul>
<li>Marguerat S, Schmidt A, Codlin S, Chen W, Aebersold R, BählerJ. 2012. &quot;Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and quiescent cells.&quot; <em>Cell</em> <strong>151:</strong>671-683.</li>
<li>Wu JQ, Pollard TD. 2005. &quot;Counting cytokinesis proteins globally and locally in fission yeast.&quot; <em>Science</em> <strong>310:</strong>310-314.</li>
</ul>
<p>We will also soon refine the display of the new expression data, and can add more datasets upon request. We thank Sam Marguerat for preparing the data from both papers for inclusion in PomBase.</p>
<p>We have also updated the PomBase site to include manual curation through April 4, 2013, and we have updated the &quot;all gene names&quot; file on the PomBase ftp site. The new file is available at<br />
<a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/allNames.tsv" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/allNames.tsv</a></p>

  </div>
  <div *ngIf="pageName == 'genedb-s-pombe-decommissioned'">
<h3 id="genedb-s.-pombe-decommissioned">GeneDB S. pombe decommissioned</h3>
<p>As of 14 May 2013, the old GeneDB database for <em>S. pombe</em> is no longer available. This resource consisted of static web pages, was not updated after March 2012, and not supported by an underlying relational database. The PomBase site fully supersedes GeneDB <em>S. pombe</em>, and provides improved infrastructure that will meet the current and future needs of the fission yeast community. Please e-mail the <a routerLink="/mailto:helpdesk@pombase.org">helpdesk</a> if you cannot find a replacement for any GeneDB functionality in PomBase.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through 13 May, 2013.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-0'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>PomBase data now includes manual curation through June 9, 2013, and represents complete annotation for 664 publications (as well as partial curation of many more). A highlight of this month's literature curation update is the addition of over 9400 phenotype annotations, representing about 95% of the phenotype data from the recently published genome-wide study of cell cycle and cell morphology (<a href="http://rsob.royalsocietypublishing.org/content/3/5/130053.abstract">Hayles <em>et al.</em> Open Biology May 2013</a>; <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23697806">PMID:23697806</a>). We have also improved the display of allele details for phenotype annotations. Other changes include better support for gene synonyms in the simple search, regular updates to the UTR data files, and a number of minor adjustments to external links in the annotation data tables and the external references section.</p>

  </div>
  <div *ngIf="pageName == 'pombase-launches-community-curation'">
<h3 id="pombase-launches-community-curation">PomBase launches community curation</h3>
<p>At the pombe 2013 conference in London, PomBase officially launched its community curation initiative, which allows researchers to contribute publication-based annotations directly to the database. PomBase curators invite lab heads by individual email to curate newly published papers, providing links to the online curation system and its documentation. Researchers can also initiate curation of any older fission yeast publication in PubMed. Community curation uses the open-source online tool <a href="http://curation.pombase.org/pombe/">Canto</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-website-update'">
<h3 id="pombase-website-update">PomBase website update</h3>
<p>We'd like to highlight a few improvements we've just made to the PomBase website. Most of the changes affect the gene pages:</p>
<ul>
<li>The basic information display at the top of each gene page is more compact.</li>
<li>For ontology annotations, the number of genes annotated is now shown, in a column labeled &quot;Count&quot; (also, changes behind the scenes involving this data mean that pages should load faster).</li>
<li>Annotation extensions for GO are displayed using human-friendly text instead of internal &quot;relation&quot; labels.</li>
<li>The Quick Links box can now be collapsed and expanded by clicking its header.</li>
<li>Display of modification annotations using PSI-MOD is improved.</li>
</ul>
<p>In addition, the Motif Search output now includes standard gene names and product descriptions. As we noted in a separate message, CDS coordinate files are once again available from the Downloads, with accurate and up-to-date data.<br />
<br />
</p>

  </div>
  <div *ngIf="pageName == 'pombelist-has-moved'">
<h3 id="pombelist-has-moved">pombelist has moved</h3>
<p>The pombe community mailing list, pombelist, has migrated from the Wellcome Trust Sanger Institute and is now hosted by EBI. The new address is <a href="mailto:pombelist@ebi.ac.uk">pombelist@ebi.ac.uk</a> (please note that the old address no longer works, and will generate an automatic notification including the new address). The link to <a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">subscribe</a> has also been updated, and the entire <a href="http://listserver.ebi.ac.uk/mailman/private/pombelist/">archive</a> is available at the new location.</p>

  </div>
  <div *ngIf="pageName == 'connecting-pombase'">
<h3 id="connecting-with-pombase">Connecting With PomBase</h3>
<p>To complement the <a routerLink="/community/mailing-lists">mailing list</a> and twitter (<a href="http://twitter.com/PomBase"><span class="citation">@PomBase</span></a>) it is now possible to follow the activities of PomBase and interact with other members of the pombe community via the new <a href="http://www.linkedin.com/groups/PomBase-5122686">LinkedIn Group</a> and <a href="https://plus.google.com/114842152049074799331">Google+</a>.</p>
<p>Links to these are also available from the front page of the PomBase.org site.</p>

  </div>
  <div *ngIf="pageName == 'send-htp-data-pombase'">
<h3 id="send-htp-data-to-pombase">Send HTP data to PomBase</h3>
<p>At the pombe 2013 meeting in London, PomBase staff received numerous requests display various published data, such as gene expression, histone modifications, etc. in the genome browser. To provide this, we now invite pombe researchers to send data: If you have published any high-throughput experiments that produced data  that can be associated with genome sequence coordinates, and thereby displayed as tracks on the PomBase genome browser, please fill out the <a href="/submit-data/data-submission-form">HTP Data Submission Form</a>. We can also accept large sets of phenotype data via the Phenotype Data Submission Form. If you have any problems or questions, contact us via the <a href="mailto:helpdesk@pombase.org">PomBase Helpdesk</a> at any time.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-1'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through August 11, 2013. We are particularly pleased to note that this update includes annotations from several dozen papers curated by the <em>S. pombe</em> community. Many thanks to all who have done, or are doing, paper curation in <a href="http://curation.pombase.org/pombe/">Canto</a>.<br />
<br />
We also have an updated version of the <em>S. pombe</em>/human ortholog table available upon <a routerLink="/mailto:helpdesk@pombase.org">request</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-user-survey-open'">
<h3 id="pombase-user-survey-open">PomBase User Survey open</h3>
<p>To guide current and future development, PomBase is now conducting a <a href="https://www.surveymonkey.com/s/NDM2BQX">user survey</a>, where we invite the fission yeast research community and any other PomBase users to evaluate the resources provided so far and comment on future priorities. The survey should take about 10 minutes to complete. Thank you for your participation!</p>
<p><a href="https://www.surveymonkey.com/s/NDM2BQX" class="uri">https://www.surveymonkey.com/s/NDM2BQX</a></p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-2'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>The PomBase web site has been updated and now includes manually curated data through October 6, 2013. The number of community-curated papers continues to increase, ensuring that PomBase gene pages contain complete and up-to-date information. We are also pleased to announce that data tracks are now available in the genome browser for data from these two publications:</p>
<ul>
<li>Woolcock KJ, Gaidatzis D, Punga T, Bühler M. 2010. Dicer associates with chromatin to repress genome activity in Schizosaccharomyces pombe. Nat Struct Mol Biol. 2011 Jan;18(1):94-9. doi: 10.1038/nsmb.1935 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21151114">PMID:21151114</a></li>
<li>Mata J. 2013. Genome-wide mapping of polyadenylation sites in fission yeast reveals widespread alternative polyadenylation. RNA Biol. 2013 Aug 1;10(8):1407-14. doi: 10.4161/rna.25758 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23900342">PMID:23900342</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'new-target-gene-page-section'">
<h3 id="new-target-of-gene-page-section">New &quot;Target Of&quot; gene page section</h3>
<p>With the October 2013 update, gene pages now include &quot;<a href="/documentation/gene-page-target">Target Of</a>&quot; annotations, which describe genes that affect the gene of interest. These annotations are essentially the reciprocal of <a href="/faqs/what-annotation-extension">ontology annotation extensions</a>. Each &quot;Target Of&quot; annotation includes a relationship that indicates how the genes are connected, the name and product of the second gene, and a reference. Genes listed under &quot;Target Of&quot; may include upstream regulators or enzymes that modify the product of the gene of interest. For example, the &quot;Target Of&quot; annotations for <a routerLink="/spombe/result/SPBC11B10.09#recipext">cdc2</a> indicate that it is a substrate of, and regulated by, the kinase Wee1 and the phosphatase Cdc25 (among others). At present, &quot;Target Of&quot; data includes annotations derived from <a href="/documentation/gene-page-gene-ontology#ann_ext">GO annotation extensions</a>. We will soon extend it to include data from <a routerLink="/documentation/gene-page-phenotypes">phenotype</a> annotation extensions.</p>

  </div>
  <div *ngIf="pageName == 'pombase-survey-results-available'">
<h3 id="pombase-survey-results-available">PomBase survey results available</h3>
<p>The 2013 PomBase user survey closed at the end of October, and the results are available <a routerLink="/community/2013-pombase-user-survey-results">here</a>. Some highlights have been sent to the pombe mailing list. Many thanks to all who completed the survey.</p>

  </div>
  <div *ngIf="pageName == '2013-meeting-mini-reviews-published'">
<h3 id="meeting-mini-reviews-published">2013 meeting mini-reviews published</h3>
<p>A series of mini-reviews, which were invited in association with the International Fission Yeast Meeting in London, have now been published in Biochemical Society Transactions: <a href="http://www.biochemsoctrans.org/bst/041/6/default.htm#c" class="uri">http://www.biochemsoctrans.org/bst/041/6/default.htm#c</a></p>
<p>(Thanks to Jürg Bahler for this item)</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-3'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through November 11, 2013. We now have future meetings available as a <a routerLink="/calendar">calendar</a> or a <a routerLink="/events">list</a>. The <a routerLink="/faq">FAQ</a> and some documentation pages have also been updated.<br />
<br />
</p>

  </div>
  <div *ngIf="pageName == 'human-ortholog-data-correction-coming-next-month'">
<h3 id="human-ortholog-data-correction-coming-next-month">Human ortholog data correction coming next month</h3>
<p>We are about to release a data update for PomBase. Please note that there is still a problem with the human orthologs, as originally described on this list in mid-December (see archived message at <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2013/003926.html" class="uri">http://listserver.ebi.ac.uk/pipermail/pombelist/2013/003926.html</a>). We will correct this problem in the next PomBase release, and apologise for any inconvenience in the meantime.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-4'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through January 10, 2014, and covers over 100 papers that have been curated in Canto by community members. We again thank all who have contributed curation via Canto.</p>
<p>We have made some improvements to the gene pages. Highlights:</p>
<ul>
<li>The Sequence section now has links to NCBI BLAST as well as Ensembl BLAST.</li>
<li>The External References section now links to the Pomb(A) polyadenylation viewer.</li>
</ul>
<p>In the genome browser, new data tracks are now available for data from these publications:</p>
<ul>
<li>Rhind N, [and many more]. 2011. Comparative functional genomics of the fission yeasts. Science 332(6032):930-6. doi: 10.1126/science.1203357. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:21511999</a></li>
<li>Schlackow M, Marguerat S, Proudfoot NJ, Bähler J, Erban R, Gullerova M. 2013. Genome-wide analysis of poly(A) site selection in <em>Schizosaccharomyces pombe</em>. RNA. 19(12):1617-31. doi:10.1261/rna.040675.113. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24152550">PMID:24152550</a></li>
<li>Soriano I, Quintales L, Antequera F. 2013. Clustered regulatory elements at nucleosome-depleted regions punctuate a constant nucleosomal landscape in Schizosaccharomyces pombe. BMC Genomics. 14:813. doi:10.1186/1471-<span class="no-break">2164-14-81</span>3. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24256300">PMID:24256300</a> (partial data;  remainder coming in the next update)</li>
<li>Xu J, Yanagisawa Y, Tsankov AM, Hart C, Aoki K, Kommajosyula N, Steinmann KE, Bochicchio J, Russ C, Regev A, Rando OJ, Nusbaum C, Niki H, Milos P, Weng Z, Rhind N. 2012. Genome-wide identification and characterization of replication origins by deep sequencing. Genome Biol. 13(4):R27. doi:10.1186/gb-2012-13-4-r27. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22531001">PMID:22531001</a></li>
</ul>
<p>Now that more data tracks are available, we have added some categories to the track configuration section to improve organization. Additional documentation is in preparation, and will be announced here when available.<br />
<br />
Genome sequences for additional <em>Schizosaccharomyces</em> species (<em>S. japonicus</em>, <em>S. octosporus</em>, and <em>S. cryophilus</em>) have recently become available in Ensembl Fungi, and the PomBase genome browser now includes comparative genomics data, with a view of region comparisons between each new genome and <em>S. pombe</em>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-5'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>Data on the PomBase web site now includes manual curation through February 24, 2014. Human orthologs that went missing from gene pages have been restored, and other small improvements have been made to gene pages. Community curation now covers over 130 publications.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-6'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through April 28, 2014. Transcriptome data from <a href="http://europepmc.org/abstract/MED/23101633">Margeurat <em>et al</em> (2012)</a> is now available as Ensembl Browser tracks.</p>
<p>Thank you to all who have done, or are doing, paper curation in Canto. Over 159 community-curated papers are now included in PomBase.</p>
<p>There are a number of routes to accelerate your data into PomBase, (either through community curation, or by supplying HTP sequence, modification or phenotype data in one of our specified formats), see <a href="http://www.pombase.org/submit-data" class="uri">http://www.pombase.org/submit-data</a> for more details.</p>
<p>As usual, please don't hesitate to alert us of any other problems with data or site performance, or if you have any questions.</p>
<p>Sincerely yours,<br />
The PomBase Staff</p>

  </div>
  <div *ngIf="pageName == 'gene-ontology-publication-annotation-extensions'">
<h3 id="gene-ontology-publication-on-annotation-extensions">Gene Ontology publication on annotation extensions</h3>
<p>PomBase was an early adopter of <a href="/documentation/gene-page-gene-ontology#ann_ext">annotation extensions</a>, which add spatial, temporal, or substrate/target details to GO annotations. The GO Consortium has now published a paper describing its implementation of annotation extensions, in which PomBase examples and its gene page display figure prominently:</p>
<p>Huntley, R.P. <em>et al.</em> (2014) A method for increasing expressivity of Gene Ontology annotations using a compositional approach. <em>BMC Bioinformatics</em> 2014, <strong>15</strong>:155. <a href="http://www.biomedcentral.com/1471-2105/15/155/abstract">doi:10.1186/1471-<span class="no-break">2105-15-15</span>5</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24885854">PMID:24885854</a></p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-7'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through June 6, 2014. In other improvements, a downloadable file of <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/FASTA/cdna_introns_utrs.fa.gz">intron sequence data</a> (FASTA format) is now available, and phenotypes are now included in the Target Of section on gene pages.</p>
<p>The gene pages also now display protein modification data from two large-scale datasets:</p>
<ul>
<li>Wilson-Grady JT, Villén J, Gygi SP. 2008 .Phosphoproteome analysis of fission yeast. <em>J Proteome Res</em>. 2008 Mar;<strong>7</strong>(3):1088-97. doi:10.1021/pr7006335. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18257517">PMID:18257517</a></li>
<li>Carpy A, Krug K, Graf S, Koch A, Popic S, Hauf S, Macek B. 2014. Absolute proteome and phosphoproteome dynamics during the cell cycle of fission yeast. <em>Mol Cell Proteomics</em>. 2014 Apr 23. [Epub ahead of print] <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24763107">PMID:24763107</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-8'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through July 8, 2014. The gene pages also now display protein modification data from an additional large-scale dataset:</p>
<p>Koch A, Krug K, Pengelley S, Macek B, Hauf S. 2011. Mitotic substrates of the kinase aurora with roles in chromatin regulation identified through quantitative phosphoproteomics of fission yeast. <em>Sci Signal</em>. <strong>4</strong>(179): rs6 doi: 10.1126/scisignal.2001588 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21712547">PMID:21712547</a><br />
<br />
We have also made corrections to some residue positions affected by sequence updates in one of the modification datasets we added last month:</p>
<p>Carpy A, Krug K, Graf S, Koch A, Popic S, Hauf S, Macek B. 2014. Absolute proteome and phosphoproteome dynamics during the cell cycle of fission yeast. <em>Mol Cell Proteomics</em>. 2014 Apr 23. [Epub ahead of print] <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24763107">PMID:24763107</a></p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-9'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through August 8, 2014. Community curation now covers over 190 papers. Gene pages now include links to the <a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/buildDetails?atlas_build_id=355"><em>S. pombe</em> PeptideAtlas</a>, a database of peptides identified in tandem mass spectrometry proteomics experiments.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-10'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through August 30, 2014. Community curation now covers over 200 papers.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-11'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through October 27, 2014, including 225 community-curated publications. The gene page Phenotype section now includes data from the high-throughput microscopy analysis of viable deletion mutants reported in:<br />
<br />
Graml V, Studera X, Lawson JL, Chessel A, Geymonat M, Bortfeld-Miller M, Walter T, Wagstaff L, Piddini E, Carazo-Salas RE. A Genomic Multiprocess Survey of Machineries that Control and Link Cell Shape, Microtubule Organization, and Cell-Cycle Progression. Dev Cell. 2014 Oct 27;31(2):227-39. doi: <a href="http://dx.doi.org/10.1016/j.devcel.2014.09.005">10.1016/j.devcel.2014.09.005</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25373780">PMID:25373780</a>. Links to the accompanying SYSGRO resource have been added to the External References section of the gene pages.</p>
<p>The genome browser now includes tracks for intron branch point data from:</p>
<p>Bitton DA, Rallis C, Jeffares DC, Smith GC, Chen YY, Codlin S, Marguerat  S, Bähler J. LaSSO, a strategy for genome-wide mapping of intronic  lariats and branch points using RNA-seq. Genome Res. 2014 Jul;24(7):1169-79. doi: <a href="http://dx.doi.org/10.1101/gr.166819.113">10.1101/gr.166819.113</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">PMID:24709818</a>.</p>
<p>We have greatly improved search results for GO and FYPO annotations: both now follow more relationship types within the ontology to retrieve genes annotated to a term. The PomBase GO search now includes the <em>regulates</em> relationships, so its search results are consistent with those in the GO Consortium's AmiGO browser. The FYPO search now uses <em>has_part,</em> <em>has_output</em>, and <em>output_of</em> as well as <em>is_a</em> and <em>part_of</em>. The Phenotype section now includes a highlighted sub-header that indicates whether a deletion mutant is viable or inviable. A file of protein complex subunits is available for download, and numerous smaller improvements have been made in the gene pages and static pages.</p>

  </div>
  <div *ngIf="pageName == 'esyn-network-visualizations-pombase'">
<h3 id="esyn-network-visualizations-in-pombase">esyN network visualizations in PomBase</h3>
<p>PomBase has implemented network visualisations for fission yeast in <a href="http://www.esyn.org/">esyN</a>, using data curated by BioGRID and PomBase. esyN is a web-based tool for building, sharing, and viewing network data developed by Dan Bean and Giorgio Favrin in the <a href="http://www.sysbiol.cam.ac.uk">Cambridge Systems Biology Centre</a>, University of Cambridge, UK.<br />
<br />
On gene pages, we have links to gene-specific interaction networks in esyN in the table headers of the Interactions sections:</p>
<ul>
<li>The Genetic Interactions section links to all interactions centred on the gene and curated in BioGRID</li>
<li>The Physical interactions section has links to two datasets:
<ul>
<li>All physical interactions curated in BioGRID for the gene</li>
<li>Interactions for the gene in the PomBase <a href="/documentation/high-confidence-physical-interaction-network">High Confidence Physical Interaction Network (HCPIN)</a></li>
</ul></li>
</ul>
<p>We also have esyN links on the <a href="/browse-curation/fission-yeast-go-slim-terms">GO Slim page</a> and on ontology term pages for GO Slim biological process terms. Each GO Slim term links to the HCPIN physical interaction network in esyN (for example, see the &quot;regulation of mitotic cell cycle&quot; <a href="http://www.esyn.org//builder.php?type=Graph&term=GO:0007346&interactionType=physical&source=pombase&includeInteractors=false">network</a>).</p>

  </div>
  <div *ngIf="pageName == 'new-compact-go-annotation-display'">
<h3 id="new-compact-go-annotation-display">New compact GO annotation display</h3>
<p>To make the Gene Ontology (GO) annotations easier to read on PomBase gene pages, we have introduced a new, streamlined display that presents just the essentials. The summary shows the term name (hyperlinked to the ontology term page), the count of genes annotated to the term, and any annotation extensions. All of the previously visible annotation details are still available -- simply click the &quot;Summary&quot; button to switch to the &quot;Full&quot; view. Or click the &quot;+&quot; and &quot;-&quot; icons to expand or collapse the annotation to a single term.<br />
 <br />
In addition, the top of the Biological Process table now lists any GO slim terms applicable to the gene.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-12'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through January 12, 2015, including 240 community-curated publications. The gene page Phenotype section now features a compact default display. A downloadable &quot;viability summary&quot; data file is now available. The PomBase BLAST server has incorporated interface changes made Ensembl-wide.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-13'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through February 2, 2015, including 245 community-curated publications. On the gene pages, the interaction tables now provides a bit of descriptive text for each annotation, indicating the nature and direction of the interaction.</p>

  </div>
  <div *ngIf="pageName == 'pombe-2015-registration-now-open'">
<h3 id="pombe-2015-registration-now-open">Pombe 2015 registration now open</h3>
<p>Registration for Pombe 2015: 8th International Fission Yeast Meeting is now open at the conference web site, <a href="https://amarys-jtb.jp/web/Pombe2015/index.html" class="uri">https://amarys-jtb.jp/web/Pombe2015/index.html</a></p>
<p>The registration deadline is 17 May 2015.</p>
<p>Thanks to Yasushi Hiraoka for this item.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-14'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through March7, 2015, including 250 community-curated publications.The autocomplete feature of the Advanced Search ontology term filter has been improved with respect to response time and relevance of suggested terms.</p>

  </div>
  <div *ngIf="pageName == 'pombe-2015-abstract-deadline-approaching'">
<h3 id="pombe-2015-abstract-deadline-approaching">Pombe 2015 abstract deadline approaching</h3>
<p>Abstracts are due on Sunday, April 19, 2015 for the <a href="https://amarys-jtb.jp/web/Pombe2015/index.html">8th International Fission Yeast Meeting</a> in Kobe, Japan. Registration will remain open until May 17, but the abstract submission deadline cannot be extended.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-15'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through April 7, 2015, including 260 community-curated publications.The Advanced Search now supports queries for proteins with a specified number of transmembrane domains.</p>

  </div>
  <div *ngIf="pageName == 'pombe-2015-poster-abstract-deadline-extended'">
<h3 id="pombe-2015-poster-abstract-deadline-extended">Pombe 2015 poster abstract deadline extended</h3>
<p>The abstract submission deadline for the <a href="https://amarys-jtb.jp/web/Pombe2015/index.html">8th International Fission Yeast Meeting</a> in Kobe, Japan has been extended until midnight Friday, April 24 for <strong>posters only</strong>. Registration is open until May 17.</p>

  </div>
  <div *ngIf="pageName == 'pombe-2015-travel-fellowships'">
<h3 id="pombe-2015-travel-fellowships">Pombe 2015 travel fellowships</h3>
<p>Applications are now being accepted for fellowships to provide financial support for students and postdocs attending the 8th International Fission Yeast Meeting in Kobe, Japan. To apply, follow the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2015/004246.html">instructions sent to the pombase mailing list</a>. The deadline is may 17, 2015 (same as the registration deadline).</p>

  </div>
  <div *ngIf="pageName == 'canto-downtime-new-version'">
<h3 id="canto-downtime-new-version">Canto downtime &amp; new version</h3>
<p>Canto, PomBase's literature curation tool, will be unavailable for approximately 3 weeks starting at 12:00 midnight UK time (BST) tonight, 27 May 2015, while we deploy an upgraded version.<br />
<br />
The upgraded Canto will feature an entirely new interface for annotating multi-allele phenotypes and the corresponding genotypes, as well as improved workflows for single-allele phenotypes, GO, etc. All existing annotations will be retained, and users can resume curation using the new and improved features in any unfinished sessions when Canto is back online.<br />
<br />
We will announce when the new version of Canto is released to the public.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-16'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through May 8, 2015, including 265 community-curated publications.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-17'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through May 26, 2015, including 270 community-curated publications. See you at Pombe 2015 in Kobe!</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-multi-allele-phenotypes'">
<h3 id="pombase-data-update-with-multi-allele-phenotypes">PomBase data update with multi-allele phenotypes</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through August 13, 2015, including 300 community-curated publications.</p>
<p>PomBase gene pages now include multi-allele phenotype annotations (i.e. phenotypes of double mutants, triple mutants, etc.). New sub-sections of the gene pages display multi-allele phenotypes at the population and individual cell level, paralleling the organisation of the single allele phenotype display. Compact and full views are available; both show phenotypes with the relevant genotypes and the alleles that make them up, and the full view adds details for evidence, expression, conditions, and references.</p>
<p>The genome browser now includes data tracks for two more publications:<br />
<br />
DNA polymerase usage from:<br />
Daigaku Y, Keszthelyi A, Müller CA, Miyabe I, Brooks T, Retkute R, Hubank M, Nieduszynski CA, Carr AM. 2015. A global profile of replicative polymerase usage. <em>Nat Struct Mol Biol.</em> 2015 Mar;<strong>22(3):</strong>192-8. doi: <a href="http://dx.doi.org/10.1038/nsmb.2962">10.1038/nsmb.2962</a> PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25664722">25664722</a><br />
<br />
Promoters and transcription start sites from:<br />
Li H, Hou J, Bai L, Hu C, Tong P, Kang Y, Zhao X, Shao Z. 2015. Genome-wide analysis of core promoter structures in Schizosaccharomyces pombe with DeepCAGE. <em>RNA Biol.</em> 2015;<strong>12(5):</strong>525-37. doi: <a href="http://dx.doi.org/10.1080/15476286.2015.1022704">10.1080/15476286.2015</a>.1022704 PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25747261">25747261</a></p>
<p>Codon adaptation index (CAI) values are now included in the Protein Properties section of the gene pages and in the downloadable PeptideStats.tsv file. A file of amino acid composition data is also available from the FTP site and the <a href="/downloads/protein-datasets">Protein Datasets</a> page.</p>
<p>The gene page section that was formerly misnamed &quot;species distribution&quot; is now called &quot;taxonomic conservation&quot;.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-viability-summary-correction-alert'">
<h3 id="pombase-data-update-viability-summary-correction-alert">PomBase data update; viability summary correction alert</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through September 6, 2015.</p>
<p>Errors in the previous FYPOviability.tsv file have been corrected, and we recommend that all users update this file, especially those who downloaded it earlier in September 2015.</p>

  </div>
  <div *ngIf="pageName == 'new-genetics-primer-fission-yeast'">
<h3 id="new-genetics-primer-for-fission-yeast">New genetics primer for fission yeast</h3>
<p>A new genetics primer, aimed at researchers interested in using fission yeast as a model system, has recently been published. The primer includes a brief history of fission yeast research, an introduction to available genetic tools, and the use of PomBase for data analysis</p>
<p>Hoffman CS, Wood V, Fantes PA. (2015) An Ancient Yeast for Young Geneticists: A Primer on the <em>Schizosaccharomyces pombe</em> Model System. <em>Genetics</em> <strong>201</strong>:403-423. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26447128">PMID:26447128</a> <a href="http://dx.doi.org/10.1534/genetics.115.181503">DOI:10.1534/genetics.115.181503</a></p>

  </div>
  <div *ngIf="pageName == 'new-advanced-search-features'">
<h3 id="new-advanced-search-features">New Advanced Search features</h3>
<p>We have introduced new features to the <a href="/spombe/query/builder">Advanced Search</a>:</p>
<ul>
<li>There are now two query reuse options: store a query in your web browser cache, or download a JSON file that can be uploaded later to re-run.</li>
<li>You can now query for genes that interact genetically or physically with a specified gene.</li>
<li>The FYPO query now offers options to choose nulls (deletions or disruptions), wild-type overexpression, or all alleles. The search results will include any genes that have an allele that matches the allele criteria and the chosen phenotype.</li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-18'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through November 9, 2015, including 340 community-curated publications.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-19'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through January 25, 2016.<br />
The genome browser includes variation data, in tracks under &quot;Variation&quot;, from natural <em>S. pombe</em> isolates, published in:<br />
<br />
Jeffares DC <em>et al.</em> 2015. The genomic and phenotypic diversity of Schizosaccharomyces pombe. <em>Nat Genet.</em> <strong>47(3):</strong> 235-241. doi:<a href="http://dx.doi.org/10.1038/ng.3215">10.1038/ng.3215</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25665008">PMID:25665008</a></p>
<p>New files are now available from the PomBase FTP site, and are linked from pages in the Download Datasets area:</p>
<ul>
<li>Non-coding RNA sequence feature coordinates (available via the <a href="/downloads/data-mappings">Data Mappings</a> page);</li>
<li>Protein features, such as domains and family assignments (available via the <a routerLink="/downloads/protein-datasets">Protein Datasets</a> page);</li>
<li>Protein modification annotations (also in Protein Datasets).</li>
</ul>
<p>The <a routerLink="/status/new-and-removed-genes">New and Removed Genes page</a> has been updated to reflect recent deletions and merges.<br />
<br />
Note: Ontology graph views are no longer available in the genome browser, so links have been removed from the GO, FYPO, and modification tables on the gene pages. For GO and FYPO, links to external ontology browsers that offer graphical views are available on the <a href="/documentation/ontology-term-page">Ontology Term pages</a>.<br />
<br />
</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-20'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through March 9, 2016.</p>
<p><strong>Important:</strong> We have corrected a problem that made erroneous interaction data and literature appear on some gene pages.</p>
<p>The gene pages now include interaction data from the Vo et al. proteome-wide study (curated by BioGRID and imported into PomBase):<br />
Vo TV <em>et al.</em> 2016. A Proteome-wide Fission Yeast Interactome Reveals Network Evolution Principles from Yeasts to Human. <em>Cell</em> <strong>164(1-2)</strong>: 310-23. doi: <a href="http://dx.doi.org/10.1016/j.cell.2015.11.037">10.1016/j.cell.2015.11.037</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26771498">PMID:26771498</a>.</p>
<p>The genome browser now includes transcriptome data published in:<br />
<br />
Eser P, Wachutka L, Maier KC, Demel C, Boroni M, Iyer S, Cramer P, Gagneur J. 2016. Determinants of RNA metabolism in the Schizosaccharomyces pombe genome. <em>Mol Syst Biol</em>. <strong>12(2):</strong> 857. doi: <a href="http://dx.doi.org/10.15252/msb.20156526">10.15252/msb.20156526</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26883383">PMID:26883383</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-21'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through April 8, 2016.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-22'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through May 12, 2016.</p>

  </div>
  <div *ngIf="pageName == 'our-model-organism-database-commentary'">
<h3 id="our-model-organism-database-commentary">Our model organism database commentary</h3>
<p>Several of the PomBase staff, joined by our advisor Sir Paul Nurse, have published a Comment in <em>BMC Biology</em> briefly describing the importance of model organism databases to the success of modern biomedical research:<br />
<br />
Oliver SG, Lock A, Harris MA, Nurse P, Wood V. 2016. Model organism databases: essential resources that need the support of both funders and users.<br />
<em>BMC Biol.</em> 2016 <strong>14(1)</strong>: 49. doi: <a href="http://dx.doi.org/10.1186/s12915-016-0276-z">10.1186/s12915-016-0276-z</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27334346">PMID:27334346</a></p>

  </div>
  <div *ngIf="pageName == 'show-your-support-database-funding'">
<h3 id="show-your-support-for-database-funding">Show your support for database funding</h3>
<p>In response to planned cuts to database funding, leading model organism researchers have prepared an open letter to NIH Director Dr. Francis Collins to demonstrate support for the independent community-focused databases that are essential to their work. Although PomBase is not directly funded by NIH, we collaborate extensively with those that are, including the GO Consortium and several model organism databases.<br />
<br />
The Genetics Society of America website where the letter can be viewed and signed is at <a href="http://www.genetics-gsa.org/MODsupport" class="uri">http://www.genetics-gsa.org/MODsupport</a><br />
<br />
Please sign the letter to add your voice in support of the databases that help make your research possible. For more information, we recommend an email that Mike Cherry sent to the GO-Friends mailing list, archived at <a href="https://mailman.stanford.edu/pipermail/go-friends/2016-June/002355.html" class="uri">https://mailman.stanford.edu/pipermail/go-friends/2016-June/002355.html</a></p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-23'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through September 11, 2016.</p>

  </div>
  <div *ngIf="pageName == '9th-international-fission-yeast-meeting-registration-open'">
<h3 id="th-international-fission-yeast-meeting-registration-open">9th International Fission Yeast Meeting registration open</h3>
<p>Registration for the 9th International Fission Yeast Meeting is now open. The meeting will be held in Banff, Canada from May 14-19, 2017. Early registration closes Dec 1, 2016! Please see our website at <a href="http://www.pombe2017.com">www.pombe2017.com</a> for details. We look forward to seeing you in Banff!<br />
<br />
- Conference Organizers: Dallan Young, Gordon Chua, Paul Young</p>

  </div>
  <div *ngIf="pageName == '9th-international-fission-yeast-meeting-early-registration-closes-soon'">
<h3 id="th-international-fission-yeast-meeting---early-registration-closes-soon">9th International Fission Yeast Meeting - early registration closes soon</h3>
<p>Reminder: early registration for the 9th International Fission Yeast Meeting in Banff closes <strong>Dec. 31, 2016</strong>. Please see the conference website at <a href="http://www.pombe2017.com">www.pombe2017.com</a> for details.</p>

  </div>
  <div *ngIf="pageName == 'new-pombase'">
<h3 id="new-pombase">New PomBase</h3>
<p>A new version of the PomBase website is under development, and will be released when it includes all features available in the current version (and many new ones). You can follow its progress via announcements on <a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">pombelist</a>, and try out the <a href="http://preview.pombase.org/">preview</a> site.</p>

  </div>
  <div *ngIf="pageName == 'index'">
<h3 id="new-pombase">New PomBase</h3>
<p><em>2017-06-01</em></p>
<p>A new version of the PomBase website is under development, and will be released when it includes all features available in the current version (and many new ones). You can follow its progress via announcements on <a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">pombelist</a>, and try out the <a href="http://preview.pombase.org/">preview</a> site.</p>
<h3 id="th-international-fission-yeast-meeting-early-registration-closes-soon">9th International Fission Yeast Meeting - early registration closes soon</h3>
<p><em>2016-12-11</em></p>
<p>Reminder: early registration for the 9th International Fission Yeast Meeting in Banff closes <strong>Dec. 31, 2016</strong>. Please see the conference website at <a href="http://www.pombe2017.com">www.pombe2017.com</a> for details.</p>
<h3 id="th-international-fission-yeast-meeting-registration-open">9th International Fission Yeast Meeting registration open</h3>
<p><em>2016-10-31</em></p>
<p>Registration for the 9th International Fission Yeast Meeting is now open. The meeting will be held in Banff, Canada from May 14-19, 2017. Early registration closes Dec 1, 2016! Please see our website at <a href="http://www.pombe2017.com">www.pombe2017.com</a> for details. We look forward to seeing you in Banff!<br />
<br />
- Conference Organizers: Dallan Young, Gordon Chua, Paul Young</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2016-10-19</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through September 11, 2016.</p>
<h3 id="show-your-support-database-funding">Show your support for database funding</h3>
<p><em>2016-06-27</em></p>
<p>In response to planned cuts to database funding, leading model organism researchers have prepared an open letter to NIH Director Dr. Francis Collins to demonstrate support for the independent community-focused databases that are essential to their work. Although PomBase is not directly funded by NIH, we collaborate extensively with those that are, including the GO Consortium and several model organism databases.<br />
<br />
The Genetics Society of America website where the letter can be viewed and signed is at <a href="http://www.genetics-gsa.org/MODsupport" class="uri">http://www.genetics-gsa.org/MODsupport</a><br />
<br />
Please sign the letter to add your voice in support of the databases that help make your research possible. For more information, we recommend an email that Mike Cherry sent to the GO-Friends mailing list, archived at <a href="https://mailman.stanford.edu/pipermail/go-friends/2016-June/002355.html" class="uri">https://mailman.stanford.edu/pipermail/go-friends/2016-June/002355.html</a></p>
<h3 id="our-model-organism-database-commentary">Our model organism database commentary</h3>
<p><em>2016-06-15</em></p>
<p>Several of the PomBase staff, joined by our advisor Sir Paul Nurse, have published a Comment in <em>BMC Biology</em> briefly describing the importance of model organism databases to the success of modern biomedical research:<br />
<br />
Oliver SG, Lock A, Harris MA, Nurse P, Wood V. 2016. Model organism databases: essential resources that need the support of both funders and users.<br />
<em>BMC Biol.</em> 2016 <strong>14(1)</strong>: 49. doi: <a href="http://dx.doi.org/10.1186/s12915-016-0276-z">10.1186/s12915-016-0276-z</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27334346">PMID:27334346</a></p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2016-05-31</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through May 12, 2016.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2016-05-09</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through April 8, 2016.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2016-04-11</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through March 9, 2016.</p>
<p><strong>Important:</strong> We have corrected a problem that made erroneous interaction data and literature appear on some gene pages.</p>
<p>The gene pages now include interaction data from the Vo et al. proteome-wide study (curated by BioGRID and imported into PomBase):<br />
Vo TV <em>et al.</em> 2016. A Proteome-wide Fission Yeast Interactome Reveals Network Evolution Principles from Yeasts to Human. <em>Cell</em> <strong>164(1-2)</strong>: 310-23. doi: <a href="http://dx.doi.org/10.1016/j.cell.2015.11.037">10.1016/j.cell.2015.11.037</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26771498">PMID:26771498</a>.</p>
<p>The genome browser now includes transcriptome data published in:<br />
<br />
Eser P, Wachutka L, Maier KC, Demel C, Boroni M, Iyer S, Cramer P, Gagneur J. 2016. Determinants of RNA metabolism in the Schizosaccharomyces pombe genome. <em>Mol Syst Biol</em>. <strong>12(2):</strong> 857. doi: <a href="http://dx.doi.org/10.15252/msb.20156526">10.15252/msb.20156526</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26883383">PMID:26883383</a>.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2016-02-11</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through January 25, 2016.<br />
The genome browser includes variation data, in tracks under &quot;Variation&quot;, from natural <em>S. pombe</em> isolates, published in:<br />
<br />
Jeffares DC <em>et al.</em> 2015. The genomic and phenotypic diversity of Schizosaccharomyces pombe. <em>Nat Genet.</em> <strong>47(3):</strong> 235-241. doi:<a href="http://dx.doi.org/10.1038/ng.3215">10.1038/ng.3215</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25665008">PMID:25665008</a></p>
<p>New files are now available from the PomBase FTP site, and are linked from pages in the Download Datasets area:</p>
<ul>
<li>Non-coding RNA sequence feature coordinates (available via the <a href="/downloads/data-mappings">Data Mappings</a> page);</li>
<li>Protein features, such as domains and family assignments (available via the <a href="/downloads/protein-datasets">Protein Datasets</a> page);</li>
<li>Protein modification annotations (also in Protein Datasets).</li>
</ul>
<p>The <a href="/status/new-and-removed-genes">New and Removed Genes page</a> has been updated to reflect recent deletions and merges.<br />
<br />
Note: Ontology graph views are no longer available in the genome browser, so links have been removed from the GO, FYPO, and modification tables on the gene pages. For GO and FYPO, links to external ontology browsers that offer graphical views are available on the <a href="/documentation/ontology-term-page">Ontology Term pages</a>.<br />
<br />
</p>
<h3 id="new-advanced-search-features">New Advanced Search features</h3>
<p><em>2015-12-02</em></p>
<p>We have introduced new features to the <a href="/spombe/query/builder">Advanced Search</a>:</p>
<ul>
<li>There are now two query reuse options: store a query in your web browser cache, or download a JSON file that can be uploaded later to re-run.</li>
<li>You can now query for genes that interact genetically or physically with a specified gene.</li>
<li>The FYPO query now offers options to choose nulls (deletions or disruptions), wild-type overexpression, or all alleles. The search results will include any genes that have an allele that matches the allele criteria and the chosen phenotype.</li>
</ul>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-12-02</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through November 9, 2015, including 340 community-curated publications.</p>
<h3 id="new-genetics-primer-fission-yeast">New genetics primer for fission yeast</h3>
<p><em>2015-10-19</em></p>
<p>A new genetics primer, aimed at researchers interested in using fission yeast as a model system, has recently been published. The primer includes a brief history of fission yeast research, an introduction to available genetic tools, and the use of PomBase for data analysis</p>
<p>Hoffman CS, Wood V, Fantes PA. (2015) An Ancient Yeast for Young Geneticists: A Primer on the <em>Schizosaccharomyces pombe</em> Model System. <em>Genetics</em> <strong>201</strong>:403-423. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26447128">PMID:26447128</a> <a href="http://dx.doi.org/10.1534/genetics.115.181503">DOI:10.1534/genetics.115.181503</a></p>
<h3 id="pombase-data-update-viability-summary-correction-alert">PomBase data update; viability summary correction alert</h3>
<p><em>2015-09-28</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through September 6, 2015.</p>
<p>Errors in the previous FYPOviability.tsv file have been corrected, and we recommend that all users update this file, especially those who downloaded it earlier in September 2015.</p>
<h3 id="pombase-data-update-multi-allele-phenotypes">PomBase data update with multi-allele phenotypes</h3>
<p><em>2015-09-03</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through August 13, 2015, including 300 community-curated publications.</p>
<p>PomBase gene pages now include multi-allele phenotype annotations (i.e. phenotypes of double mutants, triple mutants, etc.). New sub-sections of the gene pages display multi-allele phenotypes at the population and individual cell level, paralleling the organisation of the single allele phenotype display. Compact and full views are available; both show phenotypes with the relevant genotypes and the alleles that make them up, and the full view adds details for evidence, expression, conditions, and references.</p>
<p>The genome browser now includes data tracks for two more publications:<br />
<br />
DNA polymerase usage from:<br />
Daigaku Y, Keszthelyi A, Müller CA, Miyabe I, Brooks T, Retkute R, Hubank M, Nieduszynski CA, Carr AM. 2015. A global profile of replicative polymerase usage. <em>Nat Struct Mol Biol.</em> 2015 Mar;<strong>22(3):</strong>192-8. doi: <a href="http://dx.doi.org/10.1038/nsmb.2962">10.1038/nsmb.2962</a> PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25664722">25664722</a><br />
<br />
Promoters and transcription start sites from:<br />
Li H, Hou J, Bai L, Hu C, Tong P, Kang Y, Zhao X, Shao Z. 2015. Genome-wide analysis of core promoter structures in Schizosaccharomyces pombe with DeepCAGE. <em>RNA Biol.</em> 2015;<strong>12(5):</strong>525-37. doi: <a href="http://dx.doi.org/10.1080/15476286.2015.1022704">10.1080/15476286.2015</a>.1022704 PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25747261">25747261</a></p>
<p>Codon adaptation index (CAI) values are now included in the Protein Properties section of the gene pages and in the downloadable PeptideStats.tsv file. A file of amino acid composition data is also available from the FTP site and the <a href="/downloads/protein-datasets">Protein Datasets</a> page.</p>
<p>The gene page section that was formerly misnamed &quot;species distribution&quot; is now called &quot;taxonomic conservation&quot;.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-06-16</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through May 26, 2015, including 270 community-curated publications. See you at Pombe 2015 in Kobe!</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-05-26</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through May 8, 2015, including 265 community-curated publications.</p>
<h3 id="canto-downtime-new-version">Canto downtime &amp; new version</h3>
<p><em>2015-05-26</em></p>
<p>Canto, PomBase's literature curation tool, will be unavailable for approximately 3 weeks starting at 12:00 midnight UK time (BST) tonight, 27 May 2015, while we deploy an upgraded version.<br />
<br />
The upgraded Canto will feature an entirely new interface for annotating multi-allele phenotypes and the corresponding genotypes, as well as improved workflows for single-allele phenotypes, GO, etc. All existing annotations will be retained, and users can resume curation using the new and improved features in any unfinished sessions when Canto is back online.<br />
<br />
We will announce when the new version of Canto is released to the public.</p>
<h3 id="pombe-travel-fellowships">Pombe 2015 travel fellowships</h3>
<p><em>2015-04-23</em></p>
<p>Applications are now being accepted for fellowships to provide financial support for students and postdocs attending the 8th International Fission Yeast Meeting in Kobe, Japan. To apply, follow the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2015/004246.html">instructions sent to the pombase mailing list</a>. The deadline is may 17, 2015 (same as the registration deadline).</p>
<h3 id="pombe-poster-abstract-deadline-extended">Pombe 2015 poster abstract deadline extended</h3>
<p><em>2015-04-19</em></p>
<p>The abstract submission deadline for the <a href="https://amarys-jtb.jp/web/Pombe2015/index.html">8th International Fission Yeast Meeting</a> in Kobe, Japan has been extended until midnight Friday, April 24 for <strong>posters only</strong>. Registration is open until May 17.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-04-19</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through April 7, 2015, including 260 community-curated publications.The Advanced Search now supports queries for proteins with a specified number of transmembrane domains.</p>
<h3 id="pombe-abstract-deadline-approaching">Pombe 2015 abstract deadline approaching</h3>
<p><em>2015-04-09</em></p>
<p>Abstracts are due on Sunday, April 19, 2015 for the <a href="https://amarys-jtb.jp/web/Pombe2015/index.html">8th International Fission Yeast Meeting</a> in Kobe, Japan. Registration will remain open until May 17, but the abstract submission deadline cannot be extended.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-03-23</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through March7, 2015, including 250 community-curated publications.The autocomplete feature of the Advanced Search ontology term filter has been improved with respect to response time and relevance of suggested terms.</p>
<h3 id="pombe-registration-now-open">Pombe 2015 registration now open</h3>
<p><em>2015-02-26</em></p>
<p>Registration for Pombe 2015: 8th International Fission Yeast Meeting is now open at the conference web site, <a href="https://amarys-jtb.jp/web/Pombe2015/index.html" class="uri">https://amarys-jtb.jp/web/Pombe2015/index.html</a></p>
<p>The registration deadline is 17 May 2015.</p>
<p>Thanks to Yasushi Hiraoka for this item.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-02-16</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through February 2, 2015, including 245 community-curated publications. On the gene pages, the interaction tables now provides a bit of descriptive text for each annotation, indicating the nature and direction of the interaction.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-01-26</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through January 12, 2015, including 240 community-curated publications. The gene page Phenotype section now features a compact default display. A downloadable &quot;viability summary&quot; data file is now available. The PomBase BLAST server has incorporated interface changes made Ensembl-wide.</p>
<h3 id="esyn-network-visualizations-pombase">esyN network visualizations in PomBase</h3>
<p><em>2014-12-10</em></p>
<p>PomBase has implemented network visualisations for fission yeast in <a href="http://www.esyn.org/">esyN</a>, using data curated by BioGRID and PomBase. esyN is a web-based tool for building, sharing, and viewing network data developed by Dan Bean and Giorgio Favrin in the <a href="http://www.sysbiol.cam.ac.uk">Cambridge Systems Biology Centre</a>, University of Cambridge, UK.<br />
<br />
On gene pages, we have links to gene-specific interaction networks in esyN in the table headers of the Interactions sections:</p>
<ul>
<li>The Genetic Interactions section links to all interactions centred on the gene and curated in BioGRID</li>
<li>The Physical interactions section has links to two datasets:
<ul>
<li>All physical interactions curated in BioGRID for the gene</li>
<li>Interactions for the gene in the PomBase <a href="/documentation/high-confidence-physical-interaction-network">High Confidence Physical Interaction Network (HCPIN)</a></li>
</ul></li>
</ul>
<p>We also have esyN links on the <a href="/browse-curation/fission-yeast-go-slim-terms">GO Slim page</a> and on ontology term pages for GO Slim biological process terms. Each GO Slim term links to the HCPIN physical interaction network in esyN (for example, see the &quot;regulation of mitotic cell cycle&quot; <a href="http://www.esyn.org//builder.php?type=Graph&amp;term=GO:0007346&amp;interactionType=physical&amp;source=pombase&amp;includeInteractors=false">network</a>).</p>
<h3 id="new-compact-go-annotation-display">New compact GO annotation display</h3>
<p><em>2014-12-10</em></p>
<p>To make the Gene Ontology (GO) annotations easier to read on PomBase gene pages, we have introduced a new, streamlined display that presents just the essentials. The summary shows the term name (hyperlinked to the ontology term page), the count of genes annotated to the term, and any annotation extensions. All of the previously visible annotation details are still available -- simply click the &quot;Summary&quot; button to switch to the &quot;Full&quot; view. Or click the &quot;+&quot; and &quot;-&quot; icons to expand or collapse the annotation to a single term.<br />
 <br />
In addition, the top of the Biological Process table now lists any GO slim terms applicable to the gene.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-11-12</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through October 27, 2014, including 225 community-curated publications. The gene page Phenotype section now includes data from the high-throughput microscopy analysis of viable deletion mutants reported in:<br />
<br />
Graml V, Studera X, Lawson JL, Chessel A, Geymonat M, Bortfeld-Miller M, Walter T, Wagstaff L, Piddini E, Carazo-Salas RE. A Genomic Multiprocess Survey of Machineries that Control and Link Cell Shape, Microtubule Organization, and Cell-Cycle Progression. Dev Cell. 2014 Oct 27;31(2):227-39. doi: <a href="http://dx.doi.org/10.1016/j.devcel.2014.09.005">10.1016/j.devcel.2014.09.005</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25373780">PMID:25373780</a>. Links to the accompanying SYSGRO resource have been added to the External References section of the gene pages.</p>
<p>The genome browser now includes tracks for intron branch point data from:</p>
<p>Bitton DA, Rallis C, Jeffares DC, Smith GC, Chen YY, Codlin S, Marguerat  S, Bähler J. LaSSO, a strategy for genome-wide mapping of intronic  lariats and branch points using RNA-seq. Genome Res. 2014 Jul;24(7):1169-79. doi: <a href="http://dx.doi.org/10.1101/gr.166819.113">10.1101/gr.166819.113</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">PMID:24709818</a>.</p>
<p>We have greatly improved search results for GO and FYPO annotations: both now follow more relationship types within the ontology to retrieve genes annotated to a term. The PomBase GO search now includes the <em>regulates</em> relationships, so its search results are consistent with those in the GO Consortium's AmiGO browser. The FYPO search now uses <em>has_part,</em> <em>has_output</em>, and <em>output_of</em> as well as <em>is_a</em> and <em>part_of</em>. The Phenotype section now includes a highlighted sub-header that indicates whether a deletion mutant is viable or inviable. A file of protein complex subunits is available for download, and numerous smaller improvements have been made in the gene pages and static pages.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-09-16</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through August 30, 2014. Community curation now covers over 200 papers.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-08-18</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through August 8, 2014. Community curation now covers over 190 papers. Gene pages now include links to the <a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/buildDetails?atlas_build_id=355"><em>S. pombe</em> PeptideAtlas</a>, a database of peptides identified in tandem mass spectrometry proteomics experiments.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-07-17</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through July 8, 2014. The gene pages also now display protein modification data from an additional large-scale dataset:</p>
<p>Koch A, Krug K, Pengelley S, Macek B, Hauf S. 2011. Mitotic substrates of the kinase aurora with roles in chromatin regulation identified through quantitative phosphoproteomics of fission yeast. <em>Sci Signal</em>. <strong>4</strong>(179): rs6 doi: 10.1126/scisignal.2001588 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21712547">PMID:21712547</a><br />
<br />
We have also made corrections to some residue positions affected by sequence updates in one of the modification datasets we added last month:</p>
<p>Carpy A, Krug K, Graf S, Koch A, Popic S, Hauf S, Macek B. 2014. Absolute proteome and phosphoproteome dynamics during the cell cycle of fission yeast. <em>Mol Cell Proteomics</em>. 2014 Apr 23. [Epub ahead of print] <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24763107">PMID:24763107</a></p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-07-08</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through June 6, 2014. In other improvements, a downloadable file of <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/FASTA/cdna_introns_utrs.fa.gz">intron sequence data</a> (FASTA format) is now available, and phenotypes are now included in the Target Of section on gene pages.</p>
<p>The gene pages also now display protein modification data from two large-scale datasets:</p>
<ul>
<li>Wilson-Grady JT, Villén J, Gygi SP. 2008 .Phosphoproteome analysis of fission yeast. <em>J Proteome Res</em>. 2008 Mar;<strong>7</strong>(3):1088-97. doi:10.1021/pr7006335. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18257517">PMID:18257517</a></li>
<li>Carpy A, Krug K, Graf S, Koch A, Popic S, Hauf S, Macek B. 2014. Absolute proteome and phosphoproteome dynamics during the cell cycle of fission yeast. <em>Mol Cell Proteomics</em>. 2014 Apr 23. [Epub ahead of print] <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24763107">PMID:24763107</a></li>
</ul>
<h3 id="gene-ontology-publication-on-annotation-extensions">Gene Ontology publication on annotation extensions</h3>
<p><em>2014-06-29</em></p>
<p>PomBase was an early adopter of <a href="/documentation/gene-page-gene-ontology#ann_ext">annotation extensions</a>, which add spatial, temporal, or substrate/target details to GO annotations. The GO Consortium has now published a paper describing its implementation of annotation extensions, in which PomBase examples and its gene page display figure prominently:</p>
<p>Huntley, R.P. <em>et al.</em> (2014) A method for increasing expressivity of Gene Ontology annotations using a compositional approach. <em>BMC Bioinformatics</em> 2014, <strong>15</strong>:155. <a href="http://www.biomedcentral.com/1471-2105/15/155/abstract">doi:10.1186/1471-2105-15-155</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24885854">PMID:24885854</a></p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-05-15</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through April 28, 2014. Transcriptome data from <a href="http://europepmc.org/abstract/MED/23101633">Margeurat <em>et al</em> (2012)</a> is now available as Ensembl Browser tracks.</p>
<p>Thank you to all who have done, or are doing, paper curation in Canto. Over 159 community-curated papers are now included in PomBase.</p>
<p>There are a number of routes to accelerate your data into PomBase, (either through community curation, or by supplying HTP sequence, modification or phenotype data in one of our specified formats), see <a href="http://www.pombase.org/submit-data" class="uri">http://www.pombase.org/submit-data</a> for more details.</p>
<p>As usual, please don't hesitate to alert us of any other problems with data or site performance, or if you have any questions.</p>
<p>Sincerely yours,<br />
The PomBase Staff</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-03-20</em></p>
<p>Data on the PomBase web site now includes manual curation through February 24, 2014. Human orthologs that went missing from gene pages have been restored, and other small improvements have been made to gene pages. Community curation now covers over 130 publications.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-02-20</em></p>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through January 10, 2014, and covers over 100 papers that have been curated in Canto by community members. We again thank all who have contributed curation via Canto.</p>
<p>We have made some improvements to the gene pages. Highlights:</p>
<ul>
<li>The Sequence section now has links to NCBI BLAST as well as Ensembl BLAST.</li>
<li>The External References section now links to the Pomb(A) polyadenylation viewer.</li>
</ul>
<p>In the genome browser, new data tracks are now available for data from these publications:</p>
<ul>
<li>Rhind N, [and many more]. 2011. Comparative functional genomics of the fission yeasts. Science 332(6032):930-6. doi: 10.1126/science.1203357. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:21511999</a></li>
<li>Schlackow M, Marguerat S, Proudfoot NJ, Bähler J, Erban R, Gullerova M. 2013. Genome-wide analysis of poly(A) site selection in <em>Schizosaccharomyces pombe</em>. RNA. 19(12):1617-31. doi:10.1261/rna.040675.113. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24152550">PMID:24152550</a></li>
<li>Soriano I, Quintales L, Antequera F. 2013. Clustered regulatory elements at nucleosome-depleted regions punctuate a constant nucleosomal landscape in Schizosaccharomyces pombe. BMC Genomics. 14:813. doi:10.1186/1471-2164-14-813. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24256300">PMID:24256300</a> (partial data;  remainder coming in the next update)</li>
<li>Xu J, Yanagisawa Y, Tsankov AM, Hart C, Aoki K, Kommajosyula N, Steinmann KE, Bochicchio J, Russ C, Regev A, Rando OJ, Nusbaum C, Niki H, Milos P, Weng Z, Rhind N. 2012. Genome-wide identification and characterization of replication origins by deep sequencing. Genome Biol. 13(4):R27. doi:10.1186/gb-2012-13-4-r27. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22531001">PMID:22531001</a></li>
</ul>
<p>Now that more data tracks are available, we have added some categories to the track configuration section to improve organization. Additional documentation is in preparation, and will be announced here when available.<br />
<br />
Genome sequences for additional <em>Schizosaccharomyces</em> species (<em>S. japonicus</em>, <em>S. octosporus</em>, and <em>S. cryophilus</em>) have recently become available in Ensembl Fungi, and the PomBase genome browser now includes comparative genomics data, with a view of region comparisons between each new genome and <em>S. pombe</em>.</p>
<h3 id="human-ortholog-data-correction-coming-next-month">Human ortholog data correction coming next month</h3>
<p><em>2014-02-19</em></p>
<p>We are about to release a data update for PomBase. Please note that there is still a problem with the human orthologs, as originally described on this list in mid-December (see archived message at <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2013/003926.html" class="uri">http://listserver.ebi.ac.uk/pipermail/pombelist/2013/003926.html</a>). We will correct this problem in the next PomBase release, and apologise for any inconvenience in the meantime.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2013-12-08</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through November 11, 2013. We now have future meetings available as a <a href="/calendar">calendar</a> or a <a href="/events">list</a>. The <a href="/faq">FAQ</a> and some documentation pages have also been updated.<br />
<br />
</p>
<h3 id="meeting-mini-reviews-published">2013 meeting mini-reviews published</h3>
<p><em>2013-11-24</em></p>
<p>A series of mini-reviews, which were invited in association with the International Fission Yeast Meeting in London, have now been published in Biochemical Society Transactions: <a href="http://www.biochemsoctrans.org/bst/041/6/default.htm#c" class="uri">http://www.biochemsoctrans.org/bst/041/6/default.htm#c</a></p>
<p>(Thanks to Jürg Bahler for this item)</p>
<h3 id="pombase-survey-results-available">PomBase survey results available</h3>
<p><em>2013-11-20</em></p>
<p>The 2013 PomBase user survey closed at the end of October, and the results are available <a href="/community/2013-pombase-user-survey-results">here</a>. Some highlights have been sent to the pombe mailing list. Many thanks to all who completed the survey.</p>
<h3 id="new-target-gene-page-section">New &quot;Target Of&quot; gene page section</h3>
<p><em>2013-10-27</em></p>
<p>With the October 2013 update, gene pages now include &quot;<a href="/documentation/gene-page-target">Target Of</a>&quot; annotations, which describe genes that affect the gene of interest. These annotations are essentially the reciprocal of <a href="/faqs/what-annotation-extension">ontology annotation extensions</a>. Each &quot;Target Of&quot; annotation includes a relationship that indicates how the genes are connected, the name and product of the second gene, and a reference. Genes listed under &quot;Target Of&quot; may include upstream regulators or enzymes that modify the product of the gene of interest. For example, the &quot;Target Of&quot; annotations for <a href="/spombe/result/SPBC11B10.09#recipext">cdc2</a> indicate that it is a substrate of, and regulated by, the kinase Wee1 and the phosphatase Cdc25 (among others). At present, &quot;Target Of&quot; data includes annotations derived from <a href="/documentation/gene-page-gene-ontology#ann_ext">GO annotation extensions</a>. We will soon extend it to include data from <a href="/documentation/gene-page-phenotypes">phenotype</a> annotation extensions.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2013-10-21</em></p>
<p>The PomBase web site has been updated and now includes manually curated data through October 6, 2013. The number of community-curated papers continues to increase, ensuring that PomBase gene pages contain complete and up-to-date information. We are also pleased to announce that data tracks are now available in the genome browser for data from these two publications:</p>
<ul>
<li>Woolcock KJ, Gaidatzis D, Punga T, Bühler M. 2010. Dicer associates with chromatin to repress genome activity in Schizosaccharomyces pombe. Nat Struct Mol Biol. 2011 Jan;18(1):94-9. doi: 10.1038/nsmb.1935 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21151114">PMID:21151114</a></li>
<li>Mata J. 2013. Genome-wide mapping of polyadenylation sites in fission yeast reveals widespread alternative polyadenylation. RNA Biol. 2013 Aug 1;10(8):1407-14. doi: 10.4161/rna.25758 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23900342">PMID:23900342</a></li>
</ul>
<h3 id="pombase-user-survey-open">PomBase User Survey open</h3>
<p><em>2013-09-18</em></p>
<p>To guide current and future development, PomBase is now conducting a <a href="https://www.surveymonkey.com/s/NDM2BQX">user survey</a>, where we invite the fission yeast research community and any other PomBase users to evaluate the resources provided so far and comment on future priorities. The survey should take about 10 minutes to complete. Thank you for your participation!</p>
<p><a href="https://www.surveymonkey.com/s/NDM2BQX" class="uri">https://www.surveymonkey.com/s/NDM2BQX</a></p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2013-09-15</em></p>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through August 11, 2013. We are particularly pleased to note that this update includes annotations from several dozen papers curated by the <em>S. pombe</em> community. Many thanks to all who have done, or are doing, paper curation in <a href="http://curation.pombase.org/pombe/">Canto</a>.<br />
<br />
We also have an updated version of the <em>S. pombe</em>/human ortholog table available upon <a href="mailto:helpdesk@pombase.org">request</a>.</p>
<h3 id="send-htp-data-pombase">Send HTP data to PomBase</h3>
<p><em>2013-08-18</em></p>
<p>At the pombe 2013 meeting in London, PomBase staff received numerous requests display various published data, such as gene expression, histone modifications, etc. in the genome browser. To provide this, we now invite pombe researchers to send data: If you have published any high-throughput experiments that produced data  that can be associated with genome sequence coordinates, and thereby displayed as tracks on the PomBase genome browser, please fill out the <a href="/submit-data/data-submission-form">HTP Data Submission Form</a>. We can also accept large sets of phenotype data via the Phenotype Data Submission Form. If you have any problems or questions, contact us via the <a href="mailto:helpdesk@pombase.org">PomBase Helpdesk</a> at any time.</p>
<h3 id="connecting-pombase">Connecting With PomBase</h3>
<p><em>2013-07-29</em></p>
<p>To complement the <a href="/community/mailing-lists">mailing list</a> and twitter (<span class="citation">[@PomBase]</span>(http://twitter.com/PomBase)) it is now possible to follow the activities of PomBase and interact with other members of the pombe community via the new <a href="http://www.linkedin.com/groups/PomBase-5122686">LinkedIn Group</a> and <a href="https://plus.google.com/114842152049074799331">Google+</a>.</p>
<p>Links to these are also available from the front page of the PomBase.org site.</p>
<h3 id="pombelist-has-moved">pombelist has moved</h3>
<p><em>2013-07-21</em></p>
<p>The pombe community mailing list, pombelist, has migrated from the Wellcome Trust Sanger Institute and is now hosted by EBI. The new address is <a href="mailto:pombelist@ebi.ac.uk">pombelist@ebi.ac.uk</a> (please note that the old address no longer works, and will generate an automatic notification including the new address). The link to <a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">subscribe</a> has also been updated, and the entire <a href="http://listserver.ebi.ac.uk/mailman/private/pombelist/">archive</a> is available at the new location.</p>
<h3 id="pombase-website-update">PomBase website update</h3>
<p><em>2013-07-18</em></p>
<p>We'd like to highlight a few improvements we've just made to the PomBase website. Most of the changes affect the gene pages:</p>
<ul>
<li>The basic information display at the top of each gene page is more compact.</li>
<li>For ontology annotations, the number of genes annotated is now shown, in a column labeled &quot;Count&quot; (also, changes behind the scenes involving this data mean that pages should load faster).</li>
<li>Annotation extensions for GO are displayed using human-friendly text instead of internal &quot;relation&quot; labels.</li>
<li>The Quick Links box can now be collapsed and expanded by clicking its header.</li>
<li>Display of modification annotations using PSI-MOD is improved.</li>
</ul>
<p>In addition, the Motif Search output now includes standard gene names and product descriptions. As we noted in a separate message, CDS coordinate files are once again available from the Downloads, with accurate and up-to-date data.<br />
<br />
</p>
<h3 id="pombase-launches-community-curation">PomBase launches community curation</h3>
<p><em>2013-06-23</em></p>
<p>At the pombe 2013 conference in London, PomBase officially launched its community curation initiative, which allows researchers to contribute publication-based annotations directly to the database. PomBase curators invite lab heads by individual email to curate newly published papers, providing links to the online curation system and its documentation. Researchers can also initiate curation of any older fission yeast publication in PubMed. Community curation uses the open-source online tool <a href="http://curation.pombase.org/pombe/">Canto</a>.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2013-06-20</em></p>
<p>PomBase data now includes manual curation through June 9, 2013, and represents complete annotation for 664 publications (as well as partial curation of many more). A highlight of this month's literature curation update is the addition of over 9400 phenotype annotations, representing about 95% of the phenotype data from the recently published genome-wide study of cell cycle and cell morphology (<a href="http://rsob.royalsocietypublishing.org/content/3/5/130053.abstract">Hayles <em>et al.</em> Open Biology May 2013</a>; <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23697806">PMID:23697806</a>). We have also improved the display of allele details for phenotype annotations. Other changes include better support for gene synonyms in the simple search, regular updates to the UTR data files, and a number of minor adjustments to external links in the annotation data tables and the external references section.</p>
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2013-05-20</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through 13 May, 2013.</p>
<h3 id="genedb-s.-pombe-decommissioned">GeneDB S. pombe decommissioned</h3>
<p><em>2013-05-13</em></p>
<p>As of 14 May 2013, the old GeneDB database for <em>S. pombe</em> is no longer available. This resource consisted of static web pages, was not updated after March 2012, and not supported by an underlying relational database. The PomBase site fully supersedes GeneDB <em>S. pombe</em>, and provides improved infrastructure that will meet the current and future needs of the fission yeast community. Please e-mail the <a href="mailto:helpdesk@pombase.org">helpdesk</a> if you cannot find a replacement for any GeneDB functionality in PomBase.</p>
<h3 id="quantitative-gene-expression-data-available-pombase">Quantitative gene expression data available in PomBase</h3>
<p><em>2013-05-07</em></p>
<p>We have extended the Gene Expression section of each gene page to support the display of quantitative expression data, and are now showing data from two publications:</p>
<ul>
<li>Marguerat S, Schmidt A, Codlin S, Chen W, Aebersold R, BählerJ. 2012. &quot;Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and quiescent cells.&quot; <em>Cell</em> <strong>151:</strong>671-683.</li>
<li>Wu JQ, Pollard TD. 2005. &quot;Counting cytokinesis proteins globally and locally in fission yeast.&quot; <em>Science</em> <strong>310:</strong>310-314.</li>
</ul>
<p>We will also soon refine the display of the new expression data, and can add more datasets upon request. We thank Sam Marguerat for preparing the data from both papers for inclusion in PomBase.</p>
<p>We have also updated the PomBase site to include manual curation through April 4, 2013, and we have updated the &quot;all gene names&quot; file on the PomBase ftp site. The new file is available at<br />
<a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/allNames.tsv" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/allNames.tsv</a></p>
<h3 id="carl-singer-foundation-established">Carl Singer Foundation Established</h3>
<p><em>2013-04-11</em></p>
<p>Carl Singer, who was an integral part of the yeast research community for many years, passed away on February 8, 2013. Throughout his career, Carl supported yeast research both with his engineering expertise and with his good cheer. In tribute to Carl, the Singer family has now set up <a href="http://www.thecarlsingerfoundation.org/">The Carl Singer Foundation</a>, a charitable foundation dedicated to supporting scientific education in the field of yeast genetics. Questions about the foundation may be directed to Harry Singer at harry [at] thecarlsingerfoundation.org.<br />
<br />
Carl’s family would be happy to receive memories of Carl’s life at regards [at] singerinstruments.com.</p>
<p>H/T <a href="http://www.yeastgenome.org/carl-singer-foundation-established">SGD</a></p>
<h3 id="pombe-registration-abstracts-by-mon-th-april">Pombe 2013: registration &amp; abstracts by Mon 8th April</h3>
<p><em>2013-04-02</em></p>
<pre><code>Dear Pombe Fans,
      Please remember the imminent deadline (Monday 8th April) to register and submit abstracts for Pombe 2013: http://events.embo.org/13-pombe
      Abstracts are also required from all who have already been invited to talk.
      And do book your accommodation if you haven&#39;t yet done so.
      More details are in previous email forwarded below.
      Cheers,
      -Jürg &amp; Jacky
      From: On Behalf Of Bahler, Jurg
      Sent: 18 March 2013 17:49
      To: pombelist at sanger.ac.uk
      Subject: [Pombelist] Pombe 2013: Accommodation, registration &amp; abstracts
      Dear Pombe Afficionados,
      Only three weeks left to register and submit abstracts for Pombe 2013, by Monday 8th April: http://events.embo.org/13-pombe
      Speakers for 10 plenary talks and all workshop talks will be selected from abstracts, and there will be attractive poster prizes.
      Payment is only requested after registration, by 10th May.
      Important: if you require accommodation, please do book this real soon now. Especially the most cost-effective student accommodation (comfortable, with private bathrooms) may not be available much longer, as it will be put on general sale shortly. Both hotels and student accommodation will sell out in June, so you have to arrange it now. Information on accommodation is available here: http://events.embo.org/13-pombe/application.html
      We will provide a number of free registrations for which you can apply during online registration (a few of which are reserved for student members of The Genetics Society: you become eligible if you join them now). The meeting is also supported by the Biochemical Society, so if you are, or become, a member you can apply to them for student bursaries or, if you have been a member for at least 1 year, also for travel grants.
      We highly appreciate all the generous contributions from our sponsors so far:
      Platinum: EMBO
      Gold: Biochemical Society, Genetics Society, Formedium, Sunrise Science Products, Singer Instruments, F1000Research, PomBase/Wellcome Trust
      Silver: MDPI - Open Access Publishing, Hybrigenics, Infors, Life Technologies, Bioneer
      Bronze: Nature Communications, m2p labs, Imsol, Open Biology
      We look very much forward to welcoming you in London this June!
      All the best,
      -Jürg &amp; Jacky</code></pre>
<h3 id="data-update-on-pombase-web-site">Data update on PomBase web site</h3>
<p><em>2013-04-01</em></p>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through March 6, 2013.<br />
<br />
We now expect to be able to update PomBase data every month, and will soon have an automated pipeline in place. We thank all of you for your patience during the long months when updates were infrequent.<br />
<br />
You should also see a few small improvements in the site:</p>
<ul>
<li>Ontology term pages now display the text definition for each term.</li>
<li>FASTA sequence retrieval should be quicker, and less likely to time out, for large gene lists.</li>
<li>There has been some tidying of the display of &quot;extension&quot; data for GO and phenotype annotations.</li>
</ul>
<p>Last month we noted an intermittent problem with the &quot;Reference&quot; column display in the data tables. The occurrence of this problem should now be greatly reduced, so please let us know if you see it recurring.<br />
<br />
As usual, please don't hesitate to alert us of any other problems with data or site performance, or if you have any questions.<br />
<br />
</p>
<h3 id="speed-improvements-and-new-data-on-pombase-web-site">Speed improvements and new data on PomBase web site</h3>
<p><em>2013-03-01</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through December 17, 2012, and reflects complete curation of an additional 70 papers.<br />
<br />
We have also made some improvements &quot;under the hood&quot; that should make gene page loading much faster. Please let us know if you have any problems with gene pages loading slowly or incompletely, whether or not you have reported issues in the past.<br />
<br />
We are aware that there is an intermittent problem with the &quot;Reference&quot; column display in the data tables -- sometimes a PubMed ID appears instead of an author name and year. This problem will be fixed as soon as possible. Please alert us if you notice anything else odd or wrong.</p>
<h3 id="new-data-and-new-features-on-pombase-web-site">New data and new features on PomBase web site</h3>
<p><em>2012-11-06</em></p>
<p>We are pleased to announce that we have updated both data and web site features for PomBase.</p>
<p>Most importantly, we have added new data types, and upgraded the gene pages to display them.</p>
<p>We have also added more annotations of existing data types, bringing the web site content up to September 11, 2012. The new annotations include the first contributions to come in via the new community curation system, and we thank the researchers who are participating in the initial phase of community curation.</p>
<p><strong>New annotation types:</strong></p>
<ul>
<li>Phenotype annotations now use the Fission Yeast Phenotype Ontology (FYPO), and include allele details, expression levels, and experimental conditions. With FYPO, more detailed phenotypes can be described, and links between terms for related phenotypes support improved phenotype searches.</li>
<li>Many GO annotations now include &quot;annotation extensions&quot; that provide additional specificity. For example, extensions may capture the substrate of a catalytic activity, the cell cycle phase during which a function or process occurs, or any of several other types of supporting information for the annotation. Annotation extensions are described in more detail below.</li>
</ul>
<p>You can see these new data types on many gene pages, such as <a href="/spombe/result/SPBC11B10.09">cdc2</a> or <a href="/spombe/result/SPBC106.10">pka1</a>.</p>
<p><strong>New web site features:</strong></p>
<ul>
<li>Annotation display - Gene page GO and phenotype displays have been revamped to show new annotation types described above.</li>
<li>Ontology term pages - Each ontology term ID now links to pages with information about the term and lists of genes annotated to it.</li>
<li>Ontology graph links - GO and phenotype annotation sections now include links to graphical ontology displays in the genome browser.</li>
<li>Sequence highlighting - Sequence download now offers an option to show colour highlighting of regions such as UTRs, introns and exons.</li>
<li>Versions - Each gene page now shows the current data version in the format PomBase:x.y, where <em>x</em> is the Ensembl Genomes (EG) version, and <em>y</em> is the Chado version. The sequence, and sequence feature locations, remain stable within any EG version, whereas annotations change with each Chado update.</li>
<li>Protein family information is now included in the Protein Features gene page section.</li>
<li>The Protein Feature section includes a link to the Pfam entry for a protein.</li>
<li>Transcript source data (e.g. for UTR coordinates) is now displayed in the Transcript Features section.</li>
<li>A Documentation page contains links to relevant portions of the Ensembl Genomes documentation. (More documentation will be added over the coming months.)</li>
</ul>
<p><strong>What are annotation extensions?</strong></p>
<p>Annotation extensions are a form of supporting data that can be added GO annotations (or other ontology annotations) to capture additional details not provided by the ontology term itself.</p>
<p>The information in GO annotation extensions encompasses several effector-target relationships, such as</p>
<ul>
<li>localisation dependencies</li>
<li>substrates of functions, e.g. targets of a protein kinase -- see the has_substrate extensions on Cdc2's &quot;protein serine/threonine kinase&quot; (GO:0004674) annotations</li>
<li>activators and inhibitors</li>
<li>regulation targets of signalling pathways or transcription factors</li>
</ul>
<p>Additional extensions describe spatial and temporal aspects of processes. For example, several S. pombe annotations now include extensions that indicate in which phase of the cell cycle a gene product is found in a cellular component or involved in a process -- see the pka1 annotations to &quot;nucleus&quot; (GO:0005634) and &quot;cytoplasm&quot; (GO:0005737).</p>
<p>You may also find the <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">GO wiki page</a> on annotation extensions informative, although it is primarily aimed at curators.</p>
<p>Annotation extensions can also be used with phenotype annotations. The most common usage of phenotype annotation extensions is to capture which gene, protein, etc. was used in an assay. For example, the sam5 (G441E) mutation of <em>pka1</em> causes nuclear accumulation of Ste11. This is represented by annotation to the ontology term &quot;nuclear protein accumulation&quot; (FYPO:0000255), with the extension &quot;assayed_using(PomBase:SPBC32C12.02)&quot;. Extensions can also indicate expressivity or penetrance for a phenotype.<br />
<br />
<br />
</p>
<h3 id="pombase-web-site-fully-live">PomBase web site fully live</h3>
<p><em>2012-07-01</em></p>
<p>We are pleased to announce that the PomBase web site, <a href="http://www.pombase.org">www.pombase.org</a>, is now fully live; the preview phase has ended. The site has been updated with an assortment of new features, datatypes, and bug fixes.</p>
<p>More recent data, reflecting additions and changes through March 20, 2012, are now available on gene pages and in search results.</p>
<p>The updated site features a Gene List Search that provides behavior equivalent to GeneDB's List Download. You can now type or paste lists into the Gene Systematic IDs and Gene Names filters, and use the Query History to combine a gene list search with other search options. For convenience, there is a direct link to a search page pre-configured to accept a list of systematic IDs available in the Find menu, on the Find page, and here: <a href="/spombe/query/builder?filter=12">http://www.pombase.org/spombe/query/builder?filter=12</a></p>
<p>The Advanced Search also now offers:</p>
<ul>
<li>options to search GO, FYPO, and other ontologies by term name or ID;</li>
<li>autocomplete for ontology term name search;</li>
<li>ability to search for genes in a region, such as centromeres or telomeres;</li>
<li>improved organization of filter selections.</li>
</ul>
<p>We have also fixed a Sequence Download error reported by some users, so that the &quot;CDS&quot;, CDS + UTRs&quot;, and &quot;CDS + UTRs + Introns&quot; options now retrieve the correct sequences.<br />
<br />
In addition, numerous minor improvements have been made. Please send any questions or comments on the PomBase web site to us at &lt;<a href="mailto:helpdesk@pombase.org">helpdesk@pombase.org</a>&gt;.</p>
<h3 id="pombase-preview-launch">PomBase preview launch</h3>
<p><em>2011-11-27</em></p>
<p>A preview of <a href="http://www.pombase.org">PomBase</a>, the new model organism database for the fission yeast <em>Schizosaccharomyces pombe</em>, has been <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000970.html">announced to the <em>S. pombe</em> community</a> for testing and feedback. For more on PomBase, see the NAR Database Issue paper (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">PubMed abstract</a>) or <a href="/about/contacts">contact the PomBase staff</a>.</p>
<h3 id="pombase-nar-paper-published-online">PomBase NAR paper published online</h3>
<p><em>2011-10-27</em></p>
<p>A paper describing <a href="http://www.pombase.org">PomBase</a> has been published online will be included in the 2012 Database Issue of <em>Nucleic Acids Research</em>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">Abstract</a> and <a href="http://nar.oxfordjournals.org/content/early/2011/10/28/nar.gkr853">open access full text</a> are available.</p>
<h3 id="genedb-s.-pombe-now-uses-latest-release-pfam-protein-family-database-.-.">GeneDB (S. pombe) now uses the latest release of the Pfam protein family database(25.0).</h3>
<p><em>2011-04-28</em></p>
<h3 id="schizosaccharomyces-comparative-genome-paper-published">Schizosaccharomyces Comparative Genome Paper Published</h3>
<p><em>2011-04-21</em></p>
<p>A paper describing the major findings of the <em>Schizosaccharomyces</em> Comparative Genome Project was <a href="http://www.sciencemag.org/content/early/2011/04/20/science.1203357">published today</a> in Science Express and reported changes are included in GeneDB.</p>
<p>Further details are described in the pombe mailing list posts:</p>
<ul>
<li><a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000855.html">Schizosaccharomyces Comparative Genome Paper Published</a></li>
<li><a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000856.html">Import of the fission yeast revisions from the Broad Institute comparative genome paper into GeneDB</a>.</li>
</ul>
<h3 id="genome-reappraisal-reveals-new-genes-and-revised-gene-structures">Genome reappraisal reveals new genes and revised gene structures</h3>
<p><em>2011-02-01</em></p>
<p>Further information on the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000815.html">pombe mailing list</a>.</p>
<h3 id="annotated-transcription-start-and-termination-sites-fission-yeast">Annotated transcription start and termination sites for fission yeast</h3>
<p><em>2011-01-31</em></p>
<p>Further details are available on the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000814.html">pombe mailing list</a>.</p>
<h3 id="analysis-fission-yeast-deletion-publication">Analysis of Fission Yeast Deletion Publication</h3>
<p><em>2010-05-15</em></p>
<p>The analysis of the fission yeast deletion collection is now published online in <a href="http://www.nature.com/nbt/journal/v28/n6/abs/nbt.1628.html">Nature Biotechnology</a>.</p>
<h3 id="funding-pombase">Funding for PomBase</h3>
<p><em>2010-02-28</em></p>
<p>Funding was awarded by the Wellcome Trust for a fission yeast Model Organism Database, <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2010-March/002731.html">PomBase</a>.</p>
<h3 id="fission-yeast-one-key-organisms-reference-genomes-project">Fission yeast is one of the 12 key organisms of the reference genomes project</h3>
<p><em>2009-11-30</em></p>
<p>Fission yeast is one of the 12 key organisms of the <a href="http://geneontology.org/GO.refgenome.shtml?all">reference genomes project</a>. The goal of this project is to completely annotate twelve reference genomes so that those annotations may be used to effectively seed the automatic annotation efforts of other genome.</p>
<h3 id="genedb-s.-pombe-now-uses-latest-update-pfam-release-.">GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0</h3>
<p><em>2009-10-31</em></p>
<p>GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0 and 88.5% of fission yeast proteins now contain a match to at least one Pfam domain (increased from 83% in version 23).</p>
<h3 id="fission-yeast-ensembl-fungi">Fission yeast in Ensembl Fungi</h3>
<p><em>2009-09-30</em></p>
<p>The fission yeast genome and annotation dataset is now available as part of <a href="http://fungi.ensembl.org/index.html">Ensembl Fungi</a>.</p>
<h3 id="genedb-now-using-version-pfam-protein-family-database.">GeneDB is now using Version 23 of the Pfam protein family database.</h3>
<p><em>2009-08-31</em></p>
<p>GeneDB is now using Version 23 of the Pfam protein family database. A total of 4154 (83%) S. pombe proteins now have at least one Pfam domain or family assignment (compared to 76% for S. cerevisiae), the highest percentage coverage for any eukaryote.</p>
<h3 id="s.-pombe-genedb-now-includes-deep-links-biological-general-repository-interaction-datasets-biogrid">S. pombe GeneDB now includes &quot;deep links&quot; to the Biological General Repository for Interaction Datasets (BioGRID)</h3>
<p><em>2008-11-30</em></p>
<p><em>S. pombe</em> GeneDB now includes &quot;deep links&quot; to the Biological General Repository for Interaction Datasets (BioGRID) interaction datasets from the 'Database Cross References' section of the individual Gene Pages.</p>
<h3 id="global-sequence-and-chip-study-examines-eukaryotic-transcription">Global sequence and chip study examines eukaryotic transcription</h3>
<p><em>2008-04-30</em></p>
<p>Dynamic repertoire of the fission yeast transcriptome reveals: 94% of the genome is transcribed; extensive variation in different stages and conditions; global and condition-specific coupling between splicing efficiency and transcription; confirms the majority of introns; refines ~75 gene structures; identifies 453 new transcripts 26 of which were predicted to code for proteins.</p>
<h3 id="the-h-mating-type-region-has-been-provided">The h- mating type region has been provided</h3>
<p><em>2008-01-31</em></p>
<p>The <a href="/status/mating-type-region">h- mating type region</a> has been provided by Xavier Marsellach and Lorena Aguilar.</p>
<h3 id="baumann-and-zakian-labs-identify-telomerase-rna">Baumann and Zakian labs identify telomerase RNA</h3>
<p><em>2007-12-31</em></p>
<p>Baumann and Zakian labs identify elusive <a href="/spombe/result/SPNCRNA.214">telomerase RNA</a> (PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18157152">18157152</a> and PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18157149">18157149</a>)</p>
<h3 id="wellcome-trust-advanced-course-genome-wide-approaches-fission-yeast">Wellcome Trust Advanced Course 'Genome-wide approaches with fission yeast</h3>
<p><em>2007-09-30</em></p>
<p>Wellcome Trust Advanced Course 'Genome-wide approaches with fission yeast' held in Hinxton.</p>
<h3 id="th-international-fission-yeast-meeting">4th International Fission Yeast Meeting</h3>
<p><em>2007-05-31</em></p>
<p>4th International Fission Yeast Meeting held in Copenhagen.</p>
<h3 id="genedb-representation-fission-yeast-data-moved-contigs-chromosomes">GeneDB representation of the fission yeast data moved from contigs to chromosomes</h3>
<p><em>2006-12-31</em></p>
<p>GeneDB representation of the fission yeast data moved from contigs to chromosomes. See the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2007/000128.html">pombelist archive</a> for details.</p>
<h3 id="yeast-special-issue-european-fission-yeast-meeting">Yeast Special Issue from the 2006 European Fission Yeast Meeting</h3>
<p><em>2006-09-30</em></p>
<p>The October issue of the journal Yeast is a <a href="http://onlinelibrary.wiley.com/doi/10.1002/yea.v23:13/issuetoc">fission yeast special issue</a> containing 13 articles and reviews commissioned as a result of the European Fission Yeast Meeting, which are FREE to download.</p>
<h3 id="the-first-fission-yeast-whole-proteome-localization-study-now-published">The first fission yeast whole proteome localization study is now published</h3>
<p><em>2006-06-30</em></p>
<p>The first fission yeast whole proteome localization study is now published: <a href="http://www.nature.com/nbt/journal/v24/n7/abs/nbt1222.html">Matsuyama A. et al (2006): ORFeome cloning and global analysis of protein localization in the fission yeast Schizosaccharomyces pombe. Nat Biotech 24, 841-7.</a><br />
 </p>
<h3 id="fission-yeast-database-survey">Fission yeast database survey</h3>
<p><em>2006-04-30</em></p>
<p>The fission yeast database survey is now closed. You can view the survey results <a href="/community/2006-fission-yeast-database-survey-results">here</a>.</p>
<h3 id="european-fission-yeast-meeting">European Fission Yeast Meeting</h3>
<p><em>2006-03-17</em></p>
<p>The European Fission Yeast Meeting (16th-18th March 2006) and The Fission Yeast Bioinformatics workshop (15th - 16th Mar 2006) both took place at the Wellcome Trust Genome Campus in Hinxton (Cambridge, UK).</p>
<h3 id="comparative-genomics-eukaryotic-microorganisms">Comparative Genomics of Eukaryotic Microorganisms</h3>
<p><em>2005-11-16</em></p>
<p>Comparative Genomics of Eukaryotic Microorganisms:<br />
Eukaryotic Genome Evolution, Approaches with Yeasts and Fungi</p>
<p><br />
This conference took place from 12th-17th November 2005 in Sant Feliu de Guixols, Spain. Full details can be found <a href="http://www.esf.org/index.php?id=940">here</a>.</p>
<h3 id="second-east-coast-regional-pombe-meeting">Second East Coast Regional pombe Meeting</h3>
<p><em>2005-11-12</em></p>
<p>Second East Coast Regional pombe Meeting<br />
<br />
This meeting took place from November 11-13, 2005 in Miami Beach, Florida.</p>
<h3 id="general-repository-interaction-datasets">General Repository for Interaction Datasets</h3>
<p><em>2004-08-31</em></p>
<p>A project to record published genetic and physical interactions is underway with Mike Tyers and the GRID group at Toronto.</p>
<h3 id="the-third-international-fission-yeast-meeting">The Third International Fission Yeast Meeting</h3>
<p><em>2004-08-29</em></p>
<p>The meeting was held at UC San Diego on August 24-29, 2004.</p>
<h3 id="methods-volume-issue">Methods Volume 33 Issue 3</h3>
<p><em>2004-04-30</em></p>
<p>This issue of Methods includes 11 papers for fission yeast protocols including DNA damage checkpoint assays, cell wall analysis, TAP, nuclear envelope integrity assays, GFP imaging, TS mutant creation and plasmid use and construction. See the <a href="http://www.sciencedirect.com/science/journal/10462023/33/3">Methods site</a> or the <a href="/fission-yeast-papers-methods-volume-33-issue-3">Table of Contents</a> (at PomBase)<a href="/fission-yeast-papers-methods-volume-33-issue-3"></a> for details of the papers including PMIDs.</p>
<h3 id="recent-genome-wide-surveys">Recent Genome wide surveys</h3>
<p><em>2003-10-31</em></p>
<p>Correlations Between Gene Expression and Gene Conservation in Fission Yeast. Mata J, Bahler J. Genome Res. 2003 Nov 12 <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14613978">PMID:14613978</a><br />
<br />
FELINES: a utility for extracting and examining EST-defined introns and exons. Drabenstot SD et al Nucleic Acids Res. 2003 Nov 15;31(22):e141. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14602934">PMID:14602934</a><br />
<br />
Genome-wide distribution of DNA replication origins at A+T-rich islands in Schizosaccharomyces pombe. Segurado M, De Luis A, Antequera F. EMBO Rep. 2003 Nov;4(11):1048-53. Epub 2003 Oct 17. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14566325">PMID:14566325</a><br />
<br />
Retrotransposons and their recognition of pol II promoters: a comprehensive survey... Bowen NJ et al Genome Res. 2003 Sep;13(9):1984-97. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12952871">PMID:12952871</a></p>
<h3 id="the-new-fission-yeast-book-now-published">The 'new' fission yeast book is now published</h3>
<p><em>2003-08-31</em></p>
<p>Egel, R., Copenhagen, Denmark (Ed.) The Molecular Biology of Schizosaccharomyces pombe Genetics, Genomics and Beyond ISBN:3-540-00693-1</p>
<h3 id="schizosaccharomyces-pombe-essential-genes-pilot-study">Schizosaccharomyces pombe Essential Genes: A pilot Study</h3>
<p><em>2003-02-28</em></p>
<p>Decottignies A, Sanchez-Perez I, Nurse P Genome Res. 2003 Mar;13(3):399-406. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12618370">PMID:12618370</a></p>
<h3 id="global-transcriptional-responses-fission-yeast-environmental-stress">Global transcriptional responses of fission yeast to environmental stress</h3>
<p><em>2002-12-31</em></p>
<p>Chen D, Toone WM, Mata J, Lyne R, Burns G, Kivinen K, Brazma A, Jones N, Bähler J. Mol Biol Cell. 2003 Jan;14(1):214-29. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12529438">PMID:12529438</a></p>

  </div>
  </div>
</div>
<div *ngIf="section == 'status'">
  <div class="docs-menu">
<h3 id="status"><a routerLink="/status">Status</a></h3>
<ul>
<li><a routerLink="/status/statistics">Statistics</a></li>
<li><a routerLink="/status/sequencing-status">Sequencing status</a>
<ul>
<li><a routerLink="/status/centromeres">Centromeres</a></li>
<li><a routerLink="/status/mating-type-region">Mating type region</a></li>
<li><a routerLink="/status/telomeres">Telomeres</a></li>
</ul></li>
<li><a routerLink="/status/sequencing-updates">Sequence updates</a></li>
<li><a routerLink="/status/sequence-updates-pending">Sequence Updates Pending</a></li>
<li><a routerLink="/status/gene-coordinate-changes">Gene coordinate changes</a></li>
<li><a routerLink="/status/new-and-removed-genes">New and removed genes</a></li>
<li><a routerLink="/status/gene-characterisation">Gene characterisation</a></li>
<li><a routerLink="/status/priority-unstudied-genes">Priority unstudied genes</a></li>
<li><a routerLink="/status/unmapped_genes">Unmapped genes</a></li>
</ul>

  </div>
  <div class="docs-content">
  <div *ngIf="pageName == 'centromeres'">
<h1 id="centromeres">Centromeres</h1>
<p>At present, each centromere is annotated as a single sequence feature in PomBase, which can be viewed in and downloaded from the Ensembl genome browser.</p>
<p>Chromosome coordinates (links to Ensembl genome browser):</p>
<ul>
<li>CEN 1: <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=I%3A3753687-3789421;site=ensemblunit">I:3753687-3789421</a></li>
<li>CEN 2: <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=II%3A1602264-1644747;site=ensemblunit">II:1602264-1644747</a></li>
<li>CEN 3: <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=III%3A1070904-1137003;site=ensemblunit">III:1070904-1137003</a></li>
</ul>
<p>The 3 centromeres are also available as contiguated sequence (the remaining gaps in CEN2 and CEN3 are represented by 100 N's)</p>
<ul>
<li><a href="http://www.pombase.org/sites/pombase.org/files/documents/cen1.gz">CEN 1</a> (gzipped EMBL format)</li>
<li><a href="http://www.pombase.org/sites/pombase.org/files/documents/cen2.gz">CEN 2</a> (gzipped EMBL format)</li>
<li><a href="http://www.pombase.org/sites/pombase.org/files/documents/cen3.gz">CEN 3</a> (gzipped EMBL format)</li>
</ul>
<p>Note that specific repeats within centromeres cannot yet be viewed or searched as features on PomBase pages, but they can be displayed on tracks in the genome browser. Turn on any or all of &quot;PomBase Annotated Centromeric Inner Repeat Region (SO:0001798)&quot;, &quot;PomBase Annotated dg Repeat (SO:0001898)&quot; or &quot;PomBase Annotated dh Repeat (SO:0001899)&quot; (see the <a href="http://www.pombase.org/faqs/how-can-i-show-or-hide-tracks-genome-browser">browser track FAQ</a> for instructions).</p>
<p>Repeats are also shown in the diagram below. To see the repeat sequences, download and unzip the <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Chromosome_contigs/">contiguated sequence files</a> and view them in <a href="http://www.sanger.ac.uk/resources/software/artemis/">Artemis</a>. (See <a href="http://www.pombase.org/faqs/there-equivalent-artemis-java-applet-pombase">this FAQ</a> for more information.)</p>
<p><span style="color: red;">Note: Recent work by Chad Ellermeier and Gerry Smith suggests that there are only 4 +/- 1 copies of the 6760 bp repeat missing from chromosome 3</span></p>
<p><span style="color: red;">Please note: This is a schematic diagram. Distances and overlaps are approximate. Please refer to the sequence data to design experimental constructs.</span></p>
<div class="figure">
<img src="images/centromeremapping.gif" alt="Centromere Map" width="779" height="540" />
<p class="caption">Centromere Map</p>
</div>
<p>Centromere Map from: The genome sequence of <em>Schizosaccharomyces pombe</em>.<br />
Nature 2002 Feb 21;415(6874):871-80. created by Rhian Gwilliam</p>

  </div>
  <div *ngIf="pageName == 'gene-characterisation'">
<h1 id="protein-coding-gene-characterisation">Protein-coding gene characterisation</h1>
<p><app-characterisation-status-table></app-characterisation-status-table></p>
<hr />
<h2 id="protein-coding-gene-characterisation-status-descriptions">Protein-coding Gene Characterisation Status Descriptions</h2>
<p><strong>Published:</strong> Completely or partially characterised in a small scale experiment, with some published information about the biological role (corresponding to any of the <a href="browse-curation/fission-yeast-go-slim-terms">fission yeast GO slim</a> biological process terms)</p>
<p><strong>Biological role inferred:</strong> A biological role (as above, a <a href="browse-curation/fission-yeast-go-slim-terms">fission yeast GO slim</a> term) is inferred from homology to an experimentally characterised gene product</p>
<p><strong>Conserved protein (unknown biological role):</strong> Conserved outside the <i>Schizosaccharomyces</i>, but nothing known about the biological role in any organism</p>
<p><strong><i>Schizosaccharomyces</i> specific protein, uncharacterized:</strong> Unpublished and found only in fission yeast (<i>S. pombe</i>, <i>S. octosporus</i>, <i>S. japonicus</i>, <i>S. cryophilus</i>); nothing known about biological role. May be single copy or a member of a multi-member family.</p>
<p><strong><i>S. pombe</i> specific protein, uncharacterized:</strong> Unpublished and found only in <i>S. pombe</i> (not detected in other <i>Schizosaccharomyces</i> species); nothing known about biological role</p>
<p><strong>Dubious:</strong> Unlikely to be protein coding</p>
<p><strong>Transposon:</strong> A predicted or experimentally verified transposable element.</p>
<p>Note: You can retrieve current lists of genes with each characterisation status using the <a routerLink="/query">Advanced Search</a>. Select the Characterisation Status filter, then choose a status from the pulldown menu, and submit. A set of <a href="/status/gene-characterisation-statistics-history">historical data</a> recorded at various intervals is available.</p>

  </div>
  <div *ngIf="pageName == 'gene-characterisation-statistics-history'">
<h1 id="protein-coding-gene-characterisation-statistics-history">Protein-coding gene characterisation statistics history</h1>
<p>==== History of characterisation status using current status descriptions ==== ===== (introduced with PomBase Chado release 46, August 2014): =====</p>
<table>
 <thead>
  <tr>
   <th style="width: 100px;">Date</th>
   <th style="width: 100px;">Published</th>
   <th style="width: 100px;">Role inferred</th>
   <th style="width: 100px;">Conserved unknown</th>
   <th style="width: 100px;"><em><em><strong>Schizo.</strong></em></em></th>
   <th style="width: 100px;"><em>S. pombe</em></th>
   <th style="width: 100px;">Dubious</th>
   <th style="width: 100px;">Transposon</th>
   <th style="width: 100px;">Total</th>
  </tr>
 </thead>
 <tbody>
  <tr>
  <td><span class="no-break">2017-01-30</span></td>
  <td>2235</td>
  <td>1996</td>
  <td>511</td>
  <td>180</td>
  <td>145</td>
  <td>54</td>
  <td>13</td>
  <td>5171</td>
 </tr>
  <tr>
  <td><span class="no-break">2016-09-12</span></td>
  <td>2227</td>
  <td>2034</td>
  <td>503</td>
  <td>190</td>
  <td>149</td>
  <td>55</td>
  <td>13</td>
  <td>5171</td>
 </tr>
 <tr>
  <td><span class="no-break">2015-01-19</span></td>
  <td>2154</td>
  <td>2050</td>
  <td>529</td>
  <td>178</td>
  <td>143</td>
  <td>76</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-09-17</span></td>
  <td>2124</td>
  <td>2066</td>
  <td>537</td>
  <td>183</td>
  <td>143</td>
  <td>77</td>
  <td>&nbsp;13</td>
  <td>5143</td>
 </tr>
 </tbody>
</table>
<p>==== History of characterisation status using previous status descriptions: ====</p>
<table>
 <thead>
  <tr>
   <th style="width: 100px;">Date</th>
   <th style="width: 100px;">Published</th>
   <th style="width: 100px;">Role inferred</th>
   <th style="width: 100px;">Conserved unknown</th>
   <th style="width: 100px;"><em>S. pombe</em></th>
   <th style="width: 100px;">Orphan</th>
   <th style="width: 100px;">Dubious</th>
   <th style="width: 100px;">Transposon</th>
   <th style="width: 100px;">Total</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td><span class="no-break">2006-08-25</span></td>
  <td>1560</td>
  <td>2433</td>
  <td>458</td>
  <td>68</td>
  <td>403</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>4979</td>
 </tr>
 <tr>
  <td><span class="no-break">2007-02-27</span></td>
  <td>1607</td>
  <td>2329</td>
  <td>572</td>
  <td>47</td>
  <td>364</td>
  <td>60</td>
  <td>&nbsp;</td>
  <td>4979</td>
 </tr>
 <tr>
  <td><span class="no-break">2008-06-05</span></td>
  <td>1715</td>
  <td>2210</td>
  <td>634</td>
  <td>42</td>
  <td>344</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5002</td>
 </tr>
 <tr>
  <td><span class="no-break">2009-01-14</span></td>
  <td>1812</td>
  <td>2132</td>
  <td>636</td>
  <td>56</td>
  <td>318</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5011</td>
 </tr>
 <tr>
  <td><span class="no-break">2010-02-22</span></td>
  <td>1848</td>
  <td>2160</td>
  <td>614</td>
  <td>55</td>
  <td>286</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5020</td>
 </tr>
 <tr>
  <td><span class="no-break">2010-06-30</span></td>
  <td>1916</td>
  <td>2163</td>
  <td>592</td>
  <td>54</td>
  <td>285</td>
  <td>58</td>
  <td>11</td>
  <td>5025</td>
 </tr>
 <tr>
  <td><span class="no-break">2011-04-27</span></td>
  <td>1916</td>
  <td>2169</td>
  <td>588</td>
  <td>64</td>
  <td>315</td>
  <td>79</td>
  <td>11</td>
  <td>5142</td>
 </tr>
 <tr>
  <td><span class="no-break">2011-12-16</span></td>
  <td>1936</td>
  <td>2165</td>
  <td>582</td>
  <td>66</td>
  <td>309</td>
  <td>73</td>
  <td>11</td>
  <td>5142</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-05-23</span></td>
  <td>2012</td>
  <td>2137</td>
  <td>541</td>
  <td>66</td>
  <td>303</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-12-06</span></td>
  <td>2083</td>
  <td>2096</td>
  <td>522</td>
  <td>64</td>
  <td>294</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-07-14</span></td>
  <td>2110</td>
  <td>2076</td>
  <td>517</td>
  <td>64</td>
  <td>292</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 </tbody>
</table>
<h3 id="status-descriptions">Status Descriptions</h3>
<table>
<thead>
 <tr>
  <th>Shorthand</th>
  <th>Full name</th>
  <th>Description</th>
  <th>Comment</th>
 </tr>
</thead>
<tbody>
 <tr>
  <td>Published</td>
  <td>Experimentally characterised (or published)</td>
  <td>Completely or partially characterised in a small scale experiment, with some published information about the biological role (corresponding to any of the fission yeast GO slim biological process terms)</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Role inferred</td>
  <td>Role inferred from homology</td>
  <td>A biological role (as above, a fission yeast GO slim term) is inferred from homology to an experimentally characterised gene product</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Conserved unknown</td>
  <td>Conserved protein (unknown biological role)</td>
  <td>Conserved outside the <em>Schizosaccharomyces</em>, but nothing known about the biological role in any organism</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><em>Schizo.</em></td>
  <td><em><em>Schizosaccharomyces</em>&nbsp;</em>specific protein, uncharacterised</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&nbsp;<em>S. octosporus,&nbsp;<em>S. japonicus, S. cryophilus</em></em><em></em>); nothing known about biological role. May be single copy or a member of a multi-member family.</td>
  <td>Introduced Sept. 2014</td>
 </tr>
 <tr>
  <td><em>S. pombe</em></td>
  <td><em><em>S. pombe</em>&nbsp;</em>specific protein, uncharacterised</td>
  <td>Unpublished and found only in&nbsp;<em>S. pombe&nbsp;</em>(not detected in other&nbsp;<em>Schizosaccharomyces</em>&nbsp;species); nothing known about biological role</td>
  <td>Introduced Sept. 2014&nbsp;</td>
 </tr>
 <tr>
  <td>Transposon</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Dubious</td>
  <td>&nbsp;</td>
  <td>Unlikely to be protein coding</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><em>S. pombe</em></td>
  <td>&nbsp;<em>S. pombe</em>&nbsp;specific families</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&nbsp;<em>S. octosporus,&nbsp;</em><em>S. japonicus, S. cryophilus</em>); nothing known about biological role, but are not single copy (duplications in fission yeast)</td>
  <td>&nbsp;Used Aug. 2006-Aug. 2014</td>
 </tr>
 <tr>
  <td>&nbsp;Orphan</td>
  <td>Sequence orphan, uncharacterised</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&nbsp;<em>S. octosporus</em>,&nbsp;<em>S. japonicus, S. cryophilus</em>); nothing known about biological role</td>
  <td>Used Aug. 2006-Aug. 2014</td>
 </tr>
 </tbody>
 </table>

  </div>
  <div *ngIf="pageName == 'gene-coordinate-changes'">
<h1 id="gene-coordinate-changes">Gene Coordinate Changes</h1>
<table>
<thead>
<tr class="header">
<th>Systematic ID</th>
<th>Standard name</th>
<th>Comments</th>
<th>Reference</th>
<th>Date</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a routerLink="/gene/SPBC577.05c">SPBC577.05c</a></td>
<td>rec27</td>
<td>Coordinates changed from complement(join(757566..757746,757796..757951,758130..758197)) to complement(join(757566..757746,757796..757951,758002..758042))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/28469148">PMID:28469148</a></td>
<td><span class="no-break">2017-09-14</span></td>
</tr>
</tbody>
</table>
<div style="white-space: pre-line;"><a routerLink="/gene/SPSNORNA.41">SPSNORNA.41</a> | snR46 | Switched from + strand to - strand | pers. comm. Francois Bachand | <span class="no-break">2017-05-25</span> |
<a routerLink="/gene/SPBC530.13">SPBC530.13</a> | lsc1 | Frameshifted by Chr_II:825012!G→A; coordinates changed from join(824374..824534,824579..824660,824874..825011,825065..825188,825245..825747) to join(824374..824534,824579..824660,824874..824999,825065..825188,825245..825747) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-17</span> |
<a routerLink="/gene/SPAC3A11.06">SPAC3A11.06</a> | mvp1 | Frameshifted by Chr_I:3460318!T→T; coordinates changed from complement(join(3459233..3459436,3459492..3459665,3459705..3459897,3459941..3460185,3460234..3460364,3460425..3460442,3460508..3460560,3460642..3461263,3461323..3461583,3461766..3461859)) to complement(join(3459233..3459436,3459492..3459665,3459705..3459897,3459941..3460185,3460234..3460318,3460318..3460372,3460425..3460442,3460508..3460560,3460642..3461263,3461323..3461583,3461766..3461859)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-06</span> |
<a routerLink="/gene/SPAC12B10.09">SPAC12B10.09</a> | pet801 | Coordinates changed from join(4588250..4588510,4588567..4588723,4588869..4589323,4589410..4589574) to join(4588487..4588510,4588567..4588723,4588869..4589323,4589410..4589574) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPBC16D10.10">SPBC16D10.10</a> | tad2 | Frameshifted by Chr_II:3619003!A→AG; coordinates changed from join(3618117..3618179,3618228..3618617,3618658..3618702,3618747..3618918,3618956..3619071,3619113..3619321,3619366..3619540) to join(3618117..3618179,3618228..3618617,3618658..3618702,3618747..3618920,3618968..3619003,3619003..3619071,3619113..3619321,3619366..3619540) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPBC13E7.01">SPBC13E7.01</a> | cwf22 | Frameshifted by Chr_II:3040332!C→CG; coordinates changed from join(3037988..3040331,3040334..3040650) to join(3037988..3040332,3040332..3040650) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPBC32F12.08c">SPBC32F12.08c</a> | duo1 | Frameshifted by Chr_II:2798040!CT→C; coordinates changed from complement(2798007..2798507) to complement(join(2797652..2798040,2798042..2798507)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPBC4F6.10">SPBC4F6.10</a> | vps901 | Frameshifted by Chr_II:2709414!G→GC; coordinates changed from join(2708076..2709415,2709418..2709793) to join(2708076..2709414,2709414..2709793) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPBC14C8.09c">SPBC14C8.09c</a> | | Frameshifted by Chr_II:2219928!A→AT; coordinates changed from complement(join(2219430..2219934,2219937..2220322)) to complement(join(2219430..2219928,2219928..2220322)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPBC23G7.06c">SPBC23G7.06c</a> | | Frameshifted by Chr_II:2108180!T→TA; coordinates changed from complement(join(2106448..2108178,2108181..2108786)) to complement(join(2106448..2108180,2108180..2108786)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPBC29A3.08">SPBC29A3.08</a> | pof4 | Frameshifted by Chr_II:2053516!G→GC; coordinates changed from join(2053033..2053517,2053519..2053600) to join(2053033..2053517,2053517..2053823) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPBC1A4.06c">SPBC1A4.06c</a> | tam41 | Frameshifted by Chr_II:1987101!CG→C and Chr_II:1987117!TG→T; coordinates changed from complement(join(1987044..1987099,1987102..1987675,1987737..1988015,1988081..1988353)) to complement(join(1987044..1987100,1987102..1987117,1987119..1987675,1987737..1988015,1988081..1988353)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPBC1E8.03c">SPBC1E8.03c</a> | | Frameshifted by Chr_II:1960392!A→AG; coordinates changed from complement(1960384..1961817) to complement(join(1960166..1960392,1960392..1961817)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPBC16E9.16c">SPBC16E9.16c</a> | lsd90 | Frameshifted by Chr_II:1948953!GA→G and Chr_II:1950050!A→AG; coordinates changed from complement(join(1947985..1948953,1948955..1950050,1950053..1950255)) to complement(join(1947985..1948952,1948954..1950048,1950048..1950255)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAC29A4.03c">SPAC29A4.03c</a> | mrps9 | Frameshifted by Chr_I:5142627!A→AG; coordinates changed from join(5142407..5142628,5142631..5143224) to join(5142407..5142626,5142626..5143224) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAC29E6.04">SPAC29E6.04</a> | nnf1 | Frameshifted by Chr_I:4410191!CG→C; coordinates changed from join(4409773..4410192, 4410194..4410391) to join(4409773..4410190,4410192..4410391) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAC29E6.03c">SPAC29E6.03c</a> | uso1 | Frameshifted by Chr_I:4407494!T→TG; coordinates changed from complement(join(4405990..4407375,4407447..4409096,4409171..4409338)) to complement(join(4405990..4407494,4407494..4409096,4409171..4409338)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAC1071.01c">SPAC1071.01c</a> | pta1 | Frameshifted by Chr_I:3855790!GT→G; coordinates changed from complement(3855780..3857792) to complement(join(3855626..3855790,3855792..3857792)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAC3A11.09">SPAC3A11.09</a> | sod22 | Frameshifted by Chr_I:3450130!GT→G; coordinates changed from complement(join(3450114..3452269,3452485..3452608)) to complement(join(3450056..3450130,3450132..3452269,3452485..3452608)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAC1486.05">SPAC1486.05</a> | nup189 | Frameshifted by Chr_I:3197528!A→AG; coordinates changed from join(3197028..3197529,3197529..3202450) to join(3197028..3197529,3197529..3202450) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAC688.08">SPAC688.08</a> | srb8 | Frameshifted by Chr_I:3125118!A→AT; coordinates changed from join(3123163..3125057,3125099..3126310,3126356..3126653) to join(3123163..3125118,3125118..3126310,3126356..3126653) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAC823.04">SPAC823.04</a> | rrp36 | Frameshifted by Chr_I:2588066!C→CA and Chr_I:2588021!C→CA; coordinates changed from join(2587729..2587897,2587952..2588028,2588031..2588143,2588182..2588620) to join(2587729..2587897,2587952..2588025,2588025..2588066,2588066..2588620) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAC22F3.11c">SPAC22F3.11c</a> | snu23 | Frameshifted by Chr_I:682993!TC-&gt;T; coordinates changed to join(682874..682996,682998..683109,683160..683446) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAC17G8.01c">SPAC17G8.01c</a> | trl1 | Frameshifted by Chr_I:2343703!G→GA; coordinates changed from complement(join(2341346..2343706,2343754..2343756)) to complement(join(2341346..2343703,2343703..2343756)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAP27G11.10c">SPAP27G11.10c</a> | nup184 | Frameshifted by Chr_I:1625092!T→TC; coordinates changed from complement(join(1625076..1629753,1629825..1629841)) to complement(join(1624885..1625092,1625092..1629753,1629825..1629841)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPAC3A12.04c">SPAC3A12.04c</a> | rpp1 | Frameshifted by Chr_I:1424708!CA→C; coordinates changed from complement(join(1424697..1425161,1425231..1425329,1425372..1425430,1425478..1425562)) to complement(join(1424660..1424708,1424710..1425161,1425231..1425329,1425372..1425430,1425478..1425562)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du | <span class="no-break">2017-04-04</span> |
<a routerLink="/gene/SPBC21B10.12">SPBC21B10.12</a> | rec6 | Coordinates changed from complement(join(1649038..1649472,1649558..1649671)) to complement(join(1649038..1649472,1649558..1649727,1649815..1649878,1650043..1650156)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26917764">PMID:26917764</a> | <span class="no-break">2016-04-19</span> |
<a routerLink="/gene/SPAC22F3.11c">SPAC22F3.11c</a> | snu23 | Coordinates changed from complement(890220..890546) to join(682874..682996,682998..683109,683160..683446) based on ribosome profiling | <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22365419">PMID:22365419</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a> | <span class="no-break">2016-03-17</span> |
<a routerLink="/gene/SPAC29A4.23">SPAC29A4.23</a> | | Frameshifted; intron/exon boundary changes; coordinates changed to join(5139024..5139190,5139192..5139424,5139427..5139575) | <a href="http://www.ncbi.nlm.nih.gov/pubmed/26494834">PMID:26494834</a> | <span class="no-break">2015-11-28</span> |
<a routerLink="/gene/SPBC13G1.04c">SPBC13G1.04c</a> | abh1 | N-terminal shortened to use downstream methionine | <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22365419">PMID:22365419</a> | <span class="no-break">2015-10-13</span> |
<a routerLink="/gene/SPAC1486.05">SPAC1486.05</a> | nup189 | Coordinates changed from join(3197028..3197443,3197530..3202450) to join(3197028..3197528,3197528..3202450) | pers. comm. Y. Hiraoka | <span class="no-break">2015-04-14</span> |
<a routerLink="/gene/SPCC320.09">SPCC320.09</a> | hem15 | Coordinates changed from complement(join(149153..150163,150279..150323,150374..150472)) to complement(join(149153..150163,150279..150323,150374..150589)) based on ribosome profiling | <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a> | <span class="no-break">2014-07-04</span> |
<a routerLink="/gene/SPCC417.03">SPCC417.03</a> | | Coordinates changed from join(1672003..1672051,1672111..1672364) to join(1672003..1672051,1672243..1672332) based on ribosome profiling | <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a> | <span class="no-break">2014-07-04</span> |
<a routerLink="/gene/SPCC162.07">SPCC162.07</a> | ent1 | Coordinates changed from complement(join(1571648..1573733,1574016..1574050)) to complement(join(1571648..1573733,1574258..1574280)) based on ribosome profiling | <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a> | <span class="no-break">2014-07-04</span> |
<a routerLink="/gene/SPBC27B12.10c">SPBC27B12.10c</a> | tom7 | Coordinates changed from complement(join(1344032..1344157,1344214..1344237)) to complement(join(1343913..1343934,1343989..1344025,1344082..1344157,1344214..1344237)) based on ribosome profiling | <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a> | <span class="no-break">2014-07-04</span> |
<a routerLink="/gene/SPBC146.01">SPBC146.01</a> | med15 | Coordinates changed from join(996843..997019,997066..1000080) to join(996797..996870,996917..997019,997066..1000080) based on ribosome profiling | <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a> | <span class="no-break">2014-07-04</span> |
<a routerLink="/gene/SPAC823.02">SPAC823.02</a> | | Coordinates changed from join(2583804..2583902,2583941..2583973) to join(2583863..2583902,2583941..2584047) based on ribosome profiling | <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437"></a> | <span class="no-break">2014-07-04</span> |
<a routerLink="/gene/SPBC839.02">SPBC839.02</a> | | Coordinates changed from join(597611..599124,599256..599334) to 597611..599125 based on ribosome profiling | <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a> | <span class="no-break">2014-07-04</span> |
<a routerLink="/gene/SPAC1D4.08">SPAC1D4.08</a> | pis1 | Coordinates changed from join(650814..650893,651173..651848) to join(650488..650545,651193..651848) based on ribosome profiling | <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a> | <span class="no-break">2014-07-04</span> |
<a routerLink="/gene/SPAC22F3.04">SPAC22F3.04</a> | mug62 | Coordinates changed from complement(join(698033..698177, 698223..698441, 698480..698841, 698884..699104, 699148..699363, 699410..702533)) to complement(join(698033..698177, 698223..698441, 698480..698841, 698884..699104, 699148..699363, 699410..702557, 702603..702755)) based on ribosome profiling | <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a> | <span class="no-break">2014-07-04</span> |
<a routerLink="/gene/SPAC110.05">SPAC110.05</a> | | Coordinates changed from complement(1905657..1905884) to complement(1905728..1905820) based on ribosome profiling | <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a> | <span class="no-break">2014-07-04</span> |
<a routerLink="/gene/SPSNRNA.07">SPSNRNA.07</a> | snu32 | Coordinates changed from 3959174-3958768 to 3959086-3958832, to represent the actual functional RNA | <a href="http://www.ncbi.nlm.nih.gov/pubmed/1560765">PMID:1560765</a> | <span class="no-break">2014-06-19</span> |
<a routerLink="/gene/SPSNRNA.06">SPSNRNA.06</a> | snu6 | Coordinates changed from (2562559-2562374,2562323-2562085) to (2562276-2562323,2562374-2562427), to represent the actual functional RNA | <a href="http://www.ncbi.nlm.nih.gov/pubmed/2909894">PMID:2909894</a> | <span class="no-break">2014-06-19</span> |
<a routerLink="/gene/SPSNRNA.05">SPSNRNA.05</a> | snu5 | Coordinates changed from 3236617-3237051 to 3236867-3236986, to represent the actual functional RNA | <a href="http://www.ncbi.nlm.nih.gov/pubmed/2587274">PMID:2587274</a> | <span class="no-break">2014-06-19</span> |
<a routerLink="/gene/SPSNRNA.04">SPSNRNA.04</a> | snu4 | Coordinates changed from 467025-467823 to 467233-4683361, to represent the actual functional RNA | <a href="http://www.ncbi.nlm.nih.gov/pubmed/2795654">PMID:2795654</a> | <span class="no-break">2014-06-19</span> |
<a routerLink="/gene/SPSNRNA.03">SPSNRNA.03</a> | snu3 | Coordinates changed from 988266-987631 to 987825-987570, to represent the actual functional RNA | <a href="http://www.ncbi.nlm.nih.gov/pubmed/3194197">PMID:3194197</a> | <span class="no-break">2014-06-19</span> |
<a routerLink="/gene/SPSNRNA.02">SPSNRNA.02</a> | snu2 | Coordinates changed from 959770-959305 to 959856-959370, to represent the actual functional RNA | <a href="http://www.ncbi.nlm.nih.gov/pubmed/3244367">PMID:3244367</a> | <span class="no-break">2014-06-19</span> |
<a routerLink="/gene/SPSNRNA.01">SPSNRNA.01</a> | snu1 | Coordinates changed from 3019819-3020472 to 3020205-3020353, to represent the actual functional RNA | <a href="http://www.ncbi.nlm.nih.gov/pubmed/2188102">PMID:2188102</a> | <span class="no-break">2014-06-19</span> |
<a routerLink="/gene/SPNCRNA.98">SPNCRNA.98</a> | srp7 | Coordinates changed from 4268566-4269064 to 4268662-4268916, to represent the actual functional RNA | <a href="http://www.ncbi.nlm.nih.gov/pubmed/2837764">PMID:2837764</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16453822">PMID:16453822</a> | <span class="no-break">2014-06-12</span> |
<a routerLink="/gene/SPAC5H10.06c">SPAC5H10.06c</a> | adh4 | N-terminal shortened to use downstream methionine; removed 43 amino acids; UTR exon annotated | <a href="http://www.ncbi.nlm.nih.gov/pubmed/24003116">PMID:24003116</a> | <span class="no-break">2014-01-07</span> |
<a routerLink="/gene/SPBC3D6.04c">SPBC3D6.04c</a> | mad1 | N-terminal shortened to use downstream methionine; removed 13 amino acids | pers. comm. Silke Hauf | <span class="no-break">2013-12-06</span> |
<a routerLink="/gene/SPBC25B2.07c">SPBC25B2.07c</a> | mmb1 | N-terminal shortened to use downstream methionine | <a href="http://www.ncbi.nlm.nih.gov/pubmed/21856157">PMID:21856157</a> | <span class="no-break">2013-12-06</span> |
<a routerLink="/gene/SPAC2F3.13c">SPAC9G1.15c</a> | mzt1 | N-terminal shortened to use downstream methionine | <a href="http://www.ncbi.nlm.nih.gov/pubmed/23885124">PMID:23885124</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24006493">PMID:24006493</a> | <span class="no-break">2013-11-01</span> |
<a routerLink="/gene/SPAC2F3.13c">SPAC2F3.13c</a> | | removed N terminal region overlapping with plp1 | | <span class="no-break">2012-12-07</span> |
<a routerLink="/gene/SPCC4B3.05c">SPCC4B3.05c</a> | hem12 | N terminal extended by 4 amino acids based on homology | | <span class="no-break">2012-01-05</span> |
<a routerLink="/gene/SPCC1281.07c">SPCC1281.07c</a> | | N terminal extended by 24 amino acids | | <span class="no-break">2012-01-05</span> |
<a routerLink="/gene/SPBC29A3.06">SPBC29A3.06</a> | | N terminal extended | | <span class="no-break">2012-01-05</span> |
<a routerLink="/gene/SPAC1006.03c">SPAC1006.03c</a> | red1 | intron donor corrected | | |
<a routerLink="/gene/SPAC22F3.11c">SPAC22F3.11c</a> | snu23 | gene structure updated | <a href="http://www.ncbi.nlm.nih.gov/pubmed/21511999">PMID:21511999</a> | <span class="no-break">2011-03-11</span> |
<a routerLink="/gene/SPBC947.10">SPBC947.10</a> | dsc1 | N-terminal extended to use upstream methionine | pers. comm. E. Stewart | <span class="no-break">2011-02-04</span> |
<a routerLink="/gene/SPACUNK4.14">SPACUNK4.14</a> | mdb1 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPAC18B11.05">SPAC18B11.05</a> | gpi18 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPAC18B11.05">SPAC18B11.05</a> | | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPAC13F5.07c">SPAC13F5.07c</a> | | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPBC1709.12">SPBC1709.12</a> | rid1 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPBC1711.10c">SPBC1711.10c</a> | npl4 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPAC3A11.03">SPAC3A11.03</a> | | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPBC16H5.12c">SPBC16H5.12c</a> | | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPBC4F6.10">SPBC4F6.10</a> | vps901 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPBC13E7.01">SPBC13E7.01</a> | cwf22 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPCC1322.16">SPCC1322.16</a> | phb2 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPCC4G3.12c">SPCC4G3.12c</a> | not2 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPAC29A4.03c">SPAC29A4.03c</a> | mrps9 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPBC25H2.11c">SPBC25H2.11c</a> | spt7 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPBC651.06">SPBC651.06</a> | mug166 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPBP8B7.31">SPBP8B7.31</a> | | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPBC29A10.17">SPBC29A10.17</a> | | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPAPB17E12.14c">SPAPB17E12.14c</a> | | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPAC688.04c">SPAC688.04c</a> | gst3 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPAC29A4.03c">SPAC29A4.03c</a> | mrps9 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPAC688.11">SPAC688.11</a> | end4 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPAC823.04">SPAC823.04</a> | | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a> | <span class="no-break">2011-02-03</span> |
<a routerLink="/gene/SPCC622.17">SPCC622.17</a> | apn1 | added 2 5' exons | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21193357">PMID:21193357</a> | <span class="no-break">2011-01-06</span> |
<a routerLink="/gene/SPCC306.10">SPCC306.10</a> | | pseudo-&gt;coding | pers. comm. Val Wood | <span class="no-break">2010-10-21</span> |
<a routerLink="/gene/SPBPB21E7.02c">SPBPB21E7.02c</a> | | pseudo-&gt;coding | pers. comm. Val Wood | <span class="no-break">2010-10-21</span> |
<a routerLink="/gene/SPBC8D2.07c">SPBC8D2.07c</a> | sfc9 | | pers. comm. Val Wood | <span class="no-break">2010-10-21</span> |
<a routerLink="/gene/SPBC3E7.04c">SPBC3E7.04c</a> | | pseudo-&gt;coding | pers. comm. Val Wood | <span class="no-break">2010-10-21</span> |
<a routerLink="/gene/SPBC1778.02">SPBC1778.02</a> | rap1 | | pers. comm. Yasushi Hiraoka | <span class="no-break">2010-10-21</span> |
<a routerLink="/gene/SPBC1348.12">SPBC1348.12</a> | | pseudo-&gt;coding | pers. comm. Val Wood | <span class="no-break">2010-10-21</span> |
<a routerLink="/gene/SPBC12D12.09">SPBC12D12.09</a> | rev7 | | pers. comm. Nicole Kosarek | <span class="no-break">2010-10-21</span> |
<a routerLink="/gene/SPAC3H1.04c">SPAC3H1.04c</a> | mdm31 | | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a> | <span class="no-break">2010-10-21</span> |
<a routerLink="/gene/SPAC22G7.04">SPAC22G7.04</a> | ubp13 | coordinates updated to join(732880..732897, 732942..733040, 733085..733251, 733369..733495, 733536..734113, 734151..735457, 735495..736465) | pers. comm. James Iben | <span class="no-break">2010-10-20</span> |
<a routerLink="/gene/SPAC27F1.09c">SPAC27F1.09c</a> | prp10 | added intron to match fully spliced | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9837997">PMID:9837997</a> | <span class="no-break">2010-09-27</span> |
<a routerLink="/gene/SPAC26H5.02c">SPAC26H5.02c</a> | nse2 | second intron extended to 115 nt | pers. comm. Val Wood | <span class="no-break">2010-08-20</span> |
<a routerLink="/gene/SPCC5E4.10c">SPCC5E4.10c</a> | | coordinates updated from 651622..651905 , 651973..651994 to 651622..651905, 651973..652045, 652115..652225, 652280..652371, 652419..652455 | pers. comm. K. Gould | <span class="no-break">2010-03-19</span> |
<a routerLink="/gene/SPAC458.04c">SPAC458.04c</a> | | coordinates updated from 4737605..4738147, 4738189..4738283, 4738330..4738714 to 4737605..4738147, 4738189..4738283, 4738330..4738740, 4738779..4738854 | pers. comm. Val Wood | <span class="no-break">2010-02-19</span> |
<a routerLink="/gene/SPAC29B12.08">SPAC29B12.08</a> | | replaced exon 5428786..5428883 with 5429038.. 5429045 | pers. comm. Val Wood | <span class="no-break">2009-12-14</span> |
<a routerLink="/gene/SPNCRNA.445">SPNCRNA.445</a> | snoR61 | updated to extend 3' and added splice and identified as snR6 | pers. comm. J. Matthews | <span class="no-break">2008-08-08</span> |
<a routerLink="/gene/SPBC2A9.07c">SPBC2A9.07c</a> | | removed intron | pers. comm. Cathrine Arnason Boe | <span class="no-break">2009-09-10</span> |
<a routerLink="/gene/SPBC800.02">SPBC800.02</a> | whi5 | increased intron by 42bp; old coordinates 253148..253231, 253310..253984, new coordinates 253148..253231, 253352..253984 | pers. comm. Rob de Bruin | <span class="no-break">2009-07-27</span> |
<a routerLink="/gene/SPBC18H10.08c">SPBC18H10.08c</a> | ubp4 | missing 2 N-terminal exons; old - complement(1784219..1785535); new - complement(join(1784219..1785748, 1785937..1786080, 1786141..1786248)) | pers. comm. I. Kouranti | <span class="no-break">2009-09-10</span> |
<a routerLink="/gene/SPBC800.02">SPBC800.02</a> | whi5 | increased intron by 42bp; old coordinates 253148..253231, 253310..253984, new coordinates 253148..253231, 253352..253984 | pers. comm. Rob de Bruin | <span class="no-break">2009-07-27</span> |
<a routerLink="/gene/SPBC16E9.16c">SPBC16E9.16c</a> | lsd90 | gene structure revised to remove introns, genome sequence contains 2 frameshifts | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18079165">PMID:18079165</a> | <span class="no-break">2008-01-03</span> |
<a routerLink="/gene/SPBP23A10.07">SPBP23A10.07</a> | rpa2 | truncated N-term to agree with orthologs; 2007960 -&gt; 2008119 | pers. comm. Val Wood | <span class="no-break">2009-03-30</span> |
<a routerLink="/gene/SPCC548.04">SPCC548.04</a> | | final exon 223687..223852 to 223687..223796, 223853.223860 | pers. comm. Marc Feuermann, Uniprot (SIB) | <span class="no-break">2009-02-23</span> |
<a routerLink="/gene/SPCC1442.04c">SPCC1442.04c</a> | | frameshifted; complement(1774200..1775429) to complement(join(1773925..1774239, 1774236..1775429)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a> | <span class="no-break">2009-02-02</span> |
<a routerLink="/gene/SPAC1F8.07c">SPAC1F8.07c</a> | | frameshifted; complement(101760..103544) to complement(join(101836..101872, 101872..103544)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a> | <span class="no-break">2009-02-02</span> |
<a routerLink="/gene/SPAC22F3.11c">SPAC22F3.11c</a> | snu23 | frameshifted; join(682874..683034, 683102..683109, 683160..683446) to join(682874..682994, 682996..683109, 683160..683446) | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a> | <span class="no-break">2009-02-02</span> |
<a routerLink="/gene/SPBC1A4.06c">SPBC1A4.06c</a> | | frameshifted; C term exon changed from 1987076..1987675 to 1987044....1987675 | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a> | <span class="no-break">2009-02-02</span> |
<a routerLink="/gene/SPBC29A3.08">SPBC29A3.08</a> | | frameshifted; truncated to 2048336..2049892, 2049894..2049896 | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a> | <span class="no-break">2009-02-02</span> |
<a routerLink="/gene/SPBC13E7.01">SPBC13E7.01</a> | | frameshifted | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a> | <span class="no-break">2009-02-02</span> |
<a routerLink="/gene/SPBC23G7.06c">SPBC23G7.06c</a> | | frameshifted; now a single exon | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a> | <span class="no-break">2009-02-02</span> |
<a routerLink="/gene/SPBC14C8.09c">SPBC14C8.09c</a> | | merged with SPBC14C8.08c; frameshifted | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a> | <span class="no-break">2009-02-02</span> |
<a routerLink="/gene/SPBC725.12">SPBC725.12</a> | nbl1 | old join(1227762..1227876, 1227926..1228098); new join(1227762..1228094, 1228166..1228307) | pers. comm. Kathy Gould | <span class="no-break">2009-01-30</span> |
<a routerLink="/gene/SPAC31G5.12c">SPAC31G5.12c</a> | maf1 | old complement(join(3008184..3008795, 3008914..3008957, 3009044..3009089)); new complement(join(3008184..3008795, 3009160..3009258, 3009312..3009317)) | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15590667">PMID:15590667</a> | <span class="no-break">2008-12-19</span> |
<a routerLink="/gene/SPAPB1E7.14">SPAPB1E7.14</a> | iec5 | old join(3322106..3322331, 3322379..3322512); new join(3321979..3322018, 3322071..3322331, 3322379..3322512) | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19040720">PMID:19040720</a> | <span class="no-break">2008-12-18</span> |
<a routerLink="/gene/SPCC962.05">SPCC962.05</a> | ast1 | intron position moved | pers comm. M. O' Connell | <span class="no-break">2008-12-06</span> |
<a routerLink="/gene/SPAC27D7.12c">SPAC27D7.12c</a> | but1 | extended N-terminal to use first MET as start | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14623327">PMID:14623327</a> | <span class="no-break">2008-11-11</span> |
<a routerLink="/gene/SPBC11B10.10c">SPBC11B10.10c</a> | pht1 | N terminal shortened to 2nd methionine; starting coordinate changed from 1503157 to 1503064 | pers. comm. L Buchanan and F Stewart | <span class="no-break">2008-09-05</span> |
<a routerLink="/gene/SPAC959.05c">SPAC959.05c</a> | | merged with SPAC959.06c; splicing is tentative | pers. comm. Charley Chahwan | <span class="no-break">2008-08-15</span> |
<a routerLink="/gene/SPBC18E5.14c">SPBC18E5.14c</a> | | merged with SPBC18E5.09c | pers. comm. Charley Chahwan | <span class="no-break">2008-07-31</span> |
<a routerLink="/gene/SPBC651.06">SPBC651.06</a> | mug166 | merged with SPBC651.07 | pers. comm. Charley Chahwan | <span class="no-break">2008-07-26</span> |
<a routerLink="/gene/SPBC31F10.08">SPBC31F10.08</a> | mde2 | old coordinates 3764820..3765449, new join(3764820..3765437, 3765491..3765709) | pers. comm. Charley Chahwan | <span class="no-break">2008-07-18</span> |
<a routerLink="/gene/SPBC2G2.09c">SPBC2G2.09c</a> | crs1 | added new C-terminal exon, deleted exon 3452822..3452877, extended intron 2 by 4 nucleotides | pers. comm. Charley Chahwan | <span class="no-break">2008-07-09</span> |
<a routerLink="/gene/SPCC338.08">SPCC338.08</a> | ctp1 | old coordinates complement(1358962..1359819), new complement(join(1358885..1359082, 1359121..1359819)); further altered to complement(join(1358891..1359082, 1359127..1359819)) in accordance with paper | pers. comm. Paul Russell; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18378696">PMID:18378696</a> | <span class="no-break">2008-01-11</span>;<span class="no-break">2008-06-19</span> |
<a routerLink="/gene/SPCC962.05">SPCC962.05</a> | | old join(554521..554722, 554792..556200), new join(554521..554744, 554865..556200) | pers. comm. Matthew O'Connell | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC6F6.16c">SPAC6F6.16c</a> | | merged with SPAC6F6.18c and five introns added | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18535244">PMID:18535244</a> | <span class="no-break">2008-06-09</span> |
<a routerLink="/gene/SPCC63.10c">SPCC63.10c</a> | | old complement(join(854651..855695, 855766..855800, 855839..855926, 855984..856213)), new complement(join(854651..855926, 855984..856213)); deleted 2nd and 3rd introns, were not required or supported | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPCC1827.02c">SPCC1827.02c</a> | | old complement(join(2377039..2378078, 2378197..2378221)), new complement(2377039..2378127); deleted first exon, unsupported | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPCC1450.14c">SPCC1450.14c</a> | | old complement(join(1759295..1760998, 1761117..1761128)), new complement(1759295..1760998); deleted first exon, unsupported | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPCC1393.06c">SPCC1393.06c</a> | | old complement(join(804627..805627, 805678..805800, 805907..806009)), new complement(join(804627..805627 , 805678..805800 , 805849..805921 , 805965..806009)); changed first intron and added a new 2nd exon, improved homology, also confirmed by transread data | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBP35G2.06c">SPBP35G2.06c</a> | | old complement(join(973049..974181, 974223..974577, 974617..975002, 975044..975137, 975172..976632)), new complement(join(973049..974181, 974223..974577, 974647..975002, 975044..975137, 975172..976632)); changed donor for intron 3 | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC32H8.08c">SPBC32H8.08c</a> | | old complement(join(1465811..1466035, 1466138..1467057, 1467311..1467456, 1467514..1467539)), new complement(join(1465811..1466035 , 1466138..1467057 , 1467311..1467431)); removed 5' intron/exon (unsupported) and trimmed to next met | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC21.03c">SPBC21.03c</a> | | old complement(join(3217985..3218424, 3218467..3218491, 3218693..3218947)), new complement(join(3217985..3218424, 3218462..3218491, 3218580..3218651, 3218704..3218947)); changed 2 internal introns, now agrees with transread data, but still potential to be incorrect | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC19G7.06">SPBC19G7.06</a> | | old join(2359429..2359479 , 2359547..2359693 , 2359845..2359928 , 2360001..2361092), new join(2359429..2359479, 2359547..2359693, 2359845..2359928, 2360064..2361092); altered 3rd intron branch acceptor to agree with published | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC16C6.03c">SPBC16C6.03c</a> | | old complement(join(4332356..4332454, 4332746..4333123)), new complement(4332698..4333123); final exon, unsupported removed | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC15D4.13c">SPBC15D4.13c</a> | | old complement(join(3032492..3032710, 3032761..3032822, 3032865..3033222)), new complement(join(3032492..3032710, 3032761..3032822, 3032889..3033222)); better donor for first intron | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAPB17E12.03">SPAPB17E12.03</a> | | old join(1271889..1271972, 1272015..1272386, 1272483..1272962), new join(1271889..1271972, 1272015..1272962); removed in-frame splice, unsupported, improved homology | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC9G1.13c">SPAC9G1.13c</a> | | old complement(join(1995208..1996148, 1996325..1996413, 1996478..1996683)), new complement(join(1995208..1996148, 1996191..1996268, 1996325..1996413, 1996478..1996683)); added a new 3rd exon, improved homology | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC8C9.19">SPAC8C9.19</a> | | old join(3660430..3660438, 3660491..3660556, 3660607..3660713, 3660784..3660886), new join(3660430..3660438, 3660491..3660556, 3660607..3660717); deleted final exon, unsupported, improved length and alignment to orthologs | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC6F12.08c">SPAC6F12.08c</a> | | old complement(join(1322439..1323955, 1323999..1324140, 1324195..1324234, 1324299..1324336)), new complement(join(1322439..1323955, 1323999..1324140, 1324195..1324252, 1324299..1324336)); improved branch acceptor for intron 1 | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC589.02c">SPAC589.02c</a> | | old complement(join(3095075..3098590, 3098626..3098775, 3098878..3098883)), new complement(join(3095075..3098590, 3098626..3098781, 3098878..3098883)); improved branch/acceptor for 1st intron | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC3A12.05c">SPAC3A12.05c</a> | | old complement(join(1426213..1427148, 1427189..1429617, 1429709..1429836, 1429882..1429913)), new complement(join(1426213..1427148, 1427189..1429617, 1429709..1429830, 1429882..1429913)); improved distance beteen branch and acceptor | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC328.05">SPAC328.05</a> | | old join(3484695..3484698, 3484752..3485065, 3485241..3485342, 3485382..3485528, 3485582..3485807, 3485910..3485974, 3486056..3486502), new join(3484695..3484698, 3484752..3485065, 3485241..3485342, 3485382..3485528, 3485582..3485974, 3486056..3486502); removed in-frame splice (exon 5), unsupported | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC2C4.06c">SPAC2C4.06c</a> | | old complement(join(4269367..4270554, 4270587..4270593, 4270638..4270674, 4270736..4270871)), new complement(join(4269367..4270554, 4270631..4270674, 4270736..4270871)); replaced 2 internal exons with a different singe exon | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC26H5.04">SPAC26H5.04</a> | | old join(4122149..4123354, 4123535..4124518), new 4122149..4124518; removed unsupported in-frame splice | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC23H3.04">SPAC23H3.04</a> | | old join(2499035..2499164, 2499325..2499353, 2499443..2499824, 2499875..2500236), new join(2499035..2499164, 2499160..2499353, 2499443..2499824, 2499875..2500236); changed 5' 1st and 2nd exon, improved homology | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC1B3.10c">SPAC1B3.10c</a> | | old complement(join(4945933..4945984, 4946094..4948084)), new complement(4946090..4948084); removed 3' exon, unsupported | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC1751.02c">SPAC1751.02c</a> | | old complement(join(385755..385887, 385938..386043, 386107..386179, 386296..386310)), new complement(join(385755..385887, 385938..386043, 386107..386149)); remove 5' exon, trimmed to new Met, improved homology | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC11D3.07c">SPAC11D3.07c</a> | | old complement(join(118195..119827, 119930..120108)), new complement(118195..120108); removed in-frame splice, not required or supported | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC1142.01">SPAC1142.01</a> | | old join(3625447..3625460, 3625555..3627544), new 3625574..3627544; removed 5' exon and trimmed to new Met, improved homology | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC1093.03">SPAC1093.03</a> | | old join(4613627..4613686, 4613723..4614081, 4614152..4614378, 4614438..4614954, 4614996..4616331), new join(4613627..4613686, 4613723..4614081, 4614164..4614378, 4614438..4614954, 4614996..4616331); improved distance between branch and acceptor, intron 2 | Solexa intron data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC16D10.10">SPBC16D10.10</a> | | old join(3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618971, 3619077..3619321, 3619366..3619540), new join(3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618918, 3618956..3619071, 3619113..3619321, 3619366..3619540); changed 2 introns (existing 4 and 5) to create a new exon 5, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAPB18E9.01">SPAPB18E9.01</a> | | old join(3974310..3974485, 3974541..3974629, 3974674..3974775, 3974879..3975132, 3975193..3975429, 3975485..3975832, 3975882..3975983), new join(3974310..3974485, 3974541..3974629, 3974674..3974775, 3974879..3975132, 3975193..3975429, 3975485..3975832, 3975882..3975962, 3976014..3976079); added small 3' exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC15A10.12c">SPAC15A10.12c</a> | | old complement(3709451..3709804), new complement(join(3709206..3709346, 3709499..3709804)); | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPCC1620.10">SPCC1620.10</a> | cwf26 | old 2163205..2164118, new join(2163205..2164118, 2164159..2164165); added 3' exon encoding single methionine | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPCC1620.09c">SPCC1620.09c</a> | tfg1 | old complement(join(2161527..2162363, 2162405..2162500, 2162584..2163123)), new complement(join(2161331..2161515, 2161565..2162363, 2162405..2162500, 2162584..2163123)); added 5' exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPCC1442.08c">SPCC1442.08c</a> | cox17 | old complement(join(1781505..1781563, 1781637..1781744, 1781830..1781914)), new complement(join(1781505..1781563, 1781637..1781744, 1781830..1781855, 1781920..1781935, 1781981..1782032)); added 2 small 5' exons, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPCC1020.11c">SPCC1020.11c</a> | | old join(757365..757424, 757506..757802), new join(757365..757424, 757506..757761, 757851..757861); added 3'intron/exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPCC16C4.16c">SPCC16C4.16c</a> | | old complement(join(697862..698127 , 698180..698399)), new complement(join(697605..697752, 697875..698127, 698180..698399)); added 3' intron/exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPCC16C4.01">SPCC16C4.01</a> | sif2 | old join(658832..659139, 659223..659488, 659567..659628, 659709..660413), new join(658832..659139, 659223..659488, 659567..659628, 659709..660197, 660298..660322); added 3' intron which changed frame of final exon, shorter product, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPCC736.12c">SPCC736.12c</a> | mmi1 | old complement(join(338026..338178, 338232..338347, 338395..339354, 339483..339567)), new complement(join(337829..337985, 338030..338178, 338232..338347, 338395..339354, 339483..339567); added 3' exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBPB2B2.18">SPBPB2B2.18</a> | | old join(4502439..4502463, 4502712..4502857, 4503068..4503286, 4503527..4503664), new join(4502692..4502725, 4502787..4502857, 4503068..4503286, 4503527..4503664); changed 5' intron/exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC21C3.07c">SPBC21C3.07c</a> | | old complement(join(3807920..3808607, 3808665..3808795), new complement(join(3807714..3807783, 3807831..3807891, 3807946..3808607, 3808665..3808795)); added two 3' exons | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC31F10.12">SPBC31F10.12</a> | | old join(3773353..3773525, 3773581..3773678, 3773795..3773999, 3774059..3774116, 3774192..3774212), new join(3773308..3773318, 3773373..3773525, 3773581..3773678, 3773795..3773999, 3774059..3774116, 3774192..3774212); added new 5' intron and exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC13G1.04c">SPBC13G1.04c</a> | | old complement(3732941..3733762), new complement(join(3732806..3732930, 3732994..3733762)); added new 3' intron and exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC16D10.02">SPBC16D10.02</a> | trm11 | old join(3588806..3589595, 3589639..3589803, 3589845..3590123, 3590163..3590287), new join(3588806..3589595, 3589639..3589803, 3589845..3590123, 3590163..3590278, 3590327..3590386); added new 3' intron/exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC2D10.16">SPBC2D10.16</a> | | old join(2996452..2996463, 2996509..2996904), new join(2996452..2996463, 2996509..2996687, 2996738..2996879); added new intron in second exon, which created a new 3rd exon, and identified homology with Cenp-S (was previously sequence orphan) | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC2D10.15c">SPBC2D10.15c</a> | | old complement(join(2995324..2995579, 2995619..2995923, 2995971..2996066)), new complement(join(2995324..2995497, 2995534..2995579, 2995619..2995923, 2995971..2996066)); altered first intron, added in-frame intron, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC2A9.07c">SPBC2A9.07c</a> | | old complement(2959078..2959902), new complement(join(2959078..2959129, 2959172..2959902)); added in-frame intron, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBP4H10.12">SPBP4H10.12</a> | | old join(2895855..2895863, 2896001..2896507), new join(2895855..2895863, 2896001..2896459, 2896516..2896587); added new 3' intron/exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC4.02c">SPBC4.02c</a> | | old complement(join(2369064..2369160, 2369246..2369416, 2369455..2371243, 2371351..2371451, 2371507..2371583)), new complement(join(2369064..2369190, 2369246..2369416, 2369455..2371243, 2371351..2371451, 2371507..2371583)); changed accceptor for final intron, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC8D2.17">SPBC8D2.17</a> | | old 1390183..1391238, new join(1390183..1391093, 1391147..1391222); added new 3' intron/exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC4.02c">SPBC4.02c</a> | | old complement(join(1185617..1186640, 1186738..1186846, 1187087..1187324)), new complement(join(1185617..1186640, 1186738..1186846, 1187087..1187162, 1187220..1187324)); added in-frame splice to first (existing) exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBPJ4664.05">SPBPJ4664.05</a> | | old 706175..706666, new join(705932..706077, 706141..706666); added new 5' intron and exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC947.03c">SPBC947.03c</a> | | old 676629..676931, new join(676629..676908, 677057..677127); added 5' inton/exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC582.04c">SPBC582.04c</a> | | old complement(join(423269..423598, 423669..423738, 423782..423871, 423912..424373, 424424..424872, 424921..425160, 425223..425350, 425410..425446)), new complement(join(423269..423598, 423669..423738, 423782..423871, 423912..424373, 424424..424692, 424747..424872, 424921..425160, 425223..425350, 425410..425446)); added in-frame intron to (existing) exon 3 | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC106.05c">SPBC106.05c</a> | tim11 | old complement(join(383527..383612, 383703..383874, 383980..384012)), new complement(join(383527..383612, 383703..383874, 383940..383960)); changed 5' exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC22E12.10c">SPAC22E12.10c</a> | etp1 | old complement(5037208..5039103), new complement(join(5037208..5037446, 5037492..5039103)); added in-frame splice | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC890.07c">SPAC890.07c</a> | rmt1 | old complement(join(5016072..5016215, 5016262..5016596, 5016649..5016943, 5016994..5017059, 5017138..5017252, 5017421..5017433, 5017700..5017703, 5017782..5017829)), new complement(join(5016020..5016026, 5016076..5016215, 5016262..5016596, 5016649..5016943, 5016994..5017059, 5017138..5017252, 5017421..5017433, 5017700..5017703, 5017782..5017829); added small N terminal exon, which added a single conserved cysteine | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC4F10.08">SPAC4F10.08</a> | mug126 | old join(4844855..4845824, 4845863..4846203), new join(4844855..4845824, 4845863..4846063, 4846104..4846129); added new 3' intron/exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC144.17c">SPAC144.17c</a> | | old complement(join(4687483..4688462, 4688513..4688831)), new complement(join(4687388..4687477, 4687522..4688462, 4688513..4688831)); added 3' intron and exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC30C2.05">SPAC30C2.05</a> | erv14 | old complement(join(4687483..4688462, 4688513..4688831)), new complement(join(4687388..4687477, 4687522..4688462, 4688513..4688831)); added new 5' intron/exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC30.03c">SPAC30.03c</a> | tsn1 | old join(4394707..4394741, 4394799..4394967, 4395010..4395092, 4395131..4395491, 4395530..4395592), new complement(join(4394707..4394741, 4394799..4394967, 4395010..4395092, 4395131..4395367, 4395416..4395491, 439 5530..4395592)); added in-frame intron to (previous) 2nd exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC2C4.11c">SPAC2C4.11c</a> | rpb28 | old complement(4278729..4279505), new complement(join(4278729..4279405, 4279457..4279505)); added small N term in-frame intron | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC2F3.18c">SPAC2F3.18c</a> | | old 3942398..3942730 new 3942299..3942364, 3942506..3942730; added small N term in-frame intron | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC1F12.08">SPAC1F12.08</a> | | old join(3817464..3817803, 3817851..3817991, 3818035..3818543), new join(3817464..3817803, 3817851..3817991, 3818035..3818130, 3818170..3818543); added new intron, in-frame splice in final exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC1556.03">SPAC1556.03</a> | azr1 | old join(3799244..3799680, 3799731..3800160), new join(3799244..3799680, 3799731..3800135, 3800184..3800241); added new 5' intron and exon/improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC1556.01c">SPAC1556.01c</a> | rad50 | old complement(join(3791855..3792229, 3792267..3795557, 3795601..3795684, 3795734..3795856)), new complement(join(3791831..3791851, 3791891..3792229, 3792267..3795557, 3795601..3795684, 3795734..3795856)); added small 3' intron and exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC15A10.10">SPAC15A10.10</a> | | old join(3697508..3697752, 3697795..3698368, 3698421..3698527, 3698570..3699538, 3699582..3699837), new join(3697508..3697752, 3697795..3698368, 3698421..3698527, 3698570..3699310, 3699359..3699538, 3699582..3699837); added in-frame intron | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC16E8.12c">SPAC16E8.12c</a> | | old complement(join(3523136..3523644, 3524027..3524030)), new complement(join(3523136..3523644, 3523925..3523996, 3524044..3524098)); added 2 5' exons | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAPB2B4.06">SPAPB2B4.06</a> | | old join(2722735..2723210, 2723273..2723300, 2723349..2723582), new join(2722735..2723210, 2723273..2723300, 2723349..2723559, 2723608..2723660); added new 3' intron/exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC823.09c">SPAC823.09c</a> | | old complement(join(2594922..2595693, 2595734..2595819)), new complement(join(2594777..2594913, 2594966..2595693, 2595734..2595819)); added new 3' intron/exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC823.04">SPAC823.04</a> | | old 2587729..2587897 , 2588085..2588620, new join(2587729..2587897, 2588106..2588620); altered to use valid acceptor | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAPB1A10.16">SPAPB1A10.16</a> | | old 1863972..1864388, new join(1863972..1864360, 1864415..1864457); added new 3' intron/exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC17A5.02c">SPAC17A5.02c</a> | dbr1 | old complement(join(1754109..1754310, 1754350..1755118, 1755164..1755285, 1755331..1755638)), new complement(join(1754033..1754145, 1754186..1754310, 1754350..1755118, 1755164..1755285, 1755331..1755638)); added new 5'intron/exon, now agrees with published date | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC9.06c">SPAC9.06c</a> | | old complement(join(1473692..1473885, 1473939..1474105, 1474153..1474212, 1474264..1474341, 1474460..1474539)), new complement(join(1473692..1473885, 1473939..1474105, 1474153..1474212, 1474264..1474341, 1474381..1474404, 1474460..1474539)); added new exon (2nd) improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAPB17E12.08">SPAPB17E12.08</a> | | old join(1279573..1279644, 1279820..1279869, 1279953..1280325), new join(1279573..1279779, 1279820..1279869, 1279953..1280325); changed first donor to make n-terminal exon longer, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC56F8.04c">SPAC56F8.04c</a> | coq2 | old complement(1133226..1134302), new complement(join(1133226..1134283, 1134332..1134356)); added 5' intron/exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC1A6.03c">SPAC1A6.03c</a> | | old complement(1070139..1072013), new complement(join(1070061..1070138, 1070205..1072013)); new 3' intron/exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC23C4.04c">SPAC23C4.04c</a> | | old 1036033..1036347, new join(1036033..1036342, 1036397..1036455); added intron | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC24H6.01c">SPAC24H6.01c</a> | | old join(490016..490262, 490338..490771, 490812..490931, 490979..491149, 491186..491980), new join(490016..490262, 490338..490771, 490812..490931, 490979..491149, 491186..491858, 491905..492011); added new 3' intron/exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC630.11">SPAC630.11</a> | vps55 | old join(366801..366817, 366873..366891, 366968..367040, 367123..367189, 367293..367348, 367395..367531), new join(366801..366817, 366873..366891, 366968..367040, 367105..367189, 367293..367348, 367395..367531); changed internal intron acceptor, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC1639.01c">SPAC1639.01c</a> | | old complement(join(251726..252337, 252392..252752, 253590..253630)), new complement(join(251726..251745, 251863..252337, 252392..252752, 253590..253630)); added new 3' intron/exon | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPAC13G6.04">SPAC13G6.04</a> | tim8 | old join(180423..180528, 180586..180776), new join(180423..180528, 180586..180721, 180890..180911); added new 3' intron/exon, improved homology | Solexa transread data | <span class="no-break">2008-05-02</span> |
<a routerLink="/gene/SPBC16C6.02c">SPBC16C6.02c</a> | | complement(join(4322221..4322742, 4322970..4331555, 4331641..4331793, 4331856..4331968, 4332007..4332028)) was split into complement(join(4322829..4331555, 4331641..4331793, 4331856..4331968, 4332007..4332028)) and new gene SPBC16C6.14 - complement(4322221..4322622) | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a> | <span class="no-break">2008-04-02</span> |
<a routerLink="/gene/SPBC16E9.16c">SPBC16E9.16c</a> | | This CDS has 2 frameshifts in GeneDB at present; coordinates changed from 1947985..1949069 , 1949218..1950058 , 1950253..1950255 to 1947985..1948953, 1948955..1950050, 1950053..1950255 | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18079165">PMID:18079165</a> | <span class="no-break">2007-01-03</span> |
<a routerLink="/gene/SPBC16D10.10">SPBC16D10.10</a> | | exon 4 splice altered; coordinates changed from 3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618920, 3619065..3619321, 3619366..3619540 to 3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618971, 3619077..3619321, 3619366..3619540 | Solexa intron data | <span class="no-break">2007-12-07</span> |
<a routerLink="/gene/SPBC19G7.06">SPBC19G7.06</a> | | coordinates changed from 2359429..2359479, 2359547..2359693, 2359845..2359928, 2360064..2361092 to 2359429..2359479, 2359547..2359693, 2359845..2359928, 2360001..2361092 | Christopher J. McInerny | <span class="no-break">2007-12-07</span> |
<a routerLink="/gene/SPAC806.09c">SPAC806.09c</a> | | changed from 251726..252337, 252392..252752, 252945..252958 to 251726..252337, 252392..252752, 253590..253630 | Solexa transcript data | <span class="no-break">2007-12-07</span> |
<a routerLink="/gene/SPAC29E6.03c">SPAC29E6.03c</a> | | changed internal splice site, better intron, slightly improved homology; deleted final small exon, unsupported; coordinates changed from 4405871..4405889, 4405994..4407291, 4407447..4409096, 4409171..4409338 to 4405990..4407375, 4407447..4409096, 4409171..4409338 | Solexa intron data | <span class="no-break">2007-12-03</span> |
<a routerLink="/gene/SPAC688.08">SPAC688.08</a> | | deleted final exon, not supported by Solexa data; coordinates changed from 3123163..3125057, 3125099..3126310, 3126356..3126631, 3126671..3126989 to 3123163..3125057, 3125099..3126310, 3126356..3126653 | Solexa intron data | <span class="no-break">2007-12-03</span> |
<a routerLink="/gene/SPAC823.04">SPAC823.04</a> | | changed first intron acceptor, which gave a different n-term exon and improved homology; coordinates changed from 2587859..2587901, 2588085..2588620 to 2587729..2587897, 2588085..2588620 | Solexa intron data | <span class="no-break">2007-12-03</span> |
<a routerLink="/gene/SPCC622.14">SPCC622.14</a> | | altered donor for intron 3, improved donor consensus; coordinates changed from 1427098..1427310, 1427632..1427709, 1427788..1428180, 1428277..1428522 to 1427098..1427310, 1427632..1427709, 1427788..1428216, 1428277..1428522 | Solexa intron data | <span class="no-break">2007-12-03</span> |
<a routerLink="/gene/SPCC1442.13c">SPCC1442.13c</a> | | deleted final exon, improved homology; coordinates changed from 1791196..1791285, 1791376..1791909 to 1791346..1791909 | Solexa intron data | <span class="no-break">2007-12-03</span> |
<a routerLink="/gene/SPCC1494.02">SPCC1494.02</a> | | 2 C-terminal exons removed, not supported by homology; coordinates changed from 2317988..2318068, 2318200..2318244, 2318398..2318508, 2318579..2318676, 2318903..2318966 to 2318335..2318508, 2318579..2318676, 2318903..2318966 | Solexa intron data | <span class="no-break">2007-12-03</span> |
<a routerLink="/gene/SPAPB1A10.03">SPAPB1A10.03</a> | | SPAPB1A10.03 split to create SPAPB1A10.03 and SPAPB1A10.16; new coordinates 1863363..1863381, 1863434..1863762 and 1863972..1864388 | Solexa intron data | <span class="no-break">2007-12-03</span> |
<a routerLink="/gene/SPBC11B10.03">SPBC11B10.03</a> | | removed N-teminal exon, poor donor, did not improve homology; coordinates changed from 1490376..1490423, 1490593..1491747 to 1490611..1491747 | Solexa intron data | <span class="no-break">2007-12-03</span> |
<a routerLink="/gene/SPBC16H5.04">SPBC16H5.04</a> | | altered intron 2 donor and acceptor, improved homology; coordinates changed from 2293393..2293824, 2293938..2294039, 2294087..2294125 to 2293393..2293775, 2293877..2294039, 2294087..2294125 | Solexa intron data | <span class="no-break">2007-12-03</span> |
<a routerLink="/gene/SPAC4C5.01">SPAC4C5.01</a> | | altered intron 5 to improve donor and acceptor, improves homology; coordinates changed from 1192545..1192592, 1192662..1192704, 1192779..1192841, 1192916..1193010, 1193062..1193175, 1193236..1193613 to 1192545..1192592, 1192662..1192704, 1192779..1192841, 1192916..1193010, 1193062..1193145, 1193197..1193613 | Solexa intron data | <span class="no-break">2007-12-03</span> |
<a routerLink="/gene/SPAC23D3.08">SPAC23D3.08</a> | usp108 | final exon deleted, not supported by Solexa intron data; coordinates changed from 4352514..4353359, 4353410..4353573, 4353623..4353761 to 4352514..4353359, 4353410..4353577 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPBC16A3.14">SPBC16A3.14</a> | | final exon deleted, not supported by Solexa intron data; coordinates changed from 4270907..4270939, 4271137..4271937 to 4271131..4271937 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPBC1604.17c">SPBC1604.17c</a> | | updated, better donor (GTAATA instead of GTTAGT); original intron prediction not supported by Solexa data; coordinates changed from 3900047..3900121, 3900166..3901146, 3901219..3901542 to 3900047..3900121, 3900166..3901170, 3901219..3901542 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPBC29A3.07c">SPBC29A3.07c</a> | | final exon deleted, not supported by Solexa intron data; homology improved; coordinates changed from 2050069..2050183, 2050304..2050358, 2050452..2050695 to 2050258..2050358, 2050452..2050695 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPBC21C3.07c">SPBC21C3.07c</a> | | removed final exon, not supported by Solexa transcript data; coordinates changed from 3807714..3807783, 3807963..3808607, 3808665..3808795 to 3807920..3808607, 3808665..3808795 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPBC21B10.02">SPBC21B10.02</a> | | intron unsupported by Solexa transcript data, removed final exon; coordinates changed from 1669624..1669695, 1669737..1670144 to 1669719..1670144 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPAC227.11c">SPAC227.11c</a> | | removed final exon, unsupported by Solexa data; coordinates changed from 514434..514489, 514557..515082, 515222..515482, 515534..515659 to 514552..515082, 515222..515482, 515534..515659 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPCC736.05">SPCC736.05</a> | wtf7 | removed final exon, unsupported by Solexa intron data; coordinates changed from 320617..320755, 320964..321335, 321387..321509, 321554..321582 to 320617..320755, 320964..321335, 321387..321523 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPCC1450.09c">SPCC1450.09c</a> | | original intron absent from Solexa data; C terminal exon unsupported so exon deleted; coordinates changed from 1741451..1741481, 1741589..1743459 to 1741588..1743459 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPAC56F8.05c">SPAC56F8.05c</a> | | revised intron structure/original intron absent from Solexa data. N terminal exon unsupported and had no branch site so exon deleted; coordinates changed from 1134697..1135061, 1135195..1135402, 1135517..1135684, 1135729..1135776, 1135932..1135970, 1136111..1136170 to 1134697..1135061, 1135195..1135402, 1135517..1135684, 1135729..1135776, 1135932..1136003 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPAPB1A10.15">SPAPB1A10.15</a> | | revised intron structure/original intron absent from Solexa data so small C-term exon deleted; coordinates changed from 1892554..1892640, 1892926..1893060, 1893098..1893504, 1893585..1893618 to 1892554..1892640, 1892926..1893060, 1893098..1893694 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPAPB17E12.08">SPAPB17E12.08</a> | | revised intron structure/original intron absent from Solexa data so small N-term exon deleted; coordinates changed from 1279492..1279506, 1279582..1279644, 1279820..1279869, 1279953..1280325 to 1279573..1279644, 1279820..1279869, 1279953..1280325 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPAC17C9.08">SPAC17C9.08</a> | | revised intron structure/original intron absent from Solexa data and unsupported from mRNA data so final exon deleted; coordinates changed from 4484758..4484804, 4484843..4485274, 4485350..4485596, 4485713..4485994 to 4484835..4485274, 4485350..4485596, 4485713..4485994 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPAC16A10.06c">SPAC16A10.06c</a> | | revised intron structure/original intron absent from Solexa data; coordinates updated from 3089583..3089996, 3090112..3090285, 3090533..3090697 to 3089583..3090047, 3090112..3090285, 3090533..3090697 | Solexa intron data | <span class="no-break">2007-12-02</span> |
<a routerLink="/gene/SPCC18.18c">SPCC18.18c</a> | | coordinates changed from 1988631..1990079 to 1988631..1990100, 1990630..1990722 | Solexa transcript data | <span class="no-break">2007-11-29</span> |
<a routerLink="/gene/SPAC631.02">SPAC631.02</a> | | coordinates updated from 2107940..2110123 to 2107470..2107515, 2107860..2110123 | Solexa transcript data | <span class="no-break">2007-11-28</span> |
<a routerLink="/gene/SPBC354.07c">SPBC354.07c</a> | | additional exon added; coordinates updated from 558335..559225, 559273..559474, 559942..560028, 560151..560170 to 558335..559225, 559273..559474, 559942..560028, 560151..560163, 560231..560246 | Solexa transcript data | <span class="no-break">2007-11-28</span> |
<a routerLink="/gene/SPAC6G9.16c">SPAC6G9.16c</a> | | coordinates changed from 3278951..3279349 to join(3278951..3279367, 3279420..3279539, 3279683..3279819, 3279865..3279985) | Solexa transcript data | <span class="no-break">2007-11-22</span> |
<a routerLink="/gene/SPBPB2B2.07c">SPBPB2B2.07c</a> | | coordinates updated from 4474434..4474931 to join(4474145..4474403, 4474438..4474931) | Solexa transcript data | <span class="no-break">2007-11-22</span> |
<a routerLink="/gene/SPCC1529.01">SPCC1529.01</a> | | new exon added; coordinates changed from join(237554..237923, 237975..238080, 238139..238533, 238588..239105) to join(237554..237923, 237975..238080, 238139..238533, 238588..239092, 239151..239250) | Solexa transcript data | <span class="no-break">2007-10-12</span> |
<a routerLink="/gene/SPCC1020.11c">SPCC1020.11c</a> | | coordinates updated from 757494..757802 to join(757365..757424, 757506..757802) | Solexa transcript data | <span class="no-break">2007-10-12</span> |
<a routerLink="/gene/SPAC22F8.07c">SPAC22F8.07c</a> | rtf1 | Two in-frame splices added | <a href="http://www.ebi.ac.uk/ena/data/view/AJ627891">EMBL:AJ627891</a> | <span class="no-break">2007-08-28</span> |
<a routerLink="/gene/SPAC9E9.02">SPAC9E9.02</a> | | altered to include stop codon | pers. comm. Val Wood | <span class="no-break">2007-07-19</span> |
<a routerLink="/gene/SPBC409.12">SPBC409.12</a> | stn1 | N-terminal extended by 2 exons | pers. comm. C. Chahwan | <span class="no-break">2007-07-06</span> |
<a routerLink="/gene/SPAC23C4.04c">SPAC23C4.04c</a> | | altered to include stop codon and trimmed to methionine | pers. comm. Val Wood | <span class="no-break">2007-06-19</span> |
<a routerLink="/gene/SPAC29E6.04">SPAC29E6.04</a> | nnf1 | Sequence frameshifted at 4410192, new coordinates give an N terminal extension from 4410194..4410391 (NOTE: Still needs resequencing and underlying sequence changing) | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17035632">PMID:17035632</a> | <span class="no-break">2007-05-16</span> |
<a routerLink="/gene/SPAC13G7.07">SPAC13G7.07</a> | | changed second exon 3' boundary and added C-terminal exons | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17310250">PMID:17310250</a> | <span class="no-break">2007-03-23</span> |
<a routerLink="/gene/SPNCRNA.82">SPNCRNA.82</a> | mrp1 | coordinates updated from 1234705..1234837 to 1234451..1234849, upon realisation that this corresponded to RNase MRP | pers. comm. Val Wood | <span class="no-break">2007-02-07</span> |
<a routerLink="/gene/SPAC23D3.08">SPAC23D3.08</a> | usp108 | final exon removed | pers. comm. Val Wood | <span class="no-break">2007-02-05</span> |
<a routerLink="/gene/SPBP22H7.09c">SPBP22H7.09c</a> | mis15 | updated sequence coordinates as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15369671">PMID:15369671</a> | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15369671">PMID:15369671</a> | <span class="no-break">2007-01-05</span> |
<a routerLink="/gene/SPCC1223.10c">SPCC1223.10c</a> | eaf1 | Added exons as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17150956">PMID:17150956</a> | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17150956">PMID:17150956</a> | <span class="no-break">2006-12-14</span> |
<a routerLink="/gene/SPAPB17E12.08">SPAPB17E12.08</a> | | coordinates updated from join(9968..9990, 10070..10442) to join(9609..9623, 9699..9761, 9937..9986, 10070..10442). Improved homology | pers. comm. Val Wood | <span class="no-break">2006-12-12</span> |
<a routerLink="/gene/SPBC30B4.08">SPBC30B4.08</a> | eri1 | gene structure updated as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16797182">PMID:16797182</a> | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16797182">PMID:16797182</a> | <span class="no-break">2006-09-22</span> |
<a routerLink="/gene/SPAC12D12.09">SPAC12D12.09</a> | rev7 | truncated penultimate exon to remove GIN | pers. comm. Nicole Kosarek | <span class="no-break">2006-08-01</span> |
<a routerLink="/gene/SPAC12D12.09">SPAC12D12.09</a> | rev7 | added new exon to insert KCIDIFGEF | pers. comm. Nicole Kosarek | <span class="no-break">2006-08-01</span> |
<a routerLink="/gene/SPCC4B3.05c">SPCC4B3.05c</a> | hem12 | Coordinates updated from join(28064..28097, 28155..28189, 28238..28816, 28866..29330) to join(28064..28097, 28155..28189, 28236..28260, 28308..28816, 28866..29330). Improves Homology | pers. comm. Val Wood | <span class="no-break">2006-06-12</span> |
<a routerLink="/gene/SPCPJ732.03">SPCPJ732.03</a> | meu15 | new coordinates as per publication | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11376151">PMID:11376151</a> | <span class="no-break">2006-06-12</span> |
<a routerLink="/gene/SPAC21E11.05c">SPAC21E11.05c</a> | cyp8 | Single base insertion allowed first 2 exons to be merged and extended (GAGT[G]GCA) | pers. comm. Trevor Pemberton (via Ivo Pedruzzi, UniProt) | <span class="no-break">2006-05-22</span> |
<a routerLink="/gene/SPAC6G9.13c">SPAC6G9.13c</a> | bqt1 | Additional C terminal exon | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16615890">PMID:16615890</a> | <span class="no-break">2006-05-03</span> |
<a routerLink="/gene/SPBCPT2R1.08c">SPBCPT2R1.08c</a> | tlh2 | truncated to 1919 a.a.; removed truncated version of a malate transporter fused to gene, not known whether these residues are included in translation | pers. comm. Liew Li Phing | <span class="no-break">2006-04-13</span> |
<a routerLink="/gene/SPAC1D4.11c">SPAC1D4.11c</a> | lkh1 | lengthened to 690 a.a. | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12565823">PMID:12565823</a> | <span class="no-break">2006-04-12</span> |
<a routerLink="/gene/SPCC4G3.05c">SPCC4G3.05c</a> | mus81 | added N terminal exon | pers. comm. Fred Kippert | <span class="no-break">2006-02-17</span> |
<a routerLink="/gene/SPBC3B9.22c">SPBC3B9.22c</a> | dad4 | added new C-terminal exon based on Pfam protein family | <a href="http://pfam.xfam.org/">Pfam</a> | <span class="no-break">2006-02-01</span> |
<a routerLink="/gene/SPAC9.05">SPAC9.05</a> | fml1 | second intron acceptor extended by 15 bp | pers. comm. Anke Schürer | <span class="no-break">2006-01-19</span> |
<a routerLink="/gene/SPAC31G5.06">SPAC31G5.06</a> | | added N-teminal exon to extend prediction | pers. comm. Val Wood | <span class="no-break">2005-11-01</span> |
<a routerLink="/gene/SPAC17H9.20">SPAC17H9.20</a> | psc3 | added intron at 37250..37297 | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16207082">PMID:16207082</a> | <span class="no-break">2005-10-10</span> |
SPCC1223.15c | spc19 | 4 amino acid extension at N-terminus | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16079914">PMID:16079914</a> | <span class="no-break">2005-09-09</span> |
<a routerLink="/gene/SPBC15D4.02">SPBC15D4.02</a> | | trimmed N terminal 28.10.03; looked overextended as large low complexity region in front in zinc finger which is usually N-term; alignment looks better | pers. comm. Val Wood | <span class="no-break">2003-10-28</span> |
<a routerLink="/gene/SPAC959.05c">SPAC959.05c</a> | | extended N-terminal to first methionine | pers. comm. Val Wood | <span class="no-break">2003-09-14</span> |
<a routerLink="/gene/SPAC29B12.08">SPAC29B12.08</a> | | new N-terminal exon | pers. comm. Aengus Stewart | <span class="no-break">2003-09-09</span> |
<a routerLink="/gene/SPAC2F3.13c">SPAC2F3.13c</a> | | annotated as SPAC2F3.13c putative tRNA-ribosyltransferase, but actually a gene merge. Split to create SPAC2F3.13c the tRNA-ribosyltransferase and new CDS SPAC2F3.17c was created from the C-terminal | pers. comm. Val Wood | <span class="no-break">2003-06-30</span> |
<a routerLink="/gene/SPAC19B12.06c">SPAC19B12.06c</a> | | added exon 18153..18218 | pers. comm. Val Wood | <span class="no-break">2003-04-22</span> |
<a routerLink="/gene/SPAC31G5.07">SPAC31G5.07</a> | | N terminal extended to use longest ORF in the absence of homology | pers. comm. Henar Valdivieso Montero | <span class="no-break">2003-01-08</span> |
<a routerLink="/gene/SPAC29A4.14c">SPAC29A4.14c</a> | | N-terminal extended | pers. comm. Val Wood | <span class="no-break">2002-09-11</span> |
<a routerLink="/gene/SPAC22F3.03c">SPAC22F3.03c</a> | rdh54 | 4th exon extended by altering acceptor in intron 3 cosmid coordinates 34092-33923 | pers. comm. Mike Catlett | <span class="no-break">2003-05-09</span> |
<a routerLink="/gene/SPAC2F3.13c">SPAC2F3.13c</a> | | split to create SPAC2F3.13c and SPAC2F3.12c | pers. comm. Val Wood | <span class="no-break">2002-06-30</span> |
<a routerLink="/gene/SPAC2F3.12c">SPAC2F3.12c</a> | | created from the C-terminal of SPAC2F3.13c | pers. comm. Val Wood | <span class="no-break">2002-06-30</span> |
<a routerLink="/gene/SPBC1347.05c">SPBC1347.05c</a> | | new C-terminal exon | pers. comm. Val Wood | <span class="no-break">2002-10-11</span> |
<a routerLink="/gene/SPAC23C11.10">SPAC23C11.10</a> | | found 3 additional C-terminal exons by eye | pers. comm. Val Wood | <span class="no-break">2002-06-18</span> |
<a routerLink="/gene/SPAC1296.04">SPAC1296.04</a> | | has 2 additional N-terminal exons | pers. comm. Val Wood | |
<a routerLink="/gene/SPBC1604.17c">SPBC1604.17c</a> | | previous coordinates (8945..8971, 9475..10455, 10528..10851) | pers. comm. Val Wood | <span class="no-break">2002-04-29</span> |
<a routerLink="/gene/SPCC1442.13c">SPCC1442.13c</a> | | C terminal exon added ; old coordinates (26947..27936); new exon extends G-patch domain | pers. comm. Val Wood | <span class="no-break">2002-04-15</span> |
<a routerLink="/gene/SPBP23A10.04">SPBP23A10.04</a> | apc2 | coordinates updated additional in-frame splice at 5325, 5371 | pers. comm. Hyun-Joo Yoon | <span class="no-break">2003-02-04</span> |
<a routerLink="/gene/SPAC23C4.08">SPAC23C4.08</a> | rho3 | N-terminal exon removed | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12653963">PMID:12653963</a> | <span class="no-break">2003-03-31</span> |
<a routerLink="/gene/SPAC16A10.04">SPAC16A10.04</a> | rho4 | N-terminal exon replaced | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12653963">PMID:12653963</a> | <span class="no-break">2003-03-31</span> |
<a routerLink="/gene/SPBC6B1.09c">SPBC6B1.09c</a> | nbs1 | additional intron 25664..25705, and exon extension 25974..25988 | pers. comm. Charly Chahwan | <span class="no-break">2003-03-07</span> |
<a routerLink="/gene/SPBC6B1.09c">SPBC6B1.09c</a> | nbs1 | prediction extended by 4 N- terminal exons, this identified FHA domain | pers. comm. Val Wood | <span class="no-break">2002-04-22</span> |
<a routerLink="/gene/SPCC320.13c">SPCC320.13c</a> | ark1 | trimmed to 2nd met | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12676091">PMID:12676091</a> | <span class="no-break">2003-01-08</span> |
<a routerLink="/gene/SPAC11E3.03">SPAC11E3.03</a> | pcs1 | N terminal truncated, removed exon 9067..9103, truncated from 9149 to 9211 start | <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12689592">PMID:12689592</a> | <span class="no-break">2003-04-13</span> |
<a routerLink="/gene/SPAC19B12.05c">SPAC19B12.05c</a> | | additional exon | <span class="no-break">pers. comm. Val Wood</span> | <span class="no-break">2003-04-22</span> |</div>

  </div>
  <div *ngIf="pageName == 'index'">
<h1 id="genome-status">Genome Status</h1>
<table>
<tbody>
<tr class="odd">
<td><a routerLink="/status/statistics">Genome Statistics</a></td>
<td>Latest information on the status of the genome</td>
</tr>
<tr class="even">
<td><a routerLink="/status/sequencing-status">Sequencing Status</a></td>
<td>Contig size, gap status and progress, including <a routerLink="/status/centromeres">centromeres</a> and <a routerLink="/status/telomeres">telomeres</a></td>
</tr>
<tr class="odd">
<td><a routerLink="/status/sequencing-updates">Sequence Updates</a></td>
<td>Changes to the genome sequence since 07/03.</td>
</tr>
<tr class="even">
<td><a routerLink="/status/sequence-updates-pending">Sequence Updates Pending</a></td>
<td>Pending changes, mainly from the Broad Institute, some supported by data from other sources</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/gene-coordinate-changes">Gene Coordinate Changes</a></td>
<td>Changes to coordinates of individual genes since publication</td>
</tr>
<tr class="even">
<td><a routerLink="/status/new-and-removed-genes">New and Removed Genes</a></td>
<td>Genes identified or removed since publication</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/gene-characterisation">Gene Characterisation</a></td>
<td>Current counts of protein coding gene status, as published in small scale experiments</td>
</tr>
<tr class="even">
<td><a routerLink="/status/gene-characterisation-statistics-history">Historical Gene Characterisation</a></td>
<td>Changes in gene characterisation statistics over time</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/priority-unstudied-genes">Priority Unstudied Genes</a></td>
<td>Unstudied nuclear genes conserved 1:1 in human</td>
</tr>
<tr class="even">
<td><a routerLink="/status/unmapped_genes">Unmapped Genes</a></td>
<td>List of genes identified genetically but not cloned or physically mapped</td>
</tr>
</tbody>
</table>
<p>Note: Many older <em>S. pombe</em> sequence submissions to the DNA databases (International Nucleotide Sequence Database Collaboration databases, i.e. ENA, GenBank, DDBJ) contain one or more errors, and we do not have the resources to maintain past sequences or flag every error in PomBase.</p>

  </div>
  <div *ngIf="pageName == 'mating-type-region'">
<h1 id="mating-type-region">Mating type region</h1>
<p>The S. pombe mating type region is located on Chromosome 2 (for 972 h-, the coordinates are <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=II%3A2129208-2137121;site=ensemblunit">II:2129208-2137121</a>). This sequence represents a version of the mat2P-mat3M region in which mat2P and the cenH element are deleted.</p>
<p>A contig of the the h90 configuration of the mat2P-mat3M region was created by Xavier Marsellach and Lorena Aguilar (Azorín lab) using available data and <em>S. kambucha</em> as a scaffold. The contig can be viewed in the <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=MTR:1-20128">Ensembl genome&amp; browser</a>. Replacing the Chromosome 2 region spanning coordinates 2129208-2137121 with the separate contig sequence yields the Chromosome 2 contig of an h90 strain.</p>

  </div>
  <div *ngIf="pageName == 'new-and-removed-genes'">
<h1 id="new-and-removed-genes">New and removed genes</h1>
<h3 id="new-genes-since-publication">New Genes Since Publication</h3>
<table>
 <thead>
  <tr>
   <th>Gene</th>
   <th>Description</th>
   <th>Date</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td><a routerLink="/gene/SPBC1709.20" routerLinkActive="active" [attr.title]="'SPBC1709.20'"><span class="gene-link">SPBC1709.20</span></a></td>
  <td>RNase P and RNase MRP subunit; identified in targeted search for <em>S. cerevisiae</em> POP8 ortholog</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC7D4.15c" routerLinkActive="active" [attr.title]="'SPAC7D4.15c'"><span class="gene-link">SPAC7D4.15c</span></a></td>
  <td>Oligosaccharyltransferase subunit (predicted)</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC119.18" routerLinkActive="active" [attr.title]="'SPBC119.18'"><span class="gene-link">SPBC119.18</span></a></td>
  <td>conserved hypothetical protein</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1751.04" routerLinkActive="active" [attr.title]="'SPAC1751.04'"><span class="gene-link">SPAC1751.04</span></a></td>
  <td>hypothetical protein; sequence orphan</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC330.19c" routerLinkActive="active" [attr.title]="'SPCC330.19c'"><span class="gene-link">SPCC330.19c</span></a></td>
  <td>hypothetical protein; sequence orphan; confirmed by EST</td>
  <td>Sept 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC959.10" routerLinkActive="active" [attr.title]="'SPAC959.10'"><span class="gene-link">SPAC959.10</span></a></td>
  <td>tRNA splicing endonuclease; similar to <em>S. cerevisiae</em> YMR059W</td>
  <td>Sept 2003</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC14C4.16" routerLinkActive="active" [attr.title]="'SPAC14C4.16'"><span class="gene-link">SPAC14C4.16</span></a></td>
  <td>hypothetical protein; sequence orphan; identified in targeted search for <em>S. cerevisiae</em> DAD3 ortholog</td>
  <td>Sept 2003</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC6B1.12c" routerLinkActive="active" [attr.title]="'SPBC6B1.12c'"><span class="gene-link">SPBC6B1.12c</span></a></td>
  <td>SAGA histone acetylase complex (predicted); involved in nuclear export (predicted); similar to <em>S. cerevisiae</em> YBR111W-A; identified in targeted search for <em>S. cerevisiae</em> SUS1 ortholog</td>
  <td>Feb 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC25H2.10c" routerLinkActive="active" [attr.title]="'SPBC25H2.10c'"><span class="gene-link">SPBC25H2.10c</span></a></td>
  <td>dihydropteroate synthase (predicted) (ISS); similar to <em>S. cerevisiae</em> YGL232W; updated from pseudogene to coding</td>
  <td>March 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC56F2.14" routerLinkActive="active" [attr.title]="'SPBC56F2.14'"><span class="gene-link">SPBC56F2.14</span></a></td>
  <td>ribosomal protein, mitochondrial L44 (predicted); similar to <em>S. cerevisiae</em> YMR225C; identified in targeted search for <em>S. cerevisiae</em> MRP144 ortholog</td>
  <td>March 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC106.05c" routerLinkActive="active" [attr.title]="'SPBC106.05c'"><span class="gene-link">SPBC106.05c</span></a></td>
  <td>proton-transporting ATP synthase complex, coupling factor F(o) (predicted); non-essential (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12618370">PMID:12618370</a>); similar to <em>S. cerevisiae</em> YDR322C-A; identified in targeted search for <em>S. cerevisiae</em> YDR322C-A ortholog</td>
  <td>March 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC19D5.11c" routerLinkActive="active" [attr.title]="'SPAC19D5.11c'"><span class="gene-link">SPAC19D5.11c</span></a></td>
  <td>similar to <em>S. cerevisiae</em> CTF8; pers. comm. Stuart MacNeil</td>
  <td>September 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC19D5.10c" routerLinkActive="active" [attr.title]="'SPAC19D5.10c'"><span class="gene-link">SPAC19D5.10c</span></a></td>
  <td>sequence orphan</td>
  <td>September 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC3A12.19" routerLinkActive="active" [attr.title]="'SPAC3A12.19'"><span class="gene-link">SPAC3A12.19</span></a></td>
  <td>mitochondrial ribosomal protein; identified in targeted search for <em>S. cerevisiae</em> YBR282W ortholog</td>
  <td>September 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC57A10.14" routerLinkActive="active" [attr.title]="'SPAC57A10.14'"><span class="gene-link">SPAC57A10.14</span></a></td>
  <td>SAGA complex subunit (predicted); identified in targeted search for <em>S. cerevisiae</em> SGF11 ortholog</td>
  <td>January 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC8C9.19" routerLinkActive="active" [attr.title]="'SPAC8C9.19'"><span class="gene-link">SPAC8C9.19</span></a></td>
  <td>conserved protein; similar to <em>D. hansenii</em> DEHA0C10780g</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC8C9.20" routerLinkActive="active" [attr.title]="'SPAC8C9.20'"><span class="gene-link">SPAC8C9.20</span></a></td>
  <td>sequence orphan</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1223.15c" routerLinkActive="active" [attr.title]="'SPCC1223.15c'"><span class="gene-link">SPCC1223.15c</span></a></td>
  <td>DASH complex subunit Spc19; pers. comm. Xiangwei He</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC3D6.16" routerLinkActive="active" [attr.title]="'SPBC3D6.16'"><span class="gene-link">SPBC3D6.16</span></a></td>
  <td>sequence orphan</td>
  <td>Added May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC3B9.22c" routerLinkActive="active" [attr.title]="'SPBC3B9.22c'"><span class="gene-link">SPBC3B9.22c</span></a></td>
  <td>DASH complex subunit Dad4 (predicted); pers. comm. Xiangwei He</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC16E9.16c" routerLinkActive="active" [attr.title]="'SPBC16E9.16c'"><span class="gene-link">SPBC16E9.16c</span></a></td>
  <td>sequence orphan; previously annotated as pseudogene; two peptides in mass spec analysis (pers. comm. Dieter Wolf)</td>
  <td>November 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC27E2.12" routerLinkActive="active" [attr.title]="'SPAC27E2.12'"><span class="gene-link">SPAC27E2.12</span></a></td>
  <td>dubious</td>
  <td>Added February 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1347.14c" routerLinkActive="active" [attr.title]="'SPBC1347.14c'"><span class="gene-link">SPBC1347.14c</span></a></td>
  <td>dubious</td>
  <td>February 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC16C4.21" routerLinkActive="active" [attr.title]="'SPAC16C4.21'"><span class="gene-link">SPAC16C4.21</span></a></td>
  <td>dubious</td>
  <td>June 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC6B12.18" routerLinkActive="active" [attr.title]="'SPAC6B12.18'"><span class="gene-link">SPAC6B12.18</span></a></td>
  <td>sequence orphan</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.17c" routerLinkActive="active" [attr.title]="'SPAC222.17c'"><span class="gene-link">SPAC222.17c</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC27E2.13" routerLinkActive="active" [attr.title]="'SPAC27E2.13'"><span class="gene-link">SPAC27E2.13</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC417.15" routerLinkActive="active" [attr.title]="'SPCC417.15'"><span class="gene-link">SPCC417.15</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1250.07" routerLinkActive="active" [attr.title]="'SPAC1250.07'"><span class="gene-link">SPAC1250.07</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1250.07" routerLinkActive="active" [attr.title]="'SPAC1250.07'"><span class="gene-link">SPAC1250.07</span></a></td>
  <td>TFIIIC subunit Tfc7; from PMID:17409385</td>
  <td>April 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1393.14" routerLinkActive="active" [attr.title]="'SPCC1393.14'"><span class="gene-link">SPCC1393.14</span></a></td>
  <td>ten1; nuclear telomere cap complex subunit; pers. comm. Paul Russell</td>
  <td>July 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAPB1A10.16" routerLinkActive="active" [attr.title]="'SPAPB1A10.16'"><span class="gene-link">SPAPB1A10.16</span></a></td>
  <td>conserved fungal protein; created from the C-term exon of SPAPB1A10.03</td>
  <td>December 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC569.09" routerLinkActive="active" [attr.title]="'SPCC569.09'"><span class="gene-link">SPCC569.09</span></a></td>
  <td>sequence orphan; longest reading frome in solexa transcript SPNG2623</td>
  <td>December 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.214" routerLinkActive="active" [attr.title]="'SPNCRNA.214'"><span class="gene-link">SPNCRNA.214</span></a></td>
  <td>telomerase RNA (predicted)</td>
  <td>January 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1183.12" routerLinkActive="active" [attr.title]="'SPCC1183.12'"><span class="gene-link">SPCC1183.12</span></a></td>
  <td>previously SPNCRNA.546; splice site not spotted because it was outside the coordinate range of the feature; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
  <td>April 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC16C6.14" routerLinkActive="active" [attr.title]="'SPBC16C6.14'"><span class="gene-link">SPBC16C6.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
  <td>April 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC70.12c" routerLinkActive="active" [attr.title]="'SPCC70.12c'"><span class="gene-link">SPCC70.12c</span></a></td>
  <td>extender of the chronological lifespan protein Ecl1; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18422613">PMID:18422613</a></td>
  <td>April 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1442.19" routerLinkActive="active" [attr.title]="'SPCC1442.19'"><span class="gene-link">SPCC1442.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.17" routerLinkActive="active" [attr.title]="'SPAC222.17'"><span class="gene-link">SPAC222.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC569.09" routerLinkActive="active" [attr.title]="'SPCC569.09'"><span class="gene-link">SPCC569.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAPB1E7.14" routerLinkActive="active" [attr.title]="'SPAPB1E7.14'"><span class="gene-link">SPAPB1E7.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC688.16" routerLinkActive="active" [attr.title]="'SPAC688.16'"><span class="gene-link">SPAC688.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAP4C9.02" routerLinkActive="active" [attr.title]="'SPAP4C9.02'"><span class="gene-link">SPAP4C9.02</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC16C4.22" routerLinkActive="active" [attr.title]="'SPCC16C4.22'"><span class="gene-link">SPCC16C4.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23D3.16" routerLinkActive="active" [attr.title]="'SPAC23D3.16'"><span class="gene-link">SPAC23D3.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1183.12" routerLinkActive="active" [attr.title]="'SPCC1183.12'"><span class="gene-link">SPCC1183.12</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC25H1.10C" routerLinkActive="active" [attr.title]="'SPAC25H1.10C'"><span class="gene-link">SPAC25H1.10C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1604.25" routerLinkActive="active" [attr.title]="'SPBC1604.25'"><span class="gene-link">SPBC1604.25</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23D3.17" routerLinkActive="active" [attr.title]="'SPAC23D3.17'"><span class="gene-link">SPAC23D3.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4G9.22" routerLinkActive="active" [attr.title]="'SPAC4G9.22'"><span class="gene-link">SPAC4G9.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC757.15" routerLinkActive="active" [attr.title]="'SPCC757.15'"><span class="gene-link">SPCC757.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC227.19C" routerLinkActive="active" [attr.title]="'SPAC227.19C'"><span class="gene-link">SPAC227.19C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC167.09" routerLinkActive="active" [attr.title]="'SPAC167.09'"><span class="gene-link">SPAC167.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC16C6.14" routerLinkActive="active" [attr.title]="'SPBC16C6.14'"><span class="gene-link">SPBC16C6.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC20G4.09" routerLinkActive="active" [attr.title]="'SPAC20G4.09'"><span class="gene-link">SPAC20G4.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.18" routerLinkActive="active" [attr.title]="'SPAC222.18'"><span class="gene-link">SPAC222.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC6F6.19" routerLinkActive="active" [attr.title]="'SPAC6F6.19'"><span class="gene-link">SPAC6F6.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC22F3.15" routerLinkActive="active" [attr.title]="'SPAC22F3.15'"><span class="gene-link">SPAC22F3.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC27E2.14" routerLinkActive="active" [attr.title]="'SPAC27E2.14'"><span class="gene-link">SPAC27E2.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23H4.21" routerLinkActive="active" [attr.title]="'SPAC23H4.21'"><span class="gene-link">SPAC23H4.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC57A7.15C" routerLinkActive="active" [attr.title]="'SPAC57A7.15C'"><span class="gene-link">SPAC57A7.15C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC12G12.17" routerLinkActive="active" [attr.title]="'SPAC12G12.17'"><span class="gene-link">SPAC12G12.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC29A10.17" routerLinkActive="active" [attr.title]="'SPBC29A10.17'"><span class="gene-link">SPBC29A10.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1556.06" routerLinkActive="active" [attr.title]="'SPAC1556.06'"><span class="gene-link">SPAC1556.06</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC26H8.16" routerLinkActive="active" [attr.title]="'SPBC26H8.16'"><span class="gene-link">SPBC26H8.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC70.12C" routerLinkActive="active" [attr.title]="'SPCC70.12C'"><span class="gene-link">SPCC70.12C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBP35G2.16c" routerLinkActive="active" [attr.title]="'SPBP35G2.16c'"><span class="gene-link">SPBP35G2.16c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19352039">PMID:19352039</a></td>
  <td>April 2009</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC8E4.12c" routerLinkActive="active" [attr.title]="'SPBC8E4.12c'"><span class="gene-link">SPBC8E4.12c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19352039">PMID:19352039</a></td>
  <td>April 2009</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1235.16" routerLinkActive="active" [attr.title]="'SPCC1235.16'"><span class="gene-link">SPCC1235.16</span></a></td>
  <td>pers. comm. Juan Mata</td>
  <td>April 2009</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC18.19C" routerLinkActive="active" [attr.title]="'SPCC18.19C'"><span class="gene-link">SPCC18.19C</span></a></td>
  <td>pers. comm. Val Wood; note first 2 exons may be incorrect, homology is within 3rd exon</td>
  <td>February 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC306.10" routerLinkActive="active" [attr.title]="'SPCC306.10'"><span class="gene-link">SPCC306.10</span></a></td>
  <td>pers. comm. Val Wood; pseudo-&gt; coding</td>
  <td>March 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.497" routerLinkActive="active" [attr.title]="'SPNCRNA.497'"><span class="gene-link">SPNCRNA.497</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.498" routerLinkActive="active" [attr.title]="'SPNCRNA.498'"><span class="gene-link">SPNCRNA.498</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.499" routerLinkActive="active" [attr.title]="'SPNCRNA.499'"><span class="gene-link">SPNCRNA.499</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.507" routerLinkActive="active" [attr.title]="'SPNCRNA.507'"><span class="gene-link">SPNCRNA.507</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.510" routerLinkActive="active" [attr.title]="'SPNCRNA.510'"><span class="gene-link">SPNCRNA.510</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.514" routerLinkActive="active" [attr.title]="'SPNCRNA.514'"><span class="gene-link">SPNCRNA.514</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.515" routerLinkActive="active" [attr.title]="'SPNCRNA.515'"><span class="gene-link">SPNCRNA.515</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.523" routerLinkActive="active" [attr.title]="'SPNCRNA.523'"><span class="gene-link">SPNCRNA.523</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.530" routerLinkActive="active" [attr.title]="'SPNCRNA.530'"><span class="gene-link">SPNCRNA.530</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.19" routerLinkActive="active" [attr.title]="'SPAC222.19'"><span class="gene-link">SPAC222.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC17G8.15" routerLinkActive="active" [attr.title]="'SPAC17G8.15'"><span class="gene-link">SPAC17G8.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1805.18" routerLinkActive="active" [attr.title]="'SPAC1805.18'"><span class="gene-link">SPAC1805.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1486.11" routerLinkActive="active" [attr.title]="'SPAC1486.11'"><span class="gene-link">SPAC1486.11</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC3H5.13" routerLinkActive="active" [attr.title]="'SPAC3H5.13'"><span class="gene-link">SPAC3H5.13</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC15A10.17" routerLinkActive="active" [attr.title]="'SPAC15A10.17'"><span class="gene-link">SPAC15A10.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4F10.22" routerLinkActive="active" [attr.title]="'SPAC4F10.22'"><span class="gene-link">SPAC4F10.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1B3.21" routerLinkActive="active" [attr.title]="'SPAC1B3.21'"><span class="gene-link">SPAC1B3.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPRRNA.56" routerLinkActive="active" [attr.title]="'SPRRNA.56'"><span class="gene-link">SPRRNA.56</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC9G1.15c" routerLinkActive="active" [attr.title]="'SPAC9G1.15c'"><span class="gene-link">SPAC9G1.15c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC13F5.08" routerLinkActive="active" [attr.title]="'SPAC13F5.08'"><span class="gene-link">SPAC13F5.08</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC6B12.19" routerLinkActive="active" [attr.title]="'SPAC6B12.19'"><span class="gene-link">SPAC6B12.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC3G9.17" routerLinkActive="active" [attr.title]="'SPAC3G9.17'"><span class="gene-link">SPAC3G9.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC16E8.18c" routerLinkActive="active" [attr.title]="'SPAC16E8.18c'"><span class="gene-link">SPAC16E8.18c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC926.10" routerLinkActive="active" [attr.title]="'SPAC926.10'"><span class="gene-link">SPAC926.10</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC19G12.17" routerLinkActive="active" [attr.title]="'SPAC19G12.17'"><span class="gene-link">SPAC19G12.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23A1.20" routerLinkActive="active" [attr.title]="'SPAC23A1.20'"><span class="gene-link">SPAC23A1.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1F7.14" routerLinkActive="active" [attr.title]="'SPAC1F7.14'"><span class="gene-link">SPAC1F7.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4D7.15" routerLinkActive="active" [attr.title]="'SPAC4D7.15'"><span class="gene-link">SPAC4D7.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4D7.14" routerLinkActive="active" [attr.title]="'SPAC4D7.14'"><span class="gene-link">SPAC4D7.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC839.20" routerLinkActive="active" [attr.title]="'SPBC839.20'"><span class="gene-link">SPBC839.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC409.23" routerLinkActive="active" [attr.title]="'SPBC409.23'"><span class="gene-link">SPBC409.23</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC8D2.23" routerLinkActive="active" [attr.title]="'SPBC8D2.23'"><span class="gene-link">SPBC8D2.23</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC3H7.18" routerLinkActive="active" [attr.title]="'SPBC3H7.18'"><span class="gene-link">SPBC3H7.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1711.18" routerLinkActive="active" [attr.title]="'SPBC1711.18'"><span class="gene-link">SPBC1711.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC24C6.13" routerLinkActive="active" [attr.title]="'SPBC24C6.13'"><span class="gene-link">SPBC24C6.13</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC32F12.16" routerLinkActive="active" [attr.title]="'SPBC32F12.16'"><span class="gene-link">SPBC32F12.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC30D10.21" routerLinkActive="active" [attr.title]="'SPBC30D10.21'"><span class="gene-link">SPBC30D10.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC887.22" routerLinkActive="active" [attr.title]="'SPBC887.22'"><span class="gene-link">SPBC887.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC839.19" routerLinkActive="active" [attr.title]="'SPBC839.19'"><span class="gene-link">SPBC839.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC530.16" routerLinkActive="active" [attr.title]="'SPBC530.16'"><span class="gene-link">SPBC530.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC14C8.19" routerLinkActive="active" [attr.title]="'SPBC14C8.19'"><span class="gene-link">SPBC14C8.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC17D1.17" routerLinkActive="active" [attr.title]="'SPBC17D1.17'"><span class="gene-link">SPBC17D1.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1105.19" routerLinkActive="active" [attr.title]="'SPBC1105.19'"><span class="gene-link">SPBC1105.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC56F2.15" routerLinkActive="active" [attr.title]="'SPBC56F2.15'"><span class="gene-link">SPBC56F2.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCP20C8.04" routerLinkActive="active" [attr.title]="'SPCP20C8.04'"><span class="gene-link">SPCP20C8.04</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC330.20" routerLinkActive="active" [attr.title]="'SPCC330.20'"><span class="gene-link">SPCC330.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC63.09" routerLinkActive="active" [attr.title]="'SPCC63.09'"><span class="gene-link">SPCC63.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC4B3.20" routerLinkActive="active" [attr.title]="'SPCC4B3.20'"><span class="gene-link">SPCC4B3.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC330.21" routerLinkActive="active" [attr.title]="'SPCC330.21'"><span class="gene-link">SPCC330.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC11D3.11c" routerLinkActive="active" [attr.title]="'SPAC11D3.11c'"><span class="gene-link">SPAC11D3.11c</span></a></td>
  <td>pseudo to coding</td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC713.14c" routerLinkActive="active" [attr.title]="'SPBC713.14c'"><span class="gene-link">SPBC713.14c</span></a></td>
  <td>identified by ribosome profiling (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24929437">PMID:24929437</a>)</td>
  <td>July 2014</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC823.17" routerLinkActive="active" [attr.title]="'SPAC823.17'"><span class="gene-link">SPAC823.17</span></a></td>
  <td>identified by ribosome profiling (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24929437">PMID:24929437</a>)</td>
  <td>July 2014</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC110.06" routerLinkActive="active" [attr.title]="'SPAC110.06'"><span class="gene-link">SPAC110.06</span></a></td>
  <td>identified by ribosome profiling (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24929437">PMID:24929437</a>)</td>
  <td>July 2014</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAPB1A11.06" routerLinkActive="active" [attr.title]="'SPAPB1A11.06'"><span class="gene-link">SPAPB1A11.06</span></a></td>
  <td>previously SPNCRNA.31 (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a>)</td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC110.06" routerLinkActive="active" [attr.title]="'SPAC110.06'"><span class="gene-link">SPAC110.06</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a></td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC29A4.23" routerLinkActive="active" [attr.title]="'SPAC29A4.23'"><span class="gene-link">SPAC29A4.23</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a></td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC110.06" routerLinkActive="active" [attr.title]="'SPAC110.06'"><span class="gene-link">SPAC110.06</span></a></td>
  <td>F1F0 ATPase synthase peripheral stalk assembly factor Ina17 (predicted) (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a>)</td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1688" routerLinkActive="active" [attr.title]="'SPNCRNA.1688'"><span class="gene-link">SPNCRNA.1688</span></a></td>
  <td>intergenic RNA nc-tgp1 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=25428589">PMID:25428589</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1701" routerLinkActive="active" [attr.title]="'SPNCRNA.1701'"><span class="gene-link">SPNCRNA.1701</span></a></td>
  <td>name prl101; synonym prl1 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1702" routerLinkActive="active" [attr.title]="'SPNCRNA.1702'"><span class="gene-link">SPNCRNA.1702</span></a></td>
  <td>name prl104; synonym prl4 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1703" routerLinkActive="active" [attr.title]="'SPNCRNA.1703'"><span class="gene-link">SPNCRNA.1703</span></a></td>
  <td>name prl105; synonym prl5 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1704" routerLinkActive="active" [attr.title]="'SPNCRNA.1704'"><span class="gene-link">SPNCRNA.1704</span></a></td>
  <td>name prl107; synonym prl7 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1705" routerLinkActive="active" [attr.title]="'SPNCRNA.1705'"><span class="gene-link">SPNCRNA.1705</span></a></td>
  <td>name prl108; synonym prl8 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1706" routerLinkActive="active" [attr.title]="'SPNCRNA.1706'"><span class="gene-link">SPNCRNA.1706</span></a></td>
  <td>name prl106; synonym prl6 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1707" routerLinkActive="active" [attr.title]="'SPNCRNA.1707'"><span class="gene-link">SPNCRNA.1707</span></a></td>
  <td>name prl102; synonym prl2 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1708" routerLinkActive="active" [attr.title]="'SPNCRNA.1708'"><span class="gene-link">SPNCRNA.1708</span></a></td>
  <td>name prl103; synonym prl3 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 </tbody>
</table>
<h3 id="genes-removed-since-publication">Genes Removed Since Publication</h3>
<table>
 <tbody>
 <tr>
  <th>Gene</th>
  <th>Comment</th>
  <th>Date</th>
 </tr>
 <tr>
  <td>SPBC19G7.12c</td>
  <td>removed because sequence similarity supports SPBC19G7.11c</td>
  <td><span class="no-break">2004-09-14</span></td>
 </tr>
 <tr>
  <td>SPAC212.10</td>
  <td>removed as part of UTR of SPAC212.11</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC3F10.14</td>
  <td>reverse strand of SPAC3F10.15c</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC428.09c</td>
  <td>presumed non-coding as no methionine or consensus splice sites</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC222.17c</td>
  <td>presumed non-coding; was a very small ORF in ncRNA prl50</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.135</td>
  <td>Initially annotated as a non-coding RNA but subsequently determined to be the 5' UTR of SPBC16D10.06 (pers. comm S. Whitehall)</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBPB21E7.10</td>
  <td>partial ORF of 35 aa, no reason to believe this is coding</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC4G8.15c</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAPB18E9.03c</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC18H10.21c</td>
  <td>alias SPBC9B6.01c; dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC9B6.01c</td>
  <td>alias of SPBC18H10.21c; dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPCC2H8.03</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC22E12.12</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.85</td>
  <td>replaced by protein coding gene SPAC227.19c</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.48</td>
  <td>replaced by protein coding gene SPAC2F3.18</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.97</td>
  <td>replaced by short protein coding gene SPAC27E2.14</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC8E11.09c</td>
  <td>previously annotated as dubious but no evidence for splicing, and non-canonical splice sites; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC22E12.15</td>
  <td>previously annotated as dubious but dubious splice prediction; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC1002.21</td>
  <td>was annotated as dubious but based on position in uracil regulatable operon, as this CDS does not appear to be transcribed. t is possible that the localization data is an artifact of the combination of overexpression and hydrophobicity. The protein sequence is also a strange composition, and there is no ortholog in octosporus or japonicus genomes. On balance, taking into account all of the available features and contextual information, it is unlikely that this feature codes for a bona fide protein.</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC8C9.20</td>
  <td>non consensus branch sites, and no coverage with transcriptome data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC22E12.12</td>
  <td>unsupported by Solexa transcript data and largely overlapping with 3' UTR of rpl24-3</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPCC1183.12</td>
  <td>previously SPNCRNA.546; splice site not spotted because it was outside the coordinate range of the feature; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC16C6.14</td>
  <td>unsupported by Solexa transcript data and largely overlapping with 3' UTR of rpl24-3</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC6F6.18c</td>
  <td>merged with SPAC6F6.16c; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18535244">PMID:18535244</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC651.07</td>
  <td>merged with SPBC651.06; pers. comm. Charly Chahwan</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.109</td>
  <td>part of protein coding gene merge</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC18E5.09c</td>
  <td>merged with SPBC18E5.14c; pers. comm. Charly Chahwan</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC959.06c</td>
  <td>merged with SPAC959.05c; pers. comm. Charly Chahwan</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPCC794.14c</td>
  <td>dubious</td>
  <td>2005-05</td>
 </tr>
 <tr>
  <td>SPNCRNA.523</td>
  <td>extended exons of adjacent protein coding gene</td>
  <td><span class="no-break">2008-12-19</span></td>
 </tr>
 <tr>
  <td>SPBC14C8.08c</td>
  <td>merged with SPBC14C8.09c; frameshifted; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
  <td><span class="no-break">2009-02-02</span></td>
 </tr>
 <tr>
  <td>SPRRNA.25</td>
  <td>small fragment of 25S rRNA; <span class="no-break">2009-11-26</span></td>
 </tr>
 <tr>
  <td>SPAC4G9.22</td>
  <td>likely to be non-coding RNA, no supporting evidence for protein</td>
  <td><span class="no-break">2010-02-21</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.509</td>
  <td>dubious and partially overlapping with Ost5</td>
  <td><span class="no-break">2010-02-18</span></td>
 </tr>
 <tr>
  <td>SPBC11C11.12</td>
  <td>cob intron fragment currently annotated as pseudogene</td>
  <td><span class="no-break">2010-03-09</span></td>
 </tr>
 <tr>
  <td>SPBCPT2R1.09c</td>
  <td>no coding region</td>
  <td><span class="no-break">2010-09-19</span></td>
 </tr>
 <tr>
  <td>SPBPB2B2.04</td>
  <td>no coding region</td>
  <td><span class="no-break">2010-09-19</span></td>
 </tr>
 <tr>
  <td>SPBPB2B2.03c</td>
  <td>no coding region</td>
  <td><span class="no-break">2010-09-19</span></td>
 </tr>
 <tr>
  <td>SPAC17A2.15</td>
  <td>was on the reverse strand of the N-terminal extension of SPAC17A2.08</td>
  <td><span class="no-break">2011-03-10</span></td>
 </tr>
 <tr>
  <td>SPBC1348.13</td>
  <td>Coordinates updated; now annotated as nuclear mitochondrial pseudogene (SO:0001044)</td>
  <td><span class="no-break">2011-11-20</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.10</td>
  <td>merged with SPNCRNA.1234</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1098</td>
  <td>merged with SPNCRNA.07</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1196</td>
  <td>merged with SPNCRNA.67</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.125</td>
  <td>merged with SPCC16A11.14</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1254</td>
  <td>merged with SPNCRNA.511</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1291</td>
  <td>merged with SPNCRNA.519</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1486</td>
  <td>merged with SPNCRNA.388</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.18</td>
  <td>merged with SPNCRNA.928</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.489</td>
  <td>merged with SPNCRNA.1187</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.502</td>
  <td>merged with SPNCRNA.1228</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.582</td>
  <td>merged with SPNCRNA.1208</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.64</td>
  <td>merged with SPCC1393.13</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.884</td>
  <td>merged with SPNCRNA.214</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.958</td>
  <td>merged with SPAC23A1.20</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPAC1F12.03c</td>
  <td>removed; replaced by a nuclear mitochondrial pseudogene (NUMT) feature</td>
  <td><span class="no-break">2012-07-16</span></td>
 </tr>
 <tr>
  <td>SPAC1F8.09c</td>
  <td>annotated in error</td>
  <td><span class="no-break">2013-02-22</span></td>
 </tr>
 <tr>
  <td>SPBC713.13</td>
  <td>deleted; ID added as synonym to SPBC713.14c&nbsp;(<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>)</td>
  <td><span class="no-break">2014-07-04</span></td>
 </tr>
 <tr>
  <td>SPAC823.02</td>
  <td>deleted; replaced by SPAC823.17 at same locus, but in different frame&nbsp;(<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>)</td>
  <td><span class="no-break">2014-07-04</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.252</td>
  <td>changed to LTR</td>
  <td><span class="no-break">2015-01-22</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.76</td>
  <td>overlapped with correctly annotated NUMT (SPNUMT.9)</td>
  <td><span class="no-break">2015-10-12</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.83</td>
  <td>overlapped with correctly annotated NUMT (SPNUMT.8)</td>
  <td><span class="no-break">2015-10-12</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.906</td>
  <td>merged with snR30</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.814</td>
  <td>merged with snR98</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1436</td>
  <td>merged with snR95</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.9061</td>
  <td>merged with snR91</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1038</td>
  <td>merged with snR90</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1394</td>
  <td>merged with snR3</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.727</td>
  <td>deleted; ID added as synonym to SPNCRNA.82 and SPSNORNA.10 (covers both)</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.727</td>
  <td>deleted; ID added as synonym to snR41/SPSNORNA.07, snR70/SPSNORNA.08 and snR51b/SPSNORNA.09 (covers all)</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.866</td>
  <td>unsupported (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:21511999</a>)</td>
  <td><span class="no-break">2015-11-30</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.867</td>
  <td>unsupported (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:21511999</a>)</td>
  <td><span class="no-break">2015-11-30</span></td>
 </tr>
 <tr>
  <td>SPAC1D4.07c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPAC27E2.13</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPAC4H3.12c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC1685.12c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC32F12.17</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC354.11c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC36.13</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC1672.14</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC188.05</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC417.04</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC622.05</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC794.16</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 </tbody>
</table>

  </div>
  <div *ngIf="pageName == 'priority-unstudied-genes'">
<h1 id="priority-unstudied-genes">Priority unstudied genes</h1>
<p>Fission yeast genes are classed as &quot;unknown&quot; if there is no information about the broad cellular role (biological process) in which it participates (corresponding to any of the <a routerLink="/browse-curation/fission-yeast-go-slim-terms">high level GO Slim classes</a>). For these, we have been unable to identify a broad biological role based on experimental data in fission yeast or any other organism. Note that, as all genes in fission yeast have been curated, these genes are documented as &quot;unknown&quot; (they are not &quot;unannotated&quot;).</p>
<p>In fission yeast the &quot;unknown&quot; inventory is <app-query-link [predefinedQueryName]="'priority-unstudied-genes:unknown_genes'"></app-query-link> entries, many of which are apparently species-specific. However, a large number (<app-query-link [predefinedQueryName]="'priority-unstudied-genes:conserved_unknown'"></app-query-link>) are conserved, and a significant number of these (<app-query-link [predefinedQueryName]="'priority-unstudied-genes:conserved_unknown_AND_conserved_in_vertebrates'"></app-query-link>) have orthologs in vertebrates. These genes are listed below.</p>
<p>You can recreate this query, or variations of it, using the Advanced Search. See the relevant FAQ and the <a routerLink="/query">Advanced Search</a> documentation for details. You can also send this list directly to the advanced search (For example to mine for phenotypes or locations of interest).</p>
<p><app-predefined-query-results [predefinedQueryName]="'priority-unstudied-genes:conserved_unknown_AND_conserved_in_vertebrates'"></app-predefined-query-results></p>

  </div>
  <div *ngIf="pageName == 'sequence-updates-pending'">
<h1 id="pending-sequence-updates">Pending sequence updates</h1>
<h2 id="individual-sequence-updates">Individual sequence updates</h2>
<ul>
<li>Chromosome III: The reference sequence is missing 4 bp -- GATC to be inserted after reference position 17744235. Pers. comm. Nathan Lee (S. Grewal lab), reported <span class="no-break">2014-03-31</span>.</li>
</ul>
<h2 id="sequence-discrepancies-from-the-broad">Sequence discrepancies from the Broad</h2>
<p>As part of a technology development project, the Broad Institute generated 200-fold sequence coverage of the genome of <em>S. pombe</em> strain 972 using Illumina (Solexa) technology. This deep coverage was used to identify discrepancies between the Illumina read data and the current reference assembly of 972, employing a polymorphism discovery algorithm written specifically for Illumina data, and tuned to have a very low false positive call rate. 190 sequence discrepancies were observed, all but two of which were either single base insertions, deletions or substitutions. These discrepancies could be due to sequence errors in the reference 972 assembly, sequence polymorphisms between the two isolates of 972 sequenced, or systematic errors in the Illumina data or analysis. Of the 190 discrepancies, 25 occur in homopolymer runs of at least 7 nucleotides, which are both more likely to produce sequencing errors and inter-strain polymorphisms, and 39 are clustered within 10 nucleotides of another discrepancy, which is more common in cases of mis-alignment of the Illumina reads. Excluding these sites, the discrepancy rate is about 1 in 100,000, which is close to the estimated error rate in published reference sequence.</p>
<p>The data listed here were updated in August 2008; the original coordinates were based on an older assembly and did not match the current assembly of the right arm of Chromosome I. One discrepancy is a known artifact. At nucleotide 1774235 of Chromosome III there is an insertion of GACT caused by an assembly glitch in the published reference, which will be corrected in future releases. In addition, there are several clusters of discrepancies which are suspicious. These discrepancies will be investigated and the discrepancy list updated as appropriate. Beyond that analysis, no validation of other discrepancies is currently planned. However, we welcome any validation efforts by members of the community. If you generate any data pertinent to these discrepancies, or if you have any comments about them, please contact <a routerLink="/mailto:nick.rhind@umassmed.edu">Nick Rhind</a> or <a href="mailto:val@sanger.ac.uk">Valerie Wood</a>. These data can be cited as follows: &quot;Data were generated at the Broad Institute. Chad Nusbaum, personal communication.&quot;</p>
<p>The data can be downloaded as an Excel spreadsheet from the <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Archived_directories/972_discrepancies_from_Broad.xls">PomBase ftp site</a>.</p>
<table>
<thead>
<tr class="header">
<th align="left">Chromosome</th>
<th align="left">Location</th>
<th align="left">Left</th>
<th align="left">Sample</th>
<th align="left">Reference</th>
<th align="left">Right</th>
<th align="left">Clustered?</th>
<th align="left">Homopolymer</th>
<th align="left">Confirmed in</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">I</td>
<td align="left">94353</td>
<td align="left">GTGACAATTGAATTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">AAATGTTTCTATATTGTTAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">101872</td>
<td align="left">GAGGACGGGCGTTGGCGGAG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">CAACAGCCTTACCCCAGTCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">470379</td>
<td align="left">TGTGATAGACAAAAAAAGAC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">GTTAAAAGAACCTAAACGAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">470390</td>
<td align="left">AAAAAAGACCGTTAAAAGAA</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">CTAAACGAATTGTAATCAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">482738</td>
<td align="left">AGCTATTCGATCCACCGCAG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TTTGTCTGATTTGTTGCCCT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">523602</td>
<td align="left">ACAATGATACTTACTCCACA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">CGATTCCATTTTCCAAAGAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">523882</td>
<td align="left">GTTTCAGCCATTGAAAGATA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">CGATCGTTCTCAGACAAAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">527155</td>
<td align="left">AATCATGGATTTAATACATA</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">CTTTTAGTTTGTATGTGATA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">554538</td>
<td align="left">GAAGATTTATTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GGAATTTATTCTGCTCGGCC</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">670033</td>
<td align="left">AGAGATTATTATTAAGATCA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CCTAAATATCGTAACGTACG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">682996</td>
<td align="left">ATAAAACTACTATACTTCCC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">GCTGGAGCTTCGGTAGGCCG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">759333</td>
<td align="left">TGTTCACAGTTTTTCCCAAA</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">TTGCTTTTTCTTTTGGCTTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">862963</td>
<td align="left">ATTAAATTCTCGAAGTTGCT</td>
<td align="left">TT</td>
<td align="left">G</td>
<td align="left">ATTTTCCAAATCAATTCATG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">1048646</td>
<td align="left">ACGGAAAAGAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TCCGCGTTCAGTAGAAGTCA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">1274804</td>
<td align="left">GTACTTGAAATTATTTTTTA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TAAAGTGAGAACGCCCAGTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">1424708</td>
<td align="left">GTGCTTCATCAGTGACAACC</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">TTTAGAATGGATCTGTATGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">1625094</td>
<td align="left">ATCAAATAGTCGATGCCTCC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">AGCATCTCAATCTTCAATTC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">1783457</td>
<td align="left">ACTCTGGGTTCTTCTTATGT</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">TGGGACGGTCTTCGGCATAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">1936298</td>
<td align="left">ACTCTTAAATTTTAGGAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GACGCTCTTTTATAGTAAGG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">1952317</td>
<td align="left">TATATGTACGTACATATTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GCGCACCAGTTAGTTTGCGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">1999191</td>
<td align="left">TTTACCGCTTGACTGTGACC</td>
<td align="left">C</td>
<td align="left">G</td>
<td align="left">TACGACCTAACAATTTATTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18088324">PMID:18088324</a></td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2104090</td>
<td align="left">AAACTGTTGTAAAAAAATTA</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">ATTATCGGTGTATTTATTTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2104096</td>
<td align="left">TTGTAAAAAAATTAGATTAT</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">GGTGTATTTATTTATATTAG</td>
<td align="left">Yes - 6</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2154492</td>
<td align="left">AATTTGTTAAAAAAGGAGTA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GTTTACCTCTCGCTTTATGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2288241</td>
<td align="left">TCTTCATTGTTATTGTTACG</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">ATTCGTCACCCAGTGAAGAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2343704</td>
<td align="left">TGCAAACTCTTTGATCACGA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GTCTACATAATTCTTGAACC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2421577</td>
<td align="left">AGCTAAATATAAGAAAAAAC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">GAGAATTATAAGCAAACGTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2499159</td>
<td align="left">AAAACATCAATTTTTTTTTT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">GACAGTAATTATACTAGAAT</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2506043</td>
<td align="left">AAATAGTGTCACATTGTTTT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">ATATTAAGTATTTTTAATCG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2507463</td>
<td align="left">TTAGGTGTTAAACATAAATA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CTTACGTTTTCATACGCGCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2588024</td>
<td align="left">AAAGAACAGCAAAAAACAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GATAATGACAGAAAAAATAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2588072</td>
<td align="left">CTGAGAAAGTTGACAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TTTTTGGCGAAAGAAAGAAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2594264</td>
<td align="left">TAATTTTTAATGAATACTCT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">AGGCTTCCTTAAATGTTTTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2594326</td>
<td align="left">TTAAAGCAGTGTCTTATTAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CATTGTAAACATCAAAAAGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2594351</td>
<td align="left">TAAACATCAAAAAGCTTCAT</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">AACAGATGTAAGAAATTAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2594393</td>
<td align="left">AATTAACTTGTGAAGACATT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">CGATGTGATTAAAGTATGAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2605763</td>
<td align="left">TGTAAAAAGCATTTGAAATT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">AAAAAGAAAAAACAAAAAAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2605862</td>
<td align="left">AACGTACTGAGGATATCGTT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">ACCCCTTGCAGAAATATAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2607673</td>
<td align="left">ATCCATGTCTGTAATATTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GGTTATTAATAAATTGAAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2683903</td>
<td align="left">CAAATATCATCAGTGCGATC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">AACAGCGGTATTTGTCTGAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2759872</td>
<td align="left">TATCTTCCCACCGTCGTTTC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TATCGGTAGCATATCCGATT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3111507</td>
<td align="left">TGCGTTTTCTGTCTGTTGTG</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">ATGTCTCGCCTGGACTTCTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3115840</td>
<td align="left">ACAGATAAAGAGAGAAAAGT</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">CAAGCGTTTAAGCACCGAAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3119298</td>
<td align="left">AAAATCTTATTGATTGAAAT</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">CAATGCTTCGTAAACAATAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3125119</td>
<td align="left">TGCCAAACTCTTTTCTTCAT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">ATGTTCTGTTGTGCCAAAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3178496</td>
<td align="left">TAACTGGCAAATCGGCTCTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">CCCATAAGTAGACGATGATC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3179237</td>
<td align="left">TGCTAAAATACAGTTAGTTA</td>
<td align="left">C</td>
<td align="left">G</td>
<td align="left">TACTTGCTCACTTATATTTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3197530</td>
<td align="left">AATACTGCTGGAAATACAGG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TTTGGTTCGCAAGGTACTGG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://getentry.ddbj.nig.ac.jp/getentry/na/LC043101/?format=flatfile&filetype=html&trace=true&show_suppressed=false&limit=10">DDJB:LC043101</a></td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3450130</td>
<td align="left">TTATTATTCCCAGGCACGGG</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">AAGTTCCAAAAATCGAATAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3460320</td>
<td align="left">TTTACAAAGTCGGACAATCC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TGGCGGCGGTGTTCTAAAAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3538415</td>
<td align="left">TAAAGTATCGGAATTACATT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">CGTCAAACCAGCGTTTATGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3538428</td>
<td align="left">TTACATTCGTCAAACCAGCG</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">TTTATGATAAAAGATCATCC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3538436</td>
<td align="left">GTCAAACCAGCGTTTATGAT</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">AAAAGATCATCCTATTCTAA</td>
<td align="left">Yes - 8</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3538448</td>
<td align="left">TTTATGATAAAAGATCATCC</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">TATTCTAATTTTGCTATCGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3538450</td>
<td align="left">TATGATAAAAGATCATCCTA</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">TTCTAATTTTGCTATCGATA</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3538454</td>
<td align="left">ATAAAAGATCATCCTATTCT</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">AATTTTGCTATCGATAGCCC</td>
<td align="left">Yes - 4</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3538456</td>
<td align="left">AAAAGATCATCCTATTCTAA</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">TTTTGCTATCGATAGCCCTT</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3583105</td>
<td align="left">TTATAAATTTATAAATGATA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">AAATAAAGTGCAAGAAGAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3665610</td>
<td align="left">CTAGATTTAACTTTGCAACG</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TTTCGCAATCGGTGAACAGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3730377</td>
<td align="left">TGCAACAATTCATTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">ACCTAATTTGTTTTCGACAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3855791</td>
<td align="left">GGCATTTTTCACGTACAGGT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">AGTCGAAACATTATTAAATA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4055068</td>
<td align="left">ATTTAAAAAAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TTATAAGACATACCCTTTCG</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4276903</td>
<td align="left">TAGAATGTTTTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GAGAATATTATTCACACGCC</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4304172</td>
<td align="left">TCAAACCTATAAACAGGAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GATAAACGAAAATAGAATTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4374343</td>
<td align="left">AGGCTAAGTCTAGGTTGTAG</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">AAAAATCGTTTAGTCTGTAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4375420</td>
<td align="left">GTCTGAAGACTTTACCTTCT</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TCTTTTGTGCTCCTATCAAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4407495</td>
<td align="left">TCTCGTATAAGTTCATCCTG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">CGAAGCCGTTGTAGCCTATT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4410193</td>
<td align="left">ATCGACACAGATCGCGGCGG</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">AAGAAAAAGCAACTGAAGAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17035632">PMID:17035632</a></td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4431859</td>
<td align="left">ATCGAAACTACGTTGTGAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TAACTTAGCAAATATATGGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4431868</td>
<td align="left">ACGTTGTGAAATAACTTAGC</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">AAATATATGGTAAATAACAC</td>
<td align="left">Yes - 9</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4442713</td>
<td align="left">TGGAAGCTGATTTTTTGTTA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">TATGTCGTAAGTTACTGTGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4455162</td>
<td align="left">TAACTAGAAAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CCAAAGATAAAAAATGAAGT</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4721884</td>
<td align="left">AGTAGGAGTAGAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TTTTCAGCGCCTCGTCTCCT</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4764269</td>
<td align="left">TTTGAAAAAAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TTTCTACATTTCTTTCTTTA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4919602</td>
<td align="left">GTTTTTTGGGTAAAGATCTA</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">AGGGTATATTGCTTTTTTAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">5142628</td>
<td align="left">CTGAGGAAGACGTTCCGAAG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TAAAGTGGAAAACGTTAGAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">5148423</td>
<td align="left">AATAAAACAGTAAAAAAAAA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">GATAAAAAGCTGAACAGTAT</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">5173097</td>
<td align="left">CACAGATAGCTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">AACAGGTACTTTATACATGG</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">5368264</td>
<td align="left">AGTATAGTTGGTCTCCCTCC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">ATATACGGTTTGGATAAGTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">5368273</td>
<td align="left">GGTCTCCCTCCCATATACGG</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">TTTGGATAAGTTTCGCCTTG</td>
<td align="left">Yes - 9</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">5400079</td>
<td align="left">TACATTTTAGGATCAGCTCA</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">AAGTGTCCCTTTTTACAAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">37049</td>
<td align="left">ATTAATAATATATTTGCTCA</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">CCTAGATTTAAAGAATTTAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">128007</td>
<td align="left">TCAGAGCAGTCAATAAAAAA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">TAAAAAATCGAACAAAGGAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">146086</td>
<td align="left">GCCCTCGCTTGTTCCCCCCC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">TTATCTTACGAGTATATAGC</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">266717</td>
<td align="left">ATTCATGAGATTCGTACACG</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">CCTTACCAAGTCTTGCCAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">266779</td>
<td align="left">TGTTAGGGGTGATATGGGTT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">CGTTTTATCTGGGATCAAGG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">276538</td>
<td align="left">GACTTTAATTTATTCATGGA</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">CCTGAAGCTGCTCAAACTGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">685727</td>
<td align="left">ATCTCGCTGCTCAAATGTCA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">ATGCTGATTTTGGCTCCAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">699950</td>
<td align="left">TTTCTTTGAATTTTTTGTTT</td>
<td align="left">G</td>
<td align="left">T</td>
<td align="left">GTATTTTAAATTTATTATCT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">699951</td>
<td align="left">TTCTTTGAATTTTTTGTTTT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">TATTTTAAATTTATTATCTT</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">699960</td>
<td align="left">TTTTTTGTTTTGTATTTTAA</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TTTATTATCTTCCTGTCTTT</td>
<td align="left">Yes - 9</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">700010</td>
<td align="left">CCTCTATTTTTACAGATTTA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">CAAGTTTTCAACTTCGAAAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">700018</td>
<td align="left">TTTACAGATTTAACAAGTTT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">CAACTTCGAAACCCTAGAAA</td>
<td align="left">Yes - 8</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">700019</td>
<td align="left">TTACAGATTTAACAAGTTTT</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">AACTTCGAAACCCTAGAAAT</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">700020</td>
<td align="left">TACAGATTTAACAAGTTTTC</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">ACTTCGAAACCCTAGAAATT</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">700023</td>
<td align="left">AGATTTAACAAGTTTTCAAC</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">TCGAAACCCTAGAAATTTTC</td>
<td align="left">Yes - 3</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">700027</td>
<td align="left">TTAACAAGTTTTCAACTTCG</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">AACCCTAGAAATTTTCATAT</td>
<td align="left">Yes - 4</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">700112</td>
<td align="left">CTTTATCAAAAGTTATCATT</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">AATTGCCCTGCAACCGATAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">869805</td>
<td align="left">CTCTTCCAAGTCGGGTTCAT</td>
<td align="left">C</td>
<td align="left">G</td>
<td align="left">ATGCACTAATGAATTGTAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">879032</td>
<td align="left">ACTAGAGAAAGAAAAAAAGT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">ACAGTGGCTGATATTACAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">888600</td>
<td align="left">AAGAAAACTTTTGGTTATAC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">GAAACTTTGTGTGAAATGTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">903106</td>
<td align="left">AGCACAAACAATTCGAACTG</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GTGCTAACATCTAATAAAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">923542</td>
<td align="left">TTTAAAAAAATGACAGACGA</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">TTAAACTATGTTTTGAAATT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">943553</td>
<td align="left">AGCAAAGAATGAAGCCCTTA</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">ATTTTTGGATAGTTTTAAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">953200</td>
<td align="left">GTCGTTTCGAAGTTTTAGAA</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">ATGAAAAAAATTGGGCGTCT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">967093</td>
<td align="left">GAATTTCCCTTGACTTTTGA</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">GGAAGCCATATCTTCGAGTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1155902</td>
<td align="left">CTTGCAAAAACAAAAATATA</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">ATCCTTTTTTCTGCTGTAAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1159677</td>
<td align="left">TGAGTTTATTCATGAATTGC</td>
<td align="left">C</td>
<td align="left">G</td>
<td align="left">CGAAAATGAGGGTCCATCCG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1159693</td>
<td align="left">TTGCGCGAAAATGAGGGTCC</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TCCGTTCTCTACGATTCTGG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1163155</td>
<td align="left">GTATATAAGTAAATAAAAAT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">AGTATGCTTCTGGTCAACTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1308626</td>
<td align="left">GTTCGGCGAAGGTTTTTTTT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">GCATTGATCAACAGGTGATG</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1672858</td>
<td align="left">GCTTCCGTTTATATACGCGT</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">GGACATATAACAATGAACTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1673001</td>
<td align="left">GATGCTCCACCACCTGAAAC</td>
<td align="left">C</td>
<td align="left">G</td>
<td align="left">TTGATGCGTTCATTAGAGTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1678692</td>
<td align="left">TTTGCATTGGCAGCGACGTT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">GTATTTGCCGGCAAGCCCAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1683323</td>
<td align="left">CATTATGGTTGGAAATGATG</td>
<td align="left">G</td>
<td align="left">T</td>
<td align="left">AAACACCGTATACTTAGCTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1697123</td>
<td align="left">AGACTTGTTAATCATGTTAG</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">ACCTATGCTTTAATATAATT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1716317</td>
<td align="left">TACCGAGAAAACTTTCGTCA</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">ATTTTATGGGCCTAATAATA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1716319</td>
<td align="left">CCGAGAAAACTTTCGTCAAA</td>
<td align="left">G</td>
<td align="left">T</td>
<td align="left">TTTATGGGCCTAATAATAAT</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1716325</td>
<td align="left">AAACTTTCGTCAAATTTTAT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">GGCCTAATAATAATCTTATG</td>
<td align="left">Yes - 6</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1784825</td>
<td align="left">GGCCACATGTCATACATTGA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">TACGACTGCAAAGTTGACCT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1849776</td>
<td align="left">TGTCAGAATCATTGTCGAGG</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">TTTTGCATATCTCGTAAGCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1861420</td>
<td align="left">GAAAAGCATAAGAAGTTAAT</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">TACATAAAAGCACTTTAAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1867805</td>
<td align="left">AAAGTACTTATTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">ACGAATAAAATGTAAAAAAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1873781</td>
<td align="left">GTGGTTTGTAATATTCCTAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CCACCCCATTTTTTTCTGGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1877305</td>
<td align="left">ACAGCCCGATTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GTCTCAGTAATTCAAGAGAC</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1941966</td>
<td align="left">GATTCACATACCGAAGCGGA</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GAAAATTAATAAATCAGAGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1941982</td>
<td align="left">CGGAGAAAATTAATAAATCA</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GAGTTTTAACTCAAAGGAAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1948954</td>
<td align="left">CGGCGTTGAAGCGATCTTGA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">GCGGCCTGGAGTGCGCGAGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1950052</td>
<td align="left">ATGTCGGCATTGCCAGAAGG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TTTTTGCTAGAGGCATAGCT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1955509</td>
<td align="left">TTCCTTAAAAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GAAAGAAAATGAAAACTCAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1960392</td>
<td align="left">AATTAAATTGTCTAAACGTA</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TTCGTTGCTCAAGCTTCCTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1962418</td>
<td align="left">TAAGAGAGAAAACTCGGTGT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">TATTGGAAAAGTGGTACTAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1962438</td>
<td align="left">TTATTGGAAAAGTGGTACTA</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">CAAGTGTGGGAAGTATGTTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1962457</td>
<td align="left">AACAAGTGTGGGAAGTATGT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">TGACAGATTCATTACATTAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1987097</td>
<td align="left">AGCTTCGCATAGACATATAC</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">AACGGAACGAATGACACCTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1987101</td>
<td align="left">TCGCATAGACATATACCAAC</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">GAACGAATGACACCTGGCAG</td>
<td align="left">Yes - 4</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1987117</td>
<td align="left">CAACGGAACGAATGACACCT</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">GCAGTGAAAATTCCCTTTAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2044928</td>
<td align="left">AATTGACTTATCTATTAATT</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">ACTTGATTTTTGTTTGTTAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2049893</td>
<td align="left">GAAACTAGCTCATTATGATT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">GAAGTTTGTGCCAGATAAAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2053517</td>
<td align="left">AGTGATTTTCTCAAAAAGGC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TCTGCTACCCGTCATATTGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2108180</td>
<td align="left">GTAAGATAATAAAGCATAAT</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CCAATCTTCTTTGTTTGAAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2187407</td>
<td align="left">CATTTTTCTTGGTGGGAAGG</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CATCAAAAAAATCCCCCTGG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2219929</td>
<td align="left">AATTGACTGTTTGGCTGTAT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GGTGATATGCTGAAAGCGAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2228774</td>
<td align="left">AAGACAAGATAAATAATAAG</td>
<td align="left">G</td>
<td align="left">T</td>
<td align="left">TGGAAAAGTATATACCTCTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2228776</td>
<td align="left">GACAAGATAAATAATAAGTT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">GAAAAGTATATACCTCTGTT</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2264919</td>
<td align="left">ATAATAAAAACATTTATTTA</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">GTTTTATTTTTGAACTTCTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2293304</td>
<td align="left">ATTTATCACAACGAAAAAAA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">CTCGTATCACTATAATTTTT</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2436579</td>
<td align="left">ACCTTGCAGCCAACTACTTC</td>
<td align="left">G</td>
<td align="left">C</td>
<td align="left">TCACCAATTACAACTTCTGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2444542</td>
<td align="left">GAATAATGCTTCCTACATTG</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">TACCGCCTATTTAATTACAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2592434</td>
<td align="left">CCTAAACATAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CTTACAATCAATGTATATGG</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2676335</td>
<td align="left">TTTTACCGGTTCTAAAATAT</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">CCAAAGACCTGAAACCCCCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2709415</td>
<td align="left">CGTAGCACAAGGCCGAGAGC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TCTTCTCATAGTGGCTCTTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left">pers. comm. D. Bitton</td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2788933</td>
<td align="left">ATATTTTGCTGGATTAAAGC</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">AAATAACTTTTGCGTCAAAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2798041</td>
<td align="left">ACATTCATGGATGAAGACCT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">GGGAACATGGGAAGGCCTAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2798514</td>
<td align="left">ATTCAGATGTCATGTTCAAA</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">AAAAAGATAGACTATCCTAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2811496</td>
<td align="left">CAGAAAGTACAAAGCAATCA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">TACAAGAAGAAGAGAATATC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2938969</td>
<td align="left">AGGCAACTTTTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">AAGGGTTGTAGTATTGTTTA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">3006136</td>
<td align="left">GTATTCTATACCTTGGGCAG</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TTCTTCAAGCAATTTGGTCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">3040333</td>
<td align="left">ACTCTAGATCTCCGTCTCCG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">CTGCACGCCCTATTTCTCGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">3619004</td>
<td align="left">CAACCGAGAAGTCCCGGTAG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TTGCGTCTTTGTATATAAAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">3838114</td>
<td align="left">AGACCTCAAAAAAAAAAAAA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">GCGAAATGCTAACTTCAAAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">4122325</td>
<td align="left">ATCGCTGTTAATGGTGCACA</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">ATCCACAACGTGATTGGAAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">4254961</td>
<td align="left">TGGCCGGGTTATTCACAAGA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">TACCTTTTATCCAGCTTGAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">4400665</td>
<td align="left">CAAATTAAACAATCCAAACC</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">TAAATCGTCACCAGTACTAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">227729</td>
<td align="left">TTTCGTGTTCACGAGGAAGA</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">CGTTGTTTTTGCACATAATG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">240810</td>
<td align="left">TGACCATACCATTGGGATGA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">CCTTACTCTTGATTAGCTCG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">284656</td>
<td align="left">AGTATAACTAGCGCAGTAGG</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">AAAACAGAAGATTGTATGAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719956</td>
<td align="left">GTTTAATATATTTAATAAAC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">TATTTTATTCTATGTCGAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719970</td>
<td align="left">ATAAACCTATTTTATTCTAT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">TCGAGAGAAGATTGAGATGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719973</td>
<td align="left">AACCTATTTTATTCTATGTC</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">AGAGAAGATTGAGATGATTA</td>
<td align="left">Yes - 3</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719975</td>
<td align="left">CCTATTTTATTCTATGTCGA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">AGAAGATTGAGATGATTAAT</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719977</td>
<td align="left">TATTTTATTCTATGTCGAGA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">AAGATTGAGATGATTAATCA</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719978</td>
<td align="left">ATTTTATTCTATGTCGAGAG</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">AGATTGAGATGATTAATCAT</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719980</td>
<td align="left">TTTATTCTATGTCGAGAGAA</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">ATTGAGATGATTAATCATTC</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719984</td>
<td align="left">TTCTATGTCGAGAGAAGATT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">AGATGATTAATCATTCCCTC</td>
<td align="left">Yes - 4</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719986</td>
<td align="left">CTATGTCGAGAGAAGATTGA</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">ATGATTAATCATTCCCTCTA</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719987</td>
<td align="left">TATGTCGAGAGAAGATTGAG</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TGATTAATCATTCCCTCTAT</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719989</td>
<td align="left">TGTCGAGAGAAGATTGAGAT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">ATTAATCATTCCCTCTATCT</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719990</td>
<td align="left">GTCGAGAGAAGATTGAGATG</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TTAATCATTCCCTCTATCTA</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">720000</td>
<td align="left">GATTGAGATGATTAATCATT</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">CCTCTATCTATTGATTGTTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">720696</td>
<td align="left">TCCACAATTTTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">ACCATCTCCTTTAAATCAAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">867874</td>
<td align="left">CTTTATTTTACTTTTTTTTT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">ATATTTTTTAAAATATATTC</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">1016312</td>
<td align="left">TAACAATGAAATAACGAAGC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TTCAGTATAATTAGTTCTAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">1168327</td>
<td align="left">TCATAGCCTTTTATTACTAT</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">TGGGAAGTCAATTTTTAGTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">1374937</td>
<td align="left">AGGTTGTTATGTTTCAGACT</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TGATCATTACGGTGATACTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">1685491</td>
<td align="left">GGCCACCACCAAGAAGAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CCAAAGAAAAAGAACAGTCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">1756401</td>
<td align="left">AAACCTTTGATGAAAGTTTA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">TAAAGCTGCCAATGTATATA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">1765400</td>
<td align="left">TAAATTGCCGTATTATATTG</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">TACTATAACGAAATAATTGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">1774235</td>
<td align="left">AACATCTTCCAAGTTCTCAT</td>
<td align="left">GATC</td>
<td align="left"> </td>
<td align="left">CTTTAAAGATTCCTCAAGGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">2381978</td>
<td align="left">AATTTGATGTCTTAAATTTG</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">TGGTTTGCTTGAAGGCATAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'sequencing-status'">
<h1 id="sequencing-status">Sequencing status</h1>
<p>Note: Many older <em>S. pombe</em> sequence submissions to the DNA databases (International Nucleotide Sequence Database Collaboration databases, i.e. ENA, GenBank, DDBJ) contain one or more errors, and we do not have the resources to maintain past sequences or flag every error in PomBase.</p>
<h2 id="chromosome-1">Chromosome 1</h2>
<table>
<thead>
<tr class="header">
<th align="left">Contig Name</th>
<th align="left">Region</th>
<th align="left">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"> </td>
<td align="left">unsequenced to chr1 left telomere</td>
<td align="left">10 kb ± 2 kb*</td>
</tr>
<tr class="even">
<td align="left">c212</td>
<td align="left">sub-telomeric left arm</td>
<td align="left">29,663 bp</td>
</tr>
<tr class="odd">
<td align="left"> </td>
<td align="left">Gap</td>
<td align="left">&lt;5kb*</td>
</tr>
<tr class="even">
<td align="left">c977</td>
<td align="left">left arm and right arm</td>
<td align="left">5,549,370 bp</td>
</tr>
<tr class="odd">
<td align="left"> </td>
<td align="left">unsequenced to chr1 left telomere</td>
<td align="left">18kb ± 3kb*</td>
</tr>
</tbody>
</table>
<p> </p>
<h2 id="chromosome-2">Chromosome 2</h2>
<table>
<thead>
<tr class="header">
<th align="left">Contig Name</th>
<th align="left">Region</th>
<th align="left">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"> </td>
<td align="left">unsequenced to chr2 left telomere</td>
<td align="left">25kb ± 5kb*</td>
</tr>
<tr class="even">
<td align="left">c1348</td>
<td align="left">sub-telomeric left arm</td>
<td align="left">80,201bp</td>
</tr>
<tr class="odd">
<td align="left"> </td>
<td align="left">Gap</td>
<td align="left">22kb ± 5kb*</td>
</tr>
<tr class="even">
<td align="left">pB10D8</td>
<td align="left">left arm to centromeric gap</td>
<td align="left">1,536,269 bp</td>
</tr>
<tr class="odd">
<td align="left"> </td>
<td align="left">Gap</td>
<td align="left">~6kb</td>
</tr>
<tr class="even">
<td align="left">pJ5566</td>
<td align="left">right arm from centromeric gap to telomeric repeats</td>
<td align="left">2,923,134 bp</td>
</tr>
</tbody>
</table>
<p> </p>
<h2 id="chromosome-3">Chromosome 3</h2>
<table>
<thead>
<tr class="header">
<th align="left">Contig Name</th>
<th align="left">Region</th>
<th align="left">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">p20C8</td>
<td align="left">left arm from centromeric gap</td>
<td align="left">1,083,348 bp</td>
</tr>
<tr class="even">
<td align="left"> </td>
<td align="left">Gap</td>
<td align="left">25.3kb ± 6kb**</td>
</tr>
<tr class="odd">
<td align="left">c1676</td>
<td align="left">right arm from centromeric gap</td>
<td align="left">1,369,435 bp</td>
</tr>
</tbody>
</table>
<p>* pers comm. Randy Hyppa and Gerry Smith.<br />
** pers comm. Chad Ellermeier and Gerry Smith.</p>
<h3 id="centromere-clone-information"><a routerLink="/status/centromeres">Centromere clone information</a></h3>
<h3 id="telomere-clone-information"><a routerLink="/status/telomeres">Telomere clone information</a></h3>
<h3 id="mating-type-region-information"><a routerLink="/status/mating-type-region">Mating type region information</a></h3>
<h3 id="ribosomal-dna-information-faq"><a href="http://www.pombase.org/faqs/are-there-any-rdna-repeat-sequences-pombase">Ribosomal DNA information (FAQ)</a></h3>

  </div>
  <div *ngIf="pageName == 'sequencing-updates'">
<h1 id="s.-pombe-sequence-updates-since-july-2003"><em>S. pombe</em> Sequence updates since July 2003</h1>
<h3 id="aug-29-2008-chromosome-1-cosmid-c30-or-c29e6-affecting-spac29e6.04cnnf1">Aug 29 2008 Chromosome 1 cosmid C30 OR c29E6 affecting SPAC29E6.04c/Nnf1</h3>
<p><strong>PENDING <span class="no-break">2008-08-29</span></strong></p>
<p>The reference sequence appears to contain an extra 'G' at position 4410193 PMID:17035632, also confirmed by Broad sequence; Pers. comm. Aki Hayashi Reported 05/16/07</p>
<h3 id="aug-29-2008-chromosome-1-cosmid-c767-affecting-spac767.01cvps1">Aug 29 2008 Chromosome 1 cosmid c767 affecting SPAC767.01c/Vps1</h3>
<p><strong>PENDING <span class="no-break">2007-08-23</span></strong></p>
<p>At 1999191 in Vps1 TGTGACC[C-&gt;G]TACGAC PMID:18088324, also confirmed by Broad sequence; Pers. comm. Isabelle Jourdain Reported 06/15/05</p>
<h3 id="aug-29-2008-chromosome-2-cosmid-c21d10-affecting-spbc21d10.06cmap4">Aug 29 2008 Chromosome 2 cosmid c21D10 affecting SPBC21D10.06c/Map4</h3>
<p><strong>PENDING <span class="no-break">2008-08-29</span></strong></p>
<p>Map4 in the reference genome contains an array of 5 repeats. In PMID:168571979 repeats are reported. This number is the correct number of repeats and will be updated via an insertion into the contig sequence shortly S. Pers. comm. Henar Valdivieso Reported 06/15/05</p>
<h3 id="aug-29-2008-chromosome-1-cosmid-c27d7-misassembly-causing-duplication-of-spac27d7.09c-spac27d7.10c">Aug 29 2008 Chromosome 1 cosmid c27D7 misassembly causing duplication of SPAC27D7.09c (SPAC27D7.10c)</h3>
<p><strong>PENDING <span class="no-break">2008-08-29</span></strong></p>
<p>An apparent repeat region on chromosome 1 coordinates 4526059..4529095 (cosmid c27D7) is caused by a missassembly and will be removed from the genomic sequence shortly. The CDS feature SPAC27D7.10c within this region is an exact duplication of SPAC27D7.09c and will be merged with this CDS. Pers. comm. Klavs Hansen. Reported 09/01/04</p>
<h3 id="jun-2008-gatc-in-overlap-c1450-and-c1442-affects-spcc1442.04c">Jun 2008 GATC in overlap c1450 and c1442 affects SPCC1442.04c</h3>
<p><strong>PENDING <span class="no-break">2008-08-26</span></strong></p>
<p>There are 4 bases missing from this sequence AACATCTTCCAAGTTCTCAT GATC CTTTAAAGATTCCTCAAGGT, which will change the C-terminal region slightly. This will be fixed in the next round of corrections</p>
<h3 id="jun-2008-197-potential-sequence-discrepancies-from-the-broad-institute">Jun 2008 197 potential sequence discrepancies from the Broad Institute</h3>
<p><strong>PENDING <span class="no-break">2008-08-26</span></strong></p>
<p>Generated as part of a technology development project, using Illumina (Solexa) technology the Broad Institute generated &gt;200-fold sequence coverage of the genome of S. pombe strain 972. These data identified 197 potential discrepancies. The discrepancy rate is about 1 in 100,000, which is close to the estimated error rate in published reference sequence. Further information and data <a routerLink="/status/sequence-updates-pending">here</a></p>
<h3 id="jan-2007-genedb-moves-from-contigs-to-chromosomes">Jan 2007 GeneDB moves from Contigs to Chromosomes</h3>
<p>Contigs merged into chromosomes: the 4 sequence gaps represented by 100 Ns<br />
(<strong>Note:</strong> that the chromosomes previously made available from the ftp site had 1000 Ns in the gaps)</p>
<h3 id="may-2006-chromosome-1-cosmid-c21e11">May 2006 Chromosome 1 cosmid c21E11:</h3>
<p>Single base insertion at c21E11 from gagtgca to gagtgGca (in cyp8) base 4257404 in new chromsome file (note reverse strand sequence reported)</p>
<h3 id="may-2004-chromosome-2-cosmid-c646">May 2004 Chromosome 2 cosmid c646:</h3>
<p>Single base transition A-&gt;C at c646 from aatggaccAtacccc to aatggaccCtacccc<br />
(<strong>Note:</strong> This change does not affect sequence coordinates)</p>
<h3 id="apr-2004-chromosome-1-cosmid-c6b12">Apr 2004 Chromosome 1 cosmid c6B12:</h3>
<p>Single base deletion at c6B12 13981 from tgttctgcAtttttcc to tgttctgctttttcc<br />
(<strong>Note:</strong> this change was not added to the spread sheet on the ftp site until Jan 18 2006)</p>
<h3 id="jul-2003-chromosome-1-gap-filled">Jul 2003 Chromosome 1 Gap Filled</h3>
<p>Added PCR product SPACPJ113 6900 with overlaps of 504 bp with c1856 and 147bp with p4C9<br />
Joined contigs p4C9 and c977</p>

  </div>
  <div *ngIf="pageName == 'statistics'">
<h1 id="genome-statistics">Genome statistics</h1>
<h3 id="genome-structure-and-organisation">Genome Structure and Organisation</h3>
<table>
<tbody>
<tr class="odd">
<td align="left">Genome Size (sequenced portion, excluding rDNA)</td>
<td align="left">12.57Mbp</td>
</tr>
<tr class="even">
<td align="left">Number of Chromosomes</td>
<td align="left">3</td>
</tr>
<tr class="odd">
<td align="left">Chromosome Size Range</td>
<td align="left">2.45Mbp - 5.58Mbp</td>
</tr>
<tr class="even">
<td align="left">Gene Count (protein coding genes on Chr I, II and III)</td>
<td align="left">5118</td>
</tr>
<tr class="odd">
<td align="left">Gene Count excluding dubious genes (protein coding genes on Chr I, II and III)</td>
<td align="left">5064</td>
</tr>
<tr class="even">
<td align="left">Gene Density (genes per Mb)</td>
<td align="left">552.66</td>
</tr>
<tr class="odd">
<td align="left">Genome GC Content</td>
<td align="left">36.06%</td>
</tr>
<tr class="even">
<td align="left">Intergenic Length (mean for protein coding genes)</td>
<td align="left">227.37</td>
</tr>
<tr class="odd">
<td align="left">Intergenic GC Content (protein coding genes)</td>
<td align="left">28.94%</td>
</tr>
<tr class="even">
<td align="left">Mitochondrial Genome Size</td>
<td align="left">19.43kbp</td>
</tr>
<tr class="odd">
<td align="left">Mitochondrial GC Content</td>
<td align="left">30.09%</td>
</tr>
<tr class="even">
<td align="left">Telomeric Located Contig Size</td>
<td align="left">20.00kbp</td>
</tr>
<tr class="odd">
<td align="left">Telomeric Located Contig GC Content</td>
<td align="left">33.17%</td>
</tr>
</tbody>
</table>
<h3 id="chromosome-i-ii-and-iii-transcript-breakdown">Chromosome (I, II and III) Transcript Breakdown</h3>
<table>
  <thead>
    <tr>
      <th rowspan="2">Feature Type</th>
      <th colspan="3" style="text-align: center;">Count</th>
      <th>Gene Density</th>
      <th colspan="6" style="text-align: center;">GC Content (%)</th>
      <th colspan="5" style="text-align: center;">Mean Length (bp)</th>
    </tr>
    <tr>
      <th>Gene</th>
      <th>Exon</th>
      <th>Intron</th>
      <th>(genes per Mb)</th>
      <th>Gene</th>
      <th>Exon (All)</th>
      <th>Exon (Coding)</th>
      <th>Intron</th>
      <th>5' UTR</th>
      <th>3' UTR</th>
      <th>Gene</th>
      <th>Exon</th>
      <th>Intron</th>
      <th>5' UTR</th>
      <th>3' UTR</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rRNA</td>
      <td>47</td>
      <td>47</td>
      <td>0</td>
      <td>3.74</td>
      <td>45.72</td>
      <td>45.72</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td style="text-align:right;">527.53</td>
      <td style="text-align:right;">527.53</td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>snoRNA</td>
      <td>57</td>
      <td>58</td>
      <td>1</td>
      <td>4.53</td>
      <td>36.54</td>
      <td>36.82</td>
      <td></td>
      <td>30.36</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">140.09</td>
      <td style="text-align:right;">131.88</td>
      <td style="text-align:right;">336.00</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>tRNA</td>
      <td>171</td>
      <td>197</td>
      <td>26</td>
      <td>13.60</td>
      <td>53.79</td>
      <td>54.15</td>
      <td></td>
      <td>38.56</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">77.32</td>
      <td style="text-align:right;">65.56</td>
      <td style="text-align:right;">11.77</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>pseudogene</td>
      <td>28</td>
      <td>50</td>
      <td>22</td>
      <td>2.23</td>
      <td>38.95</td>
      <td>39.25</td>
      <td></td>
      <td>30.90</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">1,047.39</td>
      <td style="text-align:right;">565.70</td>
      <td style="text-align:right;">47.36</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>mRNA (protein_coding)</td>
      <td>5118</td>
      <td>10425</td>
      <td>5307</td>
      <td>407.10</td>
      <td>37.08</td>
      <td>37.39</td>
      <td>39.63</td>
      <td>29.72</td>
      <td>33.24</td>
      <td>32.27</td>
      <td style="text-align:right;">2,147.68</td>
      <td style="text-align:right;">1,011.98</td>
      <td style="text-align:right;">83.27</td>
      <td style="text-align:right;">290.05</td>
      <td style="text-align:right;">428.79</td>
    </tr>
    <tr>
      <td>ncRNA</td>
      <td>1522</td>
      <td>1527</td>
      <td>5</td>
      <td>121.06</td>
      <td>35.82</td>
      <td>35.82</td>
      <td></td>
      <td>27.94</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">1,248.46</td>
      <td style="text-align:right;">1,244.21</td>
      <td style="text-align:right;">49.40</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>snRNA</td>
      <td>5</td>
      <td>6</td>
      <td>1</td>
      <td>0.40</td>
      <td>40.03</td>
      <td>40.47</td>
      <td></td>
      <td>34.00</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">147.40</td>
      <td style="text-align:right;">114.50</td>
      <td style="text-align:right;">50.00</td>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>
<h3 id="mitochondrial-transcript-breakdown">Mitochondrial transcript breakdown</h3>
<table>
<thead>
<tr class="header">
<th align="left">Feature type</th>
<th align="left">Count</th>
<th align="left">GC content (%)</th>
<th align="left">Mean length (bp)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">rRNA</td>
<td align="left">2</td>
<td align="left">31.97</td>
<td align="left">2,122.00</td>
</tr>
<tr class="even">
<td align="left">mRNA (protein_coding)</td>
<td align="left">11</td>
<td align="left">29.39</td>
<td align="left">1,393.00</td>
</tr>
<tr class="odd">
<td align="left">tRNA</td>
<td align="left">25</td>
<td align="left">38.75</td>
<td align="left">74.52</td>
</tr>
</tbody>
</table>
<p><br />
### Telomeric located contig transcript breakdown</p>
<table>
<thead>
<tr class="header">
<th align="left">Feature type</th>
<th align="left">Count</th>
<th align="left">GC content (%)</th>
<th align="left">Mean length (bp)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">mRNA (protein_coding)</td>
<td align="left">5</td>
<td align="left">38.46</td>
<td align="left">1,428.00</td>
</tr>
</tbody>
</table>
<p><br />
### Mating type region</p>
<table>
<thead>
<tr class="header">
<th align="left">Feature type</th>
<th align="left">Count</th>
<th align="left">GC content (%)</th>
<th align="left">Mean length (bp)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">mRNA (protein_coding)</td>
<td align="left">4</td>
<td align="left">35.94</td>
<td align="left">395.75</td>
</tr>
</tbody>
</table>
<p><br />
#### <strong>Release Version:</strong> 30th Jan 2017</p>

  </div>
  <div *ngIf="pageName == 'telomeres'">
<h1 id="telomeres">Telomeres</h1>
<p>The fission yeast complete genome sequence currently stops short of the telomeric repeats. See the <a href="status/sequencing-status">Sequencing Status</a> page for the current assembly status.</p>
<p>The most proximal anchored cosmids to each telomere are (links to genome browser):</p>
<ul>
<li>Chromosome I left c212 (coordinates <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=I%3A1-29663;site=ensemblunit">1-29663</a>)</li>
<li>Cromosome I right c750 (<a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=I%3A5554844-5579133;site=ensemblunit">5554844-5579133</a>)</li>
<li>Chromosome II left c1348 (<a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=II%3A1-39186;site=ensemblunit">1-39186</a>)</li>
<li>Chromosome II right pT2R1 (<a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=II%3A4500619-4539800;site=ensemblunit">4500619-4539800</a>)</li>
<li>There are no telomere proximal clones for chromosome III as the unsequenced rDNA blocks occur inbetween the sequenced portion and the telomeres on both chromosome arms.</li>
</ul>
<p>Details of all clones used for the assembly, and their order, length and overlap details is provided in a set of <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Archived_directories/Cosmid_assembly_data/">spreadsheets</a> available from the <em>S. pombe</em> FTP site.</p>
<p>A contig extending the left arm of Chromosome II was sequenced by Sasaki <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18727152">PMID:18727152</a>) and will be attached to the assembly in the next round of sequence changes. In the meantime, the contig can be viewed in the <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?g=SPBC460.01c;r=AB325691:1479-3197;t=SPBC460.01c.1">PomBase Genome Browser</a>.</p>
<p>A set of <a href="ftp://ftp.sanger.ac.uk/pub/yeast/sequences/pombe/telomeres/">small insert clones</a> (FTP link; also see table) from a telomere plasmid library has been made available by <a routerLink="/mailto:sugawara@hydra.rose.brandeis.edu">Neal Sugawara</a>. None can be assigned to a chromosome at present.</p>
<p>Clones included in the final sequence are available from the Wellcome Trust Sanger Institute; send email to <a href="mailto:archives@sanger.ac.uk">archives@sanger.ac.uk</a></p>
<table>
<thead>
<tr class="header">
<th align="left">Plasmid</th>
<th align="left">Size of insert</th>
<th align="left">Location of telomeric sequence</th>
<th align="left">Comments</th>
<th align="left">Vector</th>
<th align="left">Laboratory</th>
<th align="left">Funded by</th>
<th align="left">Stage</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">pNSU28</td>
<td align="left">approx 1kb</td>
<td align="left"> </td>
<td align="left">Lies in pNSU21</td>
<td align="left">pUC19</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="even">
<td align="left">pNSU31</td>
<td align="left">approx 1kb</td>
<td align="left"> </td>
<td align="left">Lies in pNSU21</td>
<td align="left">pUC19</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="odd">
<td align="left">pNSU68</td>
<td align="left">423 bp</td>
<td align="left">Internal</td>
<td align="left">Contains 195bp of telomeric DNA and 123bp from the rDNA .</td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="even">
<td align="left">pNSU71</td>
<td align="left">15 kb</td>
<td align="left">Terminal</td>
<td align="left"> </td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="odd">
<td align="left">pNSU64 <sup>*</sup></td>
<td align="left">6.9 kb</td>
<td align="left">Terminal</td>
<td align="left"> </td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="even">
<td align="left">pNSU70</td>
<td align="left">7.1 kb</td>
<td align="left"> </td>
<td align="left"> </td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="odd">
<td align="left">pNSU77 <sup>*</sup></td>
<td align="left">12 kb</td>
<td align="left">Internal</td>
<td align="left">Fusion between telomere sequences (7.1kb) and rDNA sequences (4.9kb)</td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="even">
<td align="left">pNSU21</td>
<td align="left">7.9 kb</td>
<td align="left">Terminal</td>
<td align="left">pNSU21 and pNSU65 were isolated from the first and second library respectively</td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="odd">
<td align="left">pNSU65</td>
<td align="left">8.1 kb</td>
<td align="left">Terminal</td>
<td align="left">pNSU21 and pNSU65 were isolated from the first and second library respectively</td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
</tbody>
</table>
<p><strong>Table notes:</strong><br />
&quot;Terminal&quot; refers to the position adjacent to the vector sequences of pMLC12 where the blunt end (SmaI end) ligated to the telomeric sequences. &quot;Internal&quot; means that an <em>S. pombe</em> sequence intervenes between the telomere and vector sequences.</p>

  </div>
  <div *ngIf="pageName == 'unmapped_genes'">
<h1 id="unmapped-genes">Unmapped genes</h1>
<h3 id="genetic-loci-not-cloned-or-physically-mapped">Genetic loci not cloned or physically mapped</h3>
<table>
 <thead>
  <tr>
   <th>Locus name</th>
   <th>Publication(s)</th>
   <th>Allelic with</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td>ala1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>all1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24177583">PMID:24177583</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>all2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24177583">PMID:24177583</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ani1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ani2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>cyh4, tri3</td>
 </tr>
 <tr>
  <td>anr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>anr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>anr3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>anr4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>aro3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=4211223">PMID:4211223</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ban1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8187760">PMID:8187760</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 <tr>
  <td>ban2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ban3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ban4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
  <tr>
  <td>ben2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7003311">PMID:7003311</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ben3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7003311">PMID:7003311</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>can2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18357653">PMID:18357653</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>chr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12746851">PMID:12746851</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15114862">PMID:15114862</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps2</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps4</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps6</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps7</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps9</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7882425">PMID:7882425</a></td>
  <td>&nbsp;</td>
 <tr>
  <td>cps11</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps13</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps14</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 <tr>
  <td>crm2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2647765">PMID:2647765</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=607151">PMID:607151</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1068110">PMID:1068110</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1068110">PMID:1068110</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1068110">PMID:1068110</a></td>
  <td>ani2, tri3</td>
 </tr>
 <tr>
  <td>glu1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7903653">PMID:7903653</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>git4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2157626">PMID:2157626</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>hsp1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7500954">PMID:7500954</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>hus3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1427071">PMID:1427071</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>hus4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1427071">PMID:1427071</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>hyd1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=23712692">PMID:23712692</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ird11</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=22095476">PMID:22095476</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=22173629">PMID:22173629</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=25117315">PMID:25117315</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mcs3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11085271">PMID:11085271</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2474475">PMID:2474475</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15509866">PMID:15509866</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mcs5</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2474475">PMID:2474475</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis7</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis8</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis9</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
  <tr>
  <td>orb1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8187760">PMID:8187760</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>orb8</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>orb9</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>orb10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>orb12</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pet1g</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld6</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld8</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld9</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld11</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld12</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ptp1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644270">PMID:1644270</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8082169">PMID:8082169</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8887563">PMID:8887563</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12058079">PMID:12058079</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ptp2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644270">PMID:1644270</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8082169">PMID:8082169</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ral4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=3071741">PMID:3071741</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec50</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec52</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec53</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec54</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec55</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec56</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec58</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec59</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec60</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>scr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9535817">PMID:9535817</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>scr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9535817">PMID:9535817</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SLrsm1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21046351">PMID:21046351</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SLrsm2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21046351">PMID:21046351</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SLrsm3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21046351">PMID:21046351</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SP331</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7937765">PMID:7937765</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ste2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1934121">PMID:1934121</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ste10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1934121">PMID:1934121</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ste15</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8575018">PMID:8575018</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts7</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts8</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts9</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts11</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tnr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7961415">PMID:7961415</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644306">PMID:1644306</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tnr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7961415">PMID:7961415</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644306">PMID:1644306</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tri3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>ani2, cyh4</td>
 </tr>
 <tr>
  <td>tri4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tri5</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>trs1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1769544">PMID:1769544</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>wtr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8913338">PMID:8913338</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15338055">PMID:15338055</a></td>
  <td>&nbsp;</td>
 </tr>
 </tbody>
</table>

  </div>
  </div>
</div>
</div>
