<div class="docs">
<div *ngIf="section == 'about'">
  <div class="docs-menu">
<h3 id="about"><a routerLink="/about" routerLinkActive="active">About</a></h3>
<ul>
<li><a routerLink="/about/contacts" routerLinkActive="active">Contacts</a></li>
<li><a routerLink="/about/citing-pombase" routerLinkActive="active">Citing PomBase</a></li>
<li><a routerLink="/help" routerLinkActive="active">Help</a></li>
<li><a routerLink="/about/resource-metrics" routerLinkActive="active">Resource Metrics</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'citing-pombase'" class="docs-content">
<h1 id="citing-pombase">Citing PomBase</h1>
<p>To cite <em>S. pombe</em> genome data and annotation, please use:</p>
<p>Wood V, Harris MA, McDowall MD, Rutherford K, Vaughan BW, Staines DM,​ Aslett M, Lock A, Bähler J, Kersey PJ, Oliver SG.<br />
PomBase: a comprehensive online resource for fission yeast.<br />
<em>Nucleic Acids Res.</em> 2012 Jan;40(Database issue):D695-9. Epub 2011 Oct 28.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">PMID:22039153</a> <a href="http://dx.doi.org/DOI:10.1093/nar/gkr853">DOI:10.1093/nar/gkr853</a></p>
<p>You may also wish to cite the most recent PomBase NAR Database Issue paper:</p>
<p>McDowall MD, Harris MA, Lock A, Rutherford K, Staines DM, Bähler J,​ Kersey PJ, Oliver SG, Wood V.<br />
PomBase 2015: updates to the fission yeast database.<br />
<em>Nucleic Acids Res.</em> 2015 Jan;43(Database issue):D665-61. Epub 2014 Oct 31.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25361970">PMID:25361970</a> <a href="http://dx.doi.org/10.1093/nar/gku1040">DOI:10.1093/nar/gku1040</a></p>
<h3 id="genome-sequence">Genome Sequence</h3>
<p>To cite the <em>S. pombe</em> genome sequence, use:</p>
<p>Wood V, Gwilliam R, Rajandream MA, Lyne M, Lyne R, Stewart A, Sgouros J,​ Peat N, Hayles J, Baker S, <em>et al.</em><br />
The genome sequence of <em>Schizosaccharomyces pombe</em>.<br />
<em>Nature</em>. 2002 Feb 21;415(6874):871-80. Erratum in: Nature 2003 Jan 2;421(6918):94. Cerrutti L [corrected to Cerutti L].</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/11859360">PMID:11859360</a></p>
<p><a href="http://www.nature.com/nature/journal/v415/n6874/full/nature724.html">Full text at Nature</a></p>
<h3 id="go-annotations">GO Annotations</h3>
<p>To cite the <em>S. pombe</em> Gene Ontology annotation data, use the 2012 NAR Database Issue paper noted above.</p>
<p>(Note that GO annotation for <em>S. pombe</em> has also been previously published, but we now recommend the 2012 NAR paper as it is more up-to-date. Aslett M, Wood V. Gene Ontology annotation status of the fission yeast genome: preliminary coverage approaches 100%. <em>Yeast</em> 2006 Oct 15;23(13):913-9. Erratum in <em>Yeast</em> 2007 May;24(5):457. PMID:17072883)</p>
<h3 id="phenotype-ontology-and-annotations">Phenotype Ontology and Annotations</h3>
<p>To cite the Fission Yeast Phenotype Ontology (FYPO) and/or <em>S. pombe</em> FYPO annotation data, use:</p>
<p>Harris MA, Lock A, **Bähler J, Oliver SG, Wood V.<br />
FYPO: The Fission Yeast Phenotype Ontology.<br />
<em>Bioinformatics</em>. 2013 Jul 1;29(13):1671-8. doi: <a href="http://dx.doi.org/10.1093/bioinformatics/btt266">10.1093/bioinformatics/btt266</a>. Epub 2013 May 8.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/23658422">PMID:23658422</a></p>
<p>Full text at Bioinformatics: <a href="http://bioinformatics.oxfordjournals.org/content/29/13/1671.long">HTML</a> or <a href="http://bioinformatics.oxfordjournals.org/content/29/13/1671.full.pdf+html">PDF</a></p>
<h3 id="canto">Canto</h3>
<p>To cite the online curaton tool Canto, use:</p>
<p>Rutherford KM, Harris MA, Lock A, Oliver SG, Wood V.<br />
Canto: An online tool for community literature curation.<br />
<em>Bioinformatics</em>. 2014 Jun 15;30(12):1791-2. Epub 2014 Feb 25.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/24574118">PMID:24574118</a> <a href="http://dx.doi.org/10.1093/bioinformatics/btu103">DOI: 10.1093/bioinformatics/btu103</a></p>
<h3 id="manually-curated-orthologs">Manually Curated Orthologs</h3>
<p>To cite the manually curated <em>S. cerevisiae</em> or human orthologs (displayed on gene pages or downloaded), use the NAR database paper as noted above. Note that the manually curated orthologs use several data sources including Compara, but are distinct from the Compara results themselves.</p>
<p>Wood V, Harris MA, McDowall MD, Rutherford K, Vaughan BW, Staines DM,​ Aslett M, Lock A, Bähler J, Kersey PJ, Oliver SG.<br />
PomBase: a comprehensive online resource for fission yeast.<br />
<em>Nucleic Acids Res.</em> 2012 Jan;40(Database issue):D695-9. Epub 2011 Oct 28.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">PMID:22039153</a></p>
<p><a href="http://nar.oxfordjournals.org/content/40/D1/D695.long">Full text at NAR</a></p>
<h3 id="compara">Compara</h3>
<p>Note that the manually curated <em>S. cerevisiae</em> and human orthologs (displayed on gene pages or downloadable) use several data sources in addition to Compara, and are therefore distinct from the Compara data. To cite the <em>S. pombe</em> Compara data, use:</p>
<p>Vilella AJ, Severin J, Ureta-Vidal A, Heng L, Durbin R, Birney E.<br />
EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates.<br />
<em>Genome Res</em> 2009 Feb;19(2):327-35. Epub 2008 Nov 24.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/19029536">PMID:19029536</a></p>
<p><a href="http://genome.cshlp.org/content/19/2/327.long">Full text at Genome Research</a></p>

  </div>
  <div *ngIf="pageName == 'contacts'" class="docs-content">
<h1 id="contacts">Contacts</h1>
<h2 id="contact-pombase-staff">Contact PomBase Staff</h2>
<p>Email: <a routerLink="/mailto:helpdesk@pombase.org" routerLinkActive="active">helpdesk@pombase.org</a></p>
<p>Post:</p>
<p><span>PomBase</span><br />
Department of Biochemistry<br />
University of Cambridge<br />
80 Tennis Court Road<br />
Cambridge<br />
<span>CB2</span> <span>1GA</span><br />
United Kingdom</p>
<h2 id="submit-data">Submit Data</h2>
<ul>
<li>To register a gene name, please see the <a href="submit-data/gene-naming-guidelines">naming guidelines</a> and complete the <a routerLink="/submit-data/registration-form" routerLinkActive="active">submission form</a></li>
<li>To submit information on individual genes, use the Contact Curators link provided on the relevant <a href="http://www.pombase.org/">PomBase</a> gene page</li>
<li>To submit sequence-linked data, use the <a href="submit-data/data-submission-form">HTP data submission form</a></li>
<li>To submit large sets of phenotype data, use the <a href="submit-data/phenotype-data-submission-form">phenotype data submission form</a></li>
</ul>
<h2 id="receive-project-updates">Receive Project Updates</h2>
<ul>
<li>Subscribe to <a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">Pombelist</a> to receive annotation and data updates</li>
</ul>

  </div>
  <div *ngIf="pageName == 'index'" class="docs-content">
<h1 id="about-pombase">About PomBase</h1>
<p>PomBase is a model organism database that provides organization of and access to scientific data for the fission yeast <em>Schizosaccharomyces pombe</em>. PomBase supports genomic sequence and features, genome-wide datasets and manual literature curation.</p>
<p>PomBase also provides a community hub for researchers, providing genome statistics, a community curation interface, news, events,​ documentation and mailing lists.</p>
<hr />
<table>
<tbody>
<tr class="odd">
<td><a href="http://www.pombase.org/faq">FAQ</a></td>
<td>Old Frequently Asked Questions</td>
</tr>
<tr class="even">
<td><a routerLink="/documentation" routerLinkActive="active">Documentation</a></td>
<td>Documentation</td>
</tr>
<tr class="odd">
<td><a routerLink="/about/version-history" routerLinkActive="active">Data versions</a></td>
<td>Versions of internal and external data loaded into current and past Pombase releases</td>
</tr>
<tr class="even">
<td><a routerLink="/about/contacts" routerLinkActive="active">Contact PomBase</a></td>
<td>How to contact staff or submit data</td>
</tr>
<tr class="odd">
<td><a routerLink="/about/citing-pombase" routerLinkActive="active">Citing PomBase</a></td>
<td>How to cite fission yeast genome data</td>
</tr>
<tr class="even">
<td><a routerLink="/about/pombase-staff" routerLinkActive="active">Staff</a></td>
<td>PomBase staff members</td>
</tr>
</tbody>
</table>
<hr />
<p>Funding: PomBase is funded by the Wellcome Trust [104967/Z/14/Z].</p>

  </div>
  <div *ngIf="pageName == 'pombase-staff'" class="docs-content">
<h2 id="university-of-cambridge"><a href="http://www.cam.ac.uk/">University of Cambridge</a></h2>
<ul>
<li>Val Wood (Curator &amp; co-PI)</li>
<li><span>Midori</span> Harris (Curator)</li>
<li>Kim Rutherford (Programmer)</li>
<li>Steve Oliver (PI)</li>
</ul>
<h2 id="university-college-london"><a href="http://www.ucl.ac.uk/">University College London</a></h2>
<ul>
<li>Antonia Lock (Curator)</li>
<li>Jürg Bähler (co-PI)</li>
</ul>
<h2 id="contact-us">Contact us</h2>
<p>Email: <a routerLink="/mailto:helpdesk@pombase.org" routerLinkActive="active">helpdesk@pombase.org</a></p>
<p>Post:</p>
<p>PomBase<br />
Department of Biochemistry<br />
University of Cambridge<br />
80 Tennis Court Road<br />
Cambridge<br />
CB2 1GA<br />
United Kingdom</p>

  </div>
  <div *ngIf="pageName == 'resource-metrics'" class="docs-content">
<h1 id="resource-metrics">Resource Metrics</h1>
<h2 id="curation">Curation</h2>
<p>The <a href="https://curation.pombase.org/pombe/stats/annotation">Curation Statistics</a> page (currently part of PomBase Canto) provides a set of metrics reflecting the current state of literature curation in PomBase.</p>
<h2 id="web-usage">Web usage</h2>
<p>Statistics forthcoming</p>

  </div>
  <div *ngIf="pageName == 'version-history'" class="docs-content">
<h1 id="pombase-data-version-history">PomBase data version history</h1>
<p><strong>Current: <span class="no-break">2016-10-21</span></strong></p>
<table>
<thead>
<tr class="header">
<th>Dataset</th>
<th>Version</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>INSDC</td>
<td>ASM294v2</td>
</tr>
<tr class="even">
<td>Gene Build</td>
<td>2.1</td>
</tr>
<tr class="odd">
<td>Annotation Version</td>
<td>61</td>
</tr>
<tr class="even">
<td>GO GAF</td>
<td><span class="no-break">2016-10-21</span></td>
</tr>
<tr class="odd">
<td>InterPro</td>
<td>InterProScan5(5.14-53.0 23 July 2015)</td>
</tr>
<tr class="even">
<td>Manually Curated Ortholog table</td>
<td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
</tr>
</tbody>
</table>
<p><strong>Past PomBase Releases</strong></p>
<table border="1">
 <tbody>
 <tr>
  <td>Release Date</td>
  <td>INSDC</td>
  <td>Gene Build</td>
  <td>Ensembl Software Release</td>
  <td>Annotation Version</td>
  <td>GO GAF</td>
  <td>InterPro</td>
  <td>Manually Curated Ortholog table</td>
 </tr>
 <tr>
  <td><span class="no-break">2016-06-01</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>60</td>
  <td><span class="no-break">2016-06-01</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-05-10</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>59</td>
  <td><span class="no-break">2016-05-11</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-04-12</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>58</td>
  <td><span class="no-break">2016-04-12</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-02-11</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!83, EG30</span></td>
  <td>57</td>
  <td><span class="no-break">2016-02-11</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-12-02</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!82, EG29</span></td>
  <td>56</td>
  <td><span class="no-break">2015-12-02</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-09-29</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!81, EG28</span></td>
  <td>55</td>
  <td><span class="no-break">2015-09-29</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-09-07</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!81, EG28</span></td>
  <td>54</td>
  <td><span class="no-break">2015-09-01</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-06-17</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!79, EG26</span></td>
  <td>53</td>
  <td><span class="no-break">2015-06-17</span></td>
  <td>InterProScan5 (5.8-49.0 20-November-2014)</td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-05-27</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-05-28</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!79, EG26</span></td>
  <td>52</td>
  <td><span class="no-break">2015-05-29</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-05-27</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-04-21</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!79, EG26</span></td>
  <td>51</td>
  <td><span class="no-break">2015-04-21</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-03-10</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-03-25</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!78, EG25</span></td>
  <td>50</td>
  <td><span class="no-break">2015-03-25</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-03-10</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-02-19</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!78, EG25</span></td>
  <td>49</td>
  <td><span class="no-break">2015-02-17</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-01-27</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!78, EG25</span></td>
  <td>48</td>
  <td><span class="no-break">2015-01-20</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2014-11-13</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!76, EG23</span></td>
  <td>47</td>
  <td><span class="no-break">2014-11-13</span></td>
  <td><span>InterProScan5 (5.7-48.0 August-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2014-09-16</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!76, EG23</span></td>
  <td>46</td>
  <td><span class="no-break">2014-09-15</span></td>
  <td><span>InterProScan5 (5.7-48.0 August-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2014-08-18</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!76, EG23</span></td>
  <td>45</td>
  <td><span class="no-break">2014-08-18</span></td>
  <td><span>InterProScan5 (5.7-48.0 August-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-07-18</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!75, EG22</span></td>
  <td>44</td>
  <td><span class="no-break">2014-07-18</span></td>
  <td><span>InterProScan5 (5.4-47.0 June-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-07-08</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!75, EG22</span></td>
  <td>43</td>
  <td><span class="no-break">2014-07-08</span></td>
  <td><span>InterProScan5 (5.4-47.0 June-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-05-15</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!75, EG22</span></td>
  <td>42</td>
  <td><span class="no-break">2014-05-13</span></td>
  <td><span>InterProScan5 (5.3-46.0 January-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-03-20</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!74, EG21</span></td>
  <td>41</td>
  <td><span class="no-break">2014-03-20</span></td>
  <td><span>InterProScan5 (5.3-46.0 January-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-02-20</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!74, EG21</span></td>
  <td>40</td>
  <td><span class="no-break">2014-02-20</span></td>
  <td><span>InterProScan5 (5.3-46.0 January-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-12-10</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!73, EG20</span></td>
  <td>39</td>
  <td><span class="no-break">2013-12-04</span></td>
  <td><span>InterProScan5 (<span class="no-break">2013-04-24</span>)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-10-22</span></td>
  <td>ASM294v2</td>
  <td>2.1&nbsp;</td>
  <td><span>e!73, EG20</span></td>
  <td>38</td>
  <td><span class="no-break">2013-11-11</span></td>
  <td><span>InterProScan5 (<span class="no-break">2013-04-24</span>)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-09-16</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!71, EG18</span></td>
  <td>37</td>
  <td><span class="no-break">2013-09-18</span></td>
  <td><span>InterProScan5 (<span class="no-break">2013-04-24</span>)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-06-21</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td>e!71, EG18</td>
  <td>36</td>
  <td><span class="no-break">2013-07-02</span></td>
  <td>InterProScan5 (<span class="no-break">2013-04-24</span>)</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-05-21</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!71, EG18</span>&nbsp;</td>
  <td>35</td>
  <td><span class="no-break">2013-06-06</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-05-07</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!71, EG18</span>&nbsp;</td>
  <td>34</td>
  <td><span class="no-break">2013-04-18</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-04-02</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td>e!70, EG17</td>
  <td>33</td>
  <td><span class="no-break">2013-03-07</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-02-26</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!70, EG17</span>&nbsp;</td>
  <td>31</td>
  <td><span class="no-break">2013-01-16</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2012-11-19</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!69, EG16</span>&nbsp;</td>
  <td>30</td>
  <td><span class="no-break">2012-11-27</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span>2012-10</span></td>
  <td><span>ASM294v2</span></td>
  <td>2.1</td>
  <td><span>e!69, EG16</span>&nbsp;</td>
  <td>29&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><span>2012-09</span></td>
  <td><span>ASM294v2</span></td>
  <td>2.1&nbsp;</td>
  <td><span>e!68, EG15</span>&nbsp;</td>
  <td>28&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 </tbody>
</table>
<p><strong>Brief explanations</strong> | | | |---------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------| | Item | Description | | INSDC | Accession of the current <em>S. pombe</em> sequence in the International Nucleotide Sequence Data Collaboration (ENA, GenBank, DDBJ) databases | | Gene Build | Version of gene feature annotation | | Ensembl Software Release | Versions of ensembl (e!) and Ensembl Genomes (EG) data and software used to build PomBase | | Annotation Version | Version of curated PomBase annotations (sequence features, ontology annotations, etc.) | | GO GAF | Date on which GO annotation data were retrieved from the GO repository | | InterPro | Date and version of InterProScan used to find matches to InterPro entries | | Manually Curated Ortholog table | Date and version of the manually curated sets of pombe/cerevisiae (and, where included, pombe/human) orthologs |</p>

  </div>
</div>
<div *ngIf="section == 'browse-curation'">
  <div class="docs-menu">
<h3 id="browse-curation"><a routerLink="/browse-curation" routerLinkActive="active">Browse curation</a></h3>
<ul>
<li><a routerLink="/browse-curation/fission-yeast-phenotype-ontology" routerLinkActive="active">Fission Yeast Phenotype Ontology</a></li>
<li><a routerLink="/browse-curation/fission-yeast-go-slim-terms" routerLinkActive="active">GO slim terms</a></li>
<li><a routerLink="/browse-curation/drugs-known-pombe-targets" routerLinkActive="active">Drugs with known targets</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'drugs-known-pombe-targets'" class="docs-content">
<h1 id="drugs-with-known-s.-pombe-targets">Drugs with known S. pombe targets</h1>
<p>This table lists drugs that affect <em>S. pombe</em>, and summarizes what is known about their targets or mechanisms of action.</p>
<table>
<thead>
<tr class="header">
<th>Drug</th>
<th>Targeted gene product, process, etc.</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1,​10-phenanthroline</td>
<td>inhibits RNA polymerases II and III</td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27518095">PMID:27518095</a></td>
</tr>
<tr class="even">
<td>CT2108A, CT2108B</td>
<td>fatty acid synthase</td>
<td> </td>
</tr>
<tr class="odd">
<td>alpha-amanitin</td>
<td>RNA polymerase II inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>alverine citrate</td>
<td>proteasome/lipid synthesis?</td>
<td> </td>
</tr>
<tr class="odd">
<td>amiloride</td>
<td>sodium transporter inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>amphotericin B</td>
<td>sterol biosynthesis inhibitor; binds ergosterol; forms membrane pores</td>
<td> </td>
</tr>
<tr class="odd">
<td>anisomycin</td>
<td>translation inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>antifungal azoles</td>
<td>sterol biosynthesis inhibitors</td>
<td> </td>
</tr>
<tr class="odd">
<td>arborcandin C</td>
<td>1,​3-beta-glucan synthase</td>
<td> </td>
</tr>
<tr class="even">
<td>benomyl</td>
<td>destabilizes microtubules</td>
<td> </td>
</tr>
<tr class="odd">
<td>bleomycins</td>
<td>fungal cell wall septation and cytokinesis inhibitors</td>
<td> </td>
</tr>
<tr class="even">
<td>brefeldin A</td>
<td>inhibits intracellular transport</td>
<td> </td>
</tr>
<tr class="odd">
<td>calcofluor white</td>
<td>cell wall synthesis</td>
<td> </td>
</tr>
<tr class="even">
<td>camptothecin</td>
<td>topoisomerase 1 (Top1) inhibitor</td>
<td> </td>
</tr>
<tr class="odd">
<td>canavanine</td>
<td>naturally occuring (alfafa) L-arginine antimetabolite</td>
<td> </td>
</tr>
<tr class="even">
<td>chloramphenicol</td>
<td>protein synthesis inhibitor</td>
<td> </td>
</tr>
<tr class="odd">
<td>compactin</td>
<td>ergosterol biosynthesis</td>
<td> </td>
</tr>
<tr class="even">
<td>Cutin-1</td>
<td>fatty acid synthase beta subunit (Fas1); inhibits nuclear envelope expansion during mitosis</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26869222">PMID:26869222</a></td>
</tr>
<tr class="odd">
<td>cycloheximide</td>
<td>translation inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>cyclosporin A</td>
<td>binds cyclophilin</td>
<td> </td>
</tr>
<tr class="odd">
<td>dihydromotuporamine C</td>
<td>proteasome/lipid synthesis?</td>
<td> </td>
</tr>
<tr class="even">
<td>echinocandin</td>
<td>cell wall synthesis</td>
<td> </td>
</tr>
<tr class="odd">
<td>fenpropimorph</td>
<td>proteasome/lipid synthesis?</td>
<td> </td>
</tr>
<tr class="even">
<td>hydroxyurea</td>
<td>inhibits ribonucleotide reductase; slows DNA synthesis; causes replication fork stalling</td>
<td> </td>
</tr>
<tr class="odd">
<td>hygromycin B</td>
<td>translation inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>isopropyl N-3-chlorophenyl carbamate</td>
<td>spindle poison</td>
<td> </td>
</tr>
<tr class="odd">
<td>itraconazole</td>
<td>spindle poison</td>
<td> </td>
</tr>
<tr class="even">
<td>latrunculin</td>
<td>inhibits actin polymerization</td>
<td> </td>
</tr>
<tr class="odd">
<td>leptomycin B</td>
<td>nuclear export receptor Crm1</td>
<td> </td>
</tr>
<tr class="even">
<td>lovastatin</td>
<td>competitive HMG CoA-reductase inhibitor</td>
<td> </td>
</tr>
<tr class="odd">
<td>methyl-2-benzimidazolecarbamate</td>
<td>reversible MT inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>monensin</td>
<td>intracellular transport</td>
<td> </td>
</tr>
<tr class="odd">
<td>natamycin</td>
<td>blocks fungal growth by binding specifically to ergosterol without permeabilizing the membrane</td>
<td> </td>
</tr>
<tr class="even">
<td>nystatin</td>
<td>ergosterol biosynthesis; antifungal; ergosta-5,​7,​22,​24,​(28)-tetraenol binding</td>
<td> </td>
</tr>
<tr class="odd">
<td>oxazolidinones</td>
<td>protein synthesis inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>phleomycin</td>
<td>causes DNA strand breaks</td>
<td> </td>
</tr>
<tr class="odd">
<td>pravastatin</td>
<td>HMG-CoA reductase inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>ricin</td>
<td>disrupts translation by inactivating the ribosome</td>
<td> </td>
</tr>
<tr class="odd">
<td>sanglifehrin A</td>
<td>binds cyclophilin</td>
<td> </td>
</tr>
<tr class="even">
<td>spliceostatin A</td>
<td>binds SF3b components and inhibits splicing</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17961508">PMID:17961508</a></td>
</tr>
<tr class="odd">
<td>staurosporine</td>
<td>protein kinase inhibitor</td>
<td> </td>
</tr>
<tr class="even">
<td>tagetitoxin/tagetin</td>
<td>RNA polymerase III inhibitor</td>
<td> </td>
</tr>
<tr class="odd">
<td>thialysine</td>
<td>naturally occuring lysine analogue</td>
<td> </td>
</tr>
<tr class="even">
<td>trichostatin A (TSA)</td>
<td>histone deacetylase inhibitor</td>
<td> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19723888">PMID:19723888</a></td>
</tr>
<tr class="odd">
<td>tunicamycin</td>
<td>inhibits N-glycosylation</td>
<td> </td>
</tr>
<tr class="even">
<td>zaragozic acid</td>
<td>inhibits N-glycosylation</td>
<td> </td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'fission-yeast-go-slim-terms'" class="docs-content">
<h1 id="fission-yeast-go-slim-terms">Fission Yeast GO slim terms</h1>
<p>The table below shows terms in the current fission yeast biological process GO slim, and the number of annotations to each term. GO IDs link to the QuickGO browser, where you can explore the ontology and annotations further. Icons beside each GO term link to <a href="http://www.esyn.org/">esyN</a>, which provides a graphical view of interactions for the genes from the PomBase High Confidence Physical Interaction Network (HCPIN) dataset. Only the subset of genes linked into the interaction network will be displayed in the esyN network view. The annotation totals link to pages with information about the term and a list of annotated genes.</p>
<p>Further information is available from:</p>
<ul>
<li><a routerLink="/documentation/pombase-go-slim-documentation" routerLinkActive="active">PomBase GO slim documentation</a></li>
<li><a routerLink="/documentation/high-confidence-physical-interaction-network" routerLinkActive="active">PomBase HCPIN documentation</a></li>
<li><a href="http://www.esyn.org/tutorial.html">esyN documentation</a></li>
<li><a routerLink="/browse-curation/fission-yeast-go-slimming-tips" routerLinkActive="active">PomBase tips for using GO slim terms and annotations</a></li>
<li><a href="http://www.geneontology.org/GO.slims.shtml">GO Slim Guide at the Gene Ontology website</a></li>
</ul>
<p>Note that both proteins and RNAs can be annotated to GO terms, and the tables on this page include annotated RNAs. For some GO terms, notably 'cytoplasmic translation', RNAs make up a significant proportion of the total annotated genes.</p>
<p><app-go-slim-table></app-go-slim-table></p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-go-slimming-tips'" class="docs-content">
<p><strong>General GO Slim hints</strong></p>
<p>These points will help you understand GO slims, and highlight some features of the fission yeast slim terms and annotations.</p>
<ol style="list-style-type: decimal">
<li>Note that the counts in the &quot;Genes&quot; column of the GO slim table are not additive, because many gene products are annotated to multiple terms.</li>
<li>It not possible to create a slim with no overlaps between terms. Although the <em>S. pombe</em> slim has been defined to include biologically informative terms, and minimise overlaps between terms, large overlaps cannot be entirely avoided. For example, most of the gene products annotated to <em>signal transduction</em> are also annotated other terms in the slim.</li>
<li>As a consequence of points 1 and 2, GO slim annotation summaries should not be presented using pie charts. Although a pie chart could show the fraction of total annotations for any slim term, it can too easily be mistaken for the fraction of total annotated gene products,​ which is not the same.</li>
<li>It is difficult to define a slim which includes all annotated gene products without including terms with very small numbers of annotations (for example, the cell aging branch has very few annotations), or very high level terms which are not particularly biologically informative (e.g. cellular process). Because we have opted not to include such terms, some gene products are annotated to process terms but do not appear in the slim annotation set.</li>
<li>Bear in mind that both proteins and RNAs can be annotated to GO terms. If you are working only with proteins you will need to make adjustments for this. For example, many tRNAs and rRNAs are annotated to cytoplasmic translation, and are therefore included in the slim set.</li>
<li>There is a difference between &quot;unknown&quot; and &quot;unannotated&quot;. All fission yeast and budding yeast gene products have been assessed and are classed as &quot;unknown&quot; for biological process if no biological process information is found (experimental or inferred). If you are making comparisons with other organisms, remember that it is possible that not all gene products have been assessed and that the &quot;unknown&quot; set is underestimated.</li>
<li>The default S. pombe slim includes all evidence codes for fission yeast. The evidence code <strong>IEA</strong> (inferred from electronic annotation) is often considered to be less accurate than other evidence codes, but it is very useful for increasing the coverage of some of the high level GO terms. Accurate annotation counts for some terms currently depends on including this evidence code (for example, there are 26 gene products annotated to transmembrane transport with IEA evidence, which are not yet covered by a manual annotation). For fission yeast, the IEA annotations improve slim coverage, but only represent a small number of annotations (654 biological process annotations as of September 2011), and have a low rate of false positives. We therefore recommend that you include them.</li>
<li>If you are making comparisons with budding yeast (or other organisms), you should consider excluding the evidence code <strong>RCA</strong>. This evidence code is used for functional predictions, and has a very high rate of false positives (for example, including RCA for budding yeast hugely will greatly and artificially inflate the number of annotations to translation).</li>
</ol>
<p><strong>Creating a user-defined slim</strong></p>
<ol style="list-style-type: decimal">
<li>You can create your own slim, or retrieve slim annotations for a gene set, using online slimming tools such as the <a href="http://go.princeton.edu/cgi-bin/GOTermMapper">GOTermMapper</a> at Princeton.</li>
<li>When creating a slim for the entire genome, you should try to ensure that it covers as many annotated genes in your set as possible (see #3 in list above). You should be aware of how many genes are annotated but not in your slim, and how many are &quot;unknown&quot; (i.e.,​ annotated only to the root node; see #5 in list above).</li>
<li>For display purposes you usually want to keep the number of terms as small as possible to convey your results. However, you should ensure that the terms you include are specific enough to capture biologically relevant information. Many terms (e.g. metabolic process (2915 annotations), cellular process (4083 annotations)) are too general for the purpose of most slim-based analyses.</li>
<li>On a related note, if you are using your slim for data analysis (e.g. to summarize an enrichment), you should ensure that the terms are specific enough to demonstrate their relevance to the biological topic of interest. For example, lumping all genes involved in transport my mask overrepresentation of transmembrane transport vs. underrepresentation of vesicle-mediated transport in your results set, so you need to ensure that the slim has categories to represent your results effectively.</li>
<li>Most current implementations of software to create &quot;GO slims&quot; include the <em>regulates</em> relationship by default, so that (for example) genes involved in <em>regulation of cytokinesis</em> will be included with the set of genes annotated to <em>cytokinesis</em>. See the <a href="http://www.geneontology.org/GO.ontology.relations.shtml">GO Ontology Relations documentation</a> for further information about relationships in GO. The annotation totals presented for the <a routerLink="/browse-curation/fission-yeast-go-slim-terms" routerLinkActive="active">PomBase default S. pombe slim</a>, by contrast,​ are calculated both explicitly including or excluding the genes which are involved in a process via regulation only. We expect this distinction to be available in future versions of slimming software.</li>
</ol>

  </div>
  <div *ngIf="pageName == 'fission-yeast-phenotype-ontology'" class="docs-content">
<h1 id="fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</h1>
<p>PomBase curators develop and use the Fission Yeast Phenotype Ontology (FYPO) to annotate phenotypes of mutant alleles and of over- or under-expressing wild type genes.</p>
<p>The contents and structure of FYPO are described on the <a href="http://curation.pombase.org/pombase-trac/wiki/FissionYeastPhenotypeOntology">FYPO page of the PomBase wiki</a> and in the publication:</p>
<p>Harris MA, Lock A, Bähler J, Oliver SG, Wood V. FYPO: The Fission Yeast Phenotype Ontology. Bioinformatics. 2013 July 1;29(13): 1671–1678. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23658422">PubMed abstract (PMID:23658422)</a> or <a href="http://bioinformatics.oxfordjournals.org/content/29/13/1671.long">Full text at Bioinformatics</a></p>
<h2 id="browsing-fypo-terms">Browsing FYPO terms</h2>
<p>At present, FYPO is included collections such as the National Center for Biomedical Ontology's <a href="http://bioportal.bioontology.org/">BioPortal</a> and EBI's <a href="http://www.ebi.ac.uk/ontology-lookup/">Ontology Lookup Service (OLS)</a>,​ both of which allow searching and browsing.</p>
<ul>
<li>BioPortal <a href="http://bioportal.bioontology.org/ontologies/FYPO">FYPO summary page</a></li>
<li>BioPortal <a href="http://bioportal.bioontology.org/ontologies/FYPO/?p=classes">FYPO terms</a> (the term display supports tree-viewer browsing, text searches, and a graphical display)</li>
<li><a href="http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=FYPO">OLS FYPO browser</a></li>
</ul>
<p>We hope to deploy a browser at PomBase in the not-too-distant future that will include annotations as well as terms.</p>
<h2 id="finding-fypo-terms-and-annotations-in-pombase">Finding FYPO terms and annotations in PomBase</h2>
<p>FYPO terms are displayed on PomBase gene pages, along with supporting evidence and allele and expression details, as described in the PomBase gene page documentation.</p>
<p>The PomBase advanced search can use either term names or IDs to search FYPO, and returns a list of genes that have alleles annotated to the specified term or any of its descendants. For example, see the <a href="http://www.pombase.org/faqs/can-i-get-list-essential-pombe-genes">FAQ on finding essential genes</a>.</p>
<h2 id="curating-phenotypes-with-fypo">Curating phenotypes with FYPO</h2>
<p>The PomBase online curation tool,​ <a href="http://curation.pombase.org/pombe">Canto</a>, uses FYPO terms for annotation of S. pombe phenotypes. <a href="http://curation.pombase.org/pombe/docs/fypo_annotation">Phenotype curation documentation</a> is available via Canto.</p>
<h2 id="submitting-bulk-data">Submitting Bulk Data</h2>
<p>If you have a large set of phenotype data to submit, you may want to do a bulk submission. See the documentation on the <a href="http://www.pombase.org/submit-data/phenotype-data-bulk-upload-format">recommended file format</a>,​ and use the Phenotype Data Submission Form.</p>

  </div>
  <div *ngIf="pageName == 'index'" class="docs-content">


  </div>
</div>
<div *ngIf="section == 'community'">
  <div class="docs-menu">
<h3 id="community"><a routerLink="/community" routerLinkActive="active">Community</a></h3>
<ul>
<li><a routerLink="/community/fission-yeast-community-curation-project" routerLinkActive="active">Community curation project</a></li>
<li><a routerLink="/community/fission-yeast-community-curation-pilot-project" routerLinkActive="active">Curation pilot project</a></li>
<li><a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">Mailing list</a></li>
<li><a routerLink="/community/internet-resources" routerLinkActive="active">External resources</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'fission-yeast-community-curation-pilot-project'" class="docs-content">
<h2 id="participating-groups-annotated-papers">Participating groups &amp; Annotated papers</h2>
<p>This table summarizes the results of the pilot project that preceded the <a routerLink="/community/fission-yeast-community-curation-project" routerLinkActive="active">Fission Yeast Community Curation Project</a>. The first phase of the project began in January 2009, and the second in November 2009. We thank all of the researchers who participated for making the pilot project a success.</p>
<table border="0" cellpadding="2" width="100%">
 <thead>
  <tr>
  <th><span>PMID</span></th>
  <th>Laboratory</th>
  <th>Submitter if different</th>
  <th>GO terms</th>
  <th>Interactions</th>
  <th>Phenotypes</th>
  <th>Other <span>curation</span></th>
  <th>Total</th>
  <th>Date</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td>PHASE 1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19101542">19101542</a></td>
  <td>Peter Folk</td>
  <td>Martin <span>Převorovský</span></td>
  <td>19</td>
  <td>0</td>
  <td>15</td>
  <td>3</td>
  <td>37</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19098712">19098712</a></td>
  <td>Juan Mata</td>
  <td>&nbsp;</td>
  <td>3</td>
  <td>7</td>
  <td>0</td>
  <td>1</td>
  <td>11</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19076239">19076239</a></td>
  <td>Paul Russell</td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>0</td>
  <td>2</td>
  <td>2</td>
  <td>9</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18675827">18675827</a></td>
  <td>Per <span>Sunnerhagen</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>1</td>
  <td>3</td>
  <td>0</td>
  <td>6</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17353264">17353264</a></td>
  <td>Chris <span>Norbury</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>2</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19057642">19057642</a></td>
  <td>Luis <span>Rokeach</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19056896">19056896</a></td>
  <td>Fred Winston</td>
  <td>Dom <span>Helmlinger</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19041767">19041767</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>John Burg</td>
  <td>4</td>
  <td>2</td>
  <td>2</td>
  <td>1</td>
  <td>9</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18556659">18556659</a></td>
  <td>Kevin <span>Hardwick</span></td>
  <td>&nbsp;</td>
  <td>16</td>
  <td>16</td>
  <td>2</td>
  <td>0</td>
  <td>34</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18430926">18430926</a></td>
  <td>Charlie Hoffman</td>
  <td>&nbsp;</td>
  <td>4</td>
  <td>0</td>
  <td>2</td>
  <td>0</td>
  <td>6</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17304215">17304215</a></td>
  <td><span>Hiroshi</span> <span>Iwasaki</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>5</td>
  <td>9</td>
  <td>19</td>
  <td>35</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18426916">18426916</a></td>
  <td><span>Hiro</span> <span>Yamano</span></td>
  <td>Michelle Tricky</td>
  <td>2</td>
  <td>1</td>
  <td>2</td>
  <td>1</td>
  <td>6</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19037101">19037101</a></td>
  <td>Nick <span>Rhind</span></td>
  <td>Nicholas Willis</td>
  <td>7</td>
  <td>6</td>
  <td>7</td>
  <td>0</td>
  <td>20</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19001497">19001497</a></td>
  <td>Ken <span>Sawin</span></td>
  <td>Lynda <span>Groocock</span></td>
  <td>9</td>
  <td>6</td>
  <td>2</td>
  <td>1</td>
  <td>18</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18368917">18368917</a></td>
  <td>Steve Aves</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18799626">18799626</a></td>
  <td>Takashi <span>Toda</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>3</td>
  <td>3</td>
  <td>0</td>
  <td>8</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19109429">19109429</a></td>
  <td>Stephen <span>Kearsey</span></td>
  <td>&nbsp;</td>
  <td>9</td>
  <td>2</td>
  <td>2</td>
  <td>0</td>
  <td>13</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17688408">17688408</a></td>
  <td>Susan <span>Forsburg</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17409062">17409062</a></td>
  <td>Iain Hagan</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19139281">19139281</a></td>
  <td>Tony Carr</td>
  <td>Edgar <span>Hartsuiker</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-15</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18065650">18065650</a></td>
  <td>Per <span>Sunnerhagen</span></td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>2</td>
  <td>4</td>
  <td>0</td>
  <td>11</td>
  <td>-</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18406331">18406331</a></td>
  <td><span>Kaz</span> <span>Shiozaki</span></td>
  <td>&nbsp;</td>
  <td>4</td>
  <td>4</td>
  <td>1</td>
  <td>1</td>
  <td>10</td>
  <td><span class="no-break">2009-01-15</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19139265">19139265</a></td>
  <td>Kathy Gould</td>
  <td>Rachel Roberts</td>
  <td>8</td>
  <td>11</td>
  <td>1</td>
  <td>1</td>
  <td>21</td>
  <td><span class="no-break">2009-01-17</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19197239">19197239</a></td>
  <td>Benoit <span>Arcangioli</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>4</td>
  <td>0</td>
  <td>6</td>
  <td><span class="no-break">2009-02-10</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19328067">19328067</a></td>
  <td>Rob Fisher</td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>4</td>
  <td>2</td>
  <td>25</td>
  <td>36</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19330768">19330768</a></td>
  <td><span>Katya</span> <span>Grishchuk</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19217404">19217404</a></td>
  <td>Robin <span>Allshire</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19217403">19217403</a></td>
  <td>Paul Russell</td>
  <td>Jessica Williams</td>
  <td>5</td>
  <td>7</td>
  <td>3</td>
  <td>0</td>
  <td>15</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19362535">19362535</a></td>
  <td>Janet Partridge</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19250904">19250904</a></td>
  <td><span>Songtao</span> <span>Jia</span></td>
  <td><span>Bahrat</span> Reddy</td>
  <td>4</td>
  <td>2</td>
  <td>2</td>
  <td>2</td>
  <td>10</td>
  <td><span class="no-break">2009-03-01</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19373772">19373772</a></td>
  <td>Jeremy <span>Hyams</span></td>
  <td>Isabelle <span>Jourdain</span></td>
  <td>5</td>
  <td>1</td>
  <td>6</td>
  <td>0</td>
  <td>12</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19299465">19299465</a></td>
  <td>Sara Mole</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19299465">19299465</a></td>
  <td>Julie Cooper</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19286980">19286980</a></td>
  <td><span>Stevan</span> Marcus</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19033386">19033386</a></td>
  <td>Iain Hagan</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19351719">19351719</a></td>
  <td>Luis <span>Rokeach</span></td>
  <td><span>Pascale</span> Beauregard</td>
  <td>2</td>
  <td>1</td>
  <td>1</td>
  <td>0</td>
  <td>4</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19160458">19160458</a></td>
  <td>Ken <span>Sawin</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19211838">19211838</a></td>
  <td>Nick <span>Rhind</span></td>
  <td>Mary Porter-Goff</td>
  <td>2</td>
  <td>2</td>
  <td>8</td>
  <td>0</td>
  <td>12</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19273851">19273851</a></td>
  <td>Ramsay <span>Macfarlane</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>0</td>
  <td>1</td>
  <td>0</td>
  <td>2</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19243310">19243310</a></td>
  <td>Takashi <span>Toda</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>4</td>
  <td>1</td>
  <td>0</td>
  <td>7</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19158663">19158663</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>1</td>
  <td>1</td>
  <td>0</td>
  <td>4</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19237545">19237545</a></td>
  <td><span>Henar</span> <span>Valdivieso</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>1</td>
  <td>3</td>
  <td>0</td>
  <td>5</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18562692">18562692</a></td>
  <td>Jonathan Miller</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16143612">16143612</a></td>
  <td>Charlie Hoffman</td>
  <td>&nbsp;</td>
  <td>0</td>
  <td>8</td>
  <td>0</td>
  <td>0</td>
  <td>8</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15448137">15448137</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>15</td>
  <td>0</td>
  <td>0</td>
  <td>5</td>
  <td>20</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18375981">18375981</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>1</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9391101">9391101</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>2</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7518718">7518718</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>2</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7958849">7958849</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>7</td>
  <td>6</td>
  <td>0</td>
  <td>0</td>
  <td>13</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19542312">19542312</a></td>
  <td>John Armstrong</td>
  <td>James <span>Dodgeson</span></td>
  <td>18</td>
  <td>0</td>
  <td>16</td>
  <td>0</td>
  <td>34</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>PHASE 2</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19620282">19620282</a></td>
  <td>Simon Whitehall</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19643199">19643199</a></td>
  <td><span>Jeramy</span> <span>Hyams</span></td>
  <td>Isabelle <span>Jourdain</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19436749">19436749</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19627505">19627505</a></td>
  <td><span>Henar</span> <span>Valdivieso</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>4</td>
  <td>5</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19664060">19664060</a></td>
  <td>Stuart <span>Macneill</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19686603">19686603</a></td>
  <td>Stuart <span>Macneill</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19666000">19666000</a></td>
  <td><span>Murakami</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19646873">19646873</a></td>
  <td><span>Fulvia</span> Verde</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19714215">19714215</a></td>
  <td><span>Jurg</span> <span>Bahler</span></td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>4</td>
  <td>5</td>
  <td>2</td>
  <td>16</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18923417">18923417</a></td>
  <td>Nick <span>Boddy</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18667531">18667531</a></td>
  <td>Nick <span>Boddy</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19686339">19686339</a></td>
  <td>Luis <span>Rokeach</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17114925">17114925</a></td>
  <td><span>Danesh</span> <span>Moazed</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19707600">19707600</a></td>
  <td>Felicity Watts</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19394293">19394293</a></td>
  <td><span>Danesh</span> <span>Moazed</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18574244">18574244</a></td>
  <td>Peter <span>Baumann</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19520858">19520858</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>&nbsp;</td>
  <td>9</td>
  <td>1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>10</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18503029">18503029</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>1</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19427212">19427212</a></td>
  <td>Anna <span>Paoletti</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18223116">18223116</a></td>
  <td>Simon <span>Labbe</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19915076">19915076</a></td>
  <td>Simon <span>Labbe</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19571115">19571115</a></td>
  <td>Shelly <span>Sazer</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18272791">18272791</a></td>
  <td>Rosa <span>Aligue</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19838064">19838064</a></td>
  <td>Jan <span>Paluh</span></td>
  <td>&nbsp;</td>
  <td>3</td>
  <td>29</td>
  <td>9</td>
  <td>4</td>
  <td>55</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19696784">19696784</a></td>
  <td>Takashi <span>Toda</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>3</td>
  <td>2</td>
  <td>&nbsp;</td>
  <td>7</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17409354">17409354</a></td>
  <td>Elena Hidalgo</td>
  <td>&nbsp;</td>
  <td>14</td>
  <td>7</td>
  <td>3</td>
  <td>2</td>
  <td>26</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19416828">19416828</a></td>
  <td>Jacob <span>Dalgaard</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19473263">19473263</a></td>
  <td>Mathias <span>Sipiczki</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19836238">19836238</a></td>
  <td>Caroline Wilkinson</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18252721">18252721</a></td>
  <td>Caroline Wilkinson</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17986863">17986863</a></td>
  <td>Erik <span>Boye</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>3</td>
  <td>0</td>
  <td>5</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=20531409">20531409</a></td>
  <td>Juan Mata</td>
  <td>&nbsp;</td>
  <td>3</td>
  <td>90</td>
  <td>0</td>
  <td>0</td>
  <td>93</td>
  <td>N/A</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19720063">19720063</a></td>
  <td><span>Nimisha</span> Sharma</td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>3</td>
  <td>3</td>
  <td>2</td>
  <td>10</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19625445">19625445</a></td>
  <td>Jose <span>Cansado</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19486165">19486165</a></td>
  <td><span>Takayoshi</span> <span>Kuno</span></td>
  <td><span>Yue</span> Fang</td>
  <td>2</td>
  <td>10</td>
  <td>4</td>
  <td>0</td>
  <td>16</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19713940">19713940</a></td>
  <td><span>Kentaro</span> <span>Nakano</span></td>
  <td><span>Masakatsu</span> <span>Takaine</span></td>
  <td>4</td>
  <td>2</td>
  <td>2</td>
  <td>0</td>
  <td>8</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19409973">19409973</a></td>
  <td>Kurt <span>Runge</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 </tbody>
</table>

  </div>
  <div *ngIf="pageName == 'fission-yeast-community-curation-project'" class="docs-content">
<h1 id="the-fission-yeast-community-curation-project">The Fission Yeast Community Curation Project</h1>
<p>Comprehensive manual curation of the S. pombe literature is a primary goal of the PomBase project, and among the most important activities of the curation staff. Increasing numbers of new publications, and a large backlog of older papers, however, mean that complete literature curation will require input from the research community as well as the efforts of dedicated professional curators.</p>
<p>The PomBase Community Curation system enables researchers to contribute annotations directly to PomBase based on their publications using Canto, a web-based tool that allows both curators and researchers to create annotations. Canto supports GO, phenotype,​ interaction, and modification annotations, and can be configured for use with other ontologies as the need arises. Annotations made in the community curation system will be prioritized for inclusion in PomBase and will therefore also be more rapidly disseminated to other databases (e.g. GenBank/ENA/DDBJ, UniProtKB, BioGRID and GO), making data from annotated papers widely visible.</p>
<p><strong>How to contribute</strong>: For newly published papers, PomBase curators email authors inviting them to participate, with specific instructions and links. Lab members are also welcome to evaluate existing annotations and create new annotations from past publications — use the PubMed ID search on the <a href="https://curation.pombase.org/pombe">main PomBase Canto page</a>. Anyone can also try the <a href="https://curation.pombase.org/demo">demo version of Canto</a>.</p>
<p><strong>Progress and attribution</strong>: <a href="https://curation.pombase.org/pombe/stats/annotation">Curation statistics</a> track S. pombe papers curated in Canto by PomBase and community curators, and the annotations obtained. Canto will soon allow contributors to register <a href="http://orcid.org/">ORCID</a> identifiers in the system, which will support more formalised attribution of community contributions to PomBase, and thereby enable researchers to cite community curation in institutional reports, funding applications, etc. To supplement the existing permanent publication-centred links in Canto, we are currently investigating possible ways to link to external resources such as ORCID or OpenRIF via ORCID IDs.</p>
<p>For a bit of historical perspective, you can browse the results of the <a routerLink="/community/fission-yeast-community-curation-pilot-project" routerLinkActive="active">Community Curation Pilot Project</a> that took place in 2009.</p>

  </div>
  <div *ngIf="pageName == 'index'" class="docs-content">
<h1 id="fission-yeast-community">Fission Yeast Community</h1>
<ul>
<li><a routerLink="/community/fission-yeast-community-curation-project" routerLinkActive="active">Community curation project</a></li>
<li><a routerLink="/community/fission-yeast-community-curation-pilot-project" routerLinkActive="active">Curation pilot project</a></li>
<li><a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">Mailing list</a></li>
<li><a routerLink="/community/internet-resources" routerLinkActive="active">External resources</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'internet-resources'" class="docs-content">
<h2 id="s.-pombe-community-websites"><em>S. pombe</em> Community Websites</h2>
<ul>
<li><a href="http://www-bcf.usc.edu/~forsburg/pombeweb.html">PombeWeb</a> maintained by Susan Forsburg
<ul>
<li><a href="http://www-bcf.usc.edu/~forsburg/pombeweb.html#people">Fission yeast researchers</a> at PombeWeb</li>
</ul></li>
<li><a href="http://www-bcf.usc.edu/~forsburg/main.html">PombeNet</a> maintained by Susan Forsburg
<ul>
<li><a href="http://www-bcf.usc.edu/~forsburg/main.html">About pombe</a> at PombeNet</li>
<li><a href="http://www-bcf.usc.edu/~forsburg/plasmids.html">Working with pombe</a> at PombeNet</li>
</ul></li>
<li><a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/erss.cgi?rss_guid=1ToPSyjDNLLW2R_UIOHFTxnRdlJouyMeQX4YfwMFXtO8d87M4r">PubMed RSS Feed</a> Feed for current <em>S. pombe</em> related publications.</li>
</ul>
<h2 id="ontologies">Ontologies</h2>
<ul>
<li><a href="http://www.geneontology.org/">The Gene Ontology Consortium</a> controlled vocabulary</li>
<li><a href="http://amigo.geneontology.org/">AmiGO</a> Search and browse GO annotation</li>
<li>The <a href="http://www.ebi.ac.uk/quickgo/">QuickGO</a> browser at EBI</li>
<li><a href="http://www.yeastgenome.org/help/function-help/gene-ontology-go">SGD GO documentation</a> to help SGD users become familiar with GO</li>
<li><a href="http://vortex.cs.wayne.edu/projects.htm#Onto-Express">Onto-Express</a> find significant GO terms in lists of genes</li>
<li><a href="http://go.princeton.edu/cgi-bin/GOTermFinder">GOTermFinder</a> find significant GO terms in lists of genes</li>
<li><a href="http://bahlerweb.cs.ucl.ac.uk/cgi-bin/GLA/GLA_input">AnGeLi: Analysis of Gene Lists</a> (UCL) uses GO, phenotypes, interactions, etc.</li>
<li><a href="http://www.sequenceontology.org/">Sequence Ontology</a></li>
</ul>
<h2 id="interactions-pathways-and-networks">Interactions, Pathways and Networks</h2>
<ul>
<li><a href="http://www.thebiogrid.org/">BioGRID</a> Genetic and physical interactions</li>
<li><a href="http://www.ebi.ac.uk/intact/pages/interactions/interactions.xhtml?query=species:4896">Interactions (IntAct)</a> Known interactions for <em>S. pombe</em> from the IntAct database</li>
<li><a href="http://reactome.org/cgi-bin/eventbrowser?DB=gk_current&FOCUS_SPECIES=Schizosaccharomyces%20pombe&ID=68324&">Reactome pathway data</a> <em>S. pombe</em> entry point.</li>
<li><a href="http://www.esyn.org/index.php">esyN</a> Web-based tool for building,​ sharing, and viewing network data</li>
<li><a href="http://string-db.org/">STRING</a> Protein–protein interaction network database</li>
</ul>
<h2 id="sequence-analysis-tools">Sequence Analysis Tools</h2>
<h3 id="general-tools">General tools</h3>
<ul>
<li><a href="http://gbrowse.org/wiki/index.php/Schizosaccharomyces_pombe">GBrowse Wiki</a> for <em>S. pombe</em></li>
<li><a href="http://www.sanger.ac.uk/science/tools/artemis">Artemis</a> Sequence viewer and analysis tool</li>
<li><a href="http://www.cbs.dtu.dk/services/GenomeAtlas-2.0/show-kingdom.php?kingdom=Eukaryotes&KLquickSearch=pombe">Genome Atlas</a> DNA structural atlas</li>
<li><a href="http://www.ncbi.nlm.nih.gov/mapview/map_search.cgi?chr=spombe.inf">Map viewer</a> at the NCBI</li>
<li><a href="http://megasun.bch.umontreal.ca/People/lang/species/spo/spombe.html">Mitochondrial Genome Information</a></li>
</ul>
<h3 id="sequence-features-protein-families-orthologues">Sequence Features, Protein Families, Orthologues</h3>
<ul>
<li>Todd Lowe's <em>S. pombe</em> genomic <a href="http://gtrnadb.ucsc.edu/GtRNAdb2/genomes/eukaryota/Schi_pomb_972h/">tRNA analysis</a></li>
<li><a href="http://wardlab.cbs.umn.edu/pombe/"><em>S. pombe</em> membrane protein library</a></li>
<li><a href="http://www.rsat.eu/">RSAT</a> Regulatory sequence analysis tools (includes <a href="http://rsat-tagc.univ-mrs.fr/rsat/">Fungi</a> server)</li>
<li><a href="http://pfam.xfam.org/">Pfam</a> protein families (search or browse)</li>
<li><a href="http://supfam.org/SUPERFAMILY/cgi-bin/gen_list.cgi?genome=iz">SCOP</a> Superfamily assignments</li>
<li><a href="https://rostlab.org/owiki/index.php/PredictNLS">PredictNLS</a> Nuclear localization signal prediction</li>
<li><a href="http://www.bahlerlab.info/YOGY/">YOGY</a> A resource for retrieving orthologous proteins</li>
<li><a href="http://epd.vital-it.ch/S_pombe/S_pombe_database.php">EPD</a> Eukaryotic Promoter Database, <em>S. pombe</em> section</li>
</ul>
<h2 id="gene-expression">Gene Expression</h2>
<ul>
<li><a href="http://www.bahlerlab.info/cgi-bin/SPGE/geexview">Gene Expression Viewer</a> at UCL</li>
<li><a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/buildDetails?atlas_build_id=355"><em>S. pombe</em> PeptideAtlas</a> compendium of peptides identified in tandem mass spectrometry proteomics experiments</li>
</ul>
<h2 id="experimental-tools">Experimental tools</h2>
<ul>
<li><a href="http://www.massgeneral.org/cancer/research/motamediresources.aspx">PRIMED</a> Primers for deleting and tagging yeast genes (<em>S. pombe</em> and <em>S. cerevisiae</em>; downloadable spreadsheets)</li>
</ul>
<h3 id="strains-and-constructs">Strains and Constructs</h3>
<ul>
<li><a href="http://yeast.lab.nig.ac.jp/nig/index_en.html">Japanese National Bio Resource Project</a></li>
<li><a href="http://www.riken.jp/SPD/index.html">Postgenome database service</a> Chemical Genetics Laboratory, Riken</li>
<li><a href="http://dna.brc.riken.jp/en/yoshidayeast_en.html">Orfeome clones</a> Riken Bioresource Centre</li>
<li><a href="http://www.addgene.org">Addgene</a> Collects and distributes plasmids</li>
</ul>
<h3 id="experimental-protocols">Experimental protocols</h3>
<ul>
<li>The <a href="http://www-bcf.usc.edu/~forsburg/plasmids.html">Forsburg lab</a> technical information e.g. <a href="http://www-bcf.usc.edu/~forsburg/Protocols.html">Protocols</a></li>
<li><a href="http://www.dieter-wolf-lab.org/protocols/">Wolf Lab Protocols</a></li>
</ul>
<h2 id="s.-cerevisiae-resources"><em>S. cerevisiae</em> Resources</h2>
<ul>
<li><a href="http://www.yeastgenome.org/">SGD</a> <em>Saccharomyces</em> Genome Database</li>
<li><a href="http://www.sanger.ac.uk/research/projects/genomeinformatics/sgrp.html"><em>Saccharomyces</em> genome resequencing</a> at Sanger</li>
</ul>

  </div>
</div>
<div *ngIf="section == 'documentation'">
  <div class="docs-menu">
<h3 id="documentation"><a routerLink="/documentation" routerLinkActive="active">Documentation</a></h3>
<ul>
<li>Gene pages
<ul>
<li><a routerLink="/documentation/gene-page-basic-information" routerLinkActive="active">Basic Information</a></li>
<li><a routerLink="/documentation/gene-page-gene-ontology" routerLinkActive="active">Gene Ontology</a></li>
<li><a routerLink="/documentation/gene-page-phenotypes" routerLinkActive="active">Phenotypes</a></li>
<li><a routerLink="/documentation/gene-page-target" routerLinkActive="active">Target Of</a></li>
<li><a routerLink="/documentation/gene-page-protein-features" routerLinkActive="active">Protein Features</a></li>
<li><a routerLink="/documentation/gene-page-modifications" routerLinkActive="active">Modifications</a></li>
<li><a routerLink="/documentation/gene-page-gene-expression" routerLinkActive="active">Gene Expression</a></li>
<li><a routerLink="/documentation/taxonomic-conservation" routerLinkActive="active">Species Distribution</a></li>
<li><a routerLink="/documentation/orthologs" routerLinkActive="active">Ortholog Curation</a></li>
<li><a routerLink="/documentation/genetic-and-physical-interactions" routerLinkActive="active">Genetic and Physical Interactions</a></li>
<li><a routerLink="/documentation/gene-page-external-references" routerLinkActive="active">External References</a></li>
<li><a routerLink="/documentation/gene-page-literature" routerLinkActive="active">Literature</a></li>
</ul></li>
<li><a routerLink="/documentation/simple-search-documentation" routerLinkActive="active">Simple Search</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'gene-page-basic-information'" class="docs-content">
<h1 id="gene-page-basic-information">Gene page basic information</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>At the top of each gene page is a set of basic information about the gene:</p>
<div class="figure">
<img src="assets/basic_info_gene_page.png" title="Gene page basic information" alt="top of gene page" width="800" height="548" />
<p class="caption">top of gene page</p>
</div>
<ol style="list-style-type: decimal">
<li>See the <a routerLink="//submit-data/gene-name-registry" routerLinkActive="active">Gene Name Registry</a> for more information about <em>S. pombe</em> gene names. Synonyms include all names other than the standard name that have been published for a gene, and thus are not guaranteed to be unique.</li>
<li>Free text description of the gene product</li>
<li>Chromosome coordinates for the gene. For protein-coding genes, the first set of numbers indicate the gene start, end, and length including the default UTRs chosen by PomBase curators (see <a href="http://www.pombase.org/faqs/how-do-you-determine-genes-full-length-transcript-utr-coordinates-transcription-start-and-end-s">this FAQ</a> for more information). The &quot;CDS&quot; numbers give the start, end, and length excluding UTRs but including introns (other than introns located entirely within UTRs).</li>
<li>Category describing how well-studied a gene is; see the <a href="/status/gene-characterisation">Gene Characterisation</a> page for details</li>
<li>Indicates whether a gene encodes protein, specifies a non-coding RNA, or is a pseudogene</li>
<li>What the three-letter acronym stands for; where available, name descriptions are provided for synonyms as well as primary names.</li>
<li>The size of the mature gene product. For protein-coding genes, the length (number of amino acid residues) and molecular weight are shown.</li>
<li>Static graphic from the PomBase Genome Browser showing the region around the gene</li>
<li>Links to gene pages for nearby genes</li>
<li>Link to the PomBase Genome Browser, centred on the location of the gene</li>
</ol>

  </div>
  <div *ngIf="pageName == 'gene-page-external-references'" class="docs-content">
<h1 id="external-references">External references</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>The External References section of a gene page provides links to information on a gene or its product in each of several external databases. For each database, the name and a brief description are shown, and the identifier used by the database is displayed and linked.</p>
<p>The resources linked in this section cover a wide range of databases and data tyeps. Because all are external, PomBase staff cannot provide assistance with any of the resources other than problems with the actual links.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-gene-expression'" class="docs-content">
<p>Gene Page: Gene Expression</p>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>The Gene Expression section of a gene page displays curated qualitative and quantitative information about the level and timing of the gene's expression.</p>
<p><br />
<img src="assets/gene_page_expression.png" title="Gene Page: Gene Expression" alt="gene page expression section" width="672" height="600" /></p>
<ol style="list-style-type: decimal">
<li>Qualitative descriptions are drawn from a small internal controlled vocabulary, with supporting extensions. Each description links to a list of all genes annotated to the same term.
<ul>
<li>a) What was measured, i.e. RNA or protein.</li>
<li>b) A qualitative description of the expression level (one of: present,​ absent, increased, decreased, unchanged, constant, fluctuates).</li>
<li>c) An optional extension indicating that expression is observed during a particular process or phase (using GO terms) or in the presence of a particular chemical (using ChEBI terms).</li>
<li>d) A brief descriptor for the type of evidence that supports the annotation. The evidence categories come from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO).</li>
<li>e) The paper from which the annotation comes.</li>
</ul></li>
<li>Quantitative data include the number of RNA or protein molecules measured per cell, and may optionally include a copy number range. Quantitative expression annotations also indicate whether the experiment measures expression in a population or a single cell, and usually include a brief description of experimental conditions,​ drawn from a small ontology maintained by PomBase curators.
<ul>
<li>f) Molecules per cell (RNA or protein, as indicated in the sub-header). For population-scale experiment types this is an average, whereas for single-cell scale experiments it is an absolute number.</li>
<li>g) The life cycle stage or cell cycle phase in which expression was measured.</li>
<li>h) Information about experimental conditions, such as temperature, type of medium used, etc. Descriptions come from a small ontology maintained by PomBase curators.</li>
<li>i) Whether the experiment was done at the level of a single cell or a population of cells.</li>
<li>j) A brief descriptor for the type of evidence that supports the annotation. The evidence categories come from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO). If a more specific description is not available, the general term &quot;experimental evidence&quot; is shown.</li>
<li>k) The paper from which the annotation comes.</li>
</ul></li>
</ol>
<p><strong>Important:</strong> Note that the condition descriptions are rather broad,​ and therefore do not necessarily capture all details of every aspect that may affect gene expression. We are also unable to capture strain details at present. We therefore recommend consulting the papers cited for the data before comparing or combining data from different publications.</p>
<p>Several of the <a href="/documentation/gene-page-external-references">External References</a> also link to gene expression data.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-gene-ontology'" class="docs-content">
<h1 id="gene-page-gene-ontology">Gene Page: Gene Ontology</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>PomBase uses the <a href="http://www.geneontology.org/">Gene Ontology</a> (GO) to describe the biological context of genes.</p>
<p>GO consists of three distinct <a href="http://www.geneontology.org/GO.contents.doc.shtml#ontology">ontologies</a> (or sets of vocabularies) that describe a gene’s:</p>
<ol style="list-style-type: decimal">
<li>Molecular Function (MF) — single-step activities such as <em>kinase activity</em> or <em>transporter activity</em></li>
<li>Biological Process (BP) — a series of functions with a common goal such as <em>MAPK cascade</em> or <em>cytokinesis</em></li>
<li>Cellular component (CC) — the location where the gene product can be found, and what complexes it is part of, for instance <em>nucleus</em> or <em>fatty acid synthase complex</em></li>
</ol>
<p>A gene product may be <a href="http://www.geneontology.org/GO.contents.doc.shtml#annotation">annotated</a> to several GO terms from each of the three ontologies; <a routerLink="//spombe/result/SPBC19C7.03" routerLinkActive="active">cyr1</a>, for instance, is annotated to adenylate cyclase activity, magnesium ion binding and protein binding (MFs), it acts in cAMP biosynthesis and positively regulates protein import into nucleus (BPs), and is found in the plasma membrane (CC).</p>
<p>The GO annotation display on PomBase gene pages includes ontology term details and supporting data. The Summary view shows just the essentials; each unique combination of gene, GO term, and extension(s) is shown once:</p>
<div class="figure">
<img src="assets/go_gene_page_summary.png" alt="summary gene page GO annotations" width="784" height="600" />
<p class="caption">summary gene page GO annotations</p>
</div>
<p><strong><br />
</strong></p>
<ol style="list-style-type: decimal">
<li>The GO term name, which links to a page with additional information,​ including the term definition, any synonyms, <a href="#go_str">relationships to other GO terms</a>, and a list of genes annotated to the term.</li>
<li>Link to a page with term details and a list of genes annotated to the term and its is_a and part_of descendants (see the GO documentation on <a href="http://www.geneontology.org/GO.ontology.structure.shtml">Ontology Structure</a> and <a href="http://www.geneontology.org/GO.ontology.relations.shtml">Ontology Relations</a> for more information).</li>
<li>GO annotations may have <a routerLink="/#ann_ext" routerLinkActive="active">extensions</a> to capture any of several types of additional detail. <em>S. pombe</em> genes link to PomBase gene pages.</li>
<li><a routerLink="//browse-curation/fission-yeast-go-slim-terms" routerLinkActive="active">GO Slim</a> terms applicable to the gene.</li>
</ol>
<p>The Full view shows more details, including separate entries for repeat determinations of a given gene/term/extension combination (if supported by more than one line of evidence and/or reported in more than one paper):</p>
<div class="figure">
<img src="assets/go_gene_page_full.png" alt="full gene page GO annotations" width="781" height="600" />
<p class="caption">full gene page GO annotations</p>
</div>
<p><strong><br />
</strong></p>
<ol style="list-style-type: decimal">
<li>The unique <a routerLink="/#id" routerLinkActive="active">ID and name</a> for a GO term. The ID links to a page with additional information, including the term definition, any synonyms, <a routerLink="/#go_str" routerLinkActive="active">relationships to other GO terms</a>, and a list of genes annotated to the term.</li>
<li>An abbreviation (code) for the type of <a routerLink="/#evid" routerLinkActive="active">evidence</a> that supports the annotation. The evidence categories come from the set of <a href="http://www.geneontology.org/GO.evidence.shtml">evidence codes</a> defined by the GO Consortium.</li>
<li>An additional identifier that provides supporting details for annotations using certain evidence codes (see below and <a href="http://www.geneontology.org/GO.format.gaf-2_0.shtml#with_from">GO documentation</a>).</li>
<li>The paper from which the annotation comes.</li>
<li>Link to a page with term details and a list of genes annotated to the term and its is_a and part_of descendants (see the GO documentation on <a href="http://www.geneontology.org/GO.ontology.structure.shtml">Ontology Structure</a> and <a href="http://www.geneontology.org/GO.ontology.relations.shtml">Ontology Relations</a> for more information).</li>
<li>GO annotations may have <a routerLink="/#ann_ext" routerLinkActive="active">extensions</a> to capture any of several types of additional detail. <em>S. pombe</em> genes link to PomBase gene pages.</li>
<li><a routerLink="//browse-curation/fission-yeast-go-slim-terms" routerLinkActive="active">GO Slim</a> terms applicable to the gene.</li>
</ol>
<p><strong>Term ID and name</strong></p>
<p>Annotations are made to the definition of a term, not the term name itself, so we recommend users always read the term definition. The definition of a term can be found by clicking the term ID. This page will also supply a list of all genes annotated to the specific term.</p>
<p><span id="go_str"></span><strong>GO Structure</strong></p>
<p>GO is structured in a hierarchal order with less specific terms being parents of more specific child terms. A child term may have multiple parents and multiple children; the BP <em>mitosis</em>, for instance, has child terms representing each phase of mitosis, as well as parent terms connecting it to both a cell cycle phase and nuclear division. Crucially, whenever an annotation is made to a term, the gene product is automatically annotated to all the parent terms. The ancestry of a term can be viewed on the page linked to the term ID. For more information,​ see the <a href="http://www.geneontology.org/GO.ontology.structure.shtml">GO Ontology Structure</a> documentation.</p>
<p>Multiple relationships exist to describe the links within the ontologies. A child term can have an <em>is_a</em> relation to the parent term, where the child is a more specific type of the parent, or a <em>part_of</em> relationship where the child makes up a part of the parent. For instance, the mitochondrion <em>is_a</em> intracellular organelle and is part_of a cell. Additionally, GO also include regulatory relationships. For more information on relationships in GO, see the <a href="http://www.geneontology.org/GO.ontology.relations.shtml">GO Ontology Relations</a> documentation.</p>
<p><strong><span id="evid"></span>Evidence codes</strong></p>
<p>In PomBase, every annotation is supported by a reference that states where the annotation comes from, and an evidence code that describes the type of data that supports the annotation. An annotation may be inferred from experimental ‘wet lab’ data, backed by a literature reference and citing experimental evidence such as <a href="http://www.geneontology.org/GO.evidence.shtml#ida">IDA</a> (Inferred from Direct Assay) or <a href="http://www.geneontology.org/GO.evidence.shtml#imp">IMP</a> (Inferred from Mutant Phenotype). Further information on evidence codes is available in the <a href="http://www.geneontology.org/GO.evidence.shtml">GO Evidence Codes</a> documentation.</p>
<p>Another source of annotations come from computational methods. Please note that all computational annotations are based on predictions. In cases where a sequence model has been used to annotate genes, but the genes annotated based on the model have not been manually checked, the <a href="http://www.geneontology.org/GO.evidence.shtml#iea">IEA</a> (Inferred from Electronic Annotation) evidence code is assigned. If the annotations have been manually checked other evidence codes may be used, for instance <a href="http://www.geneontology.org/GO.evidence.shtml#iso">ISO</a> (Inferred from Sequence Orthology) or <a href="http://www.geneontology.org/GO.evidence.shtml#ism">ISM</a> (Inferred from Sequence Model). PomBase uses ISO to cross-reference to the roles of known <em>S. cerevisiae</em> genes, and uses ISM when domains present in a gene product can give clues to its biological role.</p>
<p>For some types of evidence, such as sequence comparisons or interaction data, it is important to note what gene or gene product was used in the comparison or detected in the interaction. In these cases the <a href="http://www.geneontology.org/GO.format.gaf-2_0.shtml#with_from">With/From column</a> provides more information regarding the source of the information.</p>
<p><strong><span id="ann_ext"></span>Annotation extensions</strong></p>
<p>Where available, annotation extensions are displayed underneath the GO term name. The extensions provide additional specificity to the annotation by linking the term to another ontology term or a gene product via a relationship. Examples include specifying substrates of molecular functions or specifying the cellular localization during a process (for instance, <a routerLink="//spombe/result/SPBC106.10" routerLinkActive="active">pka1</a> has protein serine/threonine kinase activity and has the substrates <a routerLink="//spombe/result/SPBC119.04" routerLinkActive="active">mei3</a> and <a routerLink="//spombe/result/SPAC6F12.02" routerLinkActive="active">rst2</a>. It is a cellular component of the nucleus during nitrogen starvation, but part of the cytoplasm during glucose starvation).</p>
<p>The GO Consortium provides further information on annotation extensions in its <a href="http://geneontology.org/page/annotation-extension">annotation documentation</a>,​ including the <a href="http://www.geneontology.org/GO.format.gaf-2_0.shtml#annotation_ext">file format guide</a>,​ on a <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">wiki page</a>, and in a <a href="http://www.biomedcentral.com/1471-2105/15/155/abstract">publication</a>. PomBase converts many extension names to more human-friendly text, as described <a routerLink="//documentation/gene-page-annotation-extension-relation-display" routerLinkActive="active">here</a>.</p>
<p>If an extension mentions another <em>S. pombe</em> gene, the extension data will also be displayed as an annotation in the &quot;<a href="/documentation/gene-page-target">Target Of</a>&quot; section of the page for that gene.</p>
<p><strong>Finding other genes annotated to a specific GO term</strong></p>
<p>From a gene page, all <em>S. pombe</em>genes annotated to a term (or its children) can be found by clicking on a term ID.</p>
<p>Additionally, the <a routerLink="//spombe/query/builder" routerLinkActive="active">advanced search</a> (or ‘query builder’) can be used to search for all genes annotated to a particular GO term (see the <a href="/documentation/advanced-search-documentation">advanced search documentation</a> for more information). To find annotations to specific GO terms in organisms other than <em>S. pombe</em> we recommend using <a href="http://amigo.geneontology.org">AmiGO</a> or <a href="http://www.ebi.ac.uk/QuickGO">QuickGO</a>.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-literature'" class="docs-content">
<h1 id="gene-page-literature">Gene page literature</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>The Literature section of a gene page lists the papers that have curated data for the gene or its product.</p>
<p>At the top of the section, there are links to search PubMed or UK PMC for any papers on the gene. This may retrieve papers not on the gene page Literature list, if there are papers not yet curated (or spurious matches to the search criteria).</p>
<p>For each paper, the full citation is shown, and links to the paper's record in PubMed and Europe PMC are provided. At present, the list is sorted alphabetically by first author.</p>
<p>If you reach the Literature section by clicking a reference link elsewhere on the gene page, the selected reference will be highlighted by a coloured background.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-modifications'" class="docs-content">
<h1 id="modifications">Modifications</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>The Modifications section lists protein modifications that have been manually curated for protein-coding genes. PomBase will add RNA modifications to this section in the future, when relevant data are curated.</p>
<p>The Protein Modifications table shows ontology annotations, using <a href="http://www.psidev.info/MOD">PSI-MOD</a> for protein modifications (as shown in the example), and the <a href="http://sequenceontology.org/">Sequence Ontology</a> (SO) for certain protein sequence features.</p>
<p><strong>Ontology Annotations for Protein Features</strong></p>
<p>Summary view:</p>
<div class="figure">
<img src="assets/gene_page_modification_summary.png" title="Protein modifications" alt="summary gene page modification ontology annotation" width="800" height="413" />
<p class="caption">summary gene page modification ontology annotation</p>
</div>
<ol style="list-style-type: decimal">
<li>Ontology (PSI-MOD or SO)</li>
<li>Name of the ontology term</li>
<li>Modification annotations may have <a routerLink="/#ann_ext" routerLinkActive="active">extensions</a> to capture any of several types of additional detail. <em>S. pombe</em> genes link to PomBase gene pages, and GO term names link to PomBase ontology term summary pages.</li>
<li>The residue(s) modified (where available)</li>
<li>The total number of genes annotated to this term or any of its descendants, linked to a list of those genes.</li>
</ol>
<p>Full view:</p>
<div class="figure">
<img src="assets/gene_page_modification_full.png" title="Protein modifications" alt="full gene page modification ontology annotation" width="800" height="559" />
<p class="caption">full gene page modification ontology annotation</p>
</div>
<ol style="list-style-type: decimal">
<li>Ontology (PSI-MOD or SO)</li>
<li>Name and ID of the ontology term</li>
<li>Modification annotations may have <a routerLink="/#ann_ext" routerLinkActive="active">extensions</a> to capture any of several types of additional detail. <em>S. pombe</em> genes link to PomBase gene pages, and GO term names link to PomBase ontology term summary pages.</li>
<li>An abbreviation (code) for the type of evidence that supports the annotation. Modification annotations use a subset of the <a href="http://www.geneontology.org/GO.evidence.shtml">evidence codes</a> defined by the GO Consortium.</li>
<li>The residue(s) modified (where available)</li>
<li>The published source of the annotation (where available)</li>
<li>The total number of genes annotated to this term or any of its descendants, linked to a list of those genes.</li>
</ol>
<p><strong><span id="ann_ext"></span>Annotation extensions</strong><br />
Where available, annotation extensions are displayed underneath the ontology term name. The extensions provide additional specificity to the annotation, often by linking the term to another ontology term or a gene product via a relationship. Examples include specifying the gene product that adds or removes a modification, specifying modified residues, or specifying that the modification is observed during a phase or process (for instance, Cdc2 is phosphorylated during G2, and phosphorylated on Tyrosine 15 by Wee1; it is phosphorylated during G2 but not M phase of the mitotic cell cycle; other extensions are also available for Cdc2).</p>
<p>The annotation extension field can also be used to indicate modification site occupancy, for experiments that measure the proportion of copies of the protein (or RNA) that have the modification.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-phenotypes'" class="docs-content">
<h1 id="gene-page-phenotypes">Gene Page: Phenotypes</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>PomBase defines a phenotype as an observable characteristic, or set of characteristics, of an organism that results from the interaction of its genotype with a given environment. In PomBase, phenotypes are annotated using terms from the Fission Yeast Phenotype Ontology (FYPO). FYPO uses several existing ontologies from the <a href="http://obofoundry.org/">Open Biological and Biomedical Ontologies</a> (OBO) collection to construct formal definitions. Basic documentation for FYPO is available at the <a href="http://obofoundry.org/cgi-bin/detail.cgi?id=fypo">OBO Foundry</a>, and further information is available on the <a href="http://curation.pombase.org/pombase-trac/wiki/FissionYeastPhenotypeOntology">PomBase wiki</a>.</p>
<p>In the phenotype annotation display on PomBase gene pages, the first item shown is a brief summary indicating whether cells with a null (deletion) allele of the gene are viable or inviable, or either depending on experimental conditions. Next, single-allele phenotypes are shown in two tables. The first table lists phenotypes observed at the population level, such as viability in culture, and the second shows cell-level phenotypes. Note that the viable/inviable population terms describe whether a gene is essential or not, and see the wiki <a href="http://curation.pombase.org/pombase-trac/wiki/FYPOContentStructure">FYPO Content and Structure</a> documentation for more information on cell and population phenotypes. Finally, two more tables list population-level and cell-level multi-allele phenotypes, i.e. phenotypes associated with double mutants,​ triple mutants, etc, and the relevant genotype details.</p>
<p>Each table includes ontology term details and supporting data. The Summary view shows just the essentials; each unique combination of gene,​ FYPO term, allele(s) and extension(s) is shown once. For single-allele phenotypes, the display includes:</p>
<div class="figure">
<img src="assets/single_fypo_gene_page_summary.png" title="Single-allele FYPO summary on gene page" alt="gene page phenotype annotations" width="800" height="417" />
<p class="caption">gene page phenotype annotations</p>
</div>
<ol style="list-style-type: decimal">
<li>The FYPO term name, which links to a page with additional information, including the term definition, any synonyms,​ relationships to other FYPO terms, and a list of genes annotated to the term.</li>
<li>Allele details, including a name (if one is used in the literature) and description (where known). The column is headed &quot;Genotypes&quot; for consistency between the single- and multi-allele displays. Mouse over the allele name to show the allele type, which indicates whether partial deletions or altered residues refer to amino acids or nucleotides. If you can provide a description for any allele shown as &quot;unknown&quot;, please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a>.</li>
<li>Link to a page with term details and a list of genes annotated to the term and its <em>is_a</em> and <em>part_of</em> descendants (the GO documentation on <a href="http://www.geneontology.org/GO.ontology.structure.shtml">Ontology Structure</a> and <a href="http://www.geneontology.org/GO.ontology.relations.shtml">Ontology Relations</a> has some useful information on this topic).</li>
<li>Phenotype annotations may have extensions to capture expressivity (the extent to which a phenotype is expressed; for PomBase annotations expressivity extensions are often used to capture information about phenotype severity) or penetrance (proportion of a population that shows the phenotype), or to document which gene or protein used in an assay for level, localisation, etc. <em>S. pombe</em> genes link to PomBase gene pages. Expressivity and penetrance use the relations <em>has_expressivity</em> and <em>has_penetrance</em> respectively, and can have values such as &quot;high&quot;, &quot;medium&quot;, or &quot;low&quot; (numerical values may be added in the future if and when the need arises). A gene or gene product used in an assay is stored using the appropriate PomBase systematic ID and the relation <em>assayed_using</em>; the relation is converted to <em>affecting</em> in the gene page display. If a mutation affects an activity that modifies another gene product, extensions may capture the affected enzyme, the affected substrate, or both. The relation <em>assayed_enzyme</em> is displayed as <em>affecting activity of</em> on gene pages, and <em>assayed_substrate</em> is shown as <em>affecting substrate</em>.</li>
</ol>
<p>The Full single-allele phenotype view shows more details, including separate entries for repeat determinations of a given gene/term/allele/extension combination (if supported by more than one line of evidence and/or reported in more than one paper):</p>
<div class="figure">
<img src="assets/single_fypo_gene_page_full.png" title="Single-allele FYPO full view on gene page" alt="gene page phenotype annotations" width="800" height="476" />
<p class="caption">gene page phenotype annotations</p>
</div>
<ol style="list-style-type: decimal">
<li>The unique ID and name for a term in the phenotype ontology. The ID links to a page with additional information, including the term definition, any synonyms, relationships to other FYPO terms, and a list of genes annotated to the term.</li>
<li>A brief descriptor for the type of evidence that supports the annotation. The evidence categories come from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO). The evidence text may be nested under the term name to save space.</li>
<li>Allele details, including a name (if one is used in the literature) and description (where known). Mouse over the allele name to show the allele type, which indicates whether partial deletions or altered residues refer to amino acids or nucleotides. If you can provide a description for any allele shown as &quot;unknown&quot;, please <a routerLink="/mailto:helpdesk@pombase.org" routerLinkActive="active">contact the PomBase curators</a>.</li>
<li>Where available, the expression level of the allele in the experiment is shown here. Both allele and expression details are subheadings under &quot;Genotype&quot;, again for consistency.</li>
<li>Information about experimental conditions, such as temperature, type of medium used, etc. Descriptions come from a small ontology maintained by PomBase curators.</li>
<li>The paper from which the annotation comes.</li>
<li>Link to a page with term details and a list of genes annotated to the term and its <em>is_a</em> and <em>part_of</em> descendants (the GO documentation on <a href="http://www.geneontology.org/GO.ontology.structure.shtml">Ontology Structure</a> and <a href="http://www.geneontology.org/GO.ontology.relations.shtml">Ontology Relations</a> has some useful information on this topic).</li>
<li>Phenotype annotations may have extensions to capture expressivity (the extent to which a phenotype is expressed; for PomBase annotations expressivity extensions are often used to capture information about phenotype severity) or penetrance (proportion of a population that shows the phenotype), or to document which gene or protein used in an assay for level, localisation, etc. <em>S. pombe</em> genes link to PomBase gene pages. Expressivity and penetrance use the relations <em>has_expressivity</em> and <em>has_penetrance</em> respectively, and can have values such as &quot;high&quot;, &quot;medium&quot;, or &quot;low&quot; (numerical values may be added in the future if and when the need arises). A gene or gene product used in an assay is stored using the appropriate PomBase systematic ID and the relation <em>assayed_using</em>; the relation is converted to <em>affecting</em> in the gene page display. If a mutation affects an activity that modifies another gene product, extensions may capture the affected enzyme, the affected substrate, or both. The relation <em>assayed_enzyme</em> is displayed as <em>affecting activity of</em> on gene pages, and <em>assayed_substrate</em> is shown as <em>affecting substrate</em>.</li>
</ol>
<p>Similar displays are used for multi-allele phenotypes. The &quot;Count&quot; column is omitted to avoid complications arising from having more than one gene connected with almost every multi-allele phenotype.</p>
<p>The multi-allele Summary shows FYPO terms and genotype descriptions:</p>
<div class="figure">
<img src="assets/multi_fypo_gene_page_summary-800x180.png" alt="multi-allele FYPO summary view on gene page" width="800" height="180" />
<p class="caption">multi-allele FYPO summary view on gene page</p>
</div>
<ol style="list-style-type: decimal">
<li>The FYPO term name, which links to a page with additional information, including the term definition, any synonyms,​ relationships to other FYPO terms, and a list of genes annotated to the term.</li>
<li>Genotype details, including a name (if one is used in the literature) and descriptions for the relevant alleles (where known). Mouse over any allele name to show the allele type.</li>
</ol>
<p>The Full view includes more details:</p>
<div class="figure">
<img src="assets/multi_fypo_gene_page_full-800x292.png" alt="multi-allele FYPO full view on gene page" width="800" height="292" />
<p class="caption">multi-allele FYPO full view on gene page</p>
</div>
<ol style="list-style-type: decimal">
<li>The unique ID and name for a term in the phenotype ontology. The ID links to a page with additional information, including the term definition, any synonyms, relationships to other FYPO terms, and a list of genes annotated to the term.</li>
<li>A brief descriptor for the type of evidence that supports the annotation. The evidence categories come from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO). The evidence text may be nested under the term name to save space.</li>
<li>Each gene included in the genotype description is listed on one line per allele present (some genotypes include two or more different alleles of the same gene), and linked to its gene page.</li>
<li>Allele descriptions are shown essentially as in the single-allele display, including the mouse-over that shows the allele type.</li>
<li>Where available, the expression level of the allele in cells with the genotype is shown here. Gene, allele and expression details are subheadings under &quot;Genotype&quot;.</li>
<li>Information about experimental conditions, such as temperature, type of medium used, etc.</li>
<li>The paper from which the annotation comes.</li>
</ol>
<p>Annotation extensions are also included in the multi-allele Summary and Full views (although none are included for the annotations in the above illustrations), using the same display as for single-allele phenotypes.</p>
<p>Also see the <a href="/documentation/advanced-search-documentation">Advanced Search documentation</a> for information on finding genes annotated to phenotype terms.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-protein-features'" class="docs-content">
<h1 id="gene-page-protein-features">Gene page: protein features</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>Gene pages for protein-coding genes have a section describing protein features. Also see the <a routerLink="//documentation/gene-page-modifications" routerLinkActive="active">Modifications</a> documentation.</p>
<div class="figure">
<img src="assets/gene_page_protein_features.png" title="Protein features" alt="gene page protein features" width="595" height="600" />
<p class="caption">gene page protein features</p>
</div>
<ol style="list-style-type: decimal">
<li>The graphical view is a static graphic from the PomBase Genome Browser, and shows the positions of domains and other features. Below the graphic is a link to the Genome Browser's protein view.</li>
<li>The table of protein families and domains is described in more detail <a routerLink="//documentation/gene-page-protein-features#families" routerLinkActive="active">below</a>.</li>
<li>This section provides manual curation that ensures that large protein families (e.g. the WD family) include all known members. These manual annotations are provided because many families in protein family databases are rebuilt during the release cycle to include new sequences, and as a result gene products are sometimes lost or gained from protein families, and false negatives are common. This section is also used to capture published subfamily members which may be collected into a single family in protein family databases.</li>
<li>Any motifs or features annotated using Sequence Ontology (SO) terms.</li>
<li>Table of the protein's physical properties</li>
</ol>
<p><strong><span id="families"></span>Protein Families and Domains</strong></p>
<div class="figure">
<img src="assets/gene_page_protein_domains.png" title="Protein families and domains" alt="gene page protein domains" width="800" height="276" />
<p class="caption">gene page protein domains</p>
</div>
<ol style="list-style-type: lower-alpha">
<li>Feature ID of this family or domain in the originating database. Where feasible, the ID links to the source database. Databases include Pfam, SMART, Prosite, Gene3D, SUPERFAMILY, TMHMM and Panther.</li>
<li>Source database name</li>
<li>ID of the <a href="http://www.ebi.ac.uk/interpro/">InterPro</a> entry that includes the feature. InterPro classifies proteins into families using predictive models provided by several different databases that make up the InterPro consortium. InterPro is useful for assessing the species distribution of a particular family or domains.</li>
<li>The description of the domain from the contributing database</li>
<li>Start and end coordinates, showing the location of the feature within the protein</li>
<li>Link to a list of other fission yeast proteins which are members of the family or have the domain</li>
<li>Link to the Pfam entry for the protein, which shows domain organization</li>
<li>Table of manually curated data indicating membership in a protein family or sub-family. Includes an ID and name for an internal PomBase ontology term, a reference (where available), and a link to other genes annotated to the same family.</li>
</ol>
<p>Note that some of the listed features (e.g. transmembrane domains from TMHMM) are predictions. Consult the contributing databases for further information, or <a routerLink="//feedback" routerLinkActive="active">contact the PomBase curators</a> if you notice any problems with annotated or predicted features.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-target'" class="docs-content">
<h1 id="gene-page-target-of">Gene Page: Target Of</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>The Target Of section describes genes that affect the gene of interest. Genes listed under &quot;Target Of&quot; may include upstream regulators or enzymes that modify the product of the gene of interest. For example,​ the &quot;Target Of&quot; annotations for <a routerLink="//spombe/result/SPBC11B10.09#recipext" routerLinkActive="active">cdc2</a> indicate that it is a substrate of, and regulated by, the kinase Wee1 and the phosphatase Cdc25 (among others).</p>
<p>Target Of data are derived by taking the reciprocal of <a href="http://www.pombase.org/faqs/what-annotation-extension">ontology annotation extensions</a> (using <a href="documentation/gene-page-gene-ontology#ann_ext">GO annotation extensions</a> at present; soon to be extended to include <a routerLink="//documentation/gene-page-phenotypes" routerLinkActive="active">phenotype</a> annotation extensions). The table includes:</p>
<ul>
<li>Relationship: indicates how the genes are connected. The relationship is the reciprocal of the relation used in the originating annotation extension, as noted in the <a href="/documentation/gene-page-annotation-extension-relation-display">Annotation Extension Relation Display documentation</a>.</li>
<li>Gene: the gene whose product affects the gene of interest.</li>
<li>Product: the targeting gene's product description.</li>
<li>Reference (hidden in &quot;summary&quot; view): the source of the annotation.</li>
</ul>

  </div>
  <div *ngIf="pageName == 'genetic-and-physical-interactions'" class="docs-content">
<h1 id="genetic-and-physical-interactions">Genetic and Physical Interactions</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>The Interactions section of a gene page displays genetic and physical interactions for a gene (or its product), in <a href="http://thebiogrid.org/">BioGRID</a> format. All interactions are curated manually by PomBase or BioGRID curators.</p>
<p>Each table has five columns:</p>
<p>The first column (with no header) describes the interaction. The text reflects the type of evidence listed in the Evidence column and, where applicable, the directionality of the interaction.</p>
<p><strong>Gene:</strong> The gene that interacts with the gene of interest, linked to its gene page</p>
<p><strong>Product:</strong> The description of the interacting gene's product, from its gene page</p>
<p><strong>Evidence:</strong> The type of genetic interaction observed, or the type of experiment performed to detect a physical interaction. The evidence categories come from BioGRID, and are described on their <a href="http://wiki.thebiogrid.org/doku.php/experimental_systems">Experimental Evidence Codes</a> documentation page.</p>
<p><strong>Reference:</strong> The paper cited to support the interaction</p>

  </div>
  <div *ngIf="pageName == 'high-confidence-physical-interaction-network'" class="docs-content">
<h1 id="high-confidence-physical-interaction-network-hcpin">High Confidence Physical Interaction Network (HCPIN)</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>The PomBase High Confidence Physical Interaction Network (HCPIN) is a collection of physical interaction data based on three subsets Gene Ontology (GO) annotations:</p>
<ol style="list-style-type: decimal">
<li>Manually curated protein binding annotations. PomBase curators annotate genes to the GO &quot;protein binding&quot; molecular function term (GO:0005515) when proteins are shown experimentally to interact in a pairwise, and presumably direct, manner, often by more than one method</li>
<li>Targets or substrates of biochemical activity (e.g. protein kinases or phosphatases, protein methyltransferases, ubiquitin-protein ligases). When one protein is shown experimentally to modify another, a GO molecular function annotation represents the activity,​ and the target(s) can be captured using <a href="/documentation/gene-page-gene-ontology#ann_ext">annotation extensions</a>. For example, <a routerLink="//spombe/result/SPBC11B10.09" routerLinkActive="active">Cdc2</a> phosphorylates <a routerLink="//spombe/result/SPBC146.03c" routerLinkActive="active">Cut3</a>, so it is annotated to 'protein serine/threonine kinase activity' with the extension 'has substrate cut3'.</li>
<li>GO protein complex annotations. The HCPIN set includes both annotations supported directly by fission yeast experiments and those inferred from an experimentally characterised ortholog.</li>
</ol>
<p>The data can be downloaded in tab-delimited format from the <a href="/downloads/datasets">Download Datasets</a> page.</p>

  </div>
  <div *ngIf="pageName == 'index'" class="docs-content">
<h1 id="pombase-web-site-help-documentation">PomBase Web Site Help &amp; Documentation</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p><strong><span id="DocsGenePage"></span>Gene Pages</strong></p>
<ul>
<li><a routerLink="/documentation/gene-page-basic-information" routerLinkActive="active">Basic Information</a></li>
<li><a routerLink="/documentation/gene-page-gene-ontology" routerLinkActive="active">Gene Ontology</a></li>
<li><a routerLink="/documentation/gene-page-phenotypes" routerLinkActive="active">Phenotypes</a></li>
<li><a routerLink="/documentation/gene-page-target" routerLinkActive="active">Target Of</a></li>
<li><a routerLink="/documentation/gene-page-transcript" routerLinkActive="active">Transcript</a></li>
<li><a routerLink="/documentation/gene-page-protein-features" routerLinkActive="active">Protein Features</a></li>
<li><a routerLink="/documentation/gene-page-modifications" routerLinkActive="active">Modifications</a></li>
<li><a routerLink="/documentation/gene-page-sequence" routerLinkActive="active">Sequence</a></li>
<li><a routerLink="/documentation/gene-page-gene-expression" routerLinkActive="active">Gene Expression</a></li>
<li><a routerLink="/documentation/taxonomic-conservation" routerLinkActive="active">Taxonomic Conservation</a></li>
<li><a routerLink="/documentation/orthologs" routerLinkActive="active">Orthologs</a></li>
<li><a routerLink="/documentation/genetic-and-physical-interactions" routerLinkActive="active">Interactions</a></li>
<li><a href="documentation/gene-page-external-references">External References</a></li>
<li><a routerLink="/documentation/gene-page-literature" routerLinkActive="active">Literature</a></li>
</ul>
<p><strong><span id="DocsSearch"></span>Searching</strong></p>
<ul>
<li><a routerLink="/documentation/simple-search-documentation" routerLinkActive="active">Simple Search</a></li>
<li><a routerLink="/documentation/advanced-search-documentation" routerLinkActive="active">Advanced Search</a></li>
</ul>
<p><strong>Other PomBase Pages</strong></p>
<ul>
<li>GO Slim <a routerLink="/documentation/pombase-go-slim-documentation" routerLinkActive="active">Documentation</a> and <a routerLink="/browse-curation/fission-yeast-go-slimming-tips" routerLinkActive="active">Usage Tips</a></li>
<li><a routerLink="/browse-curation/fission-yeast-phenotype-ontology" routerLinkActive="active">Fission Yeast Phenotype Ontology (FYPO) Page</a></li>
<li><a routerLink="/documentation/high-confidence-physical-interaction-network" routerLinkActive="active">High Confidence Physical Interaction Network (HCPIN)</a></li>
<li><a routerLink="/about/version-history" routerLinkActive="active">Data versions</a></li>
<li><a routerLink="/documentation/ontology-term-page" routerLinkActive="active">Ontology Term Pages</a></li>
</ul>
<p><strong>How to Cite PomBase</strong></p>
<ul>
<li>See the <a routerLink="/about/citing-pombase" routerLinkActive="active">Citing PomBase</a> page</li>
</ul>
<p>****File Formats for Data Submission****</p>
<p>PomBase accepts batch submissions of certain types of data that appear on PomBase gene pages. For these data types, we use dedicated PomBase-specific formats:</p>
<ul>
<li><a routerLink="/submit-data/phenotype-data-bulk-upload-format" routerLinkActive="active">Phenotypes</a></li>
<li><a routerLink="/submit-data/modification-bulk-upload-file-format" routerLinkActive="active">Modifications</a></li>
<li><a routerLink="/submit-data/qualitative-gene-expression-bulk-upload-file-format" routerLinkActive="active">Qualitative gene expression</a></li>
<li><a routerLink="/submit-data/quantitative-gene-expression-bulk-upload-file-format" routerLinkActive="active">Quantitative gene expression</a></li>
</ul>
<p>For data that can be connected with sequence features or coordinates,​ and displayed as tracks in the genome browser, see the <a href="faqs/what-file-formats-can-i-use-submit-high-throughput-data">data format FAQ</a> and further details linked there.</p>
<p><strong>Linking to and from PomBase</strong></p>
<ul>
<li>Linking to PomBase: To link to any PomBase gene page, use the systematic ID for the gene in a URL with the syntax http://www.pombase.org/gene/[systematic ID]. For example,​ <a routerLink="//gene/SPBC11B10.09" routerLinkActive="active">http://www.pombase.org/gene/SPBC11B10.09</a> links to the gene page for <a routerLink="/spombe/result/SPBC11B10.09" routerLinkActive="active">cdc2</a>.</li>
<li>Linking from PomBase to external resources: We can provide links from PomBase gene pages to gene- or gene product-specific <em>S. pombe</em> data for any resource that uses URLs with PomBase systematic IDs . Please contact the PomBase Curators for more information.</li>
</ul>
<h2 id="comparative-genomics">Comparative Genomics</h2>
<p>We plan to add specific documentation about comparative genomics using <em>S. pombe</em> here. In the meantime, you can look at the documentation for <a href="http://genomebrowser.pombase.org/info/docs/compara/index.html">Ensembl Compara</a>.</p>
<h2 id="more-information-...">More information ...</h2>
<p>Additional PomBase documentation and help will be available here soon. In the meantime, you can <app-contact-email class="contact-email" [linkName]="'contact the curators.'"
                   [subject]="'Comment or problem report ...'"></app-contact-email></p>

  </div>
  <div *ngIf="pageName == 'ontology-term-page'" class="docs-content">
<h1 id="ontology-term-pages">Ontology term pages</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>For each ontology term loaded into PomBase, a page summarizes essential details about the term, and lists genes annotated to it or its descendants:</p>
<div class="figure">
<img src="assets/ontology_term_page.png" alt="ontology term page" width="800" height="488" />
<p class="caption">ontology term page</p>
</div>
<ol style="list-style-type: decimal">
<li>Ontology term details. The term's name and, where applicable, ID,​ definition and synonyms are shown.</li>
<li>Links to external browsers are available for some ontologies. GO and FYPO terms have links to <a href="http://bioportal.bioontology.org/">BioPortal</a>, and GO term pages also link to <a href="http://amigo.geneontology.org/amigo">AmiGO</a> and <a href="http://www.ebi.ac.uk/QuickGO/">QuickGO</a>. SO IDs link to <a href="http://sequenceontology.org/browser/obob.cgi">MISO</a>.</li>
<li>Link to the Advanced Search with the gene list (see 4 below) loaded into the Gene ID filter.</li>
<li>List of genes annotated to the term or its descendants.</li>
</ol>
<p>Ontology term pages are available for <a href="http://geneontology.org/">GO</a>,​ <a routerLink="//browse-curation/fission-yeast-phenotype-ontology" routerLinkActive="active">FYPO</a>,​ <a href="http://sequenceontology.org/">SO</a>,​ <a href="http://www.psidev.info/MOD">PSI-MOD</a> and <a href="http://dev.pombase.org/faqs/what-pbo-option-advanced-search">PBO</a>. The internal ontology we use for phenotype experimental conditions, PECO,​ has minimal term pages with term names and IDs.</p>

  </div>
  <div *ngIf="pageName == 'orthologs'" class="docs-content">
<h1 id="ortholog-curation">Ortholog curation</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>The &quot;Manually curated orthologous groups&quot; section of a gene page lists any orthologous genes in budding yeast (<em>Saccharomyces cerevisiae</em>) or human (<em>Homo sapiens</em>) that have been assessed and recorded manually by curators. Budding yeast entries link to the <a href="http://www.yeastgenome.org/"><em>Saccharomyces</em> Genome Database</a> (SGD), and human genes link to <a href="http://www.ensembl.org/Homo_sapiens/">Ensembl</a> and <a href="http://www.uniprot.org/">UniProtKB</a>; the text descriptions come from these databases.</p>
<p>Ortholog prediction: The source of the ortholog prediction may vary. In some cases the consensus ortholog from the major ortholog predictors (<a href="http://www.ensembl.org/info/docs/compara/index.html">Compara</a>,​ <a href="http://inparanoid.sbc.su.se/">Inparanoid</a>,​ <a href="http://www.orthomcl.org/">OrthoMCL</a>) is used. Many distant orthologs have also been identified by <span class="st">PSI-BLAST</span> matches that are uniformly one-to one across species; these alignments have been submitted to the <a href="http://pfam.sanger.ac.uk/">Pfam</a> protein family database. Other ortholog predictions come from experimental data demonstrating functional correspondence or involving membership of corresponding complexes. These predictions are also aligned and submitted to Pfam before inclusion. PomBase's approach ensures that the breadth of coverage is greater than any individual prediction method, and includes many ortholog calls which are not detected by any automated method. Gradually, we will add and display supporting references for all orthology calls.</p>
<p>For species other than budding yeast and human, we provide links to Compara to display orthologs from fungi or from all species. for further information, see the <a href="http://www.pombase.org/faqs/how-can-i-find-s-pombe-orthologs-species-other-human-and-s-cerevisiae">FAQ on orthologs in additional species</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-go-slim-documentation'" class="docs-content">
<h1 id="go-slims---gene-ontology-subsets">GO Slims - Gene Ontology subsets</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>&quot;GO slims&quot; are subsets of the Gene Ontology (GO) that provide a broad overview of annotation distribution. Slims can offer a useful overview of a genome or the results of a large-scale experiment. For more information on GO slims, please see PomBase <a href="/browse-curation/fission-yeast-go-slimming-tips">GO Slimming Tips</a> and the <a href="http://www.geneontology.org/GO.slims.shtml">GO Slim Guide</a> at the Gene Ontology website.</p>
<p>The <a routerLink="//browse-curation/fission-yeast-go-slim-terms" routerLinkActive="active">GO slim term set</a> constructed for fission yeast provides almost complete coverage of gene products annotated to biological process terms. This slim set is a good starting point for users who wish to identify terms of biological interest to create a &quot;custom slim&quot;, or to become familiar with the genome contents.</p>
<p>In the GO slim <a routerLink="//browse-curation/fission-yeast-go-slim-terms" routerLinkActive="active">table</a>, GO IDs link to the QuickGO browser, where you can explore the ontology and annotations further. The annotation totals include annotations to the slim term and to descendants following the <em>is_a</em>, <em>part_of,​ regulates, positively_regulates,​</em>and <em>negatively_regulates</em> relationships, and link to gene lists.</p>

  </div>
  <div *ngIf="pageName == 'simple-search-documentation'" class="docs-content">
<h1 id="simple-search">Simple search</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>The simple search is available via the search box in the header on every PomBase page.</p>
<p>It searches gene names (standard names and synonyms). systematic IDs. and gene product descriptions for fission yeast genes.</p>

  </div>
  <div *ngIf="pageName == 'taxonomic-conservation'" class="docs-content">
<h1 id="taxonomic-conservation">Taxonomic Conservation</h1>
<div style="color: red">
<p>Note: because of changes in this preview, many of the screenshots in this documentation are out of date.</p>
</div>
<p>The Taxonomic Conservation section provides classifiers of taxon distribution (at the domain/kingdom level) of the product of a protein-coding gene, which are assigned manually. The classifiers come from a small controlled vocabulary maintained by PomBase curators. All applicable terms are assigned to a protein.</p>
<p><strong>Species Distribution terms</strong></p>
<p>Species Distribution terms</p>
<ul>
<li>These terms identify the taxonomic groups in which gene product ortholog(s) have been identified:
<ul>
<li>conserved in archaea</li>
<li>conserved in bacteria</li>
<li>conserved in eukaryotes</li>
<li>conserved in fungi</li>
<li>conserved in metazoa</li>
<li>conserved in vertebrates</li>
</ul></li>
<li>These two terms represent taxon restrictions, and indicate that a gene product has not been detected outside these taxa:
<ul>
<li>conserved in fungi only</li>
<li>conserved in eukaryotes only</li>
</ul></li>
<li>Other terms
<ul>
<li>species distribution, Schizosaccharomyces specific - indicates that a gene product is only identified in the genus <em>Schizosaccharomyces</em> at present, but is present in more than one species in this genus</li>
<li>species distribution, Schizosaccharomyces pombe specific - indicates that a gene product is identified only in <em>S. pombe</em>,​ not in any other species even in the <em>Schizosaccharomyces</em> genus</li>
<li>no apparent <em>S. cerevisiae</em> ortholog - indicates a gene product which is conserved at least outside the <em>Schizosaccharomyces</em> but is absent from <em>S. cerevisiae</em></li>
<li>faster evolving duplicate - indicates that the copy of a gene product which has duplicated after the divergence of <em>Schizosaccharomyces</em> from other yeasts is evolving faster than its duplicate partner (judged because the fission yeast duplicate is more closely related to the orthologs in other species, but is the best hit of this gene product)</li>
<li>predominantly single copy (one to one) - the gene product is conserved largely one-to one across species with minor exceptions (for example, frequently duplicated in the vertebrate lineage)</li>
<li>orthologs cannot be distinguished - the gene product appears to be conserved, but it is not possible to identify orthologs unambiguously</li>
</ul></li>
</ul>
<p><strong>Additional notes:</strong></p>
<ul>
<li>Any term which has &quot;conserved in fungi&quot; but does <em>not</em> have &quot;no apparent <em>S. cerevisiae</em> ortholog&quot; will have a manually annotated <em>S. cerevisiae</em> ortholog.</li>
<li>Any gene product which is annotated as &quot;conserved in vertebrates&quot; will have a manually annotated human ortholog.</li>
<li>Note that this section was previously called &quot;species distribution&quot;; it has been changed to avoid confusion with the correct usage (in ecology, the geographical range where a species is found).</li>
</ul>

  </div>
</div>
<div *ngIf="section == 'help'">
  <div class="docs-menu">
<h3 id="help"><a routerLink="/help" routerLinkActive="active">Help</a></h3>
<ul>
<li><a href="http://www.pombase.org/faq">Frequently Asked Questions (FAQ)</a></li>
<li><a routerLink="/documentation" routerLinkActive="active">PomBase Documentation</a></li>
<li><a routerLink="/about/contacts" routerLinkActive="active">Contact Curators</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'index'" class="docs-content">
<h1 id="help">Help</h1>
<h3 id="old-frequently-asked-questions-faq"><a href="http://www.pombase.org/faq">Old Frequently Asked Questions (FAQ)</a></h3>
<h3 id="pombase-documentation"><a routerLink="/documentation" routerLinkActive="active">PomBase Documentation</a></h3>
<h3 id="contact-curators"><a routerLink="/about/contacts" routerLinkActive="active">Contact Curators</a></h3>

  </div>
</div>
<div *ngIf="section == 'status'">
  <div class="docs-menu">
<h3 id="status"><a routerLink="/status" routerLinkActive="active">Status</a></h3>
<ul>
<li><a routerLink="/status/statistics" routerLinkActive="active">Statistics</a></li>
<li><a routerLink="/status/sequencing-status" routerLinkActive="active">Sequencing status</a>
<ul>
<li><a routerLink="/status/centromeres" routerLinkActive="active">Centromeres</a></li>
<li><a routerLink="/status/mating-type-region" routerLinkActive="active">Mating type region</a></li>
<li><a routerLink="/status/telomeres" routerLinkActive="active">Telomeres</a></li>
</ul></li>
<li><a routerLink="/status/sequencing-updates" routerLinkActive="active">Sequence updates</a></li>
<li><a routerLink="/status/sequence-updates-pending" routerLinkActive="active">Sequence Updates Pending</a></li>
<li><a routerLink="/status/gene-coordinate-changes" routerLinkActive="active">Gene coordinate changes</a></li>
<li><a routerLink="/status/new-and-removed-genes" routerLinkActive="active">New and removed genes</a></li>
<li><a routerLink="/status/gene-characterisation" routerLinkActive="active">Gene characterisation</a></li>
<li><a routerLink="/status/gene-characterisation-statistics-history" routerLinkActive="active">Gene characterisation history</a></li>
<li><a routerLink="/status/priority-unstudied-genes" routerLinkActive="active">Priority unstudied genes</a></li>
<li><a routerLink="/status/unmapped_genes" routerLinkActive="active">Unmapped genes</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'centromeres'" class="docs-content">
<h1 id="centromeres">Centromeres</h1>
<p>At present, each <span>centromere</span> is annotated as a single sequence feature in <span>PomBase</span>, which can be viewed in and downloaded from the <span>Ensembl</span> genome browser.</p>
<p>Chromosome coordinates (links to <span>Ensembl</span> genome browser):</p>
<ul>
<li>CEN 1: <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=I%3A3753687-3789421;site=ensemblunit">I:3753687-3789421</a></li>
<li>CEN 2: <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=II%3A1602264-1644747;site=ensemblunit">II:1602264-1644747</a></li>
<li>CEN 3: <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=III%3A1070904-1137003;site=ensemblunit">III:1070904-1137003</a></li>
</ul>
<p>The 3 centromeres are also available as contiguated sequence (the remaining gaps in CEN2 and CEN3 are represented by 100 N's)</p>
<ul>
<li><a href="http://www.pombase.org/sites/pombase.org/files/documents/cen1.gz">CEN 1</a> (gzipped EMBL format)</li>
<li><a href="http://www.pombase.org/sites/pombase.org/files/documents/cen2.gz">CEN 2</a> (gzipped EMBL format)</li>
<li><a href="http://www.pombase.org/sites/pombase.org/files/documents/cen3.gz">CEN 3</a> (gzipped EMBL format)</li>
</ul>
<p>Note that specific repeats within centromeres cannot yet be viewed or searched as features on PomBase pages, but they can be displayed on tracks in the genome browser. Turn on any or all of &quot;PomBase Annotated Centromeric Inner Repeat Region (SO:0001798)&quot;, &quot;PomBase Annotated dg Repeat (SO:0001898)&quot; or &quot;PomBase Annotated dh Repeat (SO:0001899)&quot; (see the <a href="http://www.pombase.org/faqs/how-can-i-show-or-hide-tracks-genome-browser">browser track FAQ</a> for instructions).</p>
<p>Repeats are also shown in the diagram below. To see the repeat sequences, download and unzip the <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Chromosome_contigs/">contiguated sequence files</a> and view them in <a href="http://www.sanger.ac.uk/resources/software/artemis/">Artemis</a>. (See <a href="http://www.pombase.org/faqs/there-equivalent-artemis-java-applet-pombase">this FAQ</a> for more information.)</p>
<p><span style="color: red;">Note: Recent work by Chad Ellermeier and Gerry Smith suggests that there are only 4 +/- 1 copies of the 6760 bp repeat missing from chromosome 3</span></p>
<p><span style="color: red;">Please note: This is a schematic diagram. Distances and overlaps are approximate. Please refer to the sequence data to design experimental constructs.</span></p>
<div class="figure">
<img src="images/centromeremapping.gif" alt="Centromere Map" width="779" height="540" />
<p class="caption">Centromere Map</p>
</div>
<p>Centromere Map from: The genome sequence of <em>Schizosaccharomyces pombe</em>.<br />
Nature 2002 Feb 21;415(6874):871-80. created by Rhian Gwilliam</p>

  </div>
  <div *ngIf="pageName == 'gene-characterisation'" class="docs-content">
<h1 id="gene-characterisation">Gene characterisation</h1>
<p><app-characterisation-status-table></app-characterisation-status-table></p>
<p>The table is updated with every PomBase data release.</p>
<hr />
<h2 id="gene-characterisation-status-descriptions">Gene Characterisation Status Descriptions</h2>
<p><strong>Experimentally characterised:</strong> Completely or partially characterised in a small scale experiment, with some published information about the biological role (corresponding to any of the <a href="browse-curation/fission-yeast-go-slim-terms">fission yeast GO slim</a> biological process terms)</p>
<p><strong>Role inferred from homology:</strong> A biological role (as above, a <a href="browse-curation/fission-yeast-go-slim-terms">fission yeast GO slim</a> term) is inferred from homology to an experimentally characterised gene product</p>
<p><strong>Conserved protein (unknown biological role):</strong> Conserved outside the <i>Schizosaccharomyces</i>,​ but nothing known about the biological role in any organism</p>
<p><strong><i>Schizosaccharomyces</i> specific protein, uncharacterised:</strong> Unpublished and found only in fission yeast (<i>S. pombe</i>, <i>S. octosporus</i>,​ <i>S. japonicus</i>, <i>S. cryophilus</i>); nothing known about biological role. May be single copy or a member of a multi-member family.</p>
<p><strong><i>S. pombe</i> specific protein, uncharacterised:</strong> Unpublished and found only in <i>S. pombe </i>(not detected in other <i>Schizosaccharomyces</i> species); nothing known about biological role</p>
<p><strong>Dubious:</strong> Unlikely to be protein coding</p>
<p>Note: You can retrieve current lists of genes with each characterisation status using the <a href="http://www.pombase.org/spombe/query/builder">Advanced Search</a>. Select the Annotation Status filter, then choose a status from the pulldown menu,​ and submit.</p>

  </div>
  <div *ngIf="pageName == 'gene-characterisation-statistics-history'" class="docs-content">
<h1 id="gene-characterisation-statistics-history">Gene characterisation statistics history</h1>
<p><strong>History of characterisation status using current status descriptions (introduced with PomBase Chado release 46, August 2014):</strong></p>
<table>
 <thead>
  <tr>
   <th style="width: 100px;">Date</th>
   <th style="width: 100px;">Published</th>
   <th style="width: 100px;">Role inferred</th>
   <th style="width: 100px;">Conserved unknown</th>
   <th style="width: 100px;"><em><em><strong>Schizo.</strong></em></em></th>
   <th style="width: 100px;"><em>S. pombe</em></th>
   <th style="width: 100px;">Dubious</th>
   <th style="width: 100px;">Transposon</th>
   <th style="width: 100px;">Total</th>
  </tr>
 </thead>
 <tbody>
  <tr>
  <td><span class="no-break">2016-09-12</span></td>
  <td>2227</td>
  <td>2034</td>
  <td>503</td>
  <td>190</td>
  <td>149</td>
  <td>55</td>
  <td>13</td>
  <td>5171</td>
 </tr>
 <tr>
  <td><span class="no-break">2015-01-19</span></td>
  <td>2154</td>
  <td>2050</td>
  <td>529</td>
  <td>178</td>
  <td>143</td>
  <td>76</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-09-17</span></td>
  <td>2124</td>
  <td>2066</td>
  <td>537</td>
  <td>183</td>
  <td>143</td>
  <td>77</td>
  <td>&nbsp;13</td>
  <td>5143</td>
 </tr>
 </tbody>
</table>
<p><strong>History of characterisation status using previous status descriptions:</strong></p>
<table>
 <thead>
  <tr>
   <th style="width: 100px;">Date</th>
   <th style="width: 100px;">Published</th>
   <th style="width: 100px;">Role inferred</th>
   <th style="width: 100px;">Conserved unknown</th>
   <th style="width: 100px;"><em>S. pombe</em></th>
   <th style="width: 100px;">Orphan</th>
   <th style="width: 100px;">Dubious</th>
   <th style="width: 100px;">Transposon</th>
   <th style="width: 100px;">Total</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td><span class="no-break">2006-08-25</span></td>
  <td>1560</td>
  <td>2433</td>
  <td>458</td>
  <td>68</td>
  <td>403</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>4979</td>
 </tr>
 <tr>
  <td><span class="no-break">2007-02-27</span></td>
  <td>1607</td>
  <td>2329</td>
  <td>572</td>
  <td>47</td>
  <td>364</td>
  <td>60</td>
  <td>&nbsp;</td>
  <td>4979</td>
 </tr>
 <tr>
  <td><span class="no-break">2008-06-05</span></td>
  <td>1715</td>
  <td>2210</td>
  <td>634</td>
  <td>42</td>
  <td>344</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5002</td>
 </tr>
 <tr>
  <td><span class="no-break">2009-01-14</span></td>
  <td>1812</td>
  <td>2132</td>
  <td>636</td>
  <td>56</td>
  <td>318</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5011</td>
 </tr>
 <tr>
  <td><span class="no-break">2010-02-22</span></td>
  <td>1848</td>
  <td>2160</td>
  <td>614</td>
  <td>55</td>
  <td>286</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5020</td>
 </tr>
 <tr>
  <td><span class="no-break">2010-06-30</span></td>
  <td>1916</td>
  <td>2163</td>
  <td>592</td>
  <td>54</td>
  <td>285</td>
  <td>58</td>
  <td>11</td>
  <td>5025</td>
 </tr>
 <tr>
  <td><span class="no-break">2011-04-27</span></td>
  <td>1916</td>
  <td>2169</td>
  <td>588</td>
  <td>64</td>
  <td>315</td>
  <td>79</td>
  <td>11</td>
  <td>5142</td>
 </tr>
 <tr>
  <td><span class="no-break">2011-12-16</span></td>
  <td>1936</td>
  <td>2165</td>
  <td>582</td>
  <td>66</td>
  <td>309</td>
  <td>73</td>
  <td>11</td>
  <td>5142</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-05-23</span></td>
  <td>2012</td>
  <td>2137</td>
  <td>541</td>
  <td>66</td>
  <td>303</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-12-06</span></td>
  <td>2083</td>
  <td>2096</td>
  <td>522</td>
  <td>64</td>
  <td>294</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-07-14</span></td>
  <td>2110</td>
  <td>2076</td>
  <td>517</td>
  <td>64</td>
  <td>292</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 </tbody>
</table>
<h3 id="status-descriptions">Status Descriptions</h3>
<table>
<thead>
 <tr>
  <th>Shorthand</th>
  <th>Full name</th>
  <th>Description</th>
  <th>Comment</th>
 </tr>
</thead>
<tbody>
 <tr>
  <td>Published</td>
  <td>Experimentally characterised (or published)</td>
  <td>Completely or partially characterised in a small scale experiment, with some published information about the biological role (corresponding to any of the fission yeast GO slim biological process terms)</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Role inferred</td>
  <td>Role inferred from homology</td>
  <td>A biological role (as above, a fission yeast GO slim term) is inferred from homology to an experimentally characterised gene product</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Conserved unknown</td>
  <td>Conserved protein (unknown biological role)</td>
  <td>Conserved outside the <em>Schizosaccharomyces</em>, but nothing known about the biological role in any organism</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><em>Schizo.</em></td>
  <td><em><em>Schizosaccharomyces</em>&nbsp;</em>specific protein, uncharacterised</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&#8203;&nbsp;<em>S. octosporus,&#8203;&nbsp;<em>S. japonicus, S. cryophilus</em></em><em></em>); nothing known about biological role. May be single copy or a member of a multi-member family.</td>
  <td>Introduced Sept. 2014</td>
 </tr>
 <tr>
  <td><em>S. pombe</em></td>
  <td><em><em>S. pombe</em>&nbsp;</em>specific protein, uncharacterised</td>
  <td>Unpublished and found only in&nbsp;<em>S. pombe&nbsp;</em>(not detected in other&nbsp;<em>Schizosaccharomyces</em>&nbsp;species); nothing known about biological role</td>
  <td>Introduced Sept. 2014&nbsp;</td>
 </tr>
 <tr>
  <td>Transposon</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Dubious</td>
  <td>&nbsp;</td>
  <td>Unlikely to be protein coding</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><em>S. pombe</em></td>
  <td>&nbsp;<em>S. pombe</em>&nbsp;specific families</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&#8203;&nbsp;<em>S. octosporus,&#8203;&nbsp;</em><em>S. japonicus, S. cryophilus</em>); nothing known about biological role, but are not single copy (duplications in fission yeast)</td>
  <td>&nbsp;Used Aug. 2006-Aug. 2014</td>
 </tr>
 <tr>
  <td>&nbsp;Orphan</td>
  <td>Sequence orphan, uncharacterised</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&#8203;&nbsp;<em>S. octosporus</em>,&#8203;&nbsp;<em>S. japonicus, S. cryophilus</em>); nothing known about biological role</td>
  <td>Used Aug. 2006-Aug. 2014</td>
 </tr>
 </tbody>
 </table>

  </div>
  <div *ngIf="pageName == 'gene-coordinate-changes'" class="docs-content">
<h1 id="gene-coordinate-changes">Gene Coordinate Changes</h1>
<table>
<thead>
<tr class="header">
<th>Systematic ID</th>
<th>Standard name</th>
<th>Comments</th>
<th>Reference</th>
<th>Date</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a routerLink="/gene/SPAC19B12.05c" routerLinkActive="active">SPAC19B12.05c</a></td>
<td></td>
<td>additional exon</td>
<td><span class="no-break">pers. comm. Val Wood</span></td>
<td><span class="no-break">2003-04-22</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC11E3.03" routerLinkActive="active">SPAC11E3.03</a></td>
<td>pcs1</td>
<td>N terminal truncated, removed exon 9067..9103, truncated from 9149 to 9211 start</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12689592">PMID:12689592</a></td>
<td><span class="no-break">2003-04-13</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC320.13c" routerLinkActive="active">SPCC320.13c</a></td>
<td>ark1</td>
<td>trimmed to 2nd met</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12676091">PMID:12676091</a></td>
<td><span class="no-break">2003-01-08</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC6B1.09c" routerLinkActive="active">SPBC6B1.09c</a></td>
<td>nbs1</td>
<td>prediction extended by 4 N- terminal exons, this identified FHA domain</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-04-22</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC6B1.09c" routerLinkActive="active">SPBC6B1.09c</a></td>
<td>nbs1</td>
<td>additional intron 25664..25705, and exon extension 25974..25988</td>
<td>pers. comm. Charly Chahwan</td>
<td><span class="no-break">2003-03-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC16A10.04" routerLinkActive="active">SPAC16A10.04</a></td>
<td>rho4</td>
<td>N-terminal exon replaced</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12653963">PMID:12653963</a></td>
<td><span class="no-break">2003-03-31</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC23C4.08" routerLinkActive="active">SPAC23C4.08</a></td>
<td>rho3</td>
<td>N-terminal exon removed</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12653963">PMID:12653963</a></td>
<td><span class="no-break">2003-03-31</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBP23A10.04" routerLinkActive="active">SPBP23A10.04</a></td>
<td>apc2</td>
<td>coordinates updated additional in-frame splice at 5325, 5371</td>
<td>pers. comm. Hyun-Joo Yoon</td>
<td><span class="no-break">2003-02-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1442.13c" routerLinkActive="active">SPCC1442.13c</a></td>
<td></td>
<td>C terminal exon added ; old coordinates (26947..27936); new exon extends G-patch domain</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-04-15</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1604.17c" routerLinkActive="active">SPBC1604.17c</a></td>
<td></td>
<td>previous coordinates (8945..8971, 9475..10455, 10528..10851)</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-04-29</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1296.04" routerLinkActive="active">SPAC1296.04</a></td>
<td></td>
<td>has 2 additional N-terminal exons</td>
<td>pers. comm. Val Wood</td>
<td></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23C11.10" routerLinkActive="active">SPAC23C11.10</a></td>
<td></td>
<td>found 3 additional C-terminal exons by eye</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-06-18</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC1347.05c" routerLinkActive="active">SPBC1347.05c</a></td>
<td></td>
<td>new C-terminal exon</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-10-11</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2F3.12c" routerLinkActive="active">SPAC2F3.12c</a></td>
<td></td>
<td>created from the C-terminal of SPAC2F3.13c</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-06-30</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC2F3.13c" routerLinkActive="active">SPAC2F3.13c</a></td>
<td></td>
<td>split to create SPAC2F3.13c and SPAC2F3.12c</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-06-30</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC22F3.03c" routerLinkActive="active">SPAC22F3.03c</a></td>
<td>rdh54</td>
<td>4th exon extended by altering acceptor in intron 3 cosmid coordinates 34092-33923</td>
<td>pers. comm. Mike Catlett</td>
<td><span class="no-break">2003-05-09</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29A4.14c" routerLinkActive="active">SPAC29A4.14c</a></td>
<td></td>
<td>N-terminal extended</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-09-11</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC31G5.07" routerLinkActive="active">SPAC31G5.07</a></td>
<td></td>
<td>N terminal extended to use longest ORF in the absence of homology</td>
<td>pers. comm. Henar Valdivieso Montero</td>
<td><span class="no-break">2003-01-08</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC19B12.06c" routerLinkActive="active">SPAC19B12.06c</a></td>
<td></td>
<td>added exon 18153..18218</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-04-22</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2F3.13c" routerLinkActive="active">SPAC2F3.13c</a></td>
<td></td>
<td>annotated as SPAC2F3.13c putative tRNA-ribosyltransferase, but actually a gene merge. Split to create SPAC2F3.13c the tRNA-ribosyltransferase and new CDS SPAC2F3.17c was created from the C-terminal</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-06-30</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29B12.08" routerLinkActive="active">SPAC29B12.08</a></td>
<td></td>
<td>new N-terminal exon</td>
<td>pers. comm. Aengus Stewart</td>
<td><span class="no-break">2003-09-09</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC959.05c" routerLinkActive="active">SPAC959.05c</a></td>
<td></td>
<td>extended N-terminal to first methionine</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-09-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC15D4.02" routerLinkActive="active">SPBC15D4.02</a></td>
<td></td>
<td>trimmed N terminal 28.10.03; looked overextended as large low complexity region in front in zinc finger which is usually N-term; alignment looks better</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-10-28</span></td>
</tr>
<tr class="even">
<td>SPCC1223.15c</td>
<td>spc19</td>
<td>4 amino acid extension at N-terminus</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16079914">PMID:16079914</a></td>
<td><span class="no-break">2005-09-09</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC17H9.20" routerLinkActive="active">SPAC17H9.20</a></td>
<td>psc3</td>
<td>added intron at 37250..37297</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16207082">PMID:16207082</a></td>
<td><span class="no-break">2005-10-10</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC31G5.06" routerLinkActive="active">SPAC31G5.06</a></td>
<td></td>
<td>added N-teminal exon to extend prediction</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2005-11-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC9.05" routerLinkActive="active">SPAC9.05</a></td>
<td>fml1</td>
<td>second intron acceptor extended by 15 bp</td>
<td>pers. comm. Anke Schürer</td>
<td><span class="no-break">2006-01-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC3B9.22c" routerLinkActive="active">SPBC3B9.22c</a></td>
<td>dad4</td>
<td>added new C-terminal exon based on Pfam protein family</td>
<td><a href="http://pfam.xfam.org/">Pfam</a></td>
<td><span class="no-break">2006-02-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC4G3.05c" routerLinkActive="active">SPCC4G3.05c</a></td>
<td>mus81</td>
<td>added N terminal exon</td>
<td>pers. comm. Fred Kippert</td>
<td><span class="no-break">2006-02-17</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1D4.11c" routerLinkActive="active">SPAC1D4.11c</a></td>
<td>lkh1</td>
<td>lengthened to 690 a.a.</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12565823">PMID:12565823</a></td>
<td><span class="no-break">2006-04-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBCPT2R1.08c" routerLinkActive="active">SPBCPT2R1.08c</a></td>
<td>tlh2</td>
<td>truncated to 1919 a.a.; removed truncated version of a malate transporter fused to gene, not known whether these residues are included in translation</td>
<td>pers. comm. Liew Li Phing</td>
<td><span class="no-break">2006-04-13</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC6G9.13c" routerLinkActive="active">SPAC6G9.13c</a></td>
<td>bqt1</td>
<td>Additional C terminal exon</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16615890">PMID:16615890</a></td>
<td><span class="no-break">2006-05-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC21E11.05c" routerLinkActive="active">SPAC21E11.05c</a></td>
<td>cyp8</td>
<td>Single base insertion allowed first 2 exons to be merged and extended (GAGT[G]GCA)</td>
<td>pers. comm. Trevor Pemberton (via Ivo Pedruzzi, UniProt)</td>
<td><span class="no-break">2006-05-22</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCPJ732.03" routerLinkActive="active">SPCPJ732.03</a></td>
<td>meu15</td>
<td>new coordinates as per publication</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11376151">PMID:11376151</a></td>
<td><span class="no-break">2006-06-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC4B3.05c" routerLinkActive="active">SPCC4B3.05c</a></td>
<td>hem12</td>
<td>Coordinates updated from join(28064..28097, 28155..28189, 28238..28816, 28866..29330) to join(28064..28097, 28155..28189, 28236..28260, 28308..28816, 28866..29330). Improves Homology</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2006-06-12</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC12D12.09" routerLinkActive="active">SPAC12D12.09</a></td>
<td>rev7</td>
<td>added new exon to insert KCIDIFGEF</td>
<td>pers. comm. Nicole Kosarek</td>
<td><span class="no-break">2006-08-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC12D12.09" routerLinkActive="active">SPAC12D12.09</a></td>
<td>rev7</td>
<td>truncated penultimate exon to remove GIN</td>
<td>pers. comm. Nicole Kosarek</td>
<td><span class="no-break">2006-08-01</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC30B4.08" routerLinkActive="active">SPBC30B4.08</a></td>
<td>eri1</td>
<td>gene structure updated as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16797182">PMID:16797182</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16797182">PMID:16797182</a></td>
<td><span class="no-break">2006-09-22</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.08" routerLinkActive="active">SPAPB17E12.08</a></td>
<td></td>
<td>coordinates updated from join(9968..9990, 10070..10442) to join(9609..9623, 9699..9761, 9937..9986, 10070..10442). Improved homology</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2006-12-12</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1223.10c" routerLinkActive="active">SPCC1223.10c</a></td>
<td>eaf1</td>
<td>Added exons as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17150956">PMID:17150956</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17150956">PMID:17150956</a></td>
<td><span class="no-break">2006-12-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP22H7.09c" routerLinkActive="active">SPBP22H7.09c</a></td>
<td>mis15</td>
<td>updated sequence coordinates as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15369671">PMID:15369671</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15369671">PMID:15369671</a></td>
<td><span class="no-break">2007-01-05</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23D3.08" routerLinkActive="active">SPAC23D3.08</a></td>
<td>usp108</td>
<td>final exon removed</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-02-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPNCRNA.82" routerLinkActive="active">SPNCRNA.82</a></td>
<td>mrp1</td>
<td>coordinates updated from 1234705..1234837 to 1234451..1234849, upon realisation that this corresponded to RNase MRP</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-02-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC13G7.07" routerLinkActive="active">SPAC13G7.07</a></td>
<td></td>
<td>changed second exon 3' boundary and added C-terminal exons</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17310250">PMID:17310250</a></td>
<td><span class="no-break">2007-03-23</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29E6.04" routerLinkActive="active">SPAC29E6.04</a></td>
<td>nnf1</td>
<td>Sequence frameshifted at 4410192, new coordinates give an N terminal extension from 4410194..4410391 (NOTE: Still needs resequencing and underlying sequence changing)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17035632">PMID:17035632</a></td>
<td><span class="no-break">2007-05-16</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23C4.04c" routerLinkActive="active">SPAC23C4.04c</a></td>
<td></td>
<td>altered to include stop codon and trimmed to methionine</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC409.12" routerLinkActive="active">SPBC409.12</a></td>
<td>stn1</td>
<td>N-terminal extended by 2 exons</td>
<td>pers. comm. C. Chahwan</td>
<td><span class="no-break">2007-07-06</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC9E9.02" routerLinkActive="active">SPAC9E9.02</a></td>
<td></td>
<td>altered to include stop codon</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-07-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F8.07c" routerLinkActive="active">SPAC22F8.07c</a></td>
<td>rtf1</td>
<td>Two in-frame splices added</td>
<td><a href="http://www.ebi.ac.uk/ena/data/view/AJ627891">EMBL:AJ627891</a></td>
<td><span class="no-break">2007-08-28</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1020.11c" routerLinkActive="active">SPCC1020.11c</a></td>
<td></td>
<td>coordinates updated from 757494..757802 to join(757365..757424, 757506..757802)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-10-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1529.01" routerLinkActive="active">SPCC1529.01</a></td>
<td></td>
<td>new exon added; coordinates changed from join(237554..237923, 237975..238080, 238139..238533, 238588..239105) to join(237554..237923, 237975..238080, 238139..238533, 238588..239092, 239151..239250)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-10-12</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPB2B2.07c" routerLinkActive="active">SPBPB2B2.07c</a></td>
<td></td>
<td>coordinates updated from 4474434..4474931 to join(4474145..4474403, 4474438..4474931)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-22</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC6G9.16c" routerLinkActive="active">SPAC6G9.16c</a></td>
<td></td>
<td>coordinates changed from 3278951..3279349 to join(3278951..3279367, 3279420..3279539, 3279683..3279819, 3279865..3279985)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-22</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC354.07c" routerLinkActive="active">SPBC354.07c</a></td>
<td></td>
<td>additional exon added; coordinates updated from 558335..559225, 559273..559474, 559942..560028, 560151..560170 to 558335..559225, 559273..559474, 559942..560028, 560151..560163, 560231..560246</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-28</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC631.02" routerLinkActive="active">SPAC631.02</a></td>
<td></td>
<td>coordinates updated from 2107940..2110123 to 2107470..2107515, 2107860..2110123</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-28</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC18.18c" routerLinkActive="active">SPCC18.18c</a></td>
<td></td>
<td>coordinates changed from 1988631..1990079 to 1988631..1990100, 1990630..1990722</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-29</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC16A10.06c" routerLinkActive="active">SPAC16A10.06c</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data; coordinates updated from 3089583..3089996, 3090112..3090285, 3090533..3090697 to 3089583..3090047, 3090112..3090285, 3090533..3090697</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC17C9.08" routerLinkActive="active">SPAC17C9.08</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data and unsupported from mRNA data so final exon deleted; coordinates changed from 4484758..4484804, 4484843..4485274, 4485350..4485596, 4485713..4485994 to 4484835..4485274, 4485350..4485596, 4485713..4485994</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.08" routerLinkActive="active">SPAPB17E12.08</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data so small N-term exon deleted; coordinates changed from 1279492..1279506, 1279582..1279644, 1279820..1279869, 1279953..1280325 to 1279573..1279644, 1279820..1279869, 1279953..1280325</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAPB1A10.15" routerLinkActive="active">SPAPB1A10.15</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data so small C-term exon deleted; coordinates changed from 1892554..1892640, 1892926..1893060, 1893098..1893504, 1893585..1893618 to 1892554..1892640, 1892926..1893060, 1893098..1893694</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC56F8.05c" routerLinkActive="active">SPAC56F8.05c</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data. N terminal exon unsupported and had no branch site so exon deleted; coordinates changed from 1134697..1135061, 1135195..1135402, 1135517..1135684, 1135729..1135776, 1135932..1135970, 1136111..1136170 to 1134697..1135061, 1135195..1135402, 1135517..1135684, 1135729..1135776, 1135932..1136003</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1450.09c" routerLinkActive="active">SPCC1450.09c</a></td>
<td></td>
<td>original intron absent from Solexa data; C terminal exon unsupported so exon deleted; coordinates changed from 1741451..1741481, 1741589..1743459 to 1741588..1743459</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC736.05" routerLinkActive="active">SPCC736.05</a></td>
<td>wtf7</td>
<td>removed final exon, unsupported by Solexa intron data; coordinates changed from 320617..320755, 320964..321335, 321387..321509, 321554..321582 to 320617..320755, 320964..321335, 321387..321523</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC227.11c" routerLinkActive="active">SPAC227.11c</a></td>
<td></td>
<td>removed final exon, unsupported by Solexa data; coordinates changed from 514434..514489, 514557..515082, 515222..515482, 515534..515659 to 514552..515082, 515222..515482, 515534..515659</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC21B10.02" routerLinkActive="active">SPBC21B10.02</a></td>
<td></td>
<td>intron unsupported by Solexa transcript data, removed final exon; coordinates changed from 1669624..1669695, 1669737..1670144 to 1669719..1670144</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC21C3.07c" routerLinkActive="active">SPBC21C3.07c</a></td>
<td></td>
<td>removed final exon, not supported by Solexa transcript data; coordinates changed from 3807714..3807783, 3807963..3808607, 3808665..3808795 to 3807920..3808607, 3808665..3808795</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC29A3.07c" routerLinkActive="active">SPBC29A3.07c</a></td>
<td></td>
<td>final exon deleted, not supported by Solexa intron data; homology improved; coordinates changed from 2050069..2050183, 2050304..2050358, 2050452..2050695 to 2050258..2050358, 2050452..2050695</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1604.17c" routerLinkActive="active">SPBC1604.17c</a></td>
<td></td>
<td>updated, better donor (GTAATA instead of GTTAGT); original intron prediction not supported by Solexa data; coordinates changed from 3900047..3900121, 3900166..3901146, 3901219..3901542 to 3900047..3900121, 3900166..3901170, 3901219..3901542</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16A3.14" routerLinkActive="active">SPBC16A3.14</a></td>
<td></td>
<td>final exon deleted, not supported by Solexa intron data; coordinates changed from 4270907..4270939, 4271137..4271937 to 4271131..4271937</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23D3.08" routerLinkActive="active">SPAC23D3.08</a></td>
<td>usp108</td>
<td>final exon deleted, not supported by Solexa intron data; coordinates changed from 4352514..4353359, 4353410..4353573, 4353623..4353761 to 4352514..4353359, 4353410..4353577</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC4C5.01" routerLinkActive="active">SPAC4C5.01</a></td>
<td></td>
<td>altered intron 5 to improve donor and acceptor, improves homology; coordinates changed from 1192545..1192592, 1192662..1192704, 1192779..1192841, 1192916..1193010, 1193062..1193175, 1193236..1193613 to 1192545..1192592, 1192662..1192704, 1192779..1192841, 1192916..1193010, 1193062..1193145, 1193197..1193613</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16H5.04" routerLinkActive="active">SPBC16H5.04</a></td>
<td></td>
<td>altered intron 2 donor and acceptor, improved homology; coordinates changed from 2293393..2293824, 2293938..2294039, 2294087..2294125 to 2293393..2293775, 2293877..2294039, 2294087..2294125</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC11B10.03" routerLinkActive="active">SPBC11B10.03</a></td>
<td></td>
<td>removed N-teminal exon, poor donor, did not improve homology; coordinates changed from 1490376..1490423, 1490593..1491747 to 1490611..1491747</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAPB1A10.03" routerLinkActive="active">SPAPB1A10.03</a></td>
<td></td>
<td>SPAPB1A10.03 split to create SPAPB1A10.03 and SPAPB1A10.16; new coordinates 1863363..1863381, 1863434..1863762 and 1863972..1864388</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1494.02" routerLinkActive="active">SPCC1494.02</a></td>
<td></td>
<td>2 C-terminal exons removed, not supported by homology; coordinates changed from 2317988..2318068, 2318200..2318244, 2318398..2318508, 2318579..2318676, 2318903..2318966 to 2318335..2318508, 2318579..2318676, 2318903..2318966</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1442.13c" routerLinkActive="active">SPCC1442.13c</a></td>
<td></td>
<td>deleted final exon, improved homology; coordinates changed from 1791196..1791285, 1791376..1791909 to 1791346..1791909</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC622.14" routerLinkActive="active">SPCC622.14</a></td>
<td></td>
<td>altered donor for intron 3, improved donor consensus; coordinates changed from 1427098..1427310, 1427632..1427709, 1427788..1428180, 1428277..1428522 to 1427098..1427310, 1427632..1427709, 1427788..1428216, 1428277..1428522</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC823.04" routerLinkActive="active">SPAC823.04</a></td>
<td></td>
<td>changed first intron acceptor, which gave a different n-term exon and improved homology; coordinates changed from 2587859..2587901, 2588085..2588620 to 2587729..2587897, 2588085..2588620</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC688.08" routerLinkActive="active">SPAC688.08</a></td>
<td></td>
<td>deleted final exon, not supported by Solexa data; coordinates changed from 3123163..3125057, 3125099..3126310, 3126356..3126631, 3126671..3126989 to 3123163..3125057, 3125099..3126310, 3126356..3126653</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29E6.03c" routerLinkActive="active">SPAC29E6.03c</a></td>
<td></td>
<td>changed internal splice site, better intron, slightly improved homology; deleted final small exon, unsupported; coordinates changed from 4405871..4405889, 4405994..4407291, 4407447..4409096, 4409171..4409338 to 4405990..4407375, 4407447..4409096, 4409171..4409338</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC806.09c" routerLinkActive="active">SPAC806.09c</a></td>
<td></td>
<td>changed from 251726..252337, 252392..252752, 252945..252958 to 251726..252337, 252392..252752, 253590..253630</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-12-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC19G7.06" routerLinkActive="active">SPBC19G7.06</a></td>
<td></td>
<td>coordinates changed from 2359429..2359479, 2359547..2359693, 2359845..2359928, 2360064..2361092 to 2359429..2359479, 2359547..2359693, 2359845..2359928, 2360001..2361092</td>
<td>Christopher J. McInerny</td>
<td><span class="no-break">2007-12-07</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16D10.10" routerLinkActive="active">SPBC16D10.10</a></td>
<td></td>
<td>exon 4 splice altered; coordinates changed from 3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618920, 3619065..3619321, 3619366..3619540 to 3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618971, 3619077..3619321, 3619366..3619540</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16E9.16c" routerLinkActive="active">SPBC16E9.16c</a></td>
<td></td>
<td>This CDS has 2 frameshifts in GeneDB at present; coordinates changed from 1947985..1949069 , 1949218..1950058 , 1950253..1950255 to 1947985..1948953, 1948955..1950050, 1950053..1950255</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18079165">PMID:18079165</a></td>
<td><span class="no-break">2007-01-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16C6.02c" routerLinkActive="active">SPBC16C6.02c</a></td>
<td></td>
<td>complement(join(4322221..4322742, 4322970..4331555, 4331641..4331793, 4331856..4331968, 4332007..4332028)) was split into complement(join(4322829..4331555, 4331641..4331793, 4331856..4331968, 4332007..4332028)) and new gene SPBC16C6.14 - complement(4322221..4322622)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
<td><span class="no-break">2008-04-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC13G6.04" routerLinkActive="active">SPAC13G6.04</a></td>
<td>tim8</td>
<td>old join(180423..180528, 180586..180776), new join(180423..180528, 180586..180721, 180890..180911); added new 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1639.01c" routerLinkActive="active">SPAC1639.01c</a></td>
<td></td>
<td>old complement(join(251726..252337, 252392..252752, 253590..253630)), new complement(join(251726..251745, 251863..252337, 252392..252752, 253590..253630)); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC630.11" routerLinkActive="active">SPAC630.11</a></td>
<td>vps55</td>
<td>old join(366801..366817, 366873..366891, 366968..367040, 367123..367189, 367293..367348, 367395..367531), new join(366801..366817, 366873..366891, 366968..367040, 367105..367189, 367293..367348, 367395..367531); changed internal intron acceptor, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC24H6.01c" routerLinkActive="active">SPAC24H6.01c</a></td>
<td></td>
<td>old join(490016..490262, 490338..490771, 490812..490931, 490979..491149, 491186..491980), new join(490016..490262, 490338..490771, 490812..490931, 490979..491149, 491186..491858, 491905..492011); added new 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23C4.04c" routerLinkActive="active">SPAC23C4.04c</a></td>
<td></td>
<td>old 1036033..1036347, new join(1036033..1036342, 1036397..1036455); added intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1A6.03c" routerLinkActive="active">SPAC1A6.03c</a></td>
<td></td>
<td>old complement(1070139..1072013), new complement(join(1070061..1070138, 1070205..1072013)); new 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC56F8.04c" routerLinkActive="active">SPAC56F8.04c</a></td>
<td>coq2</td>
<td>old complement(1133226..1134302), new complement(join(1133226..1134283, 1134332..1134356)); added 5' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.08" routerLinkActive="active">SPAPB17E12.08</a></td>
<td></td>
<td>old join(1279573..1279644, 1279820..1279869, 1279953..1280325), new join(1279573..1279779, 1279820..1279869, 1279953..1280325); changed first donor to make n-terminal exon longer, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC9.06c" routerLinkActive="active">SPAC9.06c</a></td>
<td></td>
<td>old complement(join(1473692..1473885, 1473939..1474105, 1474153..1474212, 1474264..1474341, 1474460..1474539)), new complement(join(1473692..1473885, 1473939..1474105, 1474153..1474212, 1474264..1474341, 1474381..1474404, 1474460..1474539)); added new exon (2nd) improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC17A5.02c" routerLinkActive="active">SPAC17A5.02c</a></td>
<td>dbr1</td>
<td>old complement(join(1754109..1754310, 1754350..1755118, 1755164..1755285, 1755331..1755638)), new complement(join(1754033..1754145, 1754186..1754310, 1754350..1755118, 1755164..1755285, 1755331..1755638)); added new 5'intron/exon, now agrees with published date</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAPB1A10.16" routerLinkActive="active">SPAPB1A10.16</a></td>
<td></td>
<td>old 1863972..1864388, new join(1863972..1864360, 1864415..1864457); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC823.04" routerLinkActive="active">SPAC823.04</a></td>
<td></td>
<td>old 2587729..2587897 , 2588085..2588620, new join(2587729..2587897, 2588106..2588620); altered to use valid acceptor</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC823.09c" routerLinkActive="active">SPAC823.09c</a></td>
<td></td>
<td>old complement(join(2594922..2595693, 2595734..2595819)), new complement(join(2594777..2594913, 2594966..2595693, 2595734..2595819)); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB2B4.06" routerLinkActive="active">SPAPB2B4.06</a></td>
<td></td>
<td>old join(2722735..2723210, 2723273..2723300, 2723349..2723582), new join(2722735..2723210, 2723273..2723300, 2723349..2723559, 2723608..2723660); added new 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC16E8.12c" routerLinkActive="active">SPAC16E8.12c</a></td>
<td></td>
<td>old complement(join(3523136..3523644, 3524027..3524030)), new complement(join(3523136..3523644, 3523925..3523996, 3524044..3524098)); added 2 5' exons</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC15A10.10" routerLinkActive="active">SPAC15A10.10</a></td>
<td></td>
<td>old join(3697508..3697752, 3697795..3698368, 3698421..3698527, 3698570..3699538, 3699582..3699837), new join(3697508..3697752, 3697795..3698368, 3698421..3698527, 3698570..3699310, 3699359..3699538, 3699582..3699837); added in-frame intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1556.01c" routerLinkActive="active">SPAC1556.01c</a></td>
<td>rad50</td>
<td>old complement(join(3791855..3792229, 3792267..3795557, 3795601..3795684, 3795734..3795856)), new complement(join(3791831..3791851, 3791891..3792229, 3792267..3795557, 3795601..3795684, 3795734..3795856)); added small 3' intron and exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1556.03" routerLinkActive="active">SPAC1556.03</a></td>
<td>azr1</td>
<td>old join(3799244..3799680, 3799731..3800160), new join(3799244..3799680, 3799731..3800135, 3800184..3800241); added new 5' intron and exon/improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1F12.08" routerLinkActive="active">SPAC1F12.08</a></td>
<td></td>
<td>old join(3817464..3817803, 3817851..3817991, 3818035..3818543), new join(3817464..3817803, 3817851..3817991, 3818035..3818130, 3818170..3818543); added new intron, in-frame splice in final exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC2F3.18c" routerLinkActive="active">SPAC2F3.18c</a></td>
<td></td>
<td>old 3942398..3942730 new 3942299..3942364, 3942506..3942730; added small N term in-frame intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2C4.11c" routerLinkActive="active">SPAC2C4.11c</a></td>
<td>rpb28</td>
<td>old complement(4278729..4279505), new complement(join(4278729..4279405, 4279457..4279505)); added small N term in-frame intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC30.03c" routerLinkActive="active">SPAC30.03c</a></td>
<td>tsn1</td>
<td>old join(4394707..4394741, 4394799..4394967, 4395010..4395092, 4395131..4395491, 4395530..4395592), new complement(join(4394707..4394741, 4394799..4394967, 4395010..4395092, 4395131..4395367, 4395416..4395491, 439 5530..4395592)); added in-frame intron to (previous) 2nd exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC30C2.05" routerLinkActive="active">SPAC30C2.05</a></td>
<td>erv14</td>
<td>old complement(join(4687483..4688462, 4688513..4688831)), new complement(join(4687388..4687477, 4687522..4688462, 4688513..4688831)); added new 5' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC144.17c" routerLinkActive="active">SPAC144.17c</a></td>
<td></td>
<td>old complement(join(4687483..4688462, 4688513..4688831)), new complement(join(4687388..4687477, 4687522..4688462, 4688513..4688831)); added 3' intron and exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC4F10.08" routerLinkActive="active">SPAC4F10.08</a></td>
<td>mug126</td>
<td>old join(4844855..4845824, 4845863..4846203), new join(4844855..4845824, 4845863..4846063, 4846104..4846129); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC890.07c" routerLinkActive="active">SPAC890.07c</a></td>
<td>rmt1</td>
<td>old complement(join(5016072..5016215, 5016262..5016596, 5016649..5016943, 5016994..5017059, 5017138..5017252, 5017421..5017433, 5017700..5017703, 5017782..5017829)), new complement(join(5016020..5016026, 5016076..5016215, 5016262..5016596, 5016649..5016943, 5016994..5017059, 5017138..5017252, 5017421..5017433, 5017700..5017703, 5017782..5017829); added small N terminal exon, which added a single conserved cysteine</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC22E12.10c" routerLinkActive="active">SPAC22E12.10c</a></td>
<td>etp1</td>
<td>old complement(5037208..5039103), new complement(join(5037208..5037446, 5037492..5039103)); added in-frame splice</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC106.05c" routerLinkActive="active">SPBC106.05c</a></td>
<td>tim11</td>
<td>old complement(join(383527..383612, 383703..383874, 383980..384012)), new complement(join(383527..383612, 383703..383874, 383940..383960)); changed 5' exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC582.04c" routerLinkActive="active">SPBC582.04c</a></td>
<td></td>
<td>old complement(join(423269..423598, 423669..423738, 423782..423871, 423912..424373, 424424..424872, 424921..425160, 425223..425350, 425410..425446)), new complement(join(423269..423598, 423669..423738, 423782..423871, 423912..424373, 424424..424692, 424747..424872, 424921..425160, 425223..425350, 425410..425446)); added in-frame intron to (existing) exon 3</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC947.03c" routerLinkActive="active">SPBC947.03c</a></td>
<td></td>
<td>old 676629..676931, new join(676629..676908, 677057..677127); added 5' inton/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPJ4664.05" routerLinkActive="active">SPBPJ4664.05</a></td>
<td></td>
<td>old 706175..706666, new join(705932..706077, 706141..706666); added new 5' intron and exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC4.02c" routerLinkActive="active">SPBC4.02c</a></td>
<td></td>
<td>old complement(join(1185617..1186640, 1186738..1186846, 1187087..1187324)), new complement(join(1185617..1186640, 1186738..1186846, 1187087..1187162, 1187220..1187324)); added in-frame splice to first (existing) exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC8D2.17" routerLinkActive="active">SPBC8D2.17</a></td>
<td></td>
<td>old 1390183..1391238, new join(1390183..1391093, 1391147..1391222); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC4.02c" routerLinkActive="active">SPBC4.02c</a></td>
<td></td>
<td>old complement(join(2369064..2369160, 2369246..2369416, 2369455..2371243, 2371351..2371451, 2371507..2371583)), new complement(join(2369064..2369190, 2369246..2369416, 2369455..2371243, 2371351..2371451, 2371507..2371583)); changed accceptor for final intron, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBP4H10.12" routerLinkActive="active">SPBP4H10.12</a></td>
<td></td>
<td>old join(2895855..2895863, 2896001..2896507), new join(2895855..2895863, 2896001..2896459, 2896516..2896587); added new 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC2A9.07c" routerLinkActive="active">SPBC2A9.07c</a></td>
<td></td>
<td>old complement(2959078..2959902), new complement(join(2959078..2959129, 2959172..2959902)); added in-frame intron, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC2D10.15c" routerLinkActive="active">SPBC2D10.15c</a></td>
<td></td>
<td>old complement(join(2995324..2995579, 2995619..2995923, 2995971..2996066)), new complement(join(2995324..2995497, 2995534..2995579, 2995619..2995923, 2995971..2996066)); altered first intron, added in-frame intron, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC2D10.16" routerLinkActive="active">SPBC2D10.16</a></td>
<td></td>
<td>old join(2996452..2996463, 2996509..2996904), new join(2996452..2996463, 2996509..2996687, 2996738..2996879); added new intron in second exon, which created a new 3rd exon, and identified homology with Cenp-S (was previously sequence orphan)</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16D10.02" routerLinkActive="active">SPBC16D10.02</a></td>
<td>trm11</td>
<td>old join(3588806..3589595, 3589639..3589803, 3589845..3590123, 3590163..3590287), new join(3588806..3589595, 3589639..3589803, 3589845..3590123, 3590163..3590278, 3590327..3590386); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC13G1.04c" routerLinkActive="active">SPBC13G1.04c</a></td>
<td></td>
<td>old complement(3732941..3733762), new complement(join(3732806..3732930, 3732994..3733762)); added new 3' intron and exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC31F10.12" routerLinkActive="active">SPBC31F10.12</a></td>
<td></td>
<td>old join(3773353..3773525, 3773581..3773678, 3773795..3773999, 3774059..3774116, 3774192..3774212), new join(3773308..3773318, 3773373..3773525, 3773581..3773678, 3773795..3773999, 3774059..3774116, 3774192..3774212); added new 5' intron and exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC21C3.07c" routerLinkActive="active">SPBC21C3.07c</a></td>
<td></td>
<td>old complement(join(3807920..3808607, 3808665..3808795), new complement(join(3807714..3807783, 3807831..3807891, 3807946..3808607, 3808665..3808795)); added two 3' exons</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPB2B2.18" routerLinkActive="active">SPBPB2B2.18</a></td>
<td></td>
<td>old join(4502439..4502463, 4502712..4502857, 4503068..4503286, 4503527..4503664), new join(4502692..4502725, 4502787..4502857, 4503068..4503286, 4503527..4503664); changed 5' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC736.12c" routerLinkActive="active">SPCC736.12c</a></td>
<td>mmi1</td>
<td>old complement(join(338026..338178, 338232..338347, 338395..339354, 339483..339567)), new complement(join(337829..337985, 338030..338178, 338232..338347, 338395..339354, 339483..339567); added 3' exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC16C4.01" routerLinkActive="active">SPCC16C4.01</a></td>
<td>sif2</td>
<td>old join(658832..659139, 659223..659488, 659567..659628, 659709..660413), new join(658832..659139, 659223..659488, 659567..659628, 659709..660197, 660298..660322); added 3' intron which changed frame of final exon, shorter product, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC16C4.16c" routerLinkActive="active">SPCC16C4.16c</a></td>
<td></td>
<td>old complement(join(697862..698127 , 698180..698399)), new complement(join(697605..697752, 697875..698127, 698180..698399)); added 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1020.11c" routerLinkActive="active">SPCC1020.11c</a></td>
<td></td>
<td>old join(757365..757424, 757506..757802), new join(757365..757424, 757506..757761, 757851..757861); added 3'intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1442.08c" routerLinkActive="active">SPCC1442.08c</a></td>
<td>cox17</td>
<td>old complement(join(1781505..1781563, 1781637..1781744, 1781830..1781914)), new complement(join(1781505..1781563, 1781637..1781744, 1781830..1781855, 1781920..1781935, 1781981..1782032)); added 2 small 5' exons, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1620.09c" routerLinkActive="active">SPCC1620.09c</a></td>
<td>tfg1</td>
<td>old complement(join(2161527..2162363, 2162405..2162500, 2162584..2163123)), new complement(join(2161331..2161515, 2161565..2162363, 2162405..2162500, 2162584..2163123)); added 5' exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1620.10" routerLinkActive="active">SPCC1620.10</a></td>
<td>cwf26</td>
<td>old 2163205..2164118, new join(2163205..2164118, 2164159..2164165); added 3' exon encoding single methionine</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC15A10.12c" routerLinkActive="active">SPAC15A10.12c</a></td>
<td></td>
<td>old complement(3709451..3709804), new complement(join(3709206..3709346, 3709499..3709804));</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB18E9.01" routerLinkActive="active">SPAPB18E9.01</a></td>
<td></td>
<td>old join(3974310..3974485, 3974541..3974629, 3974674..3974775, 3974879..3975132, 3975193..3975429, 3975485..3975832, 3975882..3975983), new join(3974310..3974485, 3974541..3974629, 3974674..3974775, 3974879..3975132, 3975193..3975429, 3975485..3975832, 3975882..3975962, 3976014..3976079); added small 3' exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16D10.10" routerLinkActive="active">SPBC16D10.10</a></td>
<td></td>
<td>old join(3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618971, 3619077..3619321, 3619366..3619540), new join(3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618918, 3618956..3619071, 3619113..3619321, 3619366..3619540); changed 2 introns (existing 4 and 5) to create a new exon 5, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1093.03" routerLinkActive="active">SPAC1093.03</a></td>
<td></td>
<td>old join(4613627..4613686, 4613723..4614081, 4614152..4614378, 4614438..4614954, 4614996..4616331), new join(4613627..4613686, 4613723..4614081, 4614164..4614378, 4614438..4614954, 4614996..4616331); improved distance between branch and acceptor, intron 2</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1142.01" routerLinkActive="active">SPAC1142.01</a></td>
<td></td>
<td>old join(3625447..3625460, 3625555..3627544), new 3625574..3627544; removed 5' exon and trimmed to new Met, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC11D3.07c" routerLinkActive="active">SPAC11D3.07c</a></td>
<td></td>
<td>old complement(join(118195..119827, 119930..120108)), new complement(118195..120108); removed in-frame splice, not required or supported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1751.02c" routerLinkActive="active">SPAC1751.02c</a></td>
<td></td>
<td>old complement(join(385755..385887, 385938..386043, 386107..386179, 386296..386310)), new complement(join(385755..385887, 385938..386043, 386107..386149)); remove 5' exon, trimmed to new Met, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1B3.10c" routerLinkActive="active">SPAC1B3.10c</a></td>
<td></td>
<td>old complement(join(4945933..4945984, 4946094..4948084)), new complement(4946090..4948084); removed 3' exon, unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23H3.04" routerLinkActive="active">SPAC23H3.04</a></td>
<td></td>
<td>old join(2499035..2499164, 2499325..2499353, 2499443..2499824, 2499875..2500236), new join(2499035..2499164, 2499160..2499353, 2499443..2499824, 2499875..2500236); changed 5' 1st and 2nd exon, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC26H5.04" routerLinkActive="active">SPAC26H5.04</a></td>
<td></td>
<td>old join(4122149..4123354, 4123535..4124518), new 4122149..4124518; removed unsupported in-frame splice</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2C4.06c" routerLinkActive="active">SPAC2C4.06c</a></td>
<td></td>
<td>old complement(join(4269367..4270554, 4270587..4270593, 4270638..4270674, 4270736..4270871)), new complement(join(4269367..4270554, 4270631..4270674, 4270736..4270871)); replaced 2 internal exons with a different singe exon</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC328.05" routerLinkActive="active">SPAC328.05</a></td>
<td></td>
<td>old join(3484695..3484698, 3484752..3485065, 3485241..3485342, 3485382..3485528, 3485582..3485807, 3485910..3485974, 3486056..3486502), new join(3484695..3484698, 3484752..3485065, 3485241..3485342, 3485382..3485528, 3485582..3485974, 3486056..3486502); removed in-frame splice (exon 5), unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC3A12.05c" routerLinkActive="active">SPAC3A12.05c</a></td>
<td></td>
<td>old complement(join(1426213..1427148, 1427189..1429617, 1429709..1429836, 1429882..1429913)), new complement(join(1426213..1427148, 1427189..1429617, 1429709..1429830, 1429882..1429913)); improved distance beteen branch and acceptor</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC589.02c" routerLinkActive="active">SPAC589.02c</a></td>
<td></td>
<td>old complement(join(3095075..3098590, 3098626..3098775, 3098878..3098883)), new complement(join(3095075..3098590, 3098626..3098781, 3098878..3098883)); improved branch/acceptor for 1st intron</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC6F12.08c" routerLinkActive="active">SPAC6F12.08c</a></td>
<td></td>
<td>old complement(join(1322439..1323955, 1323999..1324140, 1324195..1324234, 1324299..1324336)), new complement(join(1322439..1323955, 1323999..1324140, 1324195..1324252, 1324299..1324336)); improved branch acceptor for intron 1</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC8C9.19" routerLinkActive="active">SPAC8C9.19</a></td>
<td></td>
<td>old join(3660430..3660438, 3660491..3660556, 3660607..3660713, 3660784..3660886), new join(3660430..3660438, 3660491..3660556, 3660607..3660717); deleted final exon, unsupported, improved length and alignment to orthologs</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC9G1.13c" routerLinkActive="active">SPAC9G1.13c</a></td>
<td></td>
<td>old complement(join(1995208..1996148, 1996325..1996413, 1996478..1996683)), new complement(join(1995208..1996148, 1996191..1996268, 1996325..1996413, 1996478..1996683)); added a new 3rd exon, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.03" routerLinkActive="active">SPAPB17E12.03</a></td>
<td></td>
<td>old join(1271889..1271972, 1272015..1272386, 1272483..1272962), new join(1271889..1271972, 1272015..1272962); removed in-frame splice, unsupported, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC15D4.13c" routerLinkActive="active">SPBC15D4.13c</a></td>
<td></td>
<td>old complement(join(3032492..3032710, 3032761..3032822, 3032865..3033222)), new complement(join(3032492..3032710, 3032761..3032822, 3032889..3033222)); better donor for first intron</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16C6.03c" routerLinkActive="active">SPBC16C6.03c</a></td>
<td></td>
<td>old complement(join(4332356..4332454, 4332746..4333123)), new complement(4332698..4333123); final exon, unsupported removed</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC19G7.06" routerLinkActive="active">SPBC19G7.06</a></td>
<td></td>
<td>old join(2359429..2359479 , 2359547..2359693 , 2359845..2359928 , 2360001..2361092), new join(2359429..2359479, 2359547..2359693, 2359845..2359928, 2360064..2361092); altered 3rd intron branch acceptor to agree with published</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC21.03c" routerLinkActive="active">SPBC21.03c</a></td>
<td></td>
<td>old complement(join(3217985..3218424, 3218467..3218491, 3218693..3218947)), new complement(join(3217985..3218424, 3218462..3218491, 3218580..3218651, 3218704..3218947)); changed 2 internal introns, now agrees with transread data, but still potential to be incorrect</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC32H8.08c" routerLinkActive="active">SPBC32H8.08c</a></td>
<td></td>
<td>old complement(join(1465811..1466035, 1466138..1467057, 1467311..1467456, 1467514..1467539)), new complement(join(1465811..1466035 , 1466138..1467057 , 1467311..1467431)); removed 5' intron/exon (unsupported) and trimmed to next met</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP35G2.06c" routerLinkActive="active">SPBP35G2.06c</a></td>
<td></td>
<td>old complement(join(973049..974181, 974223..974577, 974617..975002, 975044..975137, 975172..976632)), new complement(join(973049..974181, 974223..974577, 974647..975002, 975044..975137, 975172..976632)); changed donor for intron 3</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1393.06c" routerLinkActive="active">SPCC1393.06c</a></td>
<td></td>
<td>old complement(join(804627..805627, 805678..805800, 805907..806009)), new complement(join(804627..805627 , 805678..805800 , 805849..805921 , 805965..806009)); changed first intron and added a new 2nd exon, improved homology, also confirmed by transread data</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1450.14c" routerLinkActive="active">SPCC1450.14c</a></td>
<td></td>
<td>old complement(join(1759295..1760998, 1761117..1761128)), new complement(1759295..1760998); deleted first exon, unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1827.02c" routerLinkActive="active">SPCC1827.02c</a></td>
<td></td>
<td>old complement(join(2377039..2378078, 2378197..2378221)), new complement(2377039..2378127); deleted first exon, unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC63.10c" routerLinkActive="active">SPCC63.10c</a></td>
<td></td>
<td>old complement(join(854651..855695, 855766..855800, 855839..855926, 855984..856213)), new complement(join(854651..855926, 855984..856213)); deleted 2nd and 3rd introns, were not required or supported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC6F6.16c" routerLinkActive="active">SPAC6F6.16c</a></td>
<td></td>
<td>merged with SPAC6F6.18c and five introns added</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18535244">PMID:18535244</a></td>
<td><span class="no-break">2008-06-09</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC962.05" routerLinkActive="active">SPCC962.05</a></td>
<td></td>
<td>old join(554521..554722, 554792..556200), new join(554521..554744, 554865..556200)</td>
<td>pers. comm. Matthew O'Connell</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC338.08" routerLinkActive="active">SPCC338.08</a></td>
<td>ctp1</td>
<td>old coordinates complement(1358962..1359819), new complement(join(1358885..1359082, 1359121..1359819)); further altered to complement(join(1358891..1359082, 1359127..1359819)) in accordance with paper</td>
<td>pers. comm. Paul Russell; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18378696">PMID:18378696</a></td>
<td><span class="no-break">2008-01-11</span>;<span class="no-break">2008-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC2G2.09c" routerLinkActive="active">SPBC2G2.09c</a></td>
<td>crs1</td>
<td>added new C-terminal exon, deleted exon 3452822..3452877, extended intron 2 by 4 nucleotides</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-09</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC31F10.08" routerLinkActive="active">SPBC31F10.08</a></td>
<td>mde2</td>
<td>old coordinates 3764820..3765449, new join(3764820..3765437, 3765491..3765709)</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-18</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC651.06" routerLinkActive="active">SPBC651.06</a></td>
<td>mug166</td>
<td>merged with SPBC651.07</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-26</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC18E5.14c" routerLinkActive="active">SPBC18E5.14c</a></td>
<td></td>
<td>merged with SPBC18E5.09c</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-31</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC959.05c" routerLinkActive="active">SPAC959.05c</a></td>
<td></td>
<td>merged with SPAC959.06c; splicing is tentative</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-08-15</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC11B10.10c" routerLinkActive="active">SPBC11B10.10c</a></td>
<td>pht1</td>
<td>N terminal shortened to 2nd methionine; starting coordinate changed from 1503157 to 1503064</td>
<td>pers. comm. L Buchanan and F Stewart</td>
<td><span class="no-break">2008-09-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC27D7.12c" routerLinkActive="active">SPAC27D7.12c</a></td>
<td>but1</td>
<td>extended N-terminal to use first MET as start</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14623327">PMID:14623327</a></td>
<td><span class="no-break">2008-11-11</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC962.05" routerLinkActive="active">SPCC962.05</a></td>
<td>ast1</td>
<td>intron position moved</td>
<td>pers comm. M. O' Connell</td>
<td><span class="no-break">2008-12-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB1E7.14" routerLinkActive="active">SPAPB1E7.14</a></td>
<td>iec5</td>
<td>old join(3322106..3322331, 3322379..3322512); new join(3321979..3322018, 3322071..3322331, 3322379..3322512)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19040720">PMID:19040720</a></td>
<td><span class="no-break">2008-12-18</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC31G5.12c" routerLinkActive="active">SPAC31G5.12c</a></td>
<td>maf1</td>
<td>old complement(join(3008184..3008795, 3008914..3008957, 3009044..3009089)); new complement(join(3008184..3008795, 3009160..3009258, 3009312..3009317))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15590667">PMID:15590667</a></td>
<td><span class="no-break">2008-12-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC725.12" routerLinkActive="active">SPBC725.12</a></td>
<td>nbl1</td>
<td>old join(1227762..1227876, 1227926..1228098); new join(1227762..1228094, 1228166..1228307)</td>
<td>pers. comm. Kathy Gould</td>
<td><span class="no-break">2009-01-30</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC14C8.09c" routerLinkActive="active">SPBC14C8.09c</a></td>
<td></td>
<td>merged with SPBC14C8.08c; frameshifted</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC23G7.06c" routerLinkActive="active">SPBC23G7.06c</a></td>
<td></td>
<td>frameshifted; now a single exon</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC13E7.01" routerLinkActive="active">SPBC13E7.01</a></td>
<td></td>
<td>frameshifted</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC29A3.08" routerLinkActive="active">SPBC29A3.08</a></td>
<td></td>
<td>frameshifted; truncated to 2048336..2049892, 2049894..2049896</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1A4.06c" routerLinkActive="active">SPBC1A4.06c</a></td>
<td></td>
<td>frameshifted; C term exon changed from 1987076..1987675 to 1987044....1987675</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F3.11c" routerLinkActive="active">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>frameshifted; join(682874..683034, 683102..683109, 683160..683446) to join(682874..682994, 682996..683109, 683160..683446)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1F8.07c" routerLinkActive="active">SPAC1F8.07c</a></td>
<td></td>
<td>frameshifted; complement(101760..103544) to complement(join(101836..101872, 101872..103544))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1442.04c" routerLinkActive="active">SPCC1442.04c</a></td>
<td></td>
<td>frameshifted; complement(1774200..1775429) to complement(join(1773925..1774239, 1774236..1775429))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC548.04" routerLinkActive="active">SPCC548.04</a></td>
<td></td>
<td>final exon 223687..223852 to 223687..223796, 223853.223860</td>
<td>pers. comm. Marc Feuermann, Uniprot (SIB)</td>
<td><span class="no-break">2009-02-23</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP23A10.07" routerLinkActive="active">SPBP23A10.07</a></td>
<td>rpa2</td>
<td>truncated N-term to agree with orthologs; 2007960 -&gt; 2008119</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2009-03-30</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16E9.16c" routerLinkActive="active">SPBC16E9.16c</a></td>
<td>lsd90</td>
<td>gene structure revised to remove introns, genome sequence contains 2 frameshifts</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18079165">PMID:18079165</a></td>
<td><span class="no-break">2008-01-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC800.02" routerLinkActive="active">SPBC800.02</a></td>
<td>whi5</td>
<td>increased intron by 42bp; old coordinates 253148..253231, 253310..253984, new coordinates 253148..253231, 253352..253984</td>
<td>pers. comm. Rob de Bruin</td>
<td><span class="no-break">2009-07-27</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC18H10.08c" routerLinkActive="active">SPBC18H10.08c</a></td>
<td>ubp4</td>
<td>missing 2 N-terminal exons; old - complement(1784219..1785535); new - complement(join(1784219..1785748, 1785937..1786080, 1786141..1786248))</td>
<td>pers. comm. I. Kouranti</td>
<td><span class="no-break">2009-09-10</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC800.02" routerLinkActive="active">SPBC800.02</a></td>
<td>whi5</td>
<td>increased intron by 42bp; old coordinates 253148..253231, 253310..253984, new coordinates 253148..253231, 253352..253984</td>
<td>pers. comm. Rob de Bruin</td>
<td><span class="no-break">2009-07-27</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC2A9.07c" routerLinkActive="active">SPBC2A9.07c</a></td>
<td></td>
<td>removed intron</td>
<td>pers. comm. Cathrine Arnason Boe</td>
<td><span class="no-break">2009-09-10</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPNCRNA.445" routerLinkActive="active">SPNCRNA.445</a></td>
<td>snoR61</td>
<td>updated to extend 3' and added splice and identified as snR6</td>
<td>pers. comm. J. Matthews</td>
<td><span class="no-break">2008-08-08</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29B12.08" routerLinkActive="active">SPAC29B12.08</a></td>
<td></td>
<td>replaced exon 5428786..5428883 with 5429038.. 5429045</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2009-12-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC458.04c" routerLinkActive="active">SPAC458.04c</a></td>
<td></td>
<td>coordinates updated from 4737605..4738147, 4738189..4738283, 4738330..4738714 to 4737605..4738147, 4738189..4738283, 4738330..4738740, 4738779..4738854</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-02-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC5E4.10c" routerLinkActive="active">SPCC5E4.10c</a></td>
<td></td>
<td>coordinates updated from 651622..651905 , 651973..651994 to 651622..651905, 651973..652045, 652115..652225, 652280..652371, 652419..652455</td>
<td>pers. comm. K. Gould</td>
<td><span class="no-break">2010-03-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC26H5.02c" routerLinkActive="active">SPAC26H5.02c</a></td>
<td>nse2</td>
<td>second intron extended to 115 nt</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-08-20</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC27F1.09c" routerLinkActive="active">SPAC27F1.09c</a></td>
<td>prp10</td>
<td>added intron to match fully spliced</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9837997">PMID:9837997</a></td>
<td><span class="no-break">2010-09-27</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22G7.04" routerLinkActive="active">SPAC22G7.04</a></td>
<td>ubp13</td>
<td>coordinates updated to join(732880..732897, 732942..733040, 733085..733251, 733369..733495, 733536..734113, 734151..735457, 735495..736465)</td>
<td>pers. comm. James Iben</td>
<td><span class="no-break">2010-10-20</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC3H1.04c" routerLinkActive="active">SPAC3H1.04c</a></td>
<td>mdm31</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC12D12.09" routerLinkActive="active">SPBC12D12.09</a></td>
<td>rev7</td>
<td></td>
<td>pers. comm. Nicole Kosarek</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1348.12" routerLinkActive="active">SPBC1348.12</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC1778.02" routerLinkActive="active">SPBC1778.02</a></td>
<td>rap1</td>
<td></td>
<td>pers. comm. Yasushi Hiraoka</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC3E7.04c" routerLinkActive="active">SPBC3E7.04c</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC8D2.07c" routerLinkActive="active">SPBC8D2.07c</a></td>
<td>sfc9</td>
<td></td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPB21E7.02c" routerLinkActive="active">SPBPB21E7.02c</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC306.10" routerLinkActive="active">SPCC306.10</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC622.17" routerLinkActive="active">SPCC622.17</a></td>
<td>apn1</td>
<td>added 2 5' exons</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21193357">PMID:21193357</a></td>
<td><span class="no-break">2011-01-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC823.04" routerLinkActive="active">SPAC823.04</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC688.11" routerLinkActive="active">SPAC688.11</a></td>
<td>end4</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29A4.03c" routerLinkActive="active">SPAC29A4.03c</a></td>
<td>mrps9</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC688.04c" routerLinkActive="active">SPAC688.04c</a></td>
<td>gst3</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.14c" routerLinkActive="active">SPAPB17E12.14c</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC29A10.17" routerLinkActive="active">SPBC29A10.17</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP8B7.31" routerLinkActive="active">SPBP8B7.31</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC651.06" routerLinkActive="active">SPBC651.06</a></td>
<td>mug166</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC25H2.11c" routerLinkActive="active">SPBC25H2.11c</a></td>
<td>spt7</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29A4.03c" routerLinkActive="active">SPAC29A4.03c</a></td>
<td>mrps9</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC4G3.12c" routerLinkActive="active">SPCC4G3.12c</a></td>
<td>not2</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1322.16" routerLinkActive="active">SPCC1322.16</a></td>
<td>phb2</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC13E7.01" routerLinkActive="active">SPBC13E7.01</a></td>
<td>cwf22</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC4F6.10" routerLinkActive="active">SPBC4F6.10</a></td>
<td>vps901</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16H5.12c" routerLinkActive="active">SPBC16H5.12c</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC3A11.03" routerLinkActive="active">SPAC3A11.03</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC1711.10c" routerLinkActive="active">SPBC1711.10c</a></td>
<td>npl4</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1709.12" routerLinkActive="active">SPBC1709.12</a></td>
<td>rid1</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC13F5.07c" routerLinkActive="active">SPAC13F5.07c</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC18B11.05" routerLinkActive="active">SPAC18B11.05</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC18B11.05" routerLinkActive="active">SPAC18B11.05</a></td>
<td>gpi18</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPACUNK4.14" routerLinkActive="active">SPACUNK4.14</a></td>
<td>mdb1</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC947.10" routerLinkActive="active">SPBC947.10</a></td>
<td>dsc1</td>
<td>N-terminal extended to use upstream methionine</td>
<td>pers. comm. E. Stewart</td>
<td><span class="no-break">2011-02-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC22F3.11c" routerLinkActive="active">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>gene structure updated</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/21511999">PMID:21511999</a></td>
<td><span class="no-break">2011-03-11</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1006.03c" routerLinkActive="active">SPAC1006.03c</a></td>
<td>red1</td>
<td>intron donor corrected</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC29A3.06" routerLinkActive="active">SPBC29A3.06</a></td>
<td></td>
<td>N terminal extended</td>
<td></td>
<td><span class="no-break">2012-01-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1281.07c" routerLinkActive="active">SPCC1281.07c</a></td>
<td></td>
<td>N terminal extended by 24 amino acids</td>
<td></td>
<td><span class="no-break">2012-01-05</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC4B3.05c" routerLinkActive="active">SPCC4B3.05c</a></td>
<td>hem12</td>
<td>N terminal extended by 4 amino acids based on homology</td>
<td></td>
<td><span class="no-break">2012-01-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC2F3.13c" routerLinkActive="active">SPAC2F3.13c</a></td>
<td></td>
<td>removed N terminal region overlapping with plp1</td>
<td></td>
<td><span class="no-break">2012-12-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2F3.13c" routerLinkActive="active">SPAC9G1.15c</a></td>
<td>mzt1</td>
<td>N-terminal shortened to use downstream methionine</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/23885124">PMID:23885124</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24006493">PMID:24006493</a></td>
<td><span class="no-break">2013-11-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC25B2.07c" routerLinkActive="active">SPBC25B2.07c</a></td>
<td>mmb1</td>
<td>N-terminal shortened to use downstream methionine</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/21856157">PMID:21856157</a></td>
<td><span class="no-break">2013-12-06</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC3D6.04c" routerLinkActive="active">SPBC3D6.04c</a></td>
<td>mad1</td>
<td>N-terminal shortened to use downstream methionine; removed 13 amino acids</td>
<td>pers. comm. Silke Hauf</td>
<td><span class="no-break">2013-12-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC5H10.06c" routerLinkActive="active">SPAC5H10.06c</a></td>
<td>adh4</td>
<td>N-terminal shortened to use downstream methionine; removed 43 amino acids; UTR exon annotated</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24003116">PMID:24003116</a></td>
<td><span class="no-break">2014-01-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPNCRNA.98" routerLinkActive="active">SPNCRNA.98</a></td>
<td>srp7</td>
<td>Coordinates changed from 4268566-4269064 to 4268662-4268916, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2837764">PMID:2837764</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16453822">PMID:16453822</a></td>
<td><span class="no-break">2014-06-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.01" routerLinkActive="active">SPSNRNA.01</a></td>
<td>snu1</td>
<td>Coordinates changed from 3019819-3020472 to 3020205-3020353, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2188102">PMID:2188102</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPSNRNA.02" routerLinkActive="active">SPSNRNA.02</a></td>
<td>snu2</td>
<td>Coordinates changed from 959770-959305 to 959856-959370, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/3244367">PMID:3244367</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.03" routerLinkActive="active">SPSNRNA.03</a></td>
<td>snu3</td>
<td>Coordinates changed from 988266-987631 to 987825-987570, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/3194197">PMID:3194197</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPSNRNA.04" routerLinkActive="active">SPSNRNA.04</a></td>
<td>snu4</td>
<td>Coordinates changed from 467025-467823 to 467233-4683361, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2795654">PMID:2795654</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.05" routerLinkActive="active">SPSNRNA.05</a></td>
<td>snu5</td>
<td>Coordinates changed from 3236617-3237051 to 3236867-3236986, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2587274">PMID:2587274</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPSNRNA.06" routerLinkActive="active">SPSNRNA.06</a></td>
<td>snu6</td>
<td>Coordinates changed from (2562559-2562374,​2562323-2562085) to (2562276-2562323,​2562374-2562427), to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2909894">PMID:2909894</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.07" routerLinkActive="active">SPSNRNA.07</a></td>
<td>snu32</td>
<td>Coordinates changed from 3959174-3958768 to 3959086-3958832, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/1560765">PMID:1560765</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC110.05" routerLinkActive="active">SPAC110.05</a></td>
<td></td>
<td>Coordinates changed from complement(1905657..1905884) to complement(1905728..1905820) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F3.04" routerLinkActive="active">SPAC22F3.04</a></td>
<td>mug62</td>
<td>Coordinates changed from complement(join(698033..698177, 698223..698441, 698480..698841, 698884..699104, 699148..699363, 699410..702533)) to complement(join(698033..698177, 698223..698441, 698480..698841, 698884..699104, 699148..699363, 699410..702557, 702603..702755)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1D4.08" routerLinkActive="active">SPAC1D4.08</a></td>
<td>pis1</td>
<td>Coordinates changed from join(650814..650893,​651173..651848) to join(650488..650545,​651193..651848) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC839.02" routerLinkActive="active">SPBC839.02</a></td>
<td></td>
<td>Coordinates changed from join(597611..599124,​599256..599334) to 597611..599125 based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC823.02" routerLinkActive="active">SPAC823.02</a></td>
<td></td>
<td>Coordinates changed from join(2583804..2583902,​2583941..2583973) to join(2583863..2583902,​2583941..2584047) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437"></a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC146.01" routerLinkActive="active">SPBC146.01</a></td>
<td>med15</td>
<td>Coordinates changed from join(996843..997019,​997066..1000080) to join(996797..996870,​996917..997019,​997066..1000080) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC27B12.10c" routerLinkActive="active">SPBC27B12.10c</a></td>
<td>tom7</td>
<td>Coordinates changed from complement(join(1344032..1344157,​1344214..1344237)) to complement(join(1343913..1343934,​1343989..1344025,​1344082..1344157,​1344214..1344237)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC162.07" routerLinkActive="active">SPCC162.07</a></td>
<td>ent1</td>
<td>Coordinates changed from complement(join(1571648..1573733,​1574016..1574050)) to complement(join(1571648..1573733,​1574258..1574280)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC417.03" routerLinkActive="active">SPCC417.03</a></td>
<td></td>
<td>Coordinates changed from join(1672003..1672051,​1672111..1672364) to join(1672003..1672051,​1672243..1672332) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC320.09" routerLinkActive="active">SPCC320.09</a></td>
<td>hem15</td>
<td>Coordinates changed from complement(join(149153..150163,​150279..150323,​150374..150472)) to complement(join(149153..150163,​150279..150323,​150374..150589)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1486.05" routerLinkActive="active">SPAC1486.05</a></td>
<td>nup189</td>
<td>Coordinates changed from join(3197028..3197443,​3197530..3202450) to join(3197028..3197528,​3197528..3202450)</td>
<td>pers. comm. Y. Hiraoka</td>
<td><span class="no-break">2015-04-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC13G1.04c" routerLinkActive="active">SPBC13G1.04c</a></td>
<td>abh1</td>
<td>N-terminal shortened to use downstream methionine</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22365419">PMID:22365419</a></td>
<td><span class="no-break">2015-10-13</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC29A4.23" routerLinkActive="active">SPAC29A4.23</a></td>
<td></td>
<td>Frameshifted; intron/exon boundary changes; coordinates changed to join(5139024..5139190,​5139192..5139424,​5139427..5139575)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26494834">PMID:26494834</a></td>
<td><span class="no-break">2015-11-28</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F3.11c" routerLinkActive="active">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>Coordinates changed from complement(890220..890546) to join(682874..682996,​682998..683109,​683160..683446) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22365419">PMID:22365419</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2016-03-17</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC21B10.12" routerLinkActive="active">SPBC21B10.12</a></td>
<td>rec6</td>
<td>Coordinates changed from complement(join(1649038..1649472,​1649558..1649671)) to complement(join(1649038..1649472,​1649558..1649727,​1649815..1649878,​1650043..1650156))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26917764">PMID:26917764</a></td>
<td><span class="no-break">2016-04-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC3A12.04c" routerLinkActive="active">SPAC3A12.04c</a></td>
<td>rpp1</td>
<td>Frameshifted by Chr_I:1424708!CA→C; coordinates changed from complement(join(1424697..1425161,​1425231..1425329,​1425372..1425430,​1425478..1425562)) to complement(join(1424660..1424708,​1424710..1425161,​1425231..1425329,​1425372..1425430,​1425478..1425562))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAP27G11.10c" routerLinkActive="active">SPAP27G11.10c</a></td>
<td>nup184</td>
<td>Frameshifted by Chr_I:1625092!T→TC; coordinates changed from complement(join(1625076..1629753,​1629825..1629841)) to complement(join(1624885..1625092,​1625092..1629753,​1629825..1629841))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC17G8.01c" routerLinkActive="active">SPAC17G8.01c</a></td>
<td>trl1</td>
<td>Frameshifted by Chr_I:2343703!G→GA; coordinates changed from complement(join(2341346..2343706,​2343754..2343756)) to complement(join(2341346..2343703,​2343703..2343756))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC22F3.11c" routerLinkActive="active">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>Frameshifted by Chr_I:682993!TC-&gt;T; coordinates changed to join(682874..682996,​682998..683109,​683160..683446)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC823.04" routerLinkActive="active">SPAC823.04</a></td>
<td>rrp36</td>
<td>Frameshifted by Chr_I:2588066!C→CA and Chr_I:2588021!C→CA; coordinates changed from join(2587729..2587897,​2587952..2588028,​2588031..2588143,​2588182..2588620) to join(2587729..2587897,​2587952..2588025,​2588025..2588066,​2588066..2588620)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC688.08" routerLinkActive="active">SPAC688.08</a></td>
<td>srb8</td>
<td>Frameshifted by Chr_I:3125118!A→AT; coordinates changed from join(3123163..3125057,​3125099..3126310,​3126356..3126653) to join(3123163..3125118,​3125118..3126310,​3126356..3126653)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC1486.05" routerLinkActive="active">SPAC1486.05</a></td>
<td>nup189</td>
<td>Frameshifted by Chr_I:3197528!A→AG; coordinates changed from join(3197028..3197529,​3197529..3202450) to join(3197028..3197529,​3197529..3202450)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC3A11.09" routerLinkActive="active">SPAC3A11.09</a></td>
<td>sod22</td>
<td>Frameshifted by Chr_I:3450130!GT→G; coordinates changed from complement(join(3450114..3452269,​3452485..3452608)) to complement(join(3450056..3450130,​3450132..3452269,​3452485..3452608))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC1071.01c" routerLinkActive="active">SPAC1071.01c</a></td>
<td>pta1</td>
<td>Frameshifted by Chr_I:3855790!GT→G; coordinates changed from complement(3855780..3857792) to complement(join(3855626..3855790,​3855792..3857792))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC29E6.03c" routerLinkActive="active">SPAC29E6.03c</a></td>
<td>uso1</td>
<td>Frameshifted by Chr_I:4407494!T→TG; coordinates changed from complement(join(4405990..4407375,​4407447..4409096,​4409171..4409338)) to complement(join(4405990..4407494,​4407494..4409096,​4409171..4409338))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC29E6.04" routerLinkActive="active">SPAC29E6.04</a></td>
<td>nnf1</td>
<td>Frameshifted by Chr_I:4410191!CG→C; coordinates changed from join(4409773..4410192, 4410194..4410391) to join(4409773..4410190,​4410192..4410391)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC29A4.03c" routerLinkActive="active">SPAC29A4.03c</a></td>
<td>mrps9</td>
<td>Frameshifted by Chr_I:5142627!A→AG; coordinates changed from join(5142407..5142628,​5142631..5143224) to join(5142407..5142626,​5142626..5143224)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC16E9.16c" routerLinkActive="active">SPBC16E9.16c</a></td>
<td>lsd90</td>
<td>Frameshifted by Chr_II:1948953!GA→G and Chr_II:1950050!A→AG; coordinates changed from complement(join(1947985..1948953,​1948955..1950050,​1950053..1950255)) to complement(join(1947985..1948952,​1948954..1950048,​1950048..1950255))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPBC1E8.03c" routerLinkActive="active">SPBC1E8.03c</a></td>
<td></td>
<td>Frameshifted by Chr_II:1960392!A→AG; coordinates changed from complement(1960384..1961817) to complement(join(1960166..1960392,​1960392..1961817))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC1A4.06c" routerLinkActive="active">SPBC1A4.06c</a></td>
<td>tam41</td>
<td>Frameshifted by Chr_II:1987101!CG→C and Chr_II:1987117!TG→T; coordinates changed from complement(join(1987044..1987099,​1987102..1987675,​1987737..1988015,​1988081..1988353)) to complement(join(1987044..1987100,​1987102..1987117,​1987119..1987675,​1987737..1988015,​1988081..1988353))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPBC29A3.08" routerLinkActive="active">SPBC29A3.08</a></td>
<td>pof4</td>
<td>Frameshifted by Chr_II:2053516!G→GC; coordinates changed from join(2053033..2053517,​2053519..2053600) to join(2053033..2053517,​2053517..2053823)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC23G7.06c" routerLinkActive="active">SPBC23G7.06c</a></td>
<td></td>
<td>Frameshifted by Chr_II:2108180!T→TA; coordinates changed from complement(join(2106448..2108178,​2108181..2108786)) to complement(join(2106448..2108180,​2108180..2108786))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPBC14C8.09c" routerLinkActive="active">SPBC14C8.09c</a></td>
<td></td>
<td>Frameshifted by Chr_II:2219928!A→AT; coordinates changed from complement(join(2219430..2219934,​2219937..2220322)) to complement(join(2219430..2219928,​2219928..2220322))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC4F6.10" routerLinkActive="active">SPBC4F6.10</a></td>
<td>vps901</td>
<td>Frameshifted by Chr_II:2709414!G→GC; coordinates changed from join(2708076..2709415,​2709418..2709793) to join(2708076..2709414,​2709414..2709793)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPBC32F12.08c" routerLinkActive="active">SPBC32F12.08c</a></td>
<td>duo1</td>
<td>Frameshifted by Chr_II:2798040!CT→C; coordinates changed from complement(2798007..2798507) to complement(join(2797652..2798040,​2798042..2798507))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC13E7.01" routerLinkActive="active">SPBC13E7.01</a></td>
<td>cwf22</td>
<td>Frameshifted by Chr_II:3040332!C→CG; coordinates changed from join(3037988..3040331,​3040334..3040650) to join(3037988..3040332,​3040332..3040650)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPBC16D10.10" routerLinkActive="active">SPBC16D10.10</a></td>
<td>tad2</td>
<td>Frameshifted by Chr_II:3619003!A→AG; coordinates changed from join(3618117..3618179,​3618228..3618617,​3618658..3618702,​3618747..3618918,​3618956..3619071,​3619113..3619321,​3619366..3619540) to join(3618117..3618179,​3618228..3618617,​3618658..3618702,​3618747..3618920,​3618968..3619003,​3619003..3619071,​3619113..3619321,​3619366..3619540)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC12B10.09" routerLinkActive="active">SPAC12B10.09</a></td>
<td>pet801</td>
<td>Coordinates changed from join(4588250..4588510,​4588567..4588723,​4588869..4589323,​4589410..4589574) to join(4588487..4588510,​4588567..4588723,​4588869..4589323,​4589410..4589574)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC3A11.06" routerLinkActive="active">SPAC3A11.06</a></td>
<td>mvp1</td>
<td>Frameshifted by Chr_I:3460318!T→T; coordinates changed from complement(join(3459233..3459436,​3459492..3459665,​3459705..3459897,​3459941..3460185,​3460234..3460364,​3460425..3460442,​3460508..3460560,​3460642..3461263,​3461323..3461583,​3461766..3461859)) to complement(join(3459233..3459436,​3459492..3459665,​3459705..3459897,​3459941..3460185,​3460234..3460318,​3460318..3460372,​3460425..3460442,​3460508..3460560,​3460642..3461263,​3461323..3461583,​3461766..3461859))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC530.13" routerLinkActive="active">SPBC530.13</a></td>
<td>lsc1</td>
<td>Frameshifted by Chr_II:825012!G→A; coordinates changed from join(824374..824534,​824579..824660,​824874..825011,​825065..825188,​825245..825747) to join(824374..824534,​824579..824660,​824874..824999,​825065..825188,​825245..825747)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-17</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPSNORNA.41" routerLinkActive="active">SPSNORNA.41</a></td>
<td>snR46</td>
<td>Switched from + strand to - strand</td>
<td>pers. comm. Francois Bachand</td>
<td><span class="no-break">2017-05-25</span></td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'index'" class="docs-content">
<h1 id="genome-status">Genome Status</h1>
<table>
<tbody>
<tr class="odd">
<td><a routerLink="/status/statistics" routerLinkActive="active">Genome Statistics</a></td>
<td>Latest information on the status of the genome</td>
</tr>
<tr class="even">
<td><a routerLink="/status/sequencing-status" routerLinkActive="active">Sequencing Status</a></td>
<td>Contig size, gap status and progress, including <a routerLink="/status/centromeres" routerLinkActive="active">centromeres</a> and <a routerLink="/status/telomeres" routerLinkActive="active">telomeres</a></td>
</tr>
<tr class="odd">
<td><a routerLink="/status/sequencing-updates" routerLinkActive="active">Sequence Updates</a></td>
<td>Changes to the genome sequence since 07/03.</td>
</tr>
<tr class="even">
<td><a routerLink="/status/sequence-updates-pending" routerLinkActive="active">Sequence Updates Pending</a></td>
<td>Pending changes, mainly from the Broad Institute, some supported by data from other sources</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/gene-coordinate-changes" routerLinkActive="active">Gene Coordinate Changes</a></td>
<td>Changes to coordinates of individual genes since publication</td>
</tr>
<tr class="even">
<td><a routerLink="/status/new-and-removed-genes" routerLinkActive="active">New and Removed Genes</a></td>
<td>Genes identified or removed since publication</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/gene-characterisation" routerLinkActive="active">Gene Characterisation</a></td>
<td>Current counts of protein coding gene status, as published in small scale experiments</td>
</tr>
<tr class="even">
<td><a routerLink="/status/gene-characterisation-statistics-history" routerLinkActive="active">Historical Gene Characterisation</a></td>
<td>Changes in gene characterisation statistics over time</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/priority-unstudied-genes" routerLinkActive="active">Priority Unstudied Genes</a></td>
<td>Unstudied nuclear genes conserved 1:1 in human</td>
</tr>
<tr class="even">
<td><a routerLink="/status/unmapped_genes" routerLinkActive="active">Unmapped Genes</a></td>
<td>List of genes identified genetically but not cloned or physically mapped</td>
</tr>
</tbody>
</table>
<p>Note: Many older <em>S. pombe</em> sequence submissions to the DNA databases (International Nucleotide Sequence Database Collaboration databases,​ i.e. ENA, GenBank, DDBJ) contain one or more errors, and we do not have the resources to maintain past sequences or flag every error in PomBase.</p>

  </div>
  <div *ngIf="pageName == 'mating-type-region'" class="docs-content">
<h1 id="mating-type-region">Mating type region</h1>
<p>The S. pombe mating type region is located on Chromosome 2 (for 972 h-, the coordinates are <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=II%3A2129208-2137121;site=ensemblunit">II:2129208-2137121</a>). This sequence represents a version of the mat2P-mat3M region in which mat2P and the cenH element are deleted.</p>
<p>A contig of the the h90 configuration of the mat2P-mat3M region was created by Xavier Marsellach and Lorena Aguilar (Azorín lab) using available data and <em>S. kambucha</em> as a scaffold. The contig can be viewed in the <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=MTR:1-20128">Ensembl genome&amp; browser</a>. Replacing the Chromosome 2 region spanning coordinates 2129208-2137121 with the separate contig sequence yields the Chromosome 2 contig of an h90 strain.</p>

  </div>
  <div *ngIf="pageName == 'new-and-removed-genes'" class="docs-content">
<h1 id="new-and-removed-genes">New and removed genes</h1>
<h3 id="new-genes-since-publication">New Genes Since Publication</h3>
<table>
 <thead>
  <tr>
   <th>Gene</th>
   <th>Description</th>
   <th>Date</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td><a routerLink="/gene/SPBC1709.20" routerLinkActive="active" [attr.title]="'SPBC1709.20'"><span class="gene-link">SPBC1709.20</span></a></td>
  <td>RNase P and RNase MRP subunit; identified in targeted search for <em>S. cerevisiae</em> POP8 ortholog</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC7D4.15c" routerLinkActive="active" [attr.title]="'SPAC7D4.15c'"><span class="gene-link">SPAC7D4.15c</span></a></td>
  <td>Oligosaccharyltransferase subunit (predicted)</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC119.18" routerLinkActive="active" [attr.title]="'SPBC119.18'"><span class="gene-link">SPBC119.18</span></a></td>
  <td>conserved hypothetical protein</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1751.04" routerLinkActive="active" [attr.title]="'SPAC1751.04'"><span class="gene-link">SPAC1751.04</span></a></td>
  <td>hypothetical protein; sequence orphan</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC330.19c" routerLinkActive="active" [attr.title]="'SPCC330.19c'"><span class="gene-link">SPCC330.19c</span></a></td>
  <td>hypothetical protein; sequence orphan; confirmed by EST</td>
  <td>Sept 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC959.10" routerLinkActive="active" [attr.title]="'SPAC959.10'"><span class="gene-link">SPAC959.10</span></a></td>
  <td>tRNA splicing endonuclease; similar to <em>S. cerevisiae</em> YMR059W</td>
  <td>Sept 2003</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC14C4.16" routerLinkActive="active" [attr.title]="'SPAC14C4.16'"><span class="gene-link">SPAC14C4.16</span></a></td>
  <td>hypothetical protein; sequence orphan; identified in targeted search for <em>S. cerevisiae</em> DAD3 ortholog</td>
  <td>Sept 2003</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC6B1.12c" routerLinkActive="active" [attr.title]="'SPBC6B1.12c'"><span class="gene-link">SPBC6B1.12c</span></a></td>
  <td>SAGA histone acetylase complex (predicted); involved in nuclear export (predicted); similar to <em>S. cerevisiae</em> YBR111W-A; identified in targeted search for <em>S. cerevisiae</em> SUS1 ortholog</td>
  <td>Feb 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC25H2.10c" routerLinkActive="active" [attr.title]="'SPBC25H2.10c'"><span class="gene-link">SPBC25H2.10c</span></a></td>
  <td>dihydropteroate synthase (predicted) (ISS); similar to <em>S. cerevisiae</em> YGL232W; updated from pseudogene to coding</td>
  <td>March 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC56F2.14" routerLinkActive="active" [attr.title]="'SPBC56F2.14'"><span class="gene-link">SPBC56F2.14</span></a></td>
  <td>ribosomal protein, mitochondrial L44 (predicted); similar to <em>S. cerevisiae</em> YMR225C; identified in targeted search for <em>S. cerevisiae</em> MRP144 ortholog</td>
  <td>March 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC106.05c" routerLinkActive="active" [attr.title]="'SPBC106.05c'"><span class="gene-link">SPBC106.05c</span></a></td>
  <td>proton-transporting ATP synthase complex, coupling factor F(o) (predicted); non-essential (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12618370">PMID:12618370</a>); similar to <em>S. cerevisiae</em> YDR322C-A; identified in targeted search for <em>S. cerevisiae</em> YDR322C-A ortholog</td>
  <td>March 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC19D5.11c" routerLinkActive="active" [attr.title]="'SPAC19D5.11c'"><span class="gene-link">SPAC19D5.11c</span></a></td>
  <td>similar to <em>S. cerevisiae</em> CTF8; pers. comm. Stuart MacNeil</td>
  <td>September 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC19D5.10c" routerLinkActive="active" [attr.title]="'SPAC19D5.10c'"><span class="gene-link">SPAC19D5.10c</span></a></td>
  <td>sequence orphan</td>
  <td>September 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC3A12.19" routerLinkActive="active" [attr.title]="'SPAC3A12.19'"><span class="gene-link">SPAC3A12.19</span></a></td>
  <td>mitochondrial ribosomal protein; identified in targeted search for <em>S. cerevisiae</em> YBR282W ortholog</td>
  <td>September 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC57A10.14" routerLinkActive="active" [attr.title]="'SPAC57A10.14'"><span class="gene-link">SPAC57A10.14</span></a></td>
  <td>SAGA complex subunit (predicted); identified in targeted search for <em>S. cerevisiae</em> SGF11 ortholog</td>
  <td>January 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC8C9.19" routerLinkActive="active" [attr.title]="'SPAC8C9.19'"><span class="gene-link">SPAC8C9.19</span></a></td>
  <td>conserved protein; similar to <em>D. hansenii</em> DEHA0C10780g</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC8C9.20" routerLinkActive="active" [attr.title]="'SPAC8C9.20'"><span class="gene-link">SPAC8C9.20</span></a></td>
  <td>sequence orphan</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1223.15c" routerLinkActive="active" [attr.title]="'SPCC1223.15c'"><span class="gene-link">SPCC1223.15c</span></a></td>
  <td>DASH complex subunit Spc19; pers. comm. Xiangwei He</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC3D6.16" routerLinkActive="active" [attr.title]="'SPBC3D6.16'"><span class="gene-link">SPBC3D6.16</span></a></td>
  <td>sequence orphan</td>
  <td>Added May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC3B9.22c" routerLinkActive="active" [attr.title]="'SPBC3B9.22c'"><span class="gene-link">SPBC3B9.22c</span></a></td>
  <td>DASH complex subunit Dad4 (predicted); pers. comm. Xiangwei He</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC16E9.16c" routerLinkActive="active" [attr.title]="'SPBC16E9.16c'"><span class="gene-link">SPBC16E9.16c</span></a></td>
  <td>sequence orphan; previously annotated as pseudogene; two peptides in mass spec analysis (pers. comm. Dieter Wolf)</td>
  <td>November 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC27E2.12" routerLinkActive="active" [attr.title]="'SPAC27E2.12'"><span class="gene-link">SPAC27E2.12</span></a></td>
  <td>dubious</td>
  <td>Added February 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1347.14c" routerLinkActive="active" [attr.title]="'SPBC1347.14c'"><span class="gene-link">SPBC1347.14c</span></a></td>
  <td>dubious</td>
  <td>February 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC16C4.21" routerLinkActive="active" [attr.title]="'SPAC16C4.21'"><span class="gene-link">SPAC16C4.21</span></a></td>
  <td>dubious</td>
  <td>June 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC6B12.18" routerLinkActive="active" [attr.title]="'SPAC6B12.18'"><span class="gene-link">SPAC6B12.18</span></a></td>
  <td>sequence orphan</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.17c" routerLinkActive="active" [attr.title]="'SPAC222.17c'"><span class="gene-link">SPAC222.17c</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC27E2.13" routerLinkActive="active" [attr.title]="'SPAC27E2.13'"><span class="gene-link">SPAC27E2.13</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC417.15" routerLinkActive="active" [attr.title]="'SPCC417.15'"><span class="gene-link">SPCC417.15</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1250.07" routerLinkActive="active" [attr.title]="'SPAC1250.07'"><span class="gene-link">SPAC1250.07</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1250.07" routerLinkActive="active" [attr.title]="'SPAC1250.07'"><span class="gene-link">SPAC1250.07</span></a></td>
  <td>TFIIIC subunit Tfc7; from PMID:17409385</td>
  <td>April 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1393.14" routerLinkActive="active" [attr.title]="'SPCC1393.14'"><span class="gene-link">SPCC1393.14</span></a></td>
  <td>ten1; nuclear telomere cap complex subunit; pers. comm. Paul Russell</td>
  <td>July 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAPB1A10.16" routerLinkActive="active" [attr.title]="'SPAPB1A10.16'"><span class="gene-link">SPAPB1A10.16</span></a></td>
  <td>conserved fungal protein; created from the C-term exon of SPAPB1A10.03</td>
  <td>December 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC569.09" routerLinkActive="active" [attr.title]="'SPCC569.09'"><span class="gene-link">SPCC569.09</span></a></td>
  <td>sequence orphan; longest reading frome in solexa transcript SPNG2623</td>
  <td>December 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.214" routerLinkActive="active" [attr.title]="'SPNCRNA.214'"><span class="gene-link">SPNCRNA.214</span></a></td>
  <td>telomerase RNA (predicted)</td>
  <td>January 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1183.12" routerLinkActive="active" [attr.title]="'SPCC1183.12'"><span class="gene-link">SPCC1183.12</span></a></td>
  <td>previously SPNCRNA.546; splice site not spotted because it was outside the coordinate range of the feature; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
  <td>April 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC16C6.14" routerLinkActive="active" [attr.title]="'SPBC16C6.14'"><span class="gene-link">SPBC16C6.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
  <td>April 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC70.12c" routerLinkActive="active" [attr.title]="'SPCC70.12c'"><span class="gene-link">SPCC70.12c</span></a></td>
  <td>extender of the chronological lifespan protein Ecl1; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18422613">PMID:18422613</a></td>
  <td>April 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1442.19" routerLinkActive="active" [attr.title]="'SPCC1442.19'"><span class="gene-link">SPCC1442.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.17" routerLinkActive="active" [attr.title]="'SPAC222.17'"><span class="gene-link">SPAC222.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC569.09" routerLinkActive="active" [attr.title]="'SPCC569.09'"><span class="gene-link">SPCC569.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAPB1E7.14" routerLinkActive="active" [attr.title]="'SPAPB1E7.14'"><span class="gene-link">SPAPB1E7.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC688.16" routerLinkActive="active" [attr.title]="'SPAC688.16'"><span class="gene-link">SPAC688.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAP4C9.02" routerLinkActive="active" [attr.title]="'SPAP4C9.02'"><span class="gene-link">SPAP4C9.02</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC16C4.22" routerLinkActive="active" [attr.title]="'SPCC16C4.22'"><span class="gene-link">SPCC16C4.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23D3.16" routerLinkActive="active" [attr.title]="'SPAC23D3.16'"><span class="gene-link">SPAC23D3.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1183.12" routerLinkActive="active" [attr.title]="'SPCC1183.12'"><span class="gene-link">SPCC1183.12</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC25H1.10C" routerLinkActive="active" [attr.title]="'SPAC25H1.10C'"><span class="gene-link">SPAC25H1.10C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1604.25" routerLinkActive="active" [attr.title]="'SPBC1604.25'"><span class="gene-link">SPBC1604.25</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23D3.17" routerLinkActive="active" [attr.title]="'SPAC23D3.17'"><span class="gene-link">SPAC23D3.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4G9.22" routerLinkActive="active" [attr.title]="'SPAC4G9.22'"><span class="gene-link">SPAC4G9.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC757.15" routerLinkActive="active" [attr.title]="'SPCC757.15'"><span class="gene-link">SPCC757.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC227.19C" routerLinkActive="active" [attr.title]="'SPAC227.19C'"><span class="gene-link">SPAC227.19C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC167.09" routerLinkActive="active" [attr.title]="'SPAC167.09'"><span class="gene-link">SPAC167.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC16C6.14" routerLinkActive="active" [attr.title]="'SPBC16C6.14'"><span class="gene-link">SPBC16C6.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC20G4.09" routerLinkActive="active" [attr.title]="'SPAC20G4.09'"><span class="gene-link">SPAC20G4.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.18" routerLinkActive="active" [attr.title]="'SPAC222.18'"><span class="gene-link">SPAC222.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC6F6.19" routerLinkActive="active" [attr.title]="'SPAC6F6.19'"><span class="gene-link">SPAC6F6.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC22F3.15" routerLinkActive="active" [attr.title]="'SPAC22F3.15'"><span class="gene-link">SPAC22F3.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC27E2.14" routerLinkActive="active" [attr.title]="'SPAC27E2.14'"><span class="gene-link">SPAC27E2.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23H4.21" routerLinkActive="active" [attr.title]="'SPAC23H4.21'"><span class="gene-link">SPAC23H4.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC57A7.15C" routerLinkActive="active" [attr.title]="'SPAC57A7.15C'"><span class="gene-link">SPAC57A7.15C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC12G12.17" routerLinkActive="active" [attr.title]="'SPAC12G12.17'"><span class="gene-link">SPAC12G12.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC29A10.17" routerLinkActive="active" [attr.title]="'SPBC29A10.17'"><span class="gene-link">SPBC29A10.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1556.06" routerLinkActive="active" [attr.title]="'SPAC1556.06'"><span class="gene-link">SPAC1556.06</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC26H8.16" routerLinkActive="active" [attr.title]="'SPBC26H8.16'"><span class="gene-link">SPBC26H8.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC70.12C" routerLinkActive="active" [attr.title]="'SPCC70.12C'"><span class="gene-link">SPCC70.12C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBP35G2.16c" routerLinkActive="active" [attr.title]="'SPBP35G2.16c'"><span class="gene-link">SPBP35G2.16c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19352039">PMID:19352039</a></td>
  <td>April 2009</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC8E4.12c" routerLinkActive="active" [attr.title]="'SPBC8E4.12c'"><span class="gene-link">SPBC8E4.12c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19352039">PMID:19352039</a></td>
  <td>April 2009</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1235.16" routerLinkActive="active" [attr.title]="'SPCC1235.16'"><span class="gene-link">SPCC1235.16</span></a></td>
  <td>pers. comm. Juan Mata</td>
  <td>April 2009</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC18.19C" routerLinkActive="active" [attr.title]="'SPCC18.19C'"><span class="gene-link">SPCC18.19C</span></a></td>
  <td>pers. comm. Val Wood; note first 2 exons may be incorrect, homology is within 3rd exon</td>
  <td>February 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC306.10" routerLinkActive="active" [attr.title]="'SPCC306.10'"><span class="gene-link">SPCC306.10</span></a></td>
  <td>pers. comm. Val Wood; pseudo-&gt; coding</td>
  <td>March 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.497" routerLinkActive="active" [attr.title]="'SPNCRNA.497'"><span class="gene-link">SPNCRNA.497</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.498" routerLinkActive="active" [attr.title]="'SPNCRNA.498'"><span class="gene-link">SPNCRNA.498</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.499" routerLinkActive="active" [attr.title]="'SPNCRNA.499'"><span class="gene-link">SPNCRNA.499</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.507" routerLinkActive="active" [attr.title]="'SPNCRNA.507'"><span class="gene-link">SPNCRNA.507</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.510" routerLinkActive="active" [attr.title]="'SPNCRNA.510'"><span class="gene-link">SPNCRNA.510</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.514" routerLinkActive="active" [attr.title]="'SPNCRNA.514'"><span class="gene-link">SPNCRNA.514</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.515" routerLinkActive="active" [attr.title]="'SPNCRNA.515'"><span class="gene-link">SPNCRNA.515</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.523" routerLinkActive="active" [attr.title]="'SPNCRNA.523'"><span class="gene-link">SPNCRNA.523</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.530" routerLinkActive="active" [attr.title]="'SPNCRNA.530'"><span class="gene-link">SPNCRNA.530</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.19" routerLinkActive="active" [attr.title]="'SPAC222.19'"><span class="gene-link">SPAC222.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC17G8.15" routerLinkActive="active" [attr.title]="'SPAC17G8.15'"><span class="gene-link">SPAC17G8.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1805.18" routerLinkActive="active" [attr.title]="'SPAC1805.18'"><span class="gene-link">SPAC1805.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1486.11" routerLinkActive="active" [attr.title]="'SPAC1486.11'"><span class="gene-link">SPAC1486.11</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC3H5.13" routerLinkActive="active" [attr.title]="'SPAC3H5.13'"><span class="gene-link">SPAC3H5.13</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC15A10.17" routerLinkActive="active" [attr.title]="'SPAC15A10.17'"><span class="gene-link">SPAC15A10.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4F10.22" routerLinkActive="active" [attr.title]="'SPAC4F10.22'"><span class="gene-link">SPAC4F10.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1B3.21" routerLinkActive="active" [attr.title]="'SPAC1B3.21'"><span class="gene-link">SPAC1B3.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPRRNA.56" routerLinkActive="active" [attr.title]="'SPRRNA.56'"><span class="gene-link">SPRRNA.56</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC9G1.15c" routerLinkActive="active" [attr.title]="'SPAC9G1.15c'"><span class="gene-link">SPAC9G1.15c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC13F5.08" routerLinkActive="active" [attr.title]="'SPAC13F5.08'"><span class="gene-link">SPAC13F5.08</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC6B12.19" routerLinkActive="active" [attr.title]="'SPAC6B12.19'"><span class="gene-link">SPAC6B12.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC3G9.17" routerLinkActive="active" [attr.title]="'SPAC3G9.17'"><span class="gene-link">SPAC3G9.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC16E8.18c" routerLinkActive="active" [attr.title]="'SPAC16E8.18c'"><span class="gene-link">SPAC16E8.18c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC926.10" routerLinkActive="active" [attr.title]="'SPAC926.10'"><span class="gene-link">SPAC926.10</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC19G12.17" routerLinkActive="active" [attr.title]="'SPAC19G12.17'"><span class="gene-link">SPAC19G12.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23A1.20" routerLinkActive="active" [attr.title]="'SPAC23A1.20'"><span class="gene-link">SPAC23A1.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1F7.14" routerLinkActive="active" [attr.title]="'SPAC1F7.14'"><span class="gene-link">SPAC1F7.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4D7.15" routerLinkActive="active" [attr.title]="'SPAC4D7.15'"><span class="gene-link">SPAC4D7.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4D7.14" routerLinkActive="active" [attr.title]="'SPAC4D7.14'"><span class="gene-link">SPAC4D7.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC839.20" routerLinkActive="active" [attr.title]="'SPBC839.20'"><span class="gene-link">SPBC839.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC409.23" routerLinkActive="active" [attr.title]="'SPBC409.23'"><span class="gene-link">SPBC409.23</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC8D2.23" routerLinkActive="active" [attr.title]="'SPBC8D2.23'"><span class="gene-link">SPBC8D2.23</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC3H7.18" routerLinkActive="active" [attr.title]="'SPBC3H7.18'"><span class="gene-link">SPBC3H7.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1711.18" routerLinkActive="active" [attr.title]="'SPBC1711.18'"><span class="gene-link">SPBC1711.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC24C6.13" routerLinkActive="active" [attr.title]="'SPBC24C6.13'"><span class="gene-link">SPBC24C6.13</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC32F12.16" routerLinkActive="active" [attr.title]="'SPBC32F12.16'"><span class="gene-link">SPBC32F12.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC30D10.21" routerLinkActive="active" [attr.title]="'SPBC30D10.21'"><span class="gene-link">SPBC30D10.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC887.22" routerLinkActive="active" [attr.title]="'SPBC887.22'"><span class="gene-link">SPBC887.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC839.19" routerLinkActive="active" [attr.title]="'SPBC839.19'"><span class="gene-link">SPBC839.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC530.16" routerLinkActive="active" [attr.title]="'SPBC530.16'"><span class="gene-link">SPBC530.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC14C8.19" routerLinkActive="active" [attr.title]="'SPBC14C8.19'"><span class="gene-link">SPBC14C8.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC17D1.17" routerLinkActive="active" [attr.title]="'SPBC17D1.17'"><span class="gene-link">SPBC17D1.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1105.19" routerLinkActive="active" [attr.title]="'SPBC1105.19'"><span class="gene-link">SPBC1105.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC56F2.15" routerLinkActive="active" [attr.title]="'SPBC56F2.15'"><span class="gene-link">SPBC56F2.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCP20C8.04" routerLinkActive="active" [attr.title]="'SPCP20C8.04'"><span class="gene-link">SPCP20C8.04</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC330.20" routerLinkActive="active" [attr.title]="'SPCC330.20'"><span class="gene-link">SPCC330.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC63.09" routerLinkActive="active" [attr.title]="'SPCC63.09'"><span class="gene-link">SPCC63.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC4B3.20" routerLinkActive="active" [attr.title]="'SPCC4B3.20'"><span class="gene-link">SPCC4B3.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC330.21" routerLinkActive="active" [attr.title]="'SPCC330.21'"><span class="gene-link">SPCC330.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC11D3.11c" routerLinkActive="active" [attr.title]="'SPAC11D3.11c'"><span class="gene-link">SPAC11D3.11c</span></a></td>
  <td>pseudo to coding</td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC713.14c" routerLinkActive="active" [attr.title]="'SPBC713.14c'"><span class="gene-link">SPBC713.14c</span></a></td>
  <td>identified by ribosome profiling (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24929437">PMID:24929437</a>)</td>
  <td>July 2014</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC823.17" routerLinkActive="active" [attr.title]="'SPAC823.17'"><span class="gene-link">SPAC823.17</span></a></td>
  <td>identified by ribosome profiling (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24929437">PMID:24929437</a>)</td>
  <td>July 2014</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC110.06" routerLinkActive="active" [attr.title]="'SPAC110.06'"><span class="gene-link">SPAC110.06</span></a></td>
  <td>identified by ribosome profiling (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24929437">PMID:24929437</a>)</td>
  <td>July 2014</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAPB1A11.06" routerLinkActive="active" [attr.title]="'SPAPB1A11.06'"><span class="gene-link">SPAPB1A11.06</span></a></td>
  <td>previously SPNCRNA.31 (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a>)</td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC110.06" routerLinkActive="active" [attr.title]="'SPAC110.06'"><span class="gene-link">SPAC110.06</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a></td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC29A4.23" routerLinkActive="active" [attr.title]="'SPAC29A4.23'"><span class="gene-link">SPAC29A4.23</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a></td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC110.06" routerLinkActive="active" [attr.title]="'SPAC110.06'"><span class="gene-link">SPAC110.06</span></a></td>
  <td>F1F0 ATPase synthase peripheral stalk assembly factor Ina17 (predicted) (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a>)</td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1688" routerLinkActive="active" [attr.title]="'SPNCRNA.1688'"><span class="gene-link">SPNCRNA.1688</span></a></td>
  <td>intergenic RNA nc-tgp1 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=25428589">PMID:25428589</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1701" routerLinkActive="active" [attr.title]="'SPNCRNA.1701'"><span class="gene-link">SPNCRNA.1701</span></a></td>
  <td>name prl101; synonym prl1 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1702" routerLinkActive="active" [attr.title]="'SPNCRNA.1702'"><span class="gene-link">SPNCRNA.1702</span></a></td>
  <td>name prl104; synonym prl4 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1703" routerLinkActive="active" [attr.title]="'SPNCRNA.1703'"><span class="gene-link">SPNCRNA.1703</span></a></td>
  <td>name prl105; synonym prl5 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1704" routerLinkActive="active" [attr.title]="'SPNCRNA.1704'"><span class="gene-link">SPNCRNA.1704</span></a></td>
  <td>name prl107; synonym prl7 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1705" routerLinkActive="active" [attr.title]="'SPNCRNA.1705'"><span class="gene-link">SPNCRNA.1705</span></a></td>
  <td>name prl108; synonym prl8 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1706" routerLinkActive="active" [attr.title]="'SPNCRNA.1706'"><span class="gene-link">SPNCRNA.1706</span></a></td>
  <td>name prl106; synonym prl6 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1707" routerLinkActive="active" [attr.title]="'SPNCRNA.1707'"><span class="gene-link">SPNCRNA.1707</span></a></td>
  <td>name prl102; synonym prl2 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1708" routerLinkActive="active" [attr.title]="'SPNCRNA.1708'"><span class="gene-link">SPNCRNA.1708</span></a></td>
  <td>name prl103; synonym prl3 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 </tbody>
</table>
<h3 id="genes-removed-since-publication">Genes Removed Since Publication</h3>
<table>
 <tbody>
 <tr>
  <th>Gene</th>
  <th>Comment</th>
  <th>Date</th>
 </tr>
 <tr>
  <td>SPBC19G7.12c</td>
  <td>removed because sequence similarity supports SPBC19G7.11c</td>
  <td><span class="no-break">2004-09-14</span></td>
 </tr>
 <tr>
  <td>SPAC212.10</td>
  <td>removed as part of UTR of SPAC212.11</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC3F10.14</td>
  <td>reverse strand of SPAC3F10.15c</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC428.09c</td>
  <td>presumed non-coding as no methionine or consensus splice sites</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC222.17c</td>
  <td>presumed non-coding; was a very small ORF in ncRNA prl50</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.135</td>
  <td>Initially annotated as a non-coding RNA but subsequently determined to be the 5' UTR of SPBC16D10.06 (pers. comm S. Whitehall)</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBPB21E7.10</td>
  <td>partial ORF of 35 aa, no reason to believe this is coding</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC4G8.15c</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAPB18E9.03c</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC18H10.21c</td>
  <td>alias SPBC9B6.01c; dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC9B6.01c</td>
  <td>alias of SPBC18H10.21c; dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPCC2H8.03</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC22E12.12</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.85</td>
  <td>replaced by protein coding gene SPAC227.19c</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.48</td>
  <td>replaced by protein coding gene SPAC2F3.18</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.97</td>
  <td>replaced by short protein coding gene SPAC27E2.14</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC8E11.09c</td>
  <td>previously annotated as dubious but no evidence for splicing, and non-canonical splice sites; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC22E12.15</td>
  <td>previously annotated as dubious but dubious splice prediction; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC1002.21</td>
  <td>was annotated as dubious but based on position in uracil regulatable operon, as this CDS does not appear to be transcribed. t is possible that the localization data is an artifact of the combination of overexpression and hydrophobicity. The protein sequence is also a strange composition, and there is no ortholog in octosporus or japonicus genomes. On balance, taking into account all of the available features and contextual information, it is unlikely that this feature codes for a bona fide protein.</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC8C9.20</td>
  <td>non consensus branch sites, and no coverage with transcriptome data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC22E12.12</td>
  <td>unsupported by Solexa transcript data and largely overlapping with 3' UTR of rpl24-3</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPCC1183.12</td>
  <td>previously SPNCRNA.546; splice site not spotted because it was outside the coordinate range of the feature; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC16C6.14</td>
  <td>unsupported by Solexa transcript data and largely overlapping with 3' UTR of rpl24-3</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC6F6.18c</td>
  <td>merged with SPAC6F6.16c; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18535244">PMID:18535244</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC651.07</td>
  <td>merged with SPBC651.06; pers. comm. Charly Chahwan</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.109</td>
  <td>part of protein coding gene merge</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC18E5.09c</td>
  <td>merged with SPBC18E5.14c; pers. comm. Charly Chahwan</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC959.06c</td>
  <td>merged with SPAC959.05c; pers. comm. Charly Chahwan</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPCC794.14c</td>
  <td>dubious</td>
  <td>2005-05</td>
 </tr>
 <tr>
  <td>SPNCRNA.523</td>
  <td>extended exons of adjacent protein coding gene</td>
  <td><span class="no-break">2008-12-19</span></td>
 </tr>
 <tr>
  <td>SPBC14C8.08c</td>
  <td>merged with SPBC14C8.09c; frameshifted; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
  <td><span class="no-break">2009-02-02</span></td>
 </tr>
 <tr>
  <td>SPRRNA.25</td>
  <td>small fragment of 25S rRNA; <span class="no-break">2009-11-26</span></td>
 </tr>
 <tr>
  <td>SPAC4G9.22</td>
  <td>likely to be non-coding RNA, no supporting evidence for protein</td>
  <td><span class="no-break">2010-02-21</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.509</td>
  <td>dubious and partially overlapping with Ost5</td>
  <td><span class="no-break">2010-02-18</span></td>
 </tr>
 <tr>
  <td>SPBC11C11.12</td>
  <td>cob intron fragment currently annotated as pseudogene</td>
  <td><span class="no-break">2010-03-09</span></td>
 </tr>
 <tr>
  <td>SPBCPT2R1.09c</td>
  <td>no coding region</td>
  <td><span class="no-break">2010-09-19</span></td>
 </tr>
 <tr>
  <td>SPBPB2B2.04</td>
  <td>no coding region</td>
  <td><span class="no-break">2010-09-19</span></td>
 </tr>
 <tr>
  <td>SPBPB2B2.03c</td>
  <td>no coding region</td>
  <td><span class="no-break">2010-09-19</span></td>
 </tr>
 <tr>
  <td>SPAC17A2.15</td>
  <td>was on the reverse strand of the N-terminal extension of SPAC17A2.08</td>
  <td><span class="no-break">2011-03-10</span></td>
 </tr>
 <tr>
  <td>SPBC1348.13</td>
  <td>Coordinates updated; now annotated as nuclear mitochondrial pseudogene (SO:0001044)</td>
  <td><span class="no-break">2011-11-20</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.10</td>
  <td>merged with SPNCRNA.1234</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1098</td>
  <td>merged with SPNCRNA.07</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1196</td>
  <td>merged with SPNCRNA.67</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.125</td>
  <td>merged with SPCC16A11.14</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1254</td>
  <td>merged with SPNCRNA.511</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1291</td>
  <td>merged with SPNCRNA.519</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1486</td>
  <td>merged with SPNCRNA.388</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.18</td>
  <td>merged with SPNCRNA.928</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.489</td>
  <td>merged with SPNCRNA.1187</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.502</td>
  <td>merged with SPNCRNA.1228</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.582</td>
  <td>merged with SPNCRNA.1208</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.64</td>
  <td>merged with SPCC1393.13</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.884</td>
  <td>merged with SPNCRNA.214</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.958</td>
  <td>merged with SPAC23A1.20</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPAC1F12.03c</td>
  <td>removed; replaced by a nuclear mitochondrial pseudogene (NUMT) feature</td>
  <td><span class="no-break">2012-07-16</span></td>
 </tr>
 <tr>
  <td>SPAC1F8.09c</td>
  <td>annotated in error</td>
  <td><span class="no-break">2013-02-22</span></td>
 </tr>
 <tr>
  <td>SPBC713.13</td>
  <td>deleted; ID added as synonym to SPBC713.14c&nbsp;(<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>)</td>
  <td><span class="no-break">2014-07-04</span></td>
 </tr>
 <tr>
  <td>SPAC823.02</td>
  <td>deleted; replaced by SPAC823.17 at same locus, but in different frame&nbsp;(<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>)</td>
  <td><span class="no-break">2014-07-04</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.252</td>
  <td>changed to LTR</td>
  <td><span class="no-break">2015-01-22</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.76</td>
  <td>overlapped with correctly annotated NUMT (SPNUMT.9)</td>
  <td><span class="no-break">2015-10-12</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.83</td>
  <td>overlapped with correctly annotated NUMT (SPNUMT.8)</td>
  <td><span class="no-break">2015-10-12</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.906</td>
  <td>merged with snR30</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.814</td>
  <td>merged with snR98</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1436</td>
  <td>merged with snR95</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.9061</td>
  <td>merged with snR91</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1038</td>
  <td>merged with snR90</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1394</td>
  <td>merged with snR3</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.727</td>
  <td>deleted; ID added as synonym to SPNCRNA.82 and SPSNORNA.10 (covers both)</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.727</td>
  <td>deleted; ID added as synonym to snR41/SPSNORNA.07, snR70/SPSNORNA.08 and snR51b/SPSNORNA.09 (covers all)</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.866</td>
  <td>unsupported (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:21511999</a>)</td>
  <td><span class="no-break">2015-11-30</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.867</td>
  <td>unsupported (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:21511999</a>)</td>
  <td><span class="no-break">2015-11-30</span></td>
 </tr>
 <tr>
  <td>SPAC1D4.07c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPAC27E2.13</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPAC4H3.12c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC1685.12c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC32F12.17</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC354.11c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC36.13</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC1672.14</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC188.05</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC417.04</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC622.05</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC794.16</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 </tbody>
</table>

  </div>
  <div *ngIf="pageName == 'sequence-updates-pending'" class="docs-content">
<h1 id="pending-sequence-updates">Pending sequence updates</h1>
<h2 id="individual-sequence-updates">Individual sequence updates</h2>
<ul>
<li>Chromosome III: The reference sequence is missing 4 bp -- GATC to be inserted after reference position 17744235. Pers. comm. Nathan Lee (S. Grewal lab), reported <span class="no-break">2014-03-31</span>.</li>
</ul>
<h2 id="sequence-discrepancies-from-the-broad">Sequence discrepancies from the Broad</h2>
<p>As part of a technology development project, the Broad Institute generated 200-fold sequence coverage of the genome of <em>S. pombe</em> strain 972 using Illumina (Solexa) technology. This deep coverage was used to identify discrepancies between the Illumina read data and the current reference assembly of 972, employing a polymorphism discovery algorithm written specifically for Illumina data, and tuned to have a very low false positive call rate. 190 sequence discrepancies were observed, all but two of which were either single base insertions, deletions or substitutions. These discrepancies could be due to sequence errors in the reference 972 assembly, sequence polymorphisms between the two isolates of 972 sequenced, or systematic errors in the Illumina data or analysis. Of the 190 discrepancies, 25 occur in homopolymer runs of at least 7 nucleotides, which are both more likely to produce sequencing errors and inter-strain polymorphisms, and 39 are clustered within 10 nucleotides of another discrepancy, which is more common in cases of mis-alignment of the Illumina reads. Excluding these sites, the discrepancy rate is about 1 in 100,​000, which is close to the estimated error rate in published reference sequence.</p>
<p>The data listed here were updated in August 2008; the original coordinates were based on an older assembly and did not match the current assembly of the right arm of Chromosome I. One discrepancy is a known artifact. At nucleotide 1774235 of Chromosome III there is an insertion of GACT caused by an assembly glitch in the published reference, which will be corrected in future releases. In addition,​ there are several clusters of discrepancies which are suspicious. These discrepancies will be investigated and the discrepancy list updated as appropriate. Beyond that analysis, no validation of other discrepancies is currently planned. However, we welcome any validation efforts by members of the community. If you generate any data pertinent to these discrepancies, or if you have any comments about them, please contact <a routerLink="/mailto:nick.rhind@umassmed.edu" routerLinkActive="active">Nick Rhind</a> or <a href="mailto:val@sanger.ac.uk">Valerie Wood</a>. These data can be cited as follows: &quot;Data were generated at the Broad Institute. Chad Nusbaum, personal communication.&quot;</p>
<p>The data can be downloaded as an Excel spreadsheet from the <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Archived_directories/972_discrepancies_from_Broad.xls">PomBase ftp site</a>.</p>
<table>
<thead>
<tr class="header">
<th align="left">Chromosome</th>
<th align="left">Location</th>
<th align="left">Left</th>
<th align="left">Sample</th>
<th align="left">Reference</th>
<th align="left">Right</th>
<th align="left">Clustered?</th>
<th align="left">Homopolymer</th>
<th align="left">Confirmed in</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">I</td>
<td align="left">94353</td>
<td align="left">GTGACAATTGAATTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">AAATGTTTCTATATTGTTAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">101872</td>
<td align="left">GAGGACGGGCGTTGGCGGAG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">CAACAGCCTTACCCCAGTCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">470379</td>
<td align="left">TGTGATAGACAAAAAAAGAC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">GTTAAAAGAACCTAAACGAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">470390</td>
<td align="left">AAAAAAGACCGTTAAAAGAA</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">CTAAACGAATTGTAATCAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">482738</td>
<td align="left">AGCTATTCGATCCACCGCAG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TTTGTCTGATTTGTTGCCCT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">523602</td>
<td align="left">ACAATGATACTTACTCCACA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">CGATTCCATTTTCCAAAGAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">523882</td>
<td align="left">GTTTCAGCCATTGAAAGATA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">CGATCGTTCTCAGACAAAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">527155</td>
<td align="left">AATCATGGATTTAATACATA</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">CTTTTAGTTTGTATGTGATA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">554538</td>
<td align="left">GAAGATTTATTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GGAATTTATTCTGCTCGGCC</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">670033</td>
<td align="left">AGAGATTATTATTAAGATCA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CCTAAATATCGTAACGTACG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">682996</td>
<td align="left">ATAAAACTACTATACTTCCC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">GCTGGAGCTTCGGTAGGCCG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">759333</td>
<td align="left">TGTTCACAGTTTTTCCCAAA</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">TTGCTTTTTCTTTTGGCTTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">862963</td>
<td align="left">ATTAAATTCTCGAAGTTGCT</td>
<td align="left">TT</td>
<td align="left">G</td>
<td align="left">ATTTTCCAAATCAATTCATG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">1048646</td>
<td align="left">ACGGAAAAGAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TCCGCGTTCAGTAGAAGTCA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">1274804</td>
<td align="left">GTACTTGAAATTATTTTTTA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TAAAGTGAGAACGCCCAGTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">1424708</td>
<td align="left">GTGCTTCATCAGTGACAACC</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">TTTAGAATGGATCTGTATGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">1625094</td>
<td align="left">ATCAAATAGTCGATGCCTCC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">AGCATCTCAATCTTCAATTC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">1783457</td>
<td align="left">ACTCTGGGTTCTTCTTATGT</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">TGGGACGGTCTTCGGCATAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">1936298</td>
<td align="left">ACTCTTAAATTTTAGGAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GACGCTCTTTTATAGTAAGG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">1952317</td>
<td align="left">TATATGTACGTACATATTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GCGCACCAGTTAGTTTGCGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">1999191</td>
<td align="left">TTTACCGCTTGACTGTGACC</td>
<td align="left">C</td>
<td align="left">G</td>
<td align="left">TACGACCTAACAATTTATTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18088324">PMID:18088324</a></td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2104090</td>
<td align="left">AAACTGTTGTAAAAAAATTA</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">ATTATCGGTGTATTTATTTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2104096</td>
<td align="left">TTGTAAAAAAATTAGATTAT</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">GGTGTATTTATTTATATTAG</td>
<td align="left">Yes - 6</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2154492</td>
<td align="left">AATTTGTTAAAAAAGGAGTA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GTTTACCTCTCGCTTTATGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2288241</td>
<td align="left">TCTTCATTGTTATTGTTACG</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">ATTCGTCACCCAGTGAAGAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2343704</td>
<td align="left">TGCAAACTCTTTGATCACGA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GTCTACATAATTCTTGAACC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2421577</td>
<td align="left">AGCTAAATATAAGAAAAAAC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">GAGAATTATAAGCAAACGTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2499159</td>
<td align="left">AAAACATCAATTTTTTTTTT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">GACAGTAATTATACTAGAAT</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2506043</td>
<td align="left">AAATAGTGTCACATTGTTTT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">ATATTAAGTATTTTTAATCG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2507463</td>
<td align="left">TTAGGTGTTAAACATAAATA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CTTACGTTTTCATACGCGCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2588024</td>
<td align="left">AAAGAACAGCAAAAAACAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GATAATGACAGAAAAAATAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2588072</td>
<td align="left">CTGAGAAAGTTGACAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TTTTTGGCGAAAGAAAGAAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2594264</td>
<td align="left">TAATTTTTAATGAATACTCT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">AGGCTTCCTTAAATGTTTTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2594326</td>
<td align="left">TTAAAGCAGTGTCTTATTAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CATTGTAAACATCAAAAAGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2594351</td>
<td align="left">TAAACATCAAAAAGCTTCAT</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">AACAGATGTAAGAAATTAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2594393</td>
<td align="left">AATTAACTTGTGAAGACATT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">CGATGTGATTAAAGTATGAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2605763</td>
<td align="left">TGTAAAAAGCATTTGAAATT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">AAAAAGAAAAAACAAAAAAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2605862</td>
<td align="left">AACGTACTGAGGATATCGTT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">ACCCCTTGCAGAAATATAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2607673</td>
<td align="left">ATCCATGTCTGTAATATTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GGTTATTAATAAATTGAAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">2683903</td>
<td align="left">CAAATATCATCAGTGCGATC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">AACAGCGGTATTTGTCTGAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">2759872</td>
<td align="left">TATCTTCCCACCGTCGTTTC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TATCGGTAGCATATCCGATT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3111507</td>
<td align="left">TGCGTTTTCTGTCTGTTGTG</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">ATGTCTCGCCTGGACTTCTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3115840</td>
<td align="left">ACAGATAAAGAGAGAAAAGT</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">CAAGCGTTTAAGCACCGAAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3119298</td>
<td align="left">AAAATCTTATTGATTGAAAT</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">CAATGCTTCGTAAACAATAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3125119</td>
<td align="left">TGCCAAACTCTTTTCTTCAT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">ATGTTCTGTTGTGCCAAAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3178496</td>
<td align="left">TAACTGGCAAATCGGCTCTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">CCCATAAGTAGACGATGATC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3179237</td>
<td align="left">TGCTAAAATACAGTTAGTTA</td>
<td align="left">C</td>
<td align="left">G</td>
<td align="left">TACTTGCTCACTTATATTTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3197530</td>
<td align="left">AATACTGCTGGAAATACAGG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TTTGGTTCGCAAGGTACTGG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://getentry.ddbj.nig.ac.jp/getentry/na/LC043101/?format=flatfile&filetype=html&trace=true&show_suppressed=false&limit=10">DDJB:LC043101</a></td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3450130</td>
<td align="left">TTATTATTCCCAGGCACGGG</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">AAGTTCCAAAAATCGAATAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3460320</td>
<td align="left">TTTACAAAGTCGGACAATCC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TGGCGGCGGTGTTCTAAAAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3538415</td>
<td align="left">TAAAGTATCGGAATTACATT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">CGTCAAACCAGCGTTTATGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3538428</td>
<td align="left">TTACATTCGTCAAACCAGCG</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">TTTATGATAAAAGATCATCC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3538436</td>
<td align="left">GTCAAACCAGCGTTTATGAT</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">AAAAGATCATCCTATTCTAA</td>
<td align="left">Yes - 8</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3538448</td>
<td align="left">TTTATGATAAAAGATCATCC</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">TATTCTAATTTTGCTATCGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3538450</td>
<td align="left">TATGATAAAAGATCATCCTA</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">TTCTAATTTTGCTATCGATA</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3538454</td>
<td align="left">ATAAAAGATCATCCTATTCT</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">AATTTTGCTATCGATAGCCC</td>
<td align="left">Yes - 4</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3538456</td>
<td align="left">AAAAGATCATCCTATTCTAA</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">TTTTGCTATCGATAGCCCTT</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3583105</td>
<td align="left">TTATAAATTTATAAATGATA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">AAATAAAGTGCAAGAAGAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3665610</td>
<td align="left">CTAGATTTAACTTTGCAACG</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TTTCGCAATCGGTGAACAGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">3730377</td>
<td align="left">TGCAACAATTCATTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">ACCTAATTTGTTTTCGACAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">3855791</td>
<td align="left">GGCATTTTTCACGTACAGGT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">AGTCGAAACATTATTAAATA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4055068</td>
<td align="left">ATTTAAAAAAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TTATAAGACATACCCTTTCG</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4276903</td>
<td align="left">TAGAATGTTTTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GAGAATATTATTCACACGCC</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4304172</td>
<td align="left">TCAAACCTATAAACAGGAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GATAAACGAAAATAGAATTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4374343</td>
<td align="left">AGGCTAAGTCTAGGTTGTAG</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">AAAAATCGTTTAGTCTGTAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4375420</td>
<td align="left">GTCTGAAGACTTTACCTTCT</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TCTTTTGTGCTCCTATCAAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4407495</td>
<td align="left">TCTCGTATAAGTTCATCCTG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">CGAAGCCGTTGTAGCCTATT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4410193</td>
<td align="left">ATCGACACAGATCGCGGCGG</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">AAGAAAAAGCAACTGAAGAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17035632">PMID:17035632</a></td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4431859</td>
<td align="left">ATCGAAACTACGTTGTGAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TAACTTAGCAAATATATGGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4431868</td>
<td align="left">ACGTTGTGAAATAACTTAGC</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">AAATATATGGTAAATAACAC</td>
<td align="left">Yes - 9</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4442713</td>
<td align="left">TGGAAGCTGATTTTTTGTTA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">TATGTCGTAAGTTACTGTGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4455162</td>
<td align="left">TAACTAGAAAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CCAAAGATAAAAAATGAAGT</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4721884</td>
<td align="left">AGTAGGAGTAGAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TTTTCAGCGCCTCGTCTCCT</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">4764269</td>
<td align="left">TTTGAAAAAAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">TTTCTACATTTCTTTCTTTA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">4919602</td>
<td align="left">GTTTTTTGGGTAAAGATCTA</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">AGGGTATATTGCTTTTTTAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">5142628</td>
<td align="left">CTGAGGAAGACGTTCCGAAG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TAAAGTGGAAAACGTTAGAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">5148423</td>
<td align="left">AATAAAACAGTAAAAAAAAA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">GATAAAAAGCTGAACAGTAT</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">5173097</td>
<td align="left">CACAGATAGCTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">AACAGGTACTTTATACATGG</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">5368264</td>
<td align="left">AGTATAGTTGGTCTCCCTCC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">ATATACGGTTTGGATAAGTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">5368273</td>
<td align="left">GGTCTCCCTCCCATATACGG</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">TTTGGATAAGTTTCGCCTTG</td>
<td align="left">Yes - 9</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">I</td>
<td align="left">5400079</td>
<td align="left">TACATTTTAGGATCAGCTCA</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">AAGTGTCCCTTTTTACAAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">37049</td>
<td align="left">ATTAATAATATATTTGCTCA</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">CCTAGATTTAAAGAATTTAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">128007</td>
<td align="left">TCAGAGCAGTCAATAAAAAA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">TAAAAAATCGAACAAAGGAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">146086</td>
<td align="left">GCCCTCGCTTGTTCCCCCCC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">TTATCTTACGAGTATATAGC</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">266717</td>
<td align="left">ATTCATGAGATTCGTACACG</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">CCTTACCAAGTCTTGCCAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">266779</td>
<td align="left">TGTTAGGGGTGATATGGGTT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">CGTTTTATCTGGGATCAAGG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">276538</td>
<td align="left">GACTTTAATTTATTCATGGA</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">CCTGAAGCTGCTCAAACTGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">685727</td>
<td align="left">ATCTCGCTGCTCAAATGTCA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">ATGCTGATTTTGGCTCCAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">699950</td>
<td align="left">TTTCTTTGAATTTTTTGTTT</td>
<td align="left">G</td>
<td align="left">T</td>
<td align="left">GTATTTTAAATTTATTATCT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">699951</td>
<td align="left">TTCTTTGAATTTTTTGTTTT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">TATTTTAAATTTATTATCTT</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">699960</td>
<td align="left">TTTTTTGTTTTGTATTTTAA</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TTTATTATCTTCCTGTCTTT</td>
<td align="left">Yes - 9</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">700010</td>
<td align="left">CCTCTATTTTTACAGATTTA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">CAAGTTTTCAACTTCGAAAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">700018</td>
<td align="left">TTTACAGATTTAACAAGTTT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">CAACTTCGAAACCCTAGAAA</td>
<td align="left">Yes - 8</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">700019</td>
<td align="left">TTACAGATTTAACAAGTTTT</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">AACTTCGAAACCCTAGAAAT</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">700020</td>
<td align="left">TACAGATTTAACAAGTTTTC</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">ACTTCGAAACCCTAGAAATT</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">700023</td>
<td align="left">AGATTTAACAAGTTTTCAAC</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">TCGAAACCCTAGAAATTTTC</td>
<td align="left">Yes - 3</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">700027</td>
<td align="left">TTAACAAGTTTTCAACTTCG</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">AACCCTAGAAATTTTCATAT</td>
<td align="left">Yes - 4</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">700112</td>
<td align="left">CTTTATCAAAAGTTATCATT</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">AATTGCCCTGCAACCGATAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">869805</td>
<td align="left">CTCTTCCAAGTCGGGTTCAT</td>
<td align="left">C</td>
<td align="left">G</td>
<td align="left">ATGCACTAATGAATTGTAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">879032</td>
<td align="left">ACTAGAGAAAGAAAAAAAGT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">ACAGTGGCTGATATTACAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">888600</td>
<td align="left">AAGAAAACTTTTGGTTATAC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">GAAACTTTGTGTGAAATGTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">903106</td>
<td align="left">AGCACAAACAATTCGAACTG</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GTGCTAACATCTAATAAAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">923542</td>
<td align="left">TTTAAAAAAATGACAGACGA</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">TTAAACTATGTTTTGAAATT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">943553</td>
<td align="left">AGCAAAGAATGAAGCCCTTA</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">ATTTTTGGATAGTTTTAAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">953200</td>
<td align="left">GTCGTTTCGAAGTTTTAGAA</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">ATGAAAAAAATTGGGCGTCT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">967093</td>
<td align="left">GAATTTCCCTTGACTTTTGA</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">GGAAGCCATATCTTCGAGTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1155902</td>
<td align="left">CTTGCAAAAACAAAAATATA</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">ATCCTTTTTTCTGCTGTAAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1159677</td>
<td align="left">TGAGTTTATTCATGAATTGC</td>
<td align="left">C</td>
<td align="left">G</td>
<td align="left">CGAAAATGAGGGTCCATCCG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1159693</td>
<td align="left">TTGCGCGAAAATGAGGGTCC</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TCCGTTCTCTACGATTCTGG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1163155</td>
<td align="left">GTATATAAGTAAATAAAAAT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">AGTATGCTTCTGGTCAACTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1308626</td>
<td align="left">GTTCGGCGAAGGTTTTTTTT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">GCATTGATCAACAGGTGATG</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1672858</td>
<td align="left">GCTTCCGTTTATATACGCGT</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">GGACATATAACAATGAACTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1673001</td>
<td align="left">GATGCTCCACCACCTGAAAC</td>
<td align="left">C</td>
<td align="left">G</td>
<td align="left">TTGATGCGTTCATTAGAGTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1678692</td>
<td align="left">TTTGCATTGGCAGCGACGTT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">GTATTTGCCGGCAAGCCCAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1683323</td>
<td align="left">CATTATGGTTGGAAATGATG</td>
<td align="left">G</td>
<td align="left">T</td>
<td align="left">AAACACCGTATACTTAGCTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1697123</td>
<td align="left">AGACTTGTTAATCATGTTAG</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">ACCTATGCTTTAATATAATT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1716317</td>
<td align="left">TACCGAGAAAACTTTCGTCA</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">ATTTTATGGGCCTAATAATA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1716319</td>
<td align="left">CCGAGAAAACTTTCGTCAAA</td>
<td align="left">G</td>
<td align="left">T</td>
<td align="left">TTTATGGGCCTAATAATAAT</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1716325</td>
<td align="left">AAACTTTCGTCAAATTTTAT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">GGCCTAATAATAATCTTATG</td>
<td align="left">Yes - 6</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1784825</td>
<td align="left">GGCCACATGTCATACATTGA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">TACGACTGCAAAGTTGACCT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1849776</td>
<td align="left">TGTCAGAATCATTGTCGAGG</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">TTTTGCATATCTCGTAAGCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1861420</td>
<td align="left">GAAAAGCATAAGAAGTTAAT</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">TACATAAAAGCACTTTAAAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1867805</td>
<td align="left">AAAGTACTTATTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">ACGAATAAAATGTAAAAAAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1873781</td>
<td align="left">GTGGTTTGTAATATTCCTAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CCACCCCATTTTTTTCTGGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1877305</td>
<td align="left">ACAGCCCGATTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GTCTCAGTAATTCAAGAGAC</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1941966</td>
<td align="left">GATTCACATACCGAAGCGGA</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GAAAATTAATAAATCAGAGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1941982</td>
<td align="left">CGGAGAAAATTAATAAATCA</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GAGTTTTAACTCAAAGGAAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1948954</td>
<td align="left">CGGCGTTGAAGCGATCTTGA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">GCGGCCTGGAGTGCGCGAGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1950052</td>
<td align="left">ATGTCGGCATTGCCAGAAGG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TTTTTGCTAGAGGCATAGCT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1955509</td>
<td align="left">TTCCTTAAAAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">GAAAGAAAATGAAAACTCAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1960392</td>
<td align="left">AATTAAATTGTCTAAACGTA</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TTCGTTGCTCAAGCTTCCTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1962418</td>
<td align="left">TAAGAGAGAAAACTCGGTGT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">TATTGGAAAAGTGGTACTAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1962438</td>
<td align="left">TTATTGGAAAAGTGGTACTA</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">CAAGTGTGGGAAGTATGTTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1962457</td>
<td align="left">AACAAGTGTGGGAAGTATGT</td>
<td align="left">A</td>
<td align="left">T</td>
<td align="left">TGACAGATTCATTACATTAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1987097</td>
<td align="left">AGCTTCGCATAGACATATAC</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">AACGGAACGAATGACACCTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">1987101</td>
<td align="left">TCGCATAGACATATACCAAC</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">GAACGAATGACACCTGGCAG</td>
<td align="left">Yes - 4</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">1987117</td>
<td align="left">CAACGGAACGAATGACACCT</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">GCAGTGAAAATTCCCTTTAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2044928</td>
<td align="left">AATTGACTTATCTATTAATT</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">ACTTGATTTTTGTTTGTTAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2049893</td>
<td align="left">GAAACTAGCTCATTATGATT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">GAAGTTTGTGCCAGATAAAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2053517</td>
<td align="left">AGTGATTTTCTCAAAAAGGC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TCTGCTACCCGTCATATTGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2108180</td>
<td align="left">GTAAGATAATAAAGCATAAT</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CCAATCTTCTTTGTTTGAAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2187407</td>
<td align="left">CATTTTTCTTGGTGGGAAGG</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CATCAAAAAAATCCCCCTGG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2219929</td>
<td align="left">AATTGACTGTTTGGCTGTAT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">GGTGATATGCTGAAAGCGAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2228774</td>
<td align="left">AAGACAAGATAAATAATAAG</td>
<td align="left">G</td>
<td align="left">T</td>
<td align="left">TGGAAAAGTATATACCTCTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2228776</td>
<td align="left">GACAAGATAAATAATAAGTT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">GAAAAGTATATACCTCTGTT</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2264919</td>
<td align="left">ATAATAAAAACATTTATTTA</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">GTTTTATTTTTGAACTTCTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2293304</td>
<td align="left">ATTTATCACAACGAAAAAAA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">CTCGTATCACTATAATTTTT</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2436579</td>
<td align="left">ACCTTGCAGCCAACTACTTC</td>
<td align="left">G</td>
<td align="left">C</td>
<td align="left">TCACCAATTACAACTTCTGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2444542</td>
<td align="left">GAATAATGCTTCCTACATTG</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">TACCGCCTATTTAATTACAA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2592434</td>
<td align="left">CCTAAACATAAAAAAAAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CTTACAATCAATGTATATGG</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2676335</td>
<td align="left">TTTTACCGGTTCTAAAATAT</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">CCAAAGACCTGAAACCCCCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2709415</td>
<td align="left">CGTAGCACAAGGCCGAGAGC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TCTTCTCATAGTGGCTCTTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left">pers. comm. D. Bitton</td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2788933</td>
<td align="left">ATATTTTGCTGGATTAAAGC</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">AAATAACTTTTGCGTCAAAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2798041</td>
<td align="left">ACATTCATGGATGAAGACCT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">GGGAACATGGGAAGGCCTAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2798514</td>
<td align="left">ATTCAGATGTCATGTTCAAA</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">AAAAAGATAGACTATCCTAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">2811496</td>
<td align="left">CAGAAAGTACAAAGCAATCA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">TACAAGAAGAAGAGAATATC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">2938969</td>
<td align="left">AGGCAACTTTTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">AAGGGTTGTAGTATTGTTTA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">3006136</td>
<td align="left">GTATTCTATACCTTGGGCAG</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TTCTTCAAGCAATTTGGTCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">3040333</td>
<td align="left">ACTCTAGATCTCCGTCTCCG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">CTGCACGCCCTATTTCTCGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">3619004</td>
<td align="left">CAACCGAGAAGTCCCGGTAG</td>
<td align="left">G</td>
<td align="left"> </td>
<td align="left">TTGCGTCTTTGTATATAAAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">3838114</td>
<td align="left">AGACCTCAAAAAAAAAAAAA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">GCGAAATGCTAACTTCAAAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">4122325</td>
<td align="left">ATCGCTGTTAATGGTGCACA</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">ATCCACAACGTGATTGGAAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">II</td>
<td align="left">4254961</td>
<td align="left">TGGCCGGGTTATTCACAAGA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">TACCTTTTATCCAGCTTGAC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">II</td>
<td align="left">4400665</td>
<td align="left">CAAATTAAACAATCCAAACC</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">TAAATCGTCACCAGTACTAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">227729</td>
<td align="left">TTTCGTGTTCACGAGGAAGA</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">CGTTGTTTTTGCACATAATG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">240810</td>
<td align="left">TGACCATACCATTGGGATGA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">CCTTACTCTTGATTAGCTCG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">284656</td>
<td align="left">AGTATAACTAGCGCAGTAGG</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">AAAACAGAAGATTGTATGAG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719956</td>
<td align="left">GTTTAATATATTTAATAAAC</td>
<td align="left"> </td>
<td align="left">C</td>
<td align="left">TATTTTATTCTATGTCGAGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719970</td>
<td align="left">ATAAACCTATTTTATTCTAT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">TCGAGAGAAGATTGAGATGA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719973</td>
<td align="left">AACCTATTTTATTCTATGTC</td>
<td align="left"> </td>
<td align="left">G</td>
<td align="left">AGAGAAGATTGAGATGATTA</td>
<td align="left">Yes - 3</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719975</td>
<td align="left">CCTATTTTATTCTATGTCGA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">AGAAGATTGAGATGATTAAT</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719977</td>
<td align="left">TATTTTATTCTATGTCGAGA</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">AAGATTGAGATGATTAATCA</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719978</td>
<td align="left">ATTTTATTCTATGTCGAGAG</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">AGATTGAGATGATTAATCAT</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719980</td>
<td align="left">TTTATTCTATGTCGAGAGAA</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">ATTGAGATGATTAATCATTC</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719984</td>
<td align="left">TTCTATGTCGAGAGAAGATT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">AGATGATTAATCATTCCCTC</td>
<td align="left">Yes - 4</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719986</td>
<td align="left">CTATGTCGAGAGAAGATTGA</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">ATGATTAATCATTCCCTCTA</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719987</td>
<td align="left">TATGTCGAGAGAAGATTGAG</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TGATTAATCATTCCCTCTAT</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">719989</td>
<td align="left">TGTCGAGAGAAGATTGAGAT</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">ATTAATCATTCCCTCTATCT</td>
<td align="left">Yes - 2</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">719990</td>
<td align="left">GTCGAGAGAAGATTGAGATG</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TTAATCATTCCCTCTATCTA</td>
<td align="left">Yes - 1</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">720000</td>
<td align="left">GATTGAGATGATTAATCATT</td>
<td align="left">T</td>
<td align="left">C</td>
<td align="left">CCTCTATCTATTGATTGTTT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">720696</td>
<td align="left">TCCACAATTTTTTTTTTTTT</td>
<td align="left">T</td>
<td align="left"> </td>
<td align="left">ACCATCTCCTTTAAATCAAA</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">867874</td>
<td align="left">CTTTATTTTACTTTTTTTTT</td>
<td align="left"> </td>
<td align="left">T</td>
<td align="left">ATATTTTTTAAAATATATTC</td>
<td align="left">No</td>
<td align="left">X</td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">1016312</td>
<td align="left">TAACAATGAAATAACGAAGC</td>
<td align="left">C</td>
<td align="left"> </td>
<td align="left">TTCAGTATAATTAGTTCTAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">1168327</td>
<td align="left">TCATAGCCTTTTATTACTAT</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">TGGGAAGTCAATTTTTAGTA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">1374937</td>
<td align="left">AGGTTGTTATGTTTCAGACT</td>
<td align="left">T</td>
<td align="left">A</td>
<td align="left">TGATCATTACGGTGATACTG</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">1685491</td>
<td align="left">GGCCACCACCAAGAAGAAAA</td>
<td align="left">A</td>
<td align="left"> </td>
<td align="left">CCAAAGAAAAAGAACAGTCA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">1756401</td>
<td align="left">AAACCTTTGATGAAAGTTTA</td>
<td align="left"> </td>
<td align="left">A</td>
<td align="left">TAAAGCTGCCAATGTATATA</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">1765400</td>
<td align="left">TAAATTGCCGTATTATATTG</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">TACTATAACGAAATAATTGC</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
<tr class="odd">
<td align="left">III</td>
<td align="left">1774235</td>
<td align="left">AACATCTTCCAAGTTCTCAT</td>
<td align="left">GATC</td>
<td align="left"> </td>
<td align="left">CTTTAAAGATTCCTCAAGGT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td align="left">III</td>
<td align="left">2381978</td>
<td align="left">AATTTGATGTCTTAAATTTG</td>
<td align="left">T</td>
<td align="left">G</td>
<td align="left">TGGTTTGCTTGAAGGCATAT</td>
<td align="left">No</td>
<td align="left"> </td>
<td align="left"> </td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'sequencing-status'" class="docs-content">
<h1 id="sequencing-status">Sequencing status</h1>
<p>Note: Many older <em>S. pombe</em> sequence submissions to the DNA databases (International Nucleotide Sequence Database Collaboration databases,​ i.e. ENA, GenBank, DDBJ) contain one or more errors, and we do not have the resources to maintain past sequences or flag every error in PomBase.</p>
<h2 id="chromosome-1">Chromosome 1</h2>
<table>
<thead>
<tr class="header">
<th align="left">Contig Name</th>
<th align="left">Region</th>
<th align="left">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"> </td>
<td align="left">unsequenced to chr1 left telomere</td>
<td align="left">10 kb ± 2 kb*</td>
</tr>
<tr class="even">
<td align="left">c212</td>
<td align="left">sub-telomeric left arm</td>
<td align="left">29,​663 bp</td>
</tr>
<tr class="odd">
<td align="left"> </td>
<td align="left">Gap</td>
<td align="left">&lt;5kb*</td>
</tr>
<tr class="even">
<td align="left">c977</td>
<td align="left">left arm and right arm</td>
<td align="left">5,​549,​370 bp</td>
</tr>
<tr class="odd">
<td align="left"> </td>
<td align="left">unsequenced to chr1 left telomere</td>
<td align="left">18kb ± 3kb*</td>
</tr>
</tbody>
</table>
<p> </p>
<h2 id="chromosome-2">Chromosome 2</h2>
<table>
<thead>
<tr class="header">
<th align="left">Contig Name</th>
<th align="left">Region</th>
<th align="left">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"> </td>
<td align="left">unsequenced to chr2 left telomere</td>
<td align="left">25kb ± 5kb*</td>
</tr>
<tr class="even">
<td align="left">c1348</td>
<td align="left">sub-telomeric left arm</td>
<td align="left">80,​201bp</td>
</tr>
<tr class="odd">
<td align="left"> </td>
<td align="left">Gap</td>
<td align="left">22kb ± 5kb*</td>
</tr>
<tr class="even">
<td align="left">pB10D8</td>
<td align="left">left arm to centromeric gap</td>
<td align="left">1,​536,​269 bp</td>
</tr>
<tr class="odd">
<td align="left"> </td>
<td align="left">Gap</td>
<td align="left">~6kb</td>
</tr>
<tr class="even">
<td align="left">pJ5566</td>
<td align="left">right arm from centromeric gap to telomeric repeats</td>
<td align="left">2,​923,​134 bp</td>
</tr>
</tbody>
</table>
<p> </p>
<h2 id="chromosome-3">Chromosome 3</h2>
<table>
<thead>
<tr class="header">
<th align="left">Contig Name</th>
<th align="left">Region</th>
<th align="left">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">p20C8</td>
<td align="left">left arm from centromeric gap</td>
<td align="left">1,​083,​348 bp</td>
</tr>
<tr class="even">
<td align="left"> </td>
<td align="left">Gap</td>
<td align="left">25.3kb ± 6kb**</td>
</tr>
<tr class="odd">
<td align="left">c1676</td>
<td align="left">right arm from centromeric gap</td>
<td align="left">1,​369,​435 bp</td>
</tr>
</tbody>
</table>
<p>* pers comm. Randy Hyppa and Gerry Smith.<br />
** pers comm. Chad Ellermeier and Gerry Smith.</p>
<h3 id="centromere-clone-information"><a routerLink="/status/centromeres" routerLinkActive="active">Centromere clone information</a></h3>
<h3 id="telomere-clone-information"><a routerLink="/status/telomeres" routerLinkActive="active">Telomere clone information</a></h3>
<h3 id="mating-type-region-information"><a routerLink="/status/mating-type-region" routerLinkActive="active">Mating type region information</a></h3>
<h3 id="ribosomal-dna-information-faq"><a href="http://www.pombase.org/faqs/are-there-any-rdna-repeat-sequences-pombase">Ribosomal DNA information (FAQ)</a></h3>

  </div>
  <div *ngIf="pageName == 'sequencing-updates'" class="docs-content">
<h1 id="s.-pombe-sequence-updates-since-july-2003"><em>S. pombe</em> Sequence Updates Since July 2003</h1>
<h3 id="jul-2003-chromosome-1-gap-filled">Jul 2003 Chromosome 1 Gap Filled</h3>
<p>Added PCR product SPACPJ113 6900 with overlaps of 504 bp with c1856 and 147bp with p4C9<br />
Joined contigs p4C9 and c977</p>
<h3 id="apr-2004-chromosome-1-cosmid-c6b12">Apr 2004 Chromosome 1 cosmid c6B12:</h3>
<p>Single base deletion at c6B12 13981 from tgttctgcAtttttcc to tgttctgctttttcc<br />
(<strong>Note:</strong> this change was not added to the spread sheet on the ftp site until Jan 18 2006)</p>
<h3 id="section"></h3>
<h3 id="may-2004-chromosome-2-cosmid-c646">May 2004 Chromosome 2 cosmid c646:</h3>
<p>Single base transition A-&gt;C at c646 from aatggaccAtacccc to aatggaccCtacccc<br />
(<strong>Note:</strong> This change does not affect sequence coordinates)</p>
<h3 id="may-2006-chromosome-1-cosmid-c21e11">May 2006 Chromosome 1 cosmid c21E11:</h3>
<p>Single base insertion at c21E11 from gagtgca to gagtgGca (in cyp8) base 4257404 in new chromsome file (note reverse strand sequence reported)</p>
<h3 id="jan-2007-genedb-moves-from-contigs-to-chromosomes">Jan 2007 GeneDB moves from Contigs to Chromosomes</h3>
<p>Contigs merged into chromosomes: the 4 sequence gaps represented by 100 Ns<br />
(<strong>Note:</strong> that the chromosomes previously made available from the ftp site had 1000 Ns in the gaps)</p>
<h3 id="jun-2008-197-potential-sequence-discrepancies-from-the-broad-institute">Jun 2008 197 potential sequence discrepancies from the Broad Institute</h3>
<p><strong>PENDING <span class="no-break">2008-08-26</span></strong></p>
<p>Generated as part of a technology development project, using Illumina (Solexa) technology the Broad Institute generated &gt;200-fold sequence coverage of the genome of S. pombe strain 972. These data identified 197 potential discrepancies. The discrepancy rate is about 1 in 100,​000,​ which is close to the estimated error rate in published reference sequence. Further information and data <a routerLink="//status/sequence-updates-pending" routerLinkActive="active">here</a></p>
<h3 id="jun-2008-gatc-in-overlap-c1450-and-c1442-affects-spcc1442.04c">Jun 2008 GATC in overlap c1450 and c1442 affects SPCC1442.04c</h3>
<p><strong>PENDING <span class="no-break">2008-08-26</span></strong></p>
<p>There are 4 bases missing from this sequence AACATCTTCCAAGTTCTCAT GATC CTTTAAAGATTCCTCAAGGT, which will change the C-terminal region slightly. This will be fixed in the next round of corrections</p>
<h3 id="aug-29-2008-chromosome-1-cosmid-c27d7-misassembly-causing-duplication-of-spac27d7.09c-spac27d7.10c">Aug 29 2008 Chromosome 1 cosmid c27D7 misassembly causing duplication of SPAC27D7.09c (SPAC27D7.10c)</h3>
<p><strong>PENDING <span class="no-break">2008-08-29</span></strong></p>
<p>An apparent repeat region on chromosome 1 coordinates 4526059..4529095 (cosmid c27D7) is caused by a missassembly and will be removed from the genomic sequence shortly. The CDS feature SPAC27D7.10c within this region is an exact duplication of SPAC27D7.09c and will be merged with this CDS. Pers. comm. Klavs Hansen. Reported 09/01/04</p>
<h3 id="aug-29-2008-chromosome-2-cosmid-c21d10-affecting-spbc21d10.06cmap4">Aug 29 2008 Chromosome 2 cosmid c21D10 affecting SPBC21D10.06c/Map4</h3>
<p><strong>PENDING <span class="no-break">2008-08-29</span></strong></p>
<p>Map4 in the reference genome contains an array of 5 repeats. In PMID:168571979 repeats are reported. This number is the correct number of repeats and will be updated via an insertion into the contig sequence shortly S. Pers. comm. Henar Valdivieso Reported 06/15/05</p>
<h3 id="aug-29-2008-chromosome-1-cosmid-c767-affecting-spac767.01cvps1">Aug 29 2008 Chromosome 1 cosmid c767 affecting SPAC767.01c/Vps1</h3>
<p><strong>PENDING <span class="no-break">2007-08-23</span></strong></p>
<p>At 1999191 in Vps1 TGTGACC[C-&gt;G]TACGAC PMID:18088324, also confirmed by Broad sequence; Pers. comm. Isabelle Jourdain Reported 06/15/05</p>
<h3 id="aug-29-2008-chromosome-1-cosmid-c30-or-c29e6-affecting-spac29e6.04cnnf1">Aug 29 2008 Chromosome 1 cosmid C30 OR c29E6 affecting SPAC29E6.04c/Nnf1</h3>
<p><strong>PENDING <span class="no-break">2008-08-29</span></strong></p>
<p>The reference sequence appears to contain an extra 'G' at position 4410193 PMID:17035632, also confirmed by Broad sequence; Pers. comm. Aki Hayashi Reported 05/16/07</p>

  </div>
  <div *ngIf="pageName == 'statistics'" class="docs-content">
<h1 id="genome-statistics">Genome statistics</h1>
<h3 id="genome-structure-and-organisation">Genome Structure and Organisation</h3>
<table>
<tbody>
<tr class="odd">
<td align="left">Genome Size (sequenced portion, excluding rDNA)</td>
<td align="left">12.57Mbp</td>
</tr>
<tr class="even">
<td align="left">Number of Chromosomes</td>
<td align="left">3</td>
</tr>
<tr class="odd">
<td align="left">Chromosome Size Range</td>
<td align="left">2.45Mbp - 5.58Mbp</td>
</tr>
<tr class="even">
<td align="left">Gene Count (protein coding genes on Chr I, II and III)</td>
<td align="left">5118</td>
</tr>
<tr class="odd">
<td align="left">Gene Count excluding dubious genes (protein coding genes on Chr I, II and III)</td>
<td align="left">5064</td>
</tr>
<tr class="even">
<td align="left">Gene Density (genes per Mb)</td>
<td align="left">552.66</td>
</tr>
<tr class="odd">
<td align="left">Genome GC Content</td>
<td align="left">36.06%</td>
</tr>
<tr class="even">
<td align="left">Intergenic Length (mean for protein coding genes)</td>
<td align="left">227.37</td>
</tr>
<tr class="odd">
<td align="left">Intergenic GC Content (protein coding genes)</td>
<td align="left">28.94%</td>
</tr>
<tr class="even">
<td align="left">Mitochondrial Genome Size</td>
<td align="left">19.43kbp</td>
</tr>
<tr class="odd">
<td align="left">Mitochondrial GC Content</td>
<td align="left">30.09%</td>
</tr>
<tr class="even">
<td align="left">Telomeric Located Contig Size</td>
<td align="left">20.00kbp</td>
</tr>
<tr class="odd">
<td align="left">Telomeric Located Contig GC Content</td>
<td align="left">33.17%</td>
</tr>
</tbody>
</table>
<h3 id="chromosome-i-ii-and-iii-transcript-breakdown">Chromosome (I, II and III) Transcript Breakdown</h3>
<table>
  <thead>
    <tr>
      <th rowspan="2">Feature Type</th>
      <th colspan="3" style="text-align: center;">Count</th>
      <th>Gene Density</th>
      <th colspan="6" style="text-align: center;">GC Content (%)</th>
      <th colspan="5" style="text-align: center;">Mean Length (bp)</th>
    </tr>
    <tr>
      <th>Gene</th>
      <th>Exon</th>
      <th>Intron</th>
      <th>(genes per Mb)</th>
      <th>Gene</th>
      <th>Exon (All)</th>
      <th>Exon (Coding)</th>
      <th>Intron</th>
      <th>5' UTR</th>
      <th>3' UTR</th>
      <th>Gene</th>
      <th>Exon</th>
      <th>Intron</th>
      <th>5' UTR</th>
      <th>3' UTR</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rRNA</td>
      <td>47</td>
      <td>47</td>
      <td>0</td>
      <td>3.74</td>
      <td>45.72</td>
      <td>45.72</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td style="text-align:right;">527.53</td>
      <td style="text-align:right;">527.53</td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>snoRNA</td>
      <td>57</td>
      <td>58</td>
      <td>1</td>
      <td>4.53</td>
      <td>36.54</td>
      <td>36.82</td>
      <td></td>
      <td>30.36</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">140.09</td>
      <td style="text-align:right;">131.88</td>
      <td style="text-align:right;">336.00</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>tRNA</td>
      <td>171</td>
      <td>197</td>
      <td>26</td>
      <td>13.60</td>
      <td>53.79</td>
      <td>54.15</td>
      <td></td>
      <td>38.56</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">77.32</td>
      <td style="text-align:right;">65.56</td>
      <td style="text-align:right;">11.77</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>pseudogene</td>
      <td>28</td>
      <td>50</td>
      <td>22</td>
      <td>2.23</td>
      <td>38.95</td>
      <td>39.25</td>
      <td></td>
      <td>30.90</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">1,&#8203;047.39</td>
      <td style="text-align:right;">565.70</td>
      <td style="text-align:right;">47.36</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>mRNA (protein_coding)</td>
      <td>5118</td>
      <td>10425</td>
      <td>5307</td>
      <td>407.10</td>
      <td>37.08</td>
      <td>37.39</td>
      <td>39.63</td>
      <td>29.72</td>
      <td>33.24</td>
      <td>32.27</td>
      <td style="text-align:right;">2,&#8203;147.68</td>
      <td style="text-align:right;">1,&#8203;011.98</td>
      <td style="text-align:right;">83.27</td>
      <td style="text-align:right;">290.05</td>
      <td style="text-align:right;">428.79</td>
    </tr>
    <tr>
      <td>ncRNA</td>
      <td>1522</td>
      <td>1527</td>
      <td>5</td>
      <td>121.06</td>
      <td>35.82</td>
      <td>35.82</td>
      <td></td>
      <td>27.94</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">1,&#8203;248.46</td>
      <td style="text-align:right;">1,&#8203;244.21</td>
      <td style="text-align:right;">49.40</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>snRNA</td>
      <td>5</td>
      <td>6</td>
      <td>1</td>
      <td>0.40</td>
      <td>40.03</td>
      <td>40.47</td>
      <td></td>
      <td>34.00</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">147.40</td>
      <td style="text-align:right;">114.50</td>
      <td style="text-align:right;">50.00</td>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>
<h3 id="mitochondrial-transcript-breakdown">Mitochondrial transcript breakdown</h3>
<table>
<thead>
<tr class="header">
<th align="left">Feature type</th>
<th align="left">Count</th>
<th align="left">GC content (%)</th>
<th align="left">Mean length (bp)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">rRNA</td>
<td align="left">2</td>
<td align="left">31.97</td>
<td align="left">2,​122.00</td>
</tr>
<tr class="even">
<td align="left">mRNA (protein_coding)</td>
<td align="left">11</td>
<td align="left">29.39</td>
<td align="left">1,​393.00</td>
</tr>
<tr class="odd">
<td align="left">tRNA</td>
<td align="left">25</td>
<td align="left">38.75</td>
<td align="left">74.52</td>
</tr>
</tbody>
</table>
<p><br />
### Telomeric located contig transcript breakdown</p>
<table>
<thead>
<tr class="header">
<th align="left">Feature type</th>
<th align="left">Count</th>
<th align="left">GC content (%)</th>
<th align="left">Mean length (bp)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">mRNA (protein_coding)</td>
<td align="left">5</td>
<td align="left">38.46</td>
<td align="left">1,​428.00</td>
</tr>
</tbody>
</table>
<p><br />
### Mating type region</p>
<table>
<thead>
<tr class="header">
<th align="left">Feature type</th>
<th align="left">Count</th>
<th align="left">GC content (%)</th>
<th align="left">Mean length (bp)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">mRNA (protein_coding)</td>
<td align="left">4</td>
<td align="left">35.94</td>
<td align="left">395.75</td>
</tr>
</tbody>
</table>
<p><br />
#### <strong>Release Version:</strong> 30th Jan 2017</p>

  </div>
  <div *ngIf="pageName == 'telomeres'" class="docs-content">
<h1 id="telomeres">Telomeres</h1>
<p>The fission yeast complete genome sequence currently stops short of the <span>telomeric</span> repeats. See the <a href="status/sequencing-status">Sequencing Status</a> page for the current assembly status.</p>
<p>The most proximal anchored <span>cosmids</span> to each <span>telomere</span> are (links to genome browser):</p>
<ul>
<li>Chromosome I left c212 (coordinates <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=I%3A1-29663;site=ensemblunit">1-29663</a>)</li>
<li>Cromosome I right c750 (<a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=I%3A5554844-5579133;site=ensemblunit">5554844-5579133</a>)</li>
<li>Chromosome II left c1348 (<a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=II%3A1-39186;site=ensemblunit">1-39186</a>)</li>
<li>Chromosome II right pT2R1 (<a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?r=II%3A4500619-4539800;site=ensemblunit">4500619-4539800</a>)</li>
<li>There are no telomere proximal clones for chromosome III as the unsequenced rDNA blocks occur inbetween the sequenced portion and the telomeres on both chromosome arms.</li>
</ul>
<p>Details of all clones used for the assembly, and their order, length and overlap details is provided in a set of <a routerLink="/ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Archived_directories/Cosmid_assembly_data/" routerLinkActive="active">spreadsheets</a> available from the <em>S. pombe</em> FTP site.</p>
<p>A contig extending the left arm of Chromosome II was sequenced by Sasaki <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18727152">PMID:18727152</a>) and will be attached to the assembly in the next round of sequence changes. In the meantime, the contig can be viewed in the <a href="http://genomebrowser.pombase.org/Schizosaccharomyces_pombe/Location/View?g=SPBC460.01c;r=AB325691:1479-3197;t=SPBC460.01c.1">PomBase Genome Browser</a>.</p>
<p>A set of <a routerLink="/ftp://ftp.sanger.ac.uk/pub/yeast/sequences/pombe/telomeres/" routerLinkActive="active">small insert clones</a> (FTP link; also see table) from a telomere plasmid library has been made available by <a routerLink="/mailto:sugawara@hydra.rose.brandeis.edu" routerLinkActive="active">Neal Sugawara</a>. None can be assigned to a chromosome at present.</p>
<p>Clones included in the final sequence are available from the Wellcome Trust Sanger Institute; send email to <a href="mailto:archives@sanger.ac.uk">archives@sanger.ac.uk</a></p>
<table>
<thead>
<tr class="header">
<th align="left">Plasmid</th>
<th align="left">Size of insert</th>
<th align="left">Location of telomeric sequence</th>
<th align="left">Comments</th>
<th align="left">Vector</th>
<th align="left">Laboratory</th>
<th align="left">Funded by</th>
<th align="left">Stage</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">pNSU28</td>
<td align="left">approx 1kb</td>
<td align="left"> </td>
<td align="left">Lies in pNSU21</td>
<td align="left">pUC19</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="even">
<td align="left">pNSU31</td>
<td align="left">approx 1kb</td>
<td align="left"> </td>
<td align="left">Lies in pNSU21</td>
<td align="left">pUC19</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="odd">
<td align="left">pNSU68</td>
<td align="left">423 bp</td>
<td align="left">Internal</td>
<td align="left">Contains 195bp of telomeric DNA and 123bp from the rDNA .</td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="even">
<td align="left">pNSU71</td>
<td align="left">15 kb</td>
<td align="left">Terminal</td>
<td align="left"> </td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="odd">
<td align="left">pNSU64 <sup>*</sup></td>
<td align="left">6.9 kb</td>
<td align="left">Terminal</td>
<td align="left"> </td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="even">
<td align="left">pNSU70</td>
<td align="left">7.1 kb</td>
<td align="left"> </td>
<td align="left"> </td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="odd">
<td align="left">pNSU77 <sup>*</sup></td>
<td align="left">12 kb</td>
<td align="left">Internal</td>
<td align="left">Fusion between telomere sequences (7.1kb) and rDNA sequences (4.9kb)</td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="even">
<td align="left">pNSU21</td>
<td align="left">7.9 kb</td>
<td align="left">Terminal</td>
<td align="left">pNSU21 and pNSU65 were isolated from the first and second library respectively</td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
<tr class="odd">
<td align="left">pNSU65</td>
<td align="left">8.1 kb</td>
<td align="left">Terminal</td>
<td align="left">pNSU21 and pNSU65 were isolated from the first and second library respectively</td>
<td align="left">pMLC28</td>
<td align="left">Hinxton</td>
<td align="left">EC</td>
<td align="left">finished</td>
</tr>
</tbody>
</table>
<p><strong>Table notes:</strong><br />
&quot;Terminal&quot; refers to the position adjacent to the vector sequences of pMLC12 where the blunt end (SmaI end) ligated to the telomeric sequences. &quot;Internal&quot; means that an <em>S. pombe</em> sequence intervenes between the telomere and vector sequences.</p>

  </div>
  <div *ngIf="pageName == 'unmapped_genes'" class="docs-content">
<h1 id="unmapped-genes">Unmapped genes</h1>
<h3 id="genetic-loci-not-cloned-or-physically-mapped">Genetic loci not cloned or physically mapped</h3>
<table>
 <thead>
  <tr>
   <th>Locus name</th>
   <th>Publication(s)</th>
   <th>Allelic with</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td>all2</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ani1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ani2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>cyh4, tri3</td>
 </tr>
 <tr>
  <td>anr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>anr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>anr3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>anr4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>aro3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=4211223">PMID:4211223</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ben2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7003311">PMID:7003311</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ben3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7003311">PMID:7003311</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>can2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18357653">PMID:18357653</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>chr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12746851">PMID:12746851</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15114862">PMID:15114862</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps2</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps4</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps6</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps7</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps9</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7882425">PMID:7882425</a></td>
  <td>&nbsp;</td>
 <tr>
  <td>cps11</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps13</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps14</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 <tr>
  <td>crm2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2647765">PMID:2647765</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=607151">PMID:607151</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1068110">PMID:1068110</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1068110">PMID:1068110</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1068110">PMID:1068110</a></td>
  <td>ani2, tri3</td>
 </tr>
 <tr>
  <td>glu1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7903653">PMID:7903653</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>git4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2157626">PMID:2157626</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>hsp1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7500954">PMID:7500954</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>hyd1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=23712692">PMID:23712692</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ird11</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=22095476">PMID:22095476</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=22173629">PMID:22173629</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=25117315">PMID:25117315</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mcs3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11085271">PMID:11085271</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2474475">PMID:2474475</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15509866">PMID:15509866</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mcs5</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2474475">PMID:2474475</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis7</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis8</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis9</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pet1g</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ptp1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644270">PMID:1644270</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8082169">PMID:8082169</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8887563">PMID:8887563</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12058079">PMID:12058079</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ptp2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644270">PMID:1644270</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8082169">PMID:8082169</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ral4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=3071741">PMID:3071741</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>scr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9535817">PMID:9535817</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>scr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9535817">PMID:9535817</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SLrsm1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21046351">PMID:21046351</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SLrsm2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21046351">PMID:21046351</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SLrsm3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21046351">PMID:21046351</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SP331</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7937765">PMID:7937765</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ste15</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8575018">PMID:8575018</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts7</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts8</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts9</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts11</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tnr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7961415">PMID:7961415</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644306">PMID:1644306</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tnr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7961415">PMID:7961415</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644306">PMID:1644306</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tri3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>ani2, cyh4</td>
 </tr>
 <tr>
  <td>tri4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tri5</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>trs1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1769544">PMID:1769544</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>wtr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8913338">PMID:8913338</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15338055">PMID:15338055</a></td>
  <td>&nbsp;</td>
 </tr>
 </tbody>
</table>

  </div>
</div>
</div>
