<div class="docs">
<div *ngIf="section == 'about'">
  <div class="docs-menu">
<h3 id="about"><a routerLink="/about" routerLinkActive="active">About</a></h3>
<ul>
<li>Contacts</li>
<li>Citing PomBase</li>
<li>Help</li>
<li>Resource Metrics</li>
</ul>

  </div>
  <div *ngIf="pageName == 'index'" class="docs-content">
<h1 id="about-pombase">About PomBase</h1>
<p>PomBase is a new model organism database that provides organization of and access to scientific data for the fission yeast Schizosaccharomyces pombe. PomBase supports genomic sequence and features, genome-wide datasets and manual literature curation.</p>
<p>PomBase also provides a community hub for researchers, providing genome statistics, a community curation interface, news, events,​ documentation and mailing lists.</p>
<hr />
<table>
<tbody>
<tr class="odd">
<td>FAQ</td>
<td>Frequently Asked Questions</td>
</tr>
<tr class="even">
<td>Documentation</td>
<td>Documentation</td>
</tr>
<tr class="odd">
<td>Data versions</td>
<td>Versions of internal and external data loaded into current and past Pombase releases</td>
</tr>
<tr class="even">
<td>Contact PomBase</td>
<td>How to contact staff or submit data</td>
</tr>
<tr class="odd">
<td>Citing PomBase</td>
<td>How to cite fission yeast genome data</td>
</tr>
<tr class="even">
<td>Staff</td>
<td>PomBase staff members</td>
</tr>
</tbody>
</table>
<hr />
<p>Funding: PomBase is funded by the Wellcome Trust [104967/Z/14/Z].</p>

  </div>
  <div *ngIf="pageName == 'pombe-mailing-list'" class="docs-content">
<h1 id="pombe-mailing-list">Pombe Mailing List</h1>
<p><em>Note: this list has moved. Please update your records!</em></p>
<p><em>The pombelist mailing list is used to announce:</em></p>
<ul>
<li>Sequence updates</li>
<li>Database updates</li>
<li>Software updates</li>
<li>Nomenclature updates</li>
</ul>
<p><em>List members can post:</em></p>
<ul>
<li>Information of general interest to the S. pombe community</li>
<li>General S. pombe queries</li>
</ul>
<p><em>To subscribe:</em></p>
<ul>
<li>Follow the instructions in the &quot;Subscribing to Pombelist&quot; section of <a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">this page</a></li>
<li>If you have problems subscribing, e-mail the <a routerLink="/mailto:helpdesk@pombase.org" routerLinkActive="active">PomBase helpdesk</a></li>
<li><a href="http://listserver.ebi.ac.uk/pipermail/pombelist/">Browse archives</a> from November 29th, 2004 onward</li>
</ul>

  </div>
</div>
<div *ngIf="section == 'browse-curation'">
  <div class="docs-menu">
<h3 id="browse-curation"><a routerLink="/browse-curation" routerLinkActive="active">Browse curation</a></h3>
<ul>
<li><a routerLink="/browse-curation/fission-yeast-phenotype-ontology" routerLinkActive="active">Fission Yeast Phenotype Ontology</a></li>
<li><a routerLink="/browse-curation/fission-yeast-go-slim-terms" routerLinkActive="active">GO slim terms</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'fission-yeast-go-slim-terms'" class="docs-content">
<h1 id="fission-yeast-go-slim-terms">Fission Yeast GO slim terms</h1>
<p>The table below shows terms in the current fission yeast biological process GO slim, and the number of annotations to each term. GO IDs link to the QuickGO browser, where you can explore the ontology and annotations further. Icons beside each GO term link to <a href="http://www.esyn.org/">esyN</a>, which provides a graphical view of interactions for the genes from the PomBase High Confidence Physical Interaction Network (HCPIN) dataset. Only the subset of genes linked into the interaction network will be displayed in the esyN network view. The annotation totals link to pages with information about the term and a list of annotated genes.</p>
<p>Further information is available from:</p>
<ul>
<li><a routerLink="/documentation/pombase-go-slim-documentation" routerLinkActive="active">PomBase GO slim documentation</a></li>
<li><a routerLink="/documentation/high-confidence-physical-interaction-network" routerLinkActive="active">PomBase HCPIN documentation</a></li>
<li><a href="http://www.esyn.org/tutorial.html">esyN documentation</a></li>
<li><a routerLink="/browse-curation/fission-yeast-go-slimming-tips" routerLinkActive="active">PomBase tips for using GO slim terms and annotations</a></li>
<li><a href="http://www.geneontology.org/GO.slims.shtml">GO Slim Guide at the Gene Ontology website</a></li>
</ul>
<p>Note that both proteins and RNAs can be annotated to GO terms, and the tables on this page include annotated RNAs. For some GO terms, notably 'cytoplasmic translation', RNAs make up a significant proportion of the total annotated genes.</p>
<p><app-go-slim-table></app-go-slim-table></p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-phenotype-ontology'" class="docs-content">
<h1 id="fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</h1>
<p>PomBase curators develop and use the Fission Yeast Phenotype Ontology (FYPO) to annotate phenotypes of mutant alleles and of over- or under-expressing wild type genes.</p>
<p>The contents and structure of FYPO are described on the <a href="http://curation.pombase.org/pombase-trac/wiki/FissionYeastPhenotypeOntology">FYPO page of the PomBase wiki</a> and in the publication:</p>
<p>Harris MA, Lock A, Bähler J, Oliver SG, Wood V. FYPO: The Fission Yeast Phenotype Ontology. Bioinformatics. 2013 July 1;29(13): 1671–1678. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23658422">PubMed abstract (PMID:23658422)</a> or <a href="http://bioinformatics.oxfordjournals.org/content/29/13/1671.long">Full text at Bioinformatics</a></p>
<h2 id="browsing-fypo-terms">Browsing FYPO terms</h2>
<p>At present, FYPO is included collections such as the National Center for Biomedical Ontology's <a href="http://bioportal.bioontology.org/">BioPortal</a> and EBI's <a href="http://www.ebi.ac.uk/ontology-lookup/">Ontology Lookup Service (OLS)</a>,​ both of which allow searching and browsing.</p>
<ul>
<li>BioPortal <a href="http://bioportal.bioontology.org/ontologies/FYPO">FYPO summary page</a></li>
<li>BioPortal <a href="http://bioportal.bioontology.org/ontologies/FYPO/?p=classes">FYPO terms</a> (the term display supports tree-viewer browsing, text searches, and a graphical display)</li>
<li><a href="http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=FYPO">OLS FYPO browser</a></li>
</ul>
<p>We hope to deploy a browser at PomBase in the not-too-distant future that will include annotations as well as terms.</p>
<h2 id="finding-fypo-terms-and-annotations-in-pombase">Finding FYPO terms and annotations in PomBase</h2>
<p>FYPO terms are displayed on PomBase gene pages, along with supporting evidence and allele and expression details, as described in the PomBase gene page documentation.</p>
<p>The PomBase advanced search can use either term names or IDs to search FYPO, and returns a list of genes that have alleles annotated to the specified term or any of its descendants. For example, see the <a href="http://www.pombase.org/faqs/can-i-get-list-essential-pombe-genes">FAQ on finding essential genes</a>.</p>
<h2 id="curating-phenotypes-with-fypo">Curating phenotypes with FYPO</h2>
<p>The PomBase online curation tool,​ <a href="http://curation.pombase.org/pombe">Canto</a>, uses FYPO terms for annotation of S. pombe phenotypes. <a href="http://curation.pombase.org/pombe/docs/fypo_annotation">Phenotype curation documentation</a> is available via Canto.</p>
<h2 id="submitting-bulk-data">Submitting Bulk Data</h2>
<p>If you have a large set of phenotype data to submit, you may want to do a bulk submission. See the documentation on the <a href="http://www.pombase.org/submit-data/phenotype-data-bulk-upload-format">recommended file format</a>,​ and use the Phenotype Data Submission Form.</p>

  </div>
  <div *ngIf="pageName == 'index'" class="docs-content">


  </div>
</div>
<div *ngIf="section == 'community'">
  <div class="docs-menu">
<h3 id="community"><a routerLink="/community" routerLinkActive="active">Community</a></h3>
<ul>
<li><a routerLink="/community/fission-yeast-community-curation-project" routerLinkActive="active">Curation</a></li>
<li>External Resources</li>
<li>Curation Pilot Project</li>
<li>News</li>
<li>Events</li>
<li>Jobs</li>
<li><a routerLink="/community/mailing-lists" routerLinkActive="active">Mailing lists</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'fission-yeast-community-curation-project'" class="docs-content">
<h1 id="the-fission-yeast-community-curation-project">The Fission Yeast Community Curation Project</h1>
<p>Comprehensive manual curation of the S. pombe literature is a primary goal of the PomBase project, and among the most important activities of the curation staff. Increasing numbers of new publications, and a large backlog of older papers, however, mean that complete literature curation will require input from the research community as well as the efforts of dedicated professional curators.</p>
<p>The PomBase Community Curation system enables researchers to contribute annotations directly to PomBase based on their publications using Canto, a web-based tool that allows both curators and researchers to create annotations. Canto supports GO, phenotype,​ interaction, and modification annotations, and can be configured for use with other ontologies as the need arises. Annotations made in the community curation system will be prioritized for inclusion in PomBase and will therefore also be more rapidly disseminated to other databases (e.g. GenBank/ENA/DDBJ, UniProtKB, BioGRID and GO), making data from annotated papers widely visible.</p>
<p><strong>How to contribute</strong>: For newly published papers, PomBase curators email authors inviting them to participate, with specific instructions and links. Lab members are also welcome to evaluate existing annotations and create new annotations from past publications — use the PubMed ID search on the <a href="https://curation.pombase.org/pombe">main PomBase Canto page</a>. Anyone can also try the <a href="https://curation.pombase.org/demo">demo version of Canto</a>.</p>
<p><strong>Progress and attribution</strong>: <a href="https://curation.pombase.org/pombe/stats/annotation">Curation statistics</a> track S. pombe papers curated in Canto by PomBase and community curators, and the annotations obtained. Canto will soon allow contributors to register <a href="http://orcid.org/">ORCID</a> identifiers in the system, which will support more formalised attribution of community contributions to PomBase, and thereby enable researchers to cite community curation in institutional reports, funding applications, etc. To supplement the existing permanent publication-centred links in Canto, we are currently investigating possible ways to link to external resources such as ORCID or OpenRIF via ORCID IDs.</p>
<p>For a bit of historical perspective, you can browse the results of the <a href="http://www.pombase.org/community/fission-yeast-community-curation-pilot-project">Community Curation Pilot Project</a> that took place in 2009.</p>

  </div>
  <div *ngIf="pageName == 'index'" class="docs-content">
<h1 id="fission-yeast-community">Fission Yeast Community</h1>
<ul>
<li><a routerLink="/community/fission-yeast-community-curation-project" routerLinkActive="active">Community curation project</a></li>
<li><a routerLink="/community/mailing-lists" routerLinkActive="active">Mailing lists</a></li>
<li>Listings on pombase.org</li>
<li>Events - List or Calendar</li>
<li>News Archive</li>
<li><p>Jobs</p></li>
<li><p>Fission Yeast Internet Resources (External)</p></li>
</ul>

  </div>
  <div *ngIf="pageName == 'mailing-lists'" class="docs-content">
<h1 id="mailing-lists">Mailing Lists</h1>
<p>Mailing lists of interest to the S. pombe community</p>
<hr />
<p><a routerLink="/about/pombe-mailing-list" routerLinkActive="active">Pombelist</a></p>
<p>Fission yeast community mailing list - follow link for further information, including how to subscribe. (Note: this list has moved. Please update your records!)</p>

  </div>
</div>
<div *ngIf="section == 'status'">
  <div class="docs-menu">
<h3 id="status"><a routerLink="/status" routerLinkActive="active">Status</a></h3>
<ul>
<li><a routerLink="/status/gene-coordinate-changes" routerLinkActive="active">Gene coordinate changes</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'gene-coordinate-changes'" class="docs-content">
<h1 id="gene-coordinate-changes">Gene Coordinate Changes</h1>
<table>
<thead>
<tr class="header">
<th>Systematic ID</th>
<th>Standard name</th>
<th>Comments</th>
<th>Reference</th>
<th>Date</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a routerLink="/gene/SPAC19B12.05c" routerLinkActive="active">SPAC19B12.05c</a></td>
<td></td>
<td>additional exon</td>
<td><span class="no-break">pers. comm. Val Wood</span></td>
<td><span class="no-break">2003-04-22</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC11E3.03" routerLinkActive="active">SPAC11E3.03</a></td>
<td>pcs1</td>
<td>N terminal truncated, removed exon 9067..9103, truncated from 9149 to 9211 start</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12689592">PMID:12689592</a></td>
<td><span class="no-break">2003-04-13</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC320.13c" routerLinkActive="active">SPCC320.13c</a></td>
<td>ark1</td>
<td>trimmed to 2nd met</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12676091">PMID:12676091</a></td>
<td><span class="no-break">2003-01-08</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC6B1.09c" routerLinkActive="active">SPBC6B1.09c</a></td>
<td>nbs1</td>
<td>prediction extended by 4 N- terminal exons, this identified FHA domain</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-04-22</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC6B1.09c" routerLinkActive="active">SPBC6B1.09c</a></td>
<td>nbs1</td>
<td>additional intron 25664..25705, and exon extension 25974..25988</td>
<td>pers. comm. Charly Chahwan</td>
<td><span class="no-break">2003-03-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC16A10.04" routerLinkActive="active">SPAC16A10.04</a></td>
<td>rho4</td>
<td>N-terminal exon replaced</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12653963">PMID:12653963</a></td>
<td><span class="no-break">2003-03-31</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC23C4.08" routerLinkActive="active">SPAC23C4.08</a></td>
<td>rho3</td>
<td>N-terminal exon removed</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12653963">PMID:12653963</a></td>
<td><span class="no-break">2003-03-31</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBP23A10.04" routerLinkActive="active">SPBP23A10.04</a></td>
<td>apc2</td>
<td>coordinates updated additional in-frame splice at 5325, 5371</td>
<td>pers. comm. Hyun-Joo Yoon</td>
<td><span class="no-break">2003-02-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1442.13c" routerLinkActive="active">SPCC1442.13c</a></td>
<td></td>
<td>C terminal exon added ; old coordinates (26947..27936); new exon extends G-patch domain</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-04-15</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1604.17c" routerLinkActive="active">SPBC1604.17c</a></td>
<td></td>
<td>previous coordinates (8945..8971, 9475..10455, 10528..10851)</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-04-29</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1296.04" routerLinkActive="active">SPAC1296.04</a></td>
<td></td>
<td>has 2 additional N-terminal exons</td>
<td>pers. comm. Val Wood</td>
<td></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23C11.10" routerLinkActive="active">SPAC23C11.10</a></td>
<td></td>
<td>found 3 additional C-terminal exons by eye</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-06-18</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC1347.05c" routerLinkActive="active">SPBC1347.05c</a></td>
<td></td>
<td>new C-terminal exon</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-10-11</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2F3.12c" routerLinkActive="active">SPAC2F3.12c</a></td>
<td></td>
<td>created from the C-terminal of SPAC2F3.13c</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-06-30</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC2F3.13c" routerLinkActive="active">SPAC2F3.13c</a></td>
<td></td>
<td>split to create SPAC2F3.13c and SPAC2F3.12c</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-06-30</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC22F3.03c" routerLinkActive="active">SPAC22F3.03c</a></td>
<td>rdh54</td>
<td>4th exon extended by altering acceptor in intron 3 cosmid coordinates 34092-33923</td>
<td>pers. comm. Mike Catlett</td>
<td><span class="no-break">2003-05-09</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29A4.14c" routerLinkActive="active">SPAC29A4.14c</a></td>
<td></td>
<td>N-terminal extended</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-09-11</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC31G5.07" routerLinkActive="active">SPAC31G5.07</a></td>
<td></td>
<td>N terminal extended to use longest ORF in the absence of homology</td>
<td>pers. comm. Henar Valdivieso Montero</td>
<td><span class="no-break">2003-01-08</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC19B12.06c" routerLinkActive="active">SPAC19B12.06c</a></td>
<td></td>
<td>added exon 18153..18218</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-04-22</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2F3.13c" routerLinkActive="active">SPAC2F3.13c</a></td>
<td></td>
<td>annotated as SPAC2F3.13c putative tRNA-ribosyltransferase, but actually a gene merge. Split to create SPAC2F3.13c the tRNA-ribosyltransferase and new CDS SPAC2F3.17c was created from the C-terminal</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-06-30</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29B12.08" routerLinkActive="active">SPAC29B12.08</a></td>
<td></td>
<td>new N-terminal exon</td>
<td>pers. comm. Aengus Stewart</td>
<td><span class="no-break">2003-09-09</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC959.05c" routerLinkActive="active">SPAC959.05c</a></td>
<td></td>
<td>extended N-terminal to first methionine</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-09-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC15D4.02" routerLinkActive="active">SPBC15D4.02</a></td>
<td></td>
<td>trimmed N terminal 28.10.03; looked overextended as large low complexity region in front in zinc finger which is usually N-term; alignment looks better</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-10-28</span></td>
</tr>
<tr class="even">
<td>SPCC1223.15c</td>
<td>spc19</td>
<td>4 amino acid extension at N-terminus</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16079914">PMID:16079914</a></td>
<td><span class="no-break">2005-09-09</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC17H9.20" routerLinkActive="active">SPAC17H9.20</a></td>
<td>psc3</td>
<td>added intron at 37250..37297</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16207082">PMID:16207082</a></td>
<td><span class="no-break">2005-10-10</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC31G5.06" routerLinkActive="active">SPAC31G5.06</a></td>
<td></td>
<td>added N-teminal exon to extend prediction</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2005-11-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC9.05" routerLinkActive="active">SPAC9.05</a></td>
<td>fml1</td>
<td>second intron acceptor extended by 15 bp</td>
<td>pers. comm. Anke Schürer</td>
<td><span class="no-break">2006-01-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC3B9.22c" routerLinkActive="active">SPBC3B9.22c</a></td>
<td>dad4</td>
<td>added new C-terminal exon based on Pfam protein family</td>
<td><a href="http://pfam.xfam.org/">Pfam</a></td>
<td><span class="no-break">2006-02-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC4G3.05c" routerLinkActive="active">SPCC4G3.05c</a></td>
<td>mus81</td>
<td>added N terminal exon</td>
<td>pers. comm. Fred Kippert</td>
<td><span class="no-break">2006-02-17</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1D4.11c" routerLinkActive="active">SPAC1D4.11c</a></td>
<td>lkh1</td>
<td>lengthened to 690 a.a.</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12565823">PMID:12565823</a></td>
<td><span class="no-break">2006-04-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBCPT2R1.08c" routerLinkActive="active">SPBCPT2R1.08c</a></td>
<td>tlh2</td>
<td>truncated to 1919 a.a.; removed truncated version of a malate transporter fused to gene, not known whether these residues are included in translation</td>
<td>pers. comm. Liew Li Phing</td>
<td><span class="no-break">2006-04-13</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC6G9.13c" routerLinkActive="active">SPAC6G9.13c</a></td>
<td>bqt1</td>
<td>Additional C terminal exon</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16615890">PMID:16615890</a></td>
<td><span class="no-break">2006-05-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC21E11.05c" routerLinkActive="active">SPAC21E11.05c</a></td>
<td>cyp8</td>
<td>Single base insertion allowed first 2 exons to be merged and extended (GAGT[G]GCA)</td>
<td>pers. comm. Trevor Pemberton (via Ivo Pedruzzi, UniProt)</td>
<td><span class="no-break">2006-05-22</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCPJ732.03" routerLinkActive="active">SPCPJ732.03</a></td>
<td>meu15</td>
<td>new coordinates as per publication</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11376151">PMID:11376151</a></td>
<td><span class="no-break">2006-06-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC4B3.05c" routerLinkActive="active">SPCC4B3.05c</a></td>
<td>hem12</td>
<td>Coordinates updated from join(28064..28097, 28155..28189, 28238..28816, 28866..29330) to join(28064..28097, 28155..28189, 28236..28260, 28308..28816, 28866..29330). Improves Homology</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2006-06-12</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC12D12.09" routerLinkActive="active">SPAC12D12.09</a></td>
<td>rev7</td>
<td>added new exon to insert KCIDIFGEF</td>
<td>pers. comm. Nicole Kosarek</td>
<td><span class="no-break">2006-08-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC12D12.09" routerLinkActive="active">SPAC12D12.09</a></td>
<td>rev7</td>
<td>truncated penultimate exon to remove GIN</td>
<td>pers. comm. Nicole Kosarek</td>
<td><span class="no-break">2006-08-01</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC30B4.08" routerLinkActive="active">SPBC30B4.08</a></td>
<td>eri1</td>
<td>gene structure updated as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16797182">PMID:16797182</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16797182">PMID:16797182</a></td>
<td><span class="no-break">2006-09-22</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.08" routerLinkActive="active">SPAPB17E12.08</a></td>
<td></td>
<td>coordinates updated from join(9968..9990, 10070..10442) to join(9609..9623, 9699..9761, 9937..9986, 10070..10442). Improved homology</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2006-12-12</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1223.10c" routerLinkActive="active">SPCC1223.10c</a></td>
<td>eaf1</td>
<td>Added exons as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17150956">PMID:17150956</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17150956">PMID:17150956</a></td>
<td><span class="no-break">2006-12-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP22H7.09c" routerLinkActive="active">SPBP22H7.09c</a></td>
<td>mis15</td>
<td>updated sequence coordinates as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15369671">PMID:15369671</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15369671">PMID:15369671</a></td>
<td><span class="no-break">2007-01-05</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23D3.08" routerLinkActive="active">SPAC23D3.08</a></td>
<td>usp108</td>
<td>final exon removed</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-02-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPNCRNA.82" routerLinkActive="active">SPNCRNA.82</a></td>
<td>mrp1</td>
<td>coordinates updated from 1234705..1234837 to 1234451..1234849, upon realisation that this corresponded to RNase MRP</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-02-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC13G7.07" routerLinkActive="active">SPAC13G7.07</a></td>
<td></td>
<td>changed second exon 3' boundary and added C-terminal exons</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17310250">PMID:17310250</a></td>
<td><span class="no-break">2007-03-23</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29E6.04" routerLinkActive="active">SPAC29E6.04</a></td>
<td>nnf1</td>
<td>Sequence frameshifted at 4410192, new coordinates give an N terminal extension from 4410194..4410391 (NOTE: Still needs resequencing and underlying sequence changing)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17035632">PMID:17035632</a></td>
<td><span class="no-break">2007-05-16</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23C4.04c" routerLinkActive="active">SPAC23C4.04c</a></td>
<td></td>
<td>altered to include stop codon and trimmed to methionine</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC409.12" routerLinkActive="active">SPBC409.12</a></td>
<td>stn1</td>
<td>N-terminal extended by 2 exons</td>
<td>pers. comm. C. Chahwan</td>
<td><span class="no-break">2007-07-06</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC9E9.02" routerLinkActive="active">SPAC9E9.02</a></td>
<td></td>
<td>altered to include stop codon</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-07-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F8.07c" routerLinkActive="active">SPAC22F8.07c</a></td>
<td>rtf1</td>
<td>Two in-frame splices added</td>
<td><a href="http://www.ebi.ac.uk/ena/data/view/AJ627891">EMBL:AJ627891</a></td>
<td><span class="no-break">2007-08-28</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1020.11c" routerLinkActive="active">SPCC1020.11c</a></td>
<td></td>
<td>coordinates updated from 757494..757802 to join(757365..757424, 757506..757802)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-10-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1529.01" routerLinkActive="active">SPCC1529.01</a></td>
<td></td>
<td>new exon added; coordinates changed from join(237554..237923, 237975..238080, 238139..238533, 238588..239105) to join(237554..237923, 237975..238080, 238139..238533, 238588..239092, 239151..239250)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-10-12</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPB2B2.07c" routerLinkActive="active">SPBPB2B2.07c</a></td>
<td></td>
<td>coordinates updated from 4474434..4474931 to join(4474145..4474403, 4474438..4474931)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-22</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC6G9.16c" routerLinkActive="active">SPAC6G9.16c</a></td>
<td></td>
<td>coordinates changed from 3278951..3279349 to join(3278951..3279367, 3279420..3279539, 3279683..3279819, 3279865..3279985)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-22</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC354.07c" routerLinkActive="active">SPBC354.07c</a></td>
<td></td>
<td>additional exon added; coordinates updated from 558335..559225, 559273..559474, 559942..560028, 560151..560170 to 558335..559225, 559273..559474, 559942..560028, 560151..560163, 560231..560246</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-28</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC631.02" routerLinkActive="active">SPAC631.02</a></td>
<td></td>
<td>coordinates updated from 2107940..2110123 to 2107470..2107515, 2107860..2110123</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-28</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC18.18c" routerLinkActive="active">SPCC18.18c</a></td>
<td></td>
<td>coordinates changed from 1988631..1990079 to 1988631..1990100, 1990630..1990722</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-29</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC16A10.06c" routerLinkActive="active">SPAC16A10.06c</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data; coordinates updated from 3089583..3089996, 3090112..3090285, 3090533..3090697 to 3089583..3090047, 3090112..3090285, 3090533..3090697</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC17C9.08" routerLinkActive="active">SPAC17C9.08</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data and unsupported from mRNA data so final exon deleted; coordinates changed from 4484758..4484804, 4484843..4485274, 4485350..4485596, 4485713..4485994 to 4484835..4485274, 4485350..4485596, 4485713..4485994</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.08" routerLinkActive="active">SPAPB17E12.08</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data so small N-term exon deleted; coordinates changed from 1279492..1279506, 1279582..1279644, 1279820..1279869, 1279953..1280325 to 1279573..1279644, 1279820..1279869, 1279953..1280325</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAPB1A10.15" routerLinkActive="active">SPAPB1A10.15</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data so small C-term exon deleted; coordinates changed from 1892554..1892640, 1892926..1893060, 1893098..1893504, 1893585..1893618 to 1892554..1892640, 1892926..1893060, 1893098..1893694</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC56F8.05c" routerLinkActive="active">SPAC56F8.05c</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data. N terminal exon unsupported and had no branch site so exon deleted; coordinates changed from 1134697..1135061, 1135195..1135402, 1135517..1135684, 1135729..1135776, 1135932..1135970, 1136111..1136170 to 1134697..1135061, 1135195..1135402, 1135517..1135684, 1135729..1135776, 1135932..1136003</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1450.09c" routerLinkActive="active">SPCC1450.09c</a></td>
<td></td>
<td>original intron absent from Solexa data; C terminal exon unsupported so exon deleted; coordinates changed from 1741451..1741481, 1741589..1743459 to 1741588..1743459</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC736.05" routerLinkActive="active">SPCC736.05</a></td>
<td>wtf7</td>
<td>removed final exon, unsupported by Solexa intron data; coordinates changed from 320617..320755, 320964..321335, 321387..321509, 321554..321582 to 320617..320755, 320964..321335, 321387..321523</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC227.11c" routerLinkActive="active">SPAC227.11c</a></td>
<td></td>
<td>removed final exon, unsupported by Solexa data; coordinates changed from 514434..514489, 514557..515082, 515222..515482, 515534..515659 to 514552..515082, 515222..515482, 515534..515659</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC21B10.02" routerLinkActive="active">SPBC21B10.02</a></td>
<td></td>
<td>intron unsupported by Solexa transcript data, removed final exon; coordinates changed from 1669624..1669695, 1669737..1670144 to 1669719..1670144</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC21C3.07c" routerLinkActive="active">SPBC21C3.07c</a></td>
<td></td>
<td>removed final exon, not supported by Solexa transcript data; coordinates changed from 3807714..3807783, 3807963..3808607, 3808665..3808795 to 3807920..3808607, 3808665..3808795</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC29A3.07c" routerLinkActive="active">SPBC29A3.07c</a></td>
<td></td>
<td>final exon deleted, not supported by Solexa intron data; homology improved; coordinates changed from 2050069..2050183, 2050304..2050358, 2050452..2050695 to 2050258..2050358, 2050452..2050695</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1604.17c" routerLinkActive="active">SPBC1604.17c</a></td>
<td></td>
<td>updated, better donor (GTAATA instead of GTTAGT); original intron prediction not supported by Solexa data; coordinates changed from 3900047..3900121, 3900166..3901146, 3901219..3901542 to 3900047..3900121, 3900166..3901170, 3901219..3901542</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16A3.14" routerLinkActive="active">SPBC16A3.14</a></td>
<td></td>
<td>final exon deleted, not supported by Solexa intron data; coordinates changed from 4270907..4270939, 4271137..4271937 to 4271131..4271937</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23D3.08" routerLinkActive="active">SPAC23D3.08</a></td>
<td>usp108</td>
<td>final exon deleted, not supported by Solexa intron data; coordinates changed from 4352514..4353359, 4353410..4353573, 4353623..4353761 to 4352514..4353359, 4353410..4353577</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC4C5.01" routerLinkActive="active">SPAC4C5.01</a></td>
<td></td>
<td>altered intron 5 to improve donor and acceptor, improves homology; coordinates changed from 1192545..1192592, 1192662..1192704, 1192779..1192841, 1192916..1193010, 1193062..1193175, 1193236..1193613 to 1192545..1192592, 1192662..1192704, 1192779..1192841, 1192916..1193010, 1193062..1193145, 1193197..1193613</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16H5.04" routerLinkActive="active">SPBC16H5.04</a></td>
<td></td>
<td>altered intron 2 donor and acceptor, improved homology; coordinates changed from 2293393..2293824, 2293938..2294039, 2294087..2294125 to 2293393..2293775, 2293877..2294039, 2294087..2294125</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC11B10.03" routerLinkActive="active">SPBC11B10.03</a></td>
<td></td>
<td>removed N-teminal exon, poor donor, did not improve homology; coordinates changed from 1490376..1490423, 1490593..1491747 to 1490611..1491747</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAPB1A10.03" routerLinkActive="active">SPAPB1A10.03</a></td>
<td></td>
<td>SPAPB1A10.03 split to create SPAPB1A10.03 and SPAPB1A10.16; new coordinates 1863363..1863381, 1863434..1863762 and 1863972..1864388</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1494.02" routerLinkActive="active">SPCC1494.02</a></td>
<td></td>
<td>2 C-terminal exons removed, not supported by homology; coordinates changed from 2317988..2318068, 2318200..2318244, 2318398..2318508, 2318579..2318676, 2318903..2318966 to 2318335..2318508, 2318579..2318676, 2318903..2318966</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1442.13c" routerLinkActive="active">SPCC1442.13c</a></td>
<td></td>
<td>deleted final exon, improved homology; coordinates changed from 1791196..1791285, 1791376..1791909 to 1791346..1791909</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC622.14" routerLinkActive="active">SPCC622.14</a></td>
<td></td>
<td>altered donor for intron 3, improved donor consensus; coordinates changed from 1427098..1427310, 1427632..1427709, 1427788..1428180, 1428277..1428522 to 1427098..1427310, 1427632..1427709, 1427788..1428216, 1428277..1428522</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC823.04" routerLinkActive="active">SPAC823.04</a></td>
<td></td>
<td>changed first intron acceptor, which gave a different n-term exon and improved homology; coordinates changed from 2587859..2587901, 2588085..2588620 to 2587729..2587897, 2588085..2588620</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC688.08" routerLinkActive="active">SPAC688.08</a></td>
<td></td>
<td>deleted final exon, not supported by Solexa data; coordinates changed from 3123163..3125057, 3125099..3126310, 3126356..3126631, 3126671..3126989 to 3123163..3125057, 3125099..3126310, 3126356..3126653</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29E6.03c" routerLinkActive="active">SPAC29E6.03c</a></td>
<td></td>
<td>changed internal splice site, better intron, slightly improved homology; deleted final small exon, unsupported; coordinates changed from 4405871..4405889, 4405994..4407291, 4407447..4409096, 4409171..4409338 to 4405990..4407375, 4407447..4409096, 4409171..4409338</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC806.09c" routerLinkActive="active">SPAC806.09c</a></td>
<td></td>
<td>changed from 251726..252337, 252392..252752, 252945..252958 to 251726..252337, 252392..252752, 253590..253630</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-12-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC19G7.06" routerLinkActive="active">SPBC19G7.06</a></td>
<td></td>
<td>coordinates changed from 2359429..2359479, 2359547..2359693, 2359845..2359928, 2360064..2361092 to 2359429..2359479, 2359547..2359693, 2359845..2359928, 2360001..2361092</td>
<td>Christopher J. McInerny</td>
<td><span class="no-break">2007-12-07</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16D10.10" routerLinkActive="active">SPBC16D10.10</a></td>
<td></td>
<td>exon 4 splice altered; coordinates changed from 3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618920, 3619065..3619321, 3619366..3619540 to 3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618971, 3619077..3619321, 3619366..3619540</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16E9.16c" routerLinkActive="active">SPBC16E9.16c</a></td>
<td></td>
<td>This CDS has 2 frameshifts in GeneDB at present; coordinates changed from 1947985..1949069 , 1949218..1950058 , 1950253..1950255 to 1947985..1948953, 1948955..1950050, 1950053..1950255</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18079165">PMID:18079165</a></td>
<td><span class="no-break">2007-01-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16C6.02c" routerLinkActive="active">SPBC16C6.02c</a></td>
<td></td>
<td>complement(join(4322221..4322742, 4322970..4331555, 4331641..4331793, 4331856..4331968, 4332007..4332028)) was split into complement(join(4322829..4331555, 4331641..4331793, 4331856..4331968, 4332007..4332028)) and new gene SPBC16C6.14 - complement(4322221..4322622)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
<td><span class="no-break">2008-04-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC13G6.04" routerLinkActive="active">SPAC13G6.04</a></td>
<td>tim8</td>
<td>old join(180423..180528, 180586..180776), new join(180423..180528, 180586..180721, 180890..180911); added new 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1639.01c" routerLinkActive="active">SPAC1639.01c</a></td>
<td></td>
<td>old complement(join(251726..252337, 252392..252752, 253590..253630)), new complement(join(251726..251745, 251863..252337, 252392..252752, 253590..253630)); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC630.11" routerLinkActive="active">SPAC630.11</a></td>
<td>vps55</td>
<td>old join(366801..366817, 366873..366891, 366968..367040, 367123..367189, 367293..367348, 367395..367531), new join(366801..366817, 366873..366891, 366968..367040, 367105..367189, 367293..367348, 367395..367531); changed internal intron acceptor, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC24H6.01c" routerLinkActive="active">SPAC24H6.01c</a></td>
<td></td>
<td>old join(490016..490262, 490338..490771, 490812..490931, 490979..491149, 491186..491980), new join(490016..490262, 490338..490771, 490812..490931, 490979..491149, 491186..491858, 491905..492011); added new 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23C4.04c" routerLinkActive="active">SPAC23C4.04c</a></td>
<td></td>
<td>old 1036033..1036347, new join(1036033..1036342, 1036397..1036455); added intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1A6.03c" routerLinkActive="active">SPAC1A6.03c</a></td>
<td></td>
<td>old complement(1070139..1072013), new complement(join(1070061..1070138, 1070205..1072013)); new 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC56F8.04c" routerLinkActive="active">SPAC56F8.04c</a></td>
<td>coq2</td>
<td>old complement(1133226..1134302), new complement(join(1133226..1134283, 1134332..1134356)); added 5' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.08" routerLinkActive="active">SPAPB17E12.08</a></td>
<td></td>
<td>old join(1279573..1279644, 1279820..1279869, 1279953..1280325), new join(1279573..1279779, 1279820..1279869, 1279953..1280325); changed first donor to make n-terminal exon longer, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC9.06c" routerLinkActive="active">SPAC9.06c</a></td>
<td></td>
<td>old complement(join(1473692..1473885, 1473939..1474105, 1474153..1474212, 1474264..1474341, 1474460..1474539)), new complement(join(1473692..1473885, 1473939..1474105, 1474153..1474212, 1474264..1474341, 1474381..1474404, 1474460..1474539)); added new exon (2nd) improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC17A5.02c" routerLinkActive="active">SPAC17A5.02c</a></td>
<td>dbr1</td>
<td>old complement(join(1754109..1754310, 1754350..1755118, 1755164..1755285, 1755331..1755638)), new complement(join(1754033..1754145, 1754186..1754310, 1754350..1755118, 1755164..1755285, 1755331..1755638)); added new 5'intron/exon, now agrees with published date</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAPB1A10.16" routerLinkActive="active">SPAPB1A10.16</a></td>
<td></td>
<td>old 1863972..1864388, new join(1863972..1864360, 1864415..1864457); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC823.04" routerLinkActive="active">SPAC823.04</a></td>
<td></td>
<td>old 2587729..2587897 , 2588085..2588620, new join(2587729..2587897, 2588106..2588620); altered to use valid acceptor</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC823.09c" routerLinkActive="active">SPAC823.09c</a></td>
<td></td>
<td>old complement(join(2594922..2595693, 2595734..2595819)), new complement(join(2594777..2594913, 2594966..2595693, 2595734..2595819)); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB2B4.06" routerLinkActive="active">SPAPB2B4.06</a></td>
<td></td>
<td>old join(2722735..2723210, 2723273..2723300, 2723349..2723582), new join(2722735..2723210, 2723273..2723300, 2723349..2723559, 2723608..2723660); added new 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC16E8.12c" routerLinkActive="active">SPAC16E8.12c</a></td>
<td></td>
<td>old complement(join(3523136..3523644, 3524027..3524030)), new complement(join(3523136..3523644, 3523925..3523996, 3524044..3524098)); added 2 5' exons</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC15A10.10" routerLinkActive="active">SPAC15A10.10</a></td>
<td></td>
<td>old join(3697508..3697752, 3697795..3698368, 3698421..3698527, 3698570..3699538, 3699582..3699837), new join(3697508..3697752, 3697795..3698368, 3698421..3698527, 3698570..3699310, 3699359..3699538, 3699582..3699837); added in-frame intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1556.01c" routerLinkActive="active">SPAC1556.01c</a></td>
<td>rad50</td>
<td>old complement(join(3791855..3792229, 3792267..3795557, 3795601..3795684, 3795734..3795856)), new complement(join(3791831..3791851, 3791891..3792229, 3792267..3795557, 3795601..3795684, 3795734..3795856)); added small 3' intron and exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1556.03" routerLinkActive="active">SPAC1556.03</a></td>
<td>azr1</td>
<td>old join(3799244..3799680, 3799731..3800160), new join(3799244..3799680, 3799731..3800135, 3800184..3800241); added new 5' intron and exon/improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1F12.08" routerLinkActive="active">SPAC1F12.08</a></td>
<td></td>
<td>old join(3817464..3817803, 3817851..3817991, 3818035..3818543), new join(3817464..3817803, 3817851..3817991, 3818035..3818130, 3818170..3818543); added new intron, in-frame splice in final exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC2F3.18c" routerLinkActive="active">SPAC2F3.18c</a></td>
<td></td>
<td>old 3942398..3942730 new 3942299..3942364, 3942506..3942730; added small N term in-frame intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2C4.11c" routerLinkActive="active">SPAC2C4.11c</a></td>
<td>rpb28</td>
<td>old complement(4278729..4279505), new complement(join(4278729..4279405, 4279457..4279505)); added small N term in-frame intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC30.03c" routerLinkActive="active">SPAC30.03c</a></td>
<td>tsn1</td>
<td>old join(4394707..4394741, 4394799..4394967, 4395010..4395092, 4395131..4395491, 4395530..4395592), new complement(join(4394707..4394741, 4394799..4394967, 4395010..4395092, 4395131..4395367, 4395416..4395491, 439 5530..4395592)); added in-frame intron to (previous) 2nd exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC30C2.05" routerLinkActive="active">SPAC30C2.05</a></td>
<td>erv14</td>
<td>old complement(join(4687483..4688462, 4688513..4688831)), new complement(join(4687388..4687477, 4687522..4688462, 4688513..4688831)); added new 5' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC144.17c" routerLinkActive="active">SPAC144.17c</a></td>
<td></td>
<td>old complement(join(4687483..4688462, 4688513..4688831)), new complement(join(4687388..4687477, 4687522..4688462, 4688513..4688831)); added 3' intron and exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC4F10.08" routerLinkActive="active">SPAC4F10.08</a></td>
<td>mug126</td>
<td>old join(4844855..4845824, 4845863..4846203), new join(4844855..4845824, 4845863..4846063, 4846104..4846129); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC890.07c" routerLinkActive="active">SPAC890.07c</a></td>
<td>rmt1</td>
<td>old complement(join(5016072..5016215, 5016262..5016596, 5016649..5016943, 5016994..5017059, 5017138..5017252, 5017421..5017433, 5017700..5017703, 5017782..5017829)), new complement(join(5016020..5016026, 5016076..5016215, 5016262..5016596, 5016649..5016943, 5016994..5017059, 5017138..5017252, 5017421..5017433, 5017700..5017703, 5017782..5017829); added small N terminal exon, which added a single conserved cysteine</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC22E12.10c" routerLinkActive="active">SPAC22E12.10c</a></td>
<td>etp1</td>
<td>old complement(5037208..5039103), new complement(join(5037208..5037446, 5037492..5039103)); added in-frame splice</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC106.05c" routerLinkActive="active">SPBC106.05c</a></td>
<td>tim11</td>
<td>old complement(join(383527..383612, 383703..383874, 383980..384012)), new complement(join(383527..383612, 383703..383874, 383940..383960)); changed 5' exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC582.04c" routerLinkActive="active">SPBC582.04c</a></td>
<td></td>
<td>old complement(join(423269..423598, 423669..423738, 423782..423871, 423912..424373, 424424..424872, 424921..425160, 425223..425350, 425410..425446)), new complement(join(423269..423598, 423669..423738, 423782..423871, 423912..424373, 424424..424692, 424747..424872, 424921..425160, 425223..425350, 425410..425446)); added in-frame intron to (existing) exon 3</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC947.03c" routerLinkActive="active">SPBC947.03c</a></td>
<td></td>
<td>old 676629..676931, new join(676629..676908, 677057..677127); added 5' inton/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPJ4664.05" routerLinkActive="active">SPBPJ4664.05</a></td>
<td></td>
<td>old 706175..706666, new join(705932..706077, 706141..706666); added new 5' intron and exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC4.02c" routerLinkActive="active">SPBC4.02c</a></td>
<td></td>
<td>old complement(join(1185617..1186640, 1186738..1186846, 1187087..1187324)), new complement(join(1185617..1186640, 1186738..1186846, 1187087..1187162, 1187220..1187324)); added in-frame splice to first (existing) exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC8D2.17" routerLinkActive="active">SPBC8D2.17</a></td>
<td></td>
<td>old 1390183..1391238, new join(1390183..1391093, 1391147..1391222); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC4.02c" routerLinkActive="active">SPBC4.02c</a></td>
<td></td>
<td>old complement(join(2369064..2369160, 2369246..2369416, 2369455..2371243, 2371351..2371451, 2371507..2371583)), new complement(join(2369064..2369190, 2369246..2369416, 2369455..2371243, 2371351..2371451, 2371507..2371583)); changed accceptor for final intron, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBP4H10.12" routerLinkActive="active">SPBP4H10.12</a></td>
<td></td>
<td>old join(2895855..2895863, 2896001..2896507), new join(2895855..2895863, 2896001..2896459, 2896516..2896587); added new 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC2A9.07c" routerLinkActive="active">SPBC2A9.07c</a></td>
<td></td>
<td>old complement(2959078..2959902), new complement(join(2959078..2959129, 2959172..2959902)); added in-frame intron, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC2D10.15c" routerLinkActive="active">SPBC2D10.15c</a></td>
<td></td>
<td>old complement(join(2995324..2995579, 2995619..2995923, 2995971..2996066)), new complement(join(2995324..2995497, 2995534..2995579, 2995619..2995923, 2995971..2996066)); altered first intron, added in-frame intron, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC2D10.16" routerLinkActive="active">SPBC2D10.16</a></td>
<td></td>
<td>old join(2996452..2996463, 2996509..2996904), new join(2996452..2996463, 2996509..2996687, 2996738..2996879); added new intron in second exon, which created a new 3rd exon, and identified homology with Cenp-S (was previously sequence orphan)</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16D10.02" routerLinkActive="active">SPBC16D10.02</a></td>
<td>trm11</td>
<td>old join(3588806..3589595, 3589639..3589803, 3589845..3590123, 3590163..3590287), new join(3588806..3589595, 3589639..3589803, 3589845..3590123, 3590163..3590278, 3590327..3590386); added new 3' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC13G1.04c" routerLinkActive="active">SPBC13G1.04c</a></td>
<td></td>
<td>old complement(3732941..3733762), new complement(join(3732806..3732930, 3732994..3733762)); added new 3' intron and exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC31F10.12" routerLinkActive="active">SPBC31F10.12</a></td>
<td></td>
<td>old join(3773353..3773525, 3773581..3773678, 3773795..3773999, 3774059..3774116, 3774192..3774212), new join(3773308..3773318, 3773373..3773525, 3773581..3773678, 3773795..3773999, 3774059..3774116, 3774192..3774212); added new 5' intron and exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC21C3.07c" routerLinkActive="active">SPBC21C3.07c</a></td>
<td></td>
<td>old complement(join(3807920..3808607, 3808665..3808795), new complement(join(3807714..3807783, 3807831..3807891, 3807946..3808607, 3808665..3808795)); added two 3' exons</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPB2B2.18" routerLinkActive="active">SPBPB2B2.18</a></td>
<td></td>
<td>old join(4502439..4502463, 4502712..4502857, 4503068..4503286, 4503527..4503664), new join(4502692..4502725, 4502787..4502857, 4503068..4503286, 4503527..4503664); changed 5' intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC736.12c" routerLinkActive="active">SPCC736.12c</a></td>
<td>mmi1</td>
<td>old complement(join(338026..338178, 338232..338347, 338395..339354, 339483..339567)), new complement(join(337829..337985, 338030..338178, 338232..338347, 338395..339354, 339483..339567); added 3' exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC16C4.01" routerLinkActive="active">SPCC16C4.01</a></td>
<td>sif2</td>
<td>old join(658832..659139, 659223..659488, 659567..659628, 659709..660413), new join(658832..659139, 659223..659488, 659567..659628, 659709..660197, 660298..660322); added 3' intron which changed frame of final exon, shorter product, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC16C4.16c" routerLinkActive="active">SPCC16C4.16c</a></td>
<td></td>
<td>old complement(join(697862..698127 , 698180..698399)), new complement(join(697605..697752, 697875..698127, 698180..698399)); added 3' intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1020.11c" routerLinkActive="active">SPCC1020.11c</a></td>
<td></td>
<td>old join(757365..757424, 757506..757802), new join(757365..757424, 757506..757761, 757851..757861); added 3'intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1442.08c" routerLinkActive="active">SPCC1442.08c</a></td>
<td>cox17</td>
<td>old complement(join(1781505..1781563, 1781637..1781744, 1781830..1781914)), new complement(join(1781505..1781563, 1781637..1781744, 1781830..1781855, 1781920..1781935, 1781981..1782032)); added 2 small 5' exons, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1620.09c" routerLinkActive="active">SPCC1620.09c</a></td>
<td>tfg1</td>
<td>old complement(join(2161527..2162363, 2162405..2162500, 2162584..2163123)), new complement(join(2161331..2161515, 2161565..2162363, 2162405..2162500, 2162584..2163123)); added 5' exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1620.10" routerLinkActive="active">SPCC1620.10</a></td>
<td>cwf26</td>
<td>old 2163205..2164118, new join(2163205..2164118, 2164159..2164165); added 3' exon encoding single methionine</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC15A10.12c" routerLinkActive="active">SPAC15A10.12c</a></td>
<td></td>
<td>old complement(3709451..3709804), new complement(join(3709206..3709346, 3709499..3709804));</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB18E9.01" routerLinkActive="active">SPAPB18E9.01</a></td>
<td></td>
<td>old join(3974310..3974485, 3974541..3974629, 3974674..3974775, 3974879..3975132, 3975193..3975429, 3975485..3975832, 3975882..3975983), new join(3974310..3974485, 3974541..3974629, 3974674..3974775, 3974879..3975132, 3975193..3975429, 3975485..3975832, 3975882..3975962, 3976014..3976079); added small 3' exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16D10.10" routerLinkActive="active">SPBC16D10.10</a></td>
<td></td>
<td>old join(3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618971, 3619077..3619321, 3619366..3619540), new join(3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618918, 3618956..3619071, 3619113..3619321, 3619366..3619540); changed 2 introns (existing 4 and 5) to create a new exon 5, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1093.03" routerLinkActive="active">SPAC1093.03</a></td>
<td></td>
<td>old join(4613627..4613686, 4613723..4614081, 4614152..4614378, 4614438..4614954, 4614996..4616331), new join(4613627..4613686, 4613723..4614081, 4614164..4614378, 4614438..4614954, 4614996..4616331); improved distance between branch and acceptor, intron 2</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1142.01" routerLinkActive="active">SPAC1142.01</a></td>
<td></td>
<td>old join(3625447..3625460, 3625555..3627544), new 3625574..3627544; removed 5' exon and trimmed to new Met, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC11D3.07c" routerLinkActive="active">SPAC11D3.07c</a></td>
<td></td>
<td>old complement(join(118195..119827, 119930..120108)), new complement(118195..120108); removed in-frame splice, not required or supported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1751.02c" routerLinkActive="active">SPAC1751.02c</a></td>
<td></td>
<td>old complement(join(385755..385887, 385938..386043, 386107..386179, 386296..386310)), new complement(join(385755..385887, 385938..386043, 386107..386149)); remove 5' exon, trimmed to new Met, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1B3.10c" routerLinkActive="active">SPAC1B3.10c</a></td>
<td></td>
<td>old complement(join(4945933..4945984, 4946094..4948084)), new complement(4946090..4948084); removed 3' exon, unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23H3.04" routerLinkActive="active">SPAC23H3.04</a></td>
<td></td>
<td>old join(2499035..2499164, 2499325..2499353, 2499443..2499824, 2499875..2500236), new join(2499035..2499164, 2499160..2499353, 2499443..2499824, 2499875..2500236); changed 5' 1st and 2nd exon, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC26H5.04" routerLinkActive="active">SPAC26H5.04</a></td>
<td></td>
<td>old join(4122149..4123354, 4123535..4124518), new 4122149..4124518; removed unsupported in-frame splice</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2C4.06c" routerLinkActive="active">SPAC2C4.06c</a></td>
<td></td>
<td>old complement(join(4269367..4270554, 4270587..4270593, 4270638..4270674, 4270736..4270871)), new complement(join(4269367..4270554, 4270631..4270674, 4270736..4270871)); replaced 2 internal exons with a different singe exon</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC328.05" routerLinkActive="active">SPAC328.05</a></td>
<td></td>
<td>old join(3484695..3484698, 3484752..3485065, 3485241..3485342, 3485382..3485528, 3485582..3485807, 3485910..3485974, 3486056..3486502), new join(3484695..3484698, 3484752..3485065, 3485241..3485342, 3485382..3485528, 3485582..3485974, 3486056..3486502); removed in-frame splice (exon 5), unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC3A12.05c" routerLinkActive="active">SPAC3A12.05c</a></td>
<td></td>
<td>old complement(join(1426213..1427148, 1427189..1429617, 1429709..1429836, 1429882..1429913)), new complement(join(1426213..1427148, 1427189..1429617, 1429709..1429830, 1429882..1429913)); improved distance beteen branch and acceptor</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC589.02c" routerLinkActive="active">SPAC589.02c</a></td>
<td></td>
<td>old complement(join(3095075..3098590, 3098626..3098775, 3098878..3098883)), new complement(join(3095075..3098590, 3098626..3098781, 3098878..3098883)); improved branch/acceptor for 1st intron</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC6F12.08c" routerLinkActive="active">SPAC6F12.08c</a></td>
<td></td>
<td>old complement(join(1322439..1323955, 1323999..1324140, 1324195..1324234, 1324299..1324336)), new complement(join(1322439..1323955, 1323999..1324140, 1324195..1324252, 1324299..1324336)); improved branch acceptor for intron 1</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC8C9.19" routerLinkActive="active">SPAC8C9.19</a></td>
<td></td>
<td>old join(3660430..3660438, 3660491..3660556, 3660607..3660713, 3660784..3660886), new join(3660430..3660438, 3660491..3660556, 3660607..3660717); deleted final exon, unsupported, improved length and alignment to orthologs</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC9G1.13c" routerLinkActive="active">SPAC9G1.13c</a></td>
<td></td>
<td>old complement(join(1995208..1996148, 1996325..1996413, 1996478..1996683)), new complement(join(1995208..1996148, 1996191..1996268, 1996325..1996413, 1996478..1996683)); added a new 3rd exon, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.03" routerLinkActive="active">SPAPB17E12.03</a></td>
<td></td>
<td>old join(1271889..1271972, 1272015..1272386, 1272483..1272962), new join(1271889..1271972, 1272015..1272962); removed in-frame splice, unsupported, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC15D4.13c" routerLinkActive="active">SPBC15D4.13c</a></td>
<td></td>
<td>old complement(join(3032492..3032710, 3032761..3032822, 3032865..3033222)), new complement(join(3032492..3032710, 3032761..3032822, 3032889..3033222)); better donor for first intron</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16C6.03c" routerLinkActive="active">SPBC16C6.03c</a></td>
<td></td>
<td>old complement(join(4332356..4332454, 4332746..4333123)), new complement(4332698..4333123); final exon, unsupported removed</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC19G7.06" routerLinkActive="active">SPBC19G7.06</a></td>
<td></td>
<td>old join(2359429..2359479 , 2359547..2359693 , 2359845..2359928 , 2360001..2361092), new join(2359429..2359479, 2359547..2359693, 2359845..2359928, 2360064..2361092); altered 3rd intron branch acceptor to agree with published</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC21.03c" routerLinkActive="active">SPBC21.03c</a></td>
<td></td>
<td>old complement(join(3217985..3218424, 3218467..3218491, 3218693..3218947)), new complement(join(3217985..3218424, 3218462..3218491, 3218580..3218651, 3218704..3218947)); changed 2 internal introns, now agrees with transread data, but still potential to be incorrect</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC32H8.08c" routerLinkActive="active">SPBC32H8.08c</a></td>
<td></td>
<td>old complement(join(1465811..1466035, 1466138..1467057, 1467311..1467456, 1467514..1467539)), new complement(join(1465811..1466035 , 1466138..1467057 , 1467311..1467431)); removed 5' intron/exon (unsupported) and trimmed to next met</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP35G2.06c" routerLinkActive="active">SPBP35G2.06c</a></td>
<td></td>
<td>old complement(join(973049..974181, 974223..974577, 974617..975002, 975044..975137, 975172..976632)), new complement(join(973049..974181, 974223..974577, 974647..975002, 975044..975137, 975172..976632)); changed donor for intron 3</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1393.06c" routerLinkActive="active">SPCC1393.06c</a></td>
<td></td>
<td>old complement(join(804627..805627, 805678..805800, 805907..806009)), new complement(join(804627..805627 , 805678..805800 , 805849..805921 , 805965..806009)); changed first intron and added a new 2nd exon, improved homology, also confirmed by transread data</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1450.14c" routerLinkActive="active">SPCC1450.14c</a></td>
<td></td>
<td>old complement(join(1759295..1760998, 1761117..1761128)), new complement(1759295..1760998); deleted first exon, unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1827.02c" routerLinkActive="active">SPCC1827.02c</a></td>
<td></td>
<td>old complement(join(2377039..2378078, 2378197..2378221)), new complement(2377039..2378127); deleted first exon, unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC63.10c" routerLinkActive="active">SPCC63.10c</a></td>
<td></td>
<td>old complement(join(854651..855695, 855766..855800, 855839..855926, 855984..856213)), new complement(join(854651..855926, 855984..856213)); deleted 2nd and 3rd introns, were not required or supported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC6F6.16c" routerLinkActive="active">SPAC6F6.16c</a></td>
<td></td>
<td>merged with SPAC6F6.18c and five introns added</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18535244">PMID:18535244</a></td>
<td><span class="no-break">2008-06-09</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC962.05" routerLinkActive="active">SPCC962.05</a></td>
<td></td>
<td>old join(554521..554722, 554792..556200), new join(554521..554744, 554865..556200)</td>
<td>pers. comm. Matthew O'Connell</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC338.08" routerLinkActive="active">SPCC338.08</a></td>
<td>ctp1</td>
<td>old coordinates complement(1358962..1359819), new complement(join(1358885..1359082, 1359121..1359819)); further altered to complement(join(1358891..1359082, 1359127..1359819)) in accordance with paper</td>
<td>pers. comm. Paul Russell; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18378696">PMID:18378696</a></td>
<td><span class="no-break">2008-01-11</span>;<span class="no-break">2008-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC2G2.09c" routerLinkActive="active">SPBC2G2.09c</a></td>
<td>crs1</td>
<td>added new C-terminal exon, deleted exon 3452822..3452877, extended intron 2 by 4 nucleotides</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-09</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC31F10.08" routerLinkActive="active">SPBC31F10.08</a></td>
<td>mde2</td>
<td>old coordinates 3764820..3765449, new join(3764820..3765437, 3765491..3765709)</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-18</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC651.06" routerLinkActive="active">SPBC651.06</a></td>
<td>mug166</td>
<td>merged with SPBC651.07</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-26</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC18E5.14c" routerLinkActive="active">SPBC18E5.14c</a></td>
<td></td>
<td>merged with SPBC18E5.09c</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-31</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC959.05c" routerLinkActive="active">SPAC959.05c</a></td>
<td></td>
<td>merged with SPAC959.06c; splicing is tentative</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-08-15</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC11B10.10c" routerLinkActive="active">SPBC11B10.10c</a></td>
<td>pht1</td>
<td>N terminal shortened to 2nd methionine; starting coordinate changed from 1503157 to 1503064</td>
<td>pers. comm. L Buchanan and F Stewart</td>
<td><span class="no-break">2008-09-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC27D7.12c" routerLinkActive="active">SPAC27D7.12c</a></td>
<td>but1</td>
<td>extended N-terminal to use first MET as start</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14623327">PMID:14623327</a></td>
<td><span class="no-break">2008-11-11</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC962.05" routerLinkActive="active">SPCC962.05</a></td>
<td>ast1</td>
<td>intron position moved</td>
<td>pers comm. M. O' Connell</td>
<td><span class="no-break">2008-12-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB1E7.14" routerLinkActive="active">SPAPB1E7.14</a></td>
<td>iec5</td>
<td>old join(3322106..3322331, 3322379..3322512); new join(3321979..3322018, 3322071..3322331, 3322379..3322512)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19040720">PMID:19040720</a></td>
<td><span class="no-break">2008-12-18</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC31G5.12c" routerLinkActive="active">SPAC31G5.12c</a></td>
<td>maf1</td>
<td>old complement(join(3008184..3008795, 3008914..3008957, 3009044..3009089)); new complement(join(3008184..3008795, 3009160..3009258, 3009312..3009317))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15590667">PMID:15590667</a></td>
<td><span class="no-break">2008-12-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC725.12" routerLinkActive="active">SPBC725.12</a></td>
<td>nbl1</td>
<td>old join(1227762..1227876, 1227926..1228098); new join(1227762..1228094, 1228166..1228307)</td>
<td>pers. comm. Kathy Gould</td>
<td><span class="no-break">2009-01-30</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC14C8.09c" routerLinkActive="active">SPBC14C8.09c</a></td>
<td></td>
<td>merged with SPBC14C8.08c; frameshifted</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC23G7.06c" routerLinkActive="active">SPBC23G7.06c</a></td>
<td></td>
<td>frameshifted; now a single exon</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC13E7.01" routerLinkActive="active">SPBC13E7.01</a></td>
<td></td>
<td>frameshifted</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC29A3.08" routerLinkActive="active">SPBC29A3.08</a></td>
<td></td>
<td>frameshifted; truncated to 2048336..2049892, 2049894..2049896</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1A4.06c" routerLinkActive="active">SPBC1A4.06c</a></td>
<td></td>
<td>frameshifted; C term exon changed from 1987076..1987675 to 1987044....1987675</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F3.11c" routerLinkActive="active">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>frameshifted; join(682874..683034, 683102..683109, 683160..683446) to join(682874..682994, 682996..683109, 683160..683446)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1F8.07c" routerLinkActive="active">SPAC1F8.07c</a></td>
<td></td>
<td>frameshifted; complement(101760..103544) to complement(join(101836..101872, 101872..103544))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1442.04c" routerLinkActive="active">SPCC1442.04c</a></td>
<td></td>
<td>frameshifted; complement(1774200..1775429) to complement(join(1773925..1774239, 1774236..1775429))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC548.04" routerLinkActive="active">SPCC548.04</a></td>
<td></td>
<td>final exon 223687..223852 to 223687..223796, 223853.223860</td>
<td>pers. comm. Marc Feuermann, Uniprot (SIB)</td>
<td><span class="no-break">2009-02-23</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP23A10.07" routerLinkActive="active">SPBP23A10.07</a></td>
<td>rpa2</td>
<td>truncated N-term to agree with orthologs; 2007960 -&gt; 2008119</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2009-03-30</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16E9.16c" routerLinkActive="active">SPBC16E9.16c</a></td>
<td>lsd90</td>
<td>gene structure revised to remove introns, genome sequence contains 2 frameshifts</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18079165">PMID:18079165</a></td>
<td><span class="no-break">2008-01-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC800.02" routerLinkActive="active">SPBC800.02</a></td>
<td>whi5</td>
<td>increased intron by 42bp; old coordinates 253148..253231, 253310..253984, new coordinates 253148..253231, 253352..253984</td>
<td>pers. comm. Rob de Bruin</td>
<td><span class="no-break">2009-07-27</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC18H10.08c" routerLinkActive="active">SPBC18H10.08c</a></td>
<td>ubp4</td>
<td>missing 2 N-terminal exons; old - complement(1784219..1785535); new - complement(join(1784219..1785748, 1785937..1786080, 1786141..1786248))</td>
<td>pers. comm. I. Kouranti</td>
<td><span class="no-break">2009-09-10</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC800.02" routerLinkActive="active">SPBC800.02</a></td>
<td>whi5</td>
<td>increased intron by 42bp; old coordinates 253148..253231, 253310..253984, new coordinates 253148..253231, 253352..253984</td>
<td>pers. comm. Rob de Bruin</td>
<td><span class="no-break">2009-07-27</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC2A9.07c" routerLinkActive="active">SPBC2A9.07c</a></td>
<td></td>
<td>removed intron</td>
<td>pers. comm. Cathrine Arnason Boe</td>
<td><span class="no-break">2009-09-10</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPNCRNA.445" routerLinkActive="active">SPNCRNA.445</a></td>
<td>snoR61</td>
<td>updated to extend 3' and added splice and identified as snR6</td>
<td>pers. comm. J. Matthews</td>
<td><span class="no-break">2008-08-08</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29B12.08" routerLinkActive="active">SPAC29B12.08</a></td>
<td></td>
<td>replaced exon 5428786..5428883 with 5429038.. 5429045</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2009-12-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC458.04c" routerLinkActive="active">SPAC458.04c</a></td>
<td></td>
<td>coordinates updated from 4737605..4738147, 4738189..4738283, 4738330..4738714 to 4737605..4738147, 4738189..4738283, 4738330..4738740, 4738779..4738854</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-02-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC5E4.10c" routerLinkActive="active">SPCC5E4.10c</a></td>
<td></td>
<td>coordinates updated from 651622..651905 , 651973..651994 to 651622..651905, 651973..652045, 652115..652225, 652280..652371, 652419..652455</td>
<td>pers. comm. K. Gould</td>
<td><span class="no-break">2010-03-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC26H5.02c" routerLinkActive="active">SPAC26H5.02c</a></td>
<td>nse2</td>
<td>second intron extended to 115 nt</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-08-20</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC27F1.09c" routerLinkActive="active">SPAC27F1.09c</a></td>
<td>prp10</td>
<td>added intron to match fully spliced</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9837997">PMID:9837997</a></td>
<td><span class="no-break">2010-09-27</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22G7.04" routerLinkActive="active">SPAC22G7.04</a></td>
<td>ubp13</td>
<td>coordinates updated to join(732880..732897, 732942..733040, 733085..733251, 733369..733495, 733536..734113, 734151..735457, 735495..736465)</td>
<td>pers. comm. James Iben</td>
<td><span class="no-break">2010-10-20</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC3H1.04c" routerLinkActive="active">SPAC3H1.04c</a></td>
<td>mdm31</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC12D12.09" routerLinkActive="active">SPBC12D12.09</a></td>
<td>rev7</td>
<td></td>
<td>pers. comm. Nicole Kosarek</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1348.12" routerLinkActive="active">SPBC1348.12</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC1778.02" routerLinkActive="active">SPBC1778.02</a></td>
<td>rap1</td>
<td></td>
<td>pers. comm. Yasushi Hiraoka</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC3E7.04c" routerLinkActive="active">SPBC3E7.04c</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC8D2.07c" routerLinkActive="active">SPBC8D2.07c</a></td>
<td>sfc9</td>
<td></td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPB21E7.02c" routerLinkActive="active">SPBPB21E7.02c</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC306.10" routerLinkActive="active">SPCC306.10</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC622.17" routerLinkActive="active">SPCC622.17</a></td>
<td>apn1</td>
<td>added 2 5' exons</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21193357">PMID:21193357</a></td>
<td><span class="no-break">2011-01-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC823.04" routerLinkActive="active">SPAC823.04</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC688.11" routerLinkActive="active">SPAC688.11</a></td>
<td>end4</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29A4.03c" routerLinkActive="active">SPAC29A4.03c</a></td>
<td>mrps9</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC688.04c" routerLinkActive="active">SPAC688.04c</a></td>
<td>gst3</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.14c" routerLinkActive="active">SPAPB17E12.14c</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC29A10.17" routerLinkActive="active">SPBC29A10.17</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP8B7.31" routerLinkActive="active">SPBP8B7.31</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC651.06" routerLinkActive="active">SPBC651.06</a></td>
<td>mug166</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC25H2.11c" routerLinkActive="active">SPBC25H2.11c</a></td>
<td>spt7</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29A4.03c" routerLinkActive="active">SPAC29A4.03c</a></td>
<td>mrps9</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC4G3.12c" routerLinkActive="active">SPCC4G3.12c</a></td>
<td>not2</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1322.16" routerLinkActive="active">SPCC1322.16</a></td>
<td>phb2</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC13E7.01" routerLinkActive="active">SPBC13E7.01</a></td>
<td>cwf22</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC4F6.10" routerLinkActive="active">SPBC4F6.10</a></td>
<td>vps901</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16H5.12c" routerLinkActive="active">SPBC16H5.12c</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC3A11.03" routerLinkActive="active">SPAC3A11.03</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC1711.10c" routerLinkActive="active">SPBC1711.10c</a></td>
<td>npl4</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1709.12" routerLinkActive="active">SPBC1709.12</a></td>
<td>rid1</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC13F5.07c" routerLinkActive="active">SPAC13F5.07c</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC18B11.05" routerLinkActive="active">SPAC18B11.05</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC18B11.05" routerLinkActive="active">SPAC18B11.05</a></td>
<td>gpi18</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPACUNK4.14" routerLinkActive="active">SPACUNK4.14</a></td>
<td>mdb1</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC947.10" routerLinkActive="active">SPBC947.10</a></td>
<td>dsc1</td>
<td>N-terminal extended to use upstream methionine</td>
<td>pers. comm. E. Stewart</td>
<td><span class="no-break">2011-02-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC22F3.11c" routerLinkActive="active">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>gene structure updated</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/21511999">PMID:21511999</a></td>
<td><span class="no-break">2011-03-11</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1006.03c" routerLinkActive="active">SPAC1006.03c</a></td>
<td>red1</td>
<td>intron donor corrected</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC29A3.06" routerLinkActive="active">SPBC29A3.06</a></td>
<td></td>
<td>N terminal extended</td>
<td></td>
<td><span class="no-break">2012-01-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1281.07c" routerLinkActive="active">SPCC1281.07c</a></td>
<td></td>
<td>N terminal extended by 24 amino acids</td>
<td></td>
<td><span class="no-break">2012-01-05</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC4B3.05c" routerLinkActive="active">SPCC4B3.05c</a></td>
<td>hem12</td>
<td>N terminal extended by 4 amino acids based on homology</td>
<td></td>
<td><span class="no-break">2012-01-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC2F3.13c" routerLinkActive="active">SPAC2F3.13c</a></td>
<td></td>
<td>removed N terminal region overlapping with plp1</td>
<td></td>
<td><span class="no-break">2012-12-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2F3.13c" routerLinkActive="active">SPAC9G1.15c</a></td>
<td>mzt1</td>
<td>N-terminal shortened to use downstream methionine</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/23885124">PMID:23885124</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24006493">PMID:24006493</a></td>
<td><span class="no-break">2013-11-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC25B2.07c" routerLinkActive="active">SPBC25B2.07c</a></td>
<td>mmb1</td>
<td>N-terminal shortened to use downstream methionine</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/21856157">PMID:21856157</a></td>
<td><span class="no-break">2013-12-06</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC3D6.04c" routerLinkActive="active">SPBC3D6.04c</a></td>
<td>mad1</td>
<td>N-terminal shortened to use downstream methionine; removed 13 amino acids</td>
<td>pers. comm. Silke Hauf</td>
<td><span class="no-break">2013-12-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC5H10.06c" routerLinkActive="active">SPAC5H10.06c</a></td>
<td>adh4</td>
<td>N-terminal shortened to use downstream methionine; removed 43 amino acids; UTR exon annotated</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24003116">PMID:24003116</a></td>
<td><span class="no-break">2014-01-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPNCRNA.98" routerLinkActive="active">SPNCRNA.98</a></td>
<td>srp7</td>
<td>Coordinates changed from 4268566-4269064 to 4268662-4268916, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2837764">PMID:2837764</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16453822">PMID:16453822</a></td>
<td><span class="no-break">2014-06-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.01" routerLinkActive="active">SPSNRNA.01</a></td>
<td>snu1</td>
<td>Coordinates changed from 3019819-3020472 to 3020205-3020353, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2188102">PMID:2188102</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPSNRNA.02" routerLinkActive="active">SPSNRNA.02</a></td>
<td>snu2</td>
<td>Coordinates changed from 959770-959305 to 959856-959370, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/3244367">PMID:3244367</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.03" routerLinkActive="active">SPSNRNA.03</a></td>
<td>snu3</td>
<td>Coordinates changed from 988266-987631 to 987825-987570, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/3194197">PMID:3194197</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPSNRNA.04" routerLinkActive="active">SPSNRNA.04</a></td>
<td>snu4</td>
<td>Coordinates changed from 467025-467823 to 467233-4683361, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2795654">PMID:2795654</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.05" routerLinkActive="active">SPSNRNA.05</a></td>
<td>snu5</td>
<td>Coordinates changed from 3236617-3237051 to 3236867-3236986, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2587274">PMID:2587274</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPSNRNA.06" routerLinkActive="active">SPSNRNA.06</a></td>
<td>snu6</td>
<td>Coordinates changed from (2562559-2562374,​2562323-2562085) to (2562276-2562323,​2562374-2562427), to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2909894">PMID:2909894</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.07" routerLinkActive="active">SPSNRNA.07</a></td>
<td>snu32</td>
<td>Coordinates changed from 3959174-3958768 to 3959086-3958832, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/1560765">PMID:1560765</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC110.05" routerLinkActive="active">SPAC110.05</a></td>
<td></td>
<td>Coordinates changed from complement(1905657..1905884) to complement(1905728..1905820) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F3.04" routerLinkActive="active">SPAC22F3.04</a></td>
<td>mug62</td>
<td>Coordinates changed from complement(join(698033..698177, 698223..698441, 698480..698841, 698884..699104, 699148..699363, 699410..702533)) to complement(join(698033..698177, 698223..698441, 698480..698841, 698884..699104, 699148..699363, 699410..702557, 702603..702755)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1D4.08" routerLinkActive="active">SPAC1D4.08</a></td>
<td>pis1</td>
<td>Coordinates changed from join(650814..650893,​651173..651848) to join(650488..650545,​651193..651848) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC839.02" routerLinkActive="active">SPBC839.02</a></td>
<td></td>
<td>Coordinates changed from join(597611..599124,​599256..599334) to 597611..599125 based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC823.02" routerLinkActive="active">SPAC823.02</a></td>
<td></td>
<td>Coordinates changed from join(2583804..2583902,​2583941..2583973) to join(2583863..2583902,​2583941..2584047) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437"></a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC146.01" routerLinkActive="active">SPBC146.01</a></td>
<td>med15</td>
<td>Coordinates changed from join(996843..997019,​997066..1000080) to join(996797..996870,​996917..997019,​997066..1000080) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC27B12.10c" routerLinkActive="active">SPBC27B12.10c</a></td>
<td>tom7</td>
<td>Coordinates changed from complement(join(1344032..1344157,​1344214..1344237)) to complement(join(1343913..1343934,​1343989..1344025,​1344082..1344157,​1344214..1344237)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC162.07" routerLinkActive="active">SPCC162.07</a></td>
<td>ent1</td>
<td>Coordinates changed from complement(join(1571648..1573733,​1574016..1574050)) to complement(join(1571648..1573733,​1574258..1574280)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC417.03" routerLinkActive="active">SPCC417.03</a></td>
<td></td>
<td>Coordinates changed from join(1672003..1672051,​1672111..1672364) to join(1672003..1672051,​1672243..1672332) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC320.09" routerLinkActive="active">SPCC320.09</a></td>
<td>hem15</td>
<td>Coordinates changed from complement(join(149153..150163,​150279..150323,​150374..150472)) to complement(join(149153..150163,​150279..150323,​150374..150589)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1486.05" routerLinkActive="active">SPAC1486.05</a></td>
<td>nup189</td>
<td>Coordinates changed from join(3197028..3197443,​3197530..3202450) to join(3197028..3197528,​3197528..3202450)</td>
<td>pers. comm. Y. Hiraoka</td>
<td><span class="no-break">2015-04-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC13G1.04c" routerLinkActive="active">SPBC13G1.04c</a></td>
<td>abh1</td>
<td>N-terminal shortened to use downstream methionine</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22365419">PMID:22365419</a></td>
<td><span class="no-break">2015-10-13</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC29A4.23" routerLinkActive="active">SPAC29A4.23</a></td>
<td></td>
<td>Frameshifted; intron/exon boundary changes; coordinates changed to join(5139024..5139190,​5139192..5139424,​5139427..5139575)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26494834">PMID:26494834</a></td>
<td><span class="no-break">2015-11-28</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F3.11c" routerLinkActive="active">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>Coordinates changed from complement(890220..890546) to join(682874..682996,​682998..683109,​683160..683446) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22365419">PMID:22365419</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2016-03-17</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC21B10.12" routerLinkActive="active">SPBC21B10.12</a></td>
<td>rec6</td>
<td>Coordinates changed from complement(join(1649038..1649472,​1649558..1649671)) to complement(join(1649038..1649472,​1649558..1649727,​1649815..1649878,​1650043..1650156))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26917764">PMID:26917764</a></td>
<td><span class="no-break">2016-04-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC3A12.04c" routerLinkActive="active">SPAC3A12.04c</a></td>
<td>rpp1</td>
<td>Frameshifted by Chr_I:1424708!CA→C; coordinates changed from complement(join(1424697..1425161,​1425231..1425329,​1425372..1425430,​1425478..1425562)) to complement(join(1424660..1424708,​1424710..1425161,​1425231..1425329,​1425372..1425430,​1425478..1425562))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAP27G11.10c" routerLinkActive="active">SPAP27G11.10c</a></td>
<td>nup184</td>
<td>Frameshifted by Chr_I:1625092!T→TC; coordinates changed from complement(join(1625076..1629753,​1629825..1629841)) to complement(join(1624885..1625092,​1625092..1629753,​1629825..1629841))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC17G8.01c" routerLinkActive="active">SPAC17G8.01c</a></td>
<td>trl1</td>
<td>Frameshifted by Chr_I:2343703!G→GA; coordinates changed from complement(join(2341346..2343706,​2343754..2343756)) to complement(join(2341346..2343703,​2343703..2343756))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC22F3.11c" routerLinkActive="active">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>Frameshifted by Chr_I:682993!TC-&gt;T; coordinates changed to join(682874..682996,​682998..683109,​683160..683446)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC823.04" routerLinkActive="active">SPAC823.04</a></td>
<td>rrp36</td>
<td>Frameshifted by Chr_I:2588066!C→CA and Chr_I:2588021!C→CA; coordinates changed from join(2587729..2587897,​2587952..2588028,​2588031..2588143,​2588182..2588620) to join(2587729..2587897,​2587952..2588025,​2588025..2588066,​2588066..2588620)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC688.08" routerLinkActive="active">SPAC688.08</a></td>
<td>srb8</td>
<td>Frameshifted by Chr_I:3125118!A→AT; coordinates changed from join(3123163..3125057,​3125099..3126310,​3126356..3126653) to join(3123163..3125118,​3125118..3126310,​3126356..3126653)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC1486.05" routerLinkActive="active">SPAC1486.05</a></td>
<td>nup189</td>
<td>Frameshifted by Chr_I:3197528!A→AG; coordinates changed from join(3197028..3197529,​3197529..3202450) to join(3197028..3197529,​3197529..3202450)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC3A11.09" routerLinkActive="active">SPAC3A11.09</a></td>
<td>sod22</td>
<td>Frameshifted by Chr_I:3450130!GT→G; coordinates changed from complement(join(3450114..3452269,​3452485..3452608)) to complement(join(3450056..3450130,​3450132..3452269,​3452485..3452608))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC1071.01c" routerLinkActive="active">SPAC1071.01c</a></td>
<td>pta1</td>
<td>Frameshifted by Chr_I:3855790!GT→G; coordinates changed from complement(3855780..3857792) to complement(join(3855626..3855790,​3855792..3857792))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC29E6.03c" routerLinkActive="active">SPAC29E6.03c</a></td>
<td>uso1</td>
<td>Frameshifted by Chr_I:4407494!T→TG; coordinates changed from complement(join(4405990..4407375,​4407447..4409096,​4409171..4409338)) to complement(join(4405990..4407494,​4407494..4409096,​4409171..4409338))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC29E6.04" routerLinkActive="active">SPAC29E6.04</a></td>
<td>nnf1</td>
<td>Frameshifted by Chr_I:4410191!CG→C; coordinates changed from join(4409773..4410192, 4410194..4410391) to join(4409773..4410190,​4410192..4410391)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC29A4.03c" routerLinkActive="active">SPAC29A4.03c</a></td>
<td>mrps9</td>
<td>Frameshifted by Chr_I:5142627!A→AG; coordinates changed from join(5142407..5142628,​5142631..5143224) to join(5142407..5142626,​5142626..5143224)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC16E9.16c" routerLinkActive="active">SPBC16E9.16c</a></td>
<td>lsd90</td>
<td>Frameshifted by Chr_II:1948953!GA→G and Chr_II:1950050!A→AG; coordinates changed from complement(join(1947985..1948953,​1948955..1950050,​1950053..1950255)) to complement(join(1947985..1948952,​1948954..1950048,​1950048..1950255))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPBC1E8.03c" routerLinkActive="active">SPBC1E8.03c</a></td>
<td></td>
<td>Frameshifted by Chr_II:1960392!A→AG; coordinates changed from complement(1960384..1961817) to complement(join(1960166..1960392,​1960392..1961817))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC1A4.06c" routerLinkActive="active">SPBC1A4.06c</a></td>
<td>tam41</td>
<td>Frameshifted by Chr_II:1987101!CG→C and Chr_II:1987117!TG→T; coordinates changed from complement(join(1987044..1987099,​1987102..1987675,​1987737..1988015,​1988081..1988353)) to complement(join(1987044..1987100,​1987102..1987117,​1987119..1987675,​1987737..1988015,​1988081..1988353))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPBC29A3.08" routerLinkActive="active">SPBC29A3.08</a></td>
<td>pof4</td>
<td>Frameshifted by Chr_II:2053516!G→GC; coordinates changed from join(2053033..2053517,​2053519..2053600) to join(2053033..2053517,​2053517..2053823)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC23G7.06c" routerLinkActive="active">SPBC23G7.06c</a></td>
<td></td>
<td>Frameshifted by Chr_II:2108180!T→TA; coordinates changed from complement(join(2106448..2108178,​2108181..2108786)) to complement(join(2106448..2108180,​2108180..2108786))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPBC14C8.09c" routerLinkActive="active">SPBC14C8.09c</a></td>
<td></td>
<td>Frameshifted by Chr_II:2219928!A→AT; coordinates changed from complement(join(2219430..2219934,​2219937..2220322)) to complement(join(2219430..2219928,​2219928..2220322))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC4F6.10" routerLinkActive="active">SPBC4F6.10</a></td>
<td>vps901</td>
<td>Frameshifted by Chr_II:2709414!G→GC; coordinates changed from join(2708076..2709415,​2709418..2709793) to join(2708076..2709414,​2709414..2709793)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPBC32F12.08c" routerLinkActive="active">SPBC32F12.08c</a></td>
<td>duo1</td>
<td>Frameshifted by Chr_II:2798040!CT→C; coordinates changed from complement(2798007..2798507) to complement(join(2797652..2798040,​2798042..2798507))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC13E7.01" routerLinkActive="active">SPBC13E7.01</a></td>
<td>cwf22</td>
<td>Frameshifted by Chr_II:3040332!C→CG; coordinates changed from join(3037988..3040331,​3040334..3040650) to join(3037988..3040332,​3040332..3040650)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPBC16D10.10" routerLinkActive="active">SPBC16D10.10</a></td>
<td>tad2</td>
<td>Frameshifted by Chr_II:3619003!A→AG; coordinates changed from join(3618117..3618179,​3618228..3618617,​3618658..3618702,​3618747..3618918,​3618956..3619071,​3619113..3619321,​3619366..3619540) to join(3618117..3618179,​3618228..3618617,​3618658..3618702,​3618747..3618920,​3618968..3619003,​3619003..3619071,​3619113..3619321,​3619366..3619540)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPAC12B10.09" routerLinkActive="active">SPAC12B10.09</a></td>
<td>pet801</td>
<td>Coordinates changed from join(4588250..4588510,​4588567..4588723,​4588869..4589323,​4589410..4589574) to join(4588487..4588510,​4588567..4588723,​4588869..4589323,​4589410..4589574)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="//gene/SPAC3A11.06" routerLinkActive="active">SPAC3A11.06</a></td>
<td>mvp1</td>
<td>Frameshifted by Chr_I:3460318!T→T; coordinates changed from complement(join(3459233..3459436,​3459492..3459665,​3459705..3459897,​3459941..3460185,​3460234..3460364,​3460425..3460442,​3460508..3460560,​3460642..3461263,​3461323..3461583,​3461766..3461859)) to complement(join(3459233..3459436,​3459492..3459665,​3459705..3459897,​3459941..3460185,​3460234..3460318,​3460318..3460372,​3460425..3460442,​3460508..3460560,​3460642..3461263,​3461323..3461583,​3461766..3461859))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="//gene/SPBC530.13" routerLinkActive="active">SPBC530.13</a></td>
<td>lsc1</td>
<td>Frameshifted by Chr_II:825012!G→A; coordinates changed from join(824374..824534,​824579..824660,​824874..825011,​825065..825188,​825245..825747) to join(824374..824534,​824579..824660,​824874..824999,​825065..825188,​825245..825747)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-17</span></td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'index'" class="docs-content">
<h1 id="genome-status">Genome Status</h1>
<table>
<tbody>
<tr class="odd">
<td><a routerLink="/status/statistics" routerLinkActive="active">Genome Statistics</a></td>
<td>Latest information on the status of the genome</td>
</tr>
<tr class="even">
<td><a routerLink="/status/sequencing-status" routerLinkActive="active">Sequencing Status</a></td>
<td>Contig size, gap status and progress, including <a href="status/centromeres">centromeres</a> and <a href="status/telomeres">telomeres</a></td>
</tr>
<tr class="odd">
<td><a routerLink="/status/sequence-updates" routerLinkActive="active">Sequence Updates</a></td>
<td>Changes to the genome sequence since 07/03.</td>
</tr>
<tr class="even">
<td><a routerLink="/status/sequence-updates-pending" routerLinkActive="active">Sequence Updates Pending</a></td>
<td>Pending changes, mainly from the Broad Institute, some supported by data from other sources</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/gene-coordinate-changes" routerLinkActive="active">Gene Coordinate Changes</a></td>
<td>Changes to coordinates of individual genes since publication</td>
</tr>
<tr class="even">
<td><a routerLink="/status/new-and-removed-genes" routerLinkActive="active">New and Removed Genes</a></td>
<td>Genes identified or removed since publication</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/gene-characterisation" routerLinkActive="active">Gene Characterisation</a></td>
<td>Current counts of protein coding gene status, as published in small scale experiments</td>
</tr>
<tr class="even">
<td><a routerLink="/status/gene-characterisation-statistics-history" routerLinkActive="active">Historical Gene Characterisation</a></td>
<td>Changes in gene characterisation statistics over time</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/priority-unstudied-genes" routerLinkActive="active">Priority Unstudied Genes</a></td>
<td>Unstudied nuclear genes conserved 1:1 in human</td>
</tr>
</tbody>
</table>
<p>Note: Many older <em>S. pombe</em> sequence submissions to the DNA databases (International Nucleotide Sequence Database Collaboration databases,​ i.e. ENA, GenBank, DDBJ) contain one or more errors, and we do not have the resources to maintain past sequences or flag every error in PomBase.</p>

  </div>
</div>
</div>
