### Email: Annotated transcription start and termination sites for fission yeast.

Originally sent by Val Wood to pombelist Fri Feb 4 12:08:34 GMT 2011

A set of genome wide transcription start and termination sites manually    
annotated  by  Lantermann et. al in  PMID: 20118936, based on    
transcriptome data from Dutrow et. al.  PMID: 18641648 have been used to    
create 5' and 3'UTR features for the reference genome, for genes with no    
previously annotated transcript from another source.   

This increases the number of manually annotated UTRs from   
298 to 3605 for 5'   
778 to 3693 for 3'   

Because it appears that transcription start and end can be  fuzzy,     
these features should be considered as an "indicator" of the likely    
extent of  transcription. We will continue to revise the reference    
transcript set as updates are reported, to represent the longest    
discrete transcript for a region as the consensus.   

Data Access   
You can access the transcription start and end from individual  GeneDB    
gene pages for genes of interest using the sequence viewers GBrowse or    
Artemis. The links to these 2 browsers are to the right and left of the    
"context map" on the "Location" section of the gene page.   
(Artemis tips: Use the "Artemis applet" option selected;  If you are    
interested in a specific gene, reduce the upstream and downstream range    
of   to 1000 before clicking "submit"; the page will take a few seconds    
to render the first time in a session)   

You can download this data as part of the complete annotation in the
"rich EMBL files" from here:
ftp://ftp.sanger.ac.uk/pub/yeast/pombe/Chromosome_contigs/ (the files
chromosome*.contig) and view them in Artemis on your desktop (see the
[Artemis FAQ](/faq/there-equivalent-artemis-java-applet-pombase) and
the [Artemis
manual](ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf)
(pdf; Sanger site) for additional information).

Hosting Further Transcriptome Datasets   
Clearly, as more transcriptome data becomes available the community will    
require access to the different datasets, in addition to the consensus    
transcripts. We intend to provide this in the future  through  the new    
PomBase using the Ensembl viewer.   
